<p>
<named-content content-type="genus-species">Y. pseudotuberculosis</named-content> IP31758 (serotype O:1b) was isolated in 1966 from the stools of a patient presenting with scarlet-like fever in the Primorski region of the former Soviet Union and was sent in 1971 to the Institut Pasteur (Paris, France) by Dr. Timofeeva (Antiplague Institute, Irkoutsk, Russia). The strain sequenced and analyzed in this study was subcultured once from that original 1971 stock culture for the purpose of this study. A collection of 46 geographically and phenotypically diverse <named-content content-type="genus-species">Y. pseudotuberculosis</named-content> and <named-content content-type="genus-species">Y. pestis</named-content> strains was screened for the presence or absence of 18 loci specific to <named-content content-type="genus-species">Y. pseudotuberculosis</named-content> IP31758 (<xref ref-type="fig" rid="pgen-0030142-g008">Figure 8</xref>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genomic DNA of <named-content content-type="genus-species">Y. pseudotuberculosis</named-content> IP31758 was subjected to random shotgun sequencing and closure strategies as previously described [<xref ref-type="bibr" rid="pgen-0030142-b103">103</xref>]. Random insert libraries of 3&#8211;5 kb and 10&#8211;12 kb were constructed, and 61,634 high-quality sequences of 837 nt average read length were obtained. A draft genome sequence was assembled using the Celera assembler [<xref ref-type="bibr" rid="pgen-0030142-b104">104</xref>]. An estimate of the copy number of each plasmid was obtained by dividing the coverage depth of the plasmid by the coverage depth of the chromosome. The chromosome and the two plasmids were manually annotated using the TIGR Manatee system (<ext-link ext-link-type="uri" ns0:href="http://manatee.sourceforge.net">http://manatee.sourceforge.net</ext-link>).</p><p>For each of the predicted proteins of <named-content content-type="genus-species">Y. pseudotuberculosis</named-content> IP31758, a BLASTP raw score was obtained for the alignment against itself (REF_SCORE) and the most similar protein (QUE_SCORE) in each of the genomes of <named-content content-type="genus-species">Y. pseudotuberculosis</named-content> IP32953 and <italic>Y. pestis</italic> CO92. These scores were normalized by dividing the QUE_SCORE obtained for each query genome protein by the REF_SCORE. Proteins with a normalized ratio of &lt;0.4 were considered to be nonhomologous. A normalized BLAST score ratio of 0.4 is generally similar to two proteins being 30% identical over their entire length [<xref ref-type="bibr" rid="pgen-0030142-b036">36</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The primer pairs are listed as supporting information in <xref ref-type="supplementary-material" rid="pgen-0030142-st007">Table S7</xref>. PCRs were performed with 1 U of <italic>Taq</italic> polymerase (Roche, <ext-link ext-link-type="uri" ns0:href="http://www.roche.com">http://www.roche.com</ext-link>) in the supplied buffer. PCR amplification reaction mixtures contained 10 &#956;M of each primer and 1 mM dNTPs. The PCR program involved one step at 94 &#176;C for 5 min, followed by 35 cycles of amplification of three steps: (1) 94 &#176;C for 30 s, (2) 60 &#176;C for 30 s, and (3) 72 &#176;C for 7 min. PCR products were maintained at 72 &#176;C for 7 min, separated by gel electrophoresis in 1% agarose gels, and stained with ethidium bromide.</p><p>The chi squares and GC skews were computed according to Nelson et al. [<xref ref-type="bibr" rid="pgen-0030142-b103">103</xref>]. For the chromosomal chi square, a window size of 2 kb and a sliding window of 1 kb was used, while a window size of 1 kb and a sliding window of 0.2 kb were used for the two plasmids. GC skews were calculated using a window size of 1 kb for the chromosome and 0.2 kb for the two plasmids. The whole-genome alignment tool NUCmer [<xref ref-type="bibr" rid="pgen-0030142-b105">105</xref>] was used to calculate the overall gene identities to the respective <named-content content-type="genus-species">Y. pseudotuberculosis</named-content> and <named-content content-type="genus-species">Y. pestis</named-content> strains.</p><p>Each of the 4,164 <named-content content-type="genus-species">Y. pseudotuberculosis</named-content> CDSs (not including the RNA genes) was blasted using BLASTP against the National Center for Biotechnology Information (NCBI) protein database (E-value &gt; 10<sup>&#8722;5</sup>). The BLAST output was parsed using a custom Perl script that recorded the taxonomic affiliation of the BLAST best hit for each protein.</p>