<p>The genotypes of strains and the sources of published alleles are in <xref ref-type="supplementary-material" rid="pgen-0030141-st003">Table S3</xref>. The <italic>rec12&#8211;201::6His-2FLAG</italic> allele, called <italic>rec12-FLAG</italic> in the text, was constructed as plasmid pJF32 (Text S1). The 1.6-kb <italic>Bam</italic>HI fragment of pJF32 was used to transform strain GP5622 (<italic>rec12&#8211;171::ura4<sup>+</sup></italic>) to fluoro-orotic acid-resistance to generate strain GP5960; the chromosomal allele was confirmed by automated nucleotide sequence analysis. The <italic>rec12&#8211;202::6His-2HA</italic> allele, called <italic>rec12-HA</italic> in the text, was similarly constructed as plasmid pJF41 and transferred to the chromosome to generate strain GP5961. Strains with these alleles are recombination proficient (<xref ref-type="supplementary-material" rid="pgen-0030141-st001">Table S1</xref>). Other strains were constructed by standard meiotic crosses.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Cells were grown and induced for meiosis as described [<xref ref-type="bibr" rid="pgen-0030141-b005">5</xref>] and then harvested, washed, and stored at &#8722;20 &#176;C. Chromatin was prepared, sonicated to yield DNA 0.5&#8211;1 kb long as determined by agarose gel electrophoresis (unpublished data), and subjected to immunoprecipitation with anti-FLAG antibody (Sigma, <ext-link ext-link-type="uri" ns0:href="http://www.sigmaaldrich.com">http://www.sigmaaldrich.com</ext-link>) as described [<xref ref-type="bibr" rid="pgen-0030141-b045">45</xref>]. Precipitates (IP) were washed, treated with proteinase K, extracted with phenol-CHCl<sub>3</sub>, and treated with RNase; the residual DNA was purified on a QIAquick column (Qiagen, <ext-link ext-link-type="uri" ns0:href="http://www1.qiagen.com">http://www1.qiagen.com</ext-link>), amplified in the presence of amino-allyl dUTP, and labeled with Cy5 as described [<xref ref-type="bibr" rid="pgen-0030141-b014">14</xref>]. DNA from unfractionated, sonicated chromatin (WCE) was prepared and labeled with Cy3 in parallel. The samples were mixed and hybridized to an Agilent 44-K <named-content content-type="genus-species">S. pombe</named-content> oligonucleotide microarray and analyzed as described [<xref ref-type="bibr" rid="pgen-0030141-b014">14</xref>]. The IP:WCE ratio for each probe on the array (&#8764;41,000) was then calculated after adjustment of the IP values to equalize the IP and WCE medians. Data were analyzed with Excel (Microsoft, <ext-link ext-link-type="uri" ns0:href="http://www.microsoft.com">http://www.microsoft.com</ext-link>), PeakFinder [<xref ref-type="bibr" rid="pgen-0030141-b018">18</xref>], and ChIPOTle [<xref ref-type="bibr" rid="pgen-0030141-b019">19</xref>] software. Peakfinder was run using the haploid and diploid datasets after noise reduction as in <xref ref-type="fig" rid="pgen-0030141-g001">Figure 1</xref>, with linear plotting, Gaussian weighting, a single round using a 13-probe window, right and left minimum smoothed delta = 100, and a threshold of 2,000. ChIPOTle was run on the haploid dataset with noise reduction as in <xref ref-type="supplementary-material" rid="pgen-0030141-sg003">Figure S3</xref>. The Gaussian background setting was used with a window size of 1 kb, a step size of 150 bp, and a cutoff of <italic>p</italic> &lt; 0.001.</p>