<p xmlns:ns0="http://www.w3.org/1999/xlink">We recruited and phenotyped 6,148 individuals, male and female, ages 14&#8211;102 y, from a cluster of four towns in the Lanusei Valley [<xref ref-type="bibr" rid="pgen-0030115-b020">20</xref>]. During physical examination of each individual, a blood sample was collected (for DNA extraction) and anthropometric traits were recorded. Here, we report analyses of hip circumference, weight, and the derived quantity BMI (which is calculated from a combination of height and weight). Genotyping was carried out using the Affymetrix 10K and 500K chips (<ext-link ext-link-type="uri" ns0:href="http://affymetrix.com/">http://affymetrix.com/</ext-link>) using standard protocols. Summary assessments of genotype data quality are provided in the Results section and in <xref ref-type="supplementary-material" rid="pgen-0030115-st001">Table S1</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To follow up on SNPs rs9930506 and rs6602024, we genotyped and examined the association between these two SNPs and BMI, hip circumference, and body weight in the GenNet study. The study comprises 3,467 individuals in total, recruited between 1995 and 2004 (1,101 AA, 839 HA, and 1,496 EA). Individuals were recruited at two field centers: EA were recruited from Tecumseh, Michigan, and AA and HA were recruited from Maywood, Illinois. Participants were recruited from families starting from a proband with high blood pressure. DNA was available for 3,205 individuals (968 AA, 824 HA, and 1471 EA). SNP genotyping was performed using the 5&#8242;-nuclease&#8211;based assay (TaqMan; ABI, <ext-link ext-link-type="uri" ns0:href="http://www.appliedbiosystems.com/">http://www.appliedbiosystems.com/</ext-link>) analyzed on an ABI Prism 7900 Real Time PCR System. Within each ethnic group, genotype completeness rates exceed 98% and there was no evidence for deviation from Hardy&#8211;Weinberg equilibrium (<italic>p</italic> &gt; 0.05).</p><p>To ensure adequate control of type I error rates, we applied an inverse normal transformation to each trait prior to analysis [<xref ref-type="bibr" rid="pgen-0030115-b020">20</xref>]. The inverse normal transformation reduces the impact of outliers and deviations from normality on statistical analysis. The transformation involves ranking all available phenotypes, transforming these ranks into quantiles and, finally, converting the resulting quantiles into normal deviates. We included sex, age, and age<sup>2</sup> as covariates in all analysis. Height was significantly associated with weight and hip circumference and was included as an additional covariate in analysis of those traits. We fitted a simple regression model to each trait and used a variance component approach to account for correlation between different observed phenotypes within each family. For individuals who had genotype data available, we coded genotypes as 0, 1, and 2 (depending on the number of copies of the allele being tested). For individuals with missing genotype data, we used the Lander&#8211;Green algorithm to estimate an expected genotype score (between 0 and 2) for each individual [<xref ref-type="bibr" rid="pgen-0030115-b024">24</xref>]. Briefly, to estimate each genotype score we first calculate the likelihood of the observed genotype data. Then, we instantiate each missing genotype to a specific value and update the pedigree likelihood. The ratio of the two likelihoods gives a posterior probability that the instantiated genotype is true, conditional on all available data. Due to computational constraints, we divided large pedigrees into subunits with &#8220;bit-complexity&#8221; of 19 or less (typically, 20&#8211;25 individuals) before estimating missing genotypes.</p><p>Our analytical approach considers all observed or estimated genotypes (rather than focusing on alleles transmitted from heterozygous parents) and thus is not immune to effects of population stratification. In homogenous populations, this type of analysis is expected to be more powerful [<xref ref-type="bibr" rid="pgen-0030115-b051">51</xref>,<xref ref-type="bibr" rid="pgen-0030115-b052">52</xref>]. To adjust for the effects of population structure and cryptic relatedness among sampled individuals, we used the genomic control method to adjust our test statistics for each trait separately [<xref ref-type="bibr" rid="pgen-0030115-b029">29</xref>]. FDRs were calculated with R's p.adjust() procedure using the method of Benjamini and Hochberg [<xref ref-type="bibr" rid="pgen-0030115-b030">30</xref>]. Since the initial analysis often identified clusters of nearby SNPs that all showed similar levels of association, we also carried out a sequential stepwise analysis. In this analysis, we selected the best SNP for each trait, and then conditioned on it to successively select the next best SNP. This sequential analysis can help identify regions with multiple independent association signals. The stepwise analysis was repeated for five rounds.</p><p>We selected 74 candidate genes previously tested for association with obesity in humans [<xref ref-type="bibr" rid="pgen-0030115-b053">53</xref>]. For each gene, we first evaluated the ability of the Affymetrix SNPs to tag common SNPs (MAF &gt; 0.05) within +/&#8722; 5 kb of the gene (<italic>r</italic>
<sup>2</sup> &gt; 0.50 or <italic>r</italic>
<sup>2</sup> &gt; 0.80) using the HapMap CEU database [<xref ref-type="bibr" rid="pgen-0030115-b054">54</xref>]. We then evaluated evidence for association using all Affymetrix SNPs within each gene as well as neighboring Affymetrix SNPs that could be used to improve coverage (<italic>r</italic>
<sup>2</sup> &gt; 0.5). For each gene, we report coverage statistics as well as the SNP that showed strongest evidence for association.</p><p>We selected 74 genes that were previously targeted in associations studies aiming to identify genetic determinants of obesity in humans [<xref ref-type="bibr" rid="pgen-0030115-b053">53</xref>]: <italic>ACE, ACTN, ADIPOQ, ADIPOR1, ADIPOR2, ADRB1, ADRB2, AGER, AHSG, APOA2, APOA4, APOA5, AR, BDNF, CASQ1, COL1A1, COMT, CRP, CYP11B2, DIO1, ENPP1, ESR1, ESR2, FABP2, FOXC2, GAD2, GFPT1, GHRHR, GNAS, GNB3, GPR40, H6PD, HSD11B1, HTR2C, ICAM1, IGF1, IGF2, IL6, IL6R, KCNJ11, KL, LEP, LEPR, LIPC, LPL, LTA, MC4R, MCHR1, MKKS, MTHFR, MTTP, NMB, NOS3, NPY, NPY2R, NR0B2, NTRK2, PARD6A, PLIN, PPARG, PPARGC1A, PRDM2, PTPN1, PYY, RETN, SCD, SELE, SERPINE1, TAS2R38, TNF, UCP1, UCP2, UCP3</italic>, and <italic>VDR</italic>. We did not consider genes associated with drug-induced body weight gain or mitochondrial genes [<xref ref-type="bibr" rid="pgen-0030115-b053">53</xref>].</p><p>The following genes have previously been investigated for their role in obesity and related traits but are not well tagged by SNPs in the Affymetrix array: <italic>ADRB3, DRD4, INS,</italic> and <italic>APOE</italic>.</p>