<p>The three BAC libraries (30&#215; total coverage) used in this study were HindIII (136-kb average insert size), EcoRI (163-kb), and MboI (167-kb) libraries [<xref ref-type="bibr" rid="pgen-0030123-b024">24</xref>,<xref ref-type="bibr" rid="pgen-0030123-b025">25</xref>], with 14.2&#215;, 7.6&#215;, and 7.8&#215; coverage respectively. The use of three BAC libraries helped to ensure the maize genome was fully represented. Maize cultivar B73 was selected because of its widespread use in breeding and because it is one of the parents of a public high-resolution genetic reference mapping population (IBM [<xref ref-type="bibr" rid="pgen-0030123-b043">43</xref>]).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">BAC clones starting with a &#8220;b&#8221; are from the B73 HindIII library (ZMMBBb). BAC clones starting with a &#8220;c&#8221; are from the B73 EcoRI and MboI libraries known as CHORI-201 (Clemson University Genomics Institute/Arizona Genomics Institute [CUGI/AGI] name ZMMBBc). In the c library, the first half (288 384-well plates) was from the EcoR1 digest, and the second half was from the MboI digest. Clones ending in &#8220;sd1&#8221; are sequenced BACs downloaded from GenBank (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/Genbank">http://www.ncbi.nlm.nih.gov/Genbank</ext-link>) digested with HindIII in silico [<xref ref-type="bibr" rid="pgen-0030123-b044">44</xref>]. If the sequences were longer than 150 kb, they were artificially fragmented into multiple overlapping clones, i.e., sd1, sd2, etc. and then digested with HindIII in silico. For the agarose method [<xref ref-type="bibr" rid="pgen-0030123-b029">29</xref>,<xref ref-type="bibr" rid="pgen-0030123-b045">45</xref>], all BAC DNAs were digested with the restriction enzyme HindIII, run on high-resolution 1.0% agarose gels, stained with SYBR Green (FMC BioProducts, <ext-link ext-link-type="uri" ns0:href="http://www.fmc.com">http://www.fmc.com</ext-link>), and the Image software [<xref ref-type="bibr" rid="pgen-0030123-b046">46</xref>] was used for interactive band calling.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For the HICF method [<xref ref-type="bibr" rid="pgen-0030123-b027">27</xref>,<xref ref-type="bibr" rid="pgen-0030123-b030">30</xref>,<xref ref-type="bibr" rid="pgen-0030123-b031">31</xref>], BAC DNAs were digested with type IIS restriction enzyme EarI, in which the resulting ends were tagged with fluorescently labeled ddNTPs and TaqI, which is to reduce the fragment sizes of the EarI digestion. The first base in a two-base overhang from TaqI is G, therefore ddCTP was not used in the end-labeling reaction because the majority of fragments would contain at least one TaqI site. Thus we only used C, T, and A overhangs for end labeling by using the fluorescent dyes ddGTP (blue), ddATP (green), and ddTTP (yellow), respectively. Red dye (GeneScan-500 ROX, Applied Biosystems, <ext-link ext-link-type="uri" ns0:href="http://www.appliedbiosystems.com">http://www.appliedbiosystems.com</ext-link>) was employed as an internal size standard, which was run in each capillary. Reaction products were resolved on ABI3700 automated DNA sequencers. ABI GeneScan version 3.7.1 software (Applied Biosystems) was used for band detection and extraction. Only fragments ranging in size from 75 to 500 bp were used for HICF assembly. Because FPC can only input one set of band values for each clone, we used the technique of Ding et al. [<xref ref-type="bibr" rid="pgen-0030123-b031">31</xref>] to convert the size/color pairs generated by HICF. Each band was multiplied by 20, and the fractional part discarded. An offset was then added to each band as follows: 0 to blue, 10,000 to green, and 20,000 to yellow bands. Since we used fragments in the range 75&#8211;500 bp, the result of this conversion was bands occupying the ranges 1,500&#8211;10,000, 11,500&#8211;20,000, and 21,500&#8211;30,000.</p><p>FPC software [<xref ref-type="bibr" rid="pgen-0030123-b026">26</xref>] was used for fingerprint assembly. To automatically build the agarose map, two clones assemble together if they share at least <italic>N</italic> markers and have a Sulston score of less than <italic>M,</italic> using the following (<italic>N, M</italic>) pairs: (0, 1e&#8722;12), (1, 1e&#8722;11), (2, 1e&#8722;10), or (3, 1e&#8722;09). False-positive overlaps generally result in a stack of clones within the contig whose bands do not align well with the underlying consensus band map; these clones are called Q (questionable) clones [<xref ref-type="bibr" rid="pgen-0030123-b047">47</xref>]. The FPC DQer function reassembles all contigs that have &gt;5 Q clones using a more stringent cutoff, such as 1e&#8722;13, 1e&#8722;14, or 1e&#8722;15. If the more stringent cutoff breaks the contig into multiple contigs, the function tries to join the subcontigs by their end clones; if this is not possible, it creates new contigs from the subcontigs. If it cannot reduce the number of Qs by a 1e&#8722;15, it leaves the contig alone. Generation of the maize FPC map was performed in stages including manual editing that utilized the agarose assembly of 4,518 contigs and all genetic and overgo markers. The addition of these markers allowed us to reduce the Sulston score for contig merging. Singletons were added in to merge contigs when they overlapped with the ends of two contigs. The total contig number decreased from initial build of 4,518 to 2,085, and the assembly was publicly released in February 2004 (<xref ref-type="supplementary-material" rid="pgen-0030123-st001">Table S1</xref>).</p><p>To automatically build the HICF map, possible contaminated clones were screened for same-plate overlaps at 1e&#8722;45, followed by removal of all remaining clones over 175 bands. The initial contigs were built at 1e&#8722;70 at the tolerance of 4, then the DQer reassembled all contigs that had &gt;15% Q clones. The end-merging function was run at 1e&#8722;61, 1e&#8722;52, 1e&#8722;45, 1e&#8722;40, and 1e&#8722;21, where the end-merging function checks the clones at the ends of contigs, and if at least two pairs of clones overlap based on the given cutoff, the contigs are merged. Singletons were placed at 1e&#8722;43 to their best location, before the final end-merge.</p><p>The agarose map was edited to split contigs with false-positive joins and to merge contigs that had overlapping end clones that were too small to be detected by FPC. First, manual end joining was performed if the end clones from two contigs had a Sulston score of <italic>N</italic> and at least <italic>M</italic> shared markers, where the valid (<italic>N, M</italic>) pairs were (0, 1e&#8722;10), (1, 1e&#8722;09), (2, 1e&#8722;08), or (3, 1e&#8722;07). We also extensively used the HICF map, marker information, and maize&#8211;rice synteny for contig merging. A new marker type called HICF was created, which was used to link each pair of clones that are bridged by an HICF clone that did not occur in the agarose map. We know that the pair of clones lie close to each other (within 150 kb) and probably overlap, but it cannot be confirmed. The agarose method produced an average of 30 bands for each maize clone (band size range from 1 to 23 kb), while the HICF method generated an average of 107 bands for each clone (size range 75 bp&#8211;500 bp [<xref ref-type="bibr" rid="pgen-0030123-b027">27</xref>]).</p><p>The metric of a FPC contig is the consensus band (CB) unit. If the length of a contig is <italic>N</italic> CB units, then its approximate length in base pairs is <italic>N</italic> &#215; 4,900. The 4,900 was calculated as the average size restriction fragment as follows: A total of 20 completely sequenced maize BACs that had corresponding fingerprints were downloaded from GenBank. The average size of all sequenced BACs was divided by the average number of fingerprinted bands, which resulted in an average band size of 4,900 bp. Since the number of bands was taken from the real fingerprint and not from the simulated fingerprint, this average size took into account missing bands. This estimation is similar to that in the rice physical map [<xref ref-type="bibr" rid="pgen-0030123-b045">45</xref>].</p><p>As an essential component to establish an integrated map for the maize genome, a saturated and evenly distributed genetic map was developed to anchor and fully orient physical contigs along the chromosome. We built a dense genetic framework using the IBM mapping population [<xref ref-type="bibr" rid="pgen-0030123-b043">43</xref>]. The inbred parents of IBM are highly polymorphic and the progeny experienced four rounds of intermating, which makes IBM nearly four times expanded in size and 18-fold higher in resolution than previous public standard populations [<xref ref-type="bibr" rid="pgen-0030123-b048">48</xref>]. The order of loci on the IBM map has proven to match closely that developed by fingerprinting and BAC assembly.</p><p>The number of genetically mapped markers was increased by intercalating locations of additional markers into the IBM, constructing &#8220;neighbors'&#8221; maps (<xref ref-type="table" rid="pgen-0030123-t001">Table 1</xref>), i.e., by importing markers from maps of other mapping populations and interpolating for genetic coordinates [<xref ref-type="bibr" rid="pgen-0030123-b049">49</xref>]. Successive iterations of neighbors' maps have incorporated new map data from other projects and have refined orders according to FPC results. Maps have been updated regularly in MaizeGDB [<xref ref-type="bibr" rid="pgen-0030123-b050">50</xref>].</p><p>We used probe hybridization on BAC filters, primer amplification methods on 6-D BAC pools, and overgo probe hybridizations to integrate genetically mapped markers with the related BAC clones in the maize physical map. Since genetic markers can hybridize to multiple contigs because of paralogous sequences, it was necessary to distinguish between marked contigs that could be genetically placed and all other copies. To this end, anchoring markers retained their original name, and any unanchored copies were assigned a suffix (.A). Markers included RFLPs, SSRs, ESTs, SNPs, and InDels. Filter hybridization has low throughput, so we used only 90 RFLP probes on filters [<xref ref-type="bibr" rid="pgen-0030123-b024">24</xref>] and carried out further integrating with two high-throughput methods.</p><p>Hybridization of overgos was conducted on four filters of a 6 &#215; 6 grid with a total of 165,888 BACs, half from each of the HindIII and EcoRI libraries. The 40-bp overgo probes were designed from unigenes of the DuPont/MMP/Incyte Genomics partnership and were screened with a maize repeat database to ensure that each probe was low copy. The overgos were classified into five marker types where the prefix identifies the marker type: (1) the prefix CL identifies clusters assembled from public EST sequences; (2) PCO represents public sequences combined with DuPont sequences (Unigene Consensus deposited in GenBank); (3) dd signifies anonymous clusters assembled by DuPont (sequences are not available); (4) si represents public singletons that did not cluster with DuPont sequences; and (5) SOG markers are overgos from the Paterson lab, derived from probes that have been mapped in sorghum and other grasses. Except for the SOG markers, which were designed for sequence conservation, all other overgos were designed for their diversity. We employed a 2-D 24 &#215; 24 pool strategy for overgo hybridization and scored dual grid spots as positive clones [<xref ref-type="bibr" rid="pgen-0030123-b028">28</xref>].</p><p>PCR methods were used to screen BACs with SSR or genomic sequence markers. A 6-D pool strategy [<xref ref-type="bibr" rid="pgen-0030123-b051">51</xref>] was used for library screening to construct 288 pools from a 48 &#215; 48 &#215; 48 block for a total of 82,944 BACs in 216 384-well plates from the HindIII library. Primers for selected sequences were applied to pools, and the data were deconvoluted using criteria that assured disambiguation by multiple confirmation.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used the SyMAP [<xref ref-type="bibr" rid="pgen-0030123-b037">37</xref>] to compute and view the syntenic blocks. The algorithm first computed anchors from the alignment of the maize markers and BES sequences to the rice genomic sequence and then calculated chains of anchors using dynamic programming and merged chains into blocks. The graphical results from a versatile CGI/HTML and Java Display allow users to observe synteny from different views, including dotplot, genome blocks, chromosome blocks, close-up chromosomes, close-up contigs, and table view. The results can be interactively browsed at <ext-link ext-link-type="uri" ns0:href="http://www.agcol.arizona.edu/symap">http://www.agcol.arizona.edu/symap</ext-link>.</p>