<p>The study population consisted of 2,685 locally available randomly selected healthy subjects and transplant donors. All research samples and data were collected according to the subjects' guidelines and protocols of the local institutional review boards. Individuals with an unknown ethnical background were excluded from the analyses.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">DNA was isolated using the local standard procedures used for HLA typing. Genotyping of ten autosomally encoded minor H antigens and of HY was performed using the PCR-SSP technique (Invitrogen, <ext-link ext-link-type="uri" ns0:href="http://www.invitrogen.com">http://www.invitrogen.com</ext-link>). The details of the minor H antigen typing methodology have been described before [<xref ref-type="bibr" rid="pgen-0030103-b037">37</xref>]. Previously typed cells from the Centre d'Etude du Polymorphisme Humain-Human Genome Diversity Project Cell Line Panel (CEPH panel) and from the reference DNA samples from the University of California, Los Angeles, International DNA Exchange Program were used as controls. All PCR-SSP samples were analyzed on agarose gel. If the internal control PCRs failed for one of the two allele of a particular minor H antigen, the typing for that minor H antigen was marked as incomplete and data were excluded from the analyses. The alleles of each minor H antigen were reported as the amino acid resulting from the relevant SNP. Typing data and sample parameters were submitted to a central online database (<ext-link ext-link-type="uri" ns0:href="http://www.lumc.nl/ihiw14">http://www.lumc.nl/ihiw14</ext-link>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Typing data were analyzed at three levels: (1) the allele frequency, (2) genotype frequency, and (3) phenotype frequencies. Allele and genotype frequencies except those of UGT2B17 were calculated by direct counting. Since the minor H antigen UGTB17 is the result of gene deletion, the applied methodology did not allow detection of UGT2B17 heterozygosity. Therefore the UGT2B17 allele frequencies were estimated as <inline-formula id="pgen-0030103-ex001"><inline-graphic mimetype="image" ns0:href="pgen.0030103.ex001.jpg" /></inline-formula>
for the nonimmunogenic allele and 1 &#8722; <inline-formula id="pgen-0030103-ex002"><inline-graphic mimetype="image" ns0:href="pgen.0030103.ex002.jpg" /></inline-formula>
for the immunogenic allele, where &#8220;<italic>aa</italic>&#8221; is the frequency of the nonimmunogenic phenotype. The heterozygosity frequencies (Aa) for this minor H antigen were estimated by 2 &#215; (1 &#8722; <inline-formula id="pgen-0030103-ex003"><inline-graphic mimetype="image" ns0:href="pgen.0030103.ex003.jpg" /></inline-formula>
) &#215; (<inline-formula id="pgen-0030103-ex004"><inline-graphic mimetype="image" ns0:href="pgen.0030103.ex004.jpg" /></inline-formula>
), assuming a Hardy-Weinberg equilibrium.
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Observed genotype frequencies were tested against Hardy-Weinberg equilibrium. The phenotype frequencies have been deduced from the genotype frequencies. Individuals who were homozygous or heterozygous positive for the immunogenic allele were regarded as phenotypically positive. Individuals carrying two nonimmunogenic alleles were marked as phenotypically negative. Since the HB-1 minor H antigen has been demonstrated to be immunogenic in either direction, phenotype frequencies have been calculated separately for the HB-1<sup>H</sup> and the HB-1<sup>Y</sup> allele as immunogenic alleles. The statistical significance of differences in allele, genotype, and phenotype frequencies between the various populations and geographic regions were determined by Chi-square analysis. The SPSS statistical software system (SPSS 11.0, <ext-link ext-link-type="uri" ns0:href="http://www.spss.com">http://www.spss.com</ext-link>) was used for these analyses. <italic>p-</italic>values lower than 0.05 were considered statistically significant.</p><p>Linkage of the alleles of ACC-1 and ACC-2 was analyzed with the case-control haplotype inference (Chaplin) software [<xref ref-type="bibr" rid="pgen-0030103-b034">34</xref>]. Individuals who were heterozygous for both ACC-1 and ACC-2 were classified as noninformative and excluded from the analysis.</p><p>The disparity rate is defined as the chance of a putative transplant recipient with a minor H antigen immunogenic phenotype having an HLA-identical related or HLA-matched unrelated stem cell donor being homozygous for the nonimmunogenic allele of the same minor H antigen encoding gene. In the case of a putative HLA-matched unrelated donor selection, the disparity rate is the frequency of homozygous immunogenic (AA) and heterozygous (Aa) individuals multiplied by the frequency of homozygous nonimmunogenic individuals (aa): ((<italic>AA</italic>) + (<italic>Aa</italic>)) &#215; (<italic>aa</italic>). Sib transplantation disparity rates were estimated by 3/16(<italic>Aa</italic>)<sup>2</sup> + 1/2(<italic>Aa</italic>)(<italic>aa</italic>). For both related and unrelated situations, the rates were multiplied by the phenotype frequency of the relevant HLA restriction molecule. To investigate the level of disparity within each population, two-tailed Wilcoxon signed rank tests were performed.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Spatial frequency distribution maps for each of the immunogenic minor H antigen phenotype frequencies were constructed for the White population samples using the Kriging procedure [<xref ref-type="bibr" rid="pgen-0030103-b038">38</xref>] with the Surfer 8 software (Golden Software, <ext-link ext-link-type="uri" ns0:href="http://www.goldensoftware.com">http://www.goldensoftware.com</ext-link>). For these analyses the geographical area between 29.000/&#8722;28.000 (latitude/longitude) and 72.000/49.000, covering the European continent, was split into a 1,000 &#215; 550 squares grid using the standard software settings. Cartesian coordinates of the participating institutes were obtained from the Getty Thesaurus of Geographic Names Online (<ext-link ext-link-type="uri" ns0:href="http://www.getty.edu/research/conducting_research/vocabularies/tgn">http://www.getty.edu/research/conducting_research/vocabularies/tgn</ext-link>).</p>