<p xmlns:ns0="http://www.w3.org/1999/xlink">BHK<sub>21</sub> (American Type Culture Collection, <ext-link ext-link-type="uri" ns0:href="http://www.atcc.org">http://www.atcc.org</ext-link>) were cultivated at 37 &#176;C, 5% CO<sub>2</sub>, in 100 cm<sup>2</sup> plates under 12 mL of Dulbecco modified Eagle's minimum essential medium (DMEM) supplemented with 10% fetal calf serum, and passaged upon confluence. The VSV infectious cDNA used to obtain population A was kindly provided by G. T. W. Wertz, University of Alabama, United States of America [<xref ref-type="bibr" rid="pgen-0030093-b023">23</xref>].</p><p xmlns:ns0="http://www.w3.org/1999/xlink">A 100-&#956;L volume of an appropriate dilution was added to 60 mm culture plates and cells were topped with DMEM medium containing 0.4% agarose. Monolayers were stained at 18&#8211;22 h post-inoculation (hpi) with a solution of 2% crystal violet (Sigma, <ext-link ext-link-type="uri" ns0:href="http://www.sigmaaldrich.com">http://www.sigmaaldrich.com</ext-link>) in 10% formaldehyde (Panreac, <ext-link ext-link-type="uri" ns0:href="http://www.panreac.com/new/ing/menu.htm">http://www.panreac.com/new/ing/menu.htm</ext-link>).</p><p>Approximately <italic>N</italic>
<sub>0</sub>
<italic>=</italic> 5 &#215; 10<sup>3</sup> pfu were inoculated to <italic>C =</italic> 10<sup>6</sup> BHK<sub>21</sub> cells in 25 cm<sup>2</sup> flasks, incubated for 24 h, and the supernatant was used to seed a new infection passage under the same conditions, up to 25 passages. Viral population sizes at the end of each infection passages were approximately <italic>N<sub>f</sub> =</italic> 5 &#215; 10<sup>9</sup> pfu. The number of viral infectious cycles (generations) per passage was estimated from <italic>N</italic>
<sub>0</sub>, <italic>N<sub>f</sub></italic>, and <italic>C</italic> as previously described [<xref ref-type="bibr" rid="pgen-0030093-b039">39</xref>], giving ~4 generations per passage.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Viral RNA was extracted from the supernatant of the infected cultures using the High Pure Viral Nucleic Acid Kit following the manufacturer instructions (Roche, <ext-link ext-link-type="uri" ns0:href="http://www.roche.com">http://www.roche.com</ext-link>). The anti-genomic VSV cDNA was synthesized using the reverse transcriptase of the Moloney murine leukemia virus (Promega, <ext-link ext-link-type="uri" ns0:href="http://www.promega.com">http://www.promega.com</ext-link>) plus a battery of random hexamers (Promega). The genomic region containing the original mutations was amplified by PCR using <italic>Taq</italic> polymerase (Amersham, <ext-link ext-link-type="uri" ns0:href="http://www.gehealthcare.com">http://www.gehealthcare.com</ext-link>) and specific primers. Sequencing was carried out using ABI PRISM BigDye Terminator v3.0 Ready Reaction Cycle Sequencing KIT (Applied Biosystems, <ext-link ext-link-type="uri" ns0:href="http://www.appliedbiosystems.com">http://www.appliedbiosystems.com</ext-link>) on an ABI 3700 automated sequencer. Sequences were visualized and edited with the Staden software package (<ext-link ext-link-type="uri" ns0:href="http://staden.sourceforge.net">http://staden.sourceforge.net</ext-link>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">RNA viral extraction, RT-PCRs, sequencing, and editing of sequences were performed as described above. A first round of PCR was carried out using Pfu polymerase (Amersham) and specific primers [<xref ref-type="bibr" rid="pgen-0030093-b040">40</xref>]. Amplified DNA products of each region were purified with High Pure PCR product Purification Kit (Roche) and directly cloned into EcoRV-digested pBluescript II SK (+) phagemid (Stratagene, <ext-link ext-link-type="uri" ns0:href="http://www.stratagene.com">http://www.stratagene.com</ext-link>). A second round of PCR was then carried out by adding a single transformed bacterial colony in each PCR tube and using Taq polymerase. Vector-based primers KS and SK (Stratagene) were used in this second round of PCR, as well as for the subsequent sequencing. Before sequencing, amplified DNA was purified using a PCR clean-up kit (Macherey-Nagel, <ext-link ext-link-type="uri" ns0:href="https://www.macherey-nagel.com">https://www.macherey-nagel.com</ext-link>). Plasmid DNA was purified with High Pure Plasmid Isolation Kit (Roche) and clones were sequenced using vector-based primers KS and SK (Stratagene). EMBL accession numbers of the obtained sequences for nonmutagenized populations and for 5-FU mutagenized populations are in the Accession Numbers list in the Supporting Information section of this paper.</p><p>Following protocols established in previous work [<xref ref-type="bibr" rid="pgen-0030093-b041">41</xref>&#8211;<xref ref-type="bibr" rid="pgen-0030093-b043">43</xref>], we used growth rate assays to estimate fitness. We seeded ~5 &#215; 10<sup>3</sup> pfu of each population into ~10<sup>5</sup> BHK<sub>21</sub> cells and incubated the culture until the population grew up to a titer of ~10<sup>7</sup> pfu/mL (i.e., 7&#8211;8 hpi). From final and initial titers, the growth rate (<italic>r</italic>) was calculated as the slope of log-titer regression against time (hpi). We defined fitness (<italic>W</italic>) as the number of descendants per individual per hour, i.e., <italic>W = e<sup>r</sup></italic> &#8722; 1. Fitness assays were done in triplicate.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Viruses were plated as detailed above, paying care to put less than 50 pfu per plate to avoid plaque overlapping. All platings were done in a single block and the same overlay medium batch was used for both populations and staining was done at 24 hpi. Pictures of the plates were taken with a 5 megapixel Canon PowerShot G5 digital camera (<ext-link ext-link-type="uri" ns0:href="http://www.canon.com">http://www.canon.com</ext-link>) and image analysis was done with AnalySIS v.3.2 software (Soft Imaging System, <ext-link ext-link-type="uri" ns0:href="http://www.soft-imaging.net">http://www.soft-imaging.net</ext-link>). The vast majority of plaques were automatically identified, whereas the rest were manually delineated in a zoomed image. After defining plaques as single objects, we automatically obtained their surface area in pixels, <italic>S</italic>.</p><p>To calibrate the relationship between plaque size and fitness, we selected 12 clones from each population that widely varied in fitness and, for each, we determined the average plaque size from four independent plates. Since we had no evidence that fitness and plaque area were linearly related, we performed a log&#8211;log regression to obtain a calibration line of the form log<italic>W = p + m</italic>log<italic>S</italic>. For population A, we obtained <italic>m =</italic> 1.907 &#177; 0.318 and <italic>p =</italic> &#8722;13.785 &#177; 2.246 (<italic>r =</italic> 0.884), whereas for population B, we obtained <italic>m =</italic> 0.927 &#177; 0.096 and <italic>p =</italic> &#8722;5.874 &#177; 0.678 (<italic>r =</italic> 0.950). The calibration line was used to transform each observed plaque size into a predicted fitness.</p><p>Although the calibrations described above seemed relatively accurate, it is unavoidable that there was some degree of uncertainty and consequently no guarantee that no bias was introduced. We therefore carried out a second analysis that bypassed the calibration step. To achieve this, we simply assumed that the number of lysed cells was proportional to the number of viruses produced. First, we estimated the number of lysed cells on a plaque, <italic>D,</italic> by dividing its pixel surface area per the total pixel surface area of the 60 mm plate and multiplying per the total number of cells on a plate, which was obtained by counting cells in an hemocytometer (Neubauer). Second, we estimated the average number of viruses produced per cell (<italic>K</italic>) by titrating a fully lysed plate. This was done separately for populations A and B, yielding <italic>K =</italic> 2,204 &#177; 32 and <italic>K =</italic> 2,712 &#177; 82 respectively. Using these estimations, plaque sizes were transformed into predicted fitness as <italic>W<sub>pred</sub> =</italic> (<italic>KD</italic>)<sup>1/24</sup> &#8722; 1. The results obtained with this latter fitness estimation method (unpublished data) were consistent with those obtained using the calibration line (population A showed higher maximum fitness and higher variance).</p><p>Standard growth rate assays were performed as described above except that in competitions between populations (<xref ref-type="fig" rid="pgen-0030093-g003">Figure 3</xref>), only 100 pfu from each competitor were inoculated to minimize the effect of rare beneficial mutations (see below). Viruses from the two genotypes were inoculated into the same well. Genotype B carried a monoclonal antibody resistance marker that allowed us to estimate the titer of each population in the mixture by plating in presence and absence of antibody. Samples were taken at 0, 7, 8, 9, 10, 11, 12, 15, 20, 25, and 30 hpi for competition with no mutagen. For competitions in the presence of 5-FU or 5-AzC, cells had been pretreated with the indicated 5-FU concentration 12 h prior infection. Samples were taken at 0, 15, 20, 25, 30, 35, 39, and 49 hpi for 20 and 40 &#956;g/mL 5-FU and at 0, 20, 25, 30, 35, 39, and 49 hpi for 80 &#956;g/mL 5-FU. Samples were taken at 0, 10, 12, 15, 24, 28, 32, 38, and 48 hpi for 5 and 10&#956;g/mL 5-AzC, and at 0, 24, 28, 32, 38, and 48 hpi for 20 &#956;g/mL 5-AzC. Growth rates for each population were calculated as the slope of the log-titer against hpi during the exponential growth phase. For competitions between populations, 12 replicates of the competition assay were done for each different mutagen dose, whereas for competitions between clones three replicates were done for each dose and each of the four clone pairs.</p><p>We reduced the initial inoculum to ~100 pfu of each population to minimize the effect of rare beneficial mutations in the outcome of the competition. Since 1,000 clones were sampled during the plaque size fitness screening, the probability that beneficial mutations at a frequency <italic>f</italic> in the population went unnoticed during this screening is given by the binomial distribution function, <italic>Bi</italic>(0, 1,000, <italic>f</italic>). For each competition assay, the probability that these beneficial mutations were present in the inoculum is 1 &#8722; <italic>Bi</italic>(0, 100, <italic>f</italic>). Therefore, for each replicate of the competition assay, the joint probability that beneficial mutations that had been missed in the plaque size screening and were present in the competition assays is <italic>p = Bi</italic>(0, 1,000, <italic>f</italic>) (1 &#8722; <italic>Bi</italic>(0, 100, <italic>f</italic>)). For all values of <italic>f, p</italic> &#8804; 0.035 for each replica of the competition assay, thus making it very unlikely that rare beneficial mutations had affected the results of the competition after 12 replicates.</p><p>The expected outcome of a competition between population A and B, based on plaque sizes, was obtained as follows. First, to mimic the inoculum size of the real competition assays, we randomly sampled a subset of 100 individuals from each population and we got their expected progeny per hour, which is equal to the predicted fitness. We then let the simulated growth proceed until the total population size was similar to that observed at the last time point of real competitions. The expected winner of the competition was the one with the larger progeny number at the final time point. The random sampling and the simulation were repeated 1,000 times.</p><p>Two random clones from populations A and B were picked from a 60 mm plate and stored at &#8722;80 &#176;C as ancestors. Twenty-four mutation accumulation lines were founded from each ancestor by randomly picking 24 lysis plaques. Following previous plaque-to-plaque experimental designs [<xref ref-type="bibr" rid="pgen-0030093-b044">44</xref>], for each lineage, viruses were plated and at 24 hpi, a lysis plaque was randomly sampled, resuspended in DMEM medium, and directly plated onto a fresh monolayer. This protocol was pursued on a daily basis for12 passages. The number of generations elapsed during each plaque-to-plaque passage was estimated to be approximately two. To obtain enough viruses for fitness assays, the ancestors and the final derived clones were given a single passage in a 96-well plate containing ~10<sup>4</sup> BHK<sub>21</sub>. Fitness assays were performed as detailed above simultaneously for the ancestors and the derived lines in three experimental blocks. In each block, the 24 derived clones of each population were assayed once, whereas the ancestors were assayed six times.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The expected change in log-fitness after <italic>t</italic> generations (<italic>t =</italic> 24) is &#916;log<italic>W =</italic> log<italic>W<sub>t</sub></italic> &#8722; log<italic>W</italic>
<sub>0</sub>
<italic>= U<sub>d</sub>tE</italic>[log(1 &#8722; <italic>s</italic>)]. The expected genetic variance for log-fitness among lineages is &#963;<italic><sub>G</sub></italic>
<sup>2</sup>(log<italic>W<sub>t</sub></italic>) &#8776; <italic>U<sub>d</sub>tE</italic>[(log(1 &#8722; <italic>s</italic>))<sup>2</sup>], and the total variance is &#963;<italic><sub>T</sub></italic>
<sup>2</sup>(log<italic>W</italic>) = &#963;<italic><sub>E</sub></italic>
<sup>2</sup>(log<italic>W</italic>) + &#963;<italic><sub>G</sub></italic>
<sup>2</sup>(log<italic>W</italic>) where sub-index <italic>E</italic> refers to environmental variance. Since genetic variance is null for the ancestor and all fitness assays were performed in the same environmental conditions, &#963;<italic><sub>E</sub></italic>
<sup>2</sup>(log<italic>W<sub>t</sub></italic>) = &#963;<italic><sub>E</sub></italic>
<sup>2</sup>(log<italic>W</italic>
<sub>0</sub>) = &#963;<italic><sub>T</sub></italic>
<sup>2</sup>(log<italic>W</italic>
<sub>0</sub>), and therefore &#963;<italic><sub>G</sub></italic>
<sup>2</sup>(log<italic>W<sub>t</sub></italic>) = &#963;<italic><sub>T</sub></italic>
<sup>2</sup>(log<italic>W<sub>t</sub></italic>) &#8722; &#963;<italic><sub>T</sub></italic>
<sup>2</sup>(log<italic>W</italic>
<sub>0</sub>) = &#916;&#963;<italic><sub>T</sub></italic>
<sup>2</sup>(log<italic>W</italic>). It follows that
<disp-formula id="pgen-0030093-e001"><graphic mimetype="image" position="anchor" ns0:href="pgen.0030093.e001.jpg" /></disp-formula>and that
<disp-formula id="pgen-0030093-e002"><graphic mimetype="image" position="anchor" ns0:href="pgen.0030093.e002.jpg" /></disp-formula>where <italic>&#952;</italic> is the coefficient of variation (standard deviation to mean ratio) of log <italic>W</italic> associated to single deleterious mutations. (1 + <italic>&#952;</italic>
<sup>2</sup>)<italic>E</italic>[log(1 &#8722; <italic>s</italic>)] and <italic>U<sub>d</sub></italic>/(1 + <italic>&#952;</italic>
<sup>2</sup>) were directly estimated from the data.
</p><p>For an exponential distribution (a simple and relatively accurate model for describing fitness effects associated to single mutations), <italic>&#952; =</italic> 1. Nonetheless, the actual distribution of mutational effects might have a heavier tail and thus a <italic>&#952; &gt;</italic> 1. A previous fitness dataset of 28 nonlethal random mutants of VSV [<xref ref-type="bibr" rid="pgen-0030093-b042">42</xref>] gave the empirical estimation <italic>&#952; =</italic> 1.598. Using this figure, we obtained estimates of <italic>U<sub>d</sub></italic> and <italic>E</italic>[log(1 &#8722; <italic>s</italic>)] and then, of <italic>s</italic>. For A, <italic>&#952; =</italic> 1.598 should be an accurate estimation of the true <italic>&#952;</italic>-value, because it was obtained for the above-mentioned full-length infectious cDNA clone. It is possible, however, that, due to its higher mutational robustness, B showed a lower <italic>&#952;</italic>-value. This potential bias could account for the slightly higher <italic>U<sub>d</sub></italic> estimate in B, but it could not account for the nearly one order of magnitude difference in <italic>s</italic>-values (in the extreme case <italic>&#952;<sub>B</sub> =</italic> 0, the true <italic>U<sub>d B</sub></italic> would be 2.598 times lower than our estimate and the true <italic>s<sub>B</sub></italic> 2.598 times higher).</p><p>To address whether differences between <italic>U<sub>d</sub></italic> and <italic>s</italic> between A and B were statistically significant, we generated 1,000 bootstrap pseudo-replicates from both the 24 averaged fitness values of the derived clones and the six averaged fitness values of the ancestors. After obtaining the corresponding 1,000 pseudo-replicates of <italic>U<sub>d</sub></italic>, we counted the number of times <italic>U<sub>d</sub></italic> was larger for A versus B, and the same for <italic>s</italic>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Parameters <italic>U<sub>d</sub></italic> and <italic>s</italic> were also estimated using the maximum likelihood approach implemented in the program MLGENOMEU (<ext-link ext-link-type="uri" ns0:href="http://homepages.ed.ac.uk/eang33/mlgenomeu/mlginstructions.html">http://homepages.ed.ac.uk/eang33/mlgenomeu/mlginstructions.html</ext-link>), in which a Gamma distribution of the form <italic>Ga</italic>(<italic>s</italic>) = &#945;<italic><sup>&#946;</sup>s<sup>&#946;</sup></italic>
<sup>&#8722; 1</sup>
<italic>e</italic> &#8722; <sup>&#945;<italic>s</italic></sup>/&#915;(<italic>&#946;</italic>) is used to describe the distribution of mutational effects (<italic>s</italic> =<italic>&#946;/&#945;</italic>) [<xref ref-type="bibr" rid="pgen-0030093-b028">28</xref>]. Following the author's recommendations, we ran the program to estimate <italic>U<sub>d</sub></italic> and <italic>&#945;</italic> at fixed <italic>&#946;</italic>-values varying from 99 to 0.01. The log likelihood monotonically increased as <italic>&#946;</italic> decreased, yielding increasingly higher <italic>&#945;</italic>-values (<italic>s</italic> &#8594; &#8734; and <italic>U<sub>d</sub></italic> &#8594; 0).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Statistics were done with MS Excel (<ext-link ext-link-type="uri" ns0:href="http://www.microsoft.com">http://www.microsoft.com</ext-link>) and the SPSS 12.0 package (<ext-link ext-link-type="uri" ns0:href="http://www.spss.com">http://www.spss.com</ext-link>). Resamplings and simulations were done using a Perl script.</p>