<p xmlns:ns0="http://www.w3.org/1999/xlink">TFBSs with experimental support were located in the literature (see <xref ref-type="supplementary-material" rid="pgen-0030127-st001">Table S1</xref>). Genome coordinates (National Center for Biotechnology [NCBI] 35.1, May 2004 release) were located using BLAT [<xref ref-type="bibr" rid="pgen-0030127-b039">39</xref>] searches against the human genome within the UCSC genome browser (<xref ref-type="supplementary-material" rid="pgen-0030127-st002">Table S2</xref>) [<xref ref-type="bibr" rid="pgen-0030127-b031">31</xref>,<xref ref-type="bibr" rid="pgen-0030127-b040">40</xref>]. If a TFBS could not be found in the genome, it was removed from analysis, which left 83, 21, and 21 binding sites for p53, NRF2, and NF&#954;B, respectively. The UCSC &#8220;multiz17way&#8221; conservation track [<xref ref-type="bibr" rid="pgen-0030127-b031">31</xref>,<xref ref-type="bibr" rid="pgen-0030127-b040">40</xref>,<xref ref-type="bibr" rid="pgen-0030127-b041">41</xref>] provided a 17-way multiple sequence alignment between current releases of the <named-content content-type="genus-species">Homo sapiens</named-content> genome and eleven additional mammals: <named-content content-type="genus-species">Pan troglodytes</named-content> (chimpanzee, November 2003), <named-content content-type="genus-species">Canis familiaris</named-content> (dog, May 2005), <named-content content-type="genus-species">Mus musculus</named-content> (mouse, May 2004), <named-content content-type="genus-species">Rattus norvegicus</named-content> (rat, June 2003), <named-content content-type="genus-species">Macaca mulatta</named-content> (rhesus monkey, January 2006), <named-content content-type="genus-species">Monodelphis domestica</named-content> (opossum, June 2005), <named-content content-type="genus-species">Bos taurus</named-content> (cow, March 2005), <named-content content-type="genus-species">Echinops telfairi</named-content> (tenrec, July 2005), <named-content content-type="genus-species">Loxodonta africana</named-content> (elephant, May 2005), <named-content content-type="genus-species">Oryctolagus cuniculus</named-content> (rabbit, May 2005), and <named-content content-type="genus-species">Dasypus novemcinctus</named-content> (armadillo, May 2005). This alignment set was used to find the corresponding locations of each TFBS within each genome. The accuracy of these alignment regions were manually inspected and verified by both confirming similar local gene organization as well as referencing independently generated paired human&#8211;mammal alignments (UCSC tables netMm7, netMonDom1, netBosTau2, netRn4, netCanFam2, netRheMac2, and netPanTro1) [<xref ref-type="bibr" rid="pgen-0030127-b042">42</xref>,<xref ref-type="bibr" rid="pgen-0030127-b043">43</xref>]. Similarly, a random list of promoter and cDNA sequences were obtained from Ensembl (<ext-link ext-link-type="uri" ns0:href="http://www.ensembl.org">http://www.ensembl.org</ext-link>) by referencing a genome coordinate list of all known human protein-encoding genes (version 35.1) with an Ensembl gene identifier [<xref ref-type="bibr" rid="pgen-0030127-b044">44</xref>]. For each gene, a coordinate range of length equal to the TFBS of interest was randomly picked in the (a) 3,500 bases 5&#8242; to the gene start site and (b) within protein coding DNA sequence. Sites from these two lists of coordinates were randomly chosen to form a set of genome regions with the same number of members as each TFBS category: NRF2 (21), NF&#954;B (21), p53 (83), p53 apoptosis (29), and p53 cell cycle/cell growth (23). This process was repeated 1,000 times with replacement to capture the variance in the data. As with the known TFBSs, the mammalian multiple alignment data from multiz17way were retrieved for each of these promoter and cDNA sites. The placement of target genes into p53 subcategories was based on the grouping used in Qian et al. and an annotated literature search (<xref ref-type="supplementary-material" rid="pgen-0030127-st001">Table S1</xref>).</p><p>Pair-wise percent identities relative to the each TFBS consensus were calculated as the percentage of RE bases that were either (a) identical between the human and second genome or (b) mismatched but do not deviate from the consensus sequence (<xref ref-type="fig" rid="pgen-0030127-g001">Figure 1</xref>). For p53 REs, the variable spacer region was not considered. The distribution of conservation for each TFBS set is shown in <xref ref-type="fig" rid="pgen-0030127-g002">Figure 2</xref>. The conservation of the randomly chosen sets of coding region and promoter sequences, which represent the high and low extremes respectively of human genome conservation for TFBS comparisons, was calculated in the same fashion, and average results per x-axis bin for 1,000 trials are shown in <xref ref-type="fig" rid="pgen-0030127-g002">Figure 2</xref>. <italic>p</italic>-Values describing the statistical difference between percent sequence identity means for each TFBS set (<xref ref-type="table" rid="pgen-0030127-t004">Table 4</xref>) were calculated using an unpaired two-tailed t-test with the assumption of unequal variances. The ORs in <xref ref-type="table" rid="pgen-0030127-t004">Table 4</xref> were calculated as OR = ad/bc, where a = mean conservation of the column element, b = mean conservation of the row element, c = 100 &#8722; a, and d = 100 &#8722; b.</p><p>A list of CNC blocks, which represent the 5% most conserved portions of the human genome, were downloaded from the &#8220;most conserved&#8221; track of the UCSC genome browser. These were generated by constructing a phylogenetic two-state Hidden Markov Model [<xref ref-type="bibr" rid="pgen-0030127-b045">45</xref>] from the 17-way multiple alignment, which includes the human 35.1 genome release [<xref ref-type="bibr" rid="pgen-0030127-b031">31</xref>,<xref ref-type="bibr" rid="pgen-0030127-b041">41</xref>,<xref ref-type="bibr" rid="pgen-0030127-b044">44</xref>]. To generate the data in <xref ref-type="table" rid="pgen-0030127-t005">Table 5</xref>, the coordinates of each TFBS set were intersected with the CNC blocks. Likewise, the average number of randomly chosen promoter regions (1,000 trials) found within CNCs was determined. The probability of finding a greater fraction of CNCs in randomly chosen promoter regions over TFBS sites was calculated using a two-tailed binomial test.</p><p>For the ROC curves in <xref ref-type="fig" rid="pgen-0030127-g003">Figure 3</xref>, the sensitivity (true positive rate) and 1-specificity (false positive rate) of TFBS prediction were calculated at each of 11 conservation thresholds. The true positive rate was the fraction of bona fide TFBSs with a consensus sequence percent identity above a given level. The false positive rate was calculated as the average number of sites (1,000 trials) that fell above a conservation threshold (consensus sequence percent identity) in an equivalently sized set of random promoter sequences. For example, 83 TFBSs composed the p53 ROC curve. Therefore, sets of 83 random promoter sites, where each site was 20 bases in length, were used to estimate the false positive rate. ROC AUCs (<xref ref-type="table" rid="pgen-0030127-t006">Table 6</xref>) and standard errors were calculated directly from the graphs using the trapezoid rule as described by Hanley and McNeil [<xref ref-type="bibr" rid="pgen-0030127-b046">46</xref>].</p><p>Positional sequence diversity for a TFBS, which is related to the inverse of sequence identity, was calculated as the percentage of human nucleotides (nt) at each position that had a nonconsensus mismatch or insertion/deletion event when compared to one of the four mammals (<xref ref-type="fig" rid="pgen-0030127-g004">Figure 4</xref>). Each TFBS of a given class (i.e., TP53) and its alignment to a mammal (i.e., mouse) was pulled from the multiz multiple alignments to produce a set of paired sequence alignments. For each RE member of this pool, we counted the number of times where the first position of the mammalian sequence in the alignment differed from the human nt due to either (a) a nonconsensus mismatch or (b) an insertion/deletion event. This count was divided by the total number of TFBSs in a group (i.e., 83 for all p53 REs or 21 for all NF&#954;B REs) to get the percentage of two-way alignments that differed at that position. This value is taken as the sequence diversity at that first position. This calculation was then performed for the remaining nts in the TFBS. Mismatches from human that did not alter the consensus motif did not increase the percent sequence diversity and were ignored.</p>