<p xmlns:ns0="http://www.w3.org/1999/xlink">DNase-chip was performed as previously described [<xref ref-type="bibr" rid="pgen-0030136-b004">4</xref>]. Briefly, intact nuclei were digested with optimized amounts of DNase. DNase digested ends were blunted, ligated to biotinylated linkers, sonicated, and enriched on a streptavidin column. Sheared ends were blunted and ligated to nonbiotinylated linkers. DNase-enriched material was amplified by linker-mediated PCR, labeled, and hybridized to NimbleGen ENCODE arrays. Randomly sheared DNA was used as a reference control. For each cell type, DNase-chip material was generated from three biological replicates and three different DNase concentrations (total of nine hybridizations per cell type). Raw ratio data from each cell type was averaged and significant signals were identified using ACME (<italic>p</italic>-value = 0.001). All DNase-chip data is publicly available on the UCSC genome browser [<xref ref-type="bibr" rid="pgen-0030136-b012">12</xref>] (<ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu">http://genome.ucsc.edu</ext-link>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Human ES cell line H9 [<xref ref-type="bibr" rid="pgen-0030136-b037">37</xref>] (WiCell Research Institute; National Institutes of Health Code WA09) was cultured on a feeder layer of mitotically inactivated mouse embryo fibroblasts in medium consisting of DMEM-F12 supplemented with 20% KSR (Invitrogen, <ext-link ext-link-type="uri" ns0:href="http://www.invitrogen.com">http://www.invitrogen.com</ext-link>), 5 ng/ml FGF2 (R&amp;D Systems, <ext-link ext-link-type="uri" ns0:href="http://www.rndsystems.com/">http://www.rndsystems.com/</ext-link>), 2mM L-glutamine, 0.1mM 2-mercaptoethanol, and 1&#215; nonessential amino acids. For analysis, hES cell colonies were separated away from the feeder layer and processed for DNaseI hypersensitive site mapping. The undifferentiated state of the cultures was determined by morphology, immunohistochemistry, and Affymetrix expression array (<ext-link ext-link-type="uri" ns0:href="http://www.affymetrix.com">http://www.affymetrix.com</ext-link>) analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Total RNA was extracted from CD4<sup>+</sup> T cells, GM06990, HeLa S3, K562, and H9 undifferentiated stem cells using Trizol (Invitrogen). RNA was analyzed by Bioanlyzer to confirm high-quality 18s and 28s ribosomal bands (Agilent, <ext-link ext-link-type="uri" ns0:href="http://www.home.agilent.com/">http://www.home.agilent.com/</ext-link>), labeled, and hybridized to Affymetrix U133 Plus 2.0 arrays. The expression data for IMR90 was publicly available (<ext-link ext-link-type="uri" ns0:href="http://licr-renlab.ucsd.edu/download.html">http://licr-renlab.ucsd.edu/download.html</ext-link>). All data was normalized together using RMA [<xref ref-type="bibr" rid="pgen-0030136-b038">38</xref>] through the BioConductor project's Affymetrix package (<ext-link ext-link-type="uri" ns0:href="http://www.bioconductor.org">http://www.bioconductor.org</ext-link>). Only genes expressed in the ENCODE regions were used for the analysis. Gene expression was categorized as expressed or not expressed by the Affymetrix A/P call.</p><p>The enhancer blocking assay was performed as described previously [<xref ref-type="bibr" rid="pgen-0030136-b007">7</xref>]. Briefly, the &#946;-globin DNaseI HS2 site, which is a known enhancer element [<xref ref-type="bibr" rid="pgen-0030136-b039">39</xref>], was cloned upstream of a NeoR gene. Putative insulators were cloned between the enhancer and NeoR gene. The previously described chicken insulator was used a positive control [<xref ref-type="bibr" rid="pgen-0030136-b040">40</xref>]. DNaseI HS sites proximal or distal to TSS that also overlapped CTCF binding sites were cloned into the enhancer block vector. All plasmids were purified from three independent bacterial cultures, linearized, and each DNA prep was electroporated independently into K562 cells. Each electroporation was plated in triplicate (total of nine experiments per plasmid). The next day, cells were transferred to soft agar media containing G418. After 16 days, plates were scanned and colonies were counted.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Publicly available ChIP-chip data of CTCF, p300, RNA Pol II, TRAP220 was obtained from <ext-link ext-link-type="uri" ns0:href="http://licr-renlab.ucsd.edu/download.html">http://licr-renlab.ucsd.edu/download.html</ext-link>. ChIP-chip data for five histone modifications were obtained from the UCSC genome browser (<ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu">http://genome.ucsc.edu</ext-link>). The coordinates for TSS were obtained from the ENCODE pilot study [<xref ref-type="bibr" rid="pgen-0030136-b014">14</xref>] (TSS set ABCDE defined in supplement 3.5 therein). These data were mapped onto the DNaseI HS sites in each cell line based on their overlapping coordinates. To determine whether distal DNaseI HS sites are still statistically near genes, we binned all DNaseI HS sites according to their distances to the closest TSS. We computed an enrichment score for each bin, defined as the ratio between the number of DNaseI HS sites and the number of all possible positions in the ENCODE regions in the same distance bin.</p><p>MEME [<xref ref-type="bibr" rid="pgen-0030136-b018">18</xref>] with the &#8220;-zoop&#8221; option was used to identify the CTCF sites in the ubiquitous distal DNaseI HS sites. Clover [<xref ref-type="bibr" rid="pgen-0030136-b024">24</xref>] was used to identify motifs overrepresented in cell type&#8211;specific DNaseI HS sites at each distance category: proximal (&lt;2 kb), distal (between 2 kb and 10 kb) and far distal (&gt;10 kb). Two background sets were used (union of ChIP-chip hits and random dinucleotide shuffling of input sequences). Overrepresented motifs (<italic>p-</italic>values &lt; 0.01) from the TRANSFAC database were reported. Overlapping motifs were reported as groups.</p>