<p xmlns:ns0="http://www.w3.org/1999/xlink">We utilize an approach derived from that of Bourgain et al <xref ref-type="bibr" rid="pgen.1001096-Bourgain1">[7]</xref> which has been discussed extensively in recent papers <xref ref-type="bibr" rid="pgen.1001096-Choi1">[3]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1001096-Astle1">[5]</xref>. This approach for accounting for relatedness between subjects in association tests adopts a &#8220;retrospective&#8221; approach towards the problem of testing for disease associations using marker data, in which (as in the Armitage test) the allele frequency of a variant is related to case-control status. In vector notation we have <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e005.jpg" /></inline-formula> of observed values for a given SNP <italic>j</italic>. Here <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e006.jpg" /></inline-formula> is the total number of subjects (cases+controls) in the study, and SNP values <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e007.jpg" /></inline-formula> are coded as (0,1,2) for the number of copies of a specified allele, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e008.jpg" /></inline-formula>, carried by subject <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e009.jpg" /></inline-formula>. (This coding of SNP genotype implies that we are interested in additive models for the relationship between disease risk and genotype but the approach can readily be extended to other codings). The retrospective approach models the mean of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e010.jpg" /></inline-formula> as a function of case-control status. If we define the <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e011.jpg" /></inline-formula> design matrix <bold>C</bold> to have rows (1 <italic>c<sub>i</sub></italic>) where <italic>c<sub>i</sub></italic> is case-control status (0 or 1) for subject <italic>i</italic> then the mean, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e012.jpg" /></inline-formula>, of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e013.jpg" /></inline-formula> is written as<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001096.e014.jpg" /></disp-formula>Relatedness between subjects induces a covariance matrix for the number of copies of a given SNP of form<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001096.e015.jpg" /><label>(1)</label></disp-formula>with <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e016.jpg" /></inline-formula> specific to each SNP but with the same matrix <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e017.jpg" /></inline-formula> for all SNPs. In fact, for known pedigree relationships, and unrelated founders, this matrix <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e018.jpg" /></inline-formula> has diagonal elements equal to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e019.jpg" /></inline-formula> where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e020.jpg" /></inline-formula> is the inbreeding coefficient for subject <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e021.jpg" /></inline-formula> and each off-diagonal element, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e022.jpg" /></inline-formula>, is twice the kinship coefficient for the relationship between subjects <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e023.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e024.jpg" /></inline-formula>
<xref ref-type="bibr" rid="pgen.1001096-Bourgain1">[7]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">It is worth noting that in general the topic of relatedness includes what is often considered to be population substructure. For example consider two large but isolated populations (freely mixing within each population) that have been separated for many generations. While a random sample of people from the same population (with sample size small relative to the population size) might be considered unrelated to each other when considering that population separately, when considering the two populations together people from one isolated population are considered to be related to each other, relative to those in the other population. In particular, genetic markers will, through a process of random drift and other factors, be able to distinguish members from the two populations, and this will be detectable when calculation of the <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e025.jpg" /></inline-formula> matrix is performed. A standard method of simulating genetic markers for divergent populations stemming from the same ancestral population (e.g. the Balding-Nichols model <xref ref-type="bibr" rid="pgen.1001096-Balding1">[11]</xref>) can readily be shown to produce covariance matrices of the form of expression (1).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">If <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e026.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e027.jpg" /></inline-formula> are both known then the best linear unbiased estimate (BLUE) of the regression vector <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e028.jpg" /></inline-formula> is of weighted least squares form<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001096.e029.jpg" /></disp-formula>and the variance covariance matrix of the estimates is in the form of<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001096.e030.jpg" /></disp-formula>Thus inference on the significance of the allele frequency difference between cases and controls may be based upon the Wald test statistic<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001096.e031.jpg" /><label>(2)</label></disp-formula>with <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e032.jpg" /></inline-formula> the (2,2) element of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e033.jpg" /></inline-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In general of course, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e034.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e035.jpg" /></inline-formula> are not known, except in the case of known pedigrees and unrelated founders, where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e036.jpg" /></inline-formula> can be computed from first principles. The estimation of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e037.jpg" /></inline-formula> using marker data has been considered by a number of authors and both method of moments <xref ref-type="bibr" rid="pgen.1001096-Rakovski1">[4]</xref>, <xref ref-type="bibr" rid="pgen.1001096-Astle1">[5]</xref> and maximum likelihood methods <xref ref-type="bibr" rid="pgen.1001096-Choi1">[3]</xref> have been considered. A method of moments estimator of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e038.jpg" /></inline-formula> can be concisely written <xref ref-type="bibr" rid="pgen.1001096-Astle1">[5]</xref> as<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001096.e039.jpg" /><label>(3)</label></disp-formula>and the estimate of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e040.jpg" /></inline-formula> as<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001096.e041.jpg" /></disp-formula>One value of this approach, which is exploited here, is that it is relatively easy to compute the power of the Wald <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e042.jpg" /></inline-formula> test if we can hypothesize a form of the relatedness matrix <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e043.jpg" /></inline-formula>. For a given form for <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e044.jpg" /></inline-formula> (below we consider several forms for both isolated population models and more complex admixed populations) then for a given sample size, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e045.jpg" /></inline-formula>, a given allele frequency for a causal SNP, and a hypothesized difference in allele frequencies between cases and controls (which can then be related to odds ratios in typical case/control analysis) we can compute the non-centrality parameter of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e046.jpg" /></inline-formula> (and hence the power of the test) as<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001096.e047.jpg" /><label>(4)</label></disp-formula>We illustrate the computation of this non-centrality parameter for a number of important special cases in the results section below. It is worth noting now, however, that the Bourgain test appears to be reasonably powerful compared to other procedures, and can sometimes be considered as a compromise between the principal components method <xref ref-type="bibr" rid="pgen.1001096-Price1">[8]</xref> and genomic control <xref ref-type="bibr" rid="pgen.1001096-Devlin1">[12]</xref>. We attempt to justify this last statement in the results section below.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In addition to the Bourgain test we used several well known tools for addressing population structure in the AABC data. For example we computed eigenvectors through the use of the program EIGENSTRAT <xref ref-type="bibr" rid="pgen.1001096-Price1">[8]</xref>. Briefly, each eigenvector explains a proportion of the genetic variation among samples in the analysis so that the leading eigenvector explains the greatest variation, followed by the second eigenvector, and so forth. The full set of eigenvectors form an orthonormal basis so that each eigenvector is scaled on the unit interval and linearly independent from all other eigenvectors. Note that the EIGENSTRAT procedure is operating on the same estimated <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e048.jpg" /></inline-formula> matrix, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001096.e049.jpg" /></inline-formula>, that we have described above.</p><p>To assess ancestry within the AABC study in relation to reference populations from HapMap, we performed a principal components analysis based on ancestry informative markers that were genotyped in both the AABC study and the HapMap Phase 3 populations. The 2,546 ancestry informative markers (contained within the Illumina 1M genotyping array which was used in the AABC scan) were selected based on low inter-marker correlation and high correlation to a previously determined eigenvector that explained African and European ancestry.</p><p>We quantified percent African ancestry for each of the nine AABC study populations by running the program STRUCTURE for each study population. The program implements a Markov Chain Monte Carlo algorithm that provides the posterior estimates of the proportion of ancestry from each of k clusters for each individual, where k is specified by the investigator. For each AABC study population, we assigned k&#8202;=&#8202;3, including genotypes from the same ancestry informative markers used in PCA genotyped in YRI, CEU, and JPT from HapMap Phase 3.</p><p>AABC included 9 epidemiological studies of breast cancer among African American women, which comprise a total of 3,153 cases and 2,831 controls. Below is a brief description of these studies.</p><p>The MEC is a prospective cohort study of 215,000 men and women in Hawaii and Los Angeles <xref ref-type="bibr" rid="pgen.1001096-Kolonel1">[1]</xref> between the ages of 45 and 75 years at baseline (1993&#8211;1996). Through December, 31 2007, a nested breast cancer case-control study in the MEC included 556 African American cases (554 invasive and 12 in situ) and 1,003 African American controls. An additional 178 African American breast cancer cases (ages: 50&#8211;84) diagnosed between June 1, 2006 and December 31, 2007 in Los Angeles County (but outside of the MEC) were combined with the MEC samples in the analysis.</p><p>The NICHD Women's CARE Study is a large multi-center population-based case-control study that was designed to examine the effects of oral contraceptive (OC) use on invasive breast cancer risk among African American women and white women ages 35&#8211;64 years in five U.S. locations <xref ref-type="bibr" rid="pgen.1001096-Marchbanks1">[13]</xref>. Cases in Los Angeles County were diagnosed from July 1, 1994 through April 30, 1998, and controls were sampled by random-digit dialing (RDD) from the same population and time period; 380 African American cases and 224 African American controls were included in stage 1 of the scan.</p><p>The WCHS is an ongoing case-control study of breast cancer among women of European and African descent residing in the New York City boroughs (Manhattan, the Bronx, Brooklyn and Queens) and in seven counties in New Jersey (Bergen, Essex, Hudson, Mercer, Middlesex, Passaic, and Union) <xref ref-type="bibr" rid="pgen.1001096-Ambrosone1">[14]</xref>. Eligible cases included women diagnosed with invasive breast cancer between 20 and 74 years of age; controls were identified through RDD. The WCHS contributed 272 invasive African American cases and 240 African American controls to stage 1 of the GWAS.</p><p>The SFBC is a population-based case-control study of invasive breast cancer in Hispanic, African American and non-Hispanic White women conducted between 1995 and 2003 in the San Francisco Bay Area <xref ref-type="bibr" rid="pgen.1001096-John1">[15]</xref>. African American cases, ages 35&#8211;79 years, were diagnosed between April 1, 1995 and April 30, 1999, with controls identified through RDD. Stage 1 included 172 invasive African American cases and 231 African American controls from SFBC.</p><p>The NC-BCFR is an on-going population-based family study conducted in the Greater San Francisco Bay Area, and is one of 6 sites collaborating in the Breast Cancer Family Registry (BCFR), an international consortium funded by NCI <xref ref-type="bibr" rid="pgen.1001096-John2">[16]</xref>. African American breast cancer cases in NC-BCFR were diagnosed after January 1, 1995 and between the ages of 18 and 64 years; population controls were identified through random digit dialing (RDD). Stage 1 genotyping was conducted for 440 invasive African American cases and 53 African American controls.</p><p>The CBCS is a population-based case-control study conducted between 1993 and 2001 in 24 counties of central and eastern North Carolina <xref ref-type="bibr" rid="pgen.1001096-Newman1">[17]</xref>. Cases were identified by rapid case ascertainment system in cooperation with the North Carolina Central Cancer Registry and controls were selected from the North Carolina Division of Motor Vehicle and United States Health Care Financing Administration beneficiary lists. Participants' ages ranged from 20 to 74 years. For stage 1, DNA samples were provided from 656 African American cases with invasive breast cancer and 608 African American controls.</p><p>PLCO, coordinated by the U.S. National Cancer Institute (NCI) in 10 U.S. centers, enrolled during 1993&#8211;2001 approximately 155,000 men and women, aged 55&#8211;74 years, in a randomized, two-arm trial to evaluate the efficacy of screening for these four cancers <xref ref-type="bibr" rid="pgen.1001096-Prorok1">[18]</xref>. A total of 64 African American invasive breast cancer cases and 133 African American controls contributed to stage 1 of the GWAS.</p><p>The NBHS is a population-based case-control study of incident breast cancer conducted in the Tennessee <xref ref-type="bibr" rid="pgen.1001096-Zheng1">[19]</xref>. The study was initiated in 2001 to recruit patients with invasive breast cancer or ductal carcinoma <italic>in situ</italic>, and controls, recruited through RDD between the ages of 25 and 75 years. NBHS contributed 310 African American invasive cases (57 in situ), and 186 African American controls to the stage 1 analysis.</p><p>African American breast cancer cases and controls in WFBC were recruited at Wake Forest University Health Sciences from November 1998 through December 2008 <xref ref-type="bibr" rid="pgen.1001096-Smith1">[20]</xref>. Controls were recruited from the patient population receiving routine mammography at the Breast Screening and Diagnostic Center. Age range of participants was 30&#8211;86 years. WFBC contributed 125 cases (116 invasive and 9 in situ) and 153 controls to the stage 1 analysis.</p><p>Genotyping in stage 1 was conducted using the Illumina Human1M-Duo BeadChip. Of the 5,984 samples from these studies (3,153 cases and 2,831 controls), we attempted genotyping of 5,932, removing samples (n&#8202;=&#8202;52) with DNA concentrations &lt;20 ng/ul by pico green assay. After clustering the genotype data we removed samples based on the following exclusion criteria: 1) unknown replicates (&#8805;98.9% genetically identical) that we were able to confirm (only one of each duplicate was removed, n&#8202;=&#8202;15); 2) unknown replicates that we were not able to confirm through discussions with study investigators (pair or triplicate removed, n&#8202;=&#8202;14); 3) samples with call rates &lt;95% after a second attempt (n&#8202;=&#8202;100); 4) samples with &#8804;5% African ancestry (n&#8202;=&#8202;36) (discussed below); and, 5) samples with &lt;15% mean heterozygosity of SNPs in the X chromosome and/or similar mean allele intensities of SNPs on the X and Y chromosomes (n&#8202;=&#8202;6) (these are likely to be males).</p><p>In the analysis, we removed SNPs with &lt;95% call rate or minor allele frequencies (MAFs) &lt;1%. To assess genotyping reproducibility we included 138 replicate samples; the average concordance rate was 99.95% (&gt;99.93% for all pairs). We also eliminated SNPs with genotyping concordance rates &lt;98% based on the replicates. The final analysis dataset included 3,016 cases and 2,745 controls, with an average SNP call rate of 99.7% and average sample call rate of 99.8%. Hardy-Weinberg equilibrium (HWE) was not used as a criterion for removing SNPs for this analysis.</p>