<p>This study was conducted according to the principles expressed in the Declaration of Helsinki and obtained informed consent from participants and approval from the appropriate institutional review boards.</p><p>All subjects included in these meta-analyses were current or former smokers of European ancestry. Results from 34 datasets, which include a total of 38,617 unrelated subjects who were assessed for cigarettes-per-day, contributed to the meta-analyses. Eight of the datasets were drawn from family-based studies and contributed only a subset of unrelated individuals to these analyses. <xref ref-type="table" rid="pgen-1001053-t001">Table 1</xref> gives sample sizes and demographics of each participating study sample. <xref ref-type="supplementary-material" rid="pgen.1001053.s007">Text S7.</xref>describes additional details for each dataset, including ascertainment criteria and genotyping methods, and documents that four datasets are also members of other consortia. All datasets contributed to the analyses of smoking. A subset of these 34 datasets also had information on lung cancer cases and lung-cancer-free smoker controls (6 datasets, N&#8202;=&#8202;13,614 smokers) and/or COPD cases and COPD-free smoker controls (4 datasets, N&#8202;=&#8202;6,182 smokers). The data for these traits are described in <xref ref-type="table" rid="pgen-1001053-t002">Table 2</xref> and <xref ref-type="table" rid="pgen-1001053-t003">Table 3</xref> respectively.</p><p>The traits examined were smoking quantity, lung cancer, and COPD. Two smoking traits were derived from measurements of cigarettes smoked per day (CPD): a 4-level categorical trait (CPD&#8804;10, 10&lt;CPD&#8804;20, 20&lt;CPD&#8804;30, and CPD&gt;30) and a dichotomous trait contrasting subjects from the lowest smoking category (CPD&#8804;10: light-smoking &#8220;controls&#8221;) to those in the two highest categories combined (CPD&gt;20: heavy smoking cases). The dichotomous trait of heavy versus light smoking was our primary trait for analysis. For one study (NAG-Finland), which used different boundaries to record CPD as detailed in the supplemental material, the distribution of CPD was examined to harmonize the phenotypes and select alternative boundaries. The numbers of subjects in each smoking category, total and by study, are given in <xref ref-type="table" rid="pgen-1001053-t001">Table 1</xref>. Lung cancer and COPD were analyzed as dichotomous traits. COPD cases were defined to have COPD as determined by post-bronchodilator spirometry as GOLD Stage II or worse (N&#8202;=&#8202;1,719), or self-reported COPD, emphysema or chronic bronchitis.</p><p>In European-ancestry populations, each of the four loci of interest can be represented by various highly correlated SNPs (SNPs having high r<sup>2</sup> with each other). For each locus, we chose one target SNP for analysis: rs16969968 (locus 1), rs578776 (locus 2), rs588765 (locus 3), and rs12914008 (locus 4); the pairwise correlations between any two of these loci are r<sup>2</sup>&lt;0.5 (<xref ref-type="supplementary-material" rid="pgen.1001053.s004">Table S1</xref>). In samples for which a given target SNP was not available, we chose a highly correlated proxy SNP based on r<sup>2</sup> computed with Haploview <xref ref-type="bibr" rid="pgen.1001053-Barrett1">[26]</xref> using downloaded HapMap CEU genotype data, Release 23 <xref ref-type="bibr" rid="pgen.1001053-The2">[27]</xref>. <xref ref-type="supplementary-material" rid="pgen.1001053.s005">Table S2</xref> lists the proxy SNPs used and their r<sup>2</sup> with the corresponding target SNPs. <xref ref-type="supplementary-material" rid="pgen.1001053.s001">Figure S1</xref> displays the SNPs for each of the 4 loci in relation to the <italic>CHRNA5-CHRNA3-CHRNB4</italic> cluster.</p><p>To ensure uniform analyses, SAS (SAS Institute, Cary, NC) and R <xref ref-type="bibr" rid="pgen.1001053-R1">[28]</xref> scripts for genetic association analyses were developed centrally and then distributed. The scripts were executed by each participating site, and the results returned to the coordinating group.</p><p>In each dataset, associations between the loci and the traits were evaluated using logistic regression. Our primary analysis model coded genotypes additively as the number of copies of the minor allele according to the HapMap CEU reference population. This allele is referred to as the &#8220;coded allele&#8221; (C) and the major allele is referred to as the &#8220;reference allele&#8221; (R). To confirm the appropriateness of the additive model, for each locus a 2 degree of freedom model including the additive term and a heterozygote deviation term was evaluated. The analyses of the 4-level CPD trait used generalized logistic regression to obtain separate effect estimates (beta coefficients) for each category with respect to the lowest smoking category as the referent. All these association analyses included sex and age as covariates. In addition, lung cancer and COPD analyses included categorical cigarettes-per-day as an unordered covariate.</p><p>Association results from each dataset, including the beta coefficient and standard error, were provided to the coordinating team. Meta-analysis was carried out using PLINK <xref ref-type="bibr" rid="pgen.1001053-Purcell1">[29]</xref> to obtain overall summary odds ratios (ORs) and statistics. The R package rmeta <xref ref-type="bibr" rid="pgen.1001053-Lumley1">[30]</xref> was used to verify results and create plots. There was no evidence of significant heterogeneity across datasets for these analyses (minimum heterogeneity p&#8202;=&#8202;0.21 for dichotomous CPD, 0.07 for lung cancer, 0.24 for COPD; for categorical CPD a nominally significant p was seen only for category 3 and locus 1 (p&#8202;=&#8202;0.007)). Because of varying study designs, ascertainment strategies, and representative SNPs, we nevertheless report results from random effects meta-analyses.</p><p>As noted earlier, locus 1 (representing rs16969968) is a highly replicated association finding and furthermore rs16969968 has been shown to have functional effects on the resulting alpha5-containing receptor <xref ref-type="bibr" rid="pgen.1001053-Bierut1">[19]</xref>. Therefore an important question is whether the remaining loci demonstrate additional independent effects on disease risk. Although loci 2, 3 and 4 are not highly correlated with rs16969968, |D'| is high. A high |D'| can correspond to a low r<sup>2</sup> if the alleles that tend to co-occur on the same haplotype have very different allele frequencies. Previous results in the COGEND data suggest that there may be independent or synergistic effects on nicotine dependence between locus 1 and locus 3 <xref ref-type="bibr" rid="pgen.1001053-Saccone2">[4]</xref>, and haplotype analyses in the Utah and LHS samples <xref ref-type="bibr" rid="pgen.1001053-Weiss1">[7]</xref>, and in the COGEND and CPS-II-CPD samples <xref ref-type="bibr" rid="pgen.1001053-Wang2">[22]</xref>, also indicate effects of haplotypes containing loci 1, 2 and 3.</p><p>To test whether additional loci contribute to dichotomous smoking quantity over and above the effect of rs16969968, we included both locus 1 and each of the other loci in the logistic regression models adjusting for sex and age, with and without a SNP&#215;SNP interaction term. For lung cancer and COPD the models also included categorical cigarettes-per-day as an unordered covariate. These results were then meta-analyzed as described above. The SNP&#215;SNP interaction term was never significant in the meta-analysis (p&gt;0.3), so we report results from the joint models without interactions. To allow comparison between single-SNP and joint results on comparable data, for each locus pair we also repeated the univariate single-SNP meta-analyses on the subset of datasets that had genotypes available at both loci. For dichotomous smoking quantity we also tabulated pair-wise joint genotype by case status counts for locus 1 (rs16969968) versus each of the other three loci across the contributing datasets that had both loci.</p><p>Across the four target loci, multiple traits (4), the multiple models (additive and additive+heterozygote deviation), and the 2-SNP joint analyses (3 loci), our study was designed to perform fewer than 80 tests. A conservative Bonferroni correction would result in an uncorrected p-value threshold of 6.25&#215;10<sup>&#8722;4</sup> corresponding to an experiment-wide alpha of 0.05. The results tables report uncorrected p-values which we compared to this threshold to determine statistical significance.</p>