<p>Frozen tissue samples of the cerebellum (CRBLM), frontal cortex (FCTX), caudal pons (PONS) and temporal cortex (TCTX) were obtained from 150 neurologically normal Caucasian subjects (<xref ref-type="supplementary-material" rid="pgen.1000952.s014">Text S1</xref>). 100&#8211;200mg aliquots of frozen tissue were sub-dissected from each tissue from all 150 subjects resulting in 600 tissue samples and used for methylation assay and expression assays. Genotyping was performed using Infinium HumanHap550 beadchips (Illumina) to assay genotypes for 561466 SNPs, from the cerebellum tissue samples. CpG methylation status was determined using HumanMethylation27 BeadChips (Illumina), which measure methylation at 27,578 CpG dinucleotides at 14,495 genes. Profiling of 22,184 mRNA transcripts was performed using HumanRef-8 Expression BeadChips (Illumina) as previously described <xref ref-type="bibr" rid="pgen.1000952-vanderBrug1">[27]</xref>.</p><p>The threshold call rate for inclusion of the subject in analysis was 95%. All 150 subjects had a call rate greater than 95%, and were included in the subsequent analyses (average call rate&#8202;=&#8202;99.86%; range 97.72%&#8211;99.95%, based on the <italic>missing</italic> procedure within the PLINK v1.04 software toolset <xref ref-type="bibr" rid="pgen.1000952-Purcell1">[17]</xref>.</p><p>The gender of the subjects reported by the brain banks was compared against their genotypic gender using PLINK's <italic>check-sex</italic> algorithm, which determines a sample's genotypic gender based on heterozygosity across the X chromosome. A gender discrepancy was detected for one subject and excluded from further analysis.</p><p>To confirm the ethnicity of the samples, Identity-By-State (IBS) clustering and multidimensional scaling analyses were performed within PLINK using the genotypes from the brain samples that had been merged with data from the four HapMap <xref ref-type="bibr" rid="pgen.1000952-1">[28]</xref> populations (n&#8202;=&#8202;32 Caucasian (CEU), 12 Han Chinese, 16 Japanese and 24 Yoruban non-trio samples previously genotyped by Illumina and assayed on the Infinium HumanHap500 version genotyping chips). Two samples were outliers based on population and excluded from further analysis (<xref ref-type="supplementary-material" rid="pgen.1000952.s001">Figure S1</xref>).</p><p>Genotype data of the samples were compared for cryptic relatedness using the Identity-By-Descent (IBD) procedure within PLINK. No samples were found to be from related individuals.</p><p>Mach software version 1.0.16 <xref ref-type="bibr" rid="pgen.1000952-Li1">[16]</xref> and HapMap CEU phase data (release 22) were used to impute genotypes for &#8764;2.5 million SNPs. Imputed SNPs were excluded if the linkage disequilibrium r<sup>2</sup> values between imputed and known genotypes was less than 0.3, and if their posterior probability averages were less than 0.8 for the most likely imputed genotype. For each of the four tissue regions, SNPs were also excluded if: (a) call rate was less than 95%, (b) Hardy-Weinberg equilibrium (HWE) p-value was less than 0.001, and (c) the SNP had less than 3 minor homozygotes present. Exact numbers of SNPs used per brain tissue and assay type are shown in <xref ref-type="table" rid="pgen-1000952-t001">Table 1</xref>.</p><p>Raw intensity values for each probe were transformed using the rank invariant normalization method <xref ref-type="bibr" rid="pgen.1000952-Workman1">[29]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1000952-Tseng1">[31]</xref> and then log<sub>2</sub> transformed for mRNA analysis. Four samples were excluded from further analysis, as they were outliers based on overall probe detection rate and/or mean expression level; one CRBLM, one FCTX and two PONS.</p><p>The threshold call rate for inclusion of samples in the analysis was 95%. Based on this metric, 9 CRBLM, 2 FCTX, 6 PONS and 8 TCTX samples were excluded from further analysis. The remaining brain samples had an average detection rate of 99.84% (range 95.0% to 99.98%).</p><p>The gender of the samples reported by the brain banks were compared against their assayed gender based on values of methylation from CpG sites on the X chromosome. Four samples with gender discrepancies were detected and were removed from subsequent analysis; two CRBLM, one PONS and one TCTX. The resulting HCL sample tree based on all Chromosome X probes, after removal of these four individuals is shown in <xref ref-type="supplementary-material" rid="pgen.1000952.s003">Figure S3</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Performing a Hierarchical Clustering (HCL) <xref ref-type="bibr" rid="pgen.1000952-Eisen1">[9]</xref> of the sample profiles using the TM4 MeV version 4.1.01 tool <xref ref-type="bibr" rid="pgen.1000952-Saeed1">[32]</xref>, with Euclidian distances and &#8216;Average Linkage clustering&#8217; resulted in the samples separating fairly well by brain tissue region. Where in mRNA separation of samples into 4 clusters matching brain tissue region was clear. The frontal and temporal cortices could not be separated in within CpG data. For clustering all detected data was used for mRNA; only detected data for autosomal probes was used in clustering of the CpG data, otherwise sub-clusters based on gender appeared. The HCL samples trees were saved a Newick tree files and plotted again using the HyperTree tool (<ext-link ext-link-type="uri" ns0:href="http://hypertree.sourceforge.net/">http://hypertree.sourceforge.net/</ext-link>).</p><p>Traits were excluded from analysis if they were detected in less than 95% of samples for each tissue region. For each tissue region and trait type (CpG and mRNA) the 95% threshold was determined using total number of analyzable samples for this pairing of region and trait. A probe is considered detected for a sample if the reported Illumina Detection p-value was less than or equal to 0.01. Numbers of analyzable samples per trait (or assay) type and region are available in <xref ref-type="table" rid="pgen-1000952-t001">Table 1</xref> overlap of detected probes among brain tissues is shown as Venn diagram in <xref ref-type="supplementary-material" rid="pgen.1000952.s002">Figure S2</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Prior to quantitative trait loci analysis each trait was adjusted using the available biological and methodological covariates in an attempt to remove the influence of these potentially confounding affects. In R each trait was regressed using the following model:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1000952.e001.jpg" /></disp-formula>Where Y is the trait profile (log<sub>2</sub> normalized mRNA expression intensities and raw values of CpG DNA methylation) and X<sub>1</sub> &#8230; X<sub>n</sub> represent the biological covariates Age and Gender and the methodological covariates post mortem interval (PMI), which Brain Bank the samples was from and which preparation/hybridization batch the sample was processed in. Within this model gender, tissue bank and batch where treated as categorical covariates. After fitting each trait to the model the residuals from the model are kept and represent the trait in following analyses. Thus variance attributable to gender, age, post-mortem interval, tissue source and hybridization batch are removed prior to QTL analysis. These covariate data are available in <xref ref-type="supplementary-material" rid="pgen.1000952.s010">Table S1</xref>. Histograms showing the proportion of traits that are potentially impacted by these covariates are shown in <xref ref-type="supplementary-material" rid="pgen.1000952.s004">Figure S4</xref> (CpG) and <xref ref-type="supplementary-material" rid="pgen.1000952.s005">Figure S5</xref> (mRNA).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For each of the four brain regions, a regression analysis was performed on the residuals described in the preceding section for (a) mRNA expression and (b) the methylation values generated for CpG sites. The trait residuals were then used as the quantitative phenotype for that probe in genome-wide association analysis looking for quantitative trait loci. These analyses were performed using the <italic>assoc</italic> function within Plink, which correlates allele dosage with change in the trait. Each of the four tissue regions was analyzed separately, and independent genome-wide association analyses were performed looking for (a) expression quantitative trait loci (eQTLs) for mRNA and (b) methylation quantitative trait loci (methQTLs) for CpG sites. The Plink toolset quantitative trait association analysis fits data to the following model:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1000952.e002.jpg" /></disp-formula>Where Y is the quantitative trait and ADD represent genotypes encoded as allele dosage. See Plink Quantitative trait association and Linear and Logistic models documentation for more information.</p><p>To correct for the large number of SNPs tested per trait, a genome-wide empirical p-value was computed for the asymptotic p-value for each SNP by using 1,000 permutations of swapping sample labels of the traits, using the maxT permutation functionality provided within Plink. A permutation based method using label swapping of the traits is an appropriate method of test correction <xref ref-type="bibr" rid="pgen.1000952-Churchill1">[18]</xref> for these analysis as it is not dependent on these quantitative traits having a normal distribution and also allows the linkage disequilibrium of the genomic regions being tested against the traits to be maintained.</p><p>To correct for the number of traits being tested per tissue region, a false discovery rate (FDR) threshold was determined based on the empirical p-values using the <italic>fwer2fdr</italic> function of the multtest package in R version 2.6.1. Empirical p-values were allowed to exceed this threshold if their linkage disequilibrium r<sup>2</sup> was greater than or equal to 0.7 with a SNP with empirical values within the FDR threshold.</p><p>Sequence variants within the sequence of the probe used to assay individual traits may cause differential hybridization and inaccurate expression and possible methylation measurements. To exclude this confound, the sequences of probes with significant correlation to a trait were examined for the presence of polymorphisms using CEU HapMap data, and, if present, that QTL was removed from the result set.</p><p>Within each tissue, we selected every pairing of CpG methylation sites and mRNA transcripts where the CpG methylation site was within 1Mb of the mRNA TSS and both the CpG methylation site or mRNA transcript had a significant <italic>cis</italic> QTL. These CpG-mRNA pairs were then expanded into triplets of SNP and CpG-mRNA pair, where the SNP was significantly correlated in <italic>cis</italic> with either the CpG methylation site or mRNA transcript of the CpG-mRNA pair.</p><p>Data resulting from these experiments is available online (GEO Accession Number: GSE15745; dbGAP Study Accession: phs000249.v1.p1).</p>