<p>
<italic>Cesmn-1(lf)</italic> homozygous strains cannot be maintained due to infertility; <italic>hT2(lethal)[myo-2p::GFP]/Cesmn-1(lf)</italic> (<italic>hT2[bli-4(e937) let-?(q782)qIs48] (I;III)</italic>) animals are fertile and were maintained using standard techniques <xref ref-type="bibr" rid="pgen.1001172-Brenner1">[95]</xref>. The lack of homologous pairing for the rearranged chromosomes LGI and LGIII in <italic>hT2</italic> animals likely results in increased maternal/zygotic expression of <italic>Cesmn-1</italic> and other balanced genes <xref ref-type="bibr" rid="pgen.1001172-Bean1">[96]</xref>. As expected, we found that the progeny of <italic>hT2</italic> animals were relatively resistant to <italic>Cesmn-1(RNAi)</italic> compared to wild type control strains in our assays (data not shown). Consequently, to keep the genetic background invariant, all animals were tested herein were the progeny of <italic>hT2(lethal)[myo-2p::GFP]</italic> parents. The use of RNAi sensitive <italic>C. elegans</italic> mutant strains was avoided as their behavior is not normal in many assays (Hart, unpublished observations) and because SMN complex/Sm proteins have been implicated in miRNA pathways <xref ref-type="bibr" rid="pgen.1001172-Mourelatos1">[20]</xref>, <xref ref-type="bibr" rid="pgen.1001172-Bilinski1">[97]</xref>, <xref ref-type="bibr" rid="pgen.1001172-Dostie1">[98]</xref>. We note that RNAi knockdown is not always effective. To control for genetic background effects, animals tested in these studies were either heterozygous for <italic>hT2</italic> balancer chromosome or progeny of <italic>hT2</italic> parents unless otherwise noted.</p><p>
<italic>plst-1(tm4255)</italic> animals were obtained from the Japanese National Bioresource Project and were backcrossed four times before further study. The <italic>tm4255</italic> allele is a 368 base pair deletion that removes one of the calponin-like, actin-binding homology (CH) domains; <italic>plst-1(tm4255)</italic> is likely a partial loss of function allele. To test the genetic interaction of <italic>plst-1</italic> with <italic>Cesmn-1</italic>, the backcrossed <italic>plst-1(tm4255)</italic> allele was used to create a double mutant with <italic>Cesmn-1(lf)</italic>. <italic>The flp-4(yn35)</italic> deletion allele was isolated by PCR-based screening of EMS-mutagenized animals. The <italic>yn35</italic> allele is a 928 base pair deletion that removes exon 3 of <italic>flp-4</italic> gene along with 5&#8242; sequences (flanking sequences, <named-content content-type="gene">ttctgaaaaacttttaataa</named-content> and <named-content content-type="gene">agctcgccgagccgagtctt</named-content>) <xref ref-type="bibr" rid="pgen.1001172-Liu1">[66]</xref>. The <italic>grk-2(rt97)</italic> loss of function allele was previously characterized <xref ref-type="bibr" rid="pgen.1001172-Fukuto1">[60]</xref>.</p><p>
<italic>Drosophila</italic> stocks were maintained on standard cornmeal/yeast/molasses/agar medium at 25&#176;C. The mutations of <italic>Smn</italic>
<sup>73Ao</sup> and <italic>Smn</italic>
<sup>f01109</sup> have been described previously <xref ref-type="bibr" rid="pgen.1001172-Chan1">[25]</xref>. <italic>Cbp20</italic>
<sup>e02787</sup> is a Piggy-Bac insertion mutation from the Exelixis collection. The insertion location is 5&#8242; upstream and adjacent to the start codon of the Cbp20 transcript. <italic>Fmrf</italic>
<sup>KG1300</sup> and Fim alleles are loss of function alleles (Flybase). The line d03334 may have an unlinked lethal mutation on another chromosome. Fim<sup>d02114</sup> and SmnRNAi; Fim<sup>d02114</sup> have Tubulin:Gal4 in the background; this Gal4 transgene does not alter Smn defects (data not shown).</p><p>
<italic>C. elegans</italic> orthologs of <italic>Drosophila</italic> and human genes were identified by BLAST searching at NCBI. When a clear ortholog was not identified by reciprocal BLAST analysis, the most similar <italic>C. elegans</italic> genes were generally tested. <italic>plst-1</italic> corresponds to exons of predicted adjacent genes Y104H12BR.1 and Y104H12BL.1 based on similarity searching. <italic>T02G5.3</italic> corresponds to exons of <italic>T02G5.3</italic>, <italic>T02G5.2</italic>, and <italic>T02G5.1</italic> based on high-throughput cDNA sequencing and gene prediction programs <xref ref-type="bibr" rid="pgen.1001172-Hillier1">[99]</xref>. New gene predictions have been reported to Wormbase. To assemble the interaction map in <xref ref-type="fig" rid="pgen-1001172-g007">Figure 7</xref>, literature pertaining to each modifier gene was examined at NCBI, AceView, <italic>C. elegans</italic> and yeast on-line databases (Wormbase and SGD) to identify functional or direct interactions between modifier genes and neurodegenerative disease genes.</p><p>The L4440 vector <xref ref-type="bibr" rid="pgen.1001172-Timmons1">[47]</xref> was used to clone PCR products corresponding to <italic>B0432.13</italic>, <italic>dhs-22</italic> and <italic>ugt-49</italic> genes. Plasmids were transformed into the bacterial strain HT115(DE3) <xref ref-type="bibr" rid="pgen.1001172-Timmons1">[47]</xref>, <xref ref-type="bibr" rid="pgen.1001172-Timmons2">[49]</xref>. Primers used for cloning were: <italic>B0432.13</italic> forward <named-content content-type="gene">5&#8242;-acaagctctcgacatcgctg-3&#8242;</named-content>, reverse <named-content content-type="gene">5&#8242;- ttaatcgccgcatcctcttg -3&#8242;</named-content>; <italic>dhs-22</italic> forward <named-content content-type="gene">5&#8242;-tatgctgtgcagaagcgaag-3&#8242;</named-content>, reverse <named-content content-type="gene">5&#8242;-ctgcttgattcctggtgtattc-3&#8242;</named-content>; <italic>ugt-49</italic> forward <named-content content-type="gene">5&#8242;-acgtggatgtagctgaatgg-3&#8242;</named-content>, reverse <named-content content-type="gene">5&#8242;- acgtgaagaacagcaacgaac-3&#8242;</named-content>.</p><p>For analysis of modifier genes, animals were reared for two generations/5 days on plates spread with bacterial RNAi strains from the Ahringer or Vidal RNAi libraries <xref ref-type="bibr" rid="pgen.1001172-Kamath2">[53]</xref>. RNAi clones corresponding to modifier genes in <xref ref-type="table" rid="pgen-1001172-t004">Table 4</xref> were sequenced to confirm accuracy. The <italic>hlh-4(RNAi)</italic> clone in the feeding library was incorrect. A <italic>Cesmn-1(ok355)</italic>;<italic>hlh-4(tm604)</italic> double mutant strain was generated. <italic>hlh-4(tm604)</italic> did not affect the pharyngeal pumping rates of <italic>Cesmn-(lf)</italic> (data not shown) and <italic>hlh-4</italic> was excluded from further analysis. Length and GFP fluorescence was determined using the COPAS Biosorter (Union Biometrica, Holliston, MA) and the percentage of large animals was determined for each genotype <xref ref-type="bibr" rid="pgen.1001172-Pulak1">[51]</xref>. Three to six independent determinations were undertaken for each genotype/RNAi culture. Significant changes from <italic>empty(RNAi)</italic> were calculated for each RNAi/genotype using the two-tailed Mann-Whitney <italic>U</italic> test.</p><p>The average pharyngeal pumping rates of animals were determined after 3 days (at 25, 25 and 20&#176;C) post-hatching on empty vector (<italic>empty(RNAi)</italic>) or candidate gene RNAi bacterial feeding strains. Animals were videotaped while feeding for 10 seconds with an AxioCam ICc1 camera on a Zeiss Stemi SV11 at 20 to 66&#215; magnification. Movies were slowed before counting pumping rates. Pharyngeal grinder movements in any axis were scored as a pumping event. Average pumping rates (&#177; standard error of the mean, S.E.M) for each genotype/treatment were calculated independently in two to four separate experiments. The percent change in pumping rate on empty vector <italic>versus</italic> candidate gene RNAi was determined for each trial for both <italic>Cesmn-1(lf)</italic> homozygous and <italic>+/Cesmn-1(lf)</italic> heterozygous animals and used to calculate the mean, S.E.M, and significance.</p><p>
<italic>hT2(bli-4(e937) let-?(q782) qIs48[myo-2p::GFP]) (I;III)</italic> animals were reared in liquid cultures in a 96-well plate format on RNAi feeding strains <xref ref-type="bibr" rid="pgen.1001172-Kamath2">[53]</xref>. At least two independent cultures corresponding to each <italic>C. elegans</italic> RNAi feeding clone were established. Concentrated dsRNA expressing bacteria was added to cultures as necessary to prevent starvation. Cultures were maintained for 8 days at 25&#176;C to generate sufficient animals for analysis. Length and fluorescence were determined using the COPAS BioSorter (Union Biometrica, Holliston, MA). Data was exported to Excel (Microsoft Corp.) for analysis. Thirty-one clones were identified that modified the average length of <italic>Cesmn-1(lf)</italic> animals relative to <italic>+/Cesmn-1(lf)</italic> siblings in both trials. Four of these genes altered <italic>Cesmn-1(lf)</italic> size relative to <italic>+/Cesmn-1(lf)</italic> siblings in at least 40% of subsequent trials and these were selected as candidate modifier genes for neuromuscular analysis as described in the text.</p><p>Three males and three virgin females were placed on fresh food at 25&#176;C on day 1. Eggs were collected for next 2 days (Set 1), and the parents transferred to fresh food. Eggs were collected for another 2 days (Set 2), and the parents discarded. The F1 animals were scored after 15 days,- on the 16<sup>th</sup> day for the first set, and the 19th day for the second set, from day 1. White pupae were scored as early stage death and black pupae were scored as late stage death. Control crosses of tubGAL4:FL26B(Smn RNAi) out-crossed to the wild type strain were used as a control for every experimental set. Significance was determined by Chi-square analysis.</p><p>Primary antibodies were used at the following dilutions: monoclonal anti-DLG (1&#8758;500) (Developmental Studies Hybridoma Bank), polyclonal anti-Synaptotagmin (1&#8758;1000) (a gift from Hugo Bellen). FITC- (1&#8758;40) and Cy5- (1&#8758;40) conjugated anti-rabbit and anti-mouse secondary antibodies were purchased from Jackson Immunoresearch Laboratories. Anti-Disc large used at 1&#8758;100 (Hybridoma) and anti-HRP used at 1&#8758;1000 (Cappell). 3<sup>rd</sup> instar larvae were dissected and fixed for 5 minutes in Bouin's fixative. Stained specimens were mounted in FluoroGuard Antifade Reagent (Bio-Rad), and images were obtained with a Zeiss LSM510 confocal microscope. Bouton numbers were counted based on the Discs large and Synaptotagmin staining in the A2 segment between muscles 6 and 7 or muscle 4 as indicated. The ratio of muscle area for the various genotypes was normalized to wild type. At least 10&#8211;12 animals of each genotype were dissected for the bouton analysis. The ANOVA multiple comparison test was used for statistical analysis.</p>