<p>Strains used in this study are listed in <xref ref-type="supplementary-material" rid="pgen.1001226.s007">Table S1</xref>. <italic>Aspergillus nidulans</italic> strains; TNO2A3 (<italic>nkuA</italic>&#916;) <xref ref-type="bibr" rid="pgen.1001226-Nayak1">[47]</xref>, AGB152 <xref ref-type="bibr" rid="pgen.1001226-Busch2">[48]</xref>, AGB154 <xref ref-type="bibr" rid="pgen.1001226-Bayram5">[49]</xref> served as wild type transformation hosts for the deletion and epitope tagging as well as overexpression experiments. Transformation of the <italic>vosA::ctap</italic> linear construct into AGB152 yielded AGB509 strain. <italic>laeA</italic> deletion cassette containing <italic>ptrA</italic> marker was transformed into TNO2A3 generating <italic>laeA</italic>&#916;/<italic>veA1</italic> (AGB468) which was then crossed with AGB154. This crossing gave rise to prototrophic deletion strains <italic>laeA</italic>&#916;/<italic>veA1</italic> (AGB512) and <italic>laeA</italic>&#916;/<italic>veA+</italic> (AGB493), respectively. AGB493 and AGB509 strains were crossed in order to obtain <italic>vosA::ctap</italic>, <italic>laeA</italic>&#916;/<italic>veA</italic>+ combination (AGB510). The <italic>velB::ctap</italic>, <italic>laeA</italic>&#916;/<italic>veA</italic>+ hybrid (AGB511) was created by crossing AGB493 with AGB389 strain. The presence of wild type <italic>veA</italic>+ allele was verified by analytical PCR of the locus followed by <italic>BstX</italic>I digestion. <italic>laeA</italic> deletion as well as <italic>vosA</italic>- and <italic>velB</italic>-tap loci were confirmed by Southern blot (<xref ref-type="supplementary-material" rid="pgen.1001226.s006">Figure S6</xref>). AGB513 strain that contains <italic>veA::ctap</italic> in <italic>laeA</italic>&#916; strain was created by introducing pME3711 into AGB512. <italic><sup>p</sup>mutA::sgfp</italic> reporter plasmid, pME3296, was introduced into AGB152 (<italic>wt</italic>) and <italic>laeA</italic>&#916; (AGB493) strains yielding AGB514 and AGB515, respectively. The BIFC plasmids, pME3714 (<italic>nyfp::velB</italic>/<italic>cyfp::vosA</italic>), pME3715 (<italic>cyfp::velB/nyfp::vosA</italic>), and pME3717 (<italic>nyfp::velB</italic>/<italic>cyfp::velB</italic>) were introduced into the recipient strain AGB506 yielding AGB516 (<italic>velB</italic>-<italic>vosA</italic>), AGB517 (<italic>vosA</italic>-<italic>velB</italic>), and AGB543 (<italic>velB</italic>-<italic>velB</italic>) BIFC strains, respectively. pME3715 was transformed into <italic>laeA</italic>&#916; (AGB468), resulting in AGB544 (<italic>velB</italic>-<italic>vosA, laeA</italic>&#916;). <italic>nosA</italic> OE construct (pME3719) was placed in AGB493, which led to AGB545. Integration of the plasmids into the genome was confirmed by diagnostic PCR. DH5&#945; and MACH-1 (I<sc>nvitrogen</sc>) <italic>Escherichia coli</italic> strains were applied for recombinant plasmid DNA. <italic>Aspergillus</italic> and <italic>E. coli</italic> strains were cultured as described previously <xref ref-type="bibr" rid="pgen.1001226-Bayram3">[14]</xref>.</p><p>Tranformation of <italic>E. coli</italic> and <italic>A. nidulans</italic> was performed as explained in detail <xref ref-type="bibr" rid="pgen.1001226-Hanahan1">[50]</xref>, <xref ref-type="bibr" rid="pgen.1001226-Punt1">[51]</xref>.</p><p>During processing and construction of linear and circular DNAs, standard recombinant DNA technology protocols were followed as given in detail <xref ref-type="bibr" rid="pgen.1001226-Sambrook1">[52]</xref>. Plasmids and oligonucleotides (I<sc>nvitrogen</sc>) employed in the course of this study are listed in <xref ref-type="supplementary-material" rid="pgen.1001226.s008">Table S2</xref> and <xref ref-type="supplementary-material" rid="pgen.1001226.s009">Table S3</xref>, respectively. PCR reactions <xref ref-type="bibr" rid="pgen.1001226-Saiki1">[53]</xref> were performed with various DNA polymerase combinations including <italic>Pfu</italic> (MBI F<sc>ermentas</sc>), <italic>Phusion</italic> (F<sc>innzymes</sc>), <italic>Platinum</italic>-<italic>Taq</italic> (I<sc>nvitrogen</sc>) and <italic>Taq</italic> polymerases.</p><p>In order to create <italic>laeA</italic> deletion construct 5&#8242; UTR region of <italic>laeA</italic> was amplified from the wild type genomic DNA with primers OSB22/24 and 3&#8242; UTR region was amplified with OSB25/27. The two amplicons were fused to the <italic>ptrA</italic> marker (from pPTRII) with fusion PCR <xref ref-type="bibr" rid="pgen.1001226-Nayak1">[47]</xref> (nested oligos OSB23/26) yielding 4324 bp linear deletion construct which was used to transform TNO2A3 to AGB468. For complementation of <italic>laeA</italic>&#916;, the <italic>laeA</italic> genomic locus (3.7 kb), containing 1.5 kb promoter and 1 kb terminator regions, was amplified from genomic DNA (OSB22/27) and cloned into the <italic>Stu</italic>I site of pAN8-1 (<italic>phleo</italic>
<sup>R</sup>) which yielded pME3635. Then pME3635 was introduced into <italic>laeA</italic>&#916; strains, (<italic>veA</italic>+, AGB493) and (<italic>veA1</italic>, AGB512), resulting in AGB494 and AGB518, respectively. In order to overexpress <italic>laeA</italic> gene, <italic>laeA</italic> cDNA was amplified from cDNA library (OZG61/62) and inserted into the <italic>Pme</italic>I site (pME3718) under nitrogen source regulable <italic>niiA</italic> promoter, generating pME3716. This plasmid was eventually introduced into AGB152, which resulted in AGB519.</p><p>To replace the <italic>vosA</italic> locus with <italic>vosA::ctap</italic>, <italic>vosA</italic> ORF including 1 kb of the <italic>vosA</italic> promoter (oligos VosA&#8211;A/C) and 1 kb <italic>vosA</italic> terminator (VosA&#8211;D/F) were amplified from genomic DNA and the resulting amplicons were fused to the <italic>ctap::natR</italic> module via fusion PCR (VosA&#8211;B/E). Gene replacement cassette was introduced into AGB152 and the substitution of the <italic>vosA</italic> locus by <italic>vosA::ctap</italic> was verified by Southern blot hybridization (<xref ref-type="supplementary-material" rid="pgen.1001226.s006">Figure S6</xref>).</p><p>
<italic>velB</italic> cDNA was amplified (OZG397/64 for <italic>n-yfp</italic>, OZG63/64 for <italic>c-yfp</italic> fusion) from sexual cDNA library. Then <italic>n-</italic> (OZG73/387) and <italic>c-yfp</italic> (OZG75/77) amplicons were fused to <italic>velB</italic> cDNAs with oligos OZG397/64 (<italic>n-yfp::velB</italic>) and OZG63/64 (<italic>c-yfp::velB</italic>), respectively. <italic>n-yfp::velB</italic> and <italic>c-yfp::velB</italic> were cloned into the <italic>Pme</italic>I site of pME3160 yielding plasmids pME3712 and 3713, respectively. <italic>vosA</italic> cDNA was also amplified (OZG436/438 for <italic>n-yfp</italic>, OZG437/438 for <italic>c-yfp</italic> fusion) from sexual cDNA library. <italic>vosA</italic> cDNA amplicons (OZG436/438) and (OZG437/438) were fused to <italic>n-yfp</italic> (OZG73/387) and <italic>c-yfp</italic> (OZG75/388) via fusion PCR <xref ref-type="bibr" rid="pgen.1001226-Szewczyk1">[54]</xref>. <italic>c-yfp::vosA</italic> and <italic>n-yfp::vosA</italic> fragments were inserted into <italic>Swa</italic>I site of pME3712 and 3713, respectively. Plasmids bearing <italic>n-yfp::velB</italic>/<italic>c-yfp::vosA</italic> and <italic>c-yfp::velB</italic>/<italic>n-yfp::vosA</italic> were named as pME3714 and pME3715, respectively. For the analysis of VelB-VelB dimer formation, <italic>c-yfp::velB</italic> fragment was cloned into the <italic>Swa</italic>I site of pME3712 generating pME3717. <italic>nosA</italic> cDNA, which was amplified from sexual cDNA library (OZG320/321), was cloned into the <italic>Pme</italic>I site of pME3718 yielding pME3719.</p><p>The <italic>veA::ctap</italic> fusion construct encompassing the promoter and terminator sequences was amplified from pME3157 with oligos OZG304/305. This amplicon was cloned in the blunted <italic>Apa</italic>I site of pNV1 <xref ref-type="bibr" rid="pgen.1001226-Seiler1">[55]</xref> generating pME3711.</p><p>Northern <xref ref-type="bibr" rid="pgen.1001226-Brown1">[56]</xref> and Southern <xref ref-type="bibr" rid="pgen.1001226-Southern1">[57]</xref> hybridization experiments were performed as given in detail <xref ref-type="bibr" rid="pgen.1001226-Bayram1">[9]</xref>. Band densities in the Northern blots were analyzed with IMAGEJ (National Institutes of Health) and normalized against rRNA. DNA and amino acid sequences were analyzed by using L<sc>asergene</sc> software (DNAstar). Northern blot probes were generated by PCR amplification of the following genes (primer sets): <italic>abaA</italic> cDNA (<italic>abaA5</italic>/<italic>abaA3</italic>), <italic>brlA</italic> cDNA (<italic>brlA5</italic>/<italic>brlA3</italic>), <italic>mutA</italic> cDNA (<italic>mutA5</italic>/<italic>mutA3</italic>), <italic>nosA</italic> cDNA (<italic>nosA5</italic>/<italic>nosA3</italic>), <italic>steA</italic> gDNA (<italic>steA5</italic>/<italic>steA3</italic>), <italic>nsdD</italic> cDNA (<italic>nsdD5</italic>/<italic>nsdD3</italic>), <italic>aflR</italic> gDNA (<italic>aflR5</italic>/<italic>aflR3</italic>), <italic>laeA</italic> cDNA (OZG61/OZG62), <italic>gpdA</italic> gDNA (<italic>gpdA5</italic>/<italic>gpdA3</italic>), <italic>tpsA</italic> gDNA (OMN176/OMN177), <italic>orlA</italic> gDNA (OMN182/OMN183), <italic>ipnA</italic> gDNA (<italic>ipnA5</italic>/<italic>ipnA3</italic>), and <italic>stcU</italic> gDNA (<italic>stcU5</italic>/<italic>stcU3</italic>).</p><p>Viability of spores was examined as described <xref ref-type="bibr" rid="pgen.1001226-Ni1">[16]</xref>. Two-day old conidia (10<sup>5</sup> per plate) of wild type and the mutants were spread on solid minimal medium (MM) and incubated at 37&#176;C. After 2&#8764;10 days the conidia were collected and counted in a hemocytometer. Approximately 200 conidia were inoculated on solid MM and incubated for 2 days at 37&#176;C. Survival rates were calculated as a ratio of the number of growing colonies to the number of spores inoculated. This test was performed in triplicate.</p><p>Trehalose was extracted from conidia and analyzed as described previously <xref ref-type="bibr" rid="pgen.1001226-Ni1">[16]</xref>, <xref ref-type="bibr" rid="pgen.1001226-dEnfert1">[58]</xref>. Two-day old conidia (2&#215;10<sup>8</sup>) were collected and washed with ddH<sub>2</sub>O. Conidia were resuspended in 200 &#181;l of ddH<sub>2</sub>O and incubated at 95&#176;C for 20 min and the supernatant was collected by centrifugation. The supernatant was mixed with equal volume of 0.2 M sodium citrate (pH 5.5) and samples were incubated at 37&#176;C for 8 h with or without 3 mU of trehalase (S<sc>igma</sc>), which hydrolyzes trehalose to glucose. The amount of glucose generated was assayed with a glucose assay kit (S<sc>igma</sc>). The amount of glucose by deducting trehalase untreated sample from trehalase-treated sample was converted into the trehalose amount (pg) per conidium (triplicate).</p><p>Oxidative stress tolerance test was carried out as described previously <xref ref-type="bibr" rid="pgen.1001226-Han2">[59]</xref>. Hydrogen peroxide sensitivity of conidia was tested by incubating 1 ml of conidial suspensions containing 10<sup>5</sup> conidia with varying concentrations (0.0, 0.25 or 0.5 M) of H<sub>2</sub>O<sub>2</sub> for 30 min at RT. Each conidia suspension was then diluted with ddH<sub>2</sub>O, and the conidia were inoculated into solid MM. After incubation at 37&#176;C for 48 h, colony numbers were counted and calculated as a ratio to the untreated control. Sensitivity to oxidative stress was also tested by spotting 10 &#181;l of serially diluted conidia (10 to 10<sup>5</sup>) on solid MM with 0, 2.5, 5 M of H<sub>2</sub>O<sub>2</sub> and incubated at 37&#176;C for 48 h.</p><p>UV tolerance test was carried out as described previously <xref ref-type="bibr" rid="pgen.1001226-Lima1">[60]</xref> with a slight modification. Two-day old conidia were collected in ddH<sub>2</sub>O and plated out on solid MM (100 conidia per plate). The plates were then irradiated immediately with UV using a UV crosslinker and the plates were further incubated at 37&#176;C for 48 h. The colony numbers were counted and calculated as a ratio to the untreated control. UV sensitivity was also tested by spotting 10 &#181;l of serially diluted conidia (10 to 10<sup>5</sup>) on solid MM, which were then irradiated with UV and incubated at 37&#176;C for 48 h.</p><p>For detection of GFP signal in 80 &#181;g protein extracts, &#945;-gfp mouse antibody (S<sc>anta</sc>C<sc>ruz</sc>) was used in combination with O<sc>ne</sc>-H<sc>our</sc> Western kit (G<sc>enscript</sc>). &#945;-Calmodulin rabbit antibodies (M<sc>illipore</sc>) in 1&#8758;1000 dilution in TBS 5% (w/v) non fat dry milk and secondary goat &#945;-rabbit antibodies 1&#8758;1000 in dilution in TBS 5% (w/v) milk were used for the recognition of TAP tag fusion proteins in 80 &#181;g protein extracts. Polyclonal &#945;-VeA antibody recognizing the native VeA protein was raised in rabbit (G<sc>enscript</sc>). &#945;-VeA antibody (5 &#181;g) in TBST 5% (w/v) milk 0.2% (v/v) Tween-20 was used for the detection of the VeA protein in 80 &#181;g protein extracts in immunoblotting.</p><p>Protein extracts were prepared in B buffer (100 mM Tris pH 7.5, 300 mM NaCl, 10% Glycerol, 0,1% NP-40, 1 mM DTT, protease inhibitor mix (R<sc>oche</sc>)) without phosphatase inhibitors. Total protein extract (1 mg) was treated with 10 units of Shrimp Alkaline Phosphatase (SAP, MBI F<sc>ermentas</sc>) at 37&#176;C for 30 min. SAP-treated extracts were used for immunoblotting.</p><p>Tap tag experiments and preparation of the protein crude extracts were performed as explained in detail <xref ref-type="bibr" rid="pgen.1001226-Bayram1">[9]</xref>.</p><p>Protocols given elsewhere <xref ref-type="bibr" rid="pgen.1001226-Bayram1">[9]</xref> were followed for further data processing and analysis of the proteins.</p><p>
<italic>A. nidulans</italic> spores (2000) were inoculated in 8 chambered borosilicate coverglass system (N<sc>unc</sc>) supplemented with liquid medium. Fluorescence photographs were taken with an A<sc>xiovert</sc> O<sc>bserver</sc>. Z1 (Z<sc>eiss</sc>) microscope equipped with a Q<sc>uant</sc>EM:512SC (P<sc>hotometrics</sc>) digital camera and the S<sc>lide</sc>B<sc>ook</sc> 5.0 software package (I<sc>ntelligent</sc> I<sc>maging</sc> I<sc>nnovations</sc>). For BIFC and GFP studies the following parameters were used; YFP filter 1000 milliseconds (ms), RFP filter 600 ms, DAPI filter 40 ms, DIC filter 200 ms, and GFP filter 400 ms.</p><p>Extraction of ST and running on TLC plates were performed as described in detail elsewhere previously <xref ref-type="bibr" rid="pgen.1001226-Bayram5">[49]</xref>.</p>