<p>All carriers were recruited to studies (<xref ref-type="table" rid="pgen-1001183-t002">Table 2</xref>) at the host institutions under IRB-approved protocols.</p><p>A total of 6,272 <italic>BRCA2</italic> carriers from 39 studies (<xref ref-type="table" rid="pgen-1001183-t002">Table 2</xref>) and 14 countries contributed DNA samples for this project. With the exception of NICC, all studies are members of the Consortium of Investigators of Modifiers of BRCA1/2 (CIMBA) <xref ref-type="bibr" rid="pgen.1001183-ChenevixTrench1">[28]</xref>. Recruitment of carriers were conducted predominantly through cancer genetics clinics, and enrolled through national or regional efforts. Other studies were recruited through population-based or community-based ascertainments. All subjects provided written informed consent. Eligible female carriers were aged 18 years or older, were self-reported &#8216;white&#8217;, and had mutations in <italic>BRCA2</italic>. Data were available on age at study recruitment, age at cancer diagnosis, age of bilateral prophylactic mastectomy, <italic>BRCA1/2</italic> mutation description, and self-reported ethnicity. Only a limited number of cases had detailed information on tumor characteristics (e.g., estrogen and progesterone receptor status); therefore, subtype analyses were not performed at this stage.</p><p>All eligible DNA samples provided by participating centers were subjected to a rigorous quality control assessment, including measures of overall DNA quality and quantity. A total of 1,156 young (&#8804;50 years) affected women and 1,038 unaffected women with high quality DNA samples were selected (<xref ref-type="table" rid="pgen-1001183-t002">Table 2</xref>). For time efficiency, stage 1 genotyping occurred in two phases: phase 1 included 421 cases and 404 controls and phase 2 included 735 cases and 634 controls.</p><p>Prior to the genome-wide scan, we genotyped five SNPs previously genotyped by the CIMBA study centers as a pre-filter for sample identification. Thirty-one samples (<xref ref-type="supplementary-material" rid="pgen.1001183.s001">Figure S1</xref>) were discordant in the two genotyping rounds and were excluded from further analysis.</p><p>The genotyping for the stage 1 GWAS was performed on 2,163 eligible carriers using the Affymetrix 6.0 GeneChip array that included 906,622 SNPs (<xref ref-type="supplementary-material" rid="pgen.1001183.s001">Figure S1</xref>). To further monitor the identity of the DNA samples, a fingerprinting panel of 14 SNPs with a minor allele frequency &gt;10% in HapMap European individuals were genotyped on all samples, using Sequenom iPLEX, before and after Affymetrix genotyping. The AMG gender assay was used for gender assessment. As an additional quality control measure, cases and controls were interleafed on each plate to eliminate technical bias. Each plate also included one HapMap CEU DNA sample.</p><p>The DNA samples and genotyping calls for both phases of stage 1 were filtered through a series of data quality control parameters using the Birdseed module of the Birdsuite software developed at Broad Institute <xref ref-type="bibr" rid="pgen.1001183-Korn1">[29]</xref>. Among the 2,163 samples genotyped in the stage 1 GWAS, 253 failed to hybridize to the chip due to poor DNA quality and were excluded (<xref ref-type="supplementary-material" rid="pgen.1001183.s001">Figure S1</xref>). Fifty-five samples were dropped with call rates &lt;95%. Three samples were contaminated, 43 were identified by genotyping to be duplicates, and 4 were male; all were dropped from analyses.</p><p>SNPs were also filtered using Birdseed and were removed if monomorphic or &gt;10% missing (n&#8202;=&#8202;38,962), genotype call rates &lt;95% (n&#8202;=&#8202;50,810), minor allele frequencies &lt;1% (n&#8202;=&#8202;104,792), departures from Hardy-Weinberg Equilibrium (p&lt;10<sup>&#8722;6</sup>; n&#8202;=&#8202;1,090), differential missingness with respect to phenotype (p&lt;10<sup>&#8722;3</sup>; n&#8202;=&#8202;275), and differential missingness with respect to nearby SNPs (p&lt;10<sup>&#8722;10</sup>; n&#8202;=&#8202;22,065). A total of 6,212 SNPs had different missingness patterns in phase 1 compared to phase 2, and were excluded. Since we found that significant missingness correlated to SNPs mapping to longer fragments of Affymetrix 6.0 digestion products, we also removed the SNPs on fragments longer than 1000bp (n&#8202;=&#8202;85,990).</p><p>With the remaining 1,805 carriers and 596,426 SNPs, an iterative process proceeded to drop all individuals with low call rates (&lt;95%), high autosomal heterozygosity rates (false discovery rate &lt;0.1%), and high identity by descent scores (&#8805;0.95) and to drop all SNPs with minor allele frequencies &lt;1% and SNP call rates &lt;95% until the final run contained individuals above the individual and SNP filter thresholds (n&#8202;=&#8202;1,747 samples and 592,566 SNPs). A more stringent HWE filter (p&lt;10<sup>&#8722;7</sup>) was then applied and 403 additional SNPs were dropped. Nine individuals with missing mutation descriptions were removed.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Finally, principal components analysis was used to identify the ethnic outliers (<xref ref-type="supplementary-material" rid="pgen.1001183.s005">Figure S5</xref>). A total of 1,743 <italic>BRCA2</italic> mutation carriers and the HapMap3 data for 210 individuals of European (CEU), Han Chinese (CHB), and Yoruba (YRI) African descent were available for multidimensional scaling using the genomic kinship matrix estimated using a set of 53,641 autosomal and uncorrelated SNPs. A cut-off of &gt;11% was used to exclude samples with non-CEU ancestry (n&#8202;=&#8202;35). Genotype-phenotype association analyses were based on 1,703 (899 young affected and 804 unaffected) <italic>BRCA2</italic> mutation carriers and 592,163 SNPs, covering 85% of the common HapMap 3 SNPs (imputed with <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001183.e016.jpg" /></inline-formula> (see below), including 64% of the markers that were removed in the QC process).</p><p>Where directly genotyped data were not available, probabilities were imputed with Beagle.3.0.2 (using the default parameters) using CEU+TSI samples on HapMap3 release2 B36 as the reference panel (410 chromosomes, 1.4 M SNPs).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The primary SNP selection strategy was based on the results of the kinship-adjusted score test of 592,163 GWAS genotyped SNPs. From stage 1, a total of 79 top independent regions (<inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001183.e017.jpg" /></inline-formula>) with pairwise r<sup>2</sup> values&lt;0.80 were selected for genotyping in stage 2 (<xref ref-type="supplementary-material" rid="pgen.1001183.s006">Figure S6</xref>). For the top 10 SNPs if available, an additional correlated SNP (pairwise <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001183.e018.jpg" /></inline-formula>; n&#8202;=&#8202;5) was selected to serve as genotyping backup. The remaining SNPs for stage 2 were selected based on two alternate strategies. First, we added the 14 (as well as <italic>FGFR2</italic> counted in the top 10 SNPs above) confirmed breast cancer SNPs from prior independent GWAS of sporadic breast cancer. Second, we also selected the 15 top independent regions (pairwise <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001183.e019.jpg" /></inline-formula>) based on the ranking of the p-values from a logistic regression analysis of 1.5 million imputed SNPs. In total for stage 2 replication phase, we selected 113 SNPs and 1,524 breast cancer carriers and 1,508 control carriers (<xref ref-type="table" rid="pgen-1001183-t002">Table 2</xref>) for genotyping using the Sequenom iPLEX platform.</p><p>Samples were excluded for call rates &#8804;95% (n&#8202;=&#8202;476), duplication in stage 2 (identity by state (IBS)&#8764;1.0; n&#8202;=&#8202;43), duplication in stage 1 and 2 (IBS; n&#8202;=&#8202;25), lack of complete phenotype data (n&#8202;=&#8202;1), and insufficient country-specific numbers (n&#8202;=&#8202;1; <xref ref-type="supplementary-material" rid="pgen.1001183.s006">Figure S6</xref>). A total of 100 SNPs were successfully multiplexed into three pools; the remaining 13 SNPs were not genotyped. Genotyping QC filters excluded 15 SNPs due to call rates &#8804;90% (n&#8202;=&#8202;14) and MAF&lt;1% (n&#8202;=&#8202;1). In summary, the final association analyses in stage 2 were based on 2,486 carriers (1,264 affected and 1,222 unaffected carriers) and 85 SNPs.</p><p>Carriers were censored at the first breast or ovarian cancer or bilateral prophylactic mastectomy, whichever occurred first. Carriers who developed any cancer were censored at time of bilateral prophylactic mastectomy if it occurred more than a year prior to the cancer diagnosis (to avoid censoring at bilateral mastectomies related to diagnosis in which rounded ages were used). The remaining carriers were censored at the age of last observation. This was defined either by the age/date at interview or age at follow-up depending on the information provided by the participating center. Carriers censored at diagnosis of breast cancer were considered cases in the analysis. Mutation carriers censored at ovarian cancer diagnosis were considered unaffected. Carriers with a censoring/last follow-up age older than age 80 were censored at age 80 because there are no reliable cancer incidence rates for <italic>BRCA1/2</italic> carriers beyond age 80.</p><p>Analyses, based on 1,703 <italic>BRCA2</italic> mutation carriers and 592,163 SNPs, were performed within a survival analysis framework. Since the mutation carriers were not selected at random with respect to their disease status, standard methods of survival (e.g., Cox regression) may lead to biased estimates of relative risk <xref ref-type="bibr" rid="pgen.1001183-Antoniou9">[30]</xref>. Therefore, analyses were conducted by modeling the retrospective likelihood of the observed genotypes conditional on the disease phenotypes. The associations between genotype and breast cancer risk at both stages were assessed using the 1-degree of freedom score test statistic based on this retrospective likelihood, as previously described <xref ref-type="bibr" rid="pgen.1001183-Antoniou4">[9]</xref>, <xref ref-type="bibr" rid="pgen.1001183-Antoniou6">[18]</xref>. All models were stratified by country of study and 6174delT (c.5946delT) mutation status, the most common <italic>BRCA2</italic> mutation in this study and a marker of the Ashkenazi Jewish population among Ashkenazi Jewish women <xref ref-type="bibr" rid="pgen.1001183-Tonin1">[31]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1001183-Neuhausen3">[33]</xref>. Since the linkage disequilibrium structure among Ashkenazi Jewish people may differ from other mutation carriers <xref ref-type="bibr" rid="pgen.1001183-Olshen1">[34]</xref>, stratifying by the *6174delT provides additional control for population stratification. To allow for the non-independence among related individuals, an adjusted version of the score test was used in which the variance of the score was derived by taking into account the correlation between the genotypes <xref ref-type="bibr" rid="pgen.1001183-Amin1">[35]</xref>, <xref ref-type="bibr" rid="pgen.1001183-Leutenegger1">[36]</xref>. Analyses were performed in R using the GenABEL libraries <xref ref-type="bibr" rid="pgen.1001183-Aulchenko1">[37]</xref> and custom written software.</p><p>To estimate the magnitude of the associations, the effect of each SNP was modelled either as a per allele hazard ratio (HR) (i.e., multiplicative model) or as separate HRs for heterozygotes and homozygotes, and these were estimated on the log scale. The HRs were assumed to be independent of age (i.e. we used a Cox proportional-hazards model). For the most significant novel associations this assumption was verified by adding a genotype-by-age interaction term to the model to fit models in which the HR changed with age. The retrospective likelihood was implemented in the pedigree-analysis software MENDEL <xref ref-type="bibr" rid="pgen.1001183-Lange1">[38]</xref> as previously described <xref ref-type="bibr" rid="pgen.1001183-Antoniou4">[9]</xref>. All analyses were stratified by country of residence and 6174delT (c.5946delT) mutation status, and used calendar-year- and cohort-specific breast cancer incidence rates for <italic>BRCA2</italic>
<xref ref-type="bibr" rid="pgen.1001183-Antoniou8">[25]</xref>. The combined stage 1 and stage 2 analyses were also stratified by stage. Parameter estimates were obtained by maximising the retrospective likelihood. To allow for the non-independence among related mutation carriers, we used a robust variance estimation approach in order to obtain standard errors for the parameters <xref ref-type="bibr" rid="pgen.1001183-Huber1">[39]</xref>, <xref ref-type="bibr" rid="pgen.1001183-Lin1">[40]</xref>. Related individuals were identified through a unique family identifier.</p><p>We also examined the association of both high-frequency and low-frequency copy number variants (CNV) to the age of diagnosis of breast cancer as a dichotomous trait using the stage 1 data <xref ref-type="bibr" rid="pgen.1001183-Korn1">[29]</xref>. We called known, common variants (copy number polymorphisms, CNPs) with Canary <xref ref-type="bibr" rid="pgen.1001183-Korn1">[29]</xref>. CNP alleles lower than 1% in frequency were removed, to maximize the number of the CNPs that were bi-allelic instead of multi-alleleic. CNPs were removed that had for call rate &lt;95%, differential missingness by genotype (p&lt;10<sup>&#8722;3</sup>), or departure from Hardy-Weinberg proportions (p&lt;10<sup>&#8722;3</sup>). Post-QC, we had 191 high-quality genotyped polymorphisms. We used PLINK to assess association using logistic regression and the same ancestry covariates of no interest as with SNPs. We similarly assessed less common CNVs discovered by Birdseye <xref ref-type="bibr" rid="pgen.1001183-Korn1">[29]</xref> for association with age at diagnosis using PLINK <xref ref-type="bibr" rid="pgen.1001183-Purcell1">[41]</xref>. Finally, we also looked specifically at CNVs overlapping the <italic>BRCA2</italic> gene itself using LOD scores and Birdseye.</p><p>We looked for evidence of excess sharing across the genome and the <italic>BRCA2</italic> region. Using GERMLINE <xref ref-type="bibr" rid="pgen.1001183-Gusev1">[23]</xref>, shared segments of greater than 5 cM were computed based on the imputed genotype dataset among both Ashkenazi (n&#8202;=&#8202;304) and non-Jewish (n&#8202;=&#8202;1,331) samples compared to samples from an autism study (n&#8202;=&#8202;808) (<xref ref-type="supplementary-material" rid="pgen.1001183.s003">Figure S3</xref>). Examining sites across the genome every 2.5 cM (excluding telomere and centromere regions), we computed the mean of the proportion, standard deviation, and the maximum values for non-Jewish and Ashkenazi women, respectively.</p><p>We tested whether 59 genes known to regulate or interact with <italic>BRCA2</italic>
<xref ref-type="bibr" rid="pgen.1001183-ODonovan1">[16]</xref> (<xref ref-type="supplementary-material" rid="pgen.1001183.s007">Table S1</xref>) were enriched for associations with age of onset of breast cancer in <italic>BRCA2</italic> mutation carriers, using a new implementation of Gene Set Enrichment Analysis (GSEA) called Meta-Analysis Gene-Set Enrichment of variaNT Associations (MAGENTA) <xref ref-type="bibr" rid="pgen.1001183-Segre1">[14]</xref>. The 59 genes were compiled using a Pubmed abstract mining software, Chilibot <xref ref-type="bibr" rid="pgen.1001183-Chen1">[42]</xref>, and were selected if they were related to the Fanconi anemia pathway <xref ref-type="bibr" rid="pgen.1001183-DAndrea1">[15]</xref> as well as others reported from literature to regulate or interact with <italic>BRCA1/2</italic>
<xref ref-type="bibr" rid="pgen.1001183-ODonovan2">[43]</xref>. An association p-value was calculated for each gene in the genome, defined as the most-significant association p-value of all genotyped SNPs that lie within 110 kb upstream and 40 kb downstream to the gene's most extreme transcript boundaries, followed by correction for gene score confounders (gene size, number of SNPs per gene and linkage disequilibrium related properties). SNP association p-values were taken from the stage 1 GWAS. To compute a GSEA p-value for the <italic>BRCA</italic> gene set, the fraction of genes with an association p-value more significant than the 95 percentile of all gene p-values in the genome was compared to a null distribution, generated by randomly sampling gene-sets of identical size from the genome 10,000 times. Of the 59 <italic>BRCA</italic> interactors, two genes were assigned the same most significant SNP due to physical proximity in the genome. To prevent potential over-estimation of gene set enrichment due to physical clustering of genes in a gene set, we retained only one gene of each subset of genes assigned the same best SNP (the gene with the most significant gene p-value) for the analysis of both the real and permuted gene sets.</p>