<p>Anonymized samples for this study were derived from allogeneic blood donor samples that already existed and would otherwise be discarded. As the samples were provided anonymously, the NIH Office Of Human Subjects Research approved the use of these samples on an exemption basis, per federal code (45CFR46), without the need for IRB review or informed consent.</p><p>The complete <italic>ERAP2</italic> coding region and some exon-adjacent intronic regions (8794 bp total, 2883 bp of which are protein coding) were sequenced in 180 individuals from 6 geographically diverse human groups. Specifically, we studied 30 individuals from each of the following HapMap <xref ref-type="bibr" rid="pgen.1001157-The2">[76]</xref> populations: Yoruba (Nigeria), Luhya (Kenya), Gujarati Indians (living in Houston, TX, USA), Han (China), and Toscani (Italy). As a representative Middle Eastern population, we also studied 30 Palestinian (Israel) individuals from the National Laboratory for the Genetics of Israeli Populations (Tel-Aviv University). The same 180 individuals were also used for sequencing portions of the <italic>ERAP1</italic> gene (9753 bp total, 2847 bp of coding sequence). The regions sequenced are shown in <xref ref-type="supplementary-material" rid="pgen.1001157.s001">Figure S1</xref>.</p><p>Regions of interest were PCR-amplified and sequenced (bidirectional Sanger-based sequencing), and SNPs were detected with Polyphred/Polyphrap. To minimize sequencing errors, variants residing within the first and last 50 bp of each amplified segment were discarded. Additionally, we manually reviewed all variants associated with discordant results between overlapping amplimers, variants with a quality score lower than 99, singletons, and triallelic SNPs. The ancestry of each SNP was inferred through comparison with the chimpanzee, orangutan, and macaque genome sequences <xref ref-type="bibr" rid="pgen.1001157-Chimpanzee1">[77]</xref>, <xref ref-type="bibr" rid="pgen.1001157-Gibbs1">[78, genome.ucsc.edu]</xref>. Fixed differences with chimpanzee were identified by comparison with the chimpanzee genome sequence <xref ref-type="bibr" rid="pgen.1001157-Chimpanzee1">[77]</xref>.</p><p>As a proxy for neutrality, we sequenced 47 control regions. Such regions consisted of unlinked, ancient processed pseudogenes that do not encode a functional protein and are thus expected to evolve in a neutral fashion. The control regions are not part of gene families, are far from genes, do not overlap putative functional elements, are conserved as pseudogenes in chimpanzees, orangutans, and macaques, and have recombination rates and GC contents similar to coding genes. Details about these control regions can be found in the <xref ref-type="supplementary-material" rid="pgen.1001157.s014">Text S1</xref>.</p><p>The generated sequence data were analyzed using three neutrality tests: MWUhigh, Tajima's D, and HKA. MWUhigh <xref ref-type="bibr" rid="pgen.1001157-Nielsen1">[37]</xref> compares the SFS of a region of interest with the SFS of a neutral region(s) (e.g., control regions) to determine whether the former is consistent with neutral expectations <xref ref-type="bibr" rid="pgen.1001157-Nielsen1">[37]</xref>. Specifically, we applied MWUhigh to the folded SFS, which becomes significant only in the case of an excess of intermediate-frequency alleles <xref ref-type="bibr" rid="pgen.1001157-Andrs1">[3]</xref>. Tajima's D <xref ref-type="bibr" rid="pgen.1001157-Tajima1">[38]</xref> compares two estimates of &#952; (the scaled mutation rate) and, when significantly positive, identifies genealogies with long internal branches consistent with long-term balancing selection. Finally, HKA <xref ref-type="bibr" rid="pgen.1001157-Hudson1">[39]</xref> identifies regions with an unusual density of polymorphisms when compared with divergence and with the patterns of neutral loci. For the HKA test, we focused only on coding regions and used the chimpanzee as an outgroup. MWUhigh was calculated using an in-house C script, while Tajima's D and HKA were calculated using libsequence <xref ref-type="bibr" rid="pgen.1001157-Thornton1">[79]</xref>.</p><p>The significance of all neutrality tests was assessed by 10,000 coalescent simulations with ms <xref ref-type="bibr" rid="pgen.1001157-Hudson3">[80]</xref>. Selecting an appropriate demographic model for the simulations is crucial to avoid spurious detection of signatures of selection. Our null model followed a recently published demographic scenario that included African, Asian, and European populations <xref ref-type="bibr" rid="pgen.1001157-Gutenkunst1">[81]</xref> and that was a better fit to our control data than previously proposed demographic models. The divergence to chimpanzee was adjusted in the simulations to fit the ratio of SNPs to fixed differences of the control regions. Simulations were conditioned on the total number of informative sites, and the recombination rate was set to 10<sup>&#8722;6</sup> per base pair, the estimated recombination rate of this genomic region (genome.ucsc.edu). All analyses were performed with an in-house PERL program (Neutrality Test Pipeline).</p><p>Haplotypes of the coding SNPs were inferred using PHASE <xref ref-type="bibr" rid="pgen.1001157-Stephens1">[82]</xref>, and the haplotype network was created with Network <xref ref-type="bibr" rid="pgen.1001157-Bandelt1">[83]</xref>. The estimated age of the haplotypes was calculated using Network and calibrated with chimpanzee, considering a divergence time of 6 Mya.</p><p>We analyzed the <italic>ERAP2</italic> cDNA from LCLs of HapMap Yoruba individuals with different genotypes: AA-homozygotes (GM18504, GM18505, GM11832, and GM07000), BB-homozygotes (GM18507, GM19240, GM12891, and GM12892), or AB-heterozygotes (GM18861, GM18870, GM19137, and GM19201). The cell lines were obtained from the Corriell Cell Repositories (ccr.coriell.org). Total RNA was isolated from each cell line using Trizol reagent (Invitrogen) and the RNeasy miniprep kit (Qiagen). cDNA was synthesized from 1 &#181;g of total RNA using the Superscript III First Strand Reverse Transcriptase Kit and random hexamers (Invitrogen). The <italic>ERAP2</italic> full-length transcript (exons 1 to 19) was amplified using Expand High Fidelity PCR System (Roche) from cDNA prepared from LCLs that were AA-homozygote (GM18504) or BB-homozygote (GM18508). These PCR products were cloned into the pCR4-TOPO vector (Invitrogen) and at least six clones for each haplotype were sequenced (3100 Genetic Analyzer, Applied Biosystems). Primer sequences for this experiment and for the exon 10 splice-variant screening can be found in <xref ref-type="supplementary-material" rid="pgen.1001157.s013">Table S4</xref>.</p><p>The effect of rs2248374 on <italic>ERAP2</italic> mRNA splicing was assessed using two <italic>in silico</italic> methods. First, we used GeneID <xref ref-type="bibr" rid="pgen.1001157-Parra1">[46]</xref> to predict the splicing of mRNA derived from the two haplotypes (<xref ref-type="supplementary-material" rid="pgen.1001157.s014">Text S1</xref>). Second, we used MaxEnt <xref ref-type="bibr" rid="pgen.1001157-Yeo1">[47]</xref> to predict the splicing potential of the constitutive splice site with: (1) the A allele: <named-content content-type="gene">ATGGT<underline>A</underline>AGG</named-content>; and (2) the G allele: <named-content content-type="gene">ATGGT<underline>G</underline>AGG</named-content>.</p><p>Western blot analysis was performed as previously described <xref ref-type="bibr" rid="pgen.1001157-Antonellis1">[84]</xref>. Briefly, protein extracts from approximately 3&#215;10<sup>3</sup> cells were separated on a 4&#8211;12% NuPage Bis-Tris gel (Invitrogen) at 125 V for 100 minutes in 1&#215; NuPage MES SDS Running Buffer (Invitrogen). After transfer to a nitrocellulose membrane, proteins were detected using a 1&#8758;5,000 dilution of primary antibody [goat anti-ERAP2 polyclonal antibody (AF3830, R&amp;D Systems) and mouse anti-ERAP2 polyclonal (ab69037, Abcam); anti-&#223;-actin monoclonal prepared in mouse (A5316, Sigma)] and a 1&#8758;10,000 dilution of secondary antibody conjugated with horseradish peroxidase (HRP) [goat anti-mouse IgG (sc-2005; Santa Cruz Biotechnology) and donkey anti-goat IgG (sc-2020; Santa Cruz Biotechnology)]. Proteins were then visualized by autoradiography after treatment with substrate to HRP (Thermo Scientific) for 5 minutes. The ratio of the intensity of the full-length ERAP2 band of AA-homozygotes to AB-heterozygotes was calculated using ImageJ (rsbweb.nih.gov/ij/index.html).</p><p>AB-heterozygote LCLs were treated with 100 &#181;g/ml of emetine (Sigma) for 7 hours to inhibit NMD <xref ref-type="bibr" rid="pgen.1001157-Noensie1">[50]</xref>. Parallel cultures were left untreated and grown at standard conditions. Total RNA was prepared from each cell line and used to generate cDNA as described earlier. We quantified haplotype-specific <italic>ERAP2</italic> cDNA in triplicate using an allele-discriminating TaqMan genotyping assay for three coding diagnostic SNPs (C_3282749_20 for rs2549782, C_25649530_10 for rs2548538, and C_25649516_10 for rs2287988; Applied Biosystems) as previously described <xref ref-type="bibr" rid="pgen.1001157-Dennis1">[85]</xref>. Briefly, for each allele-specific assay, we generated a standard curve consisting of serial dilutions of two HapMap genomic DNA samples homozygous for either the Haplotype A (GM18504) or Haplotype B (GM18508) allele. We used a heterozygous genomic DNA sample (GM18861) to validate the regression equation, in which we expect to see a mean allelic ratio of 1.0 since both the Haplotype-A and Haplotype-B alleles are present in an equal proportion.</p><p>Two experiments (labeled 1 and 2 in <xref ref-type="fig" rid="pgen-1001157-g006">Figure 6</xref>) were performed with 16 samples each. Human peripheral blood mononuclear cells (PBMCs) were isolated from buffy coats using a Ficoll/Histopaque gradient (Lonza). PBMCs were washed and cultured using RPMI 1640 supplemented with 10% fetal calf serum, 1% penicillin and streptomycin, 0.2 M L-glutamine, and 20 mM Hepes. Surface staining was measured by flow cytometry using fluorescence-labeled antibodies specific to CD19 (labeled with APC; clone HIB19; eBioscience) and HLA-ABC (labeled with FITC; clone W6/32; eBioscience) which reacts to HLA-A, B, and C. Flow-cytometry data analysis was performed with Flojo software (Treestar). Specifically, we measured HLA-ABC MFIs from a population of B cells gated by CD19 (a constitutive B-cell marker) intensity. Gating and analysis were carried out blindly with respect to genotypes. In order to standardize HLA-ABC MFI in light of the intrinsic variability among human samples, a standardized HLA-ABC measure was calculated for each sample by dividing the HLA-ABC MFI by the CD19 MFI for each sample. The values were partitioned by experiment and sub-partitioned by genotype; within each of these groups, outliers were removed (defined as samples with values under or over 1.5-times the inter-quartile range). It is worth noting that the inclusion of outliers did not affect the results. Two sets of analyses were performed for each of these three measures (HLA-ABC, CD19, and standardized HLA-ABC) as the dependent variable. First, a T-test was used to detect differences between cells with AA and BB genotypes for each experiment. Second, a two-factor ANOVA was performed for each measure using the data generated with all AA or BB samples, where the two factors of the ANOVA were genotype and experiment. Genotyping was performed by PCR amplification and sequencing of DNA prepared from the PBMCs (DNeasy Blood and Tissue kit, Qiagen) using primers flanking rs2248374 (see <xref ref-type="supplementary-material" rid="pgen.1001157.s013">Table S4</xref> for primer sequences).</p>