<p>Cases included in the study were unrelated individuals with AS of white European ancestry from the UK, USA, Canada and Australia. The diagnosis of AS met the modified New York definition criteria <xref ref-type="bibr" rid="pgen.1001195-vanderLinden1">[21]</xref>. Ethnically matched unrelated control individuals were selected from the 1958 British Birth Cohort (BBC) and an Oxfordshire (UK) healthy blood donor cohort. All patients gave informed, written consent, and the study was approved by the relevant ethics review committees.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">SNPs chosen for the study were selected from previously reported risk loci identified in a CD GWA study <xref ref-type="bibr" rid="pgen.1001195-Barrett1">[22]</xref>. Three candidate loci of high interest were studied in greater depth (<italic>STAT3</italic>, <italic>IL12B</italic> and <italic>ATG16L1</italic>). For these genes, tagging SNPs (r<sup>2</sup>&#8805;0.8) were selected using Tagger in Haploview (<ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/haploview/haploview">http://www.broadinstitute.org/haploview/haploview</ext-link>) and using the CEU population as a reference panel from the International HapMap Project database (<ext-link ext-link-type="uri" ns0:href="http://www.hapmap.org/">http://www.hapmap.org/</ext-link>). Whenever assay design for selected SNPs failed, perfect proxy SNPs (r<sup>2</sup>&#8202;=&#8202;1) were selected.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genotyping was carried out in two phases. In phase 1, 53 SNPs from 30 distinct genomic regions were selected for genotyping in 1,230 cases. Case genotype results were compared with historical genotypes from 1,295 unrelated individuals from the 1958 British Birth Cohort, which had been typed with both the Illumina HumanHap 550 and Affymetrix SNP Array 5.0 microarrays. Whenever selected markers were missing from the control genotypes, imputation was carried out using MACH (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/MACH/index.html">http://www.sph.umich.edu/csg/abecasis/MACH/index.html</ext-link>) against the HapMap CEU dataset (reference panel). Imputed markers with low overall quality scores (Q&lt;0.95) and/or in low LD with typed markers (r<sup>2</sup>&lt;0.3) were excluded from further analysis. Only genotypes with quality scores &#8805;0.95 were included in the study. All nominally associated markers (<italic>P</italic>&lt;0.1) were taken forward for genotyping in phase 2 on 1,543 cases and 920 controls.</p><p>In phase 2, genotyping was performed using Sequenom (Sequenom Inc., San Diego, USA), and Applied Biosystems TaqMan and OpenArray technologies (Life Technologies, Carlsbad, USA). For Sequenom, SNPs were assayed and typed using iPLEX chemistry on a matrix assisted laser desorption/ionization time-of-flight (MALDI_TOF) mass spectrometer. PCR reactions, cycling conditions and post-PCR extension reactions were all performed as recommended by the manufacturer. The iPLEX reaction products were desalted and spotted on SpectroChip and were processed and analysed in a compact mass spectrometer (MassARRAY Workstation). MassARRAY Typer 4.0 software was used for automated and manual genotype calling. OpenArray is a new genotyping platform technology from Applied Biosystems for medium-throughput experiment. SNPs were typed using TaqMan genotyping chemistry supported on a metal-based array. DNA samples were loaded and amplified on arrays as recommended by the manufacturer. Arrays were scanned on the OpenArray NT imager and genotypes were called using the OpenArray SNP Genotyping analysis software. Whenever assay design or the genotyping assay failed, markers were then genotyped using single TaqMan probe technology as recommended by the manufacturer.</p><p>Association statistics were calculated using PLINK software <xref ref-type="bibr" rid="pgen.1001195-Purcell1">[23]</xref>. Analysis was carried out using the Cochran-Armitage test for trend, excluding markers failing the following criteria; missingness rate &gt;0.1, minor allele frequency (MAF)&lt;0.01 and exact Hardy-Weinberg equilibrium <italic>P</italic>&lt;5&#215;10<sup>&#8722;5</sup>. We also excluded individuals with a missing genotype rate &gt;0.1 from the analysis. Experiment-wise significance (<italic>P</italic>&#8202;=&#8202;9.4&#215;10<sup>&#8722;4</sup>) was determined by Bonferroni correction based on the total number of markers genotyped in phase1 of the study.</p>