<p>Hemizygous R6/1 and R6/2 (160 CAG) and R6/2 (100 CAG) mice from the Jackson Laboratory were maintained on a mixed CBAxC57BL/6 genetic background <xref ref-type="bibr" rid="pgen.1000749-Mangiarini2">[21]</xref>. <italic>Hdh<sup>Q111</sup></italic>, <italic>Hdh<sup>Q50</sup></italic> and <italic>Hdh<sup>Q20</sup></italic> heterozygotes were maintained on a CD1 outbred genetic background <xref ref-type="bibr" rid="pgen.1000749-Wheeler3">[23]</xref>. The experiments were approved by the ethical committee C.R.E.M.E.A.S (Comite Regional d'Ethique en Matiere d'Experimentation Animale de Strasbourg).</p><p>The isolation of genomic DNA from mouse striatum and cerebellum for analysis of oxidative DNA damage was performed under conditions that minimize <italic>ex vivo</italic> oxidation artifacts according to the protocol developed by <italic>Lu et al.</italic>
<xref ref-type="bibr" rid="pgen.1000749-Lu1">[22]</xref>. To this end, the silica-gel-membrane based DNeasy Tissue Kit (Qiagen) was used. Importantly, 50 &#181;M of the free radical spin trap phenyl-tert-butyl nitrone (PBN, Sigma) was included in all buffers. High temperature and phenol use were avoided. DNA extracts were treated with RNase I. Striatum and cerebellum DNA extracted under these conditions was also used for CAG repeat sizing. Tail DNA for CAG repeats sizing was isolated using a standard protocol. Total RNA for CAG sizing and quantitative RT-PCR analysis was prepared using the RNeasy Mini Kit (Qiagen).</p><p>CAG repeat size was determined by PCR amplification using the HEX labeled primer 31329 and primer 33934 previously described <xref ref-type="bibr" rid="pgen.1000749-Mangiarini1">[20]</xref>. PCR reactions were performed using the expand high fidelity DNA polymerase (Roche), according to manufacturer's instructions. PCR reaction products were subsequently purified using the Nucleospin Extraction II Kit (Macherey-Nagel). Products were then analyzed using the ABI Prism 3100 DNA analyzer instrument and GeneScan and Genotyper softwares. Size calibration was performed by including ROX 500 or ROX 1000 (Applied Biosystems) with the analyzed PCR products. Amplitude of Genescan profile was determined by calculating the number of peaks above 10% of the maximum fluorescent peak intensity. From the amplitude value, we deduced the median peak.</p><p>To quantitatively amplify CAG repeats from R6/1 and R6/2 DNA or RNA, we used the following protocol. DNA was amplified with primers 31329 and 33934, previously described <xref ref-type="bibr" rid="pgen.1000749-Mangiarini1">[20]</xref> using the Herculase Hotstart DNA Polymerase (Stratagene). Concentrations of dNTP and primers were those recommended by the manufacturer. 8% DMSO was included in the reaction, as well as Sybr green (Molecular probe). The PCR reactions were performed and analyzed on a Light Cycler instrument (Roche). PCR cycling conditions were as follows: DNA was first inactivated for 3 min at 98&#176;C, followed by 45 cycles consisting in 40 seconds at 98&#176;C, 30 seconds at 60&#176;C and 2 min at 72&#176;C.</p><p>Reverse transcription was performed on 1 &#181;g of total RNA using SuperScriptII (Invitrogen) and random hexamers according to the manufacturer instructions. We performed PCR amplification of cDNA on a Light-Cycler instrument (Roche). PCR primers for detection of <italic>Ogg1</italic>, <italic>Ape</italic>1, <italic>Pol&#946;</italic>, <italic>Fen1</italic>, <italic>Hprt</italic>, <italic>Gapdh</italic> and <italic>36B4</italic> are available upon request.</p><p>Detection of AP sites was performed using the DNA damage quantification kit (Biovision), according to the manufacturer's instructions. The method is based on specific reaction of the Aldehyde Reactive (ARP) reagent with the open ring form of AP sites. Briefly, genomic DNA was treated with ARP tagged with biotin residues. AP sites in the DNA were then quantified using an avidin-biotin assay followed by a colorimetric reaction <xref ref-type="bibr" rid="pgen.1000749-Atamna1">[50]</xref>.</p><p>DNA damage at specific gene loci was assayed by cleavage of genomic DNA with bacterial formamidopyrimidine glycosylase (Fpg). Fpg is an AP-lyase that specifically excises 8-oxoG among other oxidized bases, and then creates a single strand break at the site of the abasic product. Quantitative real time PCR was used to determine the level of intact DNA at CAG repeats before and after DNA cleavage by Fpg. As a control, a portion of the murine <italic>Htt</italic> gene, which is similar in size and GC content to the CAG-expanded locus, was amplified. The ratio of PCR products after Fpg cleavage to those present in untreated DNA was used to determine the level of intact DNA. We followed the protocol described by Lu <italic>et al.</italic>
<xref ref-type="bibr" rid="pgen.1000749-Lu1">[22]</xref>, with the following modifications: Fpg from Sigma was used and incubated with genomic DNA for 1h at 37&#176;C at a concentration of 0.2 &#181;g Fpg/&#181;g of DNA. These conditions, which were predetermined by an Fpg dose response curve and time course, allowed the reaction to reach a steady state. After inactivation at 60&#176;C for 5 min, the Fpg reaction underwent an ethanol precipitation step. DNA was recovered in water and then submitted for quantitative PCR. Conditions for quantitative amplification of the CAG-expanded locus are described above. Quantitative amplification of the <italic>Hdh</italic> locus was performed using a commercial PCR master mix (Qiagen) with forward (<named-content content-type="gene">5&#8242;-TCGAGTCGCTCAAGTCGTTT-3&#8242;</named-content>) and reverse (<named-content content-type="gene">5&#8242;-ACTTCGCAAACTGGGAACGG-3&#8242;</named-content>) primers and PCR conditions were as follows: a first step of denaturation was performed at 95&#176;C for 15 min, followed by 45 cycles consisting in a denaturation at 95&#176;c for 30 seconds, hybridation for 30 seconds at 55&#176;C and elongation for 1 min at 72&#176;C.</p><p>Striata and cerebella were dissected and homogenized in lysis buffer containing 50 mM Tris-HCl pH 8.0, 10% glycerol, 5 mM EDTA, 150 mM KCl, a cocktail of protease inhibitors (Roche) and 1% NP-40. The extracts were incubated for 15 min on ice and centrifuged for 20 min at 13000 rpm and 4&#176;C. Supernatants were collected and analyzed on SDS-PAGE gels. Rabbit &#945;-OGG1 (Abcam), mouse &#945;-POL&#946; (Biovision), rabbit &#945;-APE1 (Abcam), rabbit &#945;-FEN1 (Santa Cruz or Abcam), rabbit &#945;-HMGB1 (Abcam) and mouse &#945;-&#946;-tubulin (Chemicon) were used at 1&#8758;1000 dilutions and revealed with appropriate &#945;-rabbit or &#945;-mouse peroxidase-conjugated secondary antibodies (Jackson immunoResearch Laboratories) and the ECL chemiluminescence reaction (Pierce or Millipore).</p><p>Hairpin DNA substrates with specific modifications (<xref ref-type="fig" rid="pgen-1000749-g003">Figure 3A</xref>) were purchased from Midland Certified Reagent Company, Inc. (Midland, TX). [&#947;<sup>32</sup>P]ATP 5&#8242; radiolabeled oligonucleotides were generated as described <xref ref-type="bibr" rid="pgen.1000749-Berquist1">[51]</xref>. After annealing, the labeled oligonucleotides were purified from unincorporated [&#947;<sup>32</sup>P]ATP by using a Bio-Rad Micro Bio Spin P30 column. Briefly, after spinning the column at 1000 g for 1 min to remove packing buffer, the columns used for purification of the 5&#8242;-<sup>32</sup>P THF substrates were washed twice with OPT buffer (25 mM Mops, pH 7.2, 100 mM KCl, 1 mM MgCl<sub>2</sub>) and the columns used for 5&#8242;-<sup>32</sup>P 8-oxoG substrates were washed twice with NEB2 buffer (New England Biolabs). Columns were centrifuged as above for each washing step. The labeled oligonucleotides were then eluted through the appropriate column by centrifugation at 1000 g for 4 min. The hairpin structure was verified by incubation of the 5&#8242;-<sup>32</sup>P labeled oligonucleotides (0.2 pmol) with <italic>Bam</italic>HI or <italic>Eco</italic>RI (0 U to 4U; New England Biolabs) and subsequent electrophoresis on a 15% polyacrylamide urea denaturing gel. OGG1 incision assays were performed by incubating 5&#8242;-<sup>32</sup>P labeled 8-oxoG substrate (0.2 pmol) with hOGG1 protein (0 to 30 ng; New England Biolabs) in NEB2 buffer at 37&#176;C for 15 min. APE1 incision assays were performed by incubating 5&#8242;-<sup>32</sup>P labeled THF substrate (0.2 pmol) with hAPE1 protein (0 to 1 ng) in OPT buffer at 37&#176;C for 5 min. Reactions were inhibited by the addition of stop buffer (95% formamide, 20 mM ethylenediaminetetraacetic acid [EDTA], 0.5% bromophenol blue and 0.5% xylene cyanol), and then heated at 95&#176;C for 5 min. Reaction products were resolved by 15% polyacrylamide urea denaturing gel electrophoresis and imaged using a Typhoon phosphoimager.</p><p>Cell extracts were prepared essentially as described <xref ref-type="bibr" rid="pgen.1000749-Takao1">[52]</xref>. Briefly, frozen striata or cerebella were homogenized in 10 mM HEPES&#8211;KOH, pH 7.7, 0.5 mM MgCl<sub>2</sub>, 10 mM KCl, 1 mM DTT buffer and then centrifuged at 2000 g at 4&#176;C for 10 min. The pellet was resuspended in 20 mM HEPES&#8211;KOH, pH 7.7, 0.5 mM MgCl<sub>2</sub>, 420 mM NaCl, 0.2 mM EDTA, 25% glycerol, PIC (Complete Protease Inhibitor Cocktail EDTA-free, ROCHE), 1 mM DTT and gently stirred at 4&#176;C for 20&#8211;30 min to allow for efficient nuclear lysis. The suspension was centrifuged at 14000 g at 4&#176;C for 15 min. The supernatant was dialyzed against 40 mM HEPES&#8211;KOH, pH 7.7, 50 mM KCl and 2 mM DTT buffer overnight at 4&#176;C. Cell extract concentrations were determined using a Bradford assay. OGG1, APE1, POL&#946; and FEN1 assays were performed using oligonucleotide substrates described by <xref ref-type="bibr" rid="pgen.1000749-Wilson1">[26]</xref>, see <xref ref-type="supplementary-material" rid="pgen.1000749.s002">Figure S2</xref>. 0.1 pmol of [&#947;<sup>32</sup>P]ATP 5&#8242;-radiolabeled oligonucleotides were incubated with 10 &#181;g of cell extract and assay buffer (75 mM KCl, 25 mM MgCl<sub>2</sub>, 7,5 mM dNTPs, 3,125 mM HEPES&#8211;KOH, pH 7.7, 1% glycerol, 0.25 &#181;M EDTA) at 37&#176;C. Incubation times were optimized for each repair activity: 3 h, 1 h, 2 h and 50 min were used for 8-oxoG incision (OGG1), AP-endonuclease (APE1), gap-filling (POL&#946;) and flap endonuclease (FEN1) assays, respectively. The samples were then treated with 0.5% SDS and 0.8 mg/mL of Proteinase K, heated at 55&#176;C for 15 min, purified by phenol-chloroform extraction and resuspended in solution by adding equal volume of 98% formamide, 10 mM EDTA, bromophenol blue and xylene cyanol buffer. Reaction products were resolved by 20% polyacrylamide urea denaturing gel electrophoresis and imaged on radiographic films. Images were captured with GeneSnap and quantified with GeneTool softwares on Syngene Chemigenus XE machine.</p><p>2 nM of purified human DNA POL&#946; <xref ref-type="bibr" rid="pgen.1000749-Lee1">[53]</xref> was incubated with increasing concentrations of purified human FEN1 <xref ref-type="bibr" rid="pgen.1000749-Nguyen1">[54]</xref> (1 nM, 2 nM, 4 nM, 8 nM and 16 nM) and 200 fmol of substrate for 30 min at 37&#176;C in 50 mM HEPES-KOH pH 7.5, 10 mM MgCl<sub>2</sub>, 0.5 mM EDTA, 2 mM DTT, 2 mM ATP, and 20 &#181;M each dATP, dCTP, dGTP, and dTTP. Reactions were stopped by the addition of stop buffer consisting of 90% formamide and 20 mM EDTA, boiled for 5 min and then loaded onto a 15% polyacrylamide-urea gel and run at 300 V for 5 hrs.</p><p>ChIP experiments were performed essentially as described <xref ref-type="bibr" rid="pgen.1000749-Compe1">[55]</xref>. For each ChIP experiment, striata and cerebellar from 2 to 4 transgenic mice were pooled, cut into small fragments, fixed by adding 37% formaldehyde to a final concentration of 1% and incubated for 10 min at room temperature. Cross-linking was stopped by addition of glycine to 0.125 M. Tissue fragments were washed three times with cold phosphate-buffered saline and treated with sonication buffer (50 mM HEPES pH 7.9, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium dodecyl sulfate, 0.1% Na-deoxycholate) containing protease and phosphatase inhibitors. Tissue was then homogenized, and lysates were sonicated to obtain DNA fragments of 200 to 1000 bp, as revealed by ethidium bromide staining of aliquots separated on agarose gels (<xref ref-type="supplementary-material" rid="pgen.1000749.s004">Figure S4</xref>). Samples were centrifuged to pellet debris and an aliquot was taken for gel analysis and input. The soluble chromatin fraction was pretreated for 1 h at 4&#176;C with protein A Agarose/Salmon Sperm DNA -50% slurry- (Millipore). Samples were then incubated overnight at 4&#176;C with &#945;-DNA POL&#946; ab194 (ChIP grade, Abcam) or &#945;-AcH3K9/14 (Upstate) antibodies. Protein A Agarose/Salmon Sperm DNA was then added, and the mixture was incubated for 2 h at 4&#176;C. Agarose beads were washed twice for 10 min with sonication buffer, twice for 10 min with wash buffer A (sonication buffer with 500 mM NaCl), twice for 10 min with wash buffer B (20 mM Tris-HCl, pH 8.0, 1 mM EDTA, &#224;.25 M LiCl, &#224;.5% NP-40, 0.5% Na-deoxycholate), and finally with Tris-EDTA (TE, pH 8.0). Immune complexes were eluted from the beads with 1% SDS in TE (pH 8.0) and protein-DNA cross-links were reversed by adding 200 mM NaCl and heating overnight at 65&#176;C. After treatment with proteinase K for 2 h at 42&#176;C, the samples were purified by phenol-chloroform-isoamyl alcohol extraction and precipitated with ethanol. One-sixth (for amplification of CAG expansions) to one-fifteen (for amplification of a fragment of the <italic>Hdh</italic> gene) of the immunoprecipitated DNA and 1% of the input DNA were quantified by real-time quantitative PCR (see above). Results are expressed relative to the amount of input DNA per ChIP.</p>