<p xmlns:ns0="http://www.w3.org/1999/xlink">Unless otherwise stated, flies were grown on a standard medium containing cornmeal, yeast, molasses, and propionic acid at 25&#176;C. The <italic>ssdp</italic> mutant strains (i.e <italic>ssdp<sup>BG1663</sup></italic>, <italic>ssdp<sup>neo4</sup></italic>
<sup>8</sup> and <italic>ssdp<sup>31</sup></italic>) used for the microarray experiment were previously described <xref ref-type="bibr" rid="pgen.1001063-Chen1">[13]</xref>, all three were balanced on TM3-GFP (FBba0000338). The rev(<italic>ssdp<sup>neo48</sup></italic>) line is a precise excision of the P element inserted in <italic>ssdp<sup>neo48</sup></italic>. UAS-RNAi lines 18873, 38686, 38186, 24959, 24959, 6367, 40871, 9026, 12823 and 15347 were obtained from VDRC <xref ref-type="bibr" rid="pgen.1001063-Dietzl1">[61]</xref>. Chromosomal deletions Df(2L)ED49, Df(2L)ED548, Df(3L)ED231, Df(3L)ED4284, Df(2L)ED1109, Df(2L)ED299, Df(1)ED7067, Df(2R)ED2222, Df(3R)ED5156, Df(3L)ED4528, Df(2L)ED270, Df(2L)ED774, Df(2L)ED746, Df(3R)ED5187, Df(2L)ED673, Df(2L)ED120, Df(1)ED6957, Df(2L)ED19, Df(3R)ED5657 and Df(3R)ED10257, were obtained from the DrosDel collection <xref ref-type="bibr" rid="pgen.1001063-Ryder1">[62]</xref>. All other fly stocks were obtained from the Bloomington Drosophila Stock Center (<ext-link ext-link-type="uri" ns0:href="http://flystocks.bio.indiana.edu">http://flystocks.bio.indiana.edu</ext-link>). Oregon-R flies were used as wild type.</p><p>The transheteroallelic combination <italic>ssdp<sup>BG1663</sup></italic>/<italic>ssdp<sup>neo48</sup></italic> was obtained by crossing <italic>ssdp<sup>BG1663</sup></italic>/TM3-GFP virgin females to <italic>ssdp<sup>neo48</sup></italic>/TM3-GFP males. The trans-heteroallelic combination <italic>ssdp<sup>BG1663</sup></italic>/<italic>ssdp<sup>31</sup></italic> was obtained by crossing <italic>ssdp<sup>BG1663</sup></italic>/TM3-GFP virgin females to <italic>ssdp<sup>31</sup></italic>/TM3-GFP males. The control single heterozygotes <italic>ssdp<sup>BG1663</sup></italic>/+ and <italic>ssdp<sup>31</sup></italic>/+ were obtained by crossing virgin females from each mutant strain to wild type Oregon-R males. The control single heterozygote, <italic>ssdp<sup>neo48</sup></italic>/+, was obtained by crossing virgin wild type females to mutant males. An additional control, <italic>ssdp<sup>BG1663</sup></italic>/rev (<italic>ssdp<sup>neo48</sup></italic>), was used instead of <italic>ssdp<sup>BG1663</sup></italic>/+ for comparison to the trans-heteroallelic combination <italic>ssdp<sup>BG1663</sup></italic>/<italic>ssdp<sup>neo48</sup></italic> since they share more genetic background. The <italic>ssdp<sup>BG1663</sup></italic>/rev (<italic>ssdp<sup>neo48</sup></italic>) combination was obtained by crossing <italic>ssdp<sup>BG1663</sup></italic> virgin females to rev(<italic>ssdp<sup>neo48</sup></italic>) males. For detailed genotypes of microarray samples see <xref ref-type="supplementary-material" rid="pgen.1001063.s004">Table S1</xref>. Crosses were made in vials containing colored medium: 7.5 g/l agar, 35 g/l flour, 50 g/l yeast, 55 g/l glucose, 2.5 ml/l p-Hydroxybenzoic Acid, 4 ml/l Propionic Acid and 0.5 ml/l Bromophenol. The colored medium allows for more precise staging of the larvae. Towards the end of the third larval stage the larvae cease to feed and the gut clears out. The colored medium can be seen through the live whole larvae. Larvae were collected when the gut was two thirds full and selected for the desired genotype using the GFP marker. Consequently only male larvae were taken for analysis to avoid artifactual differential expression due to sex biased expression in populations with different sex ratios.</p><p>Virgin <italic>Dlmo<sup>Bx2</sup></italic> females were crossed to males harboring a mutation in a single target gene. Each cross was set up in 50 ml vials with 10 females and 7-10 males in each vial. All resultant F1 phenotypic classes were counted. The double heterozygote offspring (i.e <italic>Dlmo<sup>Bx2</sup></italic>/+; <italic>target gene<sup>&#8722;</sup></italic>/+ females and <italic>Dlmo<sup>Bx2</sup></italic>/Y; <italic>target gene<sup>&#8722;</sup></italic>/+ males) were counted according to their wing notching severity class. Class 1 representing flies with wild type wings; class 2 represents flies that have anterior notching of one wing; class 3 represents flies that have anterior notching of both wings; class 4 represents flies that have anterior notching of both wings and posterior notching of one wing; class 5 represents flies that have anterior and posterior notching of both wings and finally class 6 representing flies displaying partial detachment of the dorsal and ventral wing layers. Most crosses were set up in three vials and results were pooled. An average of 100 double heterozygote females and 114 double heterozygote males were counted for each target gene tested. The control <italic>Dlmo<sup>Bx2</sup></italic>/+ females and <italic>Dlmo<sup>Bx2</sup></italic>/Y males were obtained by crossing the <italic>Dlmo<sup>Bx2</sup></italic> females to wild type Oregon-R males. A control cross was set up parallel to each set of test crosses. Rarely crosses were discarded if the control distribution was not consistent with previous control crosses. Finally, data from all the control crosses was combined to a single distribution for females and a single distribution for males and all the test distributions were compared to these two master controls. Significance was determined according to the Wilcoxon signed-rank test <xref ref-type="bibr" rid="pgen.1001063-Litchfield1">[63]</xref>. For 17 target genes more than one allele was tested (9 enhancers, 2 suppressors and 5 non-interacting). For a target gene to be designated as an interactor the same interaction was observed in both males and females. In addition where more than one allele was tested both alleles had to give the same interaction. If a target gene was tested by more than one allele and one allele gave the same interaction with males and females but the other only significantly affected one of the sexes it was still designated an interactor (2 enhancers and 1 suppressor). The DrosDel chromosomal deletions were compared to control <italic>Dlmo<sup>Bx2</sup></italic>/<italic>w<sup>1118</sup></italic> female and <italic>Dlmo<sup>Bx2</sup></italic>/Y male flies obtained by crossing the <italic>Dlmo<sup>Bx2</sup></italic> virgin females to males of the isogenic <italic>w<sup>1118</sup></italic> line used to create the chromosomal deletions.</p><p>Virgin <italic>T(2;3)ap</italic>
<sup>Xa</sup>/In(2R)<italic>Gla Bc Elp</italic> females were crossed to <italic>ssdp<sup>L7</sup></italic>/TM6- <italic>Tb Sb e</italic> males or males caring insertion mutations in different SSDP target genes. Each cross was set up in three 30 ml vials with 5 females and 3&#8211;5 males in each vial. The control <italic>T(2;3)ap<sup>Xa</sup></italic>/+ flies were obtained by crossing the <italic>T(2;3)ap</italic>
<sup>Xa</sup>/In(2R)<italic>Gla Bc Elp</italic> females to wild type Oregon-R males. A control cross was set up parallel to each set of test crosses. Each test cross was compared to the control cross done in parallel. The resultant F1 of genotypes <italic>T(2;3)ap<sup>Xa</sup></italic>/+, <italic>T(2;3)ap<sup>Xa</sup></italic>/+; <italic>ssdp<sup>L7</sup></italic>/+ or <italic>T(2;3)ap</italic>
<sup>Xa</sup>/+; <italic>target genes<sup>-</sup></italic>/+ were counted according to their phenotypic severity class. Flies of genotypes <italic>T(2;3)ap<sup>Xa</sup></italic>/+ or <italic>T(2;3)ap<sup>Xa</sup></italic>/+; <italic>target genes&#8722;</italic>/+ exhibited two severity classes, class 1 being the least severe. Flies of the genotype <italic>T(2;3)ap<sup>Xa</sup></italic>/+; <italic>ssdp<sup>L7</sup></italic>/+ exhibited three severity classes which were different then those observed for <italic>T(2;3)ap<sup>Xa</sup></italic>/+ or <italic>T(2;3)ap<sup>Xa</sup></italic>/+; <italic>target genes&#8722;</italic>/+ flies and were therefore classified independently. Class 1 being the list severe and class 3 being the most severe.</p><p>Virgin <italic>ssdp<sup>L7</sup></italic>/TM6-Tb Sb e or <italic>Chip<sup>e5.5</sup></italic>/CyO-GFP females were crossed to males harboring a mutation in any single target gene. Each cross was set up in 50 ml vials with 10 females and 7&#8211;10 males in each vial. All resultant F1 phenotypic classes were counted. The double heterozygote offspring (i.e <italic>ssdp<sup>L7</sup></italic>/+; <italic>target gene<sup>&#8722;</sup></italic>/+ or <italic>Chip<sup>e5.5</sup></italic>/+; <italic>target gene<sup>&#8722;</sup></italic>/+) were counted and monitored for duplications of scutellar bristles. A control cross was set up parallel to each set of test crosses. The control <italic>ssdp<sup>L7</sup></italic>/+ or <italic>Chip<sup>e5.5</sup></italic>/+ flies were obtained by crossing the <italic>ssdp<sup>L7</sup></italic>/TM6-Tb Sb e or <italic>Chip<sup>e5.5</sup></italic>/CyO-GFP females to wild type Oregon-R males. Each test cross was compared to the control cross done in parallel. For the control cross the frequency of appearance of the duplicated bristle phenotype was calculated as p&#8202;=&#8202;(d+1)/(n+1) where n is the total number of flies and d is the number of flies displaying the duplicated bristle phenotype. Significance was determined using binomial cumulative distribution function with parameters p and m, m being the total number of flies in the test cross. The p-values calculated were corrected for multiple hypotheses testing using the false discovery rate procedure <xref ref-type="bibr" rid="pgen.1001063-Benjamini1">[64]</xref>.</p><p>The insertion alleles <italic>ap<sup>MD544</sup></italic> and <italic>pnr<sup>MD237</sup></italic> were used as <italic>ap</italic>-Gal4 and <italic>pnr</italic>-Gal4 respectively. Each cross was set up in three 30 ml vials with 5 females and 3&#8211;5 males in each vial and results were pulled. The control <italic>ap</italic>-Gal4/+ flies were obtained by crossing the <italic>ap</italic>-Gal4/In(2R)Gla Bc Elp females to wild type Oregon-R males. A control cross was set up parallel to each set of test crosses. Each test cross was compared to the control cross done in parallel.</p><p>RNA handling was performed exactly as described <xref ref-type="bibr" rid="pgen.1001063-Gupta1">[65]</xref>. Briefly, larvae were flash frozen. Total RNA was extracted using Trizol (Life Technologies, Carlsbad, USA), followed by mRNA isolation using an Oligotex poly(A) extraction kit (Qiagen, Valencia, USA). RNA concentration was determined using RiboGreen dye (Molecular Probes, Oak Ridge, USA). RNA quality was determined by capillary electrophoresis using the 6000 Nano Assay kit (Agilent). All procedures were carried according to the manufacturer's instructions.</p><p>The FlyGem platform is available under GEO accession GPL20 <xref ref-type="bibr" rid="pgen.1001063-Edgar1">[66]</xref> and the experiments described in this work are available under the series accession GSE20074. GEO sample accessions are given in <xref ref-type="supplementary-material" rid="pgen.1001063.s004">Table S1</xref>.</p><p>Microarray experiments were conducted exactly as described <xref ref-type="bibr" rid="pgen.1001063-Gupta1">[65]</xref>. Briefly, samples were labeled with Cy3- or Cy5-labeled random nonamers (Trilink Biosciences, San Diego, USA). Hybridizations of samples to the microarrays were performed at 60&#176;C, followed by washes. Arrays were scanned using an Axon GenePix 3000A fluorescence reader (Molecular Devices Corporation, Union City, USA). GenePix v.4.1 image acquisition software (Molecular Devices Corporation) was used to extract signal for each target element.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The array data was analyzed using R, which is an integrated suite of software facilities for data manipulation; calculation and graphical display (see <ext-link ext-link-type="uri" ns0:href="http://www.r-project.org">http://www.r-project.org</ext-link>). The raw intensity data normalized within-arrays using the PrintTipLoess algorithm <xref ref-type="bibr" rid="pgen.1001063-Smyth1">[67]</xref>, and next between-arrays, using Quantile <xref ref-type="bibr" rid="pgen.1001063-Bolstad1">[68]</xref>. This normalization allows adjustment of microarray data according to effects that arise from variation in the technology rather than from biological differences between the RNA samples. Array elements whose intensity was lower than the median intensity in both channels were discarded. No other background correction method was used. We did not average duplicate array elements as this would reduce statistical power in later steps.</p><p>All 22 hybridizations were used to select for candidate target genes that were significantly differentially expressed between <italic>ssdp</italic> trans-heteroallelic combinations and their corresponding heterozygotes. An ANOVA fixed-model was used to determine significance. The p-value calculated was corrected for multiple hypotheses testing using the false discovery rate (FDR) procedure <xref ref-type="bibr" rid="pgen.1001063-Benjamini1">[64]</xref>, <xref ref-type="bibr" rid="pgen.1001063-Reiner1">[69]</xref>, the threshold was set at &lt;0.05.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All genomic sequences were obtained from the UCSC genome browser (<ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</ext-link>, assembly Apr. 2006 for the <italic>D. melanogaster</italic> genome) <xref ref-type="bibr" rid="pgen.1001063-Karolchik1">[70]</xref>. The 500 bp upstream of the 189 candidate genes scanned using two algorithms termed PRIMA <xref ref-type="bibr" rid="pgen.1001063-Elkon1">[30]</xref> and DEMON, for identifying enrichment of transcription factors binding sites in a set of co-regulated genes. Both methods require a background set for comparison (in this case all the annotated genes in <italic>Drosophila</italic>).</p><p>The SSDP binding site was constructed from the linear sequence in Bayarsaihan et al, <xref ref-type="bibr" rid="pgen.1001063-Bayarsaihan1">[29]</xref> and tested individually. The values in the matrix were set to 1 or 0 according to the appearance or nonappearance of the nucleotides in each position respectively. Those values were then corrected to allow flexibility in the recognition of this binding site. The binding site matrix for PANNIER was taken from <xref ref-type="bibr" rid="pgen.1001063-Haenlin1">[32]</xref> and tested together with all the binding site matrices available from the TransFac database (release 11.1) <xref ref-type="bibr" rid="pgen.1001063-Wingender1">[31]</xref>.The p-values calculated are corrected for multiple hypotheses testing using the false discovery rate procedure <xref ref-type="bibr" rid="pgen.1001063-Benjamini1">[64]</xref>.</p><p>Finds putative appearances of transcription factors binding sites in the promoters using a threshold score and then employs a hyper-geometric statistical test to examine whether those appearances are significantly over-represented in the data set with respect to the background set <xref ref-type="bibr" rid="pgen.1001063-Elkon1">[30]</xref>.</p><p>Based on hidden Markov models (HMMs) of promoter sequences regulated by a given transcription factor that take into account multiple binding sites of varying affinities in a promoter. DEMON builds an HMM for each one of the transcription factors binding sites and scores each pair of HMM-promoter for all the HMMs and the promoters in the data set. The score reflects how likely it is that the motif modeled by this HMM appears in this promoter. The scores are then utilized to obtain a p-value for each transcription factor binding site that reflects the probability that the binding site is enriched in the given set of promoters compared to a background set.</p><p>Analysis for enrichment of GO functions was conducted using the <bold>d</bold>atabase for <bold>a</bold>nnotation, <bold>v</bold>isualization and <bold>i</bold>ntegrated <bold>d</bold>iscovery (<bold>DAVID</bold>) <xref ref-type="bibr" rid="pgen.1001063-Huangda1">[71]</xref>, <xref ref-type="bibr" rid="pgen.1001063-Dennis1">[72]</xref>. Default setting were used and the enrichment cut off was set to p&#8202;=&#8202;0.05 after FDR correction.</p><p>Fly <italic>ssdp</italic> was PCR amplified, cloned into pZEX plasmid and expressed with a GST tag in <italic>E.coli</italic> BL-21. Crude cell extract or purified GST-SSDP fusion protein was used for binding assays. GST-SSDP was purified on a glutathione agarose column (Sigma G4510). The <italic>ssdp</italic> single stranded CT oligonucleotide <xref ref-type="bibr" rid="pgen.1001063-Bayarsaihan1">[29]</xref> was used as prob. Binding assays were carried out using the DIG Gel shift kit 2<sup>nd</sup> generation (Roche, Mannheim, Germany) according to the manufacturer instruction in a final volume of 20 &#181;l containing labeled DNA (150 fmoles), 1 &#181;l of poly-L-lysine and 3 &#181;l poly-[d(I-C)], 140 ng cell extract. For competition experiments 90 or 360 ng of unlabeled probe were added. Following a 20 min incubation at room temperature, the binding reaction products were separated on a native 6% polyacrylamide gel in 0.5% TBE (pH&#8202;=&#8202;8). The gel was contact blotted onto a Hybond-N<sup>+</sup> membrane (Amersham Biosciences). The chemiluminescent detection was performed following the manufacturer's instructions (Roche, Mannheim, Germany). The membrane was exposed to X-ray film (FUJI) for 15 min at 37&#176;C.</p>