<p>The generation of <italic>Ogg1</italic> null mice was described elsewhere <xref ref-type="bibr" rid="pgen.1000951-Klungland1">[36]</xref>. <italic>Ogg1<sup>&#8722;/&#8722;</sup></italic> mice were further backcrossed into C57BL/6 background. Wild type and <italic>Ogg1<sup>&#8722;/&#8722;</sup></italic> mice were derived from heterozygous (<italic>Ogg1<sup>+/&#8722;</sup></italic>) breeders. Primary MEFs were isolated from 13.5 day embryos of <italic>Ogg1<sup>+/&#8722;</sup></italic> female bred with <italic>Ogg1<sup>+/&#8722;</sup></italic> male and cultured in Dulbecco's Modified Eagle Medium containing 10% fetal bovine serum. Splenocytes were prepared from mouse spleens, cultured in RPMI 1640 with 10% FBS and 0.1% 2-mercaptoethanol, and stimulated with 50 &#181;g/ml <italic>Escherichia coli</italic> LPS serotype O111:B4 (Sigma-Aldrich) and 50 ng/ml mouse IL-4 (R&amp;D Systems). Bone marrow cells were flushed from femurs and tibias and cultured with Iscove's modified Dulbecco's medium (IBCO-BRL) supplemented with 20% fetal calf serum (Hyclone) in the presence of interleukin 6 (200 U/mL; Peprotech) and stem cell factor (100 ng/mL; Peprotech). To decrease or enhance oxidative stress, mouse cells were cultured in 3% oxygen (SANYO O<sub>2</sub>/CO<sub>2</sub> incubator, MCO-18M) or 20% oxygen or in the presence of paraquat. All animal experiments were carried out according to the &#8220;Guide for the Care and Use of Laboratory Animals&#8221; (National Academy Press, USA, 1996), and were approved by the Institutional Animal Care and Use Committee of National Institute on Aging.</p><p>The telomere fluorescence in cell populations of spleen, bone marrow, and primary MEFs was measured by Flow cytometry and FISH (Flow-FISH) according to previously published protocol <xref ref-type="bibr" rid="pgen.1000951-Rufer1">[43]</xref>. A telomere specific FITC conjugated (<named-content content-type="gene">CCCTAA</named-content>)<sub>3</sub> PNA probe (0.3 &#181;g/ml, Panagene) was used.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Quantitative FISH (Q-FISH) was performed as previously described <xref ref-type="bibr" rid="pgen.1000951-Zijlmans1">[44]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1000951-Gomez1">[46]</xref>. Metaphase spreads were prepared from freshly isolated or subcultured mouse bone marrow cells, activated splenocytes, and primary MEFs. Briefly, mice were injected with 100 &#181;l of 0.5% colchicine intraperitoneally for approximately 30 minutes before being sacrificed. Bone marrow cells were then collected by flushing 1ml of PBS through femurs and tibias. Cultured mouse cells were incubated with 0.1 &#181;g/ml colcemid for 2&#8211;6 hr at 37&#176;C to allow mitotic cells to accumulate. Metaphase spreads were obtained by incubating colchicine- or colcemid- treated mouse cells in 0.075 M KCl for 15 minutes in 37&#176;C, followed by fixing cells in ice-cold 3&#8758;1 methanol and glacial acetic acid and dropping the fixed cells onto slides. Metaphase spreads were hybridized with Cy3-labeled (<named-content content-type="gene">CCCTAA</named-content>)<sub>3</sub> (0.3 &#181;g/ml, Panagene), washed, and then counterstained with 4,6 diamidino-2-phenylindole (DAPI). For the detection of telomere signal intensity in G and C strands, metaphase spreads were initially hybridized with FITC-labeled (<named-content content-type="gene">CCCTAA</named-content>)<sub>3</sub> PNA probes (0.3 &#181;g/ml, Panagene). The free-(<named-content content-type="gene">CCCTAA</named-content>)<sub>3</sub> probe were washed off the slides, and then hybridized with TRAMA-labeled (<named-content content-type="gene">TTAGGG</named-content>)<sub>3</sub> (0.3 &#181;g/ml, Panagene). Images were captured using Cytovision software (Applied Imaging) on a fluorescence microscope (Axio2; Carl Zeiss); followed by quantification of telomere fluorescence signals using the TFL-Telo software (a kind gift from Dr. Peter Lansdorp). For histograms and box-plots, data from different mice of each genotype were scored and R statistical package (<ext-link ext-link-type="uri" ns0:href="http://www.r-project.org/">http://www.r-project.org/</ext-link>) along with R.utils package and Biobase package (<ext-link ext-link-type="uri" ns0:href="http://www.bioconductor.org/">http://www.bioconductor.org/</ext-link>) were used. The frequencies of telomeres within a given range of telomere signal intensities were plotted against the telomere signal intensity using arbitrary units. Metaphases from different mice of each genotype were scored for chromosomal and telomeric abnormalities as previously described <xref ref-type="bibr" rid="pgen.1000951-Wang1">[45]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1000951-Gomez1">[46]</xref>.</p><p>Telomerase activity was measured by Biomax Telomerase Detection Kit (Biomax) according to the manufacturer's recommendations. Briefly, mouse cell extracts were added to a pre-mix for quantitative telomerase activity in a real-time PCR reaction. MyiQ Single-Color Real-Time PCR Detection System (Bio-Rad) was used to perform the reactions, where each sample was done in triplicates and performed according to the manufacturer's instructions. HeLa cell extracts were used as positive control. <italic>Tert</italic> knockout mouse cell extracts and RNase-treated HeLa cell extracts were used as negative controls. Relative telomerase activity was expressed as log of C<sub>T</sub> value.</p><p>CO-FISH was used to measure T-SCEs and telomere lagging or leading strand loss <xref ref-type="bibr" rid="pgen.1000951-Crabbe1">[16]</xref>, <xref ref-type="bibr" rid="pgen.1000951-Bailey1">[47]</xref>. Briefly, mice were injected with 3&#8758;1 ratio of BrdU/BrdC (Sigma) at a final concentration of 1&#215;10<sup>&#8722;5</sup> M intraperitoneally for approximately 20 hours, and subsequently with 100 &#181;l of 0.5% colchicine for approximately 30 minutes before being sacrificed. Bone marrow cells were then collected by flushing 1ml of PBS through femurs and tibias. MEFs were cultured in medium containing a 3&#8758;1 ratio of BrdU/BrdC (Sigma) at a final concentration of 1&#215;10<sup>&#8722;5</sup> M for 24 hours, and colcemid (0.1 &#181;g/ml) was added 4 hours before harvest. Metaphase spreads were prepared from mouse bone marrow cells or MEFs, stained with Hoechst 33258, exposed to UV light, and digested with exonuclease III to remove newly synthesized DNA strands. Hybridization and wash conditions were identical to those described for Q-FISH. FITC-labeled (<named-content content-type="gene">CCCTAA</named-content>)<sub>3</sub> and TRAMA-labeled (<named-content content-type="gene">TTAGGG</named-content>)<sub>3</sub> PNA probes were used for the detection of lagging and leading strand, respectively. A chromosome with more than two telomeric DNA signals by both FITC-labeled (<named-content content-type="gene">CCCTAA</named-content>)<sub>3</sub> and TRAMA-labeled (<named-content content-type="gene">TTAGGG</named-content>)<sub>3</sub> PNA probes was scored as T-SCE positive. A chromosome with loss of one or two telomeric DNA signals by either FITC-labeled (<named-content content-type="gene">CCCTAA</named-content>)<sub>3</sub> or TRAMA-labeled (<named-content content-type="gene">TTAGGG</named-content>)<sub>3</sub> PNA probes was scored for telomere lagging or leading strand loss.</p><p>TEL-FISH was performed as described previously <xref ref-type="bibr" rid="pgen.1000951-Gomez1">[46]</xref> with minor modifications. Briefly, cells were fixed in 1&#8758;1 methanol&#8758;acetone (Sigma) at &#8722;20&#176;C for 10 minutes, permeabilized with 0.5% NP-40, and blocked in 1% Bovine serum albumin (BSA) (IgG-free, Sigma). Cells were first immunostained with a rabbit anti-&#947;H2AX antibody (16193, Upstate Biotechnology), a rabbit anti-53BP1 antibody (BN 100&#8211;304, Novus Biologicals), or a mouse anti-XRCC1 antibody (X0629, Sigma) overnight at 4&#176;C followed by Alexa 488-labeled secondary antibody (1&#8758;500; Molecular Probes) for one hour at 37&#176;C. Slides were washed with PBS for 15 minutes, fixed in 2% paraformaldehyde at room temperature for 10 minutes, dehydrated through ethanol series, and air-dried briefly. Slides were then hybridized to a TRAMA-labeled (<named-content content-type="gene">CCCTAA</named-content>)<sub>3</sub> PNA probe (Panagene), then counterstained with DAPI. Z-stack images were captured and deconvoluted using Axiovision 4.6.3 software on a fluorescence microscope (Axiovert 200M; Carl Zeiss).</p><p>Identification of oxidative base lesions in telomeres was performed as previously described <xref ref-type="bibr" rid="pgen.1000951-Kruk1">[48]</xref> with modifications. In brief, DNA was isolated from mouse liver or primary MEFs by salting out. 4 &#181;g of DNA was treated with <italic>HinfI</italic> and <italic>RsaI</italic> restriction enzyme at 37&#176;C overnight. The reaction was heated at 65&#176;C for 15 minutes and then divided into two equal portions; one was treated with 8 units of <italic>E. coli</italic> formamidopyrimidine-DNA glycosylase (Fpg) (New England Biolabs) and another was treated with a mock buffer at 37&#176;C for 30 minutes. Fpg was inactivated by heating at 60&#176;C for 15 minutes. Genomic single-stranded DNA fragments were separated on 1% alkaline agarose gel according to their sizes, treated with UV light, then transferred to a nylon membrane. Single-stranded telomere DNA fragments were detected by <sup>32</sup>P-labeled (<named-content content-type="gene">CCCTAA</named-content>)<sub>4</sub> probe and visualized by autoradiography. ImageQuant software was applied in quantifying DNA cleavage in mock and Fpg-treated samples. A grid object was created as a single column with multiple rows and was placed over the lane corresponding to the molecular size markers. The density measurement was conducted in each row in which each marker was recorded. The mean length (ML) was calculated as a center of mass and expressed in kb: ML&#8202;=&#8202;&#931; (MWi &#215; ODi)/&#931; (ODi), in which MWi is the length of the telomeric DNA at each row and ODi is the densitometer output at each row. The frequencies of Fpg-sensitive lesions in a sample were calculated based on ML values in Fpg- and mock-treated samples: lesions&#8202;=&#8202;(ML untreated/ML treated)&#8722;1, in which ML is expressed in kb. Fold-changes in each sample were further normalized with respect to the number of Fpg-sensitive lesions in a control.</p>