<p>
<italic>C. elegans</italic> wild-type N2 strain worms were grown either in an axenic liquid mCeHR-2 medium <xref ref-type="bibr" rid="pgen.1001044-Nass1">[47]</xref> or on NGM agar plates spotted with <italic>E. coli</italic> OP50 or HT115(<italic>DE3</italic>) strains <xref ref-type="bibr" rid="pgen.1001044-Stiernagle1">[48]</xref>. Synchronized, L1 larvae were obtained by bleaching P<sub>0</sub> gravid worms grown in mCeHR-2 medium supplemented with hemin chloride <xref ref-type="bibr" rid="pgen.1001044-Rao1">[11]</xref>. Hemin chloride and gallium protoporphyrin IX were purchased from Frontier Scientific, Inc (Logan, UT). Plasmids for cloning and injecting into worms were part of the Fire Vector Kit (Addgene, Cambridge, MA). Primers designed to PCR amplify worm open reading frames were based on Wormbase predictions and ordered from IDT (Coralville, IA). The PCR products were TA cloned into the L4440 plasmid.</p><p>Equal numbers of F<sub>1</sub> larvae in the L1 stage were inoculated in mCeHR-2 medium with 4, 20, or 500 &#181;M hemin chloride and grown with gentle shaking at 20&#176;C. Synchronized, F<sub>2</sub> larvae in the L1 stage were obtained by hatching the eggs obtained from F<sub>1</sub> gravid adults in M9 buffer containing 4, 20, or 500 &#181;M hemin. Equal numbers of F<sub>2</sub> larvae in the L1 stage were inoculated in mCeHR-2 medium supplemented with 4, 20, or 500 &#181;M hemin. The F<sub>2</sub> worms were allowed to develop to the late L4 stage, harvested, flash frozen in liquid nitrogen, and stored at &#8722;80&#176;C. Frozen worm pellets were ground into a fine powder, and total RNA was extracted using Trizol (Invitrogen, Carlsbad, CA). RNA thus obtained was subjected to RNase-free DNase treatment for 1 h at 37&#176;C and purified using the RNeasy kit (Qiagen, Germantown, MD). Total RNA from three biological replicates was used to make cDNA, which was then hybridized to <italic>C. elegans</italic> Whole Genome Arrays (Affymetrix, Santa Clara, CA).</p><p>First strand cDNA was synthesized using 2 &#181;g of total RNA using a Superscript II First Strand cDNA synthesis kit (Invitrogen). For quantitative real-time PCR (qRT-PCR), primers spanning at least one intron were designed using Primer Express (Applied Biosystems) and Beacon designer 4 (Premier Biosoft) programs. PCR was performed using the iCycler iQ Real-time PCR Detection System (BioRad) with 0.12 U/&#181;l Taq DNA polymerase, 40 nM fluorescein (Invitrogen), and SYBR Green I Nucleic Acid Gel Stain (Invitrogen) diluted 1:10. The PCR amplification was run for 40 cycles. The PCR products were between 150 and 200 bp in length. Quality of the PCR products was determined by dissociation curve analysis and gel electrophoresis. Each experiment was performed in triplicate. Average C<sub>T</sub> values were used for 2<sup>&#8722;&#916;&#916;Ct</sup> calculations of relative fold changes in gene expression <xref ref-type="bibr" rid="pgen.1001044-Livak1">[49]</xref>.</p><p>Expression data were normalized and analyzed using MAS 5.0 suite software (Affymetrix). Data from worms grown in mCeHR-2 medium with 4 and 500 &#181;M hemin were compared to data from worms grown in medium containing 20 &#181;M hemin (baseline samples). Microarray data were verified with the Robust Multichip Average Method (RMA, R package). Quantile normalization and background corrections were performed using perfect match probe intensities. Using an initial cut-off of &#8805;1.2-fold change in mRNA expression for RMA and a &#8805;1.6-fold change for MAS 5.0 resulted in the identification of 370 genes. Increasing the stringency to &#8805;1.6-fold change for both RMA and MAS 5.0 reduced the number of genes identified as heme responsive to 288 genes. To identify putative human orthologs, worm protein sequences were used to query human genome databases at NCBI by reciprocal BLAST analysis with an E-value cut-off &#8805;10<sup>&#8722;4</sup>. Sequences for each of these 288 genes were obtained from WormBase and further analyzed for topology (TMHMM 2.0, SOSUI), motifs (ELM, BLOCKS, Pfam), and pathway classification (GO and KEGG).</p><p>The Ahringer and Vidal feeding libraries were replicated to individual 96-well plates <xref ref-type="bibr" rid="pgen.1001044-Kamath1">[50]</xref>, <xref ref-type="bibr" rid="pgen.1001044-Reboul1">[51]</xref>. Thirty-four clones in the initial list of 370 <italic>hrg</italic>s were absent from both libraries. To complete the <italic>hrg</italic> mini-library, we PCR amplified the missing genes from N2 worm genomic DNA and cloned the PCR fragments by TA cloning into the RNAi feeding vector pL4440. Only 19 of the 34 RNAi clones were in the final list of 288 <italic>hrg</italic>s. DNA for all 288 <italic>hrg</italic>s was sequenced to confirm authenticity.</p><p>NGM agar plates containing IPTG, carbenicillin, and tetracycline were seeded with HT115(<italic>DE3</italic>) bacteria expressing double-stranded RNA (dsRNA) against each clone in the <italic>hrg</italic> mini-library. Duplicate bacterial cultures of each clone had been grown for 5.5 h in LB containing carbenicillin and tetracycline and 5 &#181;M or 25 &#181;M heme. Plates were seeded with a lawn of bacteria and dsRNA induction occurred for &#8776;20 h at room temperature. Subsequently, forty L1 larvae from gravid IQ6011 worms which had been grown in liquid media supplemented with 10 &#181;M heme were added to each well of the 12-well plates. Each 12-well plate had 10 wells seeded with experimental clones and one well seeded with each of the control clones &#8211; vector and <italic>hrg-4</italic>. The plates were incubated at 15&#176;C overnight and then incubated at 20&#176;C for three additional days. The GFP levels in gravid adults were observed visually using a Leica Microsystems MZ16FA stereoscope. The intensity and pattern of GFP in gravid worms feeding on bacteria producing dsRNA against each <italic>hrg</italic> was compared to the intensity and pattern of GFP in same-stage worms feeding on bacteria transformed with the empty vector. Worms that displayed altered GFP in both replicates were designated as potential modulators. Potential modulators were screened in a strain that produced GFP under the control of a promoter that was not responsive to heme (<italic>vha-6::gfp</italic>). Any clone that altered GFP levels in the <italic>vha-6::gfp</italic> strain worms was removed from the list of modulators, since the change in GFP was not in response to heme.</p><p>A COPAS BioSort worm sorter (Union Biometrica, Holliston, MA) was used to measure GFP levels in live worms. Plates, bacteria, and worms were prepared and treated as described in the previous section. After 84 h on RNAi plates, P<sub>0</sub> gravid and F<sub>1</sub> L1-stage worms were washed from each well with 600 &#181;L of M9 buffer containing 0.01% Tween-20, transferred to a 1.5-mL microcentrifuge tube, and allowed to settle for 5 min. The supernatant was removed and discarded. Each worm pellet was transferred to an individual well of a 96-well plate. Duplicate samples were transferred to successive wells in the 96-well plate and were separated from other samples by an empty well, which served to flush the flow cell where worms are analyzed and prevent contamination of subsequent samples. The contents of each well were washed, aspirated, and analyzed by a COPAS BioSort worm sorter. The GFP gain was set to 2, and the GFP PMT setting was 400. Using highly synchronized worms in the gravid stage, we had previously defined the gate settings in order to ensure that the data obtained from P<sub>0</sub> gravid animals would be easily and quickly separated from the data obtained from worms in other developmental stages. Text file data was imported into Microsoft Excel and sorted based on the gating parameters recorded in the &#8220;Status Select&#8221; column. The worm sorter records a fluorescence profile of each worm in the form of a curve, which reflects the intensity of GFP from the mouth to the tail. The &#8220;Green&#8221; column recorded the GFP value of the area under the curve, reduced by a factor of 40,000. The background levels of GFP were subtracted from all values used to generate <xref ref-type="fig" rid="pgen-1001044-g007">Figure 7A</xref>. The background level of GFP was equal to the GFP levels in IQ6011 worms feeding on HT115(<italic>DE3</italic>) bacteria transformed with the <italic>gfp</italic> RNAi vector. The COPAS BioSort detects very low levels of GFP in these worms. The mean of all values for each sample was determined, and the average of each duplicate was calculated. This mean was normalized to the average value for the GFP obtained from the vector-only sample, and reported in arbitrary units &#177; SEM for each clone analyzed.</p><p>Synchronized, F<sub>1</sub> wild-type worms in the L1 larval stage were obtained from P<sub>0</sub> worms grown in mCeHR-2 containing 1.5 &#181;M hemin. Equal numbers of these F<sub>1</sub> worms were placed on NGM agar plates containing 2 mM IPTG, 50 &#181;g/mL carbenicillin, 12 &#181;g/mL IPTG and plated with a lawn of HT115(<italic>DE3</italic>) RNAi feeding bacteria harboring the respective L4440 plasmid that had been grown in LB broth with carbenicillin and tetracycline <xref ref-type="bibr" rid="pgen.1001044-Rajagopal1">[12]</xref>. Worms were fed on the RNAi bacteria for &#8776;60 h and allowed to develop to the late L4 stage. At this point, worms were transferred to fresh RNAi plates containing 1.5 &#181;M GaPP. Worms developed to the gravid stage and laid eggs. After 24 h of egg-laying, the P<sub>0</sub> worms (all in the gravid stage) were discarded in order to prevent additional eggs from being laid. On day 5, both the total number of surviving larvae and the number of unhatched eggs were counted. <italic>P</italic> values for statistical significance were calculated by using a one-way ANOVA with Student&#8211;Newman&#8211;Keuls multiple comparisons test by using GraphPad InStat v. 3.06 (GraphPad, San Diego, CA).</p><p>Equal numbers of synchronized N2 L1 larvae obtained from P<sub>0</sub> worms grown in mCeHR-2 plus 2 &#181;M hemin were exposed to the RNAi bacteria on NGM plates containing 2 mM IPTG for 72 h. This was followed by exposure to 5 &#181;M ZnMP plus 1.5 &#181;M hemin chloride for 16 h in mCeHR-2 medium. ZnMP fluorescence intensity was measured as described previously <xref ref-type="bibr" rid="pgen.1001044-Rajagopal1">[12]</xref>.</p><p>GFP reporter fusion constructs were created using the Gateway cloning system (Invitrogen, Frederick, MD). The promoter of interest, <italic>gfp</italic> gene, and the 3&#8242; untranslated region of the <italic>unc-54</italic> gene were cloned by recombination into the entry vectors pDONR P4-P1R, pDONR 221, and pDONR P2R-P3, respectively, using the Gateway BP Clonase kit. Sequence verified entry clones were then recombined into a destination vector pDEST R4-R3 using the Gateway LR Clonase II plus enzyme kit to produce the final recombinant plasmid.</p><p>For microparticle bombardment, &#8776;5&#215;10<sup>6</sup>
<italic>unc-119</italic> (<italic>ed3</italic>) gravid worms were co-bombarded with 10 &#181;g of Gateway reporter construct and 5 &#181;g of <italic>unc-119</italic> rescue plasmid (pDM016B) using the PDS-1000 particle delivery system (Bio-Rad, Hercules, CA). Worms were washed from bombardment plates and transferred to plates seeded with a lawn of <italic>E. coli</italic> strain JM109. After two-weeks at 25&#176;C, multiple wild-type F<sub>2</sub> worms were screened for gene integration either by PCR or transgene expression. Individual transgenic lines were isolated and transferred to axenic liquid mCeHR-2 medium supplemented with antibiotics. After two weeks of serial passages, worms were bleached and maintained as transgenic strains in axenic liquid mCeHR-2 medium.</p><p>The microarray data was submitted to GEO on Aug 6, 2007. The GEO accession number is GSE8696 and available at</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE8696">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE8696</ext-link>.</p>