<p>DT40 cells were cultured in RPMI-1640 medium supplemented with 10<sup>&#8722;5</sup> M &#946;-mercaptoethanol, penicillin, streptomycin, 10% fetal calf serum (FCS), and 1% chicken serum (Sigma, St Louis, MO, USA) at 39.5&#176;C.</p><p>To generate <italic>CtIP</italic> gene disruption constructs, genomic DNA sequences of DT40 cells were amplified using primers <named-content content-type="gene">5&#8242;-GGATGCGGAGAGGCTTGAAGAGTTTTACAC-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-TTACAGCACAACGATCACATAATCCCGCTC-3&#8242;</named-content> for the 5&#8242; arm, and <named-content content-type="gene">5&#8242;-GGAGCTTCTAGCAATACGCGGAACAACTCA-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-GCTTCCCCTCCAATTCTTGACTGAGAATCA-3&#8242;</named-content> for the 3&#8242; arm. The amplified PCR products were cloned into the pCR2.1-TOPO vector (Invitrogen, CA, USA). The <italic>Bam</italic>HI site in the plasmid that contains the 5&#8242; arm was disrupted by blunt-self ligation. The 1.6-kb <italic>Hind</italic>III fragment was ligated into the partially-digested <italic>Hind</italic>III site of the 3.0-kb 3&#8242; arm containing the plasmid. A drug-resistance gene (hisD or bsr) was inserted into the <italic>Bam</italic>HI site of the pCR2.1 vector containing both the 5&#8242; and 3&#8242; arms. To generate <italic>CtIP<sup>+/&#8722;/&#8722;</sup></italic> cells, linearized <italic>CtIP</italic> gene-disruption constructs were transfected sequentially by electroporation (BioRad). The genomic DNA of the transfectants was digested with <italic>Sac</italic>I and the targeted clones were confirmed by Southern blot analysis. The 0.5-kb fragment was amplified using primers <named-content content-type="gene">5&#8242;-GATTGTATGCTTCAGAGGCTCCTGC-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-GAAATTCCCAACTTTAGCTCCCCTTGAC-3&#8242;</named-content> and used as a probe. To construct the <italic>CtIP</italic> expression plasmid, chicken the <italic>CtIP</italic> open reading frame was amplified by PCR, using the primers <named-content content-type="gene">5&#8242;-GGGGACAAGTTTGTACAAAAAAGCAGGCTTCGAACCATGAATGCGTCTGGGGGAACTTGTG-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-GGGGACCACTTTGTACAAGAAAGCTGGGTCTTATGTCTTCTGCTCTTTGCCTTTTGG-3&#8242;</named-content>, and cloned into a Gateway donor vector, pDONR207 (Invitrogen, CA, USA), by BP reaction. The <italic>CtIP</italic> gene in the donor vector was transferred to an expression vector (pA-puro) containing the Gateway conversion cassette under the <italic>&#946;-actin</italic> promoter by LR reaction. To construct a <italic>CtIP</italic> expression vector under the control of a tetracycline-repressible promoter, <italic>CtIP</italic> cDNA was amplified by PCR, using the primers <named-content content-type="gene">5&#8242;-CTCGAGATGAATGCGTCTGGGGGAACTTGTG-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-GTCGACTTATGTCTTCTGCTCTTTGCCTTTTGG-3&#8242;</named-content>, and cloned into pCR2.1-TOPO vector. The <italic>Xho</italic>I-<italic>Sal</italic>I fragment containing the <italic>CtIP</italic> cDNA was blunted and cloned into the <italic>Eco</italic>RV site of a modified pTRE2 vector (Clontech, CA, USA) containing a loxP-flanked puromycin-resistant cassette.</p><p>
<italic>CtIP<sup>+/&#8722;/&#8722;</sup></italic> cells were introduced with the tetracycline-controlled trans-activator (tTA) gene through retrovirus infection. Infected cells were sub-cloned, and tTA expression was confirmed by western blot analysis. The resulting tTA-expressing <italic>CtIP<sup>+/&#8722;/&#8722;</sup></italic> cells were transfected with the pTRE2 puro<sup>R</sup>/CtIP, and puromycin-resistant clones were selected to isolate the <italic>CtIP<sup>+/&#8722;/&#8722;</sup>tetCtIP</italic> cells. The puromycin-resistance gene was then deleted by transiently expressing the Cre recombinase (Amaxa solution T, program B-23). Puromycin-sensitive <italic>CtIP<sup>+/&#8722;/&#8722;</sup>tetCtIP</italic> cells were transfected with the <italic>CtIP</italic> gene-disruption construct carrying the puromycin-resistant cassette to generate <italic>CtIP<sup>&#8722;/&#8722;/&#8722;</sup>tetCtIP</italic> cells.</p><p>The targeting vectors for the <italic>CtIP</italic> S332A mutants were generated by site-directed mutagenesis. To generate the S332A knock-in vector, genomic DNA was amplified by PCR, using primers <named-content content-type="gene">5&#8242;-ATTATGCCCCTGAAAGAAGGGAAAC-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-TTTCCTGGGTTTGCTCTTGATTTT-3&#8242;</named-content>, and cloned into the pCR2.1-TOPO vector (Invitrogen, CA, USA). Site-directed mutagenesis was performed using primers <named-content content-type="gene">5&#8242;-GATTCTCAGGTAGTTGCTCCTGTTTTCGGA-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-TCCGAAAACAGGAGCAACTACCTGAGAATC-3&#8242;</named-content>. The puromycin-resistance gene was inserted into the <italic>Hpa</italic>I site of the resulting plasmid. After transfection of the S332A knock-in vector into the <italic>CtIP<sup>+/+/&#8722;</sup></italic> cells, the targeted clones were selected against puromycin and then identified by Southern blot analysis of genomic DNA digested with <italic>Hind</italic>III. To make probe DNAs, the 0.6-kb fragments were amplified using primers <named-content content-type="gene">5&#8242;-GACTAACAAAGATCAACCTGTC-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-GTGCATGAGATTTTGGTCGTTG-3&#8242;</named-content>. After the deletion of the puromycin-resistance gene by transiently expressing Cre recombinase by nucleofection (Amaxa, Germany), the third allele of the <italic>CtIP</italic> gene was disrupted by transfecting the <italic>CtIP</italic> gene-disruption construct carrying the puromycin-resistance gene. The insertion of the S332A mutation into the endogenous <italic>CtIP</italic> gene was confirmed by RT-PCR followed by sequencing amplified DNA.</p><p>The puromycin-resistant cassette in the targeting vector for the <italic>BRCA1</italic> gene <xref ref-type="bibr" rid="pgen.1000828-Martin1">[31]</xref> was replaced with the neomycin-resistant cassette. <italic>CtIP<sup>+/&#8722;/&#8722;</sup></italic> and <italic>CtIP<sup>S332A/&#8722;/&#8722;</sup></italic> cells were sequentially transfected with targeting vectors containing the puromycin- and neomycin-resistant gene, and selected against G418 and puromycin, respectively. The clones with the disrupted <italic>BRCA1</italic> gene were identified by Southern blot analysis as described previously <xref ref-type="bibr" rid="pgen.1000828-Martin1">[31]</xref>.</p><p>Quantitative real-time PCR was performed in an ABI Prism 7000 sequence detector (Applied Biosystems) using SYBR Green PCR Master Mix reagent (Applied Biosystems) according to the manufacturer's instruction. <italic>CtIP</italic> cDNA was amplified using primers <named-content content-type="gene">5&#8242;-GGAATTGGAGGAGCAAAAGCAAC-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-GAAACTCACTGTTGCTCTTTG-3&#8242;</named-content>. The expression level of <italic>CtIP</italic> was normalized against <italic>&#946;-actin</italic> using the comparative CT method.</p><p>For Western blot analysis, the antibodies specific for CtIP (BL1914, Bethyl, TX, USA), &#946;-actin (Sigma, MO, USA), Rad51 (Ab-1, Calbiochem, CA, USA) were used for detection of each protein. Secondary antibodies were horseradish peroxidase (HRP)-conjugated antibodies to mouse Ig (GE Healthcare, MA, USA) and HRP-conjugated antibody to rabbit Ig (Santa Cruz, CA, USA).</p><p>Karyotype analysis was performed as described previously <xref ref-type="bibr" rid="pgen.1000828-Sonoda2">[25]</xref>. To measure the number of &#947;-ray-induced chromosome breaks in mitotic cells, we exposed cells to 2 Gy &#947;-rays and immediately added colcemid. At 3 hours after irradiation, mitotic cells were harvested and subjected to chromosome analysis.</p><p>Methylcellulose colony formation assays were performed as described previously <xref ref-type="bibr" rid="pgen.1000828-Takata1">[30]</xref>,<xref ref-type="bibr" rid="pgen.1000828-Okada1">[46]</xref>. Since in this assay the plating efficiency of <italic>BRCA1</italic>-deficient cells was less than 50%, we used a different assay to measure cellular sensitivity to DNA-damaging agents. Cells (1&#215;10<sup>3</sup>) were seeded into 24-well plates containing 1 ml culture medium per well and the DNA-damaging agents, and then incubated at 39.5&#176;C for 48 hours. To assess the number of live cells, we measured the amount of ATP in the cellular lysates. We confirmed that the number of live cells was closely correlated with the amount of ATP. This ATP assay was carried out with 96-well plates using a CellTiter-Glo Luminescent Cell Viability Assay Kit (Promega Corporation, WI, USA). Briefly, we transferred 100 &#181;l of cell suspension to the individual wells of the plates, placed the plates at room temperature for approximately 30 minutes, added 100 &#181;l of CellTiter-Glo Reagent, and mixed the contents for 2 minutes on an orbital shaker to induce cell lysis. The plate was then incubated at room temperature for 10 minutes to stabilize the luminescent signal. Luminescence was measured by Fluoroskan Ascent FL (Thermo Fisher Scientific Inc., MA, USA).</p><p>The measurement of homologous recombination frequencies using a <italic>SCneo</italic> cassette <xref ref-type="bibr" rid="pgen.1000828-Fukushima1">[28]</xref> and <italic>CENP-H-EGFP</italic> was performed as described previously <xref ref-type="bibr" rid="pgen.1000828-Kikuchi1">[47]</xref>. After the I-<italic>Sce</italic>-I vector was transfected into the cells, the frequency of neomycin-resistant colony formation was measured.</p><p>To enrich DT40 cells in the G<sub>1</sub> phase, cells were synchronized by centrifugal counterflow elutriation (Hitachi Industrial, Japan). The cell suspension (&#8764;5&#215;10<sup>7</sup>) was loaded at a flow rate of 11 ml/min into an elutriation chamber running at 2,000 rpm. The first 50 ml was discarded, and the following 100 ml was used as a G<sub>1</sub>-phase cell fraction.</p><p>Fluorescence microscopy was carried out and images were obtained and processed using the IX81 (Olympus, Japan). Cells were cultured in medium containing BrdU (10 &#181;M) for 24&#8211;48 h to sensitize them to DSB generation by means of a 405 nm laser from a confocal microscope (FV-1000, Olympus, Japan). During laser treatment, cells were incubated in phenol red-free Opti medium (GIBCO, NY, USA) to prevent the absorption of the laser's wavelength. &#947;-irradiation was performed using <sup>137</sup>C (Gammacell 40, Nordion, Kanata, Ontario, Canada). Antibodies against Rad51 (Ab-1, Calbiochem, CA, USA), FK2 (Nippon Biotest Laboratories, Japan), RPA p32 (GeneTex, TX, USA), rabbit Ig (Alexa 488-conjugated antibody, Molecular Probe, OR, USA), and mouse Ig (Alexa 594-conjugated antibody, Molecular probe, OR, USA) were used for visualization.</p>