<p>All participating centers provided local institutional review board approval for genetic analysis, and each participant provided informed consent for genetic testing.</p><p>Patients have been included from two sources (<italic><xref ref-type="supplementary-material" rid="pgen.1000791.s016">Text S1</xref></italic>): <xref ref-type="bibr" rid="pgen.1000791-Telenti1">[1]</xref> Euro-CHAVI, a Consortium of 8 European and 1 Australian Cohorts/Studies; and <xref ref-type="bibr" rid="pgen.1000791-OBrien1">[2]</xref> MACS, the Multicenter AIDS Cohort Study that enrolled homosexual and bisexual men in 4 US cities. In general, patients had viral load (VL) and CD4 count monitoring at least 6 monthly. Eligible patients had 3 or more stable plasma HIV RNA results in the absence of antiretroviral treatment, and met one of the following criteria: a valid seroconversion date estimation proven by documents or biological markers; or, for seroprevalent patients, VL data over a period of at least 3 years, diverging by no more than 0.5 log. Only individuals with known date of seroconversion and at least 2 CD4 T cell determinations in absence of potent antiretroviral treatment (cART) were included in the progression analysis.</p><p>HIV-1 set point was defined as the average of the remaining VL results after careful assessment of each individual data and elimination of VL outliers: see <italic><xref ref-type="supplementary-material" rid="pgen.1000791.s016">Text S1</xref></italic> for criteria used to identify and exclude outlier VL data. Of note, our phenotype definition leads to the exclusion rapid progressors that never reach a stable VL plateau. The disease progression phenotype was defined as <xref ref-type="bibr" rid="pgen.1000791-Telenti1">[1]</xref> the drop of CD4 T cells below 350/ul, or <xref ref-type="bibr" rid="pgen.1000791-OBrien1">[2]</xref> the initiation of cART, but only if the last CD4 T cell count before cART start was &lt;500/ul. This later criterion was made necessary by the different rationales behind treatment initiation between patients and over time: a large part of patients starting cART with CD4 T cell counts approaching the 350/ul threshold did so because their CD4 slope actually showed a significant decrease, whereas most of patients who started treatment with normal or subnormal CD4 T cell counts had stable CD4 T cell profiles. Since progression has been represented in a number of different ways in genetic studies, we used the known genetic determinants to confirm that our measure is the most accurate, by comparing it to survival analyses that used cART start either as a censoring event (i.e. all patients starting cART are considered non-progressors) or as a progression event (i.e. all patients starting cART are considered progressors) (<xref ref-type="supplementary-material" rid="pgen.1000791.s012">Table S8</xref>).</p><p>All participants were genotyped using Illumina BeadChips: We used HumanHap550 Beadchips for 1633 samples and Human1M Beadchips for 921 samples. Most common SNPs found in the HapMap CEU population are readily covered by both chips; however, it is not clear yet whether common variants that have not been genotyped in the HapMap project will be measured equally well. To increase the coverage of the MHC region in the samples genotyped with the 550K chip, we designed a customized MHC-chip that contained an additional 8000 SNPs, largely overlapping with the variants that are present on the Human1M chip. We carried out a series of data cleaning and quality control procedures: SNPs were filtered based on missingness (drop if call rate &lt;99%), MAF (drop if &lt;0.006) and Hardy-Weinberg Equilibrium deviation. Participants were filtered based on call rate, gender check (heterozygosity testing), cryptic relatedness and population structure (see EIGENSTRAT below). The genetic variants located in HIV-1 candidate genes that were not represented directly or indirectly on the genome-wide chips were independently genotyped with TaqMan assays.</p><p>The quality control of the genotyping data included a check on the gender specification obtained from the phenotype database, using the observed genotypes of SNPs on chromosome X and Y. Subjects that were identified as &#8220;male&#8221; in the phenotype file but had a significant amount of heterozygous X genotypes (&gt;&#8202;=&#8202;1%), as well as subjects that was identified as &#8220;female&#8221; in the phenotype file but had a high frequency of homozygous X genotypes (&gt;&#8202;=&#8202;80%) and Y genotype readings were excluded.</p><p>To control for the possibility of spurious associations resulting from population stratification we used a modified EIGENSTRAT approach. This method derives the principal components of the correlations among gene variants and corrects for those correlations in the association tests. In principle therefore the principal components in the analyses should reflect population ancestry. Having noticed however that some of the leading axes depend on other sources of correlation, such as sets of variants near one another that show extended association (LD), we inspected the SNP loadings and followed a series of pruning procedures to ensure that EIGENSTRAT axes reflected only effects that applied equally across the whole genome (<italic><xref ref-type="supplementary-material" rid="pgen.1000791.s016">Text S1</xref></italic>).</p><p>The core association analyses on HIV-1 setpoint focused on single-marker genotype-trend tests of the quality control&#8211;passed SNPs using linear regression and including age, gender, and the significant EIGENSTRAT axes values as covariates. Associations with progression were tested using a Cox proportional hazards model. We assessed significance with a Bonferroni correction taking into account 1 million tests (p-value cutoff &#8202;=&#8202; 5E&#8211;08).</p><p>To search for additional SNPs effects in the MHC region we used a forward selection algorithm to investigate all MHC SNPs with p&lt;1E&#8211;04 (N&#8202;=&#8202;331). The top 2 associated variants and the standard covariates were fixed in a linear regression model, while new SNPs were added and selected into the model one at a time. In the forward selection process, a newly selected SNP is expected to explain a certain proportion of the set point variation independently of all variables already in the model. To control for multiple testing, a permutation procedure was used to assess the empirical significance cut-off value. In the permutation, the LD patterns among SNPs were retained while the associations between SNPs and set point were permuted. 1000 permutation runs were performed and the 5<sup>th</sup> percentile of the empirical distributions of the partial R-squares and the p-values of the 1<sup>st</sup>, 2<sup>nd</sup>, 3<sup>rd</sup>, etc. selected SNPs were recorded and compared to the observed sample statistics. The permutation p-value was defined as the probability that the statistics observed in the permutation was more extreme than the statistics observed in the real sample. The forward selection algorithm is expected to be conservative. To verify this, a large-scale statistical simulation was performed, considering different effect sizes, various LD patterns among SNPs and different sample sizes (<italic><xref ref-type="supplementary-material" rid="pgen.1000791.s016">Text S1</xref></italic>). The simulation results confirmed that the forward selection algorithm with permutation procedure is conservative.</p><p>We performed a two-way interaction test between each QC-passed SNP and the top associated variants. For each genome-wide significant polymorphism, the screening procedure involved the calculation of the p-value associated with each of the &#8764;1 million multivariate linear models incorporating the 2 polymorphisms. The p-value associated with the interaction term for this model was retained and then the top p-values for all &#8764;1 million tests considered. This approach followed recommendations that the search space for interactions can be appropriately reduced and therefore the power can be improved by focusing the search on pairs of polymorphisms including at least one that is known to be significant <xref ref-type="bibr" rid="pgen.1000791-Brem1">[39]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The SNPs that were represented in all genotyping chips were mapped to their closest gene (if &lt;500 kb away) and used in the GSEA analysis if minor allele frequency was &gt;0.05, Hardy-Weinberg Equilibrium test p-value was &gt;0.001, and at least 90% individuals were successfully genotyped. Among the 639 canonical pathway gene sets that are represented in the Molecular Signatures Database (MsigDB, <ext-link ext-link-type="uri" ns0:href="http://www.broad.mit.edu/gsea/msigdb/collections.jsp">http://www.broad.mit.edu/gsea/msigdb/collections.jsp</ext-link>), we tested the 298 gene sets that had at least 20 and at most 200 genes represented in the study (<italic><xref ref-type="supplementary-material" rid="pgen.1000791.s016">Text S1</xref></italic>).</p><p>A permutation procedure was developed to test whether genetic variants with known functional role were more likely to have a lower p-values distribution than supposedly neutral variants (<italic><xref ref-type="supplementary-material" rid="pgen.1000791.s016">Text S1</xref></italic>). We compared the overall p-value distribution of 12,535 putatively functional polymorphisms, including stop-gained, stop-lost, frame-shift coding, non-synonymous coding, and essential splicing site genetic variants, with the global distribution of p-values of all genotyped SNPs (with 10,000 permutations). We also ran the same analysis for the MHC region only (318 functional variants) and for the rest of the genome (12217 functional variants).</p>