<p>Chicken 6C2 erythroleukemia cells carrying <italic>IL-2R</italic> reporter transgenes (8103, wild type HS4; 10401, &#916;FI; 10506, &#916;FII; 10615, &#916;FIII; 10901, &#916;FIV; 8d5, &#916;FV) were cultured and assayed for IL-2R expression by FACS as described previously <xref ref-type="bibr" rid="pgen.1000804-RecillasTarga1">[12]</xref>,<xref ref-type="bibr" rid="pgen.1000804-West3">[14]</xref>. Genomic DNA was extracted from cell lines after 30 and 90 days of culture and bisulfite modified (EZ Methylation, Zymo Research, CA). The upstream double-core HS4 elements and the IL-2R promoter were PCR amplified from each line. We were unable to amplify bisulfite modified double-core HS4 elements from the 10506, &#916;FII line to sufficient levels to provide representative sequence data. We therefore opted to amplify the outermost HS4 copy only. The 8d5, &#916;FV line also only contains one copy of HS4 in the upstream location <xref ref-type="bibr" rid="pgen.1000804-RecillasTarga1">[12]</xref>. All PCR products were gel purified and cloned, followed by sequencing (GATC Biotech, Konstanz, Germany) of 10 clones for each region of interest.</p><p>
<italic>Gel mobility shift assays</italic> were performed as described previously <xref ref-type="bibr" rid="pgen.1000804-West3">[14]</xref>. Recombinant VEZF1, SP1, SP3 and ZF5 were produced by <italic>in vitro</italic> translation using rabbit reticulocyte lysate (Promega).</p><p>FI- and FIII-binding proteins were purified from adult chicken red blood nuclear protein extracts by ion exchange chromatography. Throughout the purification, eluate fractions were analyzed for FI- and FIII-binding activity with gel mobility shift assays. The binding specificity of partially purified proteins was checked by competition analysis after each purification step. FI- and FIII-binding activities co-fractionated following ion exchange chromatography with SP XL and Heparin sepharose (GE Healthcare). Phosphocellulose, Q and Sephacryl S300 columns were used in early purification attempts to resolve FI- and FIII-binding activities but they co-fractionated in each case (data not shown). FI- and FIII-binding activities both eluted in two distinct fractions of approximately 200 and 400 kDa following gel filtration (data not shown). The active fractions eluted from heparin sepharose were pooled then split into two and fractionated in parallel by FI or FIII DNA affinity as described previously <xref ref-type="bibr" rid="pgen.1000804-West3">[14]</xref>. Polypeptides electrophoresed on 7% Tris-acetate gels (Invitrogen) were excised, digested with trypsin and sequenced at the Harvard Microchemistry Facility by microcapillary reverse-phase HPLC nano-electrospray tandem mass spectrometry (&#956;LC/MS/MS) on a Finnigan LCQ DECA quadrupole ion trap mass spectrometer.</p><p>Chicken VEZF1/BGP1 cDNA was cloned following RT-PCR from 6C2 cell total RNA based on an assumption of conservation of 5&#8242; cDNA sequence with human VEZF1/DB1. The oligonucleotide Adaptor-A <named-content content-type="gene">5&#8242;CATGCCGCTCGAGCGGTTTTTTTTTTTTTTTTT</named-content> was used in first strand cDNA synthesis with Superscript II reverse transcriptase (Invitrogen). The primers VEZF1_5&#8242;, <named-content content-type="gene">5&#8242;CCATGACCCATGGGCAGAGCCAAAGT</named-content> and Adaptor <named-content content-type="gene">5&#8242;CATGCCGCTCGAGCGG</named-content> were used to amplify a full length chicken VEZF1 cDNA by PCR which was TA cloned into pCRII (Invitrogen). VEZF1 cDNA was sub-cloned into pCITE4b (Novagen) to generate p4bVEZFfull for the purpose of <italic>in vitro</italic> transcription. cDNA encoding chicken ZF5 was isolated by RT-PCR from 6C2 cell total RNA using primers designed from the published sequence (U51640). We found that the bases CpG 1306-7 in the published sequence were GpC in our clone, causing codon 436 to translate as alanine instead of arginine. We obtained the vectors pCDNA3-ZF5 and pEVRFO-ZF5 (a kind gift of W. Stumph, San Diego State University) and we also found the CpG to GpC conflict with the published sequence. Full length chicken SP1 and SP3 cDNAs cloned in the pBluescript-based vectors pH-SP1 and pH-SP3-3 were a kind gift from Marc Castellazzi (INSERM, Lyon).</p><p>Polyclonal antibodies were raised (Rockland Immunochemicals) against chicken VEZF1 (Ser376-Ala547) and chicken ZF5 peptides (Ser131-Lys248) produced in <italic>E.coli</italic> (QIAexpress, Qiagen). Peptides were produced in M15 [pREP4] <italic>E. coli</italic> followed by rapid lysis with B-PER reagent (Pierce). ZF5 peptide was soluble and purified on Ni-NTA agarose (Qiagen). VEZF1 peptide was insoluble and resulting inclusion bodies were prepared using B-PER reagent (Pierce), solubilized with 6M guanidium hydrochloride and immobilized on TALON Sepharose resin (Clontech) at pH 7. VEZF1 peptide was renatured in a stepwise manner with 6, 4, 3, 2, 1 and 0.5 M guanidium hydrochloride prior to elution. VEZF1 and ZF5 polyclonal IgG antibodies (Rockland Immunochemicals) were purified from rabbit serum using PROSEP-A media (Montage, Millipore). Anti-full length chicken VEZF1 antibodies were raised previously <xref ref-type="bibr" rid="pgen.1000804-Gowher1">[27]</xref>. Antibodies raised against CTCF (06-917), SP1 (PEP2X, H-225X), SP3 (D20X) and USF1(B01) were obtained from Millipore, Santa Cruz Biotechnology and Abnova, respectively.</p><p>ChIP analysis of transcription factor binding in chicken cells was performed using formaldehyde crosslinked chromatin prepared from chicken 10 day embryonic erythrocytes isolated from fertilized White Leghorn eggs (CBT Farms, Chestertown, MD) or cultured 6C2 erythroleukemia cells. 10 day erythrocytes were washed and resuspended in 25 mls of PBS (2&#215;10<sup>7</sup> cells/ml) and fixed with a final concentration of 0.25% formaldehyde at room temperature for 30 seconds. 6C2 cells were harvested in mid-exponential growth phase, divided into 30 ml aliquots containing 1&#215;10<sup>8</sup> cells in fresh media and fixed with a final concentration of 0.8% formaldehyde at room temperature for 5 minutes. Reactions were stopped by adding glycine to a final concentration of 0.125 M. The cells were washed in PBS and resuspended in cell lysis buffer (0.25% Triton X-100, 10 mM EDTA, 0.5 mM EGTA, 10 mM Tris pH 8.0) to isolate nuclei. Chromatin was prepared following washing (0.2 M NaCl, 1 mM EDTA, 0.5 mM EGTA, 10 mM Tris pH 8.0) and lysis (NLB: 50 mM Tris-HCl pH 8.0, 10 mM EDTA, 0.5% SDS) of nuclei. Crosslinked chromatin was fragmented by sonicaton (Misonix) for a total time of 10 minutes in regular 10 second pulses at 4&#176;C. Debris was removed by centrifugation at 15000 g for 10 minutes and chromatin was diluted in 10 volumes of X-ChIP buffer (1.1% TX-100, 1.2 mM EDTA, 16.7 mM Tris pH 8.1, 167 mM NaCl). Agarose gel electrophoresis was used to confirm that chromatin fragments were &#8764;500 bp in length on average.</p><p>Chromatin was pre-cleared with 100 &#181;l of protein A agarose (50% slurry in X-ChIP buffer, Millipore) and 50 &#181;g of normal rabbit IgG (Santa Cruz) for 3 hours at 4&#176;C on a rotating wheel. Aliquots of chromatin were taken to generate input DNA and protein for western analysis. Individual ChIPs were performed using chromatin from 1&#215;10<sup>7</sup> cells in a total volume of 1 ml by diluting the pre-cleared chromatin with modified X-ChIP buffer (1 part NLB to 9 parts X-ChIP). Incubation with antibodies was performed overnight at 4&#176;C on a rotating wheel. Between 10 and 30 &#181;g of specific antibodies or 10 &#181;g of normal rabbit IgG (Santa Cruz) were used per ChIP. Chromatin was precipitated with protein A agarose (50 &#181;l slurry in X-ChIP, Millipore) for 4 hours at 4&#176;C with rotation. Immunoprecipitated chromatin was collected, washed, eluted and crosslinks reversed. DNA was then extracted by phenol-chloroform and ethanol precipitated in the presence of 10 &#181;g glycogen.</p><p>Relative DNA enrichments were quantified by TaqMan real-time qPCR using the comparative Ct method relative to input DNA and normalized to the primer set 10.35 within the 16 kb condensed chromatin region upstream of the chicken <italic>&#946;-globin</italic> locus as described previously <xref ref-type="bibr" rid="pgen.1000804-West3">[14]</xref>. The TaqMan primer sets 10.35 (&#8220;16 kb&#8221;), 21.54 (HS4 core, &#8220;end HS4&#8221;), 39.806 (beta-adult promoter), 50.861 (3&#8242;HS) and PGI5&#8242; (5&#8242; HS4 elements on IL-2R transgene, &#8220;trans HS4&#8221;) were used in this study.</p><p>10.35_For:&#8195;<named-content content-type="gene">GGAACAAGTTGGCAAGGTCCTAT</named-content>
</p><p>10.35_Rev:&#8195;<named-content content-type="gene">TCTTCTGCCCTGCCCGTAT</named-content>
</p><p>10.35_TM:&#8195;<named-content content-type="gene">FAM-TGCAGTTCCCTGTTCATGTGCTTTTCG-TAMRA</named-content>
</p><p>21.54_For:&#8195;<named-content content-type="gene">TCCTGGAAGGTCCTGGAAG</named-content>
</p><p>21.54_Rev:&#8195;<named-content content-type="gene">CGGGGGAGGGACGTAAT</named-content>
</p><p>21.54_TM:&#8195;<named-content content-type="gene">6FAM-CCCAAAGCCCCCAGGGATGT-TAMRA</named-content>
</p><p>39.8_For:&#8195;<named-content content-type="gene">CTGTGGTCTCCTGCCTCACA</named-content>
</p><p>39.8_Rev:&#8195;<named-content content-type="gene">AGGCTGGGTGCCCCTC</named-content>
</p><p>39.8_TM:&#8195;<named-content content-type="gene">FAM-CAATGCAGAGTGCTGTGGTTTGGAACTG-TAMRA</named-content>
</p><p>PGI_5&#8242;_For&#8195;<named-content content-type="gene">CACAGGAAACAGCTATGACATGATT</named-content>
</p><p>PGI_5&#8242;_Rev&#8195;<named-content content-type="gene">TCTGCCTTCTCCCTGATAACG</named-content>
</p><p>PGI_5&#8242;_TM&#8195;<named-content content-type="gene">6FAM-AATTCCTGCCCACACCCTCCTGC-TAMRA</named-content>
</p><p>ChIP analysis of transcription factor binding in murine E14Tg2A.4 ES cell lines was performed using formaldehyde crosslinked chromatin prepared from 1&#215;10<sup>9</sup> cells treated with 1% formaldehyde for 5 minutes. Chromatin was prepared as described above, where fragments sizes ranged from 500&#8211;700 bp. Semi-quantitative PCR was performed using the following primers</p><p>APRT1/2_For: <named-content content-type="gene">AAAGGCGTGCGGGAGCCAGAAAT</named-content>
</p><p>APRT1/2_Rev: <named-content content-type="gene">CCTTGGTAGGTGGGG</named-content>
</p><p>APRT3_For: <named-content content-type="gene">CCCTGTTCCTGGGCTCC</named-content>
</p><p>APRT3_Rev: <named-content content-type="gene">TGACTGGCCAGGAGG</named-content>
</p><p>ChIP analysis from human embryonic kidney 293-T cell line SD5 that contains a stably integrated copy of the 275 bp HS4 core chicken insulator was performed as described for cultured chicken cells above. SD5 cells were crosslinked with 1.6% formaldehyde for 5 minutes. The following primers were used in SYBR quantitative PCR analysis:</p><p>HS4_21.726_F: <named-content content-type="gene">CGGGATCGCTTTCCTCTGA</named-content>
</p><p>HS4_21.726_R: <named-content content-type="gene">CCGTATCCCCCAGGTGTCT</named-content>
</p><p>P_DHFR_F: <named-content content-type="gene">TCGCCTGCACAAATAGGGAC</named-content>
</p><p>P_DHFR_R: <named-content content-type="gene">AGAACGCGCGGTCAAGTTT</named-content>
</p><p>Control</p><p>VEZF1_CDS_F: <named-content content-type="gene">GACAGCAGCCGAACTTCGTT</named-content>
</p><p>VEZF1_CDS_R: <named-content content-type="gene">TGGTGCCCGAGGAAGATG</named-content>
</p><p>
<italic>APRT elements.</italic> The following elements were amplified from Hamster liver genomic DNA:</p><p>Sp1-like motifs in red. <italic>Ava</italic>II and <italic>Hpa</italic>II restriction sites in bold.</p><p>
<named-content content-type="gene">CTAGAGGATCCGGACAACACCCACACCGGCCCCTCCAGGTCCAGAAAGCTGGCCCTGCGAGAAGCGGGACTGAAAAGGCGTGCGGGAGCCAGAAATCCAAAAGGGTGCCAAGGCATGCGTCCTTTTTCCACCCAGAAATAACCCCAGGCTTTCAATTTGAGGTTATTTCAATATCCAGCAAATGCGTTACTTCCTGCCAAAAGCCAGCCTCCCCGCAACCCACTCTCCCAGAGGCCCCGCCCCGTCCCGCCCCCTCCCGGCCTCTCCTCGTGCTGGATCGCTCCCTAAGGACGCCCCGCTCCAGAAGCCCCACCTACCAAGGACGCCCCACCCTTGTTCCCGGACTGGTATGACCCCAGCCTGCTGACATCCCTCCGCCCTTTCTCGTGCACGCGGCTATGGCGGAATCTGAGTTGCAGCTGGTGGCGCAGCGATCCGCAGTTTCCCCGACTTCCCCATCCCCGGCGTGCTGTTTAGGTGAGATCACGAGCCAGCAAGGCGTTGGAGCCCTGTTCCTGGGCTCCCGGCGAGGCGCATGGGCAGTCTCGGGGATCTTGTGGGGTCTCCGCCCCCCTTTCCCCGGCCACCAGCCTCTCCTTGTTCCCAGGGATATCTCGCCCCTCCTGAAGGACCCCGCCTCCTTCCGAGCTTCCATCCGCCTCCTGGCCAGTCACCTTAAGTCCACGCATGGCGGCAAGATCGACTACATCGCAGGTCTA</named-content>
</p><p>Mutated bases in blue were introduced by site directed mutagenesis. Overall CpG content increased from 46 to 48 following these mutations. <italic>Ava</italic>II and <italic>Hpa</italic>II restriction sites in bold.</p><p>
<named-content content-type="gene">CTAGAGGATCCGGACAACACCCACACCGGCCCCTCCAGGTCCAGAAAGCTGGCCCTGCGAGAAGCGGGACTGAAAAGGCGTGCGGGAGCCAGAAATCCAAAAGGGTGCCAAGGCATGCGTCCTTTTTCCACCCAGAAATAACCCCAGGCTTTCAATTTGAGGTTATTTCAATATCCAGCAAATGCGTTACTTCCTGCCAAAAGCCAGCCTCCCCGCAACCCACTCTCCCAGAGACCCCCCGCATCCCCGACGCTACCCCCCGCATCCCCGATCTCCTCGTGCTGGATCGCTCCCTAAGGACGCCCCGCTCCAGAAGCCCCACCTACCAAGGACGCCCCACCCTTGTTCCCGGACTGGTATGACCCCAGCCTGCTGACATCCCTCCGCCCTTTCTCGTGCACGCGGCTATGGCGGAATCTGAGTTGCAGCTGGTGGCGCAGCGATCCGCAGTTTCCCCGACTTCCCCATCCCCGGCGTGCTGTTTAGGTGAGATCACGAGCCAGCAAGGCGTTGGAGCCCTGTTCCTGGGCTCCCGGCGAGGCGCATGGGCAGTCTCGGGGATCTTGTGGGGTCCGCTCCCCCCGCATCCCCGACACCAGCCTCTCCTTGTTCCCAGGGATATCTCGCCCCTCCTGAAGGACCCCGCCTCCTTCCGAGCTTCCATCCGCCTCCTGGCCAGTCACCTTAAGTCCACGCATGGCGGCAAGATCGACTACATCGCAGGTCTA</named-content>
</p><p>
<italic>Hpa</italic>II site in bold.</p><p>
<named-content content-type="gene">TCTAGATTGCTAGGAGTAGCACCTAAGATGAACTAGATGCTAAAAAATGCTGTATCTTTGGGGCACACGAGGGCATGCCTGGGCAGGCTTAGAGCCTGGTAGTCTCAGGGGCTGCACCAAAGTGTAATTCTTGTGCTAAATAACTTTCACTTACCAGTGCCAAGCACGGGCTTCAGAAACACCCTAGGGTCGCTGAATGTCCACCAGGGGAGTCAGACATGTCCAGAGGGTGAGAACCCCAGAGAATTCGGTAGCCCTGACATGTGCTACAATTACTGATGCCCACTTCCTACTGGTTCCTCCTGGCCATACCTCAGGAATTAGGGCATGCTTTCTGCCTGCTACAGTAGCTCATCCTCCCTGGAAGTGACCCCAGACATATACCCTGAACTGTAACCGATAAAGTGCGCCTGGGCAGATGTATTTGAGAGGTGGCAAAAGTAAACCATAGGTGTCCCCGAGCTAGATACAGAAGGCAGATAACATCCCCAAGGCTAAGCTGCTGCCCCAATAGCCATCAGCCTTCTAGTTATAGCTAGTAAGACCTAGTATTCCTGGTCAATACTATTCACTCAATCCTTACACCTCAGCCCTAACACGCCCCCTCTCTCATCCTAACAGGCCTAGACTCCAGGGGATTCTTGTTTGGCCCCTCCCTAGCTCAGGAGCTGGGCCTGGGCTGTGTGCTCATCCGGAAGCGAGGGAAGCTGCCAGGCCCCACAGTGTCAGCCTCCTATGCTCTCGAGTATGGCAAGGTAAGCAGGCAGTGGGTAGCTGTCTAGGAGTAAATGTGGGGGCTCAGAGAGGTTAAGTCATCAGGCCAGGTTTATACCACCAGGAAACATGGAGAAGCTAGGGGTGGTGGTTCTAGA</named-content>
</p><p>&#8764;720 bp wild type and mutant <italic>APRT</italic> CpG islands and an 870 bp non-island region of the <italic>APRT</italic> gene were cloned into pUC19. Each element was PCR amplified, half of which was subject to <italic>in vitro</italic> CpG methylation by M.<italic>Sss</italic>I methyltransferase(New England Biolabs). <italic>In vitro</italic> methylation was validated by <italic>Hpa</italic>II and <italic>Msp</italic>I digestion. 1 &#181;g of each <italic>APRT</italic> element was transfected into murine E14Tg2A.4 ES cells (BayGenomics) with 1 &#181;g of the <italic>Xba</italic>I fragment of pREP7 (Invitrogen). Hygromycin resistant clones were grown in LIF without feeder co-culture. Genomic DNA was extracted after two weeks of culture. Probes for Southern blotting were generated using Ready-to-go dCTP beads (GE Healthcare).</p><p>Oligonucleotides were generated on an ABI 394 DNA synthesizer.</p><p>The top strand sequences for each of the duplexes used for gel mobility shift analyses were:</p><p>FI wt <named-content content-type="gene">5&#8242; GGAGCTCACGGG<underline>GACAGCCCCCCCCCAAA</underline>GCCCCCAGGGA</named-content>,</p><p>FIII wt <named-content content-type="gene">5&#8242; aggcgcgccCCGGTCCGG<underline>CGCTCCCCCCGCATCCCCGA</underline>GCCGGggcgcgcct</named-content>,</p><p>FV wt <named-content content-type="gene">5&#8242; CCTGCAGACA<underline>CCTGGGGGGATACGGGGAAA</underline>AAGCTTTAGG</named-content>,</p><p>Sp1 <named-content content-type="gene">5&#8242; ATTCGATCGGGGCGGGGCGAGC</named-content>,</p><p>glo wt <named-content content-type="gene">5&#8242; AATTGCAGAGCTGGGAATCGGGGGGGGGGGGGGGGCGGGTGGTGGTGTGG</named-content>,</p><p>glo 7G <named-content content-type="gene">5&#8242; AATTGCAGAGCTGGGAATCGGGGGGGCGGGTGGTGGTGTGG</named-content>,</p><p>glo 6G <named-content content-type="gene">5&#8242; AATTGCAGAGCTGGGAATCGGGGGGCGGGTGGTGGTGTGG</named-content>.</p><p>
<italic>Asc</italic> I restriction sites used for cloning FIII sites in an earlier study are shown as lower case. All FI and FIII oligonucleotides were identical to the wt sequences above except for mutations indicated in <xref ref-type="fig" rid="pgen-1000804-g001">Figure 1E</xref>.</p><p>DNA affinity columns were prepared using the following oligonucleotides</p><p>FI-DA TOP <named-content content-type="gene">5&#8242; gatcTCACGGGGACAGCCCCCCCCCAAAGCCCCCA</named-content>
</p><p>FI-DA BOTTOM <named-content content-type="gene">5&#8242; gatcTGGGGGCTTTGGGGGGGGGCTGTCCCCGTGA</named-content>
</p><p>FIII-DA TOP <named-content content-type="gene">5&#8242; gatcGGTCCGGCGCTCCCCCCGCATCCCCGAGCCGGCA</named-content>
</p><p>FIII-DA BOTTOM <named-content content-type="gene">5&#8242; gatcTGCCGGCTCGGGGATGCGGGGGGAGCGCCGGACC</named-content>
</p><p>PCR primers used in bisulfite sequencing were</p><p>HS4 5&#8242; double forward&#8195;<named-content content-type="gene">5&#8242; GGTATTAGAGTAGATTGTATTGAGAGTGTA</named-content>
</p><p>HS4 5&#8242; double reverse&#8195;<named-content content-type="gene">5&#8242; CATAACTATTTCCTATATAAATCCCC</named-content>
</p><p>HS4 5&#8242; single forward&#8195;<named-content content-type="gene">5&#8242; AGAGTAGATTGTATTGAGAGTGTATTATA</named-content>
</p><p>HS4 5&#8242; single reverse&#8195;<named-content content-type="gene">5&#8242; ACATCCCTAAAAACTTTAAAAAAAA</named-content>
</p><p>IL-2R forward&#8195;<named-content content-type="gene">5&#8242; GTTAAGGTTGGGGGTTTTTT</named-content>
</p><p>IL-2R reverse&#8195;<named-content content-type="gene">5&#8242; AAAACTCTACCTAACAACCAAACAC</named-content>
</p><p>PCR primers used in RT-PCR gene expression analysis were</p><p>GgVEZF805anti&#8195;<named-content content-type="gene">5&#8242;CAGTGCACGTTTGGCATTTGAAG</named-content>
</p><p>GgVEZF716sense&#8195;<named-content content-type="gene">5&#8242;GAAAAGGCTTCTCGAGGCCTGATC</named-content>
</p><p>GgGAPDH-247T&#8195;<named-content content-type="gene">5&#8242;-6FAM-TCCAGGAGCGTGACCCCAGCA-TAMRA</named-content>
</p><p>GgGAPDH-226F&#8195;<named-content content-type="gene">5&#8242; ATGGGCACGCCATCACTATC</named-content>
</p><p>GgGAPDH-302R&#8195;<named-content content-type="gene">5&#8242; AACATACTCAGCACCTGCATCTG</named-content>
</p><p>GgB-ACTIN_F&#8195;<named-content content-type="gene">5&#8242; TGCTGCGCTCGTTGTTGA</named-content>
</p><p>GgB-ACTIN_R&#8195;<named-content content-type="gene">5&#8242; CATCGTCCCCGGCGA</named-content>
</p><p>GgB-ACTIN_T&#8195;<named-content content-type="gene">5&#8242;-6FAM-TGGCTCCGGTATGTGCAAGGCC-TAMRA</named-content>
</p><p>The primers used for methylation specific PCR analysis of the <italic>APRT</italic> non-island element were:</p><p>APRTNI_<named-content content-type="gene">5&#8242;&#8195;TCTAGATTGCTAGGAGTAGC</named-content>
</p><p>APRTNI_<named-content content-type="gene">3&#8242;&#8195;TCTAGAACCACCCCTAGC</named-content>
</p>