<p>A total of 25 fungus gardens from 5 healthy colonies (5 gardens each) of the leaf-cutter ant <italic>Atta colombica</italic> were collected at the end of May and beginning of June, 2008. These colonies are located along Pipeline Road in Soberan&#237;a National Park, Panama (latitude 9&#176; 7&#8242; 0&#8243; N, longitude 79&#176; 42&#8242; 0&#8243; W) and designated N9, N11, N12, N13, and N14, respectively. Each fungus garden was vertically cross-sectioned into thirds with the top third designated as the &#8220;top&#8221; of the garden and the bottom third designated as the &#8220;bottom&#8221; of the garden. All material was frozen and transported back to the University of Wisconsin-Madison where it was stored at &#8722;20&#176;C prior to processing.</p><p>From all 5 colonies (3 gardens per colony), 5 independent samples from fungus garden tops and bottoms of each garden were collected for sugar composition analysis. Thus, a total of 75 fungus garden samples each from the top and bottom were used for this part of our study. This material was tested for crystalline cellulose and hemicellulose (matrix polysaccharide) content as follows.</p><p>Cellulose content of fungus garden plant biomass was determined by first washing each sample using Updegraff reagent <xref ref-type="bibr" rid="pgen.1001129-Updegraff1">[31]</xref>, which removes matrix polysaccharides such as hemicelluloses, pectins and amorphous glucan. The remaining residue, containing only crystalline cellulose, was hydrolyzed using Saeman hydrolysis <xref ref-type="bibr" rid="pgen.1001129-Selvendra1">[32]</xref>. The resulting glucose monosaccharides were then quantified with an anthrone colourmetric assay as previously described <xref ref-type="bibr" rid="pgen.1001129-Selvendra1">[32]</xref>.</p><p>For the composition of the matrix polysaccharide content, the following components were tested: arabinose, fucose, galactose, glucose, rhamnose, mannose and xylose. Quantification of these sugars were performed by treating finely ground materials with solvents to remove pigments, proteins, lipids, and DNA from the material as previous described <xref ref-type="bibr" rid="pgen.1001129-York1">[33]</xref>. The residue was de-starched with an amylase treatment, resulting in only cell wall material. This material was then treated with 2M trifluoroacetic acid solubilizing the matrix polysaccharides in form of their monosaccharides, and subsequently derivatized into their corresponding alditol-acetates, which were separated and quantified by GC-MS as previously described <xref ref-type="bibr" rid="pgen.1001129-Albersheim1">[34]</xref>.</p><p>The same set of samples used for sugar composition analysis was also used for lignin content analysis, as previously described <xref ref-type="bibr" rid="pgen.1001129-Jung1">[35]</xref>. Briefly, all samples were dried to 60&#176;C and ground using a 1-mm cyclone mill and analyzed for total non-lignin organic matter, lignin, and ash (organic and inorganic) content. Total carbohydrate content was assessed through a two-step acid hydrolysis with neutral sugars quantified using GC and uronic acids quantified using colorimetry. Klason lignin was quantified from the ash-free residue from the two-step acid hydrolysis. Ash content was quantified by combustion at 450&#176;C for 18 h and the average &#181;g/mg of material was calculated.</p><p>Total DNA was extracted in preparation for either 16S rDNA sequencing or community metagenomic sequencing. For 16S rDNA sequencing, a total of 5 gardens each from 3 <italic>Atta colombica</italic> colonies (N9, N11, and N12) were used. A total of 1 g (wet weight) of fungus garden material was sampled from the top layer of each garden corresponding to each colony, for a combined final weight of 5 g of fungus garden material. Total DNA from this sample was then extracted using a MoBio Power Soil DNA Extraction Kit (MOBIO Laboratories, Carlsbad, CA, USA). The same procedures were performed for all fungus garden bottom layer samples for all 3 colonies.</p><p>For community metagenomic sequencing, total community DNA was extracted from 5 whole fungus gardens each from all 5 <italic>Atta colombica</italic> colonies used in this study. A total of 1 g of fungus garden material was sampled from top, middle, and bottom layers from all fungus gardens and combined to produce a final sample weight of 75 g. This material was then enriched for bacteria using a modification of a previously-described protocol <xref ref-type="bibr" rid="pgen.1001129-Apajalahti1">[36]</xref>. Briefly, total fungus garden material was buffered in 1X PBS (137 mM NaCl, 2.7 mM KCl, 10 mM Na<sub>2</sub>HPO<sub>4</sub>, 2 mM KH<sub>2</sub>PO<sub>4</sub>) containing 0.1% Tween and then centrifuged for 5 minutes at 40&#215;g. This resulted in a 3-layer mixture containing leaf-material at the top, fungal mass in the middle, and bacteria at the bottom. The top and middle layers were carefully removed, buffered with 1X PBS containing 0.1% Tween, and washed using the same centrifugation method an additional 3 times. The final mixture was then centrifuged for 30 minutes at 2800&#215;g, re-suspended in 1X PBS containing 0.1% Tween and filtered through a 100 um filter. Total DNA from this resulting sample was then extracted using a Qiagen DNeasy Plant Maxi Kit (Qiagen Sciences, Germantown, MD, USA).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Extracted DNA from fungus gardens was PCR amplified (20 cycles) using full-length universal bacterial (27F [<named-content content-type="gene">5&#8242;-AGA GTT TGA TCC TGG CTC AG-3&#8242;</named-content>] and 1391R [<named-content content-type="gene">5&#8242;- GAC GGG CRG TGW GTR CA-3&#8242;</named-content>]) and archaeal (4aF [<named-content content-type="gene">5&#8242;- TCC GGT TGA TCC TGC CRG-3&#8242;</named-content>] and 1391R [<named-content content-type="gene">5&#8242;- GAC GGG CRG TGW GTR CA-3&#8242;</named-content>]) primers and cloned into the pCR4-TOPO vector (Invitrogen) (See <ext-link ext-link-type="uri" ns0:href="http://my.jgi.doe.gov/general/protocols/SOP_16S18S_rRNA_PCR_Library_Creation.pdf">http://my.jgi.doe.gov/general/protocols/SOP_16S18S_rRNA_PCR_Library_Creation.pdf</ext-link>). This was then sequenced using standard Sanger-based capillary sequencing and assembled as previously described <xref ref-type="bibr" rid="pgen.1001129-Warnecke1">[2]</xref> (<ext-link ext-link-type="uri" ns0:href="http://www.jgi.doe.gov/sequencing/protocols/prots_production.html">http://www.jgi.doe.gov/sequencing/protocols/prots_production.html</ext-link>). These same samples were then pyrosequenced by first PCR amplifying all samples with prokaryote-specific primers corresponding to the V6-V8 region (1492R [<named-content content-type="gene">5&#8242;- TAC GCY TAC CTT GTT ACG ACT T - 3&#8242;</named-content>] and 926F [<named-content content-type="gene">5&#8242;- AAA CTY AAA KGA ATT GAC GG - 3&#8242;</named-content>] fused to 5-base barcodes (reverse primer only) and 454-Titanium adapter sequences) and then sequenced on a Roche 454 FLX GS Titanium pyrosequencer <xref ref-type="bibr" rid="pgen.1001129-Margulies1">[14]</xref>. All 16S rDNA sequences generated in this study are deposited in GenBank with accessions HM545912&#8211;HM556124 and HM556125&#8211;HM559218 for near full-length 16S rDNA sequences and pyrotagged 16S rDNA sequences, respectively.</p><p>Assembled full-length 16S contigs were first compared against the National Center for Biotechnological Information's (NCBI) non-redundant nucleotide (nt) and environmental nucleotide (env_nt) databases (accessed: 05/01/2009) using BLAST <xref ref-type="bibr" rid="pgen.1001129-Altschul1">[37]</xref> to verify that all sequences were bacterial. A small number of eukaryotic 18S sequences belonging to the fungus the ants cultivate, <italic>Leucoagaricus gongylophorus</italic>, which were likely amplified due to the cross-reactivity of the 16S primers, were removed. No sequences identified as archaeal were detected from our library generated using archaeal-specific primers, and only bacterial sequences were amplified. Sequences were prepared for alignment by orienting each sequence in the same direction using the computer program Orientation Checker <xref ref-type="bibr" rid="pgen.1001129-Ashelford1">[38]</xref>, putative chimeras were removed using Bellerophon <xref ref-type="bibr" rid="pgen.1001129-Huber1">[39]</xref>, and each set was de-replicated to remove exact duplicates.</p><p>Finalized sets for each sample were then analyzed using the ARB <xref ref-type="bibr" rid="pgen.1001129-Ludwig1">[40]</xref> software environment as follows. All full-length 16S rDNA sequences were imported and then aligned using the ARB fast-aligner tool <xref ref-type="bibr" rid="pgen.1001129-Ludwig1">[40]</xref> against a user-constructed PT-Server (constructed from the SILVA <xref ref-type="bibr" rid="pgen.1001129-Pruesse1">[41]</xref> 16S SSU rDNA preconfigured ARB reference database with 7,682 columns and 134,095 bacterial sequences; accessed: 01/15/2009). The full alignment was manually curated using the ARBprimary editor (ARB_EDIT4) in preparation for phylogenetic and community analysis. Once an acceptable alignment was obtained we created a PHYLIP <xref ref-type="bibr" rid="pgen.1001129-Felenstein1">[42]</xref> distance matrix in ARB using the filter-by-base-frequency method (column filter; minimal similarity &#8202;=&#8202;50%; gaps ignored if occurred in &gt;50% of the samples; 1,320 valid columns). The PHYLIP distance matrix was exported to the MOTHUR software package v.1.5.0 <xref ref-type="bibr" rid="pgen.1001129-Schloss1">[43]</xref> for community analysis and OTU designation. Briefly, the distance matrix was read into MOTHUR and clustered using the furthest neighbor algorithm. From here, we performed rarefaction, rank-abundance, species abundance, and shared analyses. Representative sequences from each OTU at 97% were re-imported into ARB for phylogenetic analysis (<xref ref-type="supplementary-material" rid="pgen.1001129.s004">Figure S4</xref>, <xref ref-type="supplementary-material" rid="pgen.1001129.s005">S5</xref>, <xref ref-type="supplementary-material" rid="pgen.1001129.s006">S6</xref>, <xref ref-type="supplementary-material" rid="pgen.1001129.s007">S7</xref>, and <xref ref-type="supplementary-material" rid="pgen.1001129.s008">S8</xref>). We used a Maximum Likelihood (RAxML <xref ref-type="bibr" rid="pgen.1001129-Stamatakis1">[44]</xref>) method for all phylogenetic analyses (normal hill-climbing search algorithm) and the above-mentioned method for positional filtering. Closest taxonomic assignment of clones was performed using the Ribosomal Database Project (RDP) <xref ref-type="bibr" rid="pgen.1001129-Cole1">[45]</xref> by comparing sequences against the type strain database (<xref ref-type="supplementary-material" rid="pgen.1001129.s019">Table S5</xref>).</p><p>For pyrotagged short-read 16S rDNA sequences, all sequences were compared against the National Center for Biotechnological Information's (NCBI) non-redundant nucleotide (nt) and environmental nucleotide (env_nt) databases (accessed: 05/01/2009) using BLASTN. Sequences were then classified as either bacterial, archaeal, or eukaryotic, and only those bacterial sequences (20,330) were retained for further analysis.</p><p>These sequences were then processed through Orientation Checker, chimeras removed using the program Mallard <xref ref-type="bibr" rid="pgen.1001129-Ashelford1">[38]</xref>, and subsequently analyzed using MOTHUR in the following fashion. First the entire dataset was de-replicated to eliminate duplicate sequences. The remaining sequences were aligned in MOTHUR against the Greengenes <xref ref-type="bibr" rid="pgen.1001129-DeSantis1">[46]</xref> reference alignment (core_set_aligned.imputed.fasta; 7,682 columns, accessed: 09/11/2009) using the Needleman alignment method with the following parameters: k-tuple size &#8202;=&#8202;9; match &#8202;=&#8202;+1; mismatch penalty &#8202;=&#8202;&#8722;3; gap extension penalty &#8202;=&#8202;&#8722;1; gap opening penalty &#8202;=&#8202;&#8722;5. Sequences were then screened to eliminate those shorter than 400 bp (gaps included). Filtration eliminated 7,062 columns resulting in a total alignment size of 620 bp (gaps included). The remaining dataset was again de-replicated to eliminate duplicate sequences and we constructed a furthest-neighbor distance matrix in MOTHUR using the twice de-replicated, filtered, alignment. All subsequent analyses (rarefaction, rank-abundance, species abundance, and shared analyses) were performed in MOTHUR using this distance matrix.</p><p>A UniFrac <xref ref-type="bibr" rid="pgen.1001129-Lozupone1">[15]</xref> analysis was performed on all full-length 16S rDNA samples generated in this study, including 3 from the top and 3 from the bottom of fungus gardens. MOTHUR was used to generate phylip distance matrices and the computer program Clearcut <xref ref-type="bibr" rid="pgen.1001129-Sheneman1">[47]</xref> was then employed to construct neighbor-joining trees. UniFrac was then used to compare these samples as shown in <xref ref-type="supplementary-material" rid="pgen.1001129.s003">Figure S3</xref>.</p><p>Whole community DNA was used to create a shotgun library which was then sequenced using a single pyrosequencing plate on a Roche 454 FLX GS Titanium sequencer. Assembly of the data was performed using the 454 de novo assembler software with default parameters. Total amounts of data generated and statistical coverage is presented in <xref ref-type="supplementary-material" rid="pgen.1001129.s016">Table S2</xref>. Raw sequence reads generated for this microbiome are deposited in NCBI's Short Read Archive under Study Accession SRP001011.1, and assembled contigs and singletons have been deposited into DDBJ/EMBL/GenBank under the accession ADWX00000000.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The complete set of assembled contigs and singletons representing the fungus garden community metagenome was phylogenetically binned using the following approach. First, the metagenome was compared against NCBI's non-redundant nucleotide (nt) and environmental nucleotide (env_nt) databases (accessed: 05/01/2009) using BLASTN (e-value: 1e-05) and the top hit was retained. The designated phylogenetic classification of the top hit for each contig and singleton was then assessed and binned into one of the following 4 sets: Bacterial, Eukaryotic, Viral, or Unknown. We then performed in-depth phylogenetic binning of the bacterial portion of the fungus garden community metagenome using the current microbial genome collection (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/genomes/lproks.cgi">http://www.ncbi.nlm.nih.gov/genomes/lproks.cgi</ext-link>, accessed: 05/15/2009). We reasoned that using the current microbial genome collection is a likely a more accurate metric for classifying the bacterial set at the genus level because each genome in this collection is correctly annotated and the current iteration of this collection contains both phylogenetic breadth and depth for many represented genera. As a result, we performed two different phylogenetic bins using the current microbial genome collection.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">First, GeneMark <xref ref-type="bibr" rid="pgen.1001129-Besemer1">[48]</xref> was used to predict open reading frames and their corresponding translated proteins of the bacterial portion of the fungus garden community metagenome using a generic bacterial gene model. This predicted proteome was then compared against a local database containing all proteomes in the current microbial genome collection (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/genomes/lproks.cgi">http://www.ncbi.nlm.nih.gov/genomes/lproks.cgi</ext-link>, accessed: 05/15/2009) supplemented with the predicted proteomes of two bacterial strains (<italic>Klebsiella variicola</italic> At-22 and <italic>Pantoea</italic> sp. At-9b, see below) isolated from the fungus gardens of a related leaf-cutting ant species, <italic>Atta cephalotes</italic>. Comparison of the fungus garden proteome against our microbial reference database was done using BLASTP (e-value: 1e-05) and the phylogenetic identity of the top hit was recorded. The total number of proteins was then tabulated to the genus level. Total nucleotide coding content for each predicted protein was then calculated to determine the total amount of nucleotide represented in each bin.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Second, we performed phylogenetic binning on the bacterial portion of the fungus garden metagenome using the entire nucleotide content of the current microbial genome collection (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/genomes/lproks.cgi">http://www.ncbi.nlm.nih.gov/genomes/lproks.cgi</ext-link>, accessed: 05/15/2009), and again supplemented with the nucleotide content from the draft genome sequences of our two bacterial isolates from <italic>Atta cephalotes</italic> leaf-cutter ant fungus gardens. Using complete nucleotide content of the current microbial genome collection is advantageous because it includes both coding and intergenic regions, and provides a more robust measure of phylogenetic identity. We compared the entire bacterial portion of the fungus garden metagenome against this database using BLASTN (e-value: 1e-05) and the phylogenetic identity of the top hit was recorded. The total number of contigs and singletons was then tabulated to the genus level and the corresponding nucleotide amounts were also calculated. Furthermore, we performed this same analysis using all high-quality reads from our fungus garden community metagenome. Finally, we employed the phylogenetic binning program PhymmBL <xref ref-type="bibr" rid="pgen.1001129-Brady1">[22]</xref>, which resulted in similar phylogenetic binning results as our comparison against the sequenced genome collection.</p><p>We performed GC content analysis on the Bacterial, Eukaryotic, and Unclassified phylogenetic bins of the leaf-cutter ant fungus garden community metagenome. For the bacterial set, we divided the sequences according to the NCBI Taxonomic Groups Acidobacteria, Actinobacteria, &#945;-proteobacteria, Bacteroidetes, &#946;-proteobacteria, and &#947;-proteobacteria. We then calculated their GC content, and tabulated the total number of sequences within each group corresponding to each percentage as shown in <xref ref-type="supplementary-material" rid="pgen.1001129.s011">Figure S11</xref>. For Eukaryotic sequences, these were divided into fungal, metazoan, and plant classifications and GC content analysis was also performed as shown in <xref ref-type="supplementary-material" rid="pgen.1001129.s012">Figure S12</xref>. Furthermore, this same analysis was performed for the unclassified portion of the community metagenome and plotted alongside our Eukaryotic GC content analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The predicted proteome from the bacterial portion of the fungus garden community metagenome was annotated using the carbohydrate active enzyme (CAZy) database <xref ref-type="bibr" rid="pgen.1001129-Cantarel1">[23]</xref> as follows. A local database of all proteins corresponding to each CAZy family from the CAZy online database (<ext-link ext-link-type="uri" ns0:href="http://www.cazy.org/">http://www.cazy.org/</ext-link>, accessed: 06/01/2009) was constructed, and this was used to align the predicted proteome of the bacterial portion of the fungus garden community metagenome using BLASTP (e-value of 1e-05). This proteome was then annotated against the protein family (Pfam <xref ref-type="bibr" rid="pgen.1001129-Finn1">[49]</xref>) database (<ext-link ext-link-type="ftp" ns0:href="ftp://ftp.ncbi.nih.gov/pub/mmdb/cdd/">ftp://ftp.ncbi.nih.gov/pub/mmdb/cdd/</ext-link>, accessed: 05/01/2009) using RPSBLAST <xref ref-type="bibr" rid="pgen.1001129-MarchlerBauer1">[50]</xref> (e-value: 1e-05). A CAZy to Pfam correlation list was then compiled based on the secondary annotations provided through the CAZy online database. Finally, only those proteins that had significant BLAST hits to a protein from our local CAZy database and its corresponding Pfam were retained and designated as a carbohydrate-associated enzyme.</p><p>A similar process was performed using the eukaryotic portion of the fungus garden metagenome. However, because of the difficulty in accurately predicting proteins from this subset, due to the lack of good gene models, we compared the contigs and singletons in this subset to our local CAZy and Pfam databases using BLASTX (e-value: 1e-05). Only those hits with significant matches to a protein from our local CAZy database, and its corresponding Pfam were retained and designated as a carbohydrate-associated enzyme in this set.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To determine the similarity of the fungus garden community metagenome with respect to other sequenced metagenomes, we performed a comparative analysis using protein domain and carbohydrate enzyme content as a comparative metric, as previously described <xref ref-type="bibr" rid="pgen.1001129-Tringe1">[51]</xref>. In general, the predicted proteome from the bacterial portion of the fungus garden metagenome was annotated according to clusters of orthologous groups (COGs <xref ref-type="bibr" rid="pgen.1001129-Tatusov1">[52]</xref>) database (<ext-link ext-link-type="ftp" ns0:href="ftp://ftp.ncbi.nih.gov/pub/mmdb/cdd/">ftp://ftp.ncbi.nih.gov/pub/mmdb/cdd/</ext-link>, accessed: 05/01/2009) using RPSBLAST (e-value: 1e-05). The predicted proteomes from the following 13 metagenomes were also annotated in the same manner: bovine rumen <xref ref-type="bibr" rid="pgen.1001129-Brulc1">[3]</xref>, chicken cecum <xref ref-type="bibr" rid="pgen.1001129-Qu1">[53]</xref>, fish gut and slime <xref ref-type="bibr" rid="pgen.1001129-Dinsdale1">[54]</xref>, gutless worm <xref ref-type="bibr" rid="pgen.1001129-Woyke1">[55]</xref>, human gut (Gill) <xref ref-type="bibr" rid="pgen.1001129-Gill1">[6]</xref>, human gut (Kurokawa) <xref ref-type="bibr" rid="pgen.1001129-Kurokawa1">[56]</xref>, Minnesota soil <xref ref-type="bibr" rid="pgen.1001129-Tringe1">[51]</xref>, lean mouse <xref ref-type="bibr" rid="pgen.1001129-Turnbaugh1">[5]</xref>, obese mouse <xref ref-type="bibr" rid="pgen.1001129-Turnbaugh1">[5]</xref>, termite hindgut <xref ref-type="bibr" rid="pgen.1001129-Warnecke1">[2]</xref>, wastewater sludge USA <xref ref-type="bibr" rid="pgen.1001129-GarciaMartin1">[57]</xref>, sastewater sludge OZ <xref ref-type="bibr" rid="pgen.1001129-GarciaMartin1">[57]</xref>, and whale fall <xref ref-type="bibr" rid="pgen.1001129-Tringe1">[51]</xref>. The COG profiles from all of these metagenomes were divided according to their COG gene category designations and plotted as a proportion as shown in <xref ref-type="supplementary-material" rid="pgen.1001129.s013">Figure S13</xref>. Cluster analysis of COG profiles for these metagenomes were performed as follows. A matrix was generated with each row corresponding to a metagenome and each column corresponding to a COG ID. The proportion of each COG with respect to the total number of annotated COGs in that metagenome was calculated and populated in the appropriate cell of the matrix. Spearman's rank correlation was then applied to this matrix to generate a similarity matrix correlating each metagenome to each other based on the similarity of each metagenome's COG profile. A distance matrix was then calculated using the neighbor program from the computer suite Phylip <xref ref-type="bibr" rid="pgen.1001129-Felenstein1">[42]</xref> (using the UPGMA method), and the resulting unrooted tree was visualized using the phylodendron tree drawing program (<ext-link ext-link-type="uri" ns0:href="http://iubio.bio.indiana.edu/treeapp/">http://iubio.bio.indiana.edu/treeapp/</ext-link>, accessed 07/25/2009). This same analysis was also performed using protein domains (Pfam) and no discernable difference in metagenome groupings was detected (data not shown).</p><p>A similar approach was used for clustering these metagenomes according to CAZy content. Each metagenome's predicted proteome was annotated using CAZy and correlated to its Pfam annotation as described above. Because each protein potentially encodes for domain that belong to multiple CAZy families (i.e. a protein may contain both a GH and a CBM), we assigned multiple CAZy annotations to a particular protein. A carbohydrate enzyme matrix was then constructed with each row corresponding to a metagenome sample and each column corresponding to a CAZy family. Each cell in this matrix was then populated with the proportion of each CAZy family with respect to the total number of annotated CAZy families in each respective metagenome. Generation of an unrooted tree using this matrix was then constructed using the same procedure outlined for clustering based on the protein domain content metric.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Pure isolates of <italic>Klebsiella variicola</italic> At-22 and <italic>Pantoea</italic> sp. At-9b were cultured from the fungus gardens of the leaf-cutter ant <italic>Atta cephalotes</italic>, as previously described <xref ref-type="bibr" rid="pgen.1001129-PintoToms1">[10]</xref>. Genomic DNA from these isolates were extracted, as previously described <xref ref-type="bibr" rid="pgen.1001129-PintoToms1">[10]</xref>. Draft genomes of <italic>Klebsiella variicola</italic> At-22 and <italic>Pantoea sp</italic>. At-9b were sequenced at the U.S. Department of Energy Joint Genome Institute (JGI) using a random shotgun approach through a combination of 454 standard and paired-end pyrosequencing (454 Life Sciences, a Roche Company) and 36 bp read Illumina sequencing (Illumina, Inc.). Sequencing using 454 was performed to an average depth of coverage of 30X for both <italic>Klebsiella</italic> and <italic>Pantoea</italic>. All general aspects of library construction and sequencing performed at the JGI can be found at <ext-link ext-link-type="uri" ns0:href="http://www.jgi.doe.gov">http://www.jgi.doe.gov</ext-link>. A draft assembly for <italic>Klebsiella variicola</italic> At-22 was compiled based on 459,192 reads; for <italic>Pantoea sp.</italic> At9b, a draft assembly was constructed using 557,748 reads. The Phred/Phrap/Consed software package (<ext-link ext-link-type="uri" ns0:href="http://www.phrap.com">http://www.phrap.com</ext-link>) was used for sequence assembly and quality assessment of both drafts <xref ref-type="bibr" rid="pgen.1001129-Gordon1">[58]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1001129-Ewing2">[60]</xref>. After the shotgun stage, reads were assembled with parallel Phrap (High Performance Software LLC). Automated annotation of these draft genomes were performed by the Computational Biology and Bioinformatics Group of the Biosciences Division of the U.S. Department of Energy Oak Ridge National Laboratory as described at <ext-link ext-link-type="uri" ns0:href="http://genome.ornl.gov/">http://genome.ornl.gov/</ext-link>. The draft genome sequence and annotation for <italic>Klebsiella variicola</italic> At-22 and <italic>Pantoea</italic> sp. At-9b were deposited in GenBank under accession numbers CP001891 and ACYJ00000000, respectively.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The full set of reads used for the assembly of the fungus garden community metagenome was used to generate a recruitment plot against the draft genomes of <italic>Klebsiella variicola</italic> At-22 and <italic>Pantoea</italic> sp. At-9b, two isolates we cultured from the fungus garden of the leaf-cutter ant <italic>Atta cephalotes</italic>
<xref ref-type="bibr" rid="pgen.1001129-PintoToms1">[10]</xref>, as previously described <xref ref-type="bibr" rid="pgen.1001129-Rusch1">[27]</xref> . Briefly, the contigs from each draft genome were concatenated together in ascending size to produce a &#8220;pseudogenome&#8221;, and the reads from the fungus garden community metagenome were aligned against a database containing both pseudogenomes, and all genomes from the current microbial genome collection (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/genomes/lproks.cgi">http://www.ncbi.nlm.nih.gov/genomes/lproks.cgi</ext-link>, accessed: 05/15/2009) using BLASTN. The top hit for each read was retained, and categorized to each genome. We then mapped reads corresponding to <italic>Klebsiella variicola</italic> At-22 and <italic>Pantoea</italic> sp. At-9b onto each organism's respective psuedogenome and further binned them according to their sequence identities as follows: 95%&#8211;100%, 90%&#8211;95%, 85%&#8211;90%, 80%&#8211;85%, and 70%&#8211;80%. Visualization of the mapped reads onto each respective draft genome was performed using the DNAPlotter software package <xref ref-type="bibr" rid="pgen.1001129-Carver1">[61]</xref>.</p><p>A CAZy analysis was performed on the predicted proteomes of <italic>Klebsiella variicola</italic> At-22 and <italic>Pantoea</italic> sp. At-9b using the same approach as described for CAZy analysis of the leaf-cutter ant fungus garden community metagenome. Furthermore, both GH8 cellulases from each of these genomes were compared against the CAZyme of the fungus garden community metagenome at the nucleotide level using BLASTN (e-value: 1e-05).</p><p>Bioassays were performed on pure cultures of <italic>Klebsiella variicola</italic> At-22 and <italic>Pantoea</italic> sp. At-9b to determine their capacity to degrade cellulose. These include carboxymethyl cellulose (CMC) assays and growth on microcrystalline cellulose. CMC assays were performed as previously described <xref ref-type="bibr" rid="pgen.1001129-Ulrich1">[62]</xref>. Briefly, pure cultures of both bacteria were inoculated onto yeast malt extract agar (YMEA, 4 g yeast extract, 10 g Bacto Peptone, 4 g Dextrose, 15 g agar) and grown at 25&#176;C for 2 days. Single colonies were then spotted onto carboxymethyl cellulose plates (15 g agar, 5 g carboxymethyl cellulose [Calbiochem, La Jolla, CA]). Detection of cellulose degradation on CMC was performed using congo red, and the ability of each isolate's capacity for cellulose degradation was measured based on the zone of clearing present on the plate. Growth on microcrystalline cellulose was performed by inoculating 10 &#181;l of pure culture into 150 &#181;l of microcrystalline cellulose broth (1 L water and 5 g cellulose powder microcrystalline cellulose [MP Biomedicals, Solon, OH]) and growth was measured using a DTX 880 Multimode Detector Plate Reader (Beckman Coulter Inc., Fullerton, CA) at an absorbance of 595 for 2 days. Positive growth on microcrystalline cellulose was correlated to an increase in the measured absorbance over this period.</p>