<p>Fly stocks were maintained at 25&#176;C on standard cornmeal medium. Lines containing <italic>P{EPgy2}DIP1<sup>EY02625</sup></italic> and <italic>P{EPgy2}EY08366</italic> were obtained from the Bloomington Drosophila Stock Center, and a line harboring <italic>PBac{PB}c06482</italic> was obtained from the Exelixis Collection at Harvard Medical School. Genomic coordinates of these P-element insertions were confirmed by PCR with primers specific to the P-elements and flanking genomic sequences followed by sequencing. For transcriptional reporter assays, transgenes were crossed or recombined into mutant backgrounds and scored against crosses to <italic>yw<sup>67c23</sup></italic> as a reference. For ChIP and immunofluorescence, Oregon-R was used as a wild type control. The <italic>y v f mal flam<sup>1</sup></italic>/<italic>FM3</italic> stock was selected for heterozygous females each generation to prevent mobilization and accumulation of TEs. For the <italic>DX1</italic> variegation assay, <italic>DX1/CyO</italic> was crossed to <italic>y w v f mal flam<sup>1</sup>/FM7c; CyO/Sp</italic> flies or <italic>yw<sup>67c23</sup></italic>; <italic>CyO/Sp</italic> as a reference.</p><p>Eye pigmentation of 40 to 60 adult males six days of age was examined, and representative eye photos were taken. To quantify overall levels of eye pigmentation, the heads of 25 male flies of each genotype were dissected, and eye pigmentation was measured as previously described <xref ref-type="bibr" rid="pgen.1000880-PalBhadra1">[36]</xref>. Briefly, heads were homogenized in 0.8 ml of methanol, acidified with 0.1% HCl and centrifuged. The absorbance of the supernatant was measured at 480 nm.</p><p>Adult fly heads or ovaries were dissected and crosslinked with 1.8% formaldehyde for 20 min at 23&#176;C. Chromatin was fragmented to an average size of 300 bp by sonication and incubated with antibodies overnight at 4&#176;C. Quantitative PCR was conducted on Applied Biosystems Real Time PCR system using SYBR Green incorporation (Affymetrix/USB). Amplicon sizes ranged between 150 and 250 bp. Chromatin was immunoprecipitated with the following antibodies: &#945;-HP1 (Covance), &#945;-Su(Hw), &#945;-Piwi (P3G11, a gift from M. Siomi), and normal rabbit or mouse IgG (Santa Cruz Biotechnology). A recombinant N-terminal His-tagged fusion protein of the N-terminal of Su(Hw) (amino acids 1&#8211;218, kind gift of M. Labrador) was purified from <italic>E.coli</italic> on a nickel-agarose column and used to immunize guinea pigs using standard procedures. Similar results were obtained using the C1A9 &#945;-HP1 antibody (Developmental Studies Hybridoma Bank), but lower quantities of DNA were obtained. Fifty to one hundred fly heads and twenty five to fifty ovaries were used per IP. The quantities of target genomic regions precipitated by different antibodies were calculated as percent input based on four-point standard curves constructed from input DNA for each primer set. Standard deviation of each PCR performed in quadruplicate was calculated to determine the error of measurement. Two independent ChIP samples were analyzed, and similar results were obtained. ChIP primers were designed to be unique, detecting only sequences present in the <italic>flam</italic> and <italic>80EF</italic> piRNA loci and verified by <italic>in silico</italic> PCR. All primers (<xref ref-type="supplementary-material" rid="pgen.1000880.s009">Table S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1000880.s010">Table S2</xref>) were checked for both specificity and efficiency by standard agarose gel electrophoresis and real time PCR respectively. Primers to piRNA clusters amplify in the same DNA dilution range as primers specific to <italic>hsp26</italic> and <italic>yellow</italic> single copy genes compared to high copy TE elements (<xref ref-type="supplementary-material" rid="pgen.1000880.s008">Figure S8</xref>). Primers to the <italic>flam</italic> locus were verified to amplify approximately two-fold more DNA from female compared to male genomic DNA. A detailed version of this protocol is available in <xref ref-type="supplementary-material" rid="pgen.1000880.s011">Text S1</xref>.</p><p>The OSC line was maintained and Piwi siRNA knockdown was performed as previously described <xref ref-type="bibr" rid="pgen.1000880-Saito2">[30]</xref>. Briefly, 3&#215;10<sup>6</sup> trypsinized cells were resuspended in 0.1 mL of Solution V of the Cell Line Nucleofector Kit V (Amaxa Biosystems) and mixed with 200 pmol of siRNA duplex. Transfection was conducted according to the manufacturer's protocol using the nucleofector program T-029, and the transfected cells were incubated at 25&#176;C for 48 hrs. Protein knockdowns were verified by Western blotting, and ChIP assays were performed on mock and <italic>piwi</italic> siRNA transfected cells (5&#215;10<sup>6</sup> cells per IP).</p><p>Preparation and immunostaining of salivary gland polytene chromosomes was performed as described previously <xref ref-type="bibr" rid="pgen.1000880-Gerasimova1">[55]</xref>. Primary antibodies directed against HP1 (Covance) and Mod(mdg4)2.2 (generated similarly as in <xref ref-type="bibr" rid="pgen.1000880-Mongelard1">[56]</xref>) and Alexa Fluor 488 labeled anti-guinea pig or Alexa Fluor 594 labeled anti-rabbit secondary antibodies (Invitrogen-Molecular Probes) were used. The chromosomes were viewed using a Leica epifluorescence microscope and photographed using a Hamamatsu digital camera.</p><p>Eye pigmentation of 100 to 200 flies was scored. The scoring of variegation was categorized into five groups: Light, Medium-Light, Medium, Medium-Dark and Dark corresponding to the percentage of pigmented facets. Percentage of flies falling into each category was graphed. Representative eye photos were taken.</p>