<p>C3H/HeJ (Stock # 000659) and C57BL/6J mice (Stock #000664) were obtained from The Jackson Laboratory (Bar Harbor, ME). Mice were housed according to our institutional Animal Review Board standards with a 14 hour light/10 hour dark cycle. These studies conformed to the principles for laboratory animal research outlined by the Animal Welfare Act (NIH/DHHS) and the ARVO Statement for the Use of Animals in Ophthalmic and Vision Research and were approved by the Institutional Animal Care and Use Committee of the University of California, Berkeley and the National Eye Institute. Ehtylnitrosourea (ENU) mutagenesis and breeding of mice were performed as previously described<xref ref-type="bibr" rid="pgen.1000870-Du1">[45]</xref>. Briefly, male C57BL/6 mice were intraperitoneally injected with ENU (90 mg/kg body weight) weekly three times. Three months after the injection, each mouse was bred to wild-type C57BL/6J female mice to produce G1 mice, which were screened for dominant eye phenotypes. Clinical examination of the posterior segment was performed on gently restrained, awake mice after dilation with one drop of 1% tropicamide (Alcon Laboratories, Inc., Fort Worth, TX) using an indirect ophthalmoscope (Keeler, Windsor, Berkshire, UK) with a 90D condensing lens (Volk, Mentor, OH). The optic nerve phenotype was graded in each eye as follows: 0&#8202;=&#8202;normal, +1&#8202;=&#8202;mildly affected (anomalous nerve with peripapillary pigment changes), +2&#8202;=&#8202;strongly affected (findings of &#8220;+1&#8221; and staphylomatous changes), or indeterminate. A mouse was deemed &#8220;affected&#8221; if it had a score of two or more for both eyes combined (i.e., at least a +1 score in each of both eyes or a +2 in one eye.) Mice were euthanized with carbon dioxide according to institutional guidelines. Enucleated adult mouse eyes were fixed in a phosphate-buffered paraformaldehyde-glutaraldehyde mixture according to published protocols <xref ref-type="bibr" rid="pgen.1000870-Smith1">[46]</xref>. Mouse embryos for histopathology were dissected on ice-cold phosphate buffered saline (PBS) and fixed overnight in phosphate-buffered 4% paraformaldehyde at 4&#176;C. Hematoxylin and eosin-stained methacrylate sections via the pupillary-optic nerve axis (eyes) or in appropriate cross section (embryos) were used for histopathology.</p><p>Microsatellite markers known to be informative for the two strains were chosen from the Mouse Mapping Primers v1.0 (Applied Biosystems, Foster City, CA). PCR conditions were as follows: 12 min denaturation at 95&#176;C; 10 cycles of 94&#176;C for 45 sec, 55&#176;C for 1 min, and 72&#176;C for 1 min; 20 cycles of 89&#176;C for 1 min, 55&#176;C for 1 min, and 72&#176;C for 1 min; 10 min final extension at 72&#176;C. The PCR products were pooled based on fluorescent labels and expected allele size. Fragment separation was achieved by capillary electrophoresis on a Genetic Analyzer 3100 using 36 cm capillary array and POP-4 polymer. The ROX400 size standard (Applied Biosystems, Foster City, CA) was run as an internal size-standard. Allele sizing was calculated using the local southern algorithm available in the GENESCAN software program (Applied Biosystems). Allele calling and binning was done using the GENOTYPER software (Applied Biosystems). All genotyping included control DNA from C57BL/6J strain, C3H/HeN strain and C57BL/6J x C3H/HeN.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The structure of wild-type Pax2 paired domain-DNA hetero-complex was modeled using the PAX6 paired domain-DNA complex structure (PDB: 6pax) from the RCSB database <ext-link ext-link-type="uri" ns0:href="http://www.rcsb.org/pdb">http://www.rcsb.org/pdb</ext-link> as the structural template <xref ref-type="bibr" rid="pgen.1000870-Abola1">[47]</xref>. Primary sequences of Pax2/PAX2 and Pax6/PAX6 from mice and human were aligned by the method of Needleman &amp; Wunsch <xref ref-type="bibr" rid="pgen.1000870-Needleman1">[48]</xref>, and incorporated in the program Look, version 3.5.2 for 3-dimensional structure prediction <xref ref-type="bibr" rid="pgen.1000870-Lee1">[49]</xref>,<xref ref-type="bibr" rid="pgen.1000870-Lee2">[50]</xref>. The wild-type Pax2 paired domain-DNA hetero-complex and mutation dup74ET were built by the automatic segment matching method in the program Look followed by 500 cycles of energy minimization <xref ref-type="bibr" rid="pgen.1000870-Levitt1">[51]</xref>. The same program generated the conformation of the proteins with the T74A and G76S mutations; and refined them by self-consistent ensemble optimization (500 cycles) <xref ref-type="bibr" rid="pgen.1000870-Lee2">[50]</xref>. The geometry of the predicted structures was tested with the program Procheck <xref ref-type="bibr" rid="pgen.1000870-Laskowski1">[52]</xref>.</p><p>Mouse embryos were dissected in PBS and fixed overnight in 4% paraformaldehyde in PBS followed by cryopreservation in 30% sucrose overnight at 4&#176;C. Whole embryos were embedded and frozen on dry ice in Neg &#8211;50 (Richard-Allan Scientific, Kalamazoo, MI). 10 &#181;M frozen sections were cut through mouse eyes and slides were stored at &#8722;80&#176;C. Tissue sections were then thawed and washed three times in PBS, and then treated with 1X target retrieval solution (pH 6) (Dako, Carpenteria, CA). After washing three times in PBS, the section was incubated in blocking reagent (10% normal donkey serum, 0.02% Triton X-100 in PBS) for 1 hr. at room temperature. Slides were incubated overnight in anti-murine Pax2 (1:1000, Covance, Berkeley, CA) at 4&#176;C. Following removal of primary antibody slides were washed four times in PBS for ten minutes each and the Pax2 antibody was detected using Donkey anti-rabbit Alexa Fluor 594 secondary antibody (1:400, Molecular Probes Inc., Eugene, OR) for one hour at room temperature. Slides were then washed again in PBS and cover-slipped with Vectashield with DAPI (Vector Laboratories, Burlingame, CA). At least 100 cells were counted for each transfection.</p><p>For cell culture, COS-7 cells (ATCC, Manassas, VA) were cultured on slides and fixed in 4% paraformaldehyde at room temperature for 10 minutes, rinsed twice with wash buffer (0.1% Tween 20, 0.5% normal goat serum in PBS) then cryoprotected by incubation in 15% and 30% sucrose for 45 minutes each and stored at &#8722;80&#176;C. When ready for use, slides were thawed, washed three times in PBS and incubated in blocking reagent (10% normal goat serum, 0.1% Tween 20 in PBS) for 30 min. at room temperature. Slides were incubated overnight in anti-murine Pax2 (1:200, Zymed, Carlsbad, CA) at 4&#176;C. Following removal of primary antibody slides were washed four times in PBS for five minutes each and the Pax2 antibody was detected using goat anti-rabbit Alexa Fluor 488 secondary antibody (1:1000, Molecular Probes Inc., Eugene, OR) for one hour at room temperature. Slides were then washed again in PBS and cover-slipped with ProLong Gold with DAPI (Molecular Probes).</p><p>Fluorescence and brightfield images were taken with a Zeiss AxioVert 200 microscope with a digital camera connected to a PC running AxioVision 4.6.3 (Carl Zeiss MicroImaging, Thornwood, NY). When making qualitative comparisons of the intensity of immunofluorescence, care was taken to standardize exposure times.</p><p>Enucleated mice eyes were fixed in 4% paraformaldehyde in PBS at room temperature for 30 minutes. After washing in PBS, retinas were dissected and isolated. Four radial incisions were then made in preparation for flat mount. Subsequently, retinas were permeabilized and blocked in a solution of 0.5% Triton X100 and 1% bovine serum albumin in PBS at room temperature for 1 hour. After PBS wash, retinas were incubated in TRITC-conjugated lectin (1:100 dilution in PBS) overnight at 4&#176;C. After rinsing, retinas were flat mounted with slow-fade medium (Pro-Long Gold, Invitrogen, Carlsbad, CA) and visualized under microscopy with a TRITC filter.</p><p>PCR-based site directed mutagenesis of the CMV-<italic>Pax2b</italic> expression construct <xref ref-type="bibr" rid="pgen.1000870-Lechner1">[32]</xref>,<xref ref-type="bibr" rid="pgen.1000870-Fickenscher1">[33]</xref> was performed according to standard protocol to introduce one of the following mutations: c.A220G, c.G223A, or c.222insGAGACC. These mutations in the mouse sequence correspond to c.A223G (p.T75A), c.G226A (p.G76S), and c.220insGAGACC (p.dup74ET) in the human sequence, respectively. Mutations were confirmed by direct sequencing. NIH/3T3 (mouse embryonic fibroblast) or COS-7 (African green monkey kidney) cells (ATCC, Manassas, VA) were plated at 0.25&#215;10<sup>6</sup> cells per well in a 6-well plate in DMEM media with 10% fetal bovine serum. The following day, cells were transiently transfected with 0.5 micrograms of PRS4-luciferase reporter construct <xref ref-type="bibr" rid="pgen.1000870-Lechner1">[32]</xref>,<xref ref-type="bibr" rid="pgen.1000870-Fickenscher1">[33]</xref>, 50 nanograms of <italic>Renilla</italic> luciferase construct (pRL-CMV, Promega, Inc., Madison, WI) and 2 micrograms of CMV-Pax2b expression construct (wild-type or mutant) <xref ref-type="bibr" rid="pgen.1000870-Cai2">[30]</xref> using 10 &#181;l/well lipofectamine 2000 (Invitrogen, Carlsbad, CA) according to the manufacturer's protocol. Control samples were transfected with an equimolar amount of the expression vector backbone. After 48 hours, cells were harvested and luciferase activity was measured using microplate reader (Optima, BMG labtech, Durham, NC). All experiments were repeated at least three times with at least three replicates per sample.</p><p>Transfected cells were harvested for protein in 1x Passive Lysis Buffer. Total protein was determined using Micro-Lowry method (Sigma-Aldrich, St. Louis, MO). Equal amounts of protein were separated (4&#8211;12% NuPAGE Bis-Tris) polyacrylamide gels and transferred to 0.2 &#181;m, PVDF membranes (Invitrogen, Carlsbad, CA). Blots were hybridized with 1:1000 dilution Rabbit anti-Pax2 antibody (Zymed, San Francisco, CA) and 1:4000 dilution of goat anti-rabbit-HRP secondary antibody (Thermo Fischer Scientific, Pierce Protein Research Products, Rockford, IL) and then developed with SuperSignal West Pico chemiluminescent substrate for detection of HRP (Thermo Fischer Scientific, PierceProtein Research Products, Rockford, IL). Quantitation was performed on a ChemiDoc EQ (Bio-Rad Laboratories, Hercules, CA) using the manufacturer's software (Quantity One, v.4.5.2, Build 070).</p><p>NIH/3T3 cells were transiently transfected with wild type or mutant Pax2 plasmids using FuGene 6 HD reagent (Roche-Applied Science, Indianapolis, IN). 48 hrs. post transfection, RNA was isolated using an RNeasy mini kit from Qiagen. 1 &#181;g of RNA was treated with DNase I to remove any DNA contamination from the samples. cDNA was prepared using High Capacity cDNA Reverse Transcription kit (Applied Biosystems, Foster City, CA). Real time PCR was performed with SYBR green master mix (Applied Biosystems) on Bio-Rad iCycler. The following primers were used for Pax2 (Forward- <named-content content-type="gene">TATGCACTGCAAAGCAGACC</named-content> and Reverse- <named-content content-type="gene">GGGGCAGTCACTCCTGTC</named-content>) and Gapdh (Forward- <named-content content-type="gene">GCATTGTGGAAGGGCTCATGACC</named-content> and Reverse-CGGCATCGAAGGTGGAAGAGTGG). The parameters for PCR amplification were 95&#176;C for 10 min followed by 40 cycles of 95&#176;C for 30 s, 55&#176;C for 30 s and 72&#176;C for30 s. Relative expression of different mRNA samples for <italic>Pax2</italic> and <italic>Gapdh</italic> was calculated using the comparative threshold cycle method.</p><p>Pax2 wild type and mutant proteins stability were characterized by transiently transfecting NIH/3T3 cells with expression vectors expressing wild type and mutant Pax2 proteins using a modified method described by Jiang et al <xref ref-type="bibr" rid="pgen.1000870-Jiang1">[53]</xref>. In brief, 24 hrs post-transfection with equal amounts of wild-type or mutant <italic>Pax2</italic> expression vector, cells were treated with cycloheximide (CHX) (100 &#181;g/ml) (Sigma-Aldrich, St. Louis, MO). Cells were washed with PBS and lysed with RIPA buffer at 0, 3 and 6 hrs after CHX treatment. The protein concentrations in the lysates were determined by BCA method (ThermoFisher Scientific-Pierce, Rockford, IL). Equal amounts of proteins were resolved in NuPAGE Novex Bis-Tris gel (4&#8211;12%), transferred to PVDF membrane and probed with Rabbit polyclonal anti-Pax2 (1:250 dilution) and anti-Gapdh (1:500 dilution) primary antibodies. HRP conjugated anti-rabbit IgG was used as secondary antibody (1:5000 dilution). SuperSignal West Pico Chemiluminescent Substrate was used to detect HRP on the blots. The blot was imaged using Autochemie System (UVP, Upland, CA) and quantitative analysis of the bands was performed using Labworks software.</p><p>COS-7 cells were transiently transfected with CMV-Pax2a expression construct (wild type or mutant) using FuGENE 6 HD reagent (Roche-Applied Science, Indianapolis, IN). The cells were harvested after 48 h and the nuclear extracts were prepared by using NE-PER nuclear and cytoplasmic extraction kit reagents (Thermo Scientific-Pierce, Rockford, IL). Gel purified sense and antisense oligonucleotides representing wild-type or mutant Pax2 DNA binding sites were labeled at the 3&#8242; end of DNA strand with Biotin -11-dUTP (Biotin 3&#8242; end DNA labeling kit, Thermo Scientific-Pierce). The forward wild-type primer was <named-content content-type="gene">5&#8242;TGGAATTCAGGAAAAATT<underline>GTCACG</underline>CATGAGTGGTTAGCTCGAGTA-3&#8242;</named-content> and the forward mutant primer was <named-content content-type="gene">5&#8242;TGGAATTCAGGAAAAATT<underline>TGATAC</underline>CATGAGTGGTTAGCTCGAGTA-3&#8242;</named-content>, where the underlined sequence represents the portion of oligonucleotide that was mutated. Gel mobility shift assays were performed with Biotin -11-dUTP labeled target DNA in 10 mM Tris-HCl pH 7.5, 100 mM KCl, 0.5 mM DTT, 05% NP-40, 2.5% glycerol and 50 ng/&#181;l poly (dI:dC) and incubated with the Wt or Mut nuclear extracts in a total volume of 20 &#181;l. After incubation for 20 minutes, protein-DNA complexes were separated on a 6% polyacrylamide gel in 0.5 x TBE buffer at 100 V/cm, transferred onto Nytran membrane and UV cross-linked. Biotin label was detected by chemiluminescent detection module (Thermo Scientific-Pierce) that uses luminol substrate for HRP-conjugated streptavidin. For competition experiments, the nuclear extracts were pre-incubated with 100 molar excess of unlabeled or mutant DNA for 10 min. before adding the biotin labeled probe.</p>