<p>Peripheral blood samples were collected from 48 children with FRDA participating in a randomized, placebo-controlled clinical trial for idebenone [registered with ClinicalTrials.gov, number NCT00229632, and approved by the NIH Institutional Review Board at the National Institute of Neurological Disorders and Stroke (NINDS), protocol # 05-N-0245]. Samples from 14 anonymous FRDA adults were collected in the reference center clinic dedicated to cerebellar ataxias and aspartic paraplegias at the University Salp&#234;tri&#232;re Hospital in Paris; samples were exempted by the NIH Institutional Review Board at the National Institute of Environmental Health Sciences (NIEHS), exempt # 3984. All controls used for transcriptional profiling were young healthy adults from an acetaminophen study <xref ref-type="bibr" rid="pgen.1000812-Harrill1">[28]</xref>, approved by the Institutional Review Board at the University of North Carolina, Chapel Hill, protocol # GCRC-2265. Controls used for the DNA damage assay were obtained from an NIH blood bank. Blood and/or apheresis samples were obtained from healthy volunteer donors who gave signed consent to participate in an IRB-approved protocol for use of their blood in laboratory research studies; these samples were approved by the Institutional Review Board at the National Cancer Institute (NCI), protocol # 99-CC-0168.</p><p>Peripheral blood samples were collected from 48 children with FRDA participating in a randomized, placebo-controlled clinical trial. All whole blood samples in this study were collected before administration of idebenone. A detailed description of all subjects and clinical endpoints was recently published <xref ref-type="bibr" rid="pgen.1000812-DiProspero1">[22]</xref>. Due to other endpoints, this study only allotted one 8.5 ml sample of blood from each patient for RNA isolation. RNA was isolated by one person, utilizing the PAXgene blood RNA isolation kit (PreAnalytiX/QIAGEN, Hilden, Germany) according to the manufacturer's protocol, including the optional on-column DNase digestion, except that the centrifugation time after proteinase K digestion was increased from 3 to 20 minutes in order to obtain a tighter debris pellet. RNA quality was assessed with an Agilent Bioanalyzer (Palo Alto, CA) to ensure that samples with intact 18S and 28S ribosomal RNA peaks were used for microarray analysis. Of the 48 patients, twenty samples were lost during the isolation procedures, leaving 28 high-quality RNA samples remaining. The demographics for these subjects are detailed in <xref ref-type="supplementary-material" rid="pgen.1000812.s005">Table S1</xref>. RNA was also isolated, using the same methods already described, from 14 anonymous FRDA adults (<xref ref-type="supplementary-material" rid="pgen.1000812.s006">Table S2</xref>). All controls used were young healthy adults (see demographics data in <xref ref-type="supplementary-material" rid="pgen.1000812.s005">Table S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1000812.s006">Table S2</xref>) from an acetaminophen study <xref ref-type="bibr" rid="pgen.1000812-Harrill1">[28]</xref>. Two independent sets of control populations were used separately to compare to the children and the adult validation cohort.</p><p>Gene expression profiling was conducted using Agilent Human 1A(V2) Oligo arrays with &#8764;20,000 genes represented (Agilent Technologies, Palo Alto, CA). Each sample was hybridized against a human universal RNA control (Stratagene, La Jolla, CA). 500 ng of total RNA was amplified and labeled using the Agilent Low RNA Input Fluorescent Linear Amplification Kit, according to manufacturer's protocol. For each two color comparison, 750 ng of each <italic>Cy3</italic>- (universal control) and <italic>Cy5</italic>-labeled (sample) cRNA were mixed and fragmented using the Agilent <italic>In Situ</italic> Hybridization Kit protocol. Hybridizations were performed for 17 hours in a rotating hybridization oven according to the Agilent 60-mer oligo microarray processing protocol prior to washing and scanning with an Agilent Scanner (Agilent Technologies, Wilmington, DE). The data were obtained with the Agilent Feature Extraction software (v9.1), using defaults for all parameters. The Feature Extraction Software performs error modeling before data are loaded into a database system. Images and GEML files, including error and <italic>p</italic>-values, were exported from the Agilent Feature Extraction software and deposited into Rosetta Resolver (version 5.0, build 5.0.0.2.48) (Rosetta Biosoftware, Kirkland, WA). All gene expression data have been deposited in the public Gene Expression Omnibus (GEO) database and are available under the series ID GSE11204.</p><p>Supervised analysis to find genes associated with case versus control or low frataxin expression versus high expression was performed using Significance Analysis of Microarrays (SAM) after pooling the raw data <xref ref-type="bibr" rid="pgen.1000812-Tusher1">[29]</xref>. The two-class unpaired SAM algorithm was used and the false discovery rate was set to less than or equal to 1% for all analyses. Gene Set Analysis (GSA) <xref ref-type="bibr" rid="pgen.1000812-Efron1">[30]</xref> was also performed for these comparisons to test the association of gene sets instead of individual genes. The database of gene sets used for GSA was obtained from the Molecular Signatures Database (MSigDb) <xref ref-type="bibr" rid="pgen.1000812-Efron1">[30]</xref>. Gene sets demonstrating a <italic>p</italic>-value less than 0.01 were considered significant.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Biologically relevant themes in the lists of significant genes from SAM were analyzed with gene ontology tools, GoMiner and DAVID (Database for Annotation, Visualization and Integrated Discovery) <xref ref-type="bibr" rid="pgen.1000812-Zeeberg1">[67]</xref>,<xref ref-type="bibr" rid="pgen.1000812-Dennis1">[68]</xref>. GO terms with <italic>p</italic>&#8804;0.05 for upregulated, downregulated, and/or combined direction of change were selected for analysis. Both tools group genes according to the GO categories of biological process, cellular component, and molecular function, based on ranking by a hypergeometric test <italic>p</italic>-value. These data were also uploaded into Ingenuity Pathway Analysis (IPA) software v 5.5.1 (Ingenuity Systems, Redwood City, CA), a program that categorizes genes into biological functions but also enables visualization of biologically relevant networks and canonical pathways (&#8220;canonical&#8221; implies &#8220;established&#8221;). Go to <ext-link ext-link-type="uri" ns0:href="http://www.Ingenuity.com">www.Ingenuity.com</ext-link> for specifics regarding the application. Unsupervised clustering and heat-map generation were carried out with Cluster and Treeview programs <xref ref-type="bibr" rid="pgen.1000812-Eisen1">[69]</xref>.</p><p>The levels of DNA damage were analyzed by Mann-Whitney U test or Spearman's Rank Correlation because the data is not normally distributed or homoskedastic.</p><p>In order to test association between gene expression and age, we used SAM. For the two DNA damage variables we used a Student's <italic>t</italic>-test to detect association with age. All these analyses were performed in the controls, and age was dichotomized by comparison to the median age of the controls.</p><p>A univariate linear model was constructed to test the association of each gene in the genotoxic gene set list (genes found in 25% of gene lists from significant gene sets) to patient frataxin levels (determined by Real-time PCR). Correlations of frataxin mRNA levels to all available clinical data for each child were also performed by univariate linear modeling.</p><p>In extracting gene expression patterns, EPIG <xref ref-type="bibr" rid="pgen.1000812-Chou1">[42]</xref> uses a filtering process where all profiles initially are considered as pattern candidates. Briefly, using all pair-wise correlations, any candidate profile, whose local cluster size is less than a predefined size M<italic><sub>t</sub></italic> or its correlation with another profile is higher (&gt;R<italic><sub>t</sub></italic>) but has a lower local cluster size M, is removed from pattern construction consideration. Among the remaining profiles, EPIG then creates representative profiles for the corresponding local clusters and removes those profiles with a signal-to-noise ratio or magnitude less than given thresholds. After this filtering processing, the remaining profiles consist of the extracted patterns, which are used to be the representatives to each of the local clusters. Subsequently, EPIG categorizes each significant gene to a pattern, for which it has the highest correlation with the gene profile. A gene not assigned to any extracted pattern is considered an &#8220;orphan&#8221; if its highest correlation <italic>r</italic>-value is lower than the given threshold R.</p><p>The frataxin probe on the Agilent chip was observed to lack sensitivity for both the individual lymphoblastoid cell lines from affected people (data not shown) and the whole blood from FRDA patients. We, therefore, decided to obtain relative gene expression levels of frataxin by TaqMan Real-time PCR <xref ref-type="bibr" rid="pgen.1000812-Heid1">[70]</xref>. The sequence information of the probe used for TaqMan is the proprietary information of PE Applied Biosystems, but we know it is 75 nucleotides that span across the exon 1 and exon 2 boundary. On the other hand, the probe on the microarray chip is mostly from the 3&#8242; UTR of frataxin.</p><p>One microgram of total RNA from 27 patients and 10 controls was used for reverse transcription with TaqMan Reverse Transcription Reagents (PE Applied Biosystems). To determine relative frataxin mRNA levels, Real-time PCR was carried out using the ABI Prism 7900HT sequence detection system. Primer and probe sets for frataxin and glucuronidase-beta were purchased as pre-developed assays from PE Applied Biosystems. Relative quantification was obtained using the threshold cycle method after verification of primer performance, following the manufacturer's guidelines. The levels of frataxin obtained are relative to the average &#916;<italic>C</italic>
<sub>T</sub> from 10 controls.</p><p>Total DNA from whole blood was successfully isolated from 47 children enrolled in the study and 15 adult controls obtained from an NIH blood bank in Bethesda, MD, (<xref ref-type="supplementary-material" rid="pgen.1000812.s010">Table S6</xref>) using the PAXgene blood DNA isolation kit (PreAnalytiX/QIAGEN, Hilden, Germany) according to the instructions of the manufacturer. Briefly, one 8.5 ml sample of whole blood, per child, was collected in PAXgene blood DNA tubes for DNA isolation; each blood sample was transferred to a processing tube containing a lysing solution. Lysed red and white blood cells were centrifuged, and the resulting pellet of nuclei and mitochondria was washed and resuspended. After digestion with protease, DNA was precipitated by addition of isopropanol and dissolved in water. All DNA samples were prepared by one person. DNA lesion frequencies were calculated as described previously <xref ref-type="bibr" rid="pgen.1000812-Santos1">[33]</xref>. Briefly, the amplification of patient samples (<italic>A<sub>patient</sub></italic>) was compared to the amplification of non-damaged controls (<italic>A<sub>control</sub></italic>) resulting in a relative amplification ratio. Assuming a random distribution of lesions and using the Poisson equation [<italic>f</italic>(<italic>x</italic>)&#8202;=&#8202;<italic>.e</italic> &#732;<sup>&#8722;&#955;</sup> &#955;<sup>x</sup>/<italic>x</italic>!, where &#955; is the average lesion frequency for the nondamaged template (i.e., the zero class; <italic>x</italic>&#8202;=&#8202;0)], the average lesion per DNA strand was determined by the following equation: .&#955;&#8202;=&#8202;.&#8722;ln <italic>A<sub>Patient</sub></italic>/<italic>A<sub>control</sub></italic>. Amplification of the large mitochondrial target was normalized to mitochondrial copy number by examination of a short mitochondrial target, which due to its short size, should be free of damage.</p>