<p>A description of each study is provided in <xref ref-type="table" rid="pgen-1001051-t001">Table 1</xref> and <xref ref-type="supplementary-material" rid="pgen.1001051.s009">Text S1</xref>. Lung cancer cases and controls for the GWAS were drawn from the Genetic Epidemiological Study of Lung Adenocarcinoma (GELAC) in Taiwan. A total of 584 never-smoking incident cases and 585 never-smoking controls were included in the GWAS. The replication studies were drawn from seven studies, including additional subjects from the GELAC study <xref ref-type="bibr" rid="pgen.1001051-Jou1">[13]</xref>, the Chinese Academy of Medical Sciences Cancer Hospital study (CAMSCH) <xref ref-type="bibr" rid="pgen.1001051-Wu1">[16]</xref>, the Wuhan lung cancer study (WHLCS) <xref ref-type="bibr" rid="pgen.1001051-Bai1">[17]</xref>, the Seoul National University study (SNU) <xref ref-type="bibr" rid="pgen.1001051-Kim1">[18]</xref>, the Korea University Medical Center study (KUMC) <xref ref-type="bibr" rid="pgen.1001051-Jung1">[19]</xref>, the Kyungpook National University Hospital study (KNUH) <xref ref-type="bibr" rid="pgen.1001051-Park1">[20]</xref>, and the Shanghai Women's Health Cohort Study (SWHS) <xref ref-type="bibr" rid="pgen.1001051-Zhang1">[21]</xref>, <xref ref-type="bibr" rid="pgen.1001051-Zheng1">[22]</xref>. In addition, data were pooled with previously published findings from the Nanjing lung cancer study (NJLCS) <xref ref-type="bibr" rid="pgen.1001051-Jin1">[23]</xref> and the Genes and Environment in Lung Cancer, Singapore study (GEL-S) <xref ref-type="bibr" rid="pgen.1001051-Truong1">[24]</xref> (<xref ref-type="table" rid="pgen-1001051-t001">Table 1</xref>). All studies are case-control studies with the exception of the SWHS, which is a prospective cohort study. The range of ages is similar in cases and controls across all studies (<xref ref-type="table" rid="pgen-1001051-t001">Table 1</xref>).</p><p>All study subjects provided informed consent and each study was approved by its respective institution's IRB.</p><p>GWAS genotyping of the GELAC samples was performed in two separate phases. In the pilot phase, 54 cases and 54 controls were genotyped by GeneTech Biotech Co., (Taiwan), using the Illumina HumanCNV370-Duo BeadChip. The cases were never-smoking females diagnosed with lung adenocarcinoma at age &#8804;51 who had questionnaire data and DNA that passed quality control criteria for scanning. The controls were never-smoking females matched by age (&#177;2 years) to cases.</p><p>Cluster definitions were determined using Illumina BeadStudio Genotyping Module v.3.3.4. Genotype calls were based on a quality score (Gene call value) of 0.25 or higher. Four blind duplicate pairs were included, and the concordance of SNP genotype calls between each pair is greater than 99.997%. Quality control metrics for data from the first phase are similar to those for data from the second phase, detailed below.</p><p>In the second phase of the GWAS, 550 cases and 549 controls were genotyped with the Illumina HumanHap610 Quad BeadChip on contract at deCODE Genetics, Iceland. The cases were the first never-smoking female lung adenocarcinoma subjects to be enrolled in the study with questionnaire data and DNA that passed quality control for scanning. Cluster definitions were determined using the Illumina BeadStudio Genotyping Module. The median genotype call rate for samples was 99.78%. 95% overall displayed call rates larger than 99.49%; the median call rate for variants is 99.91%, with 95% of variants with call rates above 99.55%. 21 blind duplicate pairs displayed an average concordance greater than 99.99%.</p><p>After quality control metrics were applied, 457,504 SNPs were used for the association analysis. SNPs (n&#8202;=&#8202;1,705) were excluded if the call rate was below 90%, (i.e., a missing rate larger than 0.1); SNPs with a minor allele frequency below 0.05 (n&#8202;=&#8202;131,558); SNPs with missing rate between 0.02 and 0.1 and non-random genotype failure with p&lt;0.02 (n&#8202;=&#8202;1,046); and, significant deviation from fitness for Hardy-Weinberg equilibrium (p&lt;0.0001 in controls) (n&#8202;=&#8202;718). 1064 unique samples from phase 2 were used in the association analysis, after two exclusion steps. The first set of exclusions was based on the quality control metrics described above and relatedness among individuals: call rates less than 90% (n&#8202;=&#8202;3); sex discrepancies based on the X chromosome heterozygosity (n&#8202;=&#8202;7); contaminated samples with high heterozygosity scores (n&#8202;=&#8202;4), first or second degree relatives identified using genome-wide pairwise identical by descent (IBD) estimates (n&#8202;=&#8202;9).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We further excluded 12 individuals from phase 2, based on population substructure analysis. In fact, to detect differences in population substructure, pairwise population concordance (PPC) test in PLINK (<ext-link ext-link-type="uri" ns0:href="http://pngu.mgh.harvard.edu/purcell/plink/">http://pngu.mgh.harvard.edu/purcell/plink/</ext-link>) <xref ref-type="bibr" rid="pgen.1001051-Purcell1">[36]</xref> were performed with a threshold of 10<sup>&#8722;20</sup> on two data sets using all autosomal SNPs that had passed the quality control metrics described above. The first data set consists of the 1184 unrelated individuals with high quality genotype data (108 from phase 1 and 1076 from phase 2). The PPC test identified 15 outliers who were distinct from the remaining 1169 (105 from phase 1 and 1064 from phase 2). The eight self-described aborigines (2 in Phase 1 and 6 in Phase 2) were among the outliers. Based on the PPC analysis, the final genome-wide association analysis was conducted using 1169 samples.</p><p>To further assess the population homogeneity in our study sample, we conducted additional analyses in our 1184 individuals with HapMap3 release 2 data <xref ref-type="bibr" rid="pgen.1001051-The2">[37]</xref>. The results indicate that for the 1184 unrelated individuals with high quality genotype data, 15 outliers were detected, thus yielding 1169 individuals with homogeneous genetic structure available for follow-up analyses. We seeded the study population with genotype data from hapmap 3 as well as hapmap 2; this included 85 CHD (Chinese in Metropolitan Denver, Colorado), in addition to our 1184 individuals and the hapmap 2 (84 CHB (Han Chinese in Beijing, China) and 86 JPT (Japanese in Tokyo, Japan)). A second analysis included our 1184 study individuals and a larger sample of HapMap3 release 2, namely the CHB, CHD, GIH (Gujarati Indians in Houston, Texas), JPT, LWK (Luhya in Webuye, Kenya), MKK (Maasai in Kinyawa, Kenya), and TSI (Toscani in Italia): the results confirmed that 15 outliers were detected whereas the 1164 represented a homogeneous population.</p><p>Although the above PPC tests seem to suggest little population substructure in our 1169 samples, we still used EIGENSTRAT <xref ref-type="bibr" rid="pgen.1001051-Price1">[38]</xref> to conduct GWAS analysis to correct possible population stratification. We found that for the SNP rs2736100, the P-value is 1.239&#215;10<sup>&#8722;11</sup> based on the Armitage trend Chi-square statistic with no stratification correction and the P-value is 2.764&#215;10<sup>&#8722;11</sup> based on EIGENSTRAT using 10 principal components (the default value) for stratification correction. There was a negligible difference in p-values with and without this correction.</p><p>The genotyping cluster plot generated by the Illumina platform for rs2736100 is presented in <xref ref-type="supplementary-material" rid="pgen.1001051.s002">Figure S2</xref>. The adjusted intensities for each allele are plotted, where each color represents a different genotype in the cluster plots. As shown in the figure, clusters of different genotypes are well separated from each other, indicating a high confidence in genotype calling in our study. The genotype call at this locus was confirmed with TaqMan genotyping (concordance of 99.7%).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">DNA was extracted from blood samples and genotyped at the National Cancer Institute Core Genotyping Facility (CGF) (<ext-link ext-link-type="uri" ns0:href="Http://cgf.nci.nih.gov">Http://cgf.nci.nih.gov</ext-link>) for four studies, SNU, KUMC, KNUH, and SWHS. TaqMan genotyping for the GELAC study (including all previously scanned cases and controls plus remaining never-smoking female cases and their matched controls) and the GEL-S studies was conducted in Taiwan and Singapore, respectively. Genotyping for the CAMSCH, WHLCS, and NJLCS studies were conducted at the Cancer Institute and Hospital, Chinese Academy of Medical Science, using TaqMan assays designed and optimized by the CGF (<ext-link ext-link-type="uri" ns0:href="http://snp500cancer.nci.nih.gov">http://snp500cancer.nci.nih.gov</ext-link>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We selected 54 SNPs optimized for Eastern Asian populations to cover the three chromosomal regions previously reported to show association for lung cancer (i.e., 15 SNPs in 5p15, 15 SNPs in 6p, and 24 SNPs in 15q25) (<xref ref-type="supplementary-material" rid="pgen.1001051.s005">Table S3</xref>). The coordinates for selecting the tag SNPs were based on an r<sup>2</sup>&lt;0.8 using the CHB samples of HapMap phase 2. The boundaries for the tag SNP selection were as follows: 5p15.33 from 1310620 to 1412939, 6p21.33 from 28782776 to 29018856 and 15q25 from 76593077 to 76702301 (Build 37). We computed genomic coverage using the GLU software package (<ext-link ext-link-type="uri" ns0:href="http://code.google.com/p/glu-genetics/">http://code.google.com/p/glu-genetics/</ext-link>) for common SNPs (MAF&#8805;0.05) based on the most recent build (Build 37) of the HapMap CHB <xref ref-type="bibr" rid="pgen.1001051-The2">[37]</xref> genotype data.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All TaqMan assays (Applied Biosystems Inc., Foster City, CA) for this study were optimized on the ABI 7900HT detection system with high concordance with sequence analysis of 102 individuals as listed on the SNP500Cancer website (<ext-link ext-link-type="uri" ns0:href="http://snp500cancer.nci.nih.gov">http://snp500cancer.nci.nih.gov</ext-link>). All of the genotype frequencies were consistent with Hardy-Weinberg equilibrium except three SNPs (rs402710, rs9368570, and rs9257280) using a chi-square test (<italic>P</italic>&lt;0.0001, <xref ref-type="supplementary-material" rid="pgen.1001051.s005">Table S3</xref>). All reported genotyped results are based on completion rates of greater than 94% across all studies.</p><p>The program PLINK <xref ref-type="bibr" rid="pgen.1001051-Purcell1">[36]</xref> was used to conduct primary statistical tests for association in the discovery phase. Association analyses between individual SNP and the lung cancer risk were carried out using computer packages in PLINK. Q-Q plots analyzed by the trend test are shown in <xref ref-type="fig" rid="pgen-1001051-g001">Figure 1</xref>. We note that for Phase 1, we imputed the genotypes at the SNPs contained in HumanHap 610 Quad BeadChip but not in HumanCNV370-Duo BeadChip by using IMPUTE developed by Marchini et al. <xref ref-type="bibr" rid="pgen.1001051-Marchini1">[39]</xref> and haplotypes of CHB in HapMap as the reference.</p><p>Unconditional logistic regression was used to estimate the ORs and 95% CIs, adjusting for age and study center. All p values are two-sided. The most prevalent homozygous genotype was used as the reference group. Tests for trend were conducted by assigning the ordinal values 1, 2, and 3 to the most prevalent genotypes in rank order of wild type, heterozygous, and variant homozygous genotypes, respectively.</p>