<p>Mice carrying the <italic>Paxip1</italic> null (<italic>Paxip1</italic>
<sup>&#8722;</sup>) and floxed (<italic>Paxip1</italic>
<sup>fl</sup>) alleles were previously described and genotyped as indicated <xref ref-type="bibr" rid="pgen.1001142-Cho2">[14]</xref>, <xref ref-type="bibr" rid="pgen.1001142-Kim2">[53]</xref>. To obtain the specific deletion of the <italic>Paxip1</italic>
<sup>fl</sup> allele in glomerular podocytes, these mice were crossed with the previously characterized 2.5P-Cre mice <xref ref-type="bibr" rid="pgen.1001142-Moeller1">[24]</xref>, <xref ref-type="bibr" rid="pgen.1001142-Moeller2">[25]</xref>, which express the Cre recombinase under the control of the human <italic>NPHS2</italic> promoter (<italic>Cre</italic>
<sup>NPHS2</sup>). Among the next generations, mice carrying the Cre allele (<italic>Paxip1</italic>
<sup>fl/fl</sup>:<italic>Cre</italic>
<sup>NPHS2</sup> and <italic>Paxip1</italic>
<sup>fl/&#8722;</sup>:<italic>Cre</italic>
<sup>NPHS2</sup> mice) were considered as conditional null mutants (PTIP&#8722;), whereas littermates that did not express the Cre recombinase were used as controls (PTIP+). All animal procedures were approved by the University Committee on Use and Care of Animals (UCUCA) of the University of Michigan and performed in compliance with ULAM recommendations.</p><p>Rabbit polyclonal antibodies used to detect Nephrin (1&#8758;1000) and Podocin (1&#8758;500) were kindly provided by L.B. Holzman (University of Pennsylvania, Philadelphia, PA). Chicken anti-PTIP was described previously <xref ref-type="bibr" rid="pgen.1001142-Lechner1">[54]</xref>. Additional antibodies were commercially available: mouse clone 6F-H2 anti-WT1 (1&#8758;1000, DAKO, Carpinteria, CA), anti-H3K4me3 and anti-H3K27me3 (AbCam, Cambridge, MA), anti-Magi2 (Sigma-Aldrich, St. Louis, MO), anti-Ntrk3 (AF1404, R &amp; D Systems, Minneapolis, MN), Alexa Fluor 488 F(ab&#8242;)2 fragment of goat anti-rabbit IgG, Alexa Fluor 568 F(ab&#8242;)2 fragment of goat anti-mouse IgG, Alexa Fluor 488 donkey anti-goat IgG (1&#8758;500; Molecular Probes, Life Technologies, Carlsbad, CA).</p><p>Mice had access to a standard breeder chow (Purina 5008) and water <italic>ad libitum</italic>. Urine was collected early in the afternoon for three consecutive days from individual mice at 1, 3, 6 and 12 months of age and stored frozen until use. After thawing, 2 &#181;L urine was run on a SDS-PAGE and stained with Coomassie Blue to test for the presence of proteins/albumin, using recombinant mouse albumin (Sigma-Aldrich, St. Louis, MO) as a control. Quantitative assessment of urine albumin and creatinine concentrations were determined by ELISA using the Albuwell M and Creatinine Companion kits (Exocell Inc., Philadelphia, PA).</p><p>Mice at 1, 3, 6, and 12 months of age were sacrificed and their kidneys were perfused, fixed, and processed for histology, indirect fluorescence and electron microcopy analyses. Briefly, mice were anesthetized by intraperitoneal injection of 40 mg/kg sodium pentobarbital and prepared for systemic perfusion. A saline solution was first injected through the abdominal aorta to the entire mouse body at a pressure of approximately 70 mmHg as previously described <xref ref-type="bibr" rid="pgen.1001142-Verma1">[55]</xref>. As soon as the general bloodstream had been cleared, a solution of 4% paraformaldehyde in PBS was substituted. It was left to perfuse at the same flow conditions for approximately 10 minutes. Kidneys were removed, decapsulated, cut into pieces, and incubated for 2 additional hours in the appropriate fixative solution before being processed for histology, indirect immunofluorescence, and electron microscopy.</p><p>Kidneys were fixed in 4% paraformaldehyde, embedded in paraffin, sectioned at 5 microns, and stained with Periodic Acid Shiff or Masson Trichrome. For immunofluorescence analyses with Nephrin, PTIP, WT1 and Magi2, sections were dewaxed, rehydrated, and microwaved for 10 minutes in a citric acid-based antigen unmasking solution (Vector Laboratories, Burlingame, CA). Sections were permeabilized with 0.3% Triton X-100 in PBS and blocked with 10% goat serum in PBS. Primary antibodies were incubated overnight at 4&#176;C in PBS, 0.1% Triton, 2% goat serum. Sections were washed twice and incubated with the secondary fluorescent antibodies and DAPI in PBS, 0.1% Triton, 2% goat serum for 1 hour in the dark at room temperature. The sections were washed again and mounted in Mowiol. Stained and fluorescent-labeled sections were analyzed under a Nikon ES800 microscope. Micrographs were taken with a digital spot camera, using equivalent exposure times among sections. For Ntrk3 staining, fresh frozen sections were dried, fixed in methanol at &#8722;20&#176;C and washed in PBS, 0.1% Tween 20 before incubation with anti-Ntrk3 antibodies at 1 &#181;g/ml.</p><p>For quantitation of immunofluorescent signals, ImageJ 1.42 was utilized. H3K4me3 stained sections were digitally captured and light intensity measured by placing a fixed size circular area over the nuclei of cells and summing all pixels over the given area. At least 6 podocytes and 6 control cells, either mesangial or endothelial, were measured for each of 8 glomerular tufts (at least 48 podocytes and 48 other cells for each genotype). The average signal intensity was then expressed as a ratio of podocyte intensity to non-podocyte cell intensity for each of the glomerular micrographs taken.</p><p>For Cre activity detection, the Rosa26-lacZ reporter strain was used <xref ref-type="bibr" rid="pgen.1001142-Soriano1">[56]</xref>. Mice carrying <italic>Cre</italic>
<sup>NPHS2</sup> and <italic>Paxip1<sup>fl/fl</sup></italic> were crossed to Rosa26-stop-lacZ:<italic>Paxip1<sup>fl/+</sup></italic> to generate <italic>Paxip1<sup>fl/fl</sup></italic>:<italic>Cre</italic>
<sup>NPHS2</sup>:Rosa26-lacZ animals. Kidneys were excised at 1 month of age and stained for &#946;-galactosidase activity as described <xref ref-type="bibr" rid="pgen.1001142-Kim3">[57]</xref>.</p><p>Longitudinal slices of kidneys from PTIP+ and PTIP&#8722; mice fixed with 2.5% glutaraldehyde in 0.1M Sorensen's buffer (pH 7.2) for 2 hours at room temperature were processed for scanning electron microscopy following standard procedures. Briefly, after several washes with the Sorensen's buffer alone, the samples were dehydrated by successive washes in graded ethanol solutions, critical point dried, mounted on a stub, sputter coated with gold-palladium, and examined under an AMRAY 1910 field emission scanning electron microscope. Pieces of the kidney cortex (1 mm<sup>3</sup>), fixed with 2.5% glutaraldehyde in Sorenson's buffer for 2 hours at room temperature, were processed for transmission electron microscopy following standard procedures. They were embedded in PolyBed 812 resin (Polysciences&#8194;Inc.), cut into 1-micron slices and stained with toluidine blue. Sample areas were selected based on the presence of glomeruli and cut into ultra-thin sections for analysis under a Philips CM-100 transmission electron microscope. The selected SEM and TEM images are representative of at least 10 different glomeruli per kidney.</p><p>Glomeruli were isolated from the kidneys of individual mice by sieving as described <xref ref-type="bibr" rid="pgen.1001142-Salant1">[58]</xref>. Briefly, 1 month-old mice were sacrificed by CO<sub>2</sub> inhalation and kidneys were removed. After decapsulation, the kidneys were finely minced on ice and passed sequentially through nylon meshes of 90 and 41 microns (Sefar Filtration Inc., Depew, NY). The glomeruli-enriched fraction (GEF) was retained on top of the 41-micron mesh, while kidney tubules were flushed through. RNA was isolated directly from the mesh.</p><p>Total RNA was extracted from the GEF of individual 1-month-old mice using the RNeasy Tissue Micro Kit (Qiagen, Valencia, CA) following the manufacturer's instructions. RNA concentration and purity were determined by nanodrop analysis on an Agilent Bioanalyzer 2100 (Agilent Technologies, Santa Clara, CA). Using the Ovation RNA Amplification System V2 (NuGEN Technologies, San Carlos, CA), 500 ng total RNA was reversed transcribed and linearly amplified into single-stranded cDNA, which concentration and purity were determined by nanodrop analysis on an Agilent Bioanalyzer 2100 (Agilent Technologies).</p><p>Microarray analyses were done by the University of Michigan Comprehensive Cancer Center (UMCCC) Affymetrix and Microarray Core Facility. The FL-Ovation cDNA Biotin Module V2 kit (NuGEN Technologies, San Carlos, CA) was used to produce biotin-labeled cRNA, which was then fragmented and hybridized to a Mouse 430 2.0 Affymetrix GeneChip 3&#8242; expression array (Affymetrix, Santa Clara, CA). Array hybridization, washes, staining, and scanning procedures were carried out according to standard Affymetrix protocols. Expression data were normalized by the robust multiarray average (RMA) method and fitted to weighted linear models in R, using the affy and limma packages of Bioconductor, respectively <xref ref-type="bibr" rid="pgen.1001142-Irizarry1">[59]</xref>, <xref ref-type="bibr" rid="pgen.1001142-Smyth1">[60]</xref>. Only probe sets with a variance over all samples superior to 0.1, a p-value inferior or equal to 0.05 after adjustment for multiplicity using the false discovery rate <xref ref-type="bibr" rid="pgen.1001142-Benjamini1">[61]</xref>, and a minimum 2-fold difference in expression were selected for the analysis. The complete data set is available from the Gene Expression Omnibus database (accession number GSE17709).</p><p>Microarray data were confirmed by real-time quantitative PCR analysis. 25&#8211;50 ng single-stranded cDNA was amplified in triplicate in a 384-well plate, using the 7900HT Fast Real Time PCR system (Applied Biosystems, Foster City, CA) and expression levels of selected genes was determined by SYBR Green or TaqMan assays (Applied Biosystems). PCR primers pairs and TaqMan probes used in this study are presented in <xref ref-type="supplementary-material" rid="pgen.1001142.s002">Table S2</xref>.</p><p>Glomeruli were isolated from 6 PTIP+ and 6 PTIP&#8722; kidneys by sieving as described above. Glomeruli were resuspended in 1 ml PBS and cross linked with 1% formaldehyde for 10 minutes with rocking at room temperature. Chromatin preparation, immunoprecipitation, and PCR analysis was essentially as described previously <xref ref-type="bibr" rid="pgen.1001142-Patel1">[13]</xref>. Primers pairs for the Ntrk3 locus were as follows: P1, <named-content content-type="gene">5&#8242;- CAATGTATTTTGCTTCCTTGCC</named-content>, <named-content content-type="gene">5&#8242;- AAGAAAGGGTTAGGGGAATCCG</named-content>
<italic>; P2, <named-content content-type="gene">5&#8242;- AACCCGTGCGTTTCGTAAGG</named-content>, <named-content content-type="gene">5&#8242;- GGAGGAAGGAGGAGAAGGAAGATG</named-content>; P3, <named-content content-type="gene">5&#8242;- GCATCTTCCTTCTCCTCCTTCCTC</named-content>, <named-content content-type="gene">5&#8242;- AAGTCACCAAGTCCCACCTCCTAG</named-content>; P4, <named-content content-type="gene">5&#8242;- TTTGCCTTCCCACCGTCTGTTG</named-content>, <named-content content-type="gene">5&#8242;- TGCCTTTGAAACGCCGAAC</named-content>.</italic>
</p>