<p>The study protocol was approved by the institutional review boards at Vanderbilt University Medical Center and at each of the collaborating institutes. Informed consent was obtained from all participants.</p><p>This study consisted of a discovery stage and two validation stages, i.e. an <italic>in silico</italic> and a <italic>de novo</italic> validation study. The overall study design is presented in <xref ref-type="supplementary-material" rid="pgen.1001127.s005">Figure S1</xref>.</p><p>The discovery stage included 1,019 T2D cases, 886 incident T2D cases from the Shanghai Women's Health Study (SWHS), an ongoing, population-based, prospective cohort study of women living in Shanghai, and 133 prevalent T2D cases identified among controls of the Shanghai Breast Cancer Study (SBCS), who were recruited in Shanghai during approximately the same period as the SWHS <xref ref-type="bibr" rid="pgen.1001127-Zheng1">[11]</xref>. Controls for the discovery phase were 1,710 non-diabetic female controls from the SBCS (for further details, see <xref ref-type="supplementary-material" rid="pgen.1001127.s008">Text S1</xref>, online). The biologic samples used for genotyping in this study were collected by the SWHS and SBCS.</p><p>DNA samples were genotyped using the Affymetrix Genome-Wide Human SNP Array 6.0. Extensive quality control (QC) procedures were implemented in the study. In the SWHS/SBCS GWAS scan, three positive QC samples purchased from Coriell Cell Repositories and a negative QC sample were included in each of the 96-well plates of the Affymetrix SNP Array 6.0. SNP data obtained from positive quality control samples showed a very high concordance rate of called genotypes based on 79,764,872 comparisons (mean, 99.87%; median, 100%). Samples with genotyping call rates less than 95% were excluded. The sex of all study samples was confirmed to be female. The identity-by-descent analysis based on identity by state was performed to detect first-degree cryptic relationships using PLINK version 1.06 <xref ref-type="bibr" rid="pgen.1001127-Purcell1">[12]</xref>. We excluded from the study 21 samples that had: 1) call rate &lt;95% (n&#8202;=&#8202;5); 2) samples that were contaminated or had mixed-up labels or that had been duplicated (n&#8202;=&#8202;12); 3) first-degree relatives, such as parent-offspring or full siblings (n&#8202;=&#8202;4).</p><p>We also excluded from the analysis SNPs that met any of following criteria: 1) MAF &lt;0.05; 2) call rate &lt;95%; 3) P for Hardy-Weinburg equilibrium HWE &lt;0.00001 in either the case or control groups or in the combined data set; 4) concordance rate &lt;95% among the duplicated QC samples; 5) significant difference in allele frequency distribution (P&lt;0.00001) between the 886 T2D cases from the SWHS and the 133 T2D cases from the SBCS; 6) significant difference in missing rates between cases and controls (P&lt;0.00001). After applying the QC filter, 590,887 SNPs remained for the analyses.</p><p>Because of financial constraints, we conducted a fast-track validation study using an approach that combined <italic>in silico</italic> and <italic>de novo</italic> replication. We selected a total of 2,100 SNPs from the discovery phase that had P-values of 1.3&#215;10<sup>&#8722;9</sup> to 5.0&#215;10<sup>&#8722;3</sup> derived from the additive model and that were not in linkage disequilibrium (LD; r<sup>2</sup>&lt;0.2 based on the HapMap CHB dataset) with any previously reported T2D GWAS SNPs for an <italic>in silico</italic> replication using the GWAS scan data from the NHS/HPFS <xref ref-type="bibr" rid="pgen.1001127-Qi1">[2]</xref>. We used the NHS/HPFS T2D GWAS scans for our first step of validation, because the Shanghai T2D GWAS was conducted concurrently and used the same genotyping platform as the NHS/HPFS T2D GWAS and <italic>a priori</italic> arrangement was made for the two studies to exchange the top 2,000 SNPs for <italic>in silico</italic> replication. The NHS/HPFS T2D GWAS included 2,591 cases and 3,052 controls of European ancestry. We recognize that this approach may have reduced our chances of finding ethnicity-specific T2D markers, however, this approach had the advantage of enhancing our ability of finding true genetic markers. From the first <italic>in silico</italic> replication, 65 SNPs with the same direction of association in both studies and with a MAF &gt;20% were chosen for a second <italic>in silico</italic> replication using GWAS scan data from a Korean T2D study, which included 1,042 cases and 2,943 controls genotyped with the Affymetrix Genome-Wide Human SNP Array 5.0 platform. In order to improve yield, only the top SNPs that are included in Affymetrix 5.0 (N&#8202;=&#8202;56) or that are in high LD (r<sup>2</sup>&gt;0.8) with at least one SNP on Affymetrix 5.0 (N&#8202;=&#8202;9) were selected for replication (<xref ref-type="supplementary-material" rid="pgen.1001127.s001">Table S1</xref>). Of the 65 SNPs, the top 8 SNPs replicated in the Korean T2D study were further investigated using GWAS data from a T2D study conducted among Singapore Chinese (2,010 cases and 1,945 controls) who were genotyped by using Illumina HumanHap 610 or Illumina Human1M (<xref ref-type="supplementary-material" rid="pgen.1001127.s002">Table S2</xref>). Four of the 8 SNPs were not directly genotyped in the Singapore study, so instead, we selected SNPs that are in strong LD with these 4 SNPs (imputed SNP information became available recently and is presented in this report). Finally, the 5 top SNPs (rs2815429, rs10906115, rs1359790, rs10751301, and rs1436955) were selected for <italic>de novo</italic> genotyping in an independent sample set of 1,645 T2D cases and 1,649 controls identified from the SWHS and Shanghai Men's Health Study (SMHS). Four of these SNPs (rs10906115, rs1359790, rs10751301, and rs1436955) were selected for the final stage of <italic>de novo</italic> genotyping replication in two independent Chinese studies, the Wuhan Diabetes Study (WDS; 1,063 cases and 1,408 controls) and the Nutrition and Health of Aging Population in China (NHAPC) study (424 cases and 1,908 controls). Detailed descriptions of the study designs and populations for each of the participating studies are presented in <xref ref-type="supplementary-material" rid="pgen.1001127.s008">Text S1</xref> online.</p><p>Genotyping for the 5 SNPs included in the SWHS and SMHS sample set was completed using the iPLEX Sequenom MassArray platform. Included in each 96-well plate as quality control samples were two negative controls, two blinded duplicates, and two samples included in the HapMap project. We also included 65 subjects who had been genotyped by the Affymetrix SNP Array 6.0 in the Sequenom genotyping. The consistency rate was 100% for all SNPs for the blinded duplicates, compared with the HapMap data and compared with data from the Affymetrix SNP Array 6.0. Genotyping for the final 4 SNPs in the WDS and NHAPC was completed using TaqMan assays at the two local institute laboratories using reagents provided by the Vanderbilt Molecular Epidemiology Laboratory. Both laboratories were asked to genotype a trial plate provided by the Vanderbilt Molecular Epidemiology Laboratory that contained DNA from 70 Chinese samples before the main study genotyping was conducted. The consistency rates for these trial samples were 100% compared with genotypes previously determined at Vanderbilt for all four SNPs in both local laboratories. In addition, replicate samples comparing 3&#8211;7% of all study samples were dispersed among genotyping plates for both studies.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The imputation of un-genotyped SNPs in all participating GWASs was carried out after the completion of the current study using the programs MACH (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/MACH/">http://www.sph.umich.edu/csg/abecasis/MACH/</ext-link>) or IMPUTE (<ext-link ext-link-type="uri" ns0:href="https://mathgen.stats.ox.ac.uk/impute">https://mathgen.stats.ox.ac.uk/impute</ext-link>) with HapMap Asian data as the reference for Asians and CEU data as the reference for European-ancestry samples. Only data with high imputation quality (RSQR &gt;0.3 for MACH) were included in the current analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">PLINK version 1.06 was used to analyze genome-wide data obtained in the SBCS/SWHS GWAS scan. Population structure was evaluated by principal component analysis using EIGENSTRAT (<ext-link ext-link-type="uri" ns0:href="http://genepath.med.harvard.edu/~reich/Software.htm">http://genepath.med.harvard.edu/~reich/Software.htm</ext-link>).A set of 12,533 SNPs with a MAF &#8805;10% in Chinese samples and a distance of &#8805;25 kb between two adjacent SNPs was selected to evaluate the population structure. The first two principal components were included in the logistic regression models for adjustment of population structures. The inflation factor &#955; was estimated to be 1.03, suggesting that population substructure, if present, should not have any appreciable effect on the results.</p><p>Pooled and meta-analyses were carried out in SAS to derive combined odds ratios (OR) by using data from studies of all stages. We applied the weighted z-statistics method, where weights are proportional to the square root of the number of subjects in each study. Results from both random and fixed effect models are presented.</p><p>ORs and 95% confidence intervals (CI) were estimated using logistic regression models with adjustment for age, BMI, population structure (for GWAS data), and gender, when appropriate. Analyses with additional adjustment for smoking were conducted by pooled analysis whenever possible and by meta-analysis when KARE data were included in order to examine the confounding and modification effects of these factors (<xref ref-type="supplementary-material" rid="pgen.1001127.s002">Table S2</xref>). Genotype distributions for the top 4 SNPs included in the final <italic>de novo</italic> genotyping were consistent with HWE (P&gt; 0.05) in each study. All P values presented are based on two-tailed tests, except where indicated otherwise.</p>