<p>For a list of strains, see <xref ref-type="table" rid="pgen-1001006-t001">Table 1</xref> below. Each simplex strain described in the table represents the genotype of at least 2 more independent isolates that originated from 2 independent duplex parents. Haploid strains were derivatives of E134 <xref ref-type="bibr" rid="pgen.1001006-Tran1">[54]</xref> and its <italic>met2</italic> and <italic>met6</italic> derivatives DAG 647 and DAG 645 respectively. <italic>ura 3&#8211;52</italic> has been replaced by a complete deletion that gave rise to strains CS1004 and CS1006 (see below). CS1004 (relevant genotype <italic>MAT&#945; met 6</italic>-DEL) was transformed with CORE cassette <xref ref-type="bibr" rid="pgen.1001006-Storici1">[55]</xref> targeted to the 3&#8242; end of <italic>TYR1</italic>, starting at nucleotide 700 of the ORF. Briefly, the following primers were used to amplify G418<sup>R</sup> in tandem with the <italic>URA3</italic> cassette from pCORE <xref ref-type="bibr" rid="pgen.1001006-Storici1">[55]</xref> in order to create a 5&#8242; <italic>tyr1</italic> allele.</p><p>
<named-content content-type="gene">5&#8242;ATCTATTCGAACAAGTGGCATGTTTACGCAGGATTAGCCATAACAAACCCAAGTGCACAT-GAGCTCGTTTTCGACACTGG</named-content> and <named-content content-type="gene">5&#8242;TTATGTATTTCTTTTTTCAGCGGCCGAACGGTCACTAGAATGACTCAGAATGGTTTTTAT-TCCTTACCATTAAGTTGATC</named-content>. In parallel the CS1006 (relevant genotype <italic>MATa met 2</italic>-DEL) was transformed with a CORE cassette targeted to the 5&#8242; part of the <italic>TYR1</italic> (nucleotide #1 at ORF) in order to create a 3&#8242; <italic>tyr1</italic> allele. This was done by amplifying CORE cassette with primers <named-content content-type="gene">5&#8242;ATGGTATCAGAGGATAAGATTGAGCAATGGAAAGCCACAAAAGTCATTGGTATAATTGGT-GAGCTCGTTTTCGACACTGG</named-content> and <named-content content-type="gene">5&#8242;TGTGTATCAGCGGAATCTTGTCATGCTCTTCATATGTCAAATAAACTTGCTTGCTTACTTTTC-TCCTTACCATTAAGTTGATC</named-content>.</p><p>After selection of the CORE integrants, the CORE was removed <xref ref-type="bibr" rid="pgen.1001006-Storici1">[55]</xref> from CS1004 (<italic>met6</italic>-DEL) using oligonucleotides <named-content content-type="gene">5&#8242;TGGCATGTTTACGCAGGATTAGCCATAACAAACCCAAGTGCACATATAAAAACCATTCTGAGTCATTCTAGTGACCGTTCGGCCGCTGAAA</named-content> and <named-content content-type="gene">5&#8242;TTTCAGCGGCCGAACGGTCACTAGAATGACTCAGAATGGTTTTTATATGTGCACTTGGGTTTGTTATGGCTAATCCTGCGTAAACATGCCA</named-content>.</p><p>The CORE was removed from CS1006 derivative (<italic>met2</italic>-DEL) by introducing a portion of the <italic>TYR1</italic> gene PCR fragment defined by primers <named-content content-type="gene">5&#8242;TGAAGGAAAGAGGACAGCATATCCACTTGATAAACAAAGTATTTACCCAAGGACTGCGACATCATTACCGTGCATTCCCTTCATG</named-content> and <named-content content-type="gene">5&#8242;GCCACTTGTTCGAATAGATTCTTAGTGATATATTAACTTTCACATTTTCT.</named-content>
</p><p>Loss of CORE was identified by conversion of strain to G418 sensitive and 5FOA resistant. At the end of this stage, two sets of strains were created; CS1004 derivative with <italic>met6</italic> DEL and with <italic>tyr1</italic> allele of 1&#8211;700 bp and CS1006 derivative with <italic>tyr1</italic> allele 300&#8211;1359 bp.</p><p>Diploid cells were made from derivatives of the above (Strains CS 1061 and CS 1064 see <xref ref-type="table" rid="pgen-1001006-t001">Table 1</xref>) by introducing plasmid (YEp-HO) encoding HO endonuclease under its native promoter to the haploid strains. Non-mating cells were identified as potential diploids and confirmed by ability to sporulate. Each a/&#945; diploid was then transformed with a pGal-HOT plasmid were HO is inducible by galactose <xref ref-type="bibr" rid="pgen.1001006-Bennett1">[56]</xref>. Cells were grown 6 hr on galactose containing media to induce mating type switching. <italic>MAT</italic>aa and <italic>&#945;&#945;</italic> cells were identified by mating test. The aa and &#945;&#945; strains were isolated from each set of haploid strains (<italic>met</italic> 6-DEL 5&#8242; <italic>tyr1</italic> allele and <italic>met 2</italic>-DEL 3&#8242; <italic>tyr1</italic> allele). The <italic>MAT</italic>aa and <italic>&#945;a</italic> cells served as Diploid 1and 2 as shown in <xref ref-type="fig" rid="pgen-1001006-g001">Figure 1</xref>.</p><p>WT tetraploid cells were obtained by crossing several diploid isolates with opposite mating strains and complementing <italic>met</italic> mutations. Cells were selected on media lacking methionine and confirmed to be non-mating. They also exhibited spontaneous and UV induced Tyr+ recombination.</p><p>The following oligonucleotides were used to create G418<sup>R</sup> and Hygromicin<sup>R</sup> cassettes that could be targeted to <italic>MCD1</italic> open reading frame using plasmids pFA6 and pAG32 respectively: <named-content content-type="gene">5&#8242;TCCATAACAAAAAAGGACTGGTCAAAGAAAAGACAACTCAATTGCACAATTACTTTACAAGAAACACGACA-CGTACGCTGCAGGTCGACGGATCCCC</named-content> and <named-content content-type="gene">5&#8242;TTAAAGTCTTTGATCTATATATGCATCAGCTTATTGGGTCCACCAAGAAATCCCCTCGGCGTAACTAGGTT-ATCGATGAATTCGAGCTCGTTTTCGG.</named-content>
</p><p>The <italic>MCD1</italic> heterozygote diploid was created by transforming Diploid 1 and 2 cells with the targeted G418<sup>R</sup> and Hygromicin<sup>R</sup> cassettes, respectively. Independent <italic>MCD1</italic> heterozygote isolates derived from Diploids 1 and 2 were crossed to create <italic>MCD1</italic> duplexes (two WT alleles with one G418<sup>R</sup> and one Hygromicin<sup>R</sup> replacement alleles; see <xref ref-type="fig" rid="pgen-1001006-g001">Figure 1</xref>). Duplexes were transformed with a <italic>URA3</italic> cassette that was targeted to an internal (23 aa in frame) portion of the open reading frame by amplifying <italic>URA3</italic> gene from pRS306 using primers <named-content content-type="gene">5&#8242;TGGTTACAGAAAATCCTCAACGTCTTACTGTTTTAAGACTTGCCACCAATAAGGGTCCATTAGCACAAATACAGAG-CAGATTGTACTGAGAGTGCACC</named-content> and <named-content content-type="gene">5&#8242;TAAGCATTGATAAACCTTTCAAATAGTGCAGGTTTGGCGTCTATTTTAATATTTCCGAATGCTTCTGTTTG-CGCATCTGTGCGGTATTTCACACCGC</named-content>.</p><p>Ura<sup>+</sup> transformants were confirmed to be <italic>MCD1</italic> simplex if they maintained G418<sup>R</sup>, Hygromicin<sup>R</sup>, were non-mating and Tyr<sup>+</sup> recombinants could be induced. In addition genomic DNA was purified from the putative simplex and <italic>MCD1</italic> locus was PCR using the flanking primers <named-content content-type="gene">5&#8242;GTCGAGAAAATCGCGTCTTTC</named-content> and <named-content content-type="gene">5&#8242;AGAAAATTTCGGCTTCACCG</named-content>.</p><p>Since the <italic>MCD1</italic> ORF size is almost identical to the G418<sup>R</sup> cassette, another set of PCR derived constructs was developed using a primer within the cassette (<named-content content-type="gene">5&#8242;CGTACGCTGCAGGTCGAC</named-content>) and a primer outside the ORF (<named-content content-type="gene">5&#8242;AGAAAATTTCGGCTTCACCG</named-content>). This analysis revealed two PCR products corresponding to G418<sup>R</sup> and Hygromicin<sup>R</sup> cassettes. Immediately after PCR verification of the simplex genotype, patches were stored at &#8722;70&#176;C. At the beginning of each experiment, cells were streaked from the frozen stock for single colonies which were tested for the presence of the simplex markers.</p><p>
<italic>SMC3</italic> simplex strains were created similarly using primers: <named-content content-type="gene">5&#8242;CATCTTTAAACAGTTTCACCATTTTTTTACAAGACGACCTGCTGGAGTAACGGTAATAGTTCACGTCTGCA-CGTACGCTGCAGGTCGACGGATCCCC</named-content> and <named-content content-type="gene">5&#8242;CAGTACCTCTGGGAACTAATCTTTCAAAAACAGCTTCAAAGTTTTCAGAAACCTTTTGGAAAGTAGAATCA&#8211;ATCGATGAATTCGAGCTCGTTTTCGA</named-content> to create G418 and Hygromycin resistant cassettes directed to the ORF as well as primers <named-content content-type="gene">5&#8242;ATGTATATCAAAAGGGTGATAATTAAGGGTTTTAAAACCTACAGGAACGAAACCATTATTGATAATTTCT-CAGAGCAGATTGTACTGAGAGTGCACC</named-content> and <named-content content-type="gene">5&#8242;ACCGCGTTAACCTTTTGTTGTTTTAACTTAACAATTAGATCTTGAATTGAGTCTTTAGATTCATCCAGCTC-CGCATCTGTGCGGTATTTCACACCGC</named-content> to amplify pRS306 to create a <italic>URA3</italic> cassette targeted to the ORF. The simplex was verified by primers flanking the locus: <named-content content-type="gene">5&#8242; CATCGAAGTGTACACCTGTCACAT</named-content> and <named-content content-type="gene">5&#8242; GAAAAGTAATCTTTTTTGTACGTCG</named-content>.</p><p>
<italic>RAD50</italic> simplexes were made in a manner similarly to above. Oligonucleotides <named-content content-type="gene">5&#8242;TTTCACGGCTTTGCCTTGT</named-content> and <named-content content-type="gene">5&#8242;TCAAAGGTGCTTACGTGCTTG</named-content> were used to amplify the flanking region around the <italic>RAD50</italic> locus in a null strain from the <italic>Saccharomyces cerevisiae</italic> deletion library (G418 cassette replaced ORF). Both Diploids 1 and 2 were transfected with a G418 cassette and after obtaining heterozygote diploid isolated, the G418 cassette of one of the diploids was switched to Hygromicin resistance. The two heterozygote diploids were crossed and the tetraploid duplex was transformed with a <italic>URA3</italic> cassette targeted to an internal portion of the ORF by amplifying the <italic>URA3</italic> gene from pRS306 using the following primers <named-content content-type="gene">5&#8242;TCTATTCAGGGCATACGGTCTTTTGACTCCAATGATAGGGAAACTATTGAATTTGGCAAGCCTCTGACTTC-AGAGCAGATTGTACTGAGAGTGCACC</named-content> and <named-content content-type="gene">5&#8242;TATCGACCCACTCAATTTGTGATTTTTGCCTATCATCTCTCTTGACTTTGAAGAAGTGATCAGTAAATGCCG-CGCATCTGTGCGGTATTTCACACCGC</named-content>. Simplex strains were selected as described above.</p><p>Construction of the <italic>RAD51</italic> simplex was done by sequential transfection of Diploid1 with G418 and <italic>URA3</italic> cassettes, thereby replacing two copies of the gene. Diploid 2 was transfected with a Hygromicin cassette targeted into the ORF of the gene. G418 and Hygromicin cassettes were made based on strains from a <italic>Saccharomyces cerevisiae</italic> deletion library using primers <named-content content-type="gene">5&#8242; TTGAGCATTCCCTGAGCATT</named-content> and <named-content content-type="gene">5&#8242;TCCCCTAAAAGGATAAAGCCG</named-content>. <italic>URA3</italic> cassette was created by amplifying pRS306 using primers <named-content content-type="gene">5&#8242;CATATATCAGAGTCACAGCTTCAGTACGGGAACGGTTCGTTGATGTCCACTGTACCAGCAGACCTTTCACCAGAGCAGATTGTACTGAGAGTGCACC</named-content> and <named-content content-type="gene">5&#8242;GGTCACCAACACCATCTTCATAGATCGCGAACACACATTCAGCCTCTGGTAAGCAAGGTGAGTCAACAAC-CGCATCTGTGCGGTATTTCACACCGC.</named-content> The <italic>RAD51</italic> simplex strain was created by crossing the transformed Diploids 1 and 2.</p><p>CS1061 and CS1064 (a and &#945; haploids) that contain the same types of <italic>tyr1</italic> truncation alleles described above crossed to yield WT diploid. Same haploid strains were transformed with an Age1 cut pVG257 <xref ref-type="bibr" rid="pgen.1001006-Guacci1">[31]</xref> to yield the <italic>mcd1-1</italic> strain. The <italic>mcd1-1</italic> cells were verified by sequencing and later crossed to the opposite mating counterpart to create a diploid <italic>mcd1-1</italic> strain.</p><p>The WT, <italic>MCD1</italic> simplex, <italic>RAD50</italic> simplex or <italic>RAD51</italic> simplex cells were grown overnight and diluted to fresh media and grown to for 3 hr in 30&#176;C and then harvested. Before Cells were washed with double-distilled water and 2&#215;10<sup>7</sup> cells were re-suspended in 0.3ml SDS-running buffer, boiled for 10 min and centrifuge 5 minutes 13,000 rpm. Typically 40 &#181;l supernatant was loaded per lane (corresponding to &#8764;2.5&#215;10<sup>6</sup> cells). Following electrophoresis, the gel was transferred to a membrane using a semi-dry transfer apparatus for 105 min., according to manufacturer's instructions using a PVDF membrane (Invitrogen Carlsbad, CA). All antibodies were diluted 1&#8758;2000 except anti Rad51 that was diluted 1&#8758;5000. Anti Mcd1 antibody was kindly provided by Dr. Alexander Strunnikov <xref ref-type="bibr" rid="pgen.1001006-Guacci1">[31]</xref>. Anti yRad50 antibody sc32862 and anti yRad51 antibody sc33626 were from Santa Cruz Biotechnology (Santa Cruz, CA). The anti yHistone 3 antibody was ab1791 from Abcam (Cambridge, MA).</p><p>The details of nocodazole arrest, and gamma irradiation have been described <xref ref-type="bibr" rid="pgen.1001006-Argueso1">[6]</xref>, <xref ref-type="bibr" rid="pgen.1001006-Westmoreland1">[18]</xref>. Briefly, nocodazole (20 &#181;g/ml, final concentration) was added to cells that were growing logarithmically at 30&#176;C in YPDA media (1% yeast extract, 2% Bacto-Peptone, 2% dextrose, 60 &#181;g/ml adenine sulfate). G<sub>2</sub> arrest was monitored by cell morphology. Cells were collected by centrifugation, washed and re-suspended in ice-cold sterile water. The cell suspensions were kept on ice while being irradiated in a <sup>137</sup>Cs irradiator (J. L. Shepherd Model 431) at a dose rate of 2.3 krads per minute. Irradiated cells were harvested by centrifugation and resuspended in YPDA at 30&#176;C with nocodazole for post-irradiation incubation.</p><p>PFGE procedures were done as previously described <xref ref-type="bibr" rid="pgen.1001006-Westmoreland1">[18]</xref>. Briefly, Contour-clamped Homogeneous Electric Field (CHEF) systems were used for electrophoresis of yeast chromosomes in this study. Using a CHEF Mapper XA system (Bio-Rad, Hercules, CA). These plugs were prepared in 0.5% LE agarose (Seakem, Rockland, ME) using 1&#8211;2&#215;10<sup>7</sup> G2-arrested cells per 100 ul plug. They were cut to a thickness of &#8764;2 mm and loaded in the bottom of a preparative well so that the entire DNA migrated very close to the bottom surface of the CHEF gel. PFGE running conditions were according to the CHEF auto-algorithem separates DNA's in the 250&#8211;1600 kb range.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To quantify DSBs in irradiated samples, pulsed-field gels were stained with SybrGold (Invitrogen, San Diego, CA) and photographed using a GelLogic200 imaging system (Eastman Kodak, Rochester, NY). Bands were measured using Kodak MI software (version 4.0) and the data were exported into Microsoft Excel (version 11.5.3) for further manipulations to determine DSBs. More details on the analysis are found in <xref ref-type="supplementary-material" rid="pgen.1001006.s001">Figure S1A</xref> of <xref ref-type="bibr" rid="pgen.1001006-Westmoreland1">[18]</xref>. Briefly, for each band corresponding to a complete unbroken Chromosome Y (any chromosome), the fraction of chromosomes remaining unbroken (<italic>F<sub>ChrY</sub></italic>) after a given dose is simply the net intensity of the band divided by the net intensity of the corresponding band in the 0 krad control lane. From the Poisson distribution, the average number of DSBs (<inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001006.e001.jpg" /></inline-formula>) is given by the formula:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001006.e002.jpg" /></disp-formula>
</p><p>Plotting the experimentally determined values of <italic>N</italic> (number of breaks per chromosome) vs Molecular Weight for each chromosome band from a given dose results in an approximate straight line whose slope is in units of DSBs/mb and is independent of the total amount of DNA loaded in each lane as long as enough DNA is loaded for accurate detection of the bands. For details see reference <xref ref-type="bibr" rid="pgen.1001006-Westmoreland1">[18]</xref>. The experimentally determined values of the slope for a given dose are highly reproducible.</p><p>Tyr<sup>+</sup> cells were grown over-night in 30&#176;C in YPDA in deep well 96 plates with shaking. Genomic DNA was purified using DNeasy of Quiagen (Valencia, CA) and amplified using primers <named-content content-type="gene">5&#8242; GAATACCGTAGCACTTGAAGGAAAGAGGACAGCATATCCA</named-content> and <named-content content-type="gene">5&#8242; CACAAAAGAAGGCCTAATATTATAGGAAATCAGCATTAAAAAC.</named-content>
</p>