<p>Bacterial strains and plasmids used in this study are listed in <xref ref-type="table" rid="pgen-1000939-t001">Table 1</xref> and <xref ref-type="table" rid="pgen-1000939-t002">Table 2</xref>, respectively. Primers are listed in <xref ref-type="supplementary-material" rid="pgen.1000939.s005">Text S1</xref>. <italic>E. coli</italic> and <italic>V. cholerae</italic> were grown in LB (10 g tryptone +5 g yeast extract +5 g NaCl per liter, pH adjusted with NaOH to &#8764;7) or M63 medium (KH<sub>2</sub>PO<sub>4</sub> 3 g + K<sub>2</sub>HPO<sub>4</sub> 7 g + (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> 2 g + FeSO<sub>4</sub> 0.5 mg + MgSO<sub>4</sub>.7H<sub>2</sub>O 0.25 g, pH adjusted with KOH to &#8764;7) supplemented with 2 mM MgSO<sub>4</sub>, 0.1 mM CaCl<sub>2</sub>, 0.01% thiamine and 0.2% glucose, and additionally 0.1% casamino acids when desired. Antibiotics were used at the following concentrations: ampicillin, 100 &#181;g/ml; chloramphenicol, 25 &#181;g/ml for <italic>E. coli</italic>, 5 &#181;g/ml for <italic>V. cholerae</italic>; erythromycin, 20 &#181;g/ml; kanamycin, 25 &#181;g/ml; spectinomycin, 50&#181;g/ml; tetracycline, 15 &#181;g/ml; and zeocin, 25 &#181;g/ml. Diaminopimelic acid (DAP) was used at 0.8 mM, L-arabinose at 2 or 0.2 mg/ml, IPTG at 100 &#181;M and thymidine at 0.3 mM.</p><p>To replace <italic>oriC</italic> (coordinates 3923756&#8211;3924022) with <italic>oriI</italic> (coordinates 2961130&#8211;364), the latter was amplified from DNA of CVC209 by PCR using primers GD113 and GD114. The PCR product was digested with EcoRI and BamHI, and ligated to similarly digested pEM7-Zeo. The resulting plasmid, pGD83, was digested with SacI and BamHI, and the fragment containing the <italic>oriI</italic>-<italic>zeo</italic> region was ligated to a similarly digested vector, pSW23, generating pGD79. The <italic>oriI</italic>-<italic>zeo</italic> region of pGD79 was amplified with primers GD124 and GD125, and the product used to replace <italic>oriC</italic> of CVC1394 by the mini-&#955; Red recombineering method <xref ref-type="bibr" rid="pgen.1000939-Court1">[45]</xref>. The mini-&#955; prophage was eliminated from the strain by a 30&#176;C to 42&#176;C temperature shift. The resultant strain was called MG1655&#916;<italic>oriC::oriI-zeo</italic> (CVC1400), and the replacement was confirmed by sequencing of the origin region. The genomic DNA of the <italic>dam</italic> mutant derivative (CVC1401) was confirmed for the absence of adenine methylation by its resistance to DpnI but not to MboI and BfuCI restriction enzymes (data not shown).</p><p>Cultures of <italic>E. coli</italic> were grown in LB to OD<sub>600</sub>&#8776;0.2 and processed for flow cytometry after replication run-out in the presence of rifampicin (150 &#181;g/ml) and cephalexin (10 &#181;g/ml) for three hours as described <xref ref-type="bibr" rid="pgen.1000939-Srivastava2">[46]</xref>. The peak fluorescence intensity of an overnight grown <italic>E. coli</italic> culture in M63 + 0.2% glucose medium (without casamino acids) was taken to represent one genome equivalent.</p><p>A fragment with six 12-mers was obtained from pGD61 by digestion with XhoI and NotI.</p><p>Fragments with three 11-mers and a pair of 12- and 11-mers were obtained from pTVC86 and pTVC88, respectively, by digestion with XhoI and BamHI. For methylated and unmethylated fragments, the plasmids were from a <italic>dam</italic>
<sup>+</sup> (BR2699) and a <italic>dam</italic>
<sup>&#8722;</sup> (CVC1060) strain, respectively. The fragments were gel-purified, dephosphorylated with Shrimp Alkaline Phosphatase (USB Corporation), and end-labeled with 50 &#181;Ci [&#947;-<sup>32</sup>P]ATP (PerkinElmer) by using 30 units of T4 polynucleotide kinase (New England Biolabs) and purified through ProbeQuant G-50 micro columns (GE Healthcare). To obtain hemimethylated DNA, oligonucleotide primers, TVC64 and TVC138 (Sigma-Genosys), were end-labeled and purified as above. The labeled primers were then used for PCR one at a time with methylated DNA as template for one cycle to obtain two populations of hemimethylated DNA, one with methylation on the top strand and the other on the bottom strand. The binding reactions were essentially as described <xref ref-type="bibr" rid="pgen.1000939-VenkovaCanova1">[47]</xref>.</p><p>A partial deletion of <italic>seqA</italic> was made by deleting codons 51 to 140 and substituting the deleted region with a zeocin cassette maintaining the <italic>seqA</italic> reading frame as follows. The <italic>seqA</italic> gene was amplified from CVC209 by PCR with primers GD87 and GD88. The product was digested with EcoRI and cloned in similarly digested vector, pSW4426T. The resultant plasmid was used as template for PCR with primers GD91 and GD92 to amplify the 5&#8242; end of <italic>seqA</italic>, the plasmid backbone and the 3&#8242;end of <italic>seqA</italic>. After digestion with MfeI, a site of which was present within GD91 and GD92 primers, the PCR product was ligated to the zeocin cassette. The cassette was obtained from pEM7-Zeo by PCR, using primers GD89 and GD90 and digested with EcoRI before ligating to the MfeI fragment. The resulting plasmid, pGD70, containing the &#916;<italic>seqA</italic>
<sub>P</sub>
<italic>::zeo</italic> allele was used to replace <italic>seqA</italic> of CVC209 by the allele-exchange method <xref ref-type="bibr" rid="pgen.1000939-LeRoux1">[48]</xref>. The resulting &#916;<italic>seqA</italic>
<sub>P</sub>
<italic>::zeo</italic> mutant (CVC1410) grew slower than the WT. In LB at 37&#176;C, the doubling times of the mutant was 32&#177;2 min as opposed to 19&#177;2 min for the WT. The &#916;<italic>seqA</italic>
<sub>P</sub>
<italic>::zeo</italic> allele is called hereafter &#916;<italic>seqA</italic>
<sub>P</sub>.</p><p>The entire <italic>seqA</italic> ORF was also deleted and substituted with the zeocin cassette as follows. First, a kilobase region located downstream the stop codon of <italic>seqA</italic> was amplified by PCR with primers GD228 and GD229, the product digested with EcoRI and BamHI and cloned in a derivate of pEM7-Zeo (pGD111), previously digested with the same enzymes, generating pGD113. pGD111 is essentially same as pEM7-Zeo except that the multi-cloning site upstream of <italic>zeo</italic> is modified to include KpnI and NdeI restriction sites. Next, a kilobase region located upstream of the start codon of <italic>seqA</italic> was amplified by PCR with primers GD230 and GD231, the product digested with KpnI and NdeI and cloned in pGD113, previously digested with the same enzymes, generating pGD114. The flanking regions of <italic>seqA</italic>, now flanking the zeocin cassette, was amplified by PCR with primers GD257 and GD258 and the linear product was introduced by natural transformation in a <italic>hapR<sup>+</sup></italic> &#916;<italic>dns</italic> derivative of N16961 (CVC1121) essentially as described <xref ref-type="bibr" rid="pgen.1000939-Blokesch1">[49]</xref>, <xref ref-type="bibr" rid="pgen.1000939-Blokesch2">[50]</xref>. The transformants were selected for zeocin resistance and checked for the replacement of the <italic>seqA</italic> gene by the zeocin cassette by PCR and DNA sequencing. The resulting &#916;<italic>seqA</italic>
<sub>T</sub>
<italic>::zeo</italic> mutant (CVC2003) grew as slow as the &#916;<italic>seqA</italic>
<sub>P</sub>
<italic>::zeo</italic> mutant with a doubling time of 32&#177;2 min. The &#916;<italic>seqA</italic>
<sub>T</sub>
<italic>::zeo</italic> allele is called hereafter &#916;<italic>seqA</italic>
<sub>T</sub>.</p><p>A complete deletion of the <italic>dam</italic> ORF and its substitution with a zeocin cassette was obtained by the allele-exchange method in the presence of a complementing plasmid, pGD93. The replication of the plasmid was thermo-sensitive and it carried the <italic>V. cholerae dam</italic> under the P<sub>BAD</sub> promoter. pGD93 was made as follows: the dam gene was amplified by PCR with primers GD72 and GD73, and the product after digestion with EcoRI and KpnI was cloned in pBAD24, previously digested with the same enzymes, generating pGD55. Next, the NdeI-HindIII fragment from pGD55 containing the <italic>dam</italic> gene was cloned in pKOBEGA, previously digested by NdeI and HindIII, generating the pGD93. For allele-exchange, a kilobase region located downstream of the stop codon of <italic>dam</italic> was amplified by PCR with primers GD261 and GD262, and the product after digestion with EcoRI and BamHI was cloned in a derivative of pEM7-zeo, previously digested with the same enzymes, generating pGD117. A 700 bp region located upstream of the start codon of <italic>dam</italic> was amplified by PCR with primers GD263 and GD264, and the product after digestion with KpnI and NdeI was cloned in pGD117, previously digested with the same enzymes, generating pGD118. The zeocin cassette with the flanking regions of <italic>dam</italic> was amplified by PCR with primers GD268 and GD269, and the product cloned as a blunt end fragment in pSW23, previously digested with SmaI, generating the pGD120. The plasmid was digested with SacI and SalI, and the fragment with the zeocin cassette was cloned into pDS132, previously digested also with the same enzymes. The resulting plasmid, pGD121, was used to replace <italic>dam</italic> of CVC209/pGD93. The resulting strain, CVC2023, was confirmed for the replacement of <italic>dam</italic> by the zeocin cassette by PCR and by DNA sequencing. To deplete Dam, single colonies grown in the presence of ampicillin (to select pGD93) and arabinose (to express <italic>dam</italic>) were used to inoculate LB without any drug but containing glucose (to repress <italic>dam</italic> expression) and the cultures were grown at 42&#176;C (to stop plasmid replication).</p><p>Genomic DNA was isolated from cells of log phase cultures (OD<sub>600</sub>&#8776;0.3), using the Genelute Bacterial Genomic DNA kit (Sigma). For analyzing chrI and <italic>E. coli</italic> DNA, 1 &#181;g of DNA was digested 2 hours with 7.5 or 15 units of HphI (New England Biolabs) at 37&#176;C, and the products resolved in a 1.5% agarose gel. For chrII, the conditions were similar except that Taq<sup>&#945;</sup>I was used at 65&#176;C. The origin probes were prepared by PCR using primers GD36 and GD37 for <italic>ori</italic>I, GD40 and GD41 for <italic>ori</italic>II, GD67 and GD68 for <italic>oriC</italic>, and GD150 and GD151 for <italic>&#916;oriC::oriI</italic>. The primers for external markers on the three chromosomes were GD38 and GD39, GD42 and GD43, and GD128 and GD129, respectively. The probes for <italic>ori</italic> and the external markers were made radioactive using the RediPrimeII random primer labeling kit (GE Healthcare) and [&#945;-<sup>32</sup>P] dCTP (PerkingElmer) and mixed separately for the two chromosomes. The band intensities were recorded and quantified as described earlier <xref ref-type="bibr" rid="pgen.1000939-Srivastava2">[46]</xref>.</p><p>Marker frequency was determined by qPCR using a PTC-200 Peltier Thermal Cycler (MJ Research) and a LightCycler 480 SYBR Green I Master (Roche). Genomic DNA was prepared from log phase cultures in LB with Genelute Bacterial Genomic DNA kit (Sigma), and 313 pg was used in each reaction as template. The primers were used at 0.3 &#181;M each. They were proximal to either <italic>oriI</italic> (GD136 and GD137) or <italic>oriII</italic> (GD156 and GD157) or <italic>terI</italic> (GD142 and GD143) or <italic>terII</italic> (GD140 and GD141) region of the two chromosomes, and were identical to those described <xref ref-type="bibr" rid="pgen.1000939-Rasmussen1">[39]</xref>. The primer pairs were such that they produced &#8764;100 to 130 bp fragments in all cases. Cp (crossing point) values were determined and used for calculating the <italic>oriI/oriII</italic>, <italic>oriI/terI</italic>, <italic>oriII/terII</italic> and <italic>terI/terII</italic> ratios. The ratios were normalized to those of a culture grown to stationary phase in supplemented M63 medium (without casamino acids). Mean ratios were obtained from DNA prepared from three cultures, each grown from independent colonies, and each DNA was analyzed in triplicate.</p><p>The method was modified from the one described by Lin and Grossman <xref ref-type="bibr" rid="pgen.1000939-Lin1">[51]</xref>. Briefly, cultures at OD<sub>600nm</sub>&#8202;=&#8202;0.3 were treated with 1% formaldehyde at room temperature for 30 min. After cell lysis and sonication, RctB complexes were precipitated with antibody against RctB (IP DNA) and Dynabeads-Protein G magnetic beads (Invitrogen), followed by stringent washings (see <xref ref-type="supplementary-material" rid="pgen.1000939.s005">Text S1</xref> for the detailed ChIP protocol). After reversal of the cross-links by incubation at 65&#176;C overnight, the samples were treated by protease K (Sigma) and then purified with a PCR purification Kit (Qiagen). To quantify the enrichment of RctB binding sites in the IP DNA, 5 &#181;l of 1&#8758;100 dilution of the IP DNA was used to perform locus-specific real-time qPCR with primers GD218 and GD219, specific to the vector backbone of the plasmid carrying the RctB binding sites, and primers GD191 and GD192, specific to a gene in the <italic>E. coli</italic> genome that served as a reference, as described <xref ref-type="bibr" rid="pgen.1000939-Lee1">[52]</xref>.</p>