<p>Fission yeast strain genotypes are listed in <xref ref-type="supplementary-material" rid="pgen.1001032.s007">Table S2</xref>. Standard fission yeast methods were used as described previously <xref ref-type="bibr" rid="pgen.1001032-Forsburg1">[39]</xref>. Gene deletions and epitope-tagging were performed as described <xref ref-type="bibr" rid="pgen.1001032-Bhler1">[105]</xref>. For synchronization of cells using <italic>cdc25-22</italic> block and release, cells containing the temperature sensitive <italic>cdc25-22</italic> allele were incubated at restrictive temperature (36&#176;C) for 4 hours to arrest the cell cycle in G2 <xref ref-type="bibr" rid="pgen.1001032-Forsburg1">[39]</xref>. Upon release to permissive temperature (25&#176;C), the cells synchronously enter the cell cycle. Progression into S-phase was monitored microscopically by counting cells that contained septa, the appearance of which correlates with S-phase <xref ref-type="bibr" rid="pgen.1001032-Forsburg1">[39]</xref>. Cells were fixed for ChIP experiments when the septation index was between 60&#8211;80%. Ectopic expression of <italic>pRep41-N-GFP-brc1<sup>+</sup></italic> for microscopy and ChIP analysis was under the control of the thiamine-repressible <italic>nmt41</italic> promoter. Induction of plasmid expression was performed in selective medium in the absence of thiamine for 18&#8211;20 hours.</p><p>For &#947;H2A and H2A Western blotting, histone-enriched protein extracts were prepared as previously described in <xref ref-type="bibr" rid="pgen.1001032-Redon1">[33]</xref>. Polyclonal anti-&#947;H2A antibody (courtesy of C. Redon) was used for &#947;H2A detection. Polyclonal anti-H2A antibody (07&#8211;146, Millipore) was used for H2A detection. Images were acquired on the Odyssey Infrared Imaging System (LI-COR Biosciences).</p><p>Cells were grown in YES liquid medium at 30&#176;C, washed once in 1x PBS and imaged live for Brc1-GFP foci, or fixed overnight in cold 70% ethanol, washed once in 1x PBS, and mixed with 500 ug/ml DAPI for analysis of chromosome segregation defects. Cells were photographed using a Nikon Eclipse E800 microscope equipped with a Photometrix Quantix charge-coupled device camera. Error bars represent the standard deviation between 3 independent experiments, or range for 2 independent experiments.</p><p>ChIP experiments were performed as described <xref ref-type="bibr" rid="pgen.1001032-Du1">[20]</xref> with the following modifications. Cleared lysates were incubated with protein G Dynabeads (Invitrogen) pre-bound to anti-&#947;H2A antibody (Courtesy of C.Redon) or anti-GFP antibody (Roche). DNA was purified using the QIAquick PCR Purification Kit (Quiagen). Quantitative PCR was performed on input and ChIP samples using the Chromo4 Real-Time PCR Detection System (Bio-Rad) using the iQ SYBR Green Supermix (Bio-Rad). QPCR primers are listed in <xref ref-type="supplementary-material" rid="pgen.1001032.s008">Table S3</xref> with the exception of the subtelomere primers, which are from <xref ref-type="bibr" rid="pgen.1001032-Gomez1">[70]</xref> and tDNA primers, which are from <xref ref-type="bibr" rid="pgen.1001032-Pebernard1">[87]</xref>. Percent of immunoprecipitated DNA (%IP) in the ChIP samples was calculated relative to the amount of DNA in the input samples. ChIP fold enrichment was calculated relative to actin. All error bars represent the standard error between experimental triplicates.</p><p>Input and ChIP DNA samples were prepared using standard Affymetrix protocols and hybridized on Affymetrix <italic>S. pombe</italic> tiling arrays FR1.0. See Supporting Information (<xref ref-type="supplementary-material" rid="pgen.1001032.s009">Text S1</xref>) for description of MAT score and other methods.</p>