<p>This study was conducted according to the principles expressed in the Declaration of Helsinki. The study was approved by the Institutional Review Board of Dana-Farber Cancer Institute. All patients provided written informed consent for the collection of samples and subsequent analysis.</p><p>A total of 180 patients treated with radical prostatectomy (RP) for prostate cancer and 92 patients treated for colon cancer consented to provide tissue. Additionally, histologically normal breast tissue from 56 from women undergoing cosmetic reduction surgery was analyzed in the present study.</p><p>Fresh frozen radical prostatectomy specimens were available from 121 subjects at the Dana-Farber Cancer Institute (DFCI) and Brigham and Women's Hospital (Boston, MA) and were reviewed by a pathologist (J.C. or R.F.). Over 95% of the patients in the cohort were diagnosed with Gleason 6 or Gleason 7 disease and median PSA was 5.1 ng/ml. Areas of tumor consisted of &gt;60% tumor cells and areas of benign tissue consisted of &gt;80% non-neoplastic epithelial cells at least 5 mm away from any tumor focus. Biopsy cores of fresh frozen tissue were processed for RNA extraction using a modified Qiagen Allprep DNA/RNA protocol.</p><p>Archival FFPE blocks were available for 59 men with prostate cancer enrolled in the Physicians&#8217; Health Study (PHS) <xref ref-type="bibr" rid="pgen.1001204-Gann1">[22]</xref>, <xref ref-type="bibr" rid="pgen.1001204-Setlur1">[40]</xref>. These men were diagnosed with prostate cancer between 1983 and 2003 and treated by radical prostatectomy. RNA were extracted from paraffin-embedded tumor tissue as described previously <xref ref-type="bibr" rid="pgen.1001204-Setlur1">[40]</xref>. Areas of tumor consisted of &gt;90% tumor cells.</p><p>Fresh frozen colorectal cancer tissue samples were reviewed by a pathologist (J.C.) and areas of benign tissue were selected where 80% of cells consisted of non-neoplastic epithelium. RNA was extracted using a modified Qiagen Allprep DNA/RNA protocol. Fresh frozen normal breast tissue samples were reviewed to identify tissue blocks containing &gt;40% normal epithelial cells. RNA was extracted using a modified Qiagen RNeasy protocol.</p><p>Ethnicity was self-reported by most, but not all, subjects. Subjects in the DFCI cohort of unknown ancestry were genotyped for 59 ancestry-informative SNPs in order to ascertain ethnicity. The marker set primarily captured ancestral differences between European and African ancestries (D. Reich, personal communication). Five samples found to be from subjects of African ancestry were excluded from analysis.</p><p>The human tissues analyzed in this study were from patients treated at Brigham and Women&#8217;s Hospital, Dana-Farber Cancer Institute or Vall d'Hebron University Hospital in Barcelona, Spain, all of whom provided informed consent. The study was approved by the Institutional Review Board at Dana-Farber Cancer Institute.</p><p>cDNA was prepared for expression analysis using Invitrogen SuperScript III Reverse Transcription kit. DFCI prostate samples, colon samples and breast samples were analyzed via competitive RT-PCR using Sequenom iPLEX matrix-assisted laser desorption/ionization (MALDI)-time of flight mass spectrometry technology. Expression levels of two <italic>MSMB</italic> isoforms and five <italic>NCOA4</italic> isoforms were measured. These splice variants represent all isoforms reported in Ensembl genebuild 52. RNA expression of <italic>TIMM23</italic> and three housekeeping genes (ACTB, MYL6 and RPL13A) were also measured. Primer, probe and competitor oligo sequences are available upon request. Reactions were performed in quadruplicate using 8 serial dilutions of competitor, and the EC50 was calculated using QGE Analyzer software (Sequenom). The PHS subgroup was analyzed using Illumina cDNA-mediated Annealing, Selection, Extension and Ligation (DASL) expression assay (Illumina).</p><p>A gene expression normalization factor was calculated using the geometric mean of expression level of the three housekeeping genes. Linear regression was used to assess whether expression levels increased (or decreased) as the number of T-alleles of rs10993994 increased; for prostate tissue, differences between prostate tumor and normal tissue levels were also assessed and random effects linear regression was used to account for within-sample correlation of tumor/normal pairs.</p><p>Genotyping of DNA from each subject was carried out using Sequenom iPLEX matrix-assisted laser desorption/ionization (MALDI)-time of flight mass spectrometry technology.</p><p>LHSAR: Prostate epithelial cells (PrECs) immortalized with hTERT, SV40 Large T and small t antigens and overexpressing androgen receptor were grown in PREGM (Lonza CC-3166). HMLE: Human mammary epithelial cells immortalized with hTERT, Large T and small t antigens were grown in MEGM (Lonza CC-3150). All growth media were supplemented with 100 ug/ml Penicillin/Streptomycin.</p><p>The bottom layer contained 0.6% agar (Sigma A5431) in DMEM and 8% FBS. The top layer contained 0.3% agar in PREGM or MEGM for LHSAR or HMLE respectively. Fifteen thousand cells were seeded in the top agar layer in triplicate wells of a 6 well plate. Colonies were counted from 2 to 6 weeks post-seeding. Image of each well was taken at a 6&#215; magnification and analyzed with Image J software. Colonies that were 50 sq. pixels or larger were counted.</p><p>Qiagen RNeasy kit was used for RNA extraction. Reverse transcription was carried out with Clonetech Advantage RT-to-PCR kit while the quantitative PCR was carried out using SYBR Green Master Mix (Applied Biosystems). Two sets of <italic>NCOA4</italic> and one set of <italic>MSMB</italic> primers were used:</p><p>NCOA4 exon 3-4 - Forward <named-content content-type="gene">CAGCAGCTCTACTCGTTATTGG</named-content>
</p><p> Reverse <named-content content-type="gene">TCTCCAGGCACACAGAGACT</named-content>
</p><p>NCOA4 exon 5-6 - Forward <named-content content-type="gene">CTCTCAAAACCATTCAAATTCCT</named-content>
</p><p> Reverse <named-content content-type="gene">CTCTGGCATGGAGATACAGC</named-content>
</p><p>MSMB exon 1-2 - Forward <named-content content-type="gene">GCTTATCACAATGAATGTTCTCCT</named-content>
</p><p> Reverse <named-content content-type="gene">AATCTCCTGGAACTCCCTCA</named-content>
</p><p>
<italic>NCOA4</italic> expression constructs were received from the human ORFeome V5.1 and cloned into pWZL-Neo retroviral expression vector. Retrovirus production, infection and selection were carried out as described previously <xref ref-type="bibr" rid="pgen.1001204-Boehm1">[41]</xref>.</p><p>Short hairpins in pLKO.1 lentiviral constructs were received from the RNAi Consortium (TRC). Lentivirus production and infection were carried out as described previously <xref ref-type="bibr" rid="pgen.1001204-Moffat1">[42]</xref>.</p><p>
<underline>Hairpin</underline> shMSMB M6 <underline>TRC Clone ID</underline> TRCN0000147237</p><p>
<underline>Target sequence</underline>
<named-content content-type="gene">GTTCTGTCAGTGAATGGATAA</named-content>
</p><p>
<underline>Hairpin</underline> shMSMB M8 <underline>TRC Clone ID</underline> TRCN0000146396</p><p>
<underline>Target sequence</underline>
<named-content content-type="gene">CACCTTCGTGACTTTATGCAA</named-content>
</p><p>
<underline>Hairpin</underline> shMSMB M9 <underline>TRC Clone ID</underline> TRCN0000146343</p><p>
<underline>Target sequence</underline>
<named-content content-type="gene">CAAAGGAAACAAACACCCAAT</named-content>
</p>