<p>All <italic>S. cerevisiae</italic> strains used in this study are isogenic to E134 (<italic>MAT&#945; ade5-1 lys2::InsEA14 trp1-289 his7-2 leu2-3,112 ura3-52</italic>) <xref ref-type="bibr" rid="pgen.1001209-Shcherbakova2">[33]</xref>. The <italic>dpb3&#916;</italic> mutant was kindly provided by P. Shcherbakova and is described in <xref ref-type="bibr" rid="pgen.1001209-Northam1">[27]</xref>. Other strains carrying <italic>dpb3&#916;</italic> were obtained as described in <xref ref-type="bibr" rid="pgen.1001209-Northam1">[27]</xref>. The <italic>dpb4&#916;</italic> mutants were constructed by transformation with PCR fragment carrying the <italic>hygB</italic> selectable marker and obtained using primers DPB4/kanMX-F (<named-content content-type="gene">5&#8242;-ATGCCACCAAAAGGTTGGAGAAAAGACGCCCAAGGGAATTACCCCCGTACGCTGCAGGTCGAC</named-content>) and DPB4/kanMX-R (<named-content content-type="gene">5&#8242;-TTACGTTTGCTCAAGGTTTTGAACTCTAGTTTCTACATCTTGGCTATCGATGAATTCGAGCTCG</named-content>) and the pAG32 plasmid as a template <xref ref-type="bibr" rid="pgen.1001209-Goldstein1">[63]</xref>. The disruption was confirmed by PCR analysis. The <italic>pol2-4</italic> mutation was obtained as described in <xref ref-type="bibr" rid="pgen.1001209-Pavlov4">[64]</xref> using YIpJB1 plasmid carrying <italic>pol2-4</italic> mutation <xref ref-type="bibr" rid="pgen.1001209-Morrison2">[65]</xref>. The <italic>pol3-5DV</italic> mutation was obtained as described in <xref ref-type="bibr" rid="pgen.1001209-Pavlov4">[64]</xref> using plasmid p170-5DV <xref ref-type="bibr" rid="pgen.1001209-Jin4">[66]</xref>.The presence of the <italic>pol2-4</italic> mutation after integration into the chromosome was confirmed by SfcI digest of short PCR fragment encompassing mutation and DNA sequencing. Deletion of the <italic>MSH6</italic> gene was obtained as described in <xref ref-type="bibr" rid="pgen.1001209-Pavlov5">[67]</xref>. The <italic>REV3</italic> gene, encoding the catalytic subunit of Pol &#950;, was deleted as described in <xref ref-type="bibr" rid="pgen.1001209-Shcherbakova4">[68]</xref>. Deletion of the <italic>MSH2</italic> gene was obtained by transformation with PCR product obtained from the pRS305 plasmid using oligos MSH2_del_F (<named-content content-type="gene">CTCCACTAGGCCAGAGCTAAAATTCTCTGATGTATCAGAGGAGAGCAGAGCAGATTGTACTGAGAGTGCACC</named-content>) and MSH2_del_R (<named-content content-type="gene">CCTTCACTTTTCTAATCCACTCTTTCAGTAAAGCCTTCAAACGAACGCATCTGTGCGGTATTTCACACCGC</named-content>). The same strategy was used for deletion of the <italic>PMS1</italic> and <italic>MLH1</italic> gene. To obtain pms1&#916; we used oligos PMS1_A (<named-content content-type="gene">TATCAAAGCTAGATCATATTTCGTAATCCTTCGAAAATGAGCTCCAATCACGTAAAATATCTTGACCGCAGTTAA</named-content>) and PMS1_S (<named-content content-type="gene">AAGGTGTAAGCAAAAGGAACAGAGGTATATCCCTGTGAAATATTTATTTAGCCCCTATGAACATATTCCATT</named-content>). The <italic>mlh1&#916;</italic> was obtained by transformation with the PCR product obtained from the pRS306 plasmid using oligos MLH1_A (<named-content content-type="gene">AAGTTAACACCTCTCAAAAACTTTGTATAGATCTGGAAGGTTGGCTATTTCCAACACCGCAGGGTAATAACTGAT</named-content>) and MLH1_S (<named-content content-type="gene">ATACGATAGTGATAGTAAATGGAAGGTAAAAATAACATAGACCTATCAATAAGCACGGTCACAGCTTGTCTGTAA</named-content>).</p><p>The fluctuation tests to determine spontaneous mutation rates were, unless otherwise indicated, performed in two to five independent experiments of nine independent cultures each with independently obtained derivatives. Single two-day-old colonies from YPD plates were inoculated in 5 ml of liquid YPD medium and were grown with strong aeration for two days and processed as described <xref ref-type="bibr" rid="pgen.1001209-Shcherbakova2">[33]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Independent His<sup>+</sup> revertants and Can<sup>r</sup> mutants were grown as small patches on YPD plates. Regions of corresponding genes were amplified by PCR. Amplified DNAs were purified by QIAGEN PCR purification kit and sequenced by MWG Biotech (<ext-link ext-link-type="uri" ns0:href="http://www.mwgdna.com">www.mwgdna.com</ext-link>). <italic>CAN1</italic> spectra obtained in four strains were compared using several statistical techniques. A Monte Carlo modification of the Pearson <bold>&#967;<sup>2</sup></bold> test of spectra homogeneity <xref ref-type="bibr" rid="pgen.1001209-Adams1">[69]</xref> was used to compare 2 x N tables (two mutation spectra, N&#8805;2). Small probability values (P&#8804;0.05) indicate a significant difference between two spectra. Calculations were done using the program COLLAPSE <xref ref-type="bibr" rid="pgen.1001209-KhromovBorisov1">[70]</xref>.</p><p>All purification steps were carried out as described in <xref ref-type="bibr" rid="pgen.1001209-Asturias1">[32]</xref>.</p><p>We used primer 3NY (<named-content content-type="gene">5&#8242;-AGGTCACGATGCGGCATAGCCTGCATTGATCGCACGATGATCAGCGGACTGCTTACC</named-content>) annealed to the template 19wt (<named-content content-type="gene">3&#8242;-TCCAGTGCTACGCCGTATCGGACGTAACTAGCGTGCTACTAGTCGCCTGACGAATGGACAGTGCCATTGTCACTG</named-content>) as a substrate for the exonuclease reaction. The primer (8 &#181;M) was labeled with 40 &#181;Ci of [&#947;-<sup>32</sup>P]ATP in a 20-&#181;l reaction with 10 U T4 polynucleotide kinase (Promega) for 1 h. The reaction was stopped with EDTA and labeled products were purified through PAGE. The end-labeled primer was annealed to the template at 1.5: 1 ratio for a 5 minute incubation at 80&#176;C followed by slow cooling to room temperature. For the exonuclease assay 0.5 nM substrate was incubated with 0.1 nM Pol &#949; or 0.5 nM Pol2/Dpb2 complex in a 65 &#181;l reaction mixture (40 mM Tris-HCl pH 7.8; 1 mM DTT; 0.2 mg/ml Ac-BSA; 8 mM MgCl<sub>2</sub>; 125 mM NaAc). Fifteen &#181;l aliquots were taken at the indicated time points and were mixed with 8 &#181;l stop solution (80% formamide; 50 mM EDTA; 1 mM bromophenol blue). Before loading the reactions on a 12% polyacrylamide-urea gel, the primer-templates were denatured at 99&#176; C for 4 min and then cooled on ice. The intensity of bands corresponding to different exonuclease products was quantified using phosphoimager plates and the ImageQuant software package supplied with a Typhoon 9400 phosphoimager (Amersham Biosciences).</p><p>For primer extension assay a [&#947;-<sup>32</sup>P]ATP &#8211;labeled (as described previously) 50-mer oligonucleotide was annealed to the pBluescript II SK(+) ssDNA in a ratio of 1&#8758;1.5. For the DNA synthesis processivity assay, the substrate (14 nM) was incubated with the four-subunit Pol &#949; (0.35 nM) or Pol2/Dpb2 complex (0.7 nM) in a reaction mixture (40 mM Tris-HCl pH 7.8; 1 mM DTT; 0.2 mg/ml Ac-BSA; 8 mM MgCl2; 125 mM NaAc; 100 &#181;M dNTP). Because the loading efficiency of the Pol2/Dpb2 complex on DNA was compromised we used a two and five-fold higher concentration of the Pol2/Dpb2 complex, compared to the full-subunit Pol &#949;, in the primer-extension assay and exonuclease assay, respectively. The conditions were empirically determined to meet single-hit kinetics, i.e. where a polymerase molecule never re-associated with a previously extended primer.</p><p>In the primer-extension assay, the termination probability at position N at each primer/template was calculated by dividing the intensity of the band N by the intensity of all bands &#8805; N. In the exonuclease processivity assay, the termination probability at position N at each primer/template was calculated by dividing the intensity of the band N by the intensity of all bands &#8804; N <xref ref-type="bibr" rid="pgen.1001209-Kokoska2">[71]</xref>.</p><p>DNA synthesis fidelity was measured using the bacteriophage M13mp2 forward mutation assay described previously <xref ref-type="bibr" rid="pgen.1001209-Bebenek2">[39]</xref>, <xref ref-type="bibr" rid="pgen.1001209-Shcherbakova3">[40]</xref>. Briefly, double-stranded M13mp2 DNA with a 407-nucleotide single-stranded region containing a portion of the <italic>lacZ</italic> gene was used as a substrate for <italic>in vitro</italic> DNA synthesis. Reactions mixtures contained &#8764;1.5 nM DNA template, 50 mM Tris-Cl (pH 7.5), 2 mM DTT, 100 &#181;g/ml BSA, 10% glycerol, 250 &#181;M dNTPs and 14 nM wild type or exonuclease-deficient 2-subunit Pol &#949;. Reactions were incubated at 30&#176;C for 30 min. Aliquots of the reactions were analyzed by agarose gel electrophoresis to confirm complete gap-filling, and another aliquot of DNA was introduced into <italic>E.coli</italic> to score the frequency of light blue and colorless plaques reflecting errors made during <italic>in vitro</italic> DNA synthesis. Single stranded DNA was isolated from independent mutant M13 plaques and the <italic>lacZ</italic> gene was sequenced. Error rates (ER) for individual types of mutation were calculated according to the following equation: ER &#8202;=&#8202; [(N<sub>i</sub>/N)&#215;MF]/(D&#215;0.6) where N<sub>i</sub> is the number of mutations of a particular type, N is the total number of mutants analyzed, MF is frequency of <italic>lacZ</italic> mutants, D is the number of detectable sites for the particular type of mutation, and 0.6 is the probability of expressing a mutant <italic>lacZ</italic> allele in <italic>E. coli</italic>.</p>