<p>CoLaus is a population-based sample from Lausanne, Switzerland, consisting of 5435 individuals between 35 and 75 years old (after QC) of which a subset of 5404 had available serum calcium measurements. The study design and protocols have been described previously <xref ref-type="bibr" rid="pgen.1001035-Firmann1">[17]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1001035-Rodondi1">[18]</xref>. The CoLaus study was approved by the Institutional Ethic's Committee of the University of Lausanne. The London Life Sciences Prospective Population Study (LOLIPOP) is an ongoing population-based cohort study of &#8764;30,000 Indian Asian and European white men and women, aged 35&#8211;75 years living in West London, United Kingdom <xref ref-type="bibr" rid="pgen.1001035-Chambers2">[59]</xref>. All study participants gave written consent including for genetic studies. The LOLIPOP study is approved by the local Research Ethics Committee. The participants included in the present study are a subset of 3947 Indian Asians and 1601 Europeans from the LOLIPOP cohort study. LOLIPOP individuals are separated by origin as well as the genotyping platform, with IAA, IAI or IAP denoting Indian Asians genotyped on Affymetrix, Illumina or Perlegen platforms, respectively, and EWA and EWI denoting Europeans genotyped on Affymetrix or Illumina platforms, respectively (see <xref ref-type="supplementary-material" rid="pgen.1001035.s005">Table S1</xref>). The InCHIANTI study is a population-based epidemiological study aimed at evaluating the factors that influence mobility in the older population living in the Chianti region in Tuscany, Italy. The details of the study have been previously reported <xref ref-type="bibr" rid="pgen.1001035-Ferrucci1">[60]</xref>. Overnight fasted blood samples were taken for genomic DNA extraction, and measurement of serum calcium. For this study, 1196 subjects with serum calcium and GWAS data were analyzed. The study protocol was approved by the Italian National Institute of Research and Care of Aging Institutional Review and Medstar Research Institute (Baltimore, MD). The Baltimore longitudinal study on Aging (BLSA) study is a population-based study aimed to evaluate contributors of healthy aging in the older population residing predominantly in the Baltimore-Washington DC area <xref ref-type="bibr" rid="pgen.1001035-Shock1">[61]</xref>. Starting in 1958, participants are examined every one to four years depending on their age. Blood samples were collected for DNA extraction. This analysis focused on a subset of the participants (N&#8202;=&#8202;717) of European ancestry. The BLSA has continuing approval from the Institutional Review Board (IRB) of Medstar Research Institute. Approval was obtained from local ethic committees for all studies and all participants signed informed written consent. The deCODE study consists of individuals who visited a private outpatient laboratory, the Laboratory in Mjodd, Reykjavik, Iceland between 1997 and 2008. The main referral center for this laboratory is a multispecialty medical clinic in Reykjavik (Laeknasetrid). For the serum calcium analysis we used information on 4,126 individuals with both genome-wide SNP data and measured serum calcium and serum albumin. The samples for bone density analysis have previously been described in detail <xref ref-type="bibr" rid="pgen.1001035-Styrkarsdottir1">[24]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1001035-Styrkarsdottir2">[25]</xref>. For this study 6,657 individuals with total hip bone mineral density (BMD) and 6,838 individuals with lumbar spine BMD and SNP data were available for analysis. All participants gave informed consent and the study was approved by the Data Protection Commission of Iceland (DPC) and the National Bioethics Committee of Iceland.</p><p>For each CoLaus participant a venous blood sample was collected under fasting conditions. Measurements were conducted using a Modular P apparatus (Roche Diagnostics, Switzerland). Total serum calcium was measured by O-cresolphtalein (2.1% &#8211; 1.5% maximum inter and intra-batch CVs); albumin was measured by bromocresol green (2.5% &#8211; 0.4%). To further characterize the identified genetic variants, we analyzed the association with several outcomes postulated to be correlated with serum calcium. Within the CoLaus study, we have questionnaire responses to queries about personal histories of osteoporosis, osteoarthritis, myocardial infarction and stroke in addition to clinical data determining hypertension status, defined as previously described <xref ref-type="bibr" rid="pgen.1001035-Firmann1">[17]</xref>. The assessment of LOLIPOP study participants was carried out by a trained research nurse, during a 45 minute appointment according to a standardized protocol and with regular QC audits. An interviewer-administered questionnaire was used to collect data on medical history, family history, current prescribed medication, and cardiovascular risk factors. Physical assessment included anthropometric measurements (height, weight, waist, hip) and blood pressure. Blood was collected after an 8 hour fast for biochemical analysis, including glucose, insulin, total and HDL cholesterol and triglycerides, and whole blood was taken for DNA extraction <xref ref-type="bibr" rid="pgen.1001035-Chambers2">[59]</xref>. InCHIANTI serum albumin concentrations were determined as percentage of total protein using agarose electrophoretic technique (Hydragel Protein (E) 15/30, Sebia, Issy-les-Moulineaux, France). Serum calcium was measured using calorimetric assay (Roch Diagnostic, GmbH, Mannheim, Germany) by a Roche-Hitachi autoanalyzer (The intra-assay CV and 0.9% and the inter-assay CV was 1.5%). Measures of bone density, bone dimensions and osteoporosis diagnosis were assessed by peripheral quantitative computed tomography (pQCT) using the XCT 2000 device (Stratec Medizintechnik, Pforzheim, Germany) <xref ref-type="bibr" rid="pgen.1001035-Russo1">[62]</xref>. BLSA albumin concentrations were measured by a calorimetric assay using bromocresol green (Ortho-Clinical Diagnostics). Calcium concentrations were measured by a calorimetric assay (Vitros 5,1,FS).</p><p>CoLaus participants were genotyped using Affymetrix Human Mapping 500 K Array. For the genome-wide association stage, genotyping in LOLIPOP participants was carried out using the Illumina 317 K mapping array, Affymetrix Human Mapping 500 K array, and Perlegen, 284 K platforms (<xref ref-type="supplementary-material" rid="pgen.1001035.s005">Table S1</xref>). Participants of the InCHIANTI and BLSA studies were genotyped using Illumina Infinium HumanHap 550 K SNP arrays were used for genotyping <xref ref-type="bibr" rid="pgen.1001035-Melzer1">[21]</xref>. Imputation of allele dosage of SNPs was performed using either MACH <xref ref-type="bibr" rid="pgen.1001035-Li2">[63]</xref> or IMPUTE <xref ref-type="bibr" rid="pgen.1001035-Marchini1">[64]</xref> with parameters and quality control filters as described in . All European cohorts imputed SNPs typed in the HapMap CEU population; LOLIPOP Indian Asian cohort imputed SNPs using mixed HapMap populations, given that this showed greater concordance with real genotypes compared with use of any one HapMap population. SNPs were excluded if cohort-specific imputation quality as assessed by r2.hat (MACH) or .info (IMPUTE) metrics were &lt;0.30. In total, 2,557,252 genotyped or imputed SNPs had data from one or more cohorts and were analyzed. Genotypes in deCODE were measured using either humanHap300, humanHap300-duo or humanCNV370.</p><p>Biologically active serum calcium is estimated by the correction, Ca_corrected &#8202;=&#8202; total serum calcium [mmol/L] + (40 - albumin [g/L])/40. Individuals with values &lt;1.9 or &gt;3 were removed as these were extreme outliers. Linear-regression analyses were carried out using an additive genetic model on log10-transformed corrected calcium levels adjusted for age and pseudosex (a factor variable with three values: males, pre-menopausal females and post-menopausal females). BLSA also included the first two and LOLIPOP included the first four ancestry principal components in the regression, respectively. Regression analyses were performed with QUICKTEST <xref ref-type="bibr" rid="pgen.1001035-Johnson1">[65]</xref> (CoLaus), MACH2qtl (LOLIPOP) <xref ref-type="bibr" rid="pgen.1001035-Li2">[63]</xref> or MERLIN <xref ref-type="bibr" rid="pgen.1001035-Abecasis1">[66]</xref> (InCHIANTI, BLSA).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The results from all cohorts were combined into a fixed-effects meta-analysis using inverse variance weighting. Tests for heterogeneity were assessed using Cochran's Q statistic and the log of the related H statistic <xref ref-type="bibr" rid="pgen.1001035-Higgins1">[67]</xref> after grouping LOLIPOP subsets into European and Indian subsets. For rs1801725 and rs1042636 the p-values were (0.07657, 0.1432) and (0.3450, 0.8876), respectively, indicating limited between-study variability. The analysis was implemented in R and run on a quad-core Linux machine. SNPs were reported provided they had effect size estimates in at least 2 of the 5 European cohorts, in at least 2 of the 3 Indian Asian cohorts, or in at least 3 of the 8 total cohorts. For the overall meta-analysis, residual inflation of the test statistic was corrected using genomic control <xref ref-type="bibr" rid="pgen.1001035-Bacanu1">[68]</xref>. The inflation factor was 1.0207 for the all combined cohorts, 1.0068 for European cohorts and 1.0286 for Indian Asian cohorts. Where reported, individual study p-values are corrected for inflation using genomic control methods for genotyped and imputed SNPs combined (inflation factors for individual studies were 1.0139 (CoLaus), 0.9891 (LOLIPOP EWA), 0.9994 (LOLIPOP EWP), 0.9967 (LOLIPOP IAA), 1.0131 (LOLIPOP IAI), 0.9985 (LOLIPOP IAP), 0.9842 (InCHIANTI), 1.0019 (BLSA)). The regional association plot (<xref ref-type="fig" rid="pgen-1001035-g003">Figure 3</xref>) was created modifying a publically available R script <xref ref-type="bibr" rid="pgen.1001035-Saxena1">[69]</xref>. The map of fine-scale recombination rates was downloaded from the HapMap website <ext-link ext-link-type="uri" ns0:href="http://www.hapmap.org/downloads/recombination/">http://www.hapmap.org/downloads/recombination/</ext-link> using Phase II HapMap data (release 21). Quantile-quantile plots of the association results are shown in <xref ref-type="supplementary-material" rid="pgen.1001035.s001">Figure S1A, S1B, S1C</xref>, study-specific quantile-quantile plots are shown in <xref ref-type="supplementary-material" rid="pgen.1001035.s002">Figure S2</xref>. Associations below p&#8202;=&#8202;5&#215;10<sup>&#8722;8</sup> were considered genome-wide significant, which corresponds to a Bonferroni correction for the estimated one million independent common variant tests in the human genome of European individuals <xref ref-type="bibr" rid="pgen.1001035-Dudbridge1">[70]</xref>. The analysis of osteoporosis status in CoLaus and InCHIANTI was performed using logistic regression including age and pseudosex as covariates in QUICKTEST <xref ref-type="bibr" rid="pgen.1001035-Johnson1">[65]</xref>. Linkage disequilibrium was estimated from HapMap CEU (2007-01, build 35 non-redundant) genotypes. LD r<sup>2</sup> statistics were estimated for SNPs within 500 kb using Haploview <xref ref-type="bibr" rid="pgen.1001035-Barrett1">[71]</xref>.</p><p>For each related trait (coronary heart disease, hypertension, kidney stones, myocardial infarction, osteoarthritis, osteoporosis and stroke) we performed a fixed-effects meta-analysis of the logistic regression coefficients. We applied the bonferroni correction to adjust for multiple testing. We performed Wald-based power calculations using a type I error of 0.05/7 and meta-analysis coefficient estimates and standard errors to estimate the sample size for each trait giving power 0.80.</p>