<p>Each of the cohorts collected written informed consent from study participants and received approval from their respective Institutional Review Boards.</p><p>The CHARGE Consortium was established to facilitate meta-analysis of GWAS for traits related to cardiovascular disease (CVD) and aging <xref ref-type="bibr" rid="pgen.1001045-Psaty1">[40]</xref>. Briefly, five large, population-based cohort studies from the United States and Europe with genome-wide genotyping information available in 2007 to 2008 were included: AGES&#8212;Reykjavik, ARIC, CHS, FHS, and RS. Detailed information about each cohort is provided in other references (AGES&#8212;Reykjavik <xref ref-type="bibr" rid="pgen.1001045-Harris1">[41]</xref>; ARIC <xref ref-type="bibr" rid="pgen.1001045-The1">[42]</xref>; CHS <xref ref-type="bibr" rid="pgen.1001045-Fried1">[43]</xref>; FHS <xref ref-type="bibr" rid="pgen.1001045-Dawber1">[44]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1001045-Splansky1">[47]</xref>; RS <xref ref-type="bibr" rid="pgen.1001045-Hofman1">[48]</xref>) and is summarized below.</p><p>The AGES&#8212;Reykjavik Study includes a sample of 5,764 survivors from the Reykjavik Study of 30,795 men and women born between 1907 and 1935. The ARIC study includes 15,792 men and women aged 45 to 64 who were enrolled in a prospective follow-up study from four US communities from 1987 to 1989. The CHS includes 5,201 mostly Caucasian participants aged 65 years or older that were randomly sampled from Medicare lists in four US communities from 1989 to 1990. The FHS recruited 5,209 participants aged 28 to 62 from Framingham, Massachusetts beginning in 1948. Beginning in 1971, 5,124 offspring of the original cohort members and the offspring's spouses were also recruited as part of the Offspring Cohort. FHS subjects in this study are from the Offspring Cohort who attended the second examination in 1971&#8211;1973. Finally, the RS recruited 7,983 subjects aged 55 years or older from Ommoord, a suburb of Rotterdam, between 1990 and 1993. Only subjects self-reporting European ancestry from each cohort are included as a part of this study.</p><p>After conducting the meta-analysis within the CHARGE Study, genotype data on an additional 948 study participants of European ancestry became available within ARIC. These individuals were independent from the ones included in the discovery sample: they were not part of the discovery, had no first-degree relationship with any individual in the discovery sample, and would not have been classified as an outlier based on allele sharing measures generated during quality control procedures of the discovery sample.</p><p>The KORA Study is a series of independent population-based epidemiological surveys of participants living in the region of Augsburg, Southern Germany <xref ref-type="bibr" rid="pgen.1001045-Wichmann1">[49]</xref>. All survey participants were residents of German nationality identified through the registration office and were examined in 1994/95 (KORA S3) and 1999/2001 (KORA S4). In 2004/05, 3,006 subjects participated in a 10-year follow-up examination of S3 (KORA F3) and in 2006/08, 3,080 subjects participated in a 7-year follow-up examination of S4 (KORA F4). Individuals for genotyping in KORA F3 and KORA F4 were randomly selected. The age range of the participants was 25 to 74 years at recruitment.</p><p>The Study of Health in Pomerania (SHIP) is a cross-sectional survey in West Pomerania, the north-east area of Germany <xref ref-type="bibr" rid="pgen.1001045-John1">[50]</xref>. A sample from the population aged 20 to 79 years was drawn from population registries. Only individuals with German citizenship and main residency in the study area were included. Of 7,008 subjects sampled, 4,310 participants comprised the final SHIP population.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Details of genotyping methods, exclusion criteria, and imputation methods for the discovery and replication samples can be found in <xref ref-type="supplementary-material" rid="pgen.1001045.s003">Table S1</xref>. Briefly, SNPs were genotyped within each cohort from 2006&#8211;2008 using commercially available whole-genome platforms, and each cohort imputed genotypes to a common set of about 2.5 million autosomal SNPs. Imputation was carried out using MACH version 1.09/.15/.16 (AGES&#8212;Reykjavik, ARIC, FHS, KORA and RS) (accessed from <ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/MACH/">http://www.sph.umich.edu/csg/abecasis/MACH/</ext-link>), BimBam version 0.99 <xref ref-type="bibr" rid="pgen.1001045-Servin1">[51]</xref> software (CHS), or IMPUTEv0.5.0 <xref ref-type="bibr" rid="pgen.1001045-Marchini1">[52]</xref> (SHIP). For the imputation, genotype data from the individual studies was combined with genotype data from HapMap CEU samples to probabilistically infer the allelic dosage for each SNP (a fractional value from 0.0 to 2.0) based on the HapMap CEU haplotype structure. Imputation quality scores were calculated for each SNP as the ratio of observed dosage-variance to the expected binomial variance.</p><p>The primary outcomes for this study were serum concentrations of magnesium, potassium and sodium. We additionally evaluated the lead SNPs identified in association with other clinically-relevant phenotypes, including hypomagnesemia (defined as serum magnesium &lt;0.7 mmol/L; CHARGE), blood pressure (CHARGE), eGFR (CHARGE) fasting glucose (MAGIC), and BMD (GEFOS). Serum magnesium (discovery: ARIC, FHS, RS; replication: ARIC, KORA F3, KORA F4, SHIP), sodium (ARIC, FHS, RS), and potassium (ARIC, CHS, FHS, RS) concentrations were measured using standard protocols from fasting blood, where possible. Serum magnesium levels were determined using the method described by Gindler and Heth with metallochromic dye, Calmigate [1,-[1-hydroxy-4-methyl-2-phenylazo)-2-napthol-4-sulfonic acid] in the ARIC Study, by METPATH in FHS, with a Merck Diagnostica kit (method Xylidyl blue) on an Elan Autoanalyzer (Merk) in RS, with a Xylidylblue kit on a Modular analyzer (Roche) in the KORA Study, or using a commercial colorimetric test (Roche Diagnostics, Mannheim, Germany) with a Hitachi 717 autoanalyzer in the SHIP Study. Sodium and potassium levels were measured using standard ion electrode devices in all cohorts.</p><p>Detailed descriptions of blood pressure and eGFR traits are given in other references <xref ref-type="bibr" rid="pgen.1001045-Levy1">[24]</xref>, <xref ref-type="bibr" rid="pgen.1001045-Kottgen1">[25]</xref> and are described in brief here. Serum creatinine, used to calculate eGFR, was measured using a modified kinetic Jaffe method (ARIC, CHS, FHS, RS) or an enzymatic method (AGES&#8212;Reykjavik). Creatinine values were calibrated to age- and sex-adjusted mean values from a nationally representative study as described previously <xref ref-type="bibr" rid="pgen.1001045-Coresh1">[53]</xref>, and eGFR (ml/min/1.73 m<sup>2</sup>) was calculated using the 4-variable MDRD Study formula <xref ref-type="bibr" rid="pgen.1001045-Levey1">[54]</xref>. Due to the skewed distribution, a natural log transformation was applied before the association analyses. Repeated resting SBP and DBP measures were recorded by trained staff in all studies, and the average of multiple readings was used. Height and weight were measured by trained study personnel in all studies and were used to calculate BMI (kg/m<sup>2</sup>). Use of blood pressure medications was defined differently in the different cohorts, but for all cohorts, hypertension medication use was determined at the time of serum electrolyte determination and included all classes of anti-hypertension medications commonly prescribed at the time, including beta-blockers, diuretics, ACE-inhibitors, angiotensin type-2 antagonists, calcium-channel blockers, as well as combination therapies.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">SNPs were modeled as allelic dosages in all analyses. Genome-wide analyses of electrolyte concentrations (magnesium, potassium, and sodium) were conducted within the R package ProbABEL (<ext-link ext-link-type="uri" ns0:href="http://mga.bionet.nsc.ru/~yurii/ABEL/">http://mga.bionet.nsc.ru/~yurii/ABEL/</ext-link>) <xref ref-type="bibr" rid="pgen.1001045-Aulchenko1">[55]</xref> for ARIC, CHS and RS, or using linear mixed effects regression models in the R kinship package to account for pedigree structure in FHS. SNP-electrolyte associations were adjusted for age, sex, and study center, where applicable. For analyses of sodium and potassium concentrations subjects using any hypertension medications at the time of electrolyte assessment were excluded to avoid a possible influence of the medications on serum concentrations of sodium and potassium. Genomic control correction based on median chi-square was used within each study to adjust for inflation of the test statistics prior to meta-analysis, as well as applied to the combined results after the meta-analysis. Inverse-variance weighted fixed-effects meta-analyses were carried out by two independent analysts using the software METAL (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/metal/">www.sph.umich.edu/csg/abecasis/metal/</ext-link>) for the &#8764;2.5 million SNPs across ARIC, FHS, RS, and CHS (potassium only). After meta-analysis, results were filtered to remove SNPs with low minor allele frequency (&lt;0.01). Statistical heterogeneity was evaluated using Cochrane's <italic>&#967;<sup>2</sup></italic> test (Q-test). P-values &lt;5&#215;10<sup>&#8722;8</sup> were used to indicate genome-wide significant results. The size of the associated regions was determined using the positions of the most upstream and downstream regional SNPs with p-values&lt;5&#215;10<sup>&#8722;5</sup>. Manhattan and Q-Q plots were generated for the meta-analyzed data using the R statistical software package (<ext-link ext-link-type="uri" ns0:href="http://www.R-project.org">http://www.R-project.org</ext-link>). Plots of the &#8211;log<sub>10</sub>(p-values) by genomic position for associations within regions of statistical significance were generated using the SNAP program (<ext-link ext-link-type="uri" ns0:href="http://www.broad.mit.edu/mpg/snap/ldsearch.php">http://www.broad.mit.edu/mpg/snap/ldsearch.php</ext-link>). In SNAP, the HapMap CEU population was used as the reference group to map LD patterns. In the family-based FHS, heritability of serum magnesium, sodium, and potassium was estimated using age and sex-adjusted residuals in a variance components model that estimated additive genetic heritability and a random environmental component using SOLAR v.1.4 <xref ref-type="bibr" rid="pgen.1001045-Almasy1">[56]</xref>.</p><p>The six lead SNPs with evidence of genome-wide significant association in discovery plus an additional three SNPs with suggestive evidence of association were evaluated for independent replication. In the replication studies, SNP-magnesium associations were determined in linear regression models as described for the discovery cohorts. Inverse-variance weighted fixed effects meta-analysis was used to determine associations across the replication samples and to calculate the overall combined associations for the discovery and replication cohorts.</p><p>For the lead SNPs, we calculated the percent of magnesium variance attributable to the SNP as the difference in the adjusted r<sup>2</sup> value for a model containing the SNP, age, sex, and study center, where applicable, to a model containing only age, sex and study center, expressed as a percent. Assuming independent effects of the SNPs, we added the individual variance across the SNPs to calculate the total variance explained by the set of SNPs. The independence assumption was verified by simultaneous inclusion of all SNPs into a regression model. We also evaluated the five lead SNPs from CHARGE with evidence for replication in logistic models of hypomagnesemia (in ARIC and RS only because of small numbers of subjects with hypomagnesemia in FHS) or linear models of eGFR (ml/min/1.73 m<sup>2</sup>) adjusted for age, sex, and study center. Results for blood pressure (mm Hg) traits in association with the SNPs were adjusted for age, age squared, sex, and BMI to be consistent with the published data from a GWAS of blood pressure in the CHARGE Consortium, and blood pressure among treated and untreated individuals was modeled as described in this publication <xref ref-type="bibr" rid="pgen.1001045-Levy1">[24]</xref>, <xref ref-type="bibr" rid="pgen.1001045-Cui1">[57]</xref>. Inverse-variance weighted fixed effects meta-analysis was used to determine summary effect estimates for these additional traits.</p><p>We further evaluated these SNPs in association with fasting glucose and BMD as an <italic>in silico</italic> lookup in large available datasets from two consortia. Fasting glucose associations were available from up to 46,180 subjects of European descent from the MAGIC Consortium <xref ref-type="bibr" rid="pgen.1001045-Dupuis1">[15]</xref>, and BMD associations (femoral neck and lumbar spine) in 19,195 subjects of Northern European descent from the GEFOS Consortium <xref ref-type="bibr" rid="pgen.1001045-Rivadeneira1">[16]</xref>.</p><p>To examine the association between serum magnesium levels and common variation in previously identified magnesium transporter proteins from model systems <xref ref-type="bibr" rid="pgen.1001045-Cole1">[7]</xref>, <xref ref-type="bibr" rid="pgen.1001045-Goytain1">[28]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1001045-Shuen1">[34]</xref> or in genes with rare variants responsible for monogenic disorders of magnesium metabolism, we examined associations with SNPs within 60 kb of the genes <xref ref-type="bibr" rid="pgen.1001045-Dehghan1">[58]</xref> and reported the association and annotation for the lead SNP within each gene region.</p>