<p>We assume that in a family-based association study, <italic>n</italic> family members have been genotyped at <italic>m</italic> loci with a genome-wide SNP-chip. For each marker locus, a family-based association test is constructed based on the offspring phenotype and the within-family information. The within-family information is defined as the difference between the observed, genetic marker score and the expected, genetic marker score, which is computed conditional upon both the parental genotypes/sufficient statistic <xref ref-type="bibr" rid="pgen.1000741-Rabinowitz1">[19]</xref> under the assumption of Mendelian transmissions. We denote the family-based association test for the <italic>i</italic>th marker locus by <italic>FBAT<sub>i</sub></italic>. Such an FBAT statistic can be the standard TDT, an FBAT for quantitative/qualitative traits, FBAT-GEE for multivariate traits, etc <xref ref-type="bibr" rid="pgen.1000741-Lange1">[4]</xref>,<xref ref-type="bibr" rid="pgen.1000741-Lange2">[6]</xref>,<xref ref-type="bibr" rid="pgen.1000741-Lange3">[18]</xref>,<xref ref-type="bibr" rid="pgen.1000741-Spielman1">[20]</xref>,<xref ref-type="bibr" rid="pgen.1000741-Lange4">[21]</xref>. Similarly, for the <italic>i</italic>th marker, the between-family information can be used to assess the evidence for association at a population-based level by computing a VanSteen-type <xref ref-type="bibr" rid="pgen.1000741-VanSteen1">[11]</xref> &#8220;screening statistic&#8221; <italic>T<sub>i</sub></italic>. The screening statistic is computed based on the data for offspring phenotype and the parental genotypes/sufficient statistic. The statistic <italic>T<sub>i</sub></italic> can be a Wald test for the genetic effect size that is estimated based on the conditional mean model <xref ref-type="bibr" rid="pgen.1000741-Lange5">[22]</xref>, or the estimated power of the family-based test <italic>FBAT<sub>i</sub></italic>
<xref ref-type="bibr" rid="pgen.1000741-Lange6">[23]</xref>, either of which is feasible. However, while the <italic>FBAT<sub>i</sub></italic> statistic is robust against population stratification, the screening statistic <italic>T<sub>i</sub></italic> is susceptible to confounding. For this reason, the VanSteen-type testing strategies have restrictively used the between-family information as weights for p-values of the <italic>FBAT-</italic>statistic, but never as a component in the test statistic itself.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In order to construct a family-based association test that incorporates both the within and the between-family information, the Z-statistics that correspond to the p-values of <italic>FBAT<sub>i</sub></italic> and <italic>T<sub>i</sub></italic> are computed. The statistic <italic>Z<sub>&#945;</sub><sup>*</sup></italic> is the <italic>&#945;</italic> quantile of standard normal distribution. <italic>pFBAT<sub>i</sub></italic> and <italic>pT<sub>i</sub></italic> are the p-value of the FBAT-statistics and one sided p-value of the screening statistic where the direction of the one sided screening statistic is defined by the directionality of <italic>FBAT<sub>i</sub></italic>. Based on the statistical independence of <italic>FBAT<sub>i</sub></italic> and <italic>T<sub>i</sub></italic>
<xref ref-type="bibr" rid="pgen.1000741-VanSteen1">[11]</xref> under the null-hypothesis, we can obtain an overall family-based association test statistic <italic>Z<sub>i</sub></italic> by combining both Z-statistics in a weighted sum, <disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1000741.e001.jpg" /></disp-formula>where the parameters <italic>w<sub>FBAT</sub></italic> and <italic>w<sub>T</sub></italic> are standardized weights so that the overall family-based association test <italic>Z<sub>i</sub></italic> has a normal distribution with mean 0 and variance 1, i.e. <italic>w<sub>FBAT</sub></italic>
<sup>2</sup>+<italic>w<sub>T</sub></italic>
<sup>2</sup>&#8202;=&#8202;1. In the literature, this approach of combining two test statistics is known as the Liptak-method <xref ref-type="bibr" rid="pgen.1000741-Liptak1">[24]</xref>. However, the Liptak-method varies here from its standard application in that the 2 test statistics have to be combined so that confounding in the screening statistic <italic>T<sub>i</sub></italic> cannot affect the validity of the overall family-based association test statistic <italic>Z<sub>i</sub></italic>. In the context of a genome-wide association study (GWAS), we are able to achieve this goal by using rank-based p-values for the screening statistic <italic>T<sub>i</sub></italic> instead of their asymptotic p-values.</p><p>The &#8220;screening statistics&#8221; <italic>T<sub>i</sub></italic> are sorted based on their evidence for association so that <italic>T</italic>
<sub>(<italic>m</italic>)</sub> denotes the screening statistic with the least amount of evidence for association and <italic>T</italic>
<sub>(1)</sub> the screening statistic with the largest amount of evidence for association. The rank-based p-value, (<italic>i</italic> &#8211; 0.5)/<italic>m</italic>, is then assigned to the ordered screening test statistics <italic>T</italic>
<sub>(<italic>i</italic>)</sub>. If there is a tie, then the average of the ranks will be used for the computation of the rank-based p-value for the <italic>i</italic>th marker. Since the null-hypothesis will be true for the vast majority of the SNPs in a GWAS, the rank-based p-values provide an alternative way to assess the significance of the population-based screening statistic <italic>T<sub>i</sub></italic>. The overall association test <italic>Z<sub>i</sub></italic> is then computed based on the Z-score for the asymptotic p-value of the FBAT-statistic and the <italic>Z</italic>-score for the ranked-based p-value of the screening statistic <italic>T<sub>i</sub></italic>. In <xref ref-type="supplementary-material" rid="pgen.1000741.s001">Text S1</xref> we show that the overall association test <italic>Z<sub>i</sub></italic> maintains the global significance level <italic>&#945;</italic>, under any situations including population admixture and stratification. This can be understood intuitively as well. The smallest rank-based p-value is 0.5/<italic>m</italic>. Using the Bonferroni-correction to adjust for multiple testing, the individual, adjusted significance level is <italic>&#945;</italic>/<italic>m</italic> which will always be smaller than the smallest rank-based p-value, 0.5/<italic>m</italic>, unless the pre-specified global significance level <italic>&#945;</italic> is great than 0.5. This implies that the overall family-based association test can never achieve genome-wide significance just based on the rank-based p-values alone. The FBAT-statistic has to contribute evidence for the association as well in order for the overall family-based association test to reach genome-wide significance. Finally, we have to address the specification of the weights <italic>w<sub>FBAT</sub></italic> and <italic>w<sub>T</sub></italic> in the overall family-based association test statistic <italic>Z<sub>i</sub></italic>. While any combination of weights <italic>w<sub>FBAT</sub></italic> and <italic>w<sub>T</sub></italic> will provide a valid test statistic <italic>Z<sub>i</sub></italic>, the most powerful overall statistic <italic>Z<sub>i</sub></italic> is approximately achieved when the ratio of the weights is equal to the ratio of the standardized effect sizes, the expected effect size of the regression coefficient divided by its (estimated) standard deviation. For quantitative traits in unascertained samples, one can show that optimal power levels are achieved for equal weights, i.e. <italic>w<sub>FBAT</sub></italic>&#8202;=&#8202;<italic>w<sub>T</sub></italic>. In general, the equal weighting scheme seems to provide good power levels for any disease mode of inheritance and for different trait types, e.g. binary traits, time-to-onset, etc. The theoretical derivation of optimal weighting schemes for such scenarios is ongoing research and will be published subsequently.</p><p>Furthermore, it is important to note that, instead of the Liptak-method, Fisher's method for combining p-values could have been used as well to construct an overall family-based association test which would have the same robustness properties as the overall-test based on the Liptak-method. However, simulation studies (data not shown) suggest that the highest power levels are consistently achieved with the Liptak method. We therefore omit the approach based on Fisher's method here.</p>