<p>The study protocol was approved by relevant ethics committees and institutional review boards of all participating centers, the International Agency for Research on Cancer (IARC), and the U.S. National Cancer Institute (NCI) at the U.S. National Institutes of Health. All study subjects and their physicians provided written informed consent.</p><p>Cases were from a hospital-based case-control study of sporadic RCC that was conducted between 1999 and 2003 in seven centers in four countries of Central and Eastern Europe (Moscow, Russia; Bucharest, Romania; Lodz, Poland; and Prague, Olomouc, Ceske-Budejovice, and Brno, Czech Republic) as previously described <xref ref-type="bibr" rid="pgen.1002312-Brennan1">[45]</xref>, <xref ref-type="bibr" rid="pgen.1002312-Hung1">[46]</xref>. All newly diagnosed and histologically confirmed cases of sporadic kidney cancer (ICD-O2 code C.64) were identified at participating hospitals in each study region between 1999 and 2003. Histological slides of renal tumor tissue from all cases were reviewed by an international renal cancer pathology expert at the U.S. National Cancer Institute for standardized confirmation and disease classification (MM). Only confirmed cases of RCC and ccRCC were included in the analyses. There were 1097 cases included in the final case-control study: 524 of the 1097 cases (48%) originally diagnosed with RCC from hospital reports that provided frozen tumor biopsies for genetic analysis. Of the 524 cases, 507 (97%) cases were confirmed with RCC by review of their biopsy material provided, and 470 of the 507 (93%) were confirmed with the ccRCC subtype.</p><p>Questionnaires were administered in person by trained interviewers at each center. Subjects were asked about their lifestyle habits, in particular tobacco consumption, anthropometric measures one year before diagnosis, personal and familial medical history, and dietary habits. Lifetime occupational information for jobs of at least 12 months duration was also collected during interviews through the use of general occupational and job-specific questionnaires. Job-specific questionnaires covered (1) possible organic and chlorinated solvent exposures, (2) hours per week of solvent exposure, (3) source of solvent exposure, and (4) a description of solvent use as previously described <xref ref-type="bibr" rid="pgen.1002312-Moore1">[42]</xref>. All coding in the re-assessment was performed while blinded with respect to the previous assessment (with respect to chlorinated solvents and TCE exposures) as well as disease status.</p><p>Frequencies of fruit, vegetable, and alcohol intake were examined as they have been inversely associated with the prevalence of <italic>VHL</italic> alterations and particular alteration subtypes <xref ref-type="bibr" rid="pgen.1002312-Hemminki1">[9]</xref>. The dietary questionnaire was comprised of 23 food items which the study investigators selected by consensus during the planning stage of the study, which had been validated as previously described <xref ref-type="bibr" rid="pgen.1002312-Hsu1">[47]</xref>. The questionnaire was repeated for two different time periods: 1) the year prior to interview, and 2) prior to political and market changes in 1989 (1991 in Russia). A lifetime weighted average intake for the two time periods was calculated as previously described <xref ref-type="bibr" rid="pgen.1002312-Hsu1">[47]</xref>.</p><p>Frozen tumor biopsies were collected from a subset of cases enrolled in the case-control study in which detailed information on tumor pathology, patient characteristics, and occupational risk factors for RCC, had been collected (<xref ref-type="supplementary-material" rid="pgen.1002312.s001">Table S1</xref>). Tumor DNA extraction was performed following an additional pathology review of each tissue sample (FW) followed by macrodissection to remove non-tumor tissue, as previously described <xref ref-type="bibr" rid="pgen.1002312-Nickerson1">[6]</xref>. Sample areas estimated to contain at least 70% tumor cells were removed for DNA extraction. For each sample, 5 mm<sup>3</sup> of tissue was sectioned and digested with 0.4 &#181;g Proteinase K per &#181;l of digestion buffer (500 mM KCl, 100 mM Tris-HCl, 15 mM MgCl<sub>2</sub>, 0.5% Tween 20) at 50&#176;C overnight. A standard protocol<sup>1</sup> was used to extract DNA from the digested samples.</p><p>PCR of <italic>VHL</italic> coding sequences, endonuclease scanning, and sequencing were performed as previously described <xref ref-type="bibr" rid="pgen.1002312-Nickerson1">[6]</xref>. PCR primers for this study for exons 1&#8211;3 were the following: <italic>VHL</italic> Exon1- Forward:<named-content content-type="gene">5&#8242;-CTACGGAGGTCGACTCGGGAG</named-content>, <italic>VHL</italic> Exon 1- Reverse:<named-content content-type="gene">5&#8242;-GGGCTTCAGACCGTGCTATCG</named-content> (amplicon length:495 bp); Exon 2- Forward:<named-content content-type="gene">5&#8242;-CCGTGCCCAGCCACCGGTGTG</named-content>, Exon 2- Reverse:<named-content content-type="gene">5&#8242;-GGATAACGTGCCTGACATCAG</named-content> (amplicon length:288 bp), Exon 3- Forward:<named-content content-type="gene">5&#8242;-CGTTCCTTGTACTGAGACCCTAG</named-content>, Exon 3- Reverse:<named-content content-type="gene">5&#8242;-GAACCAGTCCTGTATCTAGATCAAG</named-content> (amplicon length:317 bp).</p><p>Amplification was carried out in 50 &#181;L reactions using 10 to 15 ng tumor DNA and HotMaster Taq DNA Polymerase (Eppendorf). Thermal cycling was accomplished using a MJ Research (Bio-Rad) DNA Engine and a touchdown PCR program with an annealing temperature of 58C. PCR products were heteroduplexed using standard conditions. PCR products were analyzed on 2% agarose gels and electrophoresis.</p><p>Heteroduplexed PCR samples were analyzed using SURVEYOR Nuclease (Transgenomic, Inc.) and standard non-denaturing HPLC conditions appropriate for DNA fragment sizing as previously described <xref ref-type="bibr" rid="pgen.1002312-Nickerson1">[6]</xref>. A 100-bp DNA ladder (New England Biolabs) was run as a size marker. Positive and negative controls were included with each plate of PCR products to monitor endonuclease cleavage efficiency.</p><p>Excess PCR primers were removed using the AMPure PCR Purification system (Agencourt Bioscience Corp.). Reaction chemistry using BigDye version 3.1 (Applied Biosystems) and cycle sequencing on an MJ Research thermal cycler were adapted from the manufacturer's recommendations. Sequencing products were purified using CleanSEQ reagents (Agencourt Bioscience). Sequence chromatograms were analyzed using Sequencher (GeneCodes, Ann Arbor, MI).</p><p>Standard methods were used for bisulfite modification of 100 to 500 ng of tumor DNA (Zymo Research Laboratories). Primers were designed to amplify both methylated and unmethylated alleles across 11 CpG dinucleotides of the <italic>VHL</italic> promoter. Primers for the methylation analysis were the following: wild-type (WT) amplicon: <italic>VHL</italic> WT-Forward-<named-content content-type="gene">CTACGGAGGTCGACTCGGGAG</named-content>, WT-Reverse-<named-content content-type="gene">GCGATTGCAGAAGATGACCTG</named-content> (amplicon length:335 bp); <italic>VHL</italic> nested primers: WT Forward-<named-content content-type="gene">CGGGTGGTCTGGATCG</named-content>, WT-Reverse <named-content content-type="gene">AGTTCACCGAGCGCAGCA</named-content> (nested amplicon length:226 bp). Post-bisulfite modification primers that annealed to both methylated (M) and unmethylated (U) alleles were as follows: <italic>VHL</italic> M/U Forward-5&#8242;-<named-content content-type="gene">TTAYGGAGGTYGATTYGGGAG</named-content>, and <italic>VHL</italic> M/U Reverse- <named-content content-type="gene">ACRATTACAAAAAATAACCTA</named-content>, (amplicon length:335 bp), nested primers <italic>VHL</italic> M/U Forward-<named-content content-type="gene">YGGGTGGTTTGGATYG</named-content>, VHL M/U Reverse <named-content content-type="gene">AATTCACCRAACRCAACA</named-content>, nested (amplicon length: 226 bp). PCR was performed using an MJ Research PTC200 thermal cycler and a touchdown PCR program with an annealing temperature of 50&#176;C. Nested PCR included 1 &#181;L of a 1&#8758;10 dilution of first-round product using cycling conditions as described above. PCR products were visualized in 2.0% agarose and bi-directionally sequenced. Cytosine positions in CpGs were inspected for thymine or cytosine signals in chromatograms, and scoring was conducted as follows: T only, not methylated; both cytosine and thymine, partially methylated; C only, fully methylated. Tumor samples with at least four of 11 methylated CpGs (&gt;36%) were considered as hypermethylated. All analyses were run in duplicate, blinded to <italic>VHL</italic> mutation status, and with positive (CpGenome Universal Methylated DNA, Chemicon/Millipore) and negative (K562 Human Genomic DNA, Promega) controls.</p><p>Each DNA sample from the same case-series was analyzed using Scanning and OncoBAC arrays. Scanning arrays were comprised of 2464 BACs selected at approximately megabase intervals along the genome as described previously <xref ref-type="bibr" rid="pgen.1002312-Hodgson1">[48]</xref>, <xref ref-type="bibr" rid="pgen.1002312-Wilhelm1">[49]</xref>. OncoBAC arrays were comprised of 960 Pa, PAC or BAC clones. About three-quarters of the clones on the OncoBAC arrays contained genes and STSs implicated in cancer development or progression. All cones were printed in quadruplicate. DNA samples for array CGH were labeled as described previously <xref ref-type="bibr" rid="pgen.1002312-Hodgson1">[48]</xref>, <xref ref-type="bibr" rid="pgen.1002312-Wilhelm1">[49]</xref>. Briefly, 500 ng each of cancer and normal female genomic DNA sample was labeled by random priming with CY3- and CY5-dUTP, respectively; denatured; and hybridized with unlabeled Cot-1 DNA to CGH arrays. After hybridization, the slides were washed and imaged using a 16-bit CCD camera through CY3, CY5, and DAPI filters <xref ref-type="bibr" rid="pgen.1002312-Pinkel1">[50]</xref>. Array CGH data image analyses were performed as described previously <xref ref-type="bibr" rid="pgen.1002312-Jain1">[51]</xref>, <xref ref-type="bibr" rid="pgen.1002312-Fridlyand1">[52]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genomic DNA was extracted from blood samples as described previously <xref ref-type="bibr" rid="pgen.1002312-Moore1">[42]</xref>. Single nucleotide polymorphisms (SNPs) were selected from the HapMap Project, using expected minor allele frequencies reported among Caucasians, to capture 80&#8211;90% of common genetic variation across the locus encompassing the <italic>VHL</italic> gene, covering an average of 15 kb upstream and downstream of the gene. This method relies upon linkage disequilibrium (LD) of highly correlated SNPs in order to identify gene regions that are associated with disease susceptibility, or in this case, specific heterogeneous subgroups of cases defined by their <italic>VHL</italic> alteration status <xref ref-type="bibr" rid="pgen.1002312-Stram1">[14]</xref>. SNPs selected (from 5&#8242; to 3&#8242; included the following: rs6442154 (<italic>c3orf10</italic>, Ex3 &#8722;90 T&gt;C), rs6796538 (<italic>VHL</italic>, &#8722;5011 A&gt;G), rs265318 (<italic>VHL</italic>, &#8722;2872 A&gt;C), rs779805 (<italic>VHL</italic>, Ex1 +19 A&gt;G), rs779812 (<italic>VHL</italic>, IVS1 &#8722;1184 G&gt;A), rs1678607 (<italic>VHL</italic>, IVS2 +108 G&gt;T), rs1642742 (<italic>VHL</italic>, Ex3 +473 A&gt;G), rs1642739 (<italic>VHL</italic> 3835 bp 3&#8242;STP C&gt;A), rs457414 (<italic>IRAK2</italic>, &#8722;3754 A&gt;C), rs11465853 (<italic>IRAK2</italic>, IVS1 +249 G&gt;C). Assays were validated at the NCI Core Genotyping Facility (CGF) in 280 control samples from the human diversity panel that included 76 African/African Americans, 66 Caucasians, 49 Native Americans/Hispanics, and 89 Pacific Rim Asians. Subsequently, minor allele frequencies (MAFs) were determined from the CGF panel among Caucasians. All selected SNPs had a minor allele frequency of at least 10%. Methods for assays can be found at: <ext-link ext-link-type="uri" ns0:href="http://snp500cancer.nci.nih.gov">http://snp500cancer.nci.nih.gov</ext-link>. DNA samples were blinded and randomized on PCR plates to avoid any potential bias and duplicate genotyping was performed for a randomly selected 5% of the total series for quality control. Concordance rates were &gt;99% for all SNPs. All SNPs analyzed were within the expected distributions of Hardy-Weinberg equilibrium (p&lt;0.05).</p><p>Tumor and patient characteristics including clinical stage, Fuhrman nuclear grade (I&#8211;IV), node stage (N0, N1, N2&#8211;3), body mass index (BMI) (&lt;25, 25&#8211;35, &gt;35), and smoking status were considered as categorical variables. Smoking status (never, former, or current smoker) was defined as status 2 years prior to the interview. Specifically, participants who were smoking in the 24 months prior interview were classified as current smokers. Other variables such as metastasis (M0, M1) self-reported hypertension (no/yes), family history of cancer or kidney cancer (no/yes), sex, and age at diagnosis (&lt;50, &#8805;50 years) were analyzed as dichotomous variables. Additional exposures and risk factors were selected based upon previous published reports of renal tumor heterogeneity including our own <xref ref-type="bibr" rid="pgen.1002312-Nickerson1">[6]</xref>. These additional factors included fruit, vegetable, and alcohol intake frequency, and <italic>VHL</italic> tagging SNPs <xref ref-type="bibr" rid="pgen.1002312-Hemminki1">[9]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1002312-Young1">[13]</xref>. Because occupational trichloroethylene (TCE) exposure was positively associated with RCC risk in this population <xref ref-type="bibr" rid="pgen.1002312-Moore1">[42]</xref> and because TCE exposure had previously been associated with an increased prevalence of <italic>VHL</italic> mutation among occupationally exposed workers <xref ref-type="bibr" rid="pgen.1002312-Brauch1">[11]</xref>, <xref ref-type="bibr" rid="pgen.1002312-Bruning1">[12]</xref>, we also evaluated associations between occupational exposure to organic and chlorinated solvents and trichloroethylene with <italic>VHL</italic> alteration prevalence. For nutritional variables, intake frequencies of related foods were summed to form food group intake categories based on tertile cutoff points that were defined by consumption frequency among controls. Categories of consumption for food-specific items were grouped as low (never to &lt;once per month), medium (&#8805;once per month but &lt;once per week), and high (&#8805;once per week to daily). Categories of alcohol consumption as grams per week of ethanol among weekly drinkers were none, low (&lt;36.5), medium (36.5&#8211;137.5), and high &#8805;137.5) as described previously <xref ref-type="bibr" rid="pgen.1002312-Hsu1">[47]</xref>.</p><p>The prevalences of <italic>VHL</italic> genetic (sequence alterations) and epigenetic alterations (promoter hypermethylation) observed in tumor DNA samples were considered as dichotomous variables per case (no/yes). <italic>VHL</italic> gene nonsynonymous sequence alterations were considered as inactivating alterations if they were located within exons 1&#8211;3 and would lead to an altered amino acid sequence or a truncated VHL protein. Splice site mutations and promoter hypermethylation were also considered as inactivating alterations. DNA sequence alterations that were synonymous or SNPs were not considered as inactivating alterations. Similarly, sequence alterations that were located 5&#8242; of codon 54, which were very rare, were not considered as inactivating alterations unless they were nonsynonymous and also would cause an alteration in the coding sequence of both pVHL19 and pVHL30 translation products. These included alterations such as insertions, deletions, and nonsense mutations. As in our previous study, the P25L variant was only considered an inactivating alteration if the case also possessed a second <italic>VHL</italic> alteration that would be considered inactivating using the above criteria. The prevalence of cases with a particular type of alteration was calculated by dividing the number of cases with that type of alteration by the total number of cases analyzed in the group. Chi-square tests were applied to contingency table (2&#215;2) analyses to test for differences between the proportion of cases with or without a particular alteration subtype within each group. Trend tests were used to analyze associations between categorical variables and cases with particular types of alterations. Ordered logistic regression was used for multivariate analyses to evaluate associations between categorical variables and case subgroups, initially adjusting for variables that were associated with the same alteration subtype in our univariate analyses (p&lt;0.20; <xref ref-type="supplementary-material" rid="pgen.1002312.s002">Table S2</xref>). With the exception of sex, age, and country, other variables remained in multivariate models if their inclusion modified risk estimates by at least 10%.</p><p>For genotyping analyses, each SNP was assessed in three categories, (G<sub>0</sub>&#8202;=&#8202;homozygote for the major allele, G<sub>1</sub>&#8202;=&#8202;heterozygote, G<sub>2</sub>&#8202;=&#8202;homozygote for the minor allele) using the most common allele as a referent. LD between SNPs was measured using Lewontin's D' statistic. Correlation (r<sup>2</sup>) between tagged regions was evaluated in Haploview. To evaluate associations between SNPs and <italic>VHL</italic> promoter hypermethylation prevalence, logistic regression models were used to calculate odds ratios (OR) and 95% confidence intervals (CI), adjusting for sex, age, country, and fruit intake. Risks were estimated using both additive and dominant models. Risk per allele and trends were calculated using unconditional logistic regression. Analyses were conducted using STATA 10.0 (Stata Corporation, College Station, TX) and all statistical tests were two-sided.</p><p>Haplotype analysis was used to explore the combined contribution of consecutive <italic>VHL</italic> tagging SNPs. A sliding window analysis was first conducted in MatLab to identify regions of interest (The MathWorks, Inc., Natick, MA). For the 10 SNPs examined, sliding windows of 2&#8211;9 SNPs were evaluated, accounting for multiple testing and correlations between SNPs. The significance of haplotype-based associations was assessed using the score test <xref ref-type="bibr" rid="pgen.1002312-Schaid1">[53]</xref>. Haplotypes of interest were analyzed using an R package Haplostats (Version 1.3.1) in (version 2.4.1) adjusting for sex, age, and country. The most common haplotype was used as the reference group and rare haplotypes with frequencies of &lt;5% were combined into one group. Generalized linear models accounting for phase ambiguity were used to estimate haplotype-specific ORs per copy <xref ref-type="bibr" rid="pgen.1002312-Lake1">[54]</xref>, <xref ref-type="bibr" rid="pgen.1002312-Schaid2">[55]</xref>.</p><p>For array CGH analyses of these same cases, those with loss of chromosome arm 3p or clone CTB110j24 to those without, the presence of a loss was considered as a dependent variable in stepwise logistic regression models to first evaluate associations with clinical risk factors including: T stage (T1, T2, T3&#8211;4), Fuhrman nuclear grade (1&#8211;2, 3&#8211;4), node positivity (N0, N1, N2&#8211;3), and presence of metastases (M0, M1, Mx). In addition, co-variates and etiologic risk factors (described above) that were previously associated with RCC risk were initially analyzed using univariate analyses, including the presence of a <italic>VHL</italic> inactivating alteration in tumor tissue (no/yes-any). The criterion for initial inclusion of a variable into multivariate models was a p-value&lt;0.20. Frequency matching variables inherent to the study design (i.e. country, sex and age) were included into all of the models regardless of associations with outcome. Variables selected were then fitted in logistic models adjusted for sex, country, and age, to obtain odds ratios (OR) and 95% confidence intervals (95% CI) as estimates of risk for both types of alterations. All analyses were conducted using SAS 9.1.3 software (SAS Institute Inc.) and STATA 10.0 and statistical tests were two-sided.</p>