<p>Mice used in this study were derived from the ES cell line (ID: RRF055, 129/Sv background). Chimeric male mouse generated from the ES cell injected C57BL/6 blastocytes were transferred into pseudopregrant CD1 female recipients. All mice works were conducted under in compliance with federal regulations and guidelines. All mice work in this study was approved by the NHGRI Animal Care and Use Committee (NIH animal study proposal: G-05-2, KM as a PI).</p><p>We followed the method described previously <xref ref-type="bibr" rid="pgen.1002245-Dertinger1">[22]</xref>. Briefly, six-week old male mice were exposed to 0.7 Gy of &#947;-ray irradiation from a <sup>137</sup>Cs source. Normochromatic erythrocytes (NCEs) and reticulocytes (RETs) in erythrocytes were distinguished by their negative and positive expressions of CD71 in cell surface, respectively. The RETs carrying micronuclei were detected by the nucleic acid binding agent propidium iodide. At least 10,000 RETs and 500,000 NCEs were analyzed in each blood sample.</p><p>Anonymized primary endometrial tumor tissues (n&#8202;=&#8202;102) and matched histologically normal tissues were obtained from the Cooperative Human Tissue Network, or from the Biosample Repository at Fox Chase Cancer Center, Philadelphia PA. Six cases of matched tumor and normal DNAs were purchased from Oncomatrix. All primary tumor tissues were pretreatment specimens, snap-frozen within 30 minutes of surgical removal. A hematoxylin and eosin (H&amp;E) stained section was cut from each tumor specimen and reviewed by a pathologist to verify histology and to delineate regions of tissue with a tumor cell content of &#8805;70%. All specimens and accompanying clinicopathological information were anonymized, and procured with appropriate IRB approval. The Office of Human Subjects Research declared exempt for human specimen usage because the specimens were anonymized when investigators received them: The exemption numbers are #3529, #3456, and #3534.</p><p>To confirm that tumor-normal pairs were consistent with derivation from the same individual, all human DNA samples were genotyped using the Coriell Identity Mapping kit (Coriell). Genotyping fragments were size separated on an ABI-3730<italic>xl</italic> DNA analyzer (Applied Biosystems). Alleles were scored using GeneMapper software.</p><p>Primer pairs to amplify <italic>ATAD5</italic> were designed using published methods <xref ref-type="bibr" rid="pgen.1002245-Chines1">[43]</xref> (<xref ref-type="supplementary-material" rid="pgen.1002245.s008">Table S4</xref>). PCR products were subjected to bidirectional Sanger sequencing using the BigDye Terminator Version 3.1 Cycle Sequencing Kit (Applied Biosystems). Sequencing reactions were run on ABI 3730<italic>xl</italic> DNA Analyzers (Applied Biosystems). Sequence data was analyzed as described in the Supplemental Experimental Procedures.</p><p>Pure populations of tumor cells were isolated from heterogeneous tissue sections by laser capture microdissection (LCM) using an Arcturus PixCell IIe system. RT-PCR products, generated from microdissected tumor cells were sequenced to discriminate between monoallelic and biallelic expression of mutant <italic>ATAD5</italic> alleles.</p><p>The wild-type and mutant <italic>Atad5</italic> alleles were distinguished by PCR using primers: 1298NF (<named-content content-type="gene">5&#8242;- GAG ACT GTC TCA CCA TGT ACA GGG-3&#8242;</named-content>), 1372NR (<named-content content-type="gene">5&#8242;- ATA AAT TGT AAA AAA TCG ATC TCT-3&#8242;</named-content>) and pGKR (<named-content content-type="gene">5&#8242;- GTG GCC TGT CCC TCT CAC CTT CTA-3&#8242;</named-content>). PCR products of 542 bp and 900 bp distinguished the wild-type and mutant alleles, respectively (<xref ref-type="fig" rid="pgen-1002245-g001">Figure 1B</xref>). PCR conditions were an initial denaturation step at 94&#176;C for 5 minutes, followed by 35 cycles of a denaturation step at 94&#176;C for 30 seconds, an annealing step at 55&#176;C for 30 seconds, and an elongation step at 72&#176;C for 60 seconds and by a final elongation step at 72&#176;C for 7 minutes. There were no additional insertions of the retroviral vector in the genome of RRF055 ES cells, as confirmed by Southern hybridization with a NEO probe. The ES cells were injected to into C57BL/6 blastocytes that were transferred into pseudopregnant CD1 female recipients. A resulting chimeric male mouse was mated with C57BL/6 females. Germline transmission was confirmed in progeny of agouti mice. The germline transmitted heterozygous mice were crossed with wild-type 129/Sv mice to increase the heterozygous population. In addition, two heterozygous mice were mated to generate <italic>Atad5<sup>m/m</sup></italic> mice. Mice were bred and maintained under a protocol (#G-05-02) approved by the Institutional Animal Care and Use committee in the NHGRI, US NIH Animal Care Facility. Young <italic>Atad5<sup>+/m</sup></italic> mice did not show any distinct phenotypic features and were fertile. However, when the <italic>Atad5<sup>+/m</sup></italic> mice were intercrossed, we did not observe any postnatal progeny that were homozygous for the <italic>Atad5</italic> allele (<italic>Atad5<sup>m/m</sup></italic>).</p><p>Mouse embryonic fibroblast (MEF) cells were isolated from both wild-type and <italic>Atad5<sup>+/m</sup></italic> embryos at day 13.5 p.c. and cultured in Dulbecco's modified Eagle's medium (DMEM) (high glucose, w/o L-glutamine, w/o sodium pyruvate, GibcoBRL) supplemented with 2 mM glutamine, 1% penicillin/streptomycin, 15% fetal calf serum, 2 mM non essential amino acids, 0.1 mM 2-&#946; mercaptoethanol in 4% oxygen concentration. RNAs isolated from wild-type and <italic>Atad5<sup>+/m</sup></italic> MEFs were analyzed by RT-PCR with primer sets specifically amplifying the region of mRNA containing exons 4 and 5, exons 18 and 19, or exons 19 and 20 respectively from 500 ng of total RNA isolated with TRIzol reagent (Invitrogen). The expression of &#946;-actin was used as an internal expression control. For quantitative RT-PCR, reactions were performed using the ABI7300 sequence Detection System (Applied Biosystems) with an annealing temperature of 60&#176;C and the following primer pairs: <italic>Atad5</italic>-Ex4F (<named-content content-type="gene">5&#8242;-GAC TGA AGA AAC AGT GGT ACC-3&#8242;</named-content>) and <italic>Atad5</italic>-Ex5R (<named-content content-type="gene">5&#8242;-CAA AGA CAG GAA TGG CTG CTC-3&#8242;</named-content>); <italic>Atad5</italic>-Ex14F (<named-content content-type="gene">5&#8242;-GCC TCT TCA CAG CGA AGT GG-3&#8242;</named-content>) and <italic>Atad5</italic>-Ex15R (<named-content content-type="gene">5&#8242;-GTG CTC GCT TCT GCC CAC T-3&#8242;</named-content>); <italic>Atad5</italic>-Ex18F (<named-content content-type="gene">5&#8242;-GTC TAG TGT TTG ATG GCT GCT TTG-3&#8242;</named-content>) and <italic>Atad5</italic>-Ex19R (<named-content content-type="gene">5&#8242;-CAC TTG TAG ATA GCT GGC AAC-3&#8242;</named-content>); <italic>Atad5</italic>-Ex19F (<named-content content-type="gene">5&#8242;-GTT GCC AGC TAT CTA CAA GTG-3&#8242;</named-content>) and <italic>Atad5</italic>-Ex20R (<named-content content-type="gene">5&#8242;-GAG CCA TCT TCT GAA CAA ACC-3&#8242;</named-content>); and LacZR (<named-content content-type="gene">5&#8242;-CTT CGC TAT TAC GCCAGC TGG-3&#8242;</named-content>).</p><p>Total cell extract was prepared by sonication of cells in lysis buffer (100 mM Tris-Cl (pH 7.5), 50 mM NaCl, 8% glycerol, and protease inhibitor cocktail (Roche) with either 0.5% Triton X-100 or 0.5% NP40 as a detergent). Supernatant after centrifugation was used as total cell extract. Proteins in total cell extract were separated by electrophoresis using 4&#8211;12% gradient polyacrylamide gels (BioRad) and transferred to PVDF membranes (BioRad). Proteins on PVDF membranes were detected by Western blotting with antibodies. Atad5 protein was detected by anti ELG1 antibody raised in rabbits against human N-terminal 1&#8211;197 amino acid fragments of ELG1. Fused Elg1 protein was detected by anti-Beta galactosidase antibody (Abcam).</p><p>The TritonX-100-insoluble fraction (chromatin bound fraction) was isolated from the Triton X-100-soluble fraction using the traditional methods with slight modifications. Harvested cells were resuspended in buffer A (100 mM NaCl, 300 mM sucrose, 3 mM MgCl2, 10 mM Pipes (pH 6.8), 1 mM EGTA, 0.2% Triton X-100, 100 mM NaVO4, 50 mM NaF, and protease inhibitors (RocheApplied Science)) and incubated for 5 minutes on ice with gentle inverting. The supernatants were recovered as the &#8220;soluble fraction&#8221; after centrifugation. Followed by washing with buffer A, the pellet was resuspended either in buffer X (100 mM Tris-HCl (pH 8.5), 250 mM NaCl, 1 mM EDTA, 1% Nonidet P-40, 100 mM NaVO4, 50 mM NaF, 2 mg/ml bovine serum albumin, and protease inhibitors) or in buffer B (50 mM Tris-HCl (pH 7.5), 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 0.1% SDS, 100 mM 100 mM NaVO4, 50 mM NaF, and protease inhibitors) for immunoblotting. Followed by a 10 min incubation on ice, the samples were sonicated and then incubated for another 10 min on ice before centrifugation to isolate the &#8220;chromatin-bound fraction.&#8221; Immunoblots were stained with PCNA (Santa Cruz) and Rad9 (Cell Signaling) antibodies.</p><p>All embryos were generated by natural mating of <italic>Atad5<sup>+/m</sup></italic> mice. The morning of the day on which a vaginal plug was detected was designated as day 0.5 p.c. and 3.5 p.c. embryos were collected by flushing uteri with M2 medium (Sigma). To genotype embryos collected at other embryonic days, individual yolk sacs were lysed in 20 &#181;l of 1xPCR lysis buffer with 0.2 mg/ml of proteinase K at 55&#176;C for overnight.</p><p>MEFs were treated with the indicated doses of methyl methane sulfonate (MMS) for 2 hours, washed with PBS three times, and incubated in media for one day. Colcemid (0.1 &#181;g/ml) was added to the medium over night before cells were collected. For each cell culture, 50 metaphases were analyzed for chromosomal abnormalities. MEFs were plated in 96 well plates (1,000 cells/well) in triplicate. The following day, cells were treated with MMS at different doses, ranging from 0.00012 to 0.01% for one hour. Cells were washed with PBS and incubated for ten more days. Cell survival was determined using the CyQuant cell proliferation assay kit (Invitrogen).</p><p>MEF cells were cultured in 2 well Lab Tek chamber slides (Nunc) chamber slide. The next day cells were fixed with 3.7% para-formaldehyde, treated with 0.2% tritonX-100 and stained with phospho-H2AX (&#945;-&#947;-H2AX; Millipore) and Rad51 (Santa Cruz) antibodies. Fluorescence conjugated anti IgG antibodies (Invitrogen-Molecular Probes) were used as a secondary antibody. Confocal images were collected with a Ziess LSM 510 NLO meta system mounted on a Ziess Axiovert 200 M microscope with an oil immersion Plan-Apochromat X 63/1.4 differential interference contrast objective lens.</p><p>MEF cells cultured in 10 cm culture dish for one day were incubated with BrdU for 36 hours. Cells were then treated overnight with Colcemid (0.1 &#181;g/ml). For each cell culture, 50 metaphases were analyzed for sister chromatid exchange abnormalities.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">MEF cells cultured in 10 cm culture dishes for a day were harvested and washed three times with PBS. Following fixation with 70% ethanol, cells were washed with PBS, stained with PI solution, and kept at 37&#176;C for 30 minutes before sorting. Data were analyzed by ModFIT software (<ext-link ext-link-type="uri" ns0:href="http://www.vsh.com/products/mflt/index.asp">http://www.vsh.com/products/mflt/index.asp</ext-link>).</p><p>The full-length chicken cDNA (DDBJ accession number AB511312) encodes a protein of 1,816 amino acids with 45.6% identity to human <italic>ATAD5</italic>. Partial chicken (ch) <italic>ATAD5</italic> cDNA and genomic sequences were identified by searching databases and cloned by PCR. To knock out <italic>ATAD5</italic> in chicken DT40 cells, a <italic>chATAD5</italic> targeting vector was designed to replace a region that is homologous to the RFC large subunit, RFC1 (PRKO4195) as assigned by conserved domain database with a drug-resistant gene cassette (<xref ref-type="supplementary-material" rid="pgen.1002245.s001">Figure S1A</xref>). The <italic>ATAD5</italic>-targeting vector was created by replacing a 2.7 kb genomic fragment containing 3 exons that correspond to chATAD5 amino acids 1338&#8211;1735, with a <italic>his</italic>-resistant gene cassette. DT40 cell cultures, electroporation and subsequent selection, RT-PCR analysis, cell growth determination, colony formation assay were performed as described <xref ref-type="bibr" rid="pgen.1002245-Yamamoto1">[44]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1002245-Kitao1">[46]</xref>. For &#947;-ray irradiation, <sup>137</sup>Cs gammacell 40 exactor (MDS Nordion) was used. One allele of <italic>chATAD5</italic> gene was targeted as evidenced by Southern blot (<xref ref-type="supplementary-material" rid="pgen.1002245.s001">Figure S1B</xref>). Semi-quantitative RT-PCR analysis showed that <italic>ATAD5</italic> mRNA levels were decreased to nearly half of the wild type level (<xref ref-type="supplementary-material" rid="pgen.1002245.s002">Figure S2C</xref>). Despite numerous attempts to disrupt the second allele, we were unable to recover homozygous <italic>chATAD5</italic> null DT40 cells. In striking contrast to the targeting efficiency of the first chATAD5 allele &#8764;10% (3/29), no double positive clones were obtained from 139 transfectants. Thus, we speculate that complete loss of the <italic>chATAD5</italic> gene might be lethal.</p><p>MEFs were treated with 0.01% MMS for one hour and then allowed to recover for 16 hours. &#8764;6&#215;10<sup>5</sup> cells were imbedded into Pulsed-Field Certified Agarose (Bio-Rad) for a final agarose concentration of 0.75%. Agarose plugs were then digested in proteinase K reaction buffer (100 mM EDTA, pH 8.0, 0.2% sodium deoxycholate, 1% sodium lauryl sarcosine, and 1 mg/ml Proteinase K) at 50&#176;C overnight and washed 4 times in wash buffer (20 mM Tris, pH 8.0, 50 mM EDTA). The plugs were loaded onto a 1% Pulsed-Field Certified Agarose gel (Bio-Rad). Separation was performed on a CHEF-DR III pulsed-field electrophoresis system (Bio-Rad; 100&#176; field angle, 1200 s switch time, 2 V/cm, 14&#176;C) for 72 hours. The gel was stained with ethidium bromide.</p><p>Genomic DNA isolated from normal and tumor tissues from the <italic>Atad5<sup>m/+</sup></italic> mice, using the DNeasy Blood and Tissue Kit (Qiagen, Valencia, CA), was used for array-CGH array analysis assisted by the NHGRI Genomics core. Data analysis was performed by the NHGRI Bioinformatics core. DNAs were hybridized to Whole Mouse Genome CGH Microarrays (105K) (Agilent Technologies) according to the manufacturer's protocol. After hybridization, arrays were washed and scanned. The scanned array images were analyzed and data was extracted using Agilent Feature Extraction (FE) Software with Linear Lowess normalization and background subtraction. Agilent's DNA Analytics Software (Version 4.0.85) was used to identify chromosomal imbalances. The altered chromosomal regions and breakpoints were detected using ADM-1 with threshold 6.0.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Expression profiling was accomplished using the Agilent High Throughput Array (HTA) GeneChip system. All the necessary enzymes and reagents were purchased from the SABiosciences unless specified below. Total RNA was isolated from normal and tumor tissues by guanidinium thiocyanate-phenol-chorofrom extraction using TRIzol reagent (Invitrogen) following the manufacturer's protocol. 600 ng of total RNA from 4 (2 wild-type and 2 tumor) samples were reverse transcribed into cDNA using T<sub>7</sub>dT<sub>24</sub> primer containing the poly T sequence and the promoter for T7 RNA polymerase. After second strand synthesis, double-stranded cDNA was used as a template to synthesize complementary RNA (cRNA) in a modified &#8220;Eberwine&#8221; type RNA amplification process. During this amplification process, cRNA was labeled with aminoallyl-UTP (Sigma). 8 &#181;g of aminoallyl-cRNA from each sample was conjugated with Cy3-NHS ester (GE Healthcare) to generate Cy3-labelled cRNA. 1.65 &#181;g Cy3-labelled cRNA generated was fragmented by the cRNA fragmentation reaction and then applied for hybridization. Hybridization was done at 65&#176;C for 17 hours with 4 rpm shaking. The Whole Mouse Genome Oligo Microarray assembly was then dissembled in the hybridization wash buffer 1 at room temperature. The microarray slides were washed with the fresh hybridization wash buffer 1 for 1 minute, followed with the hybridization with the wash buffer 2 for one minute at 37&#176;C, with acetonitrile for 1 minute at room temperature, and with the stabilization and drying solution for 30 seconds. The dried microarray slide was scanned into a TIFF image file on an Agilent microarray scanner at 5 &#181;m resolution using default settings. Microarray intensity data were extracted from the TIFF image using the Agilent Feature Extraction Software 9.1.3. The Feature Extraction Software also processed the signal intensities from Cy3 channel for each microarray. The processed data were then imported into GeneSpring GX 10 (Agilent Technologies) software and normalized using the standard settings for Agilent microarray including per chip global normalization. Global normalization was performed by dividing each measurement by the 50<sup>th</sup> percentile of all measurements in that sample. The percentile was calculated using only genes marked as present. Data for genes that were absent in all samples and all the control probes were excluded from analyses. Paired t-test were performed to identify a list of statistically significant (P&lt;0.05) differentially expressed genes between normal and tumor samples with a cut-off criteria of expression differences greater than or equal to 2-fold. Each differentially expressed gene was classified according to its Gene Ontology (GO), in which genes are organized into hierarchical categories based on biological process, molecular function and cellular compartment. GO term analysis was performed using Genespring GX 10 software. To determine the functional relationships among the identified genes, differentially expressed genes were imported into Ingenuity Pathway Analysis software (IPA) (<ext-link ext-link-type="uri" ns0:href="http://www.ingenuity.com">http://www.ingenuity.com</ext-link>). IPA contains the most literature knowledge of biological interactions among the gene products. This web-based software generated a set of molecular network based on interaction between uploaded genes and all other genes present in the knowledge base. In order to compare the measurements obtained from CGH array analysis and microarray expression data, the genes that were present in regions of amplification and deletion detected by array-CGH were chosen to determine their correlation in microarray expression. The goal of this analysis was to detect genes where a copy number change is correlated with the change in expression. Multiple public expression database, Oncomine (<ext-link ext-link-type="uri" ns0:href="https://www.oncomine.org/resource/login.html">https://www.oncomine.org/resource/login.html</ext-link>) was manually interrogated for mouse <italic>Atad5</italic> and its aliases. Only studies comparing normal against tumor tissues were queried for this gene.</p><p>Genomic DNA was isolated from macrodissected tissue with greater than 70% tumor purity using the Puregene kit (Qiagen). Total RNA was isolated with TRIzol (Invitrogen), according to the manufacturer's instructions. cDNA was synthesized using the Super Script III First Strand Synthesis System (Invitrogen).</p><p>Human embryonic kidney (HEK) 293T cells were maintained in Dulbecco's modified Eagle's medium (DMEM) with 10% fetal bovine serum (FBS) (Hyclone), 100 U/mL penicillin G and 100 &#181;g/mL streptomycin. Transfections were carried out using Lipofectamine2000 (Invitrogen) and cells were incubated for 48 h prior to harvesting. For RNA interference, transfection was performed two times with an interval of 24 h.</p><p>Full-length <italic>ATAD5</italic> cDNA was cloned into p3XFLAG CMV10 expression vector and used as a template for site-directed mutagenesis with the QuickChange Site-Directed Mutagenesis kit (Stratagene). Construct integrity was verified by nucleotide sequencing. siRNAs (Dharmacon) to target the 3&#8242; untranslated region (Utr) of <italic>ATAD5</italic> were: 5&#8242;- GUAUAUUUCUCGAUGUAC A-3&#8242; (sense) and <named-content content-type="gene">5&#8242;- UGUACAUCGAGAAAUAUACUU -3&#8242;</named-content> (antisense).</p><p>Triton X-100 soluble and insoluble fractions were isolated using a modified version of the method described by Kannouche et al <xref ref-type="bibr" rid="pgen.1002245-Kannouche1">[47]</xref>. For immunoprecipitation, proteins were pre-cleared with protein G Sepharose beads (GE Healthcare), incubated with specific antibodies, and then the complex recognized by antibodies was precipitated with protein G Sepharose beads. Proteins or immunoprecipitated samples were resolved on NuPAGE Novex 4&#8211;12% Bis-Tris Gel (Invitrogen) and transferred to PVDF membranes (Bio-Rad), followed by immunoblotting.</p><p>Total RNAs (500 ng), prepared using TRIzol (Invitrogen) were subjected to qRT-PCR using the SuperScriptIII Platinum Two Step qRT-PCR kit (Invitrogen) and a 7300 Real Time PCR system (Applied Biosystems). Primer sequences used for qRT-PCR were: ATAD5, <named-content content-type="gene">5&#8242;-CGG AGA CGA AGA AAG CAA AG-3&#8242;</named-content> (sense) and <named-content content-type="gene">5&#8242;-CAA TGA GAA ACA AGG GCA GA-3&#8242;</named-content> (antisense), &#946;-actin, <named-content content-type="gene">5&#8242;-GCT CGT CGT CGA CAA CGG CTC-3&#8242;</named-content> (sense) and <named-content content-type="gene">5&#8242;-CAA ACA TGA TCT GGG TCA TCT TCT C-3&#8242;</named-content> (antisense). &#946;-actin expression served as an internal control.</p><p>PCR was performed in a total volume of 10 &#181;l containing 5 ng genomic DNA, 0.16 &#181;M sense primer, 0.16 &#181;M antisense primer, and Immomix (Bioline). PCR amplification was performed on MBS Satellite 384 Thermalcyclers (Thermo Scientific). Cycling conditions were 95&#176;C for 3 min followed by 40 cycles of 95&#176;C for 15 s; 60&#176;C for 15 s; 72&#176;C, 1 min; followed by a final extension of 72&#176;C for 5 min.</p><p>Sequence trace quality was assessed with the base-calling program, Phred <xref ref-type="bibr" rid="pgen.1002245-Ewing1">[48]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1002245-Gordon1">[50]</xref>. All traces were included in the subsequent analysis, since deletion-insertion polymorphisms can mimic poor quality data from a Phred-quality measure, but may contain valid sequence data. All sequences for a given primer pair were assembled using Consed <xref ref-type="bibr" rid="pgen.1002245-Gordon1">[50]</xref>; overlapping amplimers were assembled separately to allow independent cross validation of calls in overlapping regions. Sequence variants, including single-nucleotide differences and short (&lt;100 base pair) insertions and deletions, were identified using PolyPhred v6.11 <xref ref-type="bibr" rid="pgen.1002245-Bhangale1">[51]</xref> and DIPDetector, an insertion-deletion (indel) detector optimized for improved sensitivity in finding insertions and deletions from aligned trace data. Since a heterozygous insertion or deletion produces traces that are superpositions of two alleles of different length, DIPDetector is able to use Phred's primary and secondary peak predictions at each base position to calculate an autocorrelation function of the trace with itself shifted by different potential insertion or deletion sizes. A maximum in the value of this function with respect to shift size indicates a potential indel. To improve accuracy, DIPDetector also examines the trace alignments for homozygous insertions and deletions, and uses these homozygous genotypes to refine its prediction for the insertion or deletion position, resulting in greater accuracy of positions when compared to PolyPhred. Human genome assembly hg18 (NCBI Build 36.1) was used as the reference sequence. Variant positions were cross-referenced to dbSNP (Build 129) entries to identify known polymorphisms. To determine whether novel variants were somatic mutations or germline polymorphisms, the appropriate tumor DNA and matched normal DNA were re-amplified in an independent PCR followed by sequence analysis of the variant position.</p><p>Frozen tissue sections (8 &#181;M) of OCT-embedded primary tumors were prepared at American HistoLabs Inc., (Gaithersburg MD), and were H&amp;E stained immediately prior to microdissection. Approximately 4000&#8211;5000 laser pulses were performed using the following infrared laser parameters: laser spot size 7.5 &#181;m, power 70 mW, duration 950 &#181;s. Each tissue section and the subsequent dissection was reviewed by a staff pathologist. RNA was isolated from LCM samples using the Arcturus Picopure RNA isolation kit (MDS Analytical Technologies). RNA (50&#8211;200 ng) was converted to cDNA using SuperScript III First-Strand Synthesis System (Invitrogen). RT-PCR primers were designed to span intronic boundaries to avoid amplification of any contaminating genomic DNA. Primer sequences (5&#8242;-3&#8242;) were: <named-content content-type="gene">CTT TTT GAG GAG GTT GAT G</named-content> (sense) and <named-content content-type="gene">GTC CAA GTC AGT GTC AAA TAG ATC AC</named-content> (antisense). Purified RT-PCR products were subjected to bidirectional sequencing to determine whether there was monoallelic or biallelic expression of mutant <italic>ATAD5</italic> alleles.</p>