<p>All studies were approved by Institutional Review Boards at their respective sites (details are given in <xref ref-type="supplementary-material" rid="pgen.1002138.s016">Text S1</xref>). PAGE study samples were drawn from four large population-based studies or consortia: EAGLE (Epidemiologic Architecture for Genes Linked to Environment), based on three National Health and Nutrition Examination Surveys (NHANES) <xref ref-type="bibr" rid="pgen.1002138-Centers1">[59]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1002138-Centers3">[61]</xref>, the Multiethnic Cohort (MEC) <xref ref-type="bibr" rid="pgen.1002138-Kolonel1">[62]</xref>, the Women's Health Initiative (WHI) <xref ref-type="bibr" rid="pgen.1002138-Design1">[63]</xref>, <xref ref-type="bibr" rid="pgen.1002138-Anderson1">[64]</xref>, and Causal Variants Across the Life Course (CALiCo), a consortium of several cohort studies: Atherosclerosis Risk in Communities Study (ARIC) <xref ref-type="bibr" rid="pgen.1002138-The1">[65]</xref>, Coronary Artery Risk in Young Adults (CARDIA) <xref ref-type="bibr" rid="pgen.1002138-Friedman1">[66]</xref>, Cardiovascular Health Study (CHS) <xref ref-type="bibr" rid="pgen.1002138-Fried1">[67]</xref>, Strong Heart Family Study (SHFS) <xref ref-type="bibr" rid="pgen.1002138-North1">[68]</xref>, and Strong Heart Cohort Study (SHS) <xref ref-type="bibr" rid="pgen.1002138-Lee1">[69]</xref> (<xref ref-type="table" rid="pgen-1002138-t001">Table 1</xref>). The PAGE study design is detailed in Matise et al <xref ref-type="bibr" rid="pgen.1002138-Matise1">[30]</xref>.</p><p>Serum HDL-C, triglycerides, and total cholesterol were measured using standard enzymatic methods. LDL-C was calculated using the Friedewald equation <xref ref-type="bibr" rid="pgen.1002138-Matise1">[30]</xref>, <xref ref-type="bibr" rid="pgen.1002138-Friedewald1">[70]</xref>, with missing values assigned for samples with triglyceride levels greater than 400 mg/dl. For PAGE study sites with longitudinal data, the baseline measurement was used for analysis. A full description of each study, along with population-specific study characteristics, is presented in <xref ref-type="supplementary-material" rid="pgen.1002138.s016">Text S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1002138.s014">Table S1</xref>.</p><p>All SNPs considered for genotyping were previously associated with HDL-C, LDL-C, and/or triglycerides in published (as of 2008) candidate gene and genome-wide association studies. A total of 52 SNPs were targeted for genotyping by two or more PAGE study sites. There is no overlap between samples used in this study and samples used in GWAS from which the SNPs were selected. The 52 targeted variants are located in or nearby 32 different genes/gene regions, with 12 of the gene/gene regions represented by two or more SNPs. Five SNPs are nonsynonymous, one SNP is a nonsense variant, and two SNPs are synonymous; the remainder are located in introns, flanking, or intergenic regions. The full list of targeted SNPs, their locations, and their previously associated lipid trait can be found in <xref ref-type="supplementary-material" rid="pgen.1002138.s015">Table S2</xref>.</p><p>Cohorts and surveys were genotyped using either commercially available genotyping arrays (Affymetrix 6.0, Illumina 370CNV BeadChip), custom mid- and low-throughput assays (TaqMan, Sequenom, Illumina GoldenGate or BeadXpress), or a combination thereof. Quality control was implemented at each study site independently. In addition to site-specific quality control, all PAGE study sites genotyped 360 DNA samples from the International HapMap Project and submitted these data to the PAGE Coordinating Center for concordance statistics <xref ref-type="bibr" rid="pgen.1002138-Matise2">[71]</xref>. Study specific genotyping details are described in <xref ref-type="supplementary-material" rid="pgen.1002138.s016">Text S1</xref>. Of the 52 targeted SNPs, three (<italic>CETP</italic> rs1800775, <italic>APOE</italic> rs429358, and <italic>APOE</italic> rs7412) failed at all PAGE study sites that attempted genotyping; therefore, a total of 49 SNPs were tested in this analysis.</p><p>All tests of association were performed by each PAGE study site using the same analysis protocol prior to meta-analysis. The study protocol excluded participants &lt;18 years of age as well as non-fasting samples (defined here as &lt;8 hours). When triglyceride level was the dependent variable, participants with &gt;1,000 mg/dl were excluded from analyses. Triglyceride (TG) levels were natural-log transformed (ln) prior to analysis.</p><p>Linear regression was performed for fasting adults regardless of lipid lowering medication use with HDL-C, LDL-C, or ln(TG) as the dependent variable and a SNP as the independent variable, assuming an additive genetic model, stratified by race/ethnicity. The coded allele is reported in <xref ref-type="table" rid="pgen-1002138-t002">Table 2</xref>, <xref ref-type="table" rid="pgen-1002138-t003">Table 3</xref>, <xref ref-type="table" rid="pgen-1002138-t004">Table 4</xref>. The beta estimate is per additional copy of the coded allele. For each SNP, four models were considered: 1) unadjusted, 2) adjusted for age (continuous in years) and sex, 3) adjusted for age, body mass index (continuous in kg/m<sup>2</sup>), current smoking (yes/no; binary), type 2 diabetes (yes/no; binary), post-menopausal status (yes/no for females only; binary), and current hormone use (yes/no for females only; binary), and 4) adjusted for age, body mass index, current smoking, type 2 diabetes, post-menopausal status, current hormone use, and previous myocardial infarction (yes/no; binary). All PAGE study sites (except for WHI, which is female only) stratified models 3 and 4 by sex given the sex-specific variables (post-menopausal status and hormone use) prior to meta-analysis. Select PAGE study sites also included study site or site of ascertainment as a covariate in all models. Results from Model 2 (adjusted for age and sex) are reported in the main text while results from Models 1, 3, and 4 are presented in <xref ref-type="supplementary-material" rid="pgen.1002138.s004">Figures S4</xref>, <xref ref-type="supplementary-material" rid="pgen.1002138.s005">S5</xref>, <xref ref-type="supplementary-material" rid="pgen.1002138.s006">S6</xref>. Model 2 excluding participants on lipid-lowering medications are presented in <xref ref-type="supplementary-material" rid="pgen.1002138.s008">Figures S8</xref>, <xref ref-type="supplementary-material" rid="pgen.1002138.s009">S9</xref>, <xref ref-type="supplementary-material" rid="pgen.1002138.s010">S10</xref>.</p><p>Meta-analyses, using a fixed-effects inverse-variance weighted approach and tests for effect size heterogeneity across studies, were performed using METAL <xref ref-type="bibr" rid="pgen.1002138-Willer2">[72]</xref>. P-values were not adjusted for multiple testing, and association results were plotted using Synthesis-View <xref ref-type="bibr" rid="pgen.1002138-Pendergrass1">[73]</xref>, <xref ref-type="bibr" rid="pgen.1002138-Pendergrass2">[74]</xref>, where indicated. Power calculations were performed using Quanto <xref ref-type="bibr" rid="pgen.1002138-Gauderman1">[75]</xref>, <xref ref-type="bibr" rid="pgen.1002138-Gauderman2">[76]</xref> assuming unrelated participants, an additive genetic model, the published effect size from European-descent populations listed in <xref ref-type="supplementary-material" rid="pgen.1002138.s014">Table S1</xref>, and the population-specific allele frequencies listed in <xref ref-type="table" rid="pgen-1002138-t002">Table 2</xref>, <xref ref-type="table" rid="pgen-1002138-t003">Table 3</xref>, <xref ref-type="table" rid="pgen-1002138-t004">Table 4</xref>. Linkage disequilibrium was calculated using HapMap European (CEU) and West African (YRI) data accessed through the Genome Variation Server. F<sub>ST</sub> was calculated using the Weir and Cockerham algorithm <xref ref-type="bibr" rid="pgen.1002138-Weir1">[77]</xref>. Aggregate data from the meta-analysis as well as individual tests of association from each PAGE study site will be made available via dbGaP <xref ref-type="bibr" rid="pgen.1002138-Matise1">[30]</xref>, <xref ref-type="bibr" rid="pgen.1002138-Mailman1">[78]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">NHGRI GWAS Catalog (<ext-link ext-link-type="uri" ns0:href="http://www.genome.gov/GWAStudies">www.genome.gov/GWAStudies</ext-link>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genome Variation Server (<ext-link ext-link-type="uri" ns0:href="http://pga.gs.washington.edu">pga.gs.washington.edu</ext-link>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Synthesis-View (<ext-link ext-link-type="uri" ns0:href="http://chgr.mc.vanderbilt.edu/ritchielab/method.php?method=synthesisview">http://chgr.mc.vanderbilt.edu/ritchielab/method.php?method=synthesisview</ext-link>).</p>