<p>The subjects enrolled in the GWAS, and in the replication study for WBC subtypes (n&#8202;=&#8202;14,792), and in the pleiotropic association study for hematological traits (n&#8202;=&#8202;30,454) consisted of patients that were classified into 27 disease groups (<xref ref-type="supplementary-material" rid="pgen.1002067.s002">Tables S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1002067.s007">S6</xref>). The subjects in the pleiotropic association study included the subjects in the GWAS and the replication study. All subjects were collected under the support of the BioBank Japan Projects <xref ref-type="bibr" rid="pgen.1002067-Nakamura1">[22]</xref>. Subjects who were determined to be of non-Japanese origin by either self-report or by PCA in GWAS were excluded from analysis. Some of the subjects in this study have also been included in our previous studies <xref ref-type="bibr" rid="pgen.1002067-Kamatani1">[17]</xref>, <xref ref-type="bibr" rid="pgen.1002067-Okada1">[21]</xref>, <xref ref-type="bibr" rid="pgen.1002067-Okada2">[45]</xref>, <xref ref-type="bibr" rid="pgen.1002067-Okada3">[46]</xref>. All participants provided written informed consent as approved by the ethical committees of the BioBank Japan Project <xref ref-type="bibr" rid="pgen.1002067-Nakamura1">[22]</xref> and the University of Tokyo. Clinical information of the subjects including age, gender, and smoking history were collected by self-report on the questionnaire. The laboratory data including the counts of the WBC subtypes and other hematological traits were collected from medical records by the professional medical coordinators according to the standardized protocol <xref ref-type="bibr" rid="pgen.1002067-Nakamura1">[22]</xref>. The details of the study enrolled by the CHARGE Consortium, including subject details and the study design, are described at length elsewhere <xref ref-type="bibr" rid="pgen.1002067-Michael1">[23]</xref>, and are summarized in <xref ref-type="supplementary-material" rid="pgen.1002067.s006">Table S5</xref>.</p><p>In the GWAS for the WBC subtypes, 592,232 SNPs were genotyped for 8,943 subjects using Illumina HumanHap610-Quad Genotyping BeadChip. We excluded 77 subjects with call rates &lt;0.98 in the process of genotyping. After this initial exclusion, SNPs with call rates &lt;0.99 or with ambiguous clustering of the intensity plots, or non-autosomal SNPs, were excluded. We excluded 67 closely related subjects based on the identity-by-descent (IBD), which was estimated using the &#8220;&#8211;genome&#8221; option implemented in PLINK version 1.06 <xref ref-type="bibr" rid="pgen.1002067-Purcell1">[47]</xref>. For each pair with a 1st or 2nd degree of kinship, we excluded the one member of the pair with lower call rates than the other. We then excluded subjects whose ancestries were estimated to be distinct from East-Asian populations using PCA performed by EIGENSTRAT version 2.0 <xref ref-type="bibr" rid="pgen.1002067-Price1">[24]</xref>. We performed PCA for the genotype data of our GWAS along with the genotype data of Phase II HapMap populations (unrelated European (CEU), African (YRU), and East-Asian (JPT + CHB) individuals) (release 24) <xref ref-type="bibr" rid="pgen.1002067-The1">[25]</xref>. Based on the PCA plot of the subjects, we visually identified and excluded 5 outliers in terms of ancestry from JPT + CHB clusters. Subsequently, the SNPs with MAF &lt;0.01 or with an exact P-value of the Hardy-Weinberg equilibrium test &lt;1.0&#215;10<sup>&#8722;7</sup> were excluded. Finally, we obtained 481,110 SNPs for 8,794 subjects.</p><p>After the quality control criteria mentioned above were applied, genotype imputation was performed using MACH 1.0 <xref ref-type="bibr" rid="pgen.1002067-Li1">[26]</xref> in a two-step procedure <xref ref-type="bibr" rid="pgen.1002067-Okada3">[46]</xref>. The genotype data of Phase II HapMap JPT and CHB individuals (release 24) <xref ref-type="bibr" rid="pgen.1002067-The1">[25]</xref> were adopted as references. In the first step of the imputation, recombination and error rate maps were estimated using 500 randomly selected subjects from those enrolled in the GWAS. In the second step, genotype imputation of all subjects was performed using the estimated recombination and error rate maps. Quality control filters of MAF &#8805;0.01 and <italic>Rsq</italic> values &#8805;0.7 were applied for the imputed SNPs.</p><p>The genotype data of the SNPs enrolled in the replication or pleiotropic association study were obtained from the genome-wide screening data of the BioBank Japan Project <xref ref-type="bibr" rid="pgen.1002067-Nakamura1">[22]</xref>. Genotyping was performed using either Illumina HumanHap550v3 Genotyping BeadChip or Illumina HumanHap610-Quad Genotyping BeadChip, and the same quality control filters and imputation procedure were applied.</p><p>The common log-transformed values of the counts of each of the WBC subtypes were adjusted for gender, age, smoking history, and the affection statuses of the subjects with the disease groups (<xref ref-type="supplementary-material" rid="pgen.1002067.s002">Table S1</xref>), using linear regression by <italic>R</italic> statistical software (version 2.11.0). Then the residuals were normalized as <italic>Z</italic> scores, and the subjects with Z score &gt;4.0 or &lt;&#8722;4.0 were excluded in each of the traits. Associations of the SNPs with the counts of the WBC subtypes were assessed by linear regression assuming the additive effects of the allele dosages on the <italic>Z</italic> scores, using mach2qtl software <xref ref-type="bibr" rid="pgen.1002067-Li1">[26]</xref>. In the replication and pleiotropic association studies, the association of the SNPs with the normalized residuals were also evaluated by the linear regression as univariate analysis for each of the phenotypes. The transformation methods used for the hematological traits in the pleiotropic association study are summarized in <xref ref-type="supplementary-material" rid="pgen.1002067.s007">Table S6</xref>. Combined study of the results of the GWAS and the replication study was performed using an inverse-variance method from the summary statistics of beta and standard error (SE). Through the combined study of the GWAS and the replication study, the locus which satisfied the genome-wide significance threshold of <italic>P</italic>&lt;5.0&#215;10<sup>&#8722;8</sup> was considered to be significant. We did not account for multiple comparisons among the traits. These significantly associated loci were subsequently enrolled in the pleiotropic association study. For the selection of the loci that were evaluated in the replication study, we adopted less stringent threshold of <italic>P</italic>&lt;5.0&#215;10<sup>&#8722;6</sup> to include potentially associated loci. For the evaluation of the identified loci using the Caucasian populations, Bonferroni correction based on the number of the evaluated loci were adopted (&#945;&#8202;=&#8202;0.05, n&#8202;=&#8202;12, <italic>P</italic>&lt;0.004). LD between the SNPs in the MHC region and HLA alleles were estimated using the genotype data of the SNPs and the high-resolution HLA alleles for Phase II HapMap JPT and CHB individuals <xref ref-type="bibr" rid="pgen.1002067-The1">[25]</xref>, <xref ref-type="bibr" rid="pgen.1002067-deBakker1">[29]</xref>. Explained proportions of the variations of the WBC subtypes by the combination of the associated SNPs were estimated based on the differences of the coefficient of determination, <italic>R<sup>2</sup></italic>, in the multivariate linear regression model for common-log transformed counts of the respective WBC subtypes, including the associated SNPs as covariates, and the model additionally including age, gender, smoking history, and the affection statuses of the subjects as covariates. Explained proportions of the correlation between the two WBC subtypes by the associated SNPs were estimated based on the following statistics: (<italic>R<sup>2</sup><sub>resi1</sub></italic> - <italic>R<sup>2</sup><sub>resi2</sub></italic>) / <italic>R<sup>2</sup><sub>nomi</sub></italic>, where <italic>R<sup>2</sup><sub>nomi</sub></italic> is the <italic>R<sup>2</sup></italic> between the log-transformed values of the counts of the WBC subtypes, <italic>R<sup>2</sup><sub>resi1</sub></italic> is the <italic>R<sup>2</sup></italic> between the residuals of the values adjusted for gender, age, smoking history, and the affection statuses of the subjects, and <italic>R<sup>2</sup><sub>resi2</sub></italic> is the <italic>R<sup>2</sup></italic> between the residuals of the values adjusted for gender, age, smoking history, the affection statuses of the subjects, and the SNPs. Significance of the statistics was evaluated using permutation procedure (&#215; 10<sup>9</sup> iteration steps).</p><p>The URLs for data presented herein are as follows.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">BioBank Japan Project, <ext-link ext-link-type="uri" ns0:href="http://biobankjp.org">http://biobankjp.org</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">MACH and mach2qtl software, <ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/MACH/index.html">http://www.sph.umich.edu/csg/abecasis/MACH/index.html</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">International HapMap Project, <ext-link ext-link-type="uri" ns0:href="http://www.hapmap.org">http://www.hapmap.org</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">PLINK software, <ext-link ext-link-type="uri" ns0:href="http://pngu.mgh.harvard.edu/~purcell/plink/index.shtml">http://pngu.mgh.harvard.edu/~purcell/plink/index.shtml</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">EIGENSTRAT software, <ext-link ext-link-type="uri" ns0:href="http://genepath.med.harvard.edu/~reich/Software.htm">http://genepath.med.harvard.edu/~reich/Software.htm</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<italic>R</italic> statistical software, <ext-link ext-link-type="uri" ns0:href="http://cran.r-project.org">http://cran.r-project.org</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">SNAP, <ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/mpg/snap/index.php">http://www.broadinstitute.org/mpg/snap/index.php</ext-link>
</p>