<p>This study arose from a collaboration between investigators from two National Institute of Health funded consortia examining the genetic basis of common complex diseases: the Population Architecture using Genomics and Epidemiology (PAGE) study, a National Human Genome Research Institute funded effort examining the epidemiologic architecture of common genetic variation that have been reproducibly associated with human diseases and traits <xref ref-type="bibr" rid="pgen.1002322-Matise1">[55]</xref> and the CARe Consortium <xref ref-type="bibr" rid="pgen.1002322-Musunuru1">[56]</xref>, a National Heart, Lung, and Blood Institute-supported resource for genetic analyses examining cardiovascular phenotypes. Briefly, PAGE investigators participating in the phenomics working group wanted to extend existing efforts examining evidence for pleiotropy in approximately 300 replicated genetic variants <xref ref-type="bibr" rid="pgen.1002322-Pendergrass1">[57]</xref> to include a more comprehensive evaluation of common SNPs. A collaboration between PAGE and CARe investigators was therefore initiated, and used data from five CARe studies of European American and African American with adequate phenotype data: ARIC, CARDIA, CHS, FHS, and MESA. All participating institutions and CARe sites obtained Institutional Review Board approval for this study. Additional information on the participating CARe studies is provided in <xref ref-type="supplementary-material" rid="pgen.1002322.s023">Text S1</xref>.</p><p>The Institute for the Translational Medicine and Therapeutics (ITMAT)-Broad-CARe (IBC) genotyping array <xref ref-type="bibr" rid="pgen.1002322-Keating1">[58]</xref> was used to evaluate approximately 2,100 genes related to cardiovascular, inflammatory, hemostasis/coagulation, and metabolic phenotypes and pathways. The IBC array tagging approach was designed to capture maximal genetic information for both common and lower frequency SNPs (&lt;5% minor allele frequency (MAF)) in HapMap as well as European American and African American populations. The array included 49,320 SNPs, 15,000 of which were gene variants not present in HapMap. Additional details of the SNP selection and tagging approach are given in <xref ref-type="supplementary-material" rid="pgen.1002322.s023">Text S1</xref>.</p><p>Imputation of untyped and missing SNP genotypes was performed using MACH 1.0.16. <xref ref-type="bibr" rid="pgen.1002322-Li4">[59]</xref> For the European samples, phased haplotypes from the CEU founders of HapMap 2 were used as reference. For African American populations, a combined CEU+YRI reference panel was created that includes SNPs segregating in both CEU and YRI, as well as SNPs segregating in one panel and monomorphic and non-missing in the other. Imputation for the IBC array was performed in two steps. First, individuals with pedigree relatedness or cryptic relatedness were filtered. A subset of individuals was randomly extracted from each panel and used to generate recombination and error rate estimates for the corresponding sample. Second, these rates were used to impute all sample individuals across the entire reference panel. Before cleaning, there were an average of 246,740 (range: 245,816, 247,505) and 227,224 (range: 225,111, 229,061) imputed SNPs in the European American and African American study populations, respectively. Imputation results were then filtered at an imputation quality limit of 0.30 and a MAF threshold of 0.01, yielding 235,077 (95.3% of total) and 227,222 (96.2% of total) SNPs for analysis in European American and African American participants, respectively.</p><p>The clustered risk factors of interest were characterized as a six-domain phenotype: atherogenic dyslipidemia, vascular dysfunction, vascular inflammation, pro-thrombotic state, elevated plasma glucose, and central obesity (<xref ref-type="fig" rid="pgen-1002322-g001">Figure 1</xref>). These domains were constructed <italic>a priori</italic> based on a review of literature examining clustering in metabolic phenotypes, placing specific emphasis on the National Cholesterol Education Program&#8217;s Adult Treatment Panel III report <xref ref-type="bibr" rid="pgen.1002322-Treatment1">[4]</xref>, <xref ref-type="bibr" rid="pgen.1002322-Grundy3">[60]</xref>. Nineteen variables were then selected to represent one of the six domains with preference for variables measured in at least four of the contributing cohort studies or variables that were highly correlated with available measures. Measurement protocols for each variable by study are provided in <xref ref-type="supplementary-material" rid="pgen.1002322.s022">Table S21</xref>. We assessed normality, and transformations were used when variables exhibited excessive skewness or kurtosis as determined by numerical summary information and visual inspection of histograms and normal probability plots. Dimension reduction using principal components analysis was then performed for each phenotype domain separately in each race/ethnic and study population. For example, principal components for the vascular inflammation domain were calculated using the following traits: albumin, C reactive protein, fibrinogen, uric acid, and white blood cell count. Principal components were chosen so that&gt;55% of the variance for each domain was explained (<xref ref-type="supplementary-material" rid="pgen.1002322.s013">Tables S12</xref>, <xref ref-type="supplementary-material" rid="pgen.1002322.s014">S13</xref>, <xref ref-type="supplementary-material" rid="pgen.1002322.s015">S14</xref>, <xref ref-type="supplementary-material" rid="pgen.1002322.s016">S15</xref>, <xref ref-type="supplementary-material" rid="pgen.1002322.s017">S16</xref>, <xref ref-type="supplementary-material" rid="pgen.1002322.s018">S17</xref>, <xref ref-type="supplementary-material" rid="pgen.1002322.s019">S18</xref>, <xref ref-type="supplementary-material" rid="pgen.1002322.s020">S19</xref>, <xref ref-type="supplementary-material" rid="pgen.1002322.s021">S20</xref>). This threshold was chosen because all of the first (waist circumference, pro-thrombotic state, elevated plasma glucose, and vascular dysfunction) and the sum of first and second (vascular inflammation and atherogenic dyslipidemia) principal components exceeded 55% across all studies and racial/ethnic groups.</p><p>For each phenotype, we fit a linear regression model relating the phenotype to the SNP genotype under the additive mode of inheritance; the model includes environmental variables (i.e., age, sex and study center) as well as the first ten principal components from EIGENSTRAT to adjust for population substructure <xref ref-type="bibr" rid="pgen.1002322-Price1">[61]</xref>. Ten population substructure components were included because each component was associated with at least one of the eight phenotypes of interest in at least one study. If the SNP genotype is not associated with any phenotype domain, then the regression coefficients for the SNP genotype are zero in all eight linear models. We tested this global null hypothesis by constructing a multivariate test statistic based on the joint distribution of the score statistics from the eight linear models, which accounted for the correlation between the eight phenotypes. We chose the score statistic because it is computationally efficient and numerically stable. The test statistic is referred to the chi-squared distribution with eight degrees of freedom. The genome-wide significance level was set as <italic>P</italic>&lt;2.13&#215;10<sup>&#8722;7</sup> (i.e. 0.05/235,077). Q-Q plots by race are not presented, as our use of a gene-centric array highly enriched for metabolic loci complicated the identification of markers with low prior probabilities of association (i.e. &#8220;null markers&#8221;) for all phenotypes of interest. The data from each cohort were analyzed separately and the results were combined via meta-analysis as described in <xref ref-type="supplementary-material" rid="pgen.1002322.s024">Text S2</xref>. All analyses were stratified by race and were performed in SAS 9.1 and C++. Further details are given in the <xref ref-type="supplementary-material" rid="pgen.1002322.s024">Text S2</xref>.</p>