<p>
<italic>Drosophila</italic> samples were collected with the help of many colleagues around the world (see Acknowledgments, <xref ref-type="table" rid="pgen-1002272-t001">Table 1</xref> and <xref ref-type="supplementary-material" rid="pgen.1002272.s002">Dataset S2</xref>). All samples were obtained from naturally occurring substrates and no artificial baits were used to attract flies. For collections done in Northern California, adults were immediately transferred to sterile no-nutrient media (agar-water) and transported to UC-Davis for dissection, which occurred within 2 hours of collection. For more remote field collections, flies were stored in 100% ethanol for transport.</p><p>Freshly collected flies were washed twice in 2.5% bleach and twice in sterile water. The entire gut was dissected in sterile insect saline and placed in sterile TES buffer (10 mM Tris-HCl [pH&#8202;=&#8202;7,5], 1 mM EDTA, 100 mM NaCl). For flies stored in ethanol, dissection was not feasible because weakening of the fly tissues caused the gut to fragment. For these samples, the entire fly body was used after three washes with sterile water. To ensure adequate removal of external bacteria, each final wash was confirmed to be free of bacterial cells by PCR with universal bacterial primers and by plating onto rich media. In no case did the final wash show evidence of bacterial contamination. For a single sample (<italic>D. melanogaster</italic> reared in the Kopp laboratory), the first wash was saved for DNA extraction to characterize the external bacterial community. Seven to 20 fly bodies or guts were combined for most samples. In one exception (<italic>D. melanogaster</italic> bodies collected from rotting grapes, sample MAW) only a single body was used. On a single occasion, the bacterial community of laboratory media within the Kopp laboratory was sampled using 1 ml of media that had been inhabited by <italic>D. melanogaster</italic> for 7&#8211;10 days. Further details regarding sample collection dates, locations, and contents can be found in <xref ref-type="supplementary-material" rid="pgen.1002272.s001">Dataset S1</xref>.</p><p>DNA was extracted from samples using a modification of the Bead Beater protocol <xref ref-type="bibr" rid="pgen.1002272-McOrist1">[93]</xref>. The tissue was homogenized by grinding and three freeze/thaw cycles on dry ice. Samples were then incubated with 50 units/ml of lysozyme for 15 minutes. Next, physical disruption was performed in a Bead-Beater (BioSpec Products, Inc., Bartlesville, OK) on the homogenize setting for three minutes. An overnight incubation with 1% SDS and 2 mg/ml Proteinase K was followed by extraction with an equal volume of 25&#8758;24&#8758;1 phenol:chloroform:isoamyl alcohol. The aqueous phase was incubated at room temperature for 30 minutes with 2.5 volumes of 100% isopropanol and 0.1 volumes of 3 M sodium acetate before centrifugation at 16,000 g for 30 minutes at 4&#176;C. The DNA pellet was washed with cold 70% ethanol and allowed to air dry before resuspension in TE (10 mM Tris-HCl pH 7.5, 1 mM EDTA.).</p><p>Approximately 100 ng of DNA was used as template for small-subunit rDNA (16S) amplification. Bacterial universal primers 27F (<named-content content-type="gene">5&#8242;- AGAGTTTGATCCTGGCTCAG</named-content>) and 1492R (<named-content content-type="gene">5&#8242;-GGTTACCTTGTTACGACTT</named-content>) were used to amplify a &#8764;1450 bp fragment (Lane, 1991). These primers were chosen for three reasons. First, although they are not truly universal, they are specific to a region that is conserved in many groups of bacteria <xref ref-type="bibr" rid="pgen.1002272-Baker1">[94]</xref>. Second, they allow for the amplification of nearly the full length of the gene, therefore providing consistent comparisons to previous studies of 16S rDNA diversity <xref ref-type="bibr" rid="pgen.1002272-Schloss3">[95]</xref>. Finally, both of these primers have been used in many similar surveys of bacterial diversity, including a previous study of bacterial diversity in <italic>Drosophila melanogaster</italic>
<xref ref-type="bibr" rid="pgen.1002272-Cox1">[32]</xref>. Using these primers allows our results to be directly comparable to those previous studies. The PCR conditions were as follows: initial denaturation for 5 minutes at 95&#176;C; 30 or 35 cycles at 95&#176;C for 30 seconds, 55&#176;C for 30 seconds, and 72&#176;C for 2 minutes; final extension for 10 minutes at 72&#176;C. These PCR conditions were used for all samples, with an annealing temperature of 55&#176;C chosen from a temperature gradient study of 48&#176;C to 58&#176;C because it produced the maximum product yield. The 16S rDNA amplicons were cloned into the pCR4-<italic>TOPO</italic> vector using the <italic>TOPO</italic> TA Cloning Kit. Clones were transformed chemically into One Shot <italic>TOP10</italic> chemically competent <italic>E. coli</italic> cells or via electroporation into ElectroMAX DH10B <italic>E. coli</italic> cells (Invitrogen, Carlsbad, CA) and plated onto agar plates with X-gal and either 50 mg/mL Kanamycin or 50 mg/mL Ampicillin. Colony PCR (20 colonies) was used to verify a &lt;10% insertless rate and &#8764;1.5 kb insert size. White colonies were arrayed into 384-well plates. Prior to sequencing, plasmids were amplified by rolling circle amplification using the TempliPhi DNA Sequencing Amplification Kit (Amersham Biosciences, Piscataway, NJ) and sequenced from both ends using the M13 (&#8722;28 or &#8722;40) primers with the BigDye kit (Applied Biosystems, Foster City, CA). Sequencing reactions were purified using magnetic beads and run on an ABI PRISM 3730 (Applied Biosystems) sequencing machine.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Vector and primer sequences were removed with <italic>cross_match</italic>, a component of the <italic>Phrap</italic> software package <xref ref-type="bibr" rid="pgen.1002272-Ewing1">[96]</xref>, <xref ref-type="bibr" rid="pgen.1002272-Ewing2">[97]</xref>, and bases with a PHRED quality score of Q&gt;&#8202;=&#8202;15 were converted to &#8220;N&#8221;s using JAZZ, the Joint Genome Institute's in-house assembly algorithm. When possible, overlapping regions from the forward and reverse reads of each clone were used to assemble a single contiguous sequence for each clone. In cases where the overlap was not sufficient for assembly, custom perl scripts were used to concatenate the forward and reverse reads with gaps inserted between them (see below). All sequence data are available via BioTorrents (<ext-link ext-link-type="uri" ns0:href="http://biotorrents.net/details.php?id=143">http://biotorrents.net/details.php?id=143</ext-link>) and have been submitted GenBank under the accession numbers JN420379 through JN426767.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used the <italic>Infernal 1.0</italic> software <xref ref-type="bibr" rid="pgen.1002272-Nawrocki1">[98]</xref> to create a single multiple sequence alignment for all of our samples. <italic>Infernal</italic> creates a Hidden Markov Model (HMM) based on a high-quality reference alignment with a fixed length of 1532. 2078 of the 4198 clones consisted of non-overlapping paired reads; for those we created a 5&#8242;-alignment (of reads beginning with the 27F primer sequence) and a 3&#8242;-alignment (of reads ending with the reverse complement of the 1492R primer sequence), and merged the two alignments, inserting gaps into the intervening columns, based on positions in the reference alignment. The concatenated sequences from this &#8220;merged&#8221; alignment were combined with the successfully-assembled, full-length clones to create a single multiple sequence alignment. This alignment is available via BioTorrents (<ext-link ext-link-type="uri" ns0:href="http://biotorrents.net/details.php?id=143">http://biotorrents.net/details.php?id=143</ext-link>).</p><p>For the purposes of OTU (operational taxonomic units <xref ref-type="bibr" rid="pgen.1002272-Schloss4">[99]</xref>) definition and phylogenetic inference, this multiple sequence alignment was further refined to remove column blocks that contained &gt;80% gaps. This resulted in the removal of the first 11 (1&#8211;11) and last 132 (1400&#8211;1532) positions, as well as positions 642&#8211;806 (164 positions) from the middle of the alignment (which primarily corresponded to the regions of non-overlap between reads). A custom perl script was used to remove sequences with fewer than 300 remaining nucleotides from this trimmed alignment. Chimeric sequences were identified and removed using the <italic>chimera.slayer</italic> function within <italic>mothur v.1.11</italic>
<xref ref-type="bibr" rid="pgen.1002272-Schloss1">[35]</xref>.</p><p>We submitted our sequences to the Ribosomal Database Project (RDP10) Classifier for taxonomic assignment <xref ref-type="bibr" rid="pgen.1002272-Cole1">[100]</xref> to the genus level. We were unable to submit a single, full-length sequence for every 16S clone that was sequenced, because for 50% of our clones, there was no overlap between the forward and reverse reads. For each clone, we assigned taxonomy independently to the 5&#8242; read, the 3&#8242; read, and to the full-length or concatenated reads (with intervening gaps inserted, as described above), and then selected a single taxonomy assignment for each 16S clone. We used the measures of confidence (bootstrap values) that are associated with the RDP taxonomy predictions to guide the selection process. Most investigators agree that &gt;70% bootstrap support is indicative of strong support for a phylogenetic clade <xref ref-type="bibr" rid="pgen.1002272-Soltis1">[101]</xref>. In order to arrive at taxonomy predictions with very high confidence, we only considered taxonomy assignments that had bootstrap values of &gt;75% at the genus level, &gt;80% at the family level, &gt;95% at the order level, and 100% at the class level. Strongly supported disagreements between the 5&#8242;, 3&#8242;, and combined data sets were rare (72 total sequences). These were handled in one of two ways: 1) if the conflict was at the level of family or above, they were considered likely to be chimeric sequences and excluded from further analysis, or 2) if the conflict was within a single family, the genus name was changed to &#8220;unclassified&#8221;.</p><p>We used the <italic>mothur</italic> program <xref ref-type="bibr" rid="pgen.1002272-Schloss1">[35]</xref> to generate a distance matrix using our trimmed <italic>Infernal</italic> alignment of 3243 sequences as input. Using the distance matrix created by <italic>mothur</italic>, sequences were clustered using the average neighbor algorithm. Using the 0.03 cutoff option (97% sequence similarity), all sequences fell into 139 OTUs. The average OTU abundance was 23.3 sequences (Min&#8202;=&#8202;1, Max&#8202;=&#8202;638). A representative sequence from each OTU was selected using the <italic>get.oturep</italic> function within <italic>mothur</italic>. This representative sequence and the <italic>dist.seqs</italic> command in <italic>mothur</italic> was used to calculate genetic distances between OTUs and representative sequences throughout this study.</p><p>Taxonomy predictions generated by RDP were mapped onto each sequence within an OTU. In many cases, this led to a clear reassignment of &#8220;unclassified&#8221; sequences to the genus level based on the dominant genus present in that OTU. In other cases, the entire OTU was comprised of &#8220;unclassified&#8221; sequences. These OTUs were assigned names based on their phylogenetic position relative to the reference sequences included, either from the RDP type strains, from other <italic>Drosophila</italic> bacterial microbiome studies, or from GenBank.</p><p>All 7448 good quality 16S sequences longer than 1200 bp from bacterial type strains were downloaded from the RDP website on 8/22/10 <xref ref-type="bibr" rid="pgen.1002272-Cole1">[100]</xref>. These representatives are usually the first identified or most fully characterized strains within a bacterial lineage. Although closely related bacteria may differ substantially in genome content, inclusion of these type strains provides important phylogenetic landmarks during tree building. All 7448 strains were aligned using <italic>Infernal,</italic> and the resulting alignment was trimmed to remove the first 11 (1&#8211;11) and last 132 (1400&#8211;1532) positions, as well as positions 642&#8211;806 (164 positions) from the middle of the alignment as described above.</p><p>All sequences from previous studies of <italic>Drosophila</italic> bacterial communities <xref ref-type="bibr" rid="pgen.1002272-Cox1">[32]</xref>, <xref ref-type="bibr" rid="pgen.1002272-CorbyHarris1">[33]</xref> (Corby-Harris, unpublished) were downloaded from GenBank. <italic>Mothur</italic> was used to create a distance matrix, and OTUs were created at a 97% similarity cutoff. The <italic>get.oturep</italic> function was used to pick a representative sequence for each OTU. Additional <italic>Drosophila</italic>-associated sequences were also included <xref ref-type="bibr" rid="pgen.1002272-Ryu1">[21]</xref>, <xref ref-type="bibr" rid="pgen.1002272-Juneja1">[37]</xref>. Finally, for OTUs in our study that do not have any closely related sequences within the RDP database (such as <italic>Enterobacteriaceae Group Orbus</italic>) the closest BLAST hits from GenBank were included. A list of the RDP and GenBank accession numbers for sequences used in the final tree are found in the <xref ref-type="supplementary-material" rid="pgen.1002272.s005">Dataset S5</xref>.</p><p>To compare our results to mammalian studies, the 17,504 ultraclean sequences from <xref ref-type="bibr" rid="pgen.1002272-Ley2">[12]</xref> were analyzed. To obtain a sample that was roughly equal to our data in taxonomic breadth, only the sequences from <italic>Artiodactyla</italic>, <italic>Carnivora</italic>, and <italic>Primate</italic> samples were analyzed. These sequences were aligned and trimmed as above and a full PCA analysis was performed using the <italic>Fast UniFrac Interface</italic>
<xref ref-type="bibr" rid="pgen.1002272-Hamady1">[62]</xref>. Taxonomic classifications were done with RDP <xref ref-type="bibr" rid="pgen.1002272-Cole1">[100]</xref>.</p><p>Diversity measurements were calculated for each library from both <italic>Drosophila</italic>
<xref ref-type="bibr" rid="pgen.1002272-CorbyHarris1">[33]</xref> and mammalian <xref ref-type="bibr" rid="pgen.1002272-Ley2">[12]</xref> datasets using <italic>mothur</italic>
<xref ref-type="bibr" rid="pgen.1002272-Schloss1">[35]</xref>. The proportion of OTUs unique to each library was calculated for each group (as in Ley et al., 2008a) <xref ref-type="bibr" rid="pgen.1002272-Ley2">[12]</xref>.</p><p>Using representative sequences from our dataset, previously identified <italic>Drosophila</italic>-associated bacteria, representative type strains from the RDP database, and sequences obtained from GenBank (see previous section), a phylogenetic tree was created with <italic>FastTree</italic>
<xref ref-type="bibr" rid="pgen.1002272-Price1">[36]</xref>. Default settings with the GTR (generalized time-reversible) model were used. The entire tree was rooted using <italic>Thermus thermophilus</italic> (RDP identifier S000381199). After an initial run with all 8,407 sequences, many clades were removed from the alignment (for example, bacterial Phyla in which no <italic>Drosophila</italic> associated sequences were present). The remaining 1349 aligned sequences were then re-run on <italic>FastTree</italic> using the settings described above. Final publication quality images were prepared using <italic>Dendroscope</italic>
<xref ref-type="bibr" rid="pgen.1002272-Huson1">[102]</xref>.</p><p>Tests of significance of differences between samples were performed using <italic>UniFrac</italic> and <italic>FastUniFrac</italic>
<xref ref-type="bibr" rid="pgen.1002272-Lozupone1">[61]</xref>, <xref ref-type="bibr" rid="pgen.1002272-Hamady1">[62]</xref>. The low depth of coverage provided by the sequencing method used is sufficient to find significant results using <italic>UniFrac</italic>
<xref ref-type="bibr" rid="pgen.1002272-Kuczynski1">[103]</xref>. Because of the correction for multiple comparisons, pairwise comparisons for each library were not feasible with the amount of data collected. We therefore parsed all data into bins representing different host diets to estimate the overall effect of this factor. The effect of different experimental treatments was determined similarly. All comparisons of bacterial communities are given in the <xref ref-type="supplementary-material" rid="pgen.1002272.s020">Text S1</xref>. Co-occurrence tests were also performed (as in <xref ref-type="bibr" rid="pgen.1002272-Chaffron1">[104]</xref>), but inadequate power precluded the finding of any significant co-occurring pairs of taxa (additional details in <xref ref-type="supplementary-material" rid="pgen.1002272.s020">Text S1</xref>).</p><p>Unless explicitly stated, flies were fed unsterilized standard lab media (<xref ref-type="supplementary-material" rid="pgen.1002272.s020">Text S1</xref>). All transfer steps were performed near a Bunsen burner flame and all surfaces and instruments were frequently sterilized with 70% ethanol. For bacterial DNA extraction, flies were washed to remove external bacterial cells and their intestines dissected as described above. All negative controls were confirmed to be bacteria-free by plating onto MRS media and PCR with universal bacterial primers. Separate libraries were created from adult Canton-S males, Oregon-R males, and Oregon-R females for Deborah Kimbrell's lab (UC-Davis, CA). In addition, a large population of wild <italic>D. melanogaster</italic> originally collected from Winters, CA was established in the Kopp lab for use in dietary treatment experiments (strain WO).</p><p>For diet experiments, approximately 25 flies were transferred to each of 5 separate diets, with one vial per treatment. The diets included standard lab media, high yeast media, high yeast supplemented with 6% ethanol, sugar-agar, and agar only (see <xref ref-type="supplementary-material" rid="pgen.1002272.s020">Text S1</xref> for media composition). All media were initially sterilized in an autoclave, with the exception of the standard lab media. Ethanol was added to the ethanol treatment after media cooled below 55&#176;C. To reduce the effect of the media-dwelling bacterial population that arose after contact with non-sterile flies, cultures were transferred daily to fresh sterile media, with the exception of the standard lab diet. These transfers continued for three days on all media except the agar-only diet, where starvation-induced death limited the experiment to two days. Four hours after the final transfer, the intestines of 10 flies per treatment were dissected for analysis.</p><p>For the multiple species experiment, approximately 25 adults each of <italic>D. melanogaster</italic>, <italic>D. elegans,</italic> and <italic>D. virilis</italic> were combined on sterilized high yeast media. After three days of daily transfers as above, 10 males per species were dissected for analysis.</p>