<p xmlns:ns0="http://www.w3.org/1999/xlink">Embryos were collected, fixed and prehybridized according to standard protocols, which are available at <ext-link ext-link-type="uri" ns0:href="http://depace.med.harvard.edu/links.html">http://depace.med.harvard.edu/links.html</ext-link>, and described in <xref ref-type="bibr" rid="pgen.1002346-LuengoHendriks1">[6]</xref>, <xref ref-type="bibr" rid="pgen.1002346-Kernen1">[26]</xref>. Briefly, <italic>D. yakuba</italic> and <italic>D. pseudoobscura</italic> cages were maintained at 23&#176;C. <italic>D. yakuba</italic> embryos were collected for 3 hours, and aged for 2 hours prior to fixation. <italic>D. pseudoobscura</italic> embryos were collected for 3 hours, and aged for 3 hours prior to fixation. Embryos were dechorionated in 50% bleach for 3 minutes, washed, and fixed in a 1&#8758;4 solution of 10% formaldehyde (Polysciences #04018) to heptane for 20 minutes with vigorous shaking. The vitelline membranes were removed by shaking with MeOH and washed 3X with 100% MeOH. Fixed embryos were stored at &#8722;20&#176;C in 100% ethanol. Embryos were pooled for prehybrization, rehydrated in PBT + Tx (PBS pH 7.2, 0.05% Tween20 and 0.2% Triton X-100), post-fixed for 20 minutes in 5% formaldehyde in PBT+Tx, washed in hybridization buffer (50% formamide, 5X SSC pH 5.2, 0.2% Triton X-100, 40 &#181;g/ml heparin, and 250 &#181;g/ml salmon sperm DNA) and incubated at 55&#176;C for 1 to 5 hours in hybridization buffer. Prehybridized embryos were stored in hybridization buffer at &#8722;20&#176;C.</p><p>There were two modifications to the staining protocol developed for <italic>D. melanogaster</italic>
<xref ref-type="bibr" rid="pgen.1002346-LuengoHendriks1">[6]</xref>. First, species-specific RNA probes were made using cDNA or genomic DNA as a template, whereas cDNA probes were used exclusively for the <italic>D. melanogaster</italic> data. Probes ranged in size from 531 bp to 2771 bp, and either encompassed the majority of the coding sequence or overlapped large exons (<xref ref-type="supplementary-material" rid="pgen.1002346.s015">Table S4</xref>). Variation in probe length did not significantly affect our measurements (<xref ref-type="supplementary-material" rid="pgen.1002346.s011">Figure S11</xref>). Second, two different haptens are required for our imaging pipeline, one for the registration gene and one for the gene of interest. While dinitrophenol (DNP) - labeled probes gave consistently clean results in all species, digoxygenin (DIG) - labeled probes yielded variable levels of background. Another commonly used hapten, biotin, was even worse. Because DIG stains were strong enough to reliably distinguish stripes, we chose to use it for the registration channel, but not include the data in the final gene expression atlases.</p><p>Probe templates were cloned by PCR amplification using either genomic DNA or cDNA libraries as a template, ligated into pGEM-Teasy, and sequence verified. Cloning primers are listed in <xref ref-type="supplementary-material" rid="pgen.1002346.s015">Table S4</xref>. Probe templates were generated by PCR with M13 forward and reverse primers. Anti-sense digoxygenin (DIG) or dinitrophenol (DNP) probes were synthesized using in vitro transcription from DNA templates using either SP6 or T7 polymerase, depending on the orientation of the clone. Probes were not carbonate-treated as this did not improve stain quality. All probes were diluted to 200 ng/&#181;l.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For <italic>in situ</italic> hybridizations, approximately 100 &#181;l of embryos were incubated for up to 48 hours at 55&#8211;57&#176;C in 300 &#181;l of hybridization buffer with 2&#8211;10 &#181;l each of a DIG and DNP probe. Embryos were then washed extensively with hybridization buffer at 55&#8211;57&#176;C, and probes were detected sequentially using horseradish-peroxidase (HRP) conjugated antibodies (anti-DIG POD, Roche 11207733910 at 1&#8758;250 or 1&#8758;500; anti-DNP Perkin Elmer NEL747 A001KT at 1&#8758;100) and either coumarin or Cy3 tyramide amplification (Perkin-Elmer NEL703 001KT, SAT 704B). To disable the HRP in the first signal detection reaction, embryos were washed in hybridization buffer at 55&#176;C and incubated in 5% formaldehyde in PBT+Tx for 20 minutes. All remaining RNA was removed by incubation with 0.18 &#181;g/ml RNAse A in 500 &#181;l PBT+Tx overnight at 37&#176;C. Nuclei were detected by staining with Sytox Green (Molecular Probes #S7020, 1&#8758;5000 in 500 &#181;l overnight at 4&#176;C). Embryos were dehydrated in an ethanol series and mounted in xylene-based DePex (Electron Microscopy Service #13514) on a slide with 2 bridging coverslips to prevent flattening of the embryos. Detailed protocols are available at <ext-link ext-link-type="uri" ns0:href="http://depace.med.harvard.edu/links.html">http://depace.med.harvard.edu/links.html</ext-link>.</p><p>Three-dimensional image stacks of individual embryos were acquired semi-automatically on a Zeiss LSM 710 using a plan-apochromat 20X 0.8NA objective. Embryos were located, staged using phase contrast optics, and the imaging parameters such as the height of the image stack and gain settings for each fluorophore were recorded. A custom built macro then acquired all marked embryos <xref ref-type="bibr" rid="pgen.1002346-Fowlkes1">[7]</xref>. All three fluorophores (Sytox Green, coumarin and Cy3) were excited simultaneously at 750 nm, using a Coherent Chameleon 2-photon laser at 4&#8211;7% power. The emission was spectrally split into 3 channels: 462&#8211;502 nm (coumarin), 514&#8211;543 nm (sytox), 599&#8211;676 nm (Cy3). Images were 1024&#215;1024, and slices were taken every 1 &#181;m. Resulting image stacks were processed by previously described algorithms to unmix channels <xref ref-type="bibr" rid="pgen.1002346-Hendriks1">[56]</xref>, and segment individual nuclei <xref ref-type="bibr" rid="pgen.1002346-LuengoHendriks1">[6]</xref>, resulting in individual pointcloud files for each embryo. These were housed in a custom-built database.</p><p>Gene expression atlases were assembled using the registration algorithms previously described in <xref ref-type="bibr" rid="pgen.1002346-Fowlkes1">[7]</xref>. For each species, a morphological model was constructed that contained an average number of nuclei. The 3D positions of the nuclei in the model were chosen to match the average egg-length, shape and density pattern measured for each of the 6 temporal cohorts. Motions of nuclei between time points in the model were constrained to be as small and smooth as possible while still recapitulating the observed changes in density and shape (see <xref ref-type="bibr" rid="pgen.1002346-Fowlkes2">[27]</xref> for details). Pointcloud data extracted for each embryo in a given cohort were aligned to the morphological template by a rigid-body transformation and isotropic scaling. For each time point, a registration template was constructed by finding average boundary locations of a registration marker gene (eve or ftz) with respect to the egg-length of the morphological model. Fine registration of individual embryo pointclouds was then carried out by non-rigid warping of the embryo to align marker gene boundaries with the template. Finally, expression values were computed for each nucleus and time point in the model by averaging measurements across those nuclei in individual pointclouds that were closest after spatial registration. Prior to averaging, gains and offsets were estimated for expression measurements within each embryo pointcloud in order to minimize the expression variance across the cohort and to match smoothed estimates of the total change in expression level between temporal cohorts (see <xref ref-type="bibr" rid="pgen.1002346-Fowlkes1">[7]</xref> for details).</p><p>Surface area was computed as the sum of areas of the triangles defined by the neighbor relation information in the Pointclouds <xref ref-type="bibr" rid="pgen.1002346-LuengoHendriks1">[6]</xref>. Local density was computed by defining a disk of 15 &#181;m radius on the surface around each nucleus, and dividing the number of nuclei in this disk by its area <xref ref-type="bibr" rid="pgen.1002346-LuengoHendriks1">[6]</xref>. These density maps were then averaged over a cohort of embryos by resampling the cylindrical projections onto a regular grid.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Cell-to-cell comparisons within and between species were made by looking at the squared distance between vectors of average expression measurements for the cell at all 6 time points and 11 genes. For a pair of nuclei <bold><italic>i</italic></bold> and <bold><italic>j</italic></bold> we computed the distance:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002346.e001.jpg" /></disp-formula>where <bold>e</bold>
<sub>i</sub>
<sup>gt</sup> is the expression of the <bold><italic>g</italic></bold>th gene recorded in the atlas for the <bold><italic>i</italic></bold>th cell at time point <bold><italic>t</italic></bold>. We used squared distance since it is additive across genes and time-points which makes the contribution of individual genes more interpretable. Prior to computing the distance, expression levels for each gene in the atlas were scaled so that the maximum expression at each time point was 1.0. In order to determine relevant cells to compare between species, only cells that were nearby were considered. Corresponding locations were estimated by scaling each atlas to unit egg length and nearby nuclei were specified as those nuclei in the target embryo that were within the 30 nearest to the cell to be matched.</p><p>Since there are often several cells that are good matches, the displacement direction to the best matching nearby cell is noisy. In order to visualize displacement, we used a weighted average of the locations of the top 10 matching cells (smallest expression distance). The 3D locations of these 10 matching cells were averaged using weights inversely proportional to the expression distance (i.e. 1/<bold><italic>d<sub>ij</sub></italic></bold>). These 3D displacement vectors were then visualized on a cylindrical projection.</p><p>We chose not to include bcd and cad in calculation of the expression distance score for the entire dataset. We excluded bcd because its expression increases over the first two time points in the <italic>D. melanogaster</italic> dataset; this is likely an experimental artifact and leads to artificially high expression distance scores in the anterior. We excluded cad because data was not available for <italic>D. pseudoobscura</italic> and we wished to compare results between the <italic>D. melanogaster/D. yakuba</italic> and <italic>D. melanogaster/D. pseudoobscura</italic> analyses.</p><p>As an alternative to expression distance scores, we also considered a hypothesis-testing framework in which two cells are declared to have different expression profiles if the expression of some gene at some time point is significantly different relative to variance in our measurements. This comparison was carried out independently for each cell, gene and time-point using a two-sample t-test with unequal sample sizes and variances. In all tests we used the Bonferroni correction to assure a family-wise error rate of less than 0.01. Visualizations and histograms in <xref ref-type="supplementary-material" rid="pgen.1002346.s005">Figures S5</xref> and <xref ref-type="supplementary-material" rid="pgen.1002346.s009">S9</xref> show the number of expression profile measurements for which a given cell was significantly different under this significance threshold.</p><p>We defined an expression neighborhood <bold>N<sub>i</sub></bold> for a given nucleus <bold>i</bold> in the following way. Choose a threshold <bold>t</bold> and find all nuclei in the atlas for which <bold>d<sub>ij</sub></bold>&lt;<bold>t</bold>. Of these nuclei with similar spatio-temporal expression profiles, let <bold>N<sub>i</sub></bold> be the largest connected component on the embryo surface that contains cell <bold>i</bold>. For areas where the expression pattern varies rapidly in space, this neighborhood of similar cells is small. In areas where the pattern changes slowly, the neighborhood is large.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To determine how these neighborhoods might expand or contract between different species, we consider the neighborhood size around corresponding nuclei. Let <bold><italic>j</italic></bold> be the nucleus in the target atlas whose expression profile best matches <bold><italic>i</italic></bold> in the source atlas. We compare the relative sizes of the two neighborhoods in order to gauge the degree of expansion or contraction measured by the log ratio of neighborhood sizes:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002346.e002.jpg" /></disp-formula>
</p><p>The log ratio is symmetric about zero with positive values indicating an expansion and negative values indicating a contraction.</p><p>One concern is that the choice of neighbor threshold may affect this analysis since there is not a meaningful way to scale the measured fluorescence levels between atlases of different species. To resolve this, we choose the threshold for each target atlas adaptively. Given a fixed threshold for the query atlas, we searched over thresholds for the target atlas in order to find a threshold in which the average expansion ratio R across all cells matched the log-ratio of the number of nuclei in the two atlases. Choosing the threshold in this way entails that ratios are visualized relative to a null hypothesis of uniform scaling between species.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Figures similar to <xref ref-type="fig" rid="pgen-1002346-g005">Figure 5</xref>, <xref ref-type="fig" rid="pgen-1002346-g006">Figure 6</xref>, <xref ref-type="supplementary-material" rid="pgen.1002346.s004">Figure S4</xref>, <xref ref-type="supplementary-material" rid="pgen.1002346.s007">Figure S7</xref>, and <xref ref-type="supplementary-material" rid="pgen.1002346.s008">Figure S8</xref> can easily be generated using MulteeSum, a custom software tool for visualizing comparative analysis of cellular gene expression profiles <xref ref-type="bibr" rid="pgen.1002346-Meyer1">[57]</xref>. Our datasets are complex and best viewed interactively. We therefore have made MulteeSum and the analyses presented here available for download at <ext-link ext-link-type="uri" ns0:href="http://depace.med.harvard.edu/downloads/MulteeSum.zip">http://depace.med.harvard.edu/downloads/MulteeSum.zip</ext-link>. We have released MulteeSum open source, and it was developed using the Processing programming language (<ext-link ext-link-type="uri" ns0:href="http://www.processing.org">http://www.processing.org</ext-link>), an open-source language for visualization. Executables for running on Mac OSX, Windows and Linux and instructions (see README.txt) are included in the download. A full description of the usage and features of MulteeSum can be found at <ext-link ext-link-type="uri" ns0:href="http://www.multeesum.org">http://www.multeesum.org</ext-link>.</p>