<p>All cohort participants gave written informed consent, including consent to participate in genetic studies. All studies received approval from local ethical oversight committees.</p><p>The data were obtained from 2 cohort studies in the CHARGE Consortium, the Atherosclerosis Risk in Communities (ARIC) Study and the Cardiovascular Health Study (CHS), and 3 additional cohort studies, the Coronary Artery Risk Development in Young Adults (CARDIA) Study, the Invecchiare in Chianti (InCHIANTI) Study, and the Multi-Ethnic Study of Atherosclerosis (MESA).</p><p>In all cohorts but InCHIANTI, plasma phospholipids were first isolated by thin layer chromatography; fatty acids were then separated by gas chromatography. In InCHIANTI, total plasma fatty acids were measured using a similar gas chromatography technique. Details of fatty acid measurements are provided in <xref ref-type="supplementary-material" rid="pgen.1002193.s001">Text S1</xref>. Levels of EPA, DHA, ALA and DPA were expressed as % of total fatty acids.</p><p>Genotyping was done in each cohort separately using high-density SNP marker platforms (ARIC, CARDIA and MESA - Affymetrix 6.0, CHS - Illumina 370, InCHIANTI - Illumina 550). Samples with call rates below 95% (ARIC, CARDIA, MESA), or 97% (CHS, InCHIANTI) at genotyped markers were excluded. Genotypes were imputed to approximately 2.5 million HapMap SNPs by using either MACH <xref ref-type="bibr" rid="pgen.1002193-Li1">[48]</xref> (ARIC, InCHIANTI), BIMBAM <xref ref-type="bibr" rid="pgen.1002193-Servin1">[49]</xref> (CHS), BEAGLE <xref ref-type="bibr" rid="pgen.1002193-Browning1">[50]</xref> (CARDIA) or IMPUTE <xref ref-type="bibr" rid="pgen.1002193-Marchini1">[51]</xref> (MESA). SNPs for which testing Hardy Weinberg equilibrium resulted in <italic>p&lt;</italic>10<sup>&#8722;5</sup> (CHS) or <italic>p</italic>&lt;10<sup>&#8722;6</sup> (ARIC) were excluded from imputation. SNPs with minor allele frequency (MAF) &#8804;1% were excluded from the meta-analyses. Additional details on genotyping and imputation per cohort are provided in <xref ref-type="supplementary-material" rid="pgen.1002193.s001">Text S1</xref>.</p><p>Association analysis between genotype and each fatty acid was done separately within each study cohort according to a pre-specified plan. All studies conducted linear regression analysis using an additive genetic model, i.e. regression of phenotype on the number of reference alleles, or equivalently the imputed dosage for imputed genotypes. All analyses were adjusted for age, sex, and site of recruitment where appropriate, and used robust standard errors. In addition, CARDIA, CHS and MESA analyses were adjusted for principal components to account for possible population genetic substructure. The results in InCHIANTI included in the present study have been previously published <xref ref-type="bibr" rid="pgen.1002193-Tanaka1">[26]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For each SNP and fatty acid, GWAS-specific results were combined using inverse-variance weighted meta-analysis in METAL (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/metal">www.sph.umich.edu/csg/abecasis/metal</ext-link>). Genomic control correction was applied to each study prior to the meta-analysis. Genomic control correction factors ranged from 1.00 to 1.07 (ALA), 1.00&#8211;1.08 (EPA), 1.00&#8211;1.03 (DPA) and 1.01&#8211;1.13 (DHA). P-values less than 5&#215;10<sup>&#8722;8</sup> were considered significant. Because total plasma levels of ALA (measured in InCHIANTI) are higher than plasma phospholipid levels of ALA (measured in the other cohorts), we performed a z-score based meta-analysis of ALA with the 5 cohorts as a sensitivity analysis. Results did not differ from that of inverse-variance weighted meta-analysis, i.e. from those presented. The proportion of fatty acid variance explained by a particular variant allele was calculated from the formula.</p><p>(&#946;<sup>2</sup>*2*MAF(1-MAF))/Var(Y), where &#946; is the regression coefficient for one copy of the allele, MAF is the minor allele frequency and Var(Y) is the variance of the fatty acid.</p><p>We tested 13 interactions using cross-products in the linear regression models. Two of the most associated SNPs available in all cohorts (rs1535 in <italic>FADS2</italic> and rs3734398 in <italic>ELOVL2</italic>) were chosen for investigation of interactions with a) fatty fish intake (dichotomized at 0.6 servings/week, a cut-point around the 25<sup>th</sup> percentile of fish consumption in the CHS and ARIC cohorts), b) plasma phospholipid ALA levels (continuous linear) and c) plasma phospholipid LA levels (continuous linear) on the outcomes of EPA and DHA. Additionally, we tested the interaction between rs1535 and plasma phospholipid EPA with DHA levels as the outcome. Interaction coefficients from cohort-specific analyses were meta-analyzed. For interactions of SNPs with ALA on the outcomes of EPA and DHA, we performed z-score meta-analysis with all the cohorts to assess statistical significance, and inverse-variance weighted meta-analysis excluding InCHIANTI to estimate the magnitude of the interaction. P-values less than 0.004 (0.05/13 tests) were considered significant for the interactions.</p><p>We investigated the association of two selected SNPs which had been directly genotyped as part of candidate gene studies in the African American cohort in CARDIA and the African, Chinese and Hispanic American cohort in MESA, and which were available from genome-wide scans on African Americans in the CHS cohort. We used linear regression and additive models as described above. Results in the 3 African American cohorts were meta-analyzed using inverse-variance weighted meta-analysis.</p>