<p>Mice were maintained on a 12&#8758;12-h light-dark cycle. All experiments and procedures were approved by the Animal Care and Use Committees of the National Institute of Diabetes and Digestive and Kidney Diseases, National Institute of Neurological Disorders and Stroke and National Institute on Deafness and Other Communication Disorders. Twirler mice were a kind gift from Drs. Miriam Meisler and Siew-Ging Gong at the University of Michigan and were maintained on a C57BL/6J background by backcrossing heterozygous <italic>Tw</italic> males to C57BL/6J females for at least 30 generations. <italic>Zeb1<sup>&#916;Ex1</sup></italic> mice <xref ref-type="bibr" rid="pgen.1002307-Takagi1">[10]</xref> were a generous gift from Dr. Douglas Darling and were serially backcrossed to C57BL/6J to maintain the line.</p><p>Bacterial artificial chromosome (BAC) clone RP23-135A18 containing mouse genomic DNA encoding exon 1 of <italic>Zeb1</italic> was digested with PacI/SphI and SphI to yield 7.6-kb and 2.6-kb homology arms, respectively, for targeting constructs (<xref ref-type="fig" rid="pgen-1002307-g006">Figure 6</xref>). Each targeting construct included loxP sites flanking a splice acceptor site and internal ribosomal entry site (IRES) (pGT1.8IresBgeo, provided by Austin Smith at University of Edinburgh) <xref ref-type="bibr" rid="pgen.1002307-Mountford1">[44]</xref>, <italic>E. coli lacZ</italic>, and a reverse-oriented pPGK-neomycin resistance cassette cloned into the pPNT plasmid <xref ref-type="bibr" rid="pgen.1002307-Tybulewicz1">[45]</xref> (<xref ref-type="fig" rid="pgen-1002307-g006">Figure 6</xref>). The wild type (<italic>KI<sup>G</sup></italic>) and Twirler (<italic>KI<sup>A</sup></italic>) 7.6-kb PacI/SphI homology arms contained G and A at position c.58+181, respectively. Bruce4 embryonic stem (ES) cells <xref ref-type="bibr" rid="pgen.1002307-Kontgen1">[46]</xref> were electroporated with the <italic>KI<sup>G</sup></italic> or <italic>KI<sup>A</sup></italic> targeting constructs and grown in the presence of G418 and ganciclovir, using standard protocols at the University of Michigan Transgenic Animal Model Core <xref ref-type="bibr" rid="pgen.1002307-Hughes1">[47]</xref>. G418-resistant ES clones were screened for homologous recombination by PCR and Southern blot analyses. At least three recombinant ES cell lines for each targeting construct were injected into C57BL/6 blastocysts. Chimeric males were mated with C57BL/6 females and offspring were analyzed by Southern blot and PCR analyses for germline transmission of <italic>KI<sup>G</sup></italic> or <italic>KI<sup>A</sup></italic> (<xref ref-type="supplementary-material" rid="pgen.1002307.s003">Figure S3</xref>). <xref ref-type="supplementary-material" rid="pgen.1002307.s004">Table S1</xref> shows PCR primer pairs used to genotype KI alleles before and after neomycin cassette removal (<xref ref-type="fig" rid="pgen-1002307-g006">Figure 6</xref>). Mice transmitting <italic>KI<sup>G</sup></italic> or <italic>KI<sup>A</sup></italic> in the germline were crossed to <italic>Cre</italic> recombinase-expressing mice (C57BL/6-TgN(Zp3-Cre)93Knw, Jackson laboratory, ME) to delete the IRES-<italic>lacZ</italic>-neomycin resistance cassette, leaving a single loxP site 606 bp downstream from <italic>Zeb1</italic> exon 1.</p><p>A comprehensive gross anatomical, histological, and serological analysis of three 15-week-old <italic>Tw</italic>/+ and three wild type littermate males was performed as described <xref ref-type="bibr" rid="pgen.1002307-BenYosef1">[48]</xref>. Tissue sections from two <italic>Tw/Tw</italic>, two <italic>Tw</italic>/+ and two wild type mice at postnatal day 0 (P0) were analyzed.</p><p>Heterozygous <italic>Tw</italic> males and females were mated. Pregnant females were identified by the presence of a vaginal plug and gestational stage was estimated by defining that morning as 0.5 days post-conception (dpc). Embryos at 14.5 dpc were harvested and processed for paint-filling as described <xref ref-type="bibr" rid="pgen.1002307-Morsli1">[49]</xref>. The length of the cochlear duct was measured along its outer contour from a ventral view <xref ref-type="bibr" rid="pgen.1002307-Bok1">[50]</xref>. For scanning electron microscopy (SEM), whole-mounted inner ears were fixed in 2.5% glutaraldehyde in 0.1 M sodium cacodylate with 2 mM CaCl<sub>2</sub> for 90 min. The organ of Corti, saccule, utricle, and crista ampullaris were dissected free in water and dehydrated with a serial dilution series of acetone. Samples were critical point-dried and sputter-coated followed by visualization with a field-emission scanning electron microscope (S-4800, Hitachi). Auditory brainstem response (ABR) thresholds were measured in response to click or pure-tone stimuli of 8, 16, or 32 kHz as described <xref ref-type="bibr" rid="pgen.1002307-Noguchi1">[51]</xref>.</p><p>Six <italic>Tw/+</italic> male, six <italic>Tw/+</italic> female, six wild type male and six wild type female mice were housed individually with regular mouse chow and water provided <italic>ad libitum</italic>. Body weights were measured weekly from 5 weeks of age. Weekly food intake was measured from weeks 6 through 22 to calculate average daily food intake. At 23 weeks of age, mice were transferred to the NIDDK Mouse Metabolism Core Laboratory for measurement of oxygen consumption, carbon dioxide production and motor activity as described <xref ref-type="bibr" rid="pgen.1002307-Kim1">[52]</xref>. Body composition was measured using Echo3-in-1 NMR analyzer (Echo Medical Systems, Houston, TX). Tail vein blood was used for serologic analyses. Fifteen-week-old female mice (eight <italic>Tw/+</italic>, six wild type) were tested for glucose and insulin tolerance as described <xref ref-type="bibr" rid="pgen.1002307-Kim1">[52]</xref>. All data are expressed as a mean &#177; SEM. Student's <italic>t</italic>-test was used to identify statistically significant differences between genotype groups.</p><p>Twirler males (C57BL/6J-<italic>Tw/+)</italic> were crossed with DBA/2J or Castaneus (CAST/Ei) females since Twirler females are poor caretakers of offspring. Male (C57BL/6J-<italic>Tw/+</italic> x DBA/2J)F1-<italic>Tw/+</italic> or (C57BL/6J-<italic>Tw/+</italic> x CAST/Ei)F1-<italic>Tw/+</italic> progeny were backcrossed with DBA/2J or C57BL/6J females, respectively, to generate 337 and 1679 N2 backcross progeny, respectively. Progeny were scored for circling behavior or obesity by visual inspection.</p><p>We genotyped short tandem repeat (STR) markers on 337 DBA/2J N2 backcross progeny to identify two STR markers (D18Mit65, D18Mit64, D18Mit19 and D18Umi1) flanking each side of <italic>Tw</italic>. These markers were genotyped in the 1679 CAST/Ei N2 backcross progeny to identify recombinations in the <italic>Tw</italic> region. The <italic>Tw</italic> map interval was defined by genotypes of additional markers in the recombinants. We genotyped MIT markers between D18Mit65 and D18Umi1, as well as 40 novel STR markers (denoted D18Nih1 through D18Nih44; PCR primer sequences listed in <xref ref-type="supplementary-material" rid="pgen.1002307.s004">Table S1</xref>) located between D18Mit19 and D18Mit219.</p><p>Genomic DNA of <italic>Tw/Tw, Tw/+</italic> and wild type mice were isolated for PCR amplification as described <xref ref-type="bibr" rid="pgen.1002307-Bork1">[53]</xref>. The primers were designed to amplify and sequence all of the annotated exons of the <italic>Zeb1, Zeb1os</italic> (MGI predicted gene <italic>Gm10125</italic>) and <italic>Zfp438</italic> genes in the <italic>Tw</italic> critical interval. Additional novel exons were identified by 5&#8242; and 3&#8242;- RACE (5&#8242; and 3&#8242; rapid amplification of cDNA ends) of the <italic>Zeb1, Zeb1os</italic> and <italic>Zfp438</italic> genes. This revealed multiple alternative first exons for <italic>Zeb1</italic> that were also sequenced. Reverse transcription (RT)-PCR was performed to amplify and sequence full-length cDNA clones of the three genes using whole body mRNA collected from embryonic <italic>Tw/Tw, Tw/+</italic> and wild type littermates.</p><p>PCR reaction conditions were modified to amplify and sequence the overlapping genomic region of <italic>Zeb1</italic> and <italic>Zeb1os</italic>. Fifty-&#181;l PCR reactions contained 50 to 100 ng of genomic DNA, 5 pmol each of forward and reverse primers, 200 mM each dNTP, 0.5 M betaine, 10% dimethyl sulfoxide (DMSO), 2.5 mM MgCl<sub>2</sub>, and 0.5 U of thermostable polymerase. Thermal cycling conditions were: 95&#176;C for 1 min; 33 cycles of 20 s at 95&#176;C, 20 s at 57&#176;C, and 45 s at 72&#176;C; and a final 2-min extension at 72&#176;C. For sequencing, 50 &#181;l PCR reaction products were purified with a QIAquick PCR purification kit (Qiagen, Hilden, Germany) and eluted with 30 &#181;l elution buffer. Three &#181;l of purified products were added to a 10-&#181;l sequencing reaction containing 3.2 pmol primer, 0.25 &#181;l Big Dye Terminator Ready Reaction mix (PE Biosystems), sequencing buffer and 10% DMSO. Cycling conditions were 96&#176;C for 2 min and 33 cycles of 96&#176;C for 10 s, 55&#176;C for 10 s, and 60&#176;C for 4 min. We also amplified and sequenced the overlapping genomic region of <italic>Zeb1</italic> and <italic>Zeb1os</italic> from normal mouse control strains 129/J, AKR/J, BALB/cJ, C3H/HeJ, C57BL/6J, C58/J, CBA/J, CE/J, DBA/2J, P/J, RF/J, SEA/GnJ and SWR/J DNA.</p><p>Double-stranded oligodeoxyribonucleotide probes were synthesized to encode genomic sequences containing c.58+181G (<named-content content-type="gene">5&#8242;-TGCTGGACTGGACCGTTATGTCTTACCTGC</named-content> and <named-content content-type="gene">5&#8242;-GCAGGTAAGACATAACGGTCCAGTCCAGCA</named-content>), c.58+181A (<named-content content-type="gene">5&#8242;-TGCTGGACTGGACCATTATGTCTTACCTGC</named-content> and <named-content content-type="gene">5&#8242;-GCAGGTAAGACATAATGGTCCAGTCCAGCA</named-content>), or a C-Myb binding site control from the <italic>mim-1</italic> gene <xref ref-type="bibr" rid="pgen.1002307-Chayka1">[19]</xref> (<named-content content-type="gene">5&#8242;-GCTCTAAAAAACCGTTATAATGTACAGATATCTT</named-content> and <named-content content-type="gene">5&#8242;-AAGATATCTGTACATTATAACGGTTTTTTAGAG</named-content>). Probes were end-labeled with [&#947;-<sup>32</sup>P]ATP by T4 Polynucleotide Kinase (New England Biolabs). Mouse C-Myb cDNA was cloned in pET-41a(+) (Novagen), and the protein was expressed in E.<italic>coli</italic> strain BL21(DE3)pLys (Invitrogen) and purified with Ni-NTA columns (Qiagen). Twenty-&#956;l reactions were performed with the EMSA Accessory Kit (Novagen). Unlabeled oligonucleotide competitors were added at 25- or 50-fold molar excess. Binding reaction products were separated by 6% DNA retardation gel electrophoresis (Invitrogen) and visualized with a Typhoon Trio+ (GE Healthcare).</p><p>Inner ears with adjacent mesenchyme were microdissected from E13.5 offspring of <italic>Tw/+</italic> x <italic>Tw/+</italic> matings. Total RNA was isolated from inner ears using PicoPure (Applied Biosystems, Foster City, CA). Total RNA from 10 to 14 ears of the same genotype was pooled and purified with the RNAeasy MinElute Cleanup kit (Qiagen). RNA integrity was measured with an Agilent 2100 Bioanalyzer (Applied Biosystems). One &#181;g of total RNA was reverse-transcribed with oligo(dT) primers and SuperScriptIII (Invitrogen, Carlsbad, CA, USA). For TaqMan real-time PCR, PCR primers were designed to amplify <italic>Zeb1</italic> exons 1a to 2, 1b to 2, 1c to 2, 1d to 2, 1e to 2, 1f to 2, and 2 to 3, <italic>Zeb1os</italic> exons 1 to 2, and <italic>Zfp438</italic> exons 3 to 4 with ZEN double-quenched probes containing a 5&#8242; FAM fluorophore, 3&#8242; IBFQ quencher, and an internal ZEN quencher (IDT, Coralville, IA). Sequences for the primers and probes are listed in <xref ref-type="supplementary-material" rid="pgen.1002307.s004">Table S1</xref>.</p><p>Comparative TaqMan assays were performed in triplicate on an ABI 7500 real-time PCR system (Applied Biosystems). PCR reactions were performed in a 50-&#181;l volume containing 5 &#181;l cDNA, 5 &#181;l primer mix (IDT), and 25 &#181;l of Universal PCR Master Mix (Applied Biosystems). Cycling conditions were 50&#176;C for 2 min, 95&#176;C for 10 min, followed by 40 cycles of 15 s at 95&#176;C and 1 min at 60&#176;C. Relative expression was normalized as the percentage of &#946;-actin expression, and calculated using the comparative threshold cycle method of 2<sup>&#8722;&#916;&#916;CT</sup>. Data are presented as mean values &#177; S.D. from six technical replicates. ANOVA was used to identify statistically significant differences between genotype groups (P&lt;0.05).</p><p>Proteins were extracted from E13.5 mouse inner ears or whole heads with NE-PER Nuclear and Cytoplasmic Extraction Reagents (Pierce Biotechnology) in the presence of Halt Protease Inhibitor Cocktail (Thermo Fisher Scientific Inc.). Proteins were separated by SDS-PAGE in 4&#8211;20% NuPage Bis-Tris gels followed by transfer to PVDF membranes (Millipore Corp., Billerica, MA). Proteins were detected with primary antibodies for Zeb1 (ab64098, Abcam, 1&#8758;200) and &#946;-actin (A2228, Sigma-Aldrich, 1&#8758;1000). Secondary antibodies were conjugated with Cy 3 or Cy 5 (GE Healthcare) and detected with a Typhoon Trio+ (GE Healthcare). Band density was measured using ImageQuant TL software. &#946;-actin levels were used for normalization. ANOVA analysis of two to six biological replicates from each genotype group was used to identify statistically significant differences (P&lt;0.05).</p><p>Mouse inner ear sections were harvested, processed and immunostained with anti-Zeb1 or anti-CD326 antibodies as described in Hertzano et al. <xref ref-type="bibr" rid="pgen.1002307-Hertzano1">[13]</xref>. CD326 is also known as epithelial cell adhesion/activating molecule (EpCAM) that serves as a specific antigenic marker for epithelial cells <xref ref-type="bibr" rid="pgen.1002307-Hertzano1">[13]</xref>.</p>