<p>All <italic>H. pylori</italic> strains used were in the 26695 genetic background <xref ref-type="bibr" rid="pgen.1002152-Tomb1">[15]</xref>. To generate gene-specific mutants, the corresponding open-reading frame (ORF) cloned into pILL570 was disrupted, leaving 5&#8242; and 3&#8242; ends (300 bp) of the gene, by a nonpolar kanamycin- (Km), apramycin- (Apr) or chloramphenicol (Cm) resistance cassette <xref ref-type="bibr" rid="pgen.1002152-Heuermann1">[59]</xref>, <xref ref-type="bibr" rid="pgen.1002152-Skouloubris1">[60]</xref>. To generate the Pol I over-expressing strain, the <italic>HP1470</italic> ORF was inserted into pADC vector, downstream of the <italic>ureA</italic> promoter, as described <xref ref-type="bibr" rid="pgen.1002152-Kang3">[61]</xref>. Plasmids were introduced into <italic>H. pylori</italic> strains by natural transformation and recombinants were selected after 3 to 5 days of growth on either 20 &#181;g/ml Km, 12.5 &#181;g/ml Apr or 8 &#181;g/ml Cm. Allelic replacement was verified by PCR. As described in the <xref ref-type="sec" rid="s2">Results</xref> section, the <italic>polA</italic> mutants used correspond, unless specified, to the replacement of the equivalent of the Klenow fragment by the resistance cassette, leaving the 5&#8242; to 3&#8242; exonuclease domain intact. Double mutants were obtained by plasmid or genomic DNA transformation of single mutant or by mixing two mutant strains together before plating the mix on double selection. <italic>H. pylori</italic> cultures were grown at 37&#176;C under a microaerobic atmosphere on BAB, blood agar base medium supplemented with an antibiotic mix and 10% defibrillated horse blood.</p><p>For all experiments, <italic>H. pylori</italic> strains were initially grown for 24 hr on plates with BAB medium. UV, MMS and gamma irradiation sensitivity assays were performed as described <xref ref-type="bibr" rid="pgen.1002152-ORourke2">[33]</xref>, <xref ref-type="bibr" rid="pgen.1002152-Marsin1">[62]</xref>. For chemical oxidative stress treatment, <italic>H pylori</italic> (OD<sub>600</sub>&#8202;=&#8202;1) cell suspensions were incubated with different concentrations of hydrogen peroxide (100, 200 and 300 mM). Cells were washed 10 min later, diluted with peptone broth and plated on BAB plates. Survival was determined as the number of cells forming colonies on plates after a given treatment divided by the number of colonies from non-treated cells. Assays to determine spontaneous mutation rates were performed as described <xref ref-type="bibr" rid="pgen.1002152-Pinto1">[7]</xref>.</p><p>All assays were performed at 37&#176;C for 30 min in 20 &#181;l reactions containing 10 mM Tris-HCl (pH 7.9), 50 mM NaCl, 10 mM MgCl<sub>2</sub>, 1 mM Dithiothreitol, 2.5 nM DNA substrate (see <xref ref-type="supplementary-material" rid="pgen.1002152.s005">Text S1</xref> and <xref ref-type="supplementary-material" rid="pgen.1002152.s006">Table S1</xref>) and variable concentrations of Pol I (specified in the figure legends). 0.1 mM of either all four dNTPs or each dNTP individually was included in the reactions except for the exonuclease activity assays. When required as a control, Klenow fragment DNA polymerase (Roche) was used. Reactions were stopped by adding loading buffer (10 mM EDTA, 95% (v/v) formamide, 0.03% (w/v) bromophenol blue, 0.03% (w/v) xylene cyanol) and subjected to electrophoresis in 8 M urea-containing 20% polyacrylamide gels. Gels were visualised and quantified using a Molecular Dynamics PhosphorImager. According to Koskoska et al. <xref ref-type="bibr" rid="pgen.1002152-Kokoska1">[63]</xref>, bypass probabilities were calculated as the proportion of DNA synthesis products extended beyond the lesion. Bypass efficiencies were then calculated for each enzyme and each substrate as the ratio of the bypass probability of a specific damaged base with respect to that of an undamaged nucleotide in the same position.</p><p>The assay was performed as described previously <xref ref-type="bibr" rid="pgen.1002152-Bebenek1">[22]</xref>. Gap-filling DNA synthesis was performed in a reaction mixture (25 &#181;l) containing 50 mM Tris (pH 6.8), 50 mM NaCl, 10 mM MgCl<sub>2</sub>, 1 mM DTT, 0.2 mM each of dNTP and 0.2 nM of gapped M13mp2 DNA substrate. Reactions were initiated by adding Pol I, incubated at 37&#176;C for 30 min, and terminated by adding EDTA to 20 mM. When DNA products were analyzed by agarose gel electrophoresis <xref ref-type="bibr" rid="pgen.1002152-Bebenek1">[22]</xref>, the majority of the gapped molecules were filled to completion. However, a minority of DNA products migrated as if synthesis had paused at the palindrome just upstream of the open reading frame of the <italic>LacZ</italic> gene. In this minority population, only about 75% of the template used to score errors had been copied. As a consequence, the <italic>lacZ</italic> mutant frequency observed for the ensemble reaction products may slightly underestimate the error rate of <italic>H. pylori</italic> Pol I.</p>