<p>All eleven collection sites in the BiGS Consortium received IRB approval for subject ascertainment, assessment, and collection of DNA for genetic studies. All participating subjects signed a statement of informed consent.</p><p>The subjects used in the GAIN <xref ref-type="bibr" rid="pgen.1002134-Smith1">[12]</xref> and WTCCC <xref ref-type="bibr" rid="pgen.1002134-WTCCC1">[8]</xref> samples have been previously described. The TGEN cases consisted of unrelated individuals from the &#8220;Wave 5&#8221; collection of the Bipolar Consortium, which included 1,310 unrelated DNA samples from families ascertained through probands with DSM IV-defined BPI disorder <xref ref-type="bibr" rid="pgen.1002134-Smith1">[12]</xref>, 1,190 of which ultimately passed QC measures. While GAIN samples were primarily from larger families with multiple BD cases or sib-pairs, TGEN samples were primarily population-based and were not required to have a family history. Controls were collected by NorthShore University HealthSystem, Evanston, IL, R01 MH59571, Pablo V. Gejman, M.D. (Collaboration Coordinator; PI) as part of a collaborative R01 application comprised of ten sites (see Acknowledgements).</p><p>Genomic DNA samples were analyzed on the Genome-Wide Human SNP 6.0 Array (Affymetrix, Inc. Santa Clara, CA) according to the manufacturer's protocols (Affymetrix Genome-Wide Human SNP Nsp/Sty 6.0 User Guide;Rev. 1 2007). Before the initiation of the assay, 50 ng of genomic DNA from each sample was examined qualitatively on a 1% Tris-acetate-EDTA agarose gel for visual signs of degradation. Any degraded DNA samples were excluded from further analysis (&#8764;3%). Samples were quantitated by Spectrometry and diluted to 50 ng/&#181;l in reduced EDTA TE buffer (10 mM Tris HCL, 0.1 mM EDTA, pH 8.0). 250 ng of DNA was then aliquotted into two 96-well reaction plates and digested with either Sty or Nsp restriction enzymes (New England Biolabs, Inc. Ipswich, MA) for 2 hours at 37&#176;C followed by 65&#176;C for 20 min. Sty and Nsp digested samples were then ligated to either the Sty 1 or the Nsp 1 adaptor (Affymetrix), respectively, with T4 DNA Ligase (New England Biolabs) for 3 hours at 16&#176;C then 20 min at 70&#176;C. The ligated samples were then diluted in molecular-grade water and subaliquotted into 3 (Sty) or 4 (Nsp) 96 well PCR plates. PCR was performed using PCR Primer 002 (Affymetrix) and Titanium Taq DNA Polymerase (Clontech, Mountain View, CA) with the following thermal cycling parameters: 1. 94&#176;C for 3 min., 2. 30 cycles of 94&#176;C for 30 sec., 60&#176;C for 30 sec., and 68&#176;C for 15 sec., and 3. 68&#176;C for 7 min. Like samples for all Sty and Nsp reactions were pooled into a single deep well plate, the DNA was bound to Agencourt AMPure beads (Beckman Coulter, Inc. Berea, CA), placed into MultiScreen filter plates (Millipore, Billerica, MA), washed with 75% ethanol and eluted with Buffer EB (QIAGEN, Valencia, CA). Purified samples were then fragmented using Fragmentation Reagent (Affymetrix) and incubated at 37&#176;C for 35 min. then at 95&#176;C for 15 min. Fragmented samples were labeled with DNA Labeling Reagent (Affymetrix) and TdT Enzyme (New England Biolabs) at 37&#176;C for 4 hours followed by 95&#176;C for 15 min. The samples were denatured at 95&#176;C for 10 min. and held at 49&#176;C until they were loaded on to the arrays. The arrays were placed into the hybridization oven at 50&#176;C and 60 rpm for 16 to 18 hours. Arrays were then washed, stained and immediately imaged on the GeneChip Scanner 3000 (Affymetrix). Birdsuite was used to call SNP genotypes from CEL files. Initial quality control measures consisted of gender-checks and a custom SNP fingerprinting approach to identify potentially duplicated or related individuals.</p><p>1,858 samples [1,310 Bipolar cases, 408 controls,140 technical controls (42 case replicates, 43 control replicates, 19 HapMap individuals, and 36 parents from 19 case families)] passed a call rate threshold of 97%, QC contrast of 0.40, and gender consistency. We further removed samples that did not have a diagnosis of Bipolar Disorder I or Schizoaffective Bipolar Disorder (78 cases filtered), that were outliers on the first 2 coordinates of an MDS plot including HapMap 3 individuals (N&#8202;=&#8202;6 controls and 35 cases), that showed poor concordance between duplicates (3 individuals out of 85 pairs), that appeared to be more similar to another individual in the GAIN study than expected (PI_HAT&gt;0.15) (6 cases that were related to someone in GAIN or appeared to be the same person that had entered the study twice), or that had high heterozygosity (&gt;0.285 averaged across all markers, N&#8202;=&#8202;1 case). A total of 1,190 cases and 401 controls remained and are included in the analysis.</p><p>SNPs were filtered for a lack of positional information from Affymetrix (N&#8202;=&#8202;1,233), low minor allele frequency (&lt;1%, N&#8202;=&#8202;145,345), significant deviation from Hardy-Weinberg equilibrium in controls (P&lt;10<sup>&#8722;6</sup>, 592), low call rate (&lt;95%, N&#8202;=&#8202;34,930), poor duplicate concordance (&gt;2 heterozygote or homozygote errors, 16,541), or &gt;1 Mendelian error within families (N&#8202;=&#8202;1,348). A total of 178,413 SNPs were removed, leaving a final count of 728,187 SNPs. Of these, 703,019 also passed QC in GAIN and were included in the merged analysis. Genotypes are reported in genome forward orientation based on NCBI build 36 <italic>via</italic> the Affymetrix annotation file GenomeWideSNP_6.na27.annot.csv.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genotype data was further filtered (MAF&gt;&#8202;=&#8202;5% and Hardy-Weinberg P&lt;10<sup>&#8722;6</sup> using all samples) and imputed to the CEU HapMap 2 (CEU_r22_b36_fwd) genotypes using MACH <xref ref-type="bibr" rid="pgen.1002134-Li1">[23]</xref> (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/MACH/index.html">http://www.sph.umich.edu/csg/abecasis/MACH/index.html</ext-link>). Imputation results were filtered such that r<sup>2</sup>&gt;&#8202;=&#8202;0.30.</p><p>Association analysis was performed on the genotype data in PLINK <xref ref-type="bibr" rid="pgen.1002134-Purcell2">[24]</xref> using logistic regression, adjusting for study in the GAIN+TGEN sample, and using the &#8211;dosage command with the predictor being the maximum likelihood estimate of the number of alleles at the locus (format&#8202;=&#8202;1). Adjusting for up to 10 MDS components did not alter the genomic inflation factor, so they were not included as covariates.</p><p>WTCCC genotype data was downloaded in TPED format and filtered as described in the accompanying documentation. Genotype calls were filtered based on CHIAMO quality scores (&gt;0.90); SNPs were filtered according to SNP lists provided by the WTCCC, including SNPs that were excluded based on poor genotype clustering; and individuals were filtered according to individual lists provided by the WTCCC for a total of 459,075 SNPs, 1,868 bipolar disorder (BD) cases, 1,926 coronary artery disease (CAD) cases, 1,748 Crohn's disease (CD) cases, 1,952 hypertension (HT) cases, 1,860 rheumatoid arthritis (RA) cases, 1,963 type 1 diabetes (T1D) cases, 1,924 type 2 diabetes (T2D) cases, 1,458 United Kingdom Blood Services (NBS) controls, and 1,480 1958 British Birth Cohort (58C) controls.</p><p>Association for WTCCC samples was performed in PLINK using logistic regression without any covariates. For each phenotype, cases were analyzed against both (NBS and 58C) control sets. Meta-analysis was performed in PLINK using the &#8211;meta-analysis command. Fixed effects P-values are reported.</p><p>SNPs that were genotyped in both WTCCC-BD and GAIN+TGEN were used to generate scores for each individual in GAIN+TGEN. Odds ratios from WTCCC-BD were natural log transformed and used as a score in the SNP scoring routine of PLINK. Subsets of SNPs achieving different P-value cutoffs were used such that weakly associated SNPs were progressively added to strongly associated SNPs. SNPs were pruned to linkage equilibrium (r<sup>2</sup>&lt;0.5) using the &#8220;&#8212;indep-pairwise&#8221; command in PLINK with a sliding window of 50 SNPs and a 5 SNP step. SNPs were clumped to independent associations using the &#8220;&#8212;clump&#8221; command in PLINK. Index SNPs were selected at P&lt;0.1, with a secondary threshold of P&lt;0.1, r<sup>2</sup>&lt;0.5, and a 250 kb window. Logistic regression was used to test for association between score and case-control status in R <xref ref-type="bibr" rid="pgen.1002134-R1">[25]</xref> (glm command). The lrm command was used to calculate a pseudo-R<sup>2</sup> statistic.</p><p>Summary-level data was obtained from a meta-analysis of height using a genotyping chip targeting genes related to cardiovascular disease, covering 49,320 SNPs and about 2,000 genes. We used the natural division of the Phase I cohort consisting of 53,394 individuals of European Ancestry as our discovery set and the collection of Phase II cohorts consisting of 37,052 individuals of European Ancestry as our test dataset. We performed meta-analysis on the Phase I and Phase II samples using METAL as described <xref ref-type="bibr" rid="pgen.1002134-Lanktree1">[19]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Power was calculated using the pwr.f2.test function from the pwr package in R with 1 degree of freedom in the numerator; N&#8722;2 degrees of freedom in the denominator, where N is the number of individuals with genotype data for the SNP in the test dataset; alpha of 0.05; and effect size f<sup>2</sup>. The effect size f<sup>2</sup> was calculated as:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002134.e001.jpg" /></disp-formula>R<sup>2</sup> was calculated as:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002134.e002.jpg" /></disp-formula>where beta is the effect size from the meta-analysis and s is standard deviation. In the study, height was expressed in cm and was not standardized to a z-score. The standard deviation of height was calculated as the sample size-weighted standard deviation across all discovery data sets (9.2 cm), and the standard deviation of the SNP was calculated from the allele frequencies assuming Hardy-Weinberg equilibrium. Because a proportion of SNPs were only tested in a small number of samples, we filtered the 20% of SNPs with the lowest sample sizes. To test the role of sample size on initial effect estimates, studies were progressively added to the discovery data and the meta-analysis was repeated on each subset of the data.</p><p>Power was calculated based on association results from WTCCC Bipolar cases and controls (NBS+58C) <xref ref-type="bibr" rid="pgen.1002134-Genomewide1">[26]</xref>. SNPs were additionally filtered for MAF (&gt;1%). In order to calculate power, the non-centrality parameter was calculated given the odds ratio (OR) from WTCCC, minor allele frequency in WTCCC controls, and the number of case and control samples in the TGEN+GAIN combined sample <xref ref-type="bibr" rid="pgen.1002134-Klein1">[27]</xref>. OR and MAF were rounded to 2 decimals. Power was calculated using G*Power 3 <xref ref-type="bibr" rid="pgen.1002134-Faul1">[28]</xref> given the non-centrality parameter and an alpha of 0.05.</p><p>Associations between power and replication at P&lt;0.05 or consistency of effect size were performed using logistic regression in R using the glm command (family&#8202;=&#8202;binomial(&#8220;logit&#8221;)). For plots, smoothing was performed using the smooth.spline function in R.</p><p>For both height and BD samples, we performed simulations in R to show that given the observed effect size, we would expect to see replication rates associated with power linearly with a slope of 1. We sampled 361 SNPs from each study across power levels, with 19 SNPs from each 5% power bracket. We simulated a population of 1,000,000 individuals with genotypes based on the allele frequency given Hardy-Weinberg expectations. For the BD case-control study, an individual's risk of disease was 1% multiplied by the odds ratio raised to the power of the number of risk alleles they carried. If this risk was greater than a random number between 0 and 1, then they were considered affected. For each SNP, we performed 100 permutations, sampling cases and controls in numbers to match the GAIN-TGEN sample and performed logistic regression. The observed replication rate is the proportion of tests that reached P&lt;0.05. For the quantitative height example, a individual's baseline height in standard deviations was modeled using a random number as a quantile of the normal distribution using the command qnorm in R. The observed effect of the SNP was then multiplied by the number of risk alleles and added to the baseline height. For 100 permutations per SNP, a random sample of individuals corresponding to the number of individuals tested for that SNP in the Phase II study was taken and association was tested using linear regression.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Exon location in the RefSeq genes was determined from refGene table (<ext-link ext-link-type="uri" ns0:href="http://genome.ucsc.edu/cgi-bin/hgTables">http://genome.ucsc.edu/cgi-bin/hgTables</ext-link>). For each SNP, the closest exon within 1 Mb was determined and the distance to that exon calculated. If a SNP was within an exon, a distance of 0 was used. An indicator variable of whether the SNP was within 2 kb or 25 kb of an exon was used in logistic regression.</p>