<p>Strains are listed in <xref ref-type="supplementary-material" rid="pgen.1002083.s003">Table S1</xref> and are SK1 derivatives <xref ref-type="bibr" rid="pgen.1002083-Kane1">[96]</xref>. The <italic>URA3</italic>-<italic>ARG4</italic> recombination interval has been described <xref ref-type="bibr" rid="pgen.1002083-Jessop1">[7]</xref>; <italic>cyh2-z</italic> is a spontaneous cycloheximide resistance mutation (Cyh<sup>R</sup>); <italic>spo11-Y135F</italic>
<xref ref-type="bibr" rid="pgen.1002083-Cha1">[97]</xref> was a gift from S. Keeney; <italic>mus81</italic>&#916; and <italic>sgs1</italic>-&#916;<italic>C795</italic> have been described <xref ref-type="bibr" rid="pgen.1002083-Jessop3">[42]</xref>, <xref ref-type="bibr" rid="pgen.1002083-Jessop2">[28]</xref>. Strains with estrogen-inducible <italic>CDC5</italic> and <italic>NDT80</italic> alleles (<italic>pGPD1-GAL4-ER pGAL1-CDC5</italic> and <italic>pGPD1-GAL4-ER pGAL1-NDT80</italic>, respectively) have been described <xref ref-type="bibr" rid="pgen.1002083-Sourirajan1">[10]</xref>. Strains were constructed by genetic crosses, or by transformation. Media formulae were as described <xref ref-type="bibr" rid="pgen.1002083-Guthrie1">[98]</xref>, <xref ref-type="bibr" rid="pgen.1002083-Goyon1">[99]</xref>.</p><p>Sporulation was as described <xref ref-type="bibr" rid="pgen.1002083-Goyon1">[99]</xref> using 400 ml cultures in a 2.8 liter baffled Fernbach flask (BellCo Glass) with a cell density of 2x 10<sup>7</sup> cells per ml at the beginning of sporulation. For RTG experiments, cells were induced to undergo meiosis for 7 hr, harvested by centrifugation, resuspended in an equal volume of liquid YPD (prewarmed to 30&#176;C) and aerated with vigorous shaking at 30&#176;C in conditions similar to those used for sporulation. For plating experiments, samples were sonicated twice for 5 seconds at baseline power (Microson XL 2005), diluted appropriately and then plated on YPD plates. To determine colony-forming units, samples were counted in a hematocytometer and the concentration of cells was determined; cells with unseparated buds were counted as a single entity. For Ndt80 or Cdc5 induction, &#946;-estradiol (ED; Sigma; 5 mM stock in ethanol) was added to a final concentration of 1 &#181;M. For no Cdc5-indcuation control experiments, the same amount of ethanol (without ED) was added. For RTG after Cdc5 induction during meiosis, cells were washed twice with sporulation medium lacking ED at 30&#176;C before resuspension in YPD.</p><p>Unless stated otherwise, all data presented are the average of two independent experiments; error bars in plots indicate standard error.</p><p>To score bud emergence and nuclear division, 1 ml of a culture was mixed with 1 ml of ethanol and stored at 4&#176;C. Just before examination, 1 &#181;l of 1 mg/ml 4&#8242;,6-diamidino-2-phenylindole (DAPI) was added and samples were left for 5 min at room temperature, washed once with an equal volume of water and resuspended in 0.5 ml water. Cell morphology was scored using phase contrast or differential interference contrast microscopy and nuclear morphology by DAPI epifluorescence microscopy, using a Zeiss Axioplan 2 epifluorescence microscope and a QICAM camera. Images were acquired using QCapture 3.1.1 and processed with Adobe Photoshop CS3.</p><p>GFP chromosome dot visualization was done using cells fixed in 3.7% formaldehyde as described <xref ref-type="bibr" rid="pgen.1002083-Lee1">[65]</xref>. Vectashield with DAPI (Vector Laboratories) was used to simultaneously stain DNA. Cells were counted as having two GFP dots if two separated GFP dots could be clearly visualized. Sample fluorescence was visualized using a Zeiss Axioplan 2 epifluorescence microscope and a Micromax 1300 CCD camera. Images were acquired using IPlab 3.7 and processed with Adobe Photoshop CS3.</p><p>Nuclear spreads were performed and stained as described <xref ref-type="bibr" rid="pgen.1002083-Bishop1">[100]</xref> using cells from 5 ml of culture. Zip1 was detected using anti-Zip1 rabbit polyclonal sera (a gift from G.S. Roeder, 1&#8758;100 dilution) as the primary antibody and Alexafluor 488 conjugated goat anti-rabbit IgG (Molecular Probes #A11034) at 1&#8758;100 as the secondary antibody. To visualize DNA, 40 &#181;l of Vectashield with DAPI (Vector Laboratories) was added. Sample fluorescence was visualized using a Zeiss Axioplan 2 epifluorescence microscope and a Micromax 1300 CCD camera. Images were acquired using IPlab 3.7 and processed with Adobe Photoshop CS3.</p><p>During RTG, cells lose synchrony and continue to further cell cycles, complicating calculation of a cumulative cell division curve. We assumed that bud emergence and nuclear division occur with the same relative timing in the first and second cell division after RTG. To distinguish between daughter and mother cells, we took advantage of the fact that after RTG, <italic>ndt80</italic>&#916; cells produce an elongated bud that can be easily distinguished from the round mother cell (<xref ref-type="fig" rid="pgen-1002083-g001">Figure 1a</xref>). The fraction of cells that had not yet budded (unbudded cells) was calculated according to the equation: unbudded cells&#8202;=&#8202; (X<sub>1</sub>-Y<sub>1</sub>)/Z<sub>1</sub> where X<sub>1</sub>&#8202;=&#8202; unbudded round cells (i.e. cells before the first mitotic division), Y<sub>1</sub>&#8202;=&#8202; unbudded elongated cells (i.e. products of the first mitotic division) and Z<sub>1</sub>&#8202;=&#8202; total cells counted. At late times, due to continuous division of the cells, the number of cells that have already undergone the first mitotic division (Y<sub>1</sub>) can exceed the number of cells that have not undergone a mitotic division (X<sub>1</sub>). In such a case, (X<sub>1</sub>-Y<sub>1</sub>) was set to zero.</p><p>The fraction of cells that had undergone the first nuclear division (post-division) was calculated according to the equation: post-division&#8202;=&#8202; X<sub>2</sub>/Y<sub>2</sub> where X<sub>2</sub>&#8202;=&#8202; round cells that were undergoing mitosis (detected as budded with a nucleus stretched between the mother and daughter cells) plus all elongated cells with a nucleus (i.e. cells that have already completed the first mitotic division) and Y<sub>2</sub>&#8202;=&#8202; all round cells. At late times, due to continuous cell division, X<sub>2</sub> may be greater than Y<sub>2</sub>. In such a case, the fraction of post-division cells was set to one.</p><p>DNA preparation and analysis on Southern blots were as described <xref ref-type="bibr" rid="pgen.1002083-Allers2">[101]</xref>, <xref ref-type="bibr" rid="pgen.1002083-Allers1">[8]</xref>. <italic>Xho</italic>I and <italic>Xmn</italic>I digests were probed with <italic>ARG4</italic> coding sequences (+165 to +1413). XhoI/<italic>Eco</italic>RI double digests were probed with <italic>HIS4</italic> coding sequences (+538 to +718).</p><p>Protein was prepared from 4 ml of sporulating culture by TCA precipitation <xref ref-type="bibr" rid="pgen.1002083-Foiani1">[102]</xref>. 5 &#181;l samples of each extract were displayed on 7.5% polyacrylamide Tris-Glycine pre-cast gels (Bio-Rad) and electroblotted to a PVDF membrane (Invitrogen), using an iBlot Dry Blotting System (Invitrogen) as recommended by the manufacturer. Blots were washed for at least one hr on an orbital shaker at room temperature in blocking buffer, 0.2% I-block (Tropix) in PBST (0.15 M NaCl, 0.053 M Na<sub>2</sub>HPO<sub>4</sub>, 0.008 M KH<sub>2</sub>PO<sub>4</sub>, 0.05% v/v Tween-20, pH 7.4). Primary antibody, diluted in blocking buffer, was added to the blot and incubated on an orbital shaker at room temperature for at least one hr. Blots were washed four times for 15 min with blocking buffer, incubated with secondary antibody for one hr with shaking at room temperature, and wash steps were repeated. Signal was developed using the chemiluminescent CDP-star substrate (Applied Biosystems), detected using a Fuji LAS3000 CCD camera, and quantified using ImageGauge V4.22 software (Fuji). Blots were stripped with OneMinute Western Blot Stripping Buffer (GM Biosciences) and reprobed for Arp7 as a loading control. Primary antisera were as follows: Arp7 &#8211; goat polyclonal (Santa Cruz Biotechnology, Inc; Sc-8961), 1&#8758;500; influenza hemagglutinin (HA) &#8211; mouse monoclonal (5 &#181;g/&#181;l; Roche Applied Science; 12CA5), 1&#8758;10,000; Cdc5 &#8211; goat polyclonal (Santa Cruz Biotechnology, Inc; Sc-6733), 1&#8758;500; Ndt80 &#8211; rabbit polyclonal (a gift from K. Benjamin), 1&#8758;10,000; Clb2 &#8211; rabbit polyclonal (Santa Cruz Biotechnology, Inc; Sc-9071), 1&#8758;500. Secondary antibodies were alkaline phosphatase conjugates of goat-anti-mouse (Sigma, A3562), goat-anti-rabbit (Sigma, A3687) and rabbit-anti-goat (Sigma, A4187), all used at 1&#8758;10,000.</p><p>To measure the frequency of recombination between the <italic>CYH2</italic> locus and the centromere of chromosome VII, we measured second division segregation pattern of the <italic>TRP1</italic> and <italic>CYH2</italic> alleles in dissected tetrads from strain MJL3548 (<italic>CYH2</italic>/<italic>cyh2-z TRP1</italic>/<italic>trp1</italic>), using <italic>TRP1</italic> as a centromere-linked marker <xref ref-type="bibr" rid="pgen.1002083-Mortimer1">[62]</xref>. Of 72 tetrads with 4 viable spores, 12 tetrads were parental ditypes, 12 were non-parental ditypes and 47 were tetratypes. One tetrad had gene conversion of <italic>cyh2-z</italic> and was not counted. Thus, as expected for a locus far removed from its centromere, the vast majority of cells undergo at least one crossover between <italic>CYH2</italic> and <italic>CEN7</italic>, and about two thirds of cells produce spores with a crossover between the <italic>CYH2</italic> locus and its centromere.</p>