<p>See also <xref ref-type="supplementary-material" rid="pgen.1002057.s009">Text S1</xref>.</p><p>
<xref ref-type="supplementary-material" rid="pgen.1002057.s008">Table S6</xref> is a list of yeast strains and plasmids used. Yeast was grown in rich media at 30&#176;C unless otherwise stated. Plasmid bearing strains were grown in synthetic complete (SC) media lacking the appropriate nutrient. For spot assays, an identical optical density (OD) of cells was serially diluted ten-fold and spotted on the indicated plate at the indicated temperature for 72 hours. Growth curves were performed as described <xref ref-type="bibr" rid="pgen.1002057-McLellan1">[35]</xref>. Briefly, logarithmic phase cultures were diluted to and OD of 0.05 in a 96-well plate in triplicate and grown for 24 hours in a TECAN M200 plate reader at 30&#176;C. <xref ref-type="fig" rid="pgen-1002057-g004">Figure 4C</xref> shows the middle curve of the three replicates for each strain. None of the TTT-<italic>ASA1</italic> ts-alleles were able to grow at the non-permissive temperature of 37&#176;C, regardless of whether <italic>TIP41</italic> was deleted (unpublished observation).</p><p>SGA was performed using a Singer RoToR essentially as described <xref ref-type="bibr" rid="pgen.1002057-Tong1">[28]</xref> and was also used to introduce the appropriate chromosomal markers for the CTF (i.e <italic>ade2-101</italic>::NatMX and CFIII or CFVII {<italic>URA3</italic>, <italic>SUP11</italic>}) and GCR (i.e. <italic>pif1</italic>&#916;::<italic>HygMX</italic> and <italic>hxt13</italic>::<italic>URA3)</italic> reporter strains <xref ref-type="bibr" rid="pgen.1002057-Smith1">[12]</xref>, <xref ref-type="bibr" rid="pgen.1002057-Yuen1">[13]</xref>. Clustering of genetic interaction profiles was done for genes and arrays by average linkage using Cluster 3.0 <xref ref-type="bibr" rid="pgen.1002057-deHoon1">[36]</xref> and viewed with Java TreeView.</p><p>Patches of the <italic>CAN1</italic>, <italic>URA3</italic>, GCR assay strains were replicated to media containing canavanine and 5&#8242;FOA at 30&#176;C to screen for GCR exactly as described <xref ref-type="bibr" rid="pgen.1002057-Smith1">[12]</xref>. CTF screening of ts-alleles was done by streaking CF containing strains onto SC media with 20% the normal amount of adenine exactly as described in <xref ref-type="bibr" rid="pgen.1002057-Yuen1">[13]</xref>. Due to the temperature sensitivity, CTF assay strains were tested in an iterative fashion with respect to temperature. Beginning at 30&#176;C, strains were deemed CTF, wildtype or inviable. Inviable strains were re-tested at 25&#176;C, CTF strains were put aside as putative hits, and wildtype strains were retested at 34&#176;C. The process was repeated with the 34&#176;C strains at 37&#176;C for wildtype strains and 32&#176;C for inviable strains. All the CTF positive strains from any temperature were retested in three independent experiments and a qualitative strength designation assigned as described <xref ref-type="bibr" rid="pgen.1002057-Yuen1">[13]</xref>. Independently generated fragments of chromosome III and VII were tested for each CTF assay strain <xref ref-type="bibr" rid="pgen.1002057-Yuen1">[13]</xref>, <xref ref-type="bibr" rid="pgen.1002057-Spencer1">[19]</xref>. Screening of DAmP alleles for CTF was conducted at 30&#176;C. The ALF screen was performed as described in <xref ref-type="bibr" rid="pgen.1002057-Yuen1">[13]</xref> except that the 1 cm<sup>2</sup> patches of each mutant strain were mated to the MAT&#945; test strain at 25&#176;C, 30&#176;C and 34&#176;C to explore a range of semi-permissive temperatures.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Enriched GO terms were calculated at (<ext-link ext-link-type="uri" ns0:href="http://go.princeton.edu/cgi-bin/GOTermFinder">http://go.princeton.edu/cgi-bin/GOTermFinder</ext-link>) using the hypergeometric distribution to define enriched terms. Terms (downloaded September 28<sup>th</sup> 2010) with a p&lt;0.05 after Bonferroni correction were considered enriched (<xref ref-type="supplementary-material" rid="pgen.1002057.s004">Table S2</xref>). Human CIN candidate genes were compared to tumor mutations found in the COSMIC database (version 49) <xref ref-type="bibr" rid="pgen.1002057-Forbes1">[23]</xref> and the Cancer Gene Census <xref ref-type="bibr" rid="pgen.1002057-Kaizuka1">[25]</xref>. The data was obtained from the Wellcome Trust Sanger Institute Cancer Genome Project web site, <ext-link ext-link-type="uri" ns0:href="http://www.sanger.ac.uk/genetics/CGP">http://www.sanger.ac.uk/genetics/CGP</ext-link>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Telomere length was determined essentially as described <xref ref-type="bibr" rid="pgen.1002057-Ungar1">[37]</xref> except that the probes and ladder were labeled with digoxigenin (DIG) and detected with anti-DIG antibodies according to manufacturers instructions (DIG High Prime DNA Labeling and Detection Starter Kit II; Roche). Mutants for telomere length analysis were chosen primarily based on <ext-link ext-link-type="uri" ns0:href="http://www.yeastgenome.org">www.yeastgenome.org</ext-link> descriptions including &#8220;Protein of Unknown Function&#8221;, and with &#8220;putative&#8221; or &#8220;potential&#8221; functions.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Logarithmic Gat1-GFP expressing cultures were grown in SC media and shifted to 37&#176;C for three hours. Live cells were mounted on concanavalin A coated slides and imaged with the GFP filter set (500 ms exposure) using a Zeiss axioscop and Metamorph software (Molecular Devices) essentially as described <xref ref-type="bibr" rid="pgen.1002057-Carroll1">[38]</xref>. Images were analyzed using Image J (<ext-link ext-link-type="uri" ns0:href="http://rsbweb.nih.gov/ij/index.html">http://rsbweb.nih.gov/ij/index.html</ext-link>). Experiments were repeated in triplicate and the proportion of cells with nuclear staining was counted for at least 100 cells from each experiment.</p><p>Logarithmic cultures at 25&#176;C were shifted to the indicated temperature for 5 hours and harvested by centrifugation (4000 &#215;g, 2 min). Cell pellets were washed with H<sub>2</sub>O and resuspended in lysis buffer at 4&#176;C for glass bead lysis (50 mM Tris-Cl pH 7.8, 150 mM NaCl, 1.5 mM Mg Acetate, 10% Glycerol, 0.5% Triton X-100, 1 mM DTT, 10 mM Na PPi, 5 mM EDTA, 0.1 mM NaVO<sub>4</sub>, 5 mM NaF, 1&#215; complete protease inhibitor (Roche). Cell lysate was centrifuged (10000 &#215;g, 5 min) and the supernatant retained. Equal amounts of protein (quantified by Bradford Assay reagent, Bio-Rad) were run on SDS-PAGE gels (10% for Pgk1p, 6% for Tel1p or Tor1p), transferred to nitrocellulose membranes and probed with the indicated antibodies.</p>