<p>All <italic>Drosophila experiments</italic> were performed on standard food in 25&#176;C incubators unless otherwise noted. Uas-(CGG)<sub>90</sub>eGFP lines 1 and 2 (designated lines BD and BC, respectively) were a kind gift from Peng Jin at the University of Emory and have been previously described <xref ref-type="bibr" rid="pgen.1001240-Jin1">[13]</xref>, <xref ref-type="bibr" rid="pgen.1001240-Jin2">[18]</xref>, <xref ref-type="bibr" rid="pgen.1001240-Sofola1">[19]</xref>. Stability of the CGG repeat was confirmed by PCR and sequencing using C and F primers as described previously <xref ref-type="bibr" rid="pgen.1001240-Jin1">[13]</xref>, <xref ref-type="bibr" rid="pgen.1001240-Chong1">[65]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1001240-Tassone5">[67]</xref>. Uas-dHDAC6L #13 and UAS-dHDAC6kd flies have been previously described and characterized <xref ref-type="bibr" rid="pgen.1001240-Pandey1">[30]</xref>. UAS-HDAC11kd #1B3 and UAS-dHDAC6KD were obtained as a kind gift from Dan Garza at Novartis Pharmaceuticals. Tub-Gal4, Gmr-GAL4, Uas-eGFP and UAS-LacZ fly lines were obtained from the Bloomington <italic>Drosophila</italic> Stock Center (Bloomington, IN).</p><p>The cDNA for <italic>Drosophila</italic> HDAC3 and HDAC6 were amplified from the expressed sequence tag (EST) clones LD23745 and LD43531, respectively. The coding region for <italic>Drosophila</italic> HDAC11 was generated by positional cloning from EST clone CG31119-RA. Primers were created to insert 5&#8242; Kpn I and 3&#8242; Xba I restriction sites and the constructs were ligated into the pAc5.1/V5 vector (Invitrogen, Carlsbad, CA) as previously described <xref ref-type="bibr" rid="pgen.1001240-Cho1">[28]</xref>. Primers were then generated to introduce Spe I at the 5&#8242; end just upstream of the Kpn I site and Avr II at the 3&#8242; end of the dHDAC constructs maintaining the V5 tag, 6xHIS, stop codon and the SV40 pA sites from the pAc5.1/V5 vector and was subcloned into a pINDY6 vector <xref ref-type="bibr" rid="pgen.1001240-KazemiEsfarjani1">[68]</xref> cis to the yeast upstream activating sequence (UAS). The pINDY6 vector containing the dHDACs was used to generate transgenic fly lines using standard germ-line transformation by Genetic Services (Cambridge, MA). The dHDAC expression was activated in genetic crosses trans to the yeast GAL4 transcription factor, which was in turn regulated by the eye-specific promoter GMR upstream of the yeast GAL4 cDNA.</p><p>Eye phenotypes of 1&#8211;2 day-old anesthetized flies were evaluated with a Leica MZ APO stereomicroscope and photographed with a Leica DFC320 digital camera as previously described <xref ref-type="bibr" rid="pgen.1001240-Pandey1">[30]</xref>. For each genotype and condition, at least 100 flies were evaluated. For fluorescent images of <italic>Drosophila</italic> heads to evaluate eGFP expression, animals were anesthetized with CO2, decapitated, and immediately imaged on a glass coverslip by epifluorescence on an inverted Olympus IX71 microscope. All images were taken at the same exposure. Images were processed using Slidebook 4.0 software and subtracted for background auto-fluorescence based on images of non-transgenic flies. Images are 10x in coronal orientation. For quantitation of the severity of the eye phenotypes, a degeneration scale was used as previously described <xref ref-type="bibr" rid="pgen.1001240-Pandey1">[30]</xref>. Briefly, flies were given a score between 0 and 10 on the following scale: 1 point for extranumerary bristles, 1 point for abnormal bristle orientation, 1 point for oomatidial fusion, 1 point for oomatidial pitting, 1 point for retinal collapse. In addition, 1 point was added if more than 5% of the eye was affected, 4 points was added if more than 50% of the eye was affected. For quantitation, at least 20 flies per group were utilized.</p><p>SEM samples were processed as previously described <xref ref-type="bibr" rid="pgen.1001240-Pandey1">[30]</xref>. Briefly, flies were collected and fixed in 2.5% glutaraldehyde (EMS) in PBS and post-fixed for 15&#8211;30 minutes in 1.5% osmium tetroxide (Stevens Metallurgical) in PBS. Samples were then dehydrated in ethanol, immersed in hexamethyldisilazane (Polysciences Inc.) and dried in a desiccator for three days. Specimens were then coated with gold:palladium using a Denton DV-503 vacuum evaporator, and analyzed using an AMRAY 1820D scanning electron microscope.</p><p>Semi-quantitative RT-PCR analysis in cells and <italic>Drosophila</italic> was performed as previously described <xref ref-type="bibr" rid="pgen.1001240-Pandey1">[30]</xref>, <xref ref-type="bibr" rid="pgen.1001240-RodriguezLebron1">[69]</xref>. Briefly, flies of the specified genotypes were decapitated under anesthesia and the heads were flash frozen on dry ice. After crushing the heads with a pipette tip, total RNA was extracted using Trizol reagent (Invitrogen, San Diego, CA) according to manufacturer's protocols. Lymphoblastoid Cell lines were homogenized directly in Trizol prior to extraction. Following spectrophotometric quantification, 1 &#181;g of total RNA was reverse-transcribed (iScript cDNA Synthesis Kit; BioRad, Hercules, CA) and the resulting cDNA subjected to quantitative real-time PCR analysis with gene specific primers <xref ref-type="bibr" rid="pgen.1001240-Biacsi1">[38]</xref> as follows: eGFP: (Fwd: <named-content content-type="gene">CTGCTGCCCGACAACCA</named-content>; Rev: <named-content content-type="gene">GAACTCCAGCAGGACCATGTG</named-content>), FMR1: (Fwd: <named-content content-type="gene">CCGAACAGATAATCGTCCACG</named-content>; Rev: <named-content content-type="gene">ACGCTGTCTGGCTTTTCCTTC</named-content>), 18S Fly: (Fwd: <named-content content-type="gene">CGGCTACCACATCCAAGGAA</named-content>; Rev: <named-content content-type="gene">GCTGGAATTACCGCGGCT</named-content>), 18SHum: (Fwd:<named-content content-type="gene">CAGCCACCCGAGATTGAGCA</named-content>; Rev:<named-content content-type="gene">TAGTAGCGACGGGCGGTGTG</named-content>), &#946; ~Actin: (Fwd: <named-content content-type="gene">GGCATCCTCACCCTGAACTA</named-content>; Rev:<named-content content-type="gene">AGAGGCGTACAGGGATAGCA</named-content>), GAPDH: (Fwd: <named-content content-type="gene">AGAAGGCTGGGGCTCATTTG</named-content> Rev: <named-content content-type="gene">AGGGGCCATCCACAGTCTTC</named-content>).</p><p>PCR analysis was performed using the iQ SYBR Green Supermix in a myiQ Single Color RTPCR system (BioRad). All runs included a standard dilution curve representing at least 10x and 0.01X the RNA concentration utilized for all primer sets to insure linearity. Further, equivalent efficiency of individual primer sets was confirmed prior to data analysis. For <italic>Drosophila</italic>, the levels of (CGG)<sub>90</sub>-eGFP mRNA were normalized to those of 18S RNA or GAPDH mRNA for each sample run and expressed as a ratio of levels found UAS-eGFP lines (fold control expression). For lymphoblast derived mRNA quantitation, levels of FMR1 mRNA were normalized to actin RNA or 18S RNA and expressed as a ratio of mean expression of normal repeat length cell lines (fold control expression). All samples were run in triplicate and all data represent at least three independent experiments.</p><p>Three cell lines were obtained as a kind gift from Stephanie Sherman at Emory University. Their brief clinical history is below:</p><p>C0140.004 (CGG 91), Definite FXTAS; 62 y.o. man with onset of symptoms around age 50, Middle cerebellar Peduncle (MCP) sign on MRI, postural kinetic tremor early, developed gait ataxia, cerebellar tremor, voice tremor and head titubation; no nystagmus or parkinsonian features.</p><p>C0051.004, (CGG 90) Definite FXTAS; Onset at 59, postural tremor with mild gait imbalance. MCP sign on MRI;</p><p>C0038.026, asymptomatic male with 30 CGG repeats.</p><p>All other lymphoblastoid cell lines were obtained from the Coriell Cell Repository (Coriell, Camden, NJ).</p><p>GM09237, symptomatic FXS male with &gt;900 CGG repeats.</p><p>GM06891, 29YO male with 118 CGG repeats, clinical status for FXTAS unknown.</p><p>GM06892, 84YO male with 93 CGG repeats, clinical status for FXTAS unknown.</p><p>GM06895, 55YO male with 23 CGG repeats; related to 6891 and 6892, clinical status unknown.</p><p>GM20244, 43 YO male with 41 CGG repeats, clinical status unknown.</p><p>Repeat size was confirmed by PCR for all lines except for the FXS patient line, as previously described using C and F primers <xref ref-type="bibr" rid="pgen.1001240-Chong1">[65]</xref>, <xref ref-type="bibr" rid="pgen.1001240-Tassone5">[67]</xref>. Cells were grown in RPMI 1640 media with Glutamine pre-added (Invitrogen) and 12% fetal bovine serum plus antibiotics as previously described<xref ref-type="bibr" rid="pgen.1001240-Peprah1">[12]</xref>.</p><p>Nine fibroblast cell lines were obtained as a kind gift from Dr Paul Hagerman and colleagues (University of California at Davis). The clinical state of these patients has been described previously and all cell lines were obtained from males <xref ref-type="bibr" rid="pgen.1001240-GarciaArocena1">[42]</xref>. Nomenclature used follows that used in the publication where these lines were first published <xref ref-type="bibr" rid="pgen.1001240-GarciaArocena1">[42]</xref>. Briefly, there are three control lines: C1 (CGG31), C4 (CGG22) and C5 (CGG30). There are three cell lines from asymptomatic pre-mutation carriers (all at least 68 years old): P3 (CGG81), P4 (CGG70) and P5 (CGG67). Lastly, there are three cell lines from patients with clinically definite FXTAS based on established criteria<xref ref-type="bibr" rid="pgen.1001240-BerryKravis1">[70]</xref>: F1 (CGG122), F2 (CGG105) and F3 (CGG97). Fibroblasts were cultured and maintained as previously described<xref ref-type="bibr" rid="pgen.1001240-GarciaArocena1">[42]</xref> and repeat size was confirmed by PCR for all lines using C and F primers <xref ref-type="bibr" rid="pgen.1001240-Chong1">[65]</xref>, <xref ref-type="bibr" rid="pgen.1001240-Tassone5">[67]</xref>.</p><p>Formaldehyde Cross linking ChIP was performed according to previously published commercial protocols using a ChIP kit (Millipore, Temecula, CA) <xref ref-type="bibr" rid="pgen.1001240-Patel1">[71]</xref>. Antibodies to Histone H3 acetyl K9 (ab10812, Abcam, Cambridge, MA), Acetyl Histone H4 (abCS200571, Millipore), or rabbit IgG (Millipore) were used to immunoprecipitate the DNA/protein complexes overnight. After reversal of crosslinks and DNA isolation, real time PCR was conducted on equal concentrations of input and IP derived DNA in triplicate. Primers used against the <italic>FMR1</italic> promoter and exon1 were previously published <xref ref-type="bibr" rid="pgen.1001240-Biacsi1">[38]</xref>. Control primers to human GAPDH optimized for ChIP were obtained commercially (Millipore). FMR1 DNA quantity from IP was normalized to input FMR1 DNA. This number was then re-normalized to the mean of the control cell lines and expressed as fold change from this mean. For both selective antibodies, IP qPCR was similar to input and &gt;10x of ChIP with rabbit serum or non-specific antibodies (e.g. Beta-gal) for both FMR1 and GAPDH primers (<xref ref-type="supplementary-material" rid="pgen.1001240.s003">Figure S3</xref>). PCR primers targeted within 100 bp upstream or downstream of the CGG repeat showed similar results, but with altered efficiency due to high GC content, complicating quantification (data not shown).</p><p>Anacardic Acid was obtained from Calbiochem and Garcinol was obtained from Sigma. All drugs were dissolved in DMSO. Cells were treated for 24 or 72 hours (Garcinol 10, 20, or 50 &#181;M or Anacardic Acid at 10, 50, 200, or 500 &#181;M) with drug added directly to the culture media. Both 18S RNA and actin were used as internal controls for qRT-PCR experiments. FMR1 mRNA expression was normalized to 18S RNA in all drug experiments rather than actin because at the highest doses of both drugs, there were alterations in actin, GAPDH and c-myc mRNA levels (data not shown).</p><p>Determination of cell viability and cytotoxicity was performed using a MultiTox-Glo Multiplex Cytoxicity Assay kit from Promega (Madison, WI, USA) according to the manufacturers' protocol. Additional viability assays were done using trypan blue exclusion with similar results.</p><p>RU-486(Sigma) or Vehicle(Ethanol) was added to standard fly food at a final concentration of 200 &#181;M as previously described <xref ref-type="bibr" rid="pgen.1001240-McGuire1">[72]</xref>. Uas-(CGG)<sub>90</sub>eGFP line 1 and line 2 fly lines were crossed with flies expressing a Gene-Switch Tubulin-Gal4 driver (Tub5, a kind gift from Scott Pletcher at the University of Michigan). At 0-3 days post eclosion, flies of the specified genotype were switched to food containing drug or vehicle at 25C at 60% humidity. Viability was assessed daily for 15 days or until all flies were dead, whichever came first. This latter time point was chosen because of concerns over the stability of RU-486 and HAT inhibitors over time.</p><p>For drug studies, Garcinol (from a stock solution of 100 mM), Anacardic acid (stock solution of 250 mM) at various concentrations or an equal volume of vehicle (DMSO) were added to food containing RU-486 as described above. Different volumes of DMSO had the same viability as RU-486 alone, so these results were pooled for statistical analysis. Data in <xref ref-type="fig" rid="pgen-1001240-g007">Figure 7</xref> represents a summary of at least 3 independent experiments with all results pooled.</p><p>For viability assays done pre-eclosion, uas-eGFP/CyO (Chr 2) or uas-(CGG)<sub>90</sub>eGFP/CyO line 1 and line 2 fly lines (Chr 2) were crossed with flies expressing a tubulin-Gal4 driver (Chr 3). Viability was assessed by determining the ratio of progeny carrying the balancer chromosome (CyO) to those carrying the UAS-transgene. Data are representative of at least 4 experiments.</p><p>Western blotting to FMRP was performed as previously described <xref ref-type="bibr" rid="pgen.1001240-Todd1">[73]</xref>. Briefly, cell lysates were lysed in RIPA buffer, sonicated, and quantified by the method of Bradford. Equal protein amounts were then boiled in 2x Laemmeli buffer and run on a 12% SDS polyacrylamide gel. After transfer to PVDF membrane at 36 amps for 1.5 hours at 4C, the blots were incubated with antibodies to FMRP (Millipore 1&#8758;1000 O/N at 4C) and GAPDH (Santa Cruz Biotechnology, used at 1&#8758;1000 O/N at 4C). Images from westerns are representative of at least 2 independent experiments with similar results.</p><p>For all graphs, error bars represent Standard error of the mean calculated in Microsoft Excel. For comparisons of single groups, a Student's two sided T-Test was performed. For drug studies in lymphoblast cell lines, these were done as a paired T-Test. For all others, unpaired T-tests were performed.</p><p>Other statistical analyses were performed using GraphPad Prism. For correlation analysis, data are presented as the best fit curve with 95% confidence intervals. r<sup>2</sup> (coefficient of determination) is included as a measurement of variance explained by the selected variable. An F test was performed to determine statistical significance. For comparisons to CGG repeat length, a Spearman correlation was used to account for the non-parametric distribution of repeat lengths in our samples. All other comparisons utilized a Pearson correlation. For survival analysis, a Log-rank (Mantel-Cox) test or a Log-rank test for Trend was performed. Flies surviving past the specified end of the study (15 days) were assigned a survival time of 15 days for statistical purposes.</p>