<p>Plants were grown hydroponically in a growth chamber as described by Famoso et al. <xref ref-type="bibr" rid="pgen.1002221-Famoso1">[8]</xref>. Al tolerance was determined based on relative root growth (RRG) after three days in Al (160 &#181;M Al<sup>3+</sup>) or control solution. The hydroponic solution used in this study was chemically designed and optimized for rice Al tolerance screening; for a detailed comparison of the phenotypic procedures employed in this work compared to previously published rice Al tolerance work see Famoso et al. (2010). To obtain uniform seedlings, 80 seeds were germinated and the 30 most uniform seedlings were visually selected and transferred to a control hydroponic solution for a 24 hour adjustment period. After the 24 hour adjustment period, root length was measured with a ruler and the 20 most uniform seedlings were selected and distributed to fresh control solution (0 uM Al<sup>3+</sup>) or Al treatment solution (160 uM Al<sup>3+</sup>). Plants were grown in their respective treatments for &#8764;72 hours and the total root system growth was quantified using an imaging and root quantification system as described by Famoso et al. (2010). The mean total root growth was calculated for Al treated and control plants and RRG was calculated as mean growth (Al)/mean growth (control). The 373 genotypes screened for Al tolerance and used in the association analysis are part of a set of 400 <italic>O. sativa</italic> genotypes that have been genotyped with 44,000 SNPs as described by Zhao et al. <xref ref-type="bibr" rid="pgen.1002221-Zhao3">[65]</xref>.</p><p>The QTL populations consisted of a population of 134 recombinant inbred lines (RILs) derived from a cross between Azucena (tolerant <italic>tropical japonica</italic>) and IR64 (susceptible <italic>indica</italic>) <xref ref-type="bibr" rid="pgen.1002221-Tung1">[67]</xref>, <xref ref-type="bibr" rid="pgen.1002221-Lin1">[70]</xref> and a population of 78 backcross introgression lines (BILs) derived from a cross between Nipponbare (tolerant <italic>temperate japonica</italic>) and Kasalath (susceptible <italic>aus</italic>) and backcrossed to Nipponbare. The Al<sup>3+</sup> activity at which Al tolerance was screened was determined by identifying the Al<sup>3+</sup> activity that provided the greatest difference in tolerance between the parents. The tolerant parent of the RIL population, Azucena, and the tolerant parent of the BIL population, Nipponbare, are similar in Al tolerance, whereas the susceptible parent of the RIL population, IR64, is significantly more tolerant than the susceptible parent of the BIL population, Kasalath (<xref ref-type="fig" rid="pgen-1002221-g001">Figure 1A</xref>). To ensure that a normal distribution was obtained in each population, a different Al<sup>3+</sup> concentration was used for each mapping population. The RIL population was screened at 250 &#181;M Al<sup>3+</sup> because the Azucena parent is very Al tolerant and the IR64 parent is only moderately susceptible. The BIL population was screened at 120 &#181;M Al<sup>3+</sup> because the Kasalath parent is extremely Al sensitive, though the Nipponbare parent is very Al tolerant. <xref ref-type="fig" rid="pgen-1002221-g001">Figure 1</xref> displays the Al tolerance of each mapping parent in reference to the 373 genetically diverse rice accessions screened at 160 &#181;M Al<sup>3+</sup>. The genetic component of the phenotypic variance was calculated as VarG&#8202;=&#8202;VarG+Var(GxE)+error. QTL analysis was conducted using composite interval mapping (CIM) function in QTL Cartographer <xref ref-type="bibr" rid="pgen.1002221-Basten1">[81]</xref>. The significance threshold was determined by 1000 permutations.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Genome-Wide Association Analysis was performed using three approaches in all samples (373) with phenotypes. The first approach was the na&#239;ve approach, which is simply the linear regression of phenotype on the genotype for each SNP marker. The second approach was principle component analysis (PCA), where we obtained the four main PCs (principle components) that reflect the global main subpopulations in the sample to correct population structure estimated from software EIGENSOFT. <xref ref-type="bibr" rid="pgen.1002221-Price1">[82]</xref>. The first four PCs are included as cofactors in the regression model to correct population structure: <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002221.e001.jpg" /></inline-formula>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Here &#946; and &#947; are coefficient vectors for SNP effects and subpopulation PCs respectively. <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002221.e002.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002221.e003.jpg" /></inline-formula> are the corresponding SNP vector and first 4 PC vectors, and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002221.e004.jpg" /></inline-formula> is the random error term. The third approach was the linear mixed model proposed by <xref ref-type="bibr" rid="pgen.1002221-Zhao2">[62]</xref>, <xref ref-type="bibr" rid="pgen.1002221-Yu2">[63]</xref>, implemented in the R package EMMA <xref ref-type="bibr" rid="pgen.1002221-Kang1">[71]</xref>, which models the different levels of population structure and relatedness. The model can be written in a matrix form as: <italic>y&#8202;=&#8202;X&#946;</italic>+<italic>C&#947;</italic>+<italic>Z&#956;</italic>+<italic>e</italic> where &#946; and &#947; are the same as above, both of which are fixed effects, and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002221.e005.jpg" /></inline-formula> is the random effect accounting for structures and relatedness, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002221.e006.jpg" /></inline-formula> is corresponding design matrices, and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002221.e007.jpg" /></inline-formula> is the random error term. Assume <italic>&#956;</italic>&#8764;<italic>N(0,&#963;<sup>2</sup><sub>g</sub>K)</italic> and <italic>e</italic>&#8764;<italic>N(0,&#963;<sup>2</sup><sub>e</sub>I)</italic>, and K is the IBS matrix, as in <xref ref-type="bibr" rid="pgen.1002221-Zhao2">[62]</xref>. We also conducted GWA using both the na&#239;ve approach and the mixed model approach in each of the four main subpopulations (IND, AUS, TEJ, TRJ). For the mixed model, the model was changed to <italic>y&#8202;=&#8202;X&#946;</italic>+<italic>Zu+e</italic>, since there was no main subpopulation division within each subpopulation sample. Linkage disequilibrium decay and haploblocks were calculated at specific chromosome/gene regions using Haploview software <xref ref-type="bibr" rid="pgen.1002221-Barrett1">[83]</xref>.</p><p>Population structure was analyzed employing Expectation-Maximization techniques on an HMM model of per-marker ancestry along a chromosome with a weak linkage model between adjacent markers on the same chromosome induced by the HMM's state dependence on the previous marker's subpopulation assignment (M. Wright, Cornell University, personal communication). The 5,467 SNPs used for admixture analysis were a subset of the 36,901 high quality SNPs on the 44 K chip, and were selected based on their information content and ability to distinguish genetic groups, rather than individuals. The two main criteria used to select the subset of SNPs were a) good genomic distribution and minimal LD among those used in the analysis, and b) MAF&gt;0.05 in at least one subpopulation. The state of the HMM at each marker corresponds to the subpopulation of origin for the marker (and by extension, the region containing the marker and its adjacent markers). The number of <italic>a priori</italic> distinct subpopulations was K&#8202;=&#8202;5, consistent with that reported previously by Garris et al. 2005 and Ali et al., 2011 <xref ref-type="bibr" rid="pgen.1002221-Garris1">[40]</xref>, <xref ref-type="bibr" rid="pgen.1002221-Ali1">[66]</xref>. A set of 50 standard non-admixed &#8220;control&#8221; lines, 10 representing each of the Garris et al. subpopulations, that were genotyped on the 44 K rice SNP array were used to develop and evaluate the method. All 50 lines were correctly assigned to each of the subpopulations and concordant with previous results using STRUCTURE <xref ref-type="bibr" rid="pgen.1002221-Pritchard1">[84]</xref>, with little or no admixture or introgressions detected. The EM/HMM method was favored over the corresponding &#8220;linkage model&#8221; of recent versions of STRUCTURE because the EM/HMM model explicitly modeled inbreeding and estimated the inbreeding coefficient for each line independently, permitting lines in various stages of purification or inbreeding to homozygosity to be analyzed. The lines phenotyped in this study that were also genotyped on the 44 K SNP array were then analyzed, combined with these 50 control lines and the local ancestry along chromosomes were assigned by maximizing the state path of the HMM while simultaneously estimating subpopulation specific allele frequencies using the forward-backward algorithm. Using this method, introgressions from a foreign subpopulation into a line with a vast majority of the genetic background originating from a single subpopulation were detected.</p>