<p>The discovery stage of the GWAS included 8,938 Caucasian men of European descent drawn from seven epidemiological cohorts: the Framingham Heart Study (FHS), the Study of Health in Pomerania (SHIP), the Gothenburg Osteoporosis and Obesity Determinants (GOOD) study, the Cooperative Health Research in the Region of Augsburg (KORA) study, the Health, Aging and Body Composition (HEALTH ABC) study, the Rotterdam Study (RS1), and the Invecchiare in Chianti (InCHIANTI) (<xref ref-type="supplementary-material" rid="pgen.1002313.s006">Table S1</xref>). The replication stage consisted of 4,620 men from two epidemiological cohorts (the European Male Ageing Study [EMAS] and the Osteoporotic Fractures in Men [MrOS] Sweden study) for <italic>de novo</italic> genotyping of the top SNPs and one additional cohort (the Young Finns Study, [YFS, n&#8202;=&#8202;871]) with genome-wide association data available and joining the study after stage one was finished for <italic>in silico</italic> replication (<xref ref-type="supplementary-material" rid="pgen.1002313.s007">Table S2</xref>).</p><p>Exclusion criteria included chemical or surgical castration and/or medications affecting sex hormones such as steroid 5-alpha reductase inhibitors, and sex hormone antagonists. All studies were approved by local ethics committees and all participants provided written informed consent. Characteristics of the study samples and detailed descriptions of the participating cohorts, genotyping methods, quality control, and imputation procedures are provided in <xref ref-type="supplementary-material" rid="pgen.1002313.s009">Text S1</xref>.</p><p>Altogether, &#8764;2.5 million SNPs, imputed using the HapMapII CEU population, were tested for association with serum testosterone in the discovery stage. Genome-wide association analyses using an additive genetic linear regression model adjusted for age, BMI, and current smoking were conducted twice within each of the discovery cohorts using serum testosterone expressed as ng/dl, as well as inverse-normal transformed serum testosterone as outcomes.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To examine the robustness of the discovery results and to reduce the risk of spurious associations due to possible testosterone measurement heterogeneity between the individual cohorts, three different types of meta-analyses were performed in the discovery stage: 1) an inverse-variance weighted fixed effect model; 2) a z-score based analysis of the untransformed serum testosterone concentrations; and 3) a z-score based meta-analysis of the inverse-normal transformed values. Model 1) was used as main analysis since it allowed the computation of effect estimates, whereas the other two analysis models were used for verification and quality control checks of the main findings. All meta-analyses were performed using METAL (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/metal/">www.sph.umich.edu/csg/abecasis/metal/</ext-link>). The random effects model of the two <italic>SHBG</italic> locus SNPs was calculated using the R-package <italic>metafor</italic> (<ext-link ext-link-type="uri" ns0:href="http://www.r-project.org">www.r-project.org</ext-link>). Imputed genotypes were analyzed in all cohorts taking the genotype uncertainties into account. Genomic control was applied to each individual cohort's results and to the discovery stage meta-analysis to correct p-values for potential effects of mild population stratification. The estimated genomic control lambda was low for both the individual cohorts (range of &#955;<sub>GC</sub>: 1.00&#8211;1.07) and the meta-analyses (range of &#955;<sub>GC</sub>: 1.01&#8211;1.02), suggesting little residual confounding due to population stratification (<xref ref-type="supplementary-material" rid="pgen.1002313.s002">Figure S2</xref>).</p><p>To reduce the variance on serum testosterone induced by SHBG concentration, the GWAS included a genome-wide test for association of untransformed serum testosterone concentrations adjusted for age, BMI, current smoking, SHBG and SHBG<sup>2</sup> concentrations, again using both an inverse-variance weighted fixed effect as main analysis and a z-score based meta-analysis for quality control purposes.</p><p>A threshold of p&lt;5&#215;10<sup>&#8722;8</sup> was established <italic>a priori</italic> as the level for genome-wide significance in the discovery analyses <xref ref-type="bibr" rid="pgen.1002313-Peer1">[30]</xref>. SNPs that reached genome-wide significance in the inverse-variance weighted meta-analysis of untransformed serum testosterone concentrations with or without adjustment for SHBG and which had association results in at least five of the seven cohorts (for chr X: two cohorts with data available) were selected for further analyses. Notably, all autosomal SNPs that fulfilled these criteria also reached genome-wide significance in the other two types of meta-analyses. From these SNPs, all independent SNPs were taken to the replication stage.</p><p>We also assessed whether the lead SNPs from the continuous trait analyses were associated with low serum testosterone concentration (defined as &lt;300 ng/dl <xref ref-type="bibr" rid="pgen.1002313-Bhasin1">[16]</xref>; this level is slightly lower than that suggested recently by Wu et al [11 nmol/l&#8202;=&#8202;317 ng/dl] <xref ref-type="bibr" rid="pgen.1002313-Wu2">[17]</xref>) by binary logistic regression including the same covariates in the model used for the main analysis and meta-analyzing the within-cohort results using inverse-variance weighted fixed-effect model. The KORA cohort was not included in the meta-analyses of low serum testosterone as testosterone was measured using plasma in this cohort.</p><p>We determined the number of low serum testosterone concentration risk alleles (0 to 4) for the two lead SNPs of the <italic>SHBG</italic> locus in each individual and assessed the risk of low serum testosterone concentrations in the three replication cohorts (MrOS Sweden, EMAS, and YFS) using a trend test. Since only two subjects in the replication cohorts had four risk alleles, individuals having three and four risk alleles were grouped into one category to obtain more reliable effect estimates during the subsequent analyses. Details of test for independence, SHBG related analysis of the top SNPs and quality control steps performed can be found in <xref ref-type="supplementary-material" rid="pgen.1002313.s009">Text S1</xref>.</p><p>Methods for the measurement of serum testosterone and SHBG are given in <xref ref-type="supplementary-material" rid="pgen.1002313.s009">Text S1</xref>. Calculated free testosterone was for all subjects with both testosterone and SHBG available (n&#8202;=&#8202;13833; <xref ref-type="table" rid="pgen-1002313-t001">Table 1</xref> and <xref ref-type="table" rid="pgen-1002313-t002">Table 2</xref>) calculated by using a modified law of mass action equation, as described by Mazer <xref ref-type="bibr" rid="pgen.1002313-Mazer1">[31]</xref>. The concentrations of testosterone and SHBG and a fixed value for SHBG's dissociation constant were used in these calculations.</p><p>Free testosterone fraction was measured by an equilibrium dialysis method in 87 subjects with the CC genotype and 32 subjects with the CT genotype of rs6258 (<xref ref-type="fig" rid="pgen-1002313-g003">Figure 3D</xref>) <xref ref-type="bibr" rid="pgen.1002313-SinhaHikim1">[32]</xref>. Detailed description of the free testosterone fraction measurements is provided in <xref ref-type="supplementary-material" rid="pgen.1002313.s009">Text S1</xref>.</p><p>In experiments evaluating SHBG binding capacity, serum SHBG concentrations were determined by two-site immunofluorometric assay (PerkinElmer Life Sciences, Turku, Finland) <xref ref-type="bibr" rid="pgen.1002313-Niemi1">[33]</xref>, or by a steroid-binding capacity assay <xref ref-type="bibr" rid="pgen.1002313-Hammond1">[34]</xref>. For steroid-binding assays, serum samples were pre-incubated with dextran-coated charcoal (DCC) to remove endogenous steroids, prior to incubation with either [<sup>3</sup>H]5&#945;-dihydrotestosterone ([<sup>3</sup>H]DHT; specific activity 50 Ci/mmol) or [<sup>3</sup>H]testosterone (specific activity 40 Ci/mmol), bound from free [<sup>3</sup>H]steroid were separated using DCC as the separation reagent <xref ref-type="bibr" rid="pgen.1002313-Hammond1">[34]</xref>. The steroid-binding properties of SHBG in diluted serum samples or tissue culture medium were determined by Scatchard analysis <xref ref-type="bibr" rid="pgen.1002313-Hammond1">[34]</xref>. For the expression of SHBG protein, wild type (corresponding to the C genotype of rs6258) and rs6258 (corresponding to the T genotype of rs6258) SHBG cDNAs in the pRC/CMV expression vector were transfected into CHO cells, and G418 was used for selection of stably transfected cells. At near confluence, cells were washed with PBS and cultured in serum-free SFM4CHO medium (Thermo Scientific HyClone, Logan, UT) for four days before the SHBG-containing medium was harvested.</p>