<p>The barley QSM doubled-haploid mapping population was generated from a single Q21861 &#215; SM89010 F<sub>1</sub> plant <xref ref-type="bibr" rid="pgen.1002208-Steffenson2">[26]</xref>, <xref ref-type="bibr" rid="pgen.1002208-Borovkova1">[77]</xref>. All data used for infection type analysis are derived from Steffenson <italic>et al</italic>. <xref ref-type="bibr" rid="pgen.1002208-Steffenson1">[12]</xref>. Briefly, three to five seeds of each doubled-haploid line or parent were planted in plastic cones and placed in flats in a completely randomized design. Plants were placed in the greenhouse at 22&#176;C with supplemental lighting by 1,000-W sodium vapor lamps for 14 hours per day at the USDA-ARS Cereal Disease Lab, University of Minnesota, St. Paul. <italic>Pgt</italic> race TTKSK isolate 04KEN156/04 was initially increased on a susceptible wheat host, collected, desiccated, and stored in tubes at &#8722;80&#176;C. Nine days after sowing (PO:0007094 - first leaf unfolded), flats were inoculated with a low density of <italic>Pgt</italic> race TTKSK urediniospores (0.004 mg/plant) suspended in a lightweight mineral oil carrier using the inoculation protocols described by Sun and Steffenson <xref ref-type="bibr" rid="pgen.1002208-Sun1">[29]</xref>. After inoculation, the plants were placed in a mist chamber for 16 hours in the dark, followed by light for 5 hours, and then moved to the greenhouse using the previously described conditions. Plants were phenotyped at 14 to 17 days after inoculation. The full experiment was repeated twice, with the evaluation of three to five plants per replicate.</p><p>Field trials were carried out at the Kenya Agricultural Research Institute in Njoro, Kenya during the 2008 growing season. QSM DH lines were planted in 0.3 m rows (20&#8211;35 seed per row) in a completely randomized design with one replicate. Parents were included at random in the planting plan in three replicates. Infection phenotypes were scored on 7 October 2008, 17 October 2008, and 10 November 2008. The majority of lines were at the mid-dough stage of development (Zadoks scale 8.5; Feeke's scale 11.2) at the first scoring date <xref ref-type="bibr" rid="pgen.1002208-Large1">[78]</xref>, <xref ref-type="bibr" rid="pgen.1002208-Zadoks1">[79]</xref>. Other lines with later maturity reached the mid-dough stage by 17 October and 10 November. The bulk of the natural inoculum found in the field was typed to <italic>Pgt</italic> races TTKSK (used in seedling resistance assays) and TTKST (same virulence pattern as TTKSK with the addition of virulence for wheat stem rust resistance gene <italic>Sr24</italic>). <italic>Pgt</italic> race TTTSK (same virulence pattern as TTSKS with the addition of virulence for <italic>Sr36</italic>) may have been present at a low frequency.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Stakman infection types (ITs) for seedling plants were normalized using a modified approach that weights the counts of ordered ITs (<xref ref-type="fig" rid="pgen-1002208-g001">Figure 1A</xref> and <xref ref-type="supplementary-material" rid="pgen.1002208.s004">Figure S1</xref>, <xref ref-type="supplementary-material" rid="pgen.1002208.s013">Table S6</xref>) <xref ref-type="bibr" rid="pgen.1002208-Druka1">[30]</xref>. Weights given were 1.0, 0.65, 0.25, and 0.1 for the 1<sup>st</sup>, 2<sup>nd</sup>, 3<sup>rd</sup>, and 4<sup>th</sup> ordered ITs, respectively. IFs were determined by averaging weights for two replicates, where full weight is given to ITs of 0, 1, 2, and 3 or partial weights for ITs of &#8216;0;&#8217;, &#8216;1&#8722;&#8217;, &#8216;1+&#8217;, &#8216;2&#8722;&#8217;, &#8216;2+&#8217;, and &#8216;3&#8722;&#8217;. For partial weights, 70% is given to the IT shown (0, 1, 2, or 3) and 30% to the modified IT (&#8216;+&#8217; to the greater IT, &#8216;&#8722;&#8217; to the lower IT). In the unique case of &#8216;3+&#8217;, a weight of 1.3 was given to IT 3. For adult plants, LES was quantified on a scale from 0.25 to 1.0 based on resistance or full susceptibility, where resistant is equal to 0.25, moderately resistant is equal to 0.5, and a fully susceptible LES score is 1.0 (<xref ref-type="fig" rid="pgen-1002208-g001">Figure 1B</xref>). The IC was determined by multiplying the SEV by LES. Principal components analysis for seedling and adult phenotypic data was performed using R (<ext-link ext-link-type="uri" ns0:href="http://www.r-project.org">www.r-project.org</ext-link>) (<xref ref-type="supplementary-material" rid="pgen.1002208.s014">Table S7</xref>). Composite interval mapping (<bold>Zmapqtl</bold>; model 6) was performed with QTL Cartographer v1.17j, with a walking speed of 2 cM, window size of 10 cM, and five background markers (<bold>SRmapqtl</bold>) <xref ref-type="bibr" rid="pgen.1002208-Basten1">[39]</xref>. EWT were computed using permuted data (<bold>Prune</bold>) with reselection of background markers (<bold>SRmapqtl</bold>), where each iteration maximum LOD scores were stored and after 1,000 runs the 95<sup>th</sup> quantile (&#945;&#8202;=&#8202;0.05) was selected as the EWT <xref ref-type="bibr" rid="pgen.1002208-Churchill1">[41]</xref>, <xref ref-type="bibr" rid="pgen.1002208-Lauter1">[42]</xref>. QTLs that exceeded the EWT were extracted using <bold>Eqtl</bold>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Two flats (each flat contained 75 doubled-haploid lines + 4 replicates of each parent &#8202;=&#8202;81 cones/flat) were grown in a completely randomized design at the USDA-ARS Cereal Disease Lab, University of Minnesota, St. Paul. For the INOC flat, a higher density of <italic>Pgt</italic> race TTKSK urediniospores (0.25 mg/plant) was used as compared to the seedling phenotypic assay. For the MOCK flat, spore-free mineral oil was used. After inoculation, both flats were placed in the same mist chamber for 16 hours in the dark, exposed to light for 5 hours, and then moved to the greenhouse for 2 hours. Five seedlings were harvested, pooled, and placed in liquid nitrogen for each line in the population within a 1.5 hour period at 24 HAI. RNA was extracted using a hot acid-phenol protocol and RNAeasy columns (Qiagen) were used for further purification of the isolated RNA. Labeling, hybridization, washing, and scanning were performed according to standard Affymetrix protocols using the Barley1 GeneChip which contains probe sets representing 22,792 (21,439 non-redundant) genes <xref ref-type="bibr" rid="pgen.1002208-Close1">[32]</xref> at the ISU GeneChip Facility (<ext-link ext-link-type="uri" ns0:href="http://www.biotech.iastate.edu/facilities/genechip/Genechip.htm">www.biotech.iastate.edu/facilities/genechip/Genechip.htm</ext-link>).</p><p xmlns:ns0="http://www.w3.org/1999/xlink">At any single genetic locus, each of the 75 doubled haploid lines carries two copies of either the Q21861 or the SM89010 allele. These two genotypes can be distinguished by differential success in hybridizing RNA to Barley1 arrays, providing robust genetic markers. Transcript-derived markers were generated as described by Potokina and colleagues <xref ref-type="bibr" rid="pgen.1002208-Potokina1">[46]</xref>, using an implementation in Python (<ext-link ext-link-type="uri" ns0:href="http://www.python.org">www.python.org</ext-link>). This technique identifies single feature polymorphisms (SFPs) by using individual probes on the Affymetrix Barley1 GeneChip as quantifiable measures of probe hybridization efficiency. After background correction and quantile normalization using R/Bioconductor (<ext-link ext-link-type="uri" ns0:href="http://www.bioconductor.org">www.bioconductor.org</ext-link>), individual probe signals were separated into two distinct groups with <italic>k</italic>-means clustering. Goodness-of-fit using a Z-statistic found over 2,500 quality markers for the QSM genetic map. This analysis was performed separately with the INOC and MOCK data sets, and only those markers conserved between these data sets (1,503 markers) that had three (of 75) or fewer data points missing were included. A scaffold of 294 markers shared with the SxM doubled-haploid mapping population was used to place the remaining 1,200 markers <xref ref-type="bibr" rid="pgen.1002208-Potokina1">[46]</xref>. Available information for the genetic positions of genes represented on the Affymetrix Barley1 GeneChip was used to confirm marker order; this included data from a recently developed SNP-derived genetic map <xref ref-type="bibr" rid="pgen.1002208-Close2">[44]</xref>. Manual curation of marker positions was assisted with visualization of two-point marker linkages using MadMapper (<xref ref-type="supplementary-material" rid="pgen.1002208.s006">Figure S3</xref>; <ext-link ext-link-type="uri" ns0:href="http://cgpdb.ucdavis.edu/XLinkage/MadMapper">http://cgpdb.ucdavis.edu/XLinkage/MadMapper</ext-link>) <xref ref-type="bibr" rid="pgen.1002208-Potokina1">[46]</xref>, <xref ref-type="bibr" rid="pgen.1002208-West2">[80]</xref>. The final map has a total of 378 unique markers (bins) with a genetic length of 1,259 cM, with an average of approximately 3.3 recombination events between bins (<xref ref-type="supplementary-material" rid="pgen.1002208.s007">Figure S4</xref>, <xref ref-type="supplementary-material" rid="pgen.1002208.s001">Dataset S1</xref>).</p><p>ANOVA was performed with SAS v9.1 (SAS Institute Inc., Cary, North Carolina). All comparisons between these data sets were generated using Python scripts. <italic>q</italic>-values were determined using histogram-based estimation proposed by Mosig and colleagues <xref ref-type="bibr" rid="pgen.1002208-Mosig1">[81]</xref>, using the implementation by Nettleton and associates <xref ref-type="bibr" rid="pgen.1002208-Nettleton1">[33]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All microarray data for eQTL analysis were normalized with the Bioconductor implementation of the MAS5.0 algorithm (<ext-link ext-link-type="uri" ns0:href="http://www.bioconductor.org">www.bioconductor.org</ext-link>). Composite interval mapping was performed with QTL Cartographer v1.17j, using a walking speed of 2 cM, window size of 10 cM, and five background markers <xref ref-type="bibr" rid="pgen.1002208-Basten1">[39]</xref>. eQTL that exceeded individual EWT were extracted using a Python script, such that two peaks within close proximity were declared different eQTL if the distance between peaks was greater than 2 LOD <xref ref-type="bibr" rid="pgen.1002208-Potokina1">[46]</xref>. Individual EWT were computed using a combination of Python scripts, bash shell scripts, and QTL Cartographer. Briefly, composite interval mapping was performed using the same criteria in the eQTL analysis except the data were permuted (<bold>Prune</bold>) with reselection of background markers (<bold>SRmapqtl</bold>) a total of 1,000 times <xref ref-type="bibr" rid="pgen.1002208-Churchill1">[41]</xref>, <xref ref-type="bibr" rid="pgen.1002208-Lauter1">[42]</xref>. Maximum LOD scores were stored and the 95<sup>th</sup> quantile (&#945; &#8202;=&#8202; 0.05) was selected as the individual EWT.</p><p>The over and under-saturation of eQTL were identified using a contingency &#967;<sup>2</sup> test on the ratio of TDM:eQTL for a region as compared to the entire experiment. To fulfill the requirements of this contingency &#967;<sup>2</sup> test, we merged successive bins in the genetic map until the sum of observed eQTL and TDMs was greater than 73 for INOC and 85 for MOCK for each set of bins. The same bins were used to analyze both experiments by incorporating the distribution of eQTL in MOCK and INOC in parallel.</p><p>A bootstrap approach was used to estimate the significance associated with alternate regulation in the INOC and MOCK data sets for genes with eQTL in both data sets, using the maximum LOD eQTL. Genetic regions were compared with non-overlapping merged bins (superbins) that were generated with a greedy approach. This approach required a minimum number of TDMs and eQTL in INOC (73 TDM and eQTL) and MOCK (85 TDM and eQTL) to be placed within a superbin. Hence, the same bins in each experiment were collected into a single superbin. This approach is similar to that used for the identification of <italic>trans</italic>-eQTL hotspots and regions for over-saturation of differentially expressed genes (see <xref ref-type="sec" rid="s3">Results</xref>). Two strategies were used to account for (1) alternate regulation on the same chromosome and (2) regulation on different chromosomes between INOC and MOCK. For both, genes were redistributed from MOCK using probabilities determined by the distribution of eQTL in INOC based on the eQTL histogram. This was repeated 1,000 times for the maximum LOD eQTL in the overlap between INOC and MOCK. Probabilities were generated differently for the first and second strategies by including all genes with eQTL (10,127 genes) in INOC and MOCK or only those genes with maximum LOD eQTL on a different chromosome between data sets (5,538 genes). Bootstrap <italic>p</italic>-values were determined by comparing the observed overlap versus the 1,000 bootstrapped samples.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Gene ontology enrichment analysis was carried out using agriGO v1.0&#946; (<ext-link ext-link-type="uri" ns0:href="http://bioinfo.cau.edu.cn/agriGO">http://bioinfo.cau.edu.cn/agriGO</ext-link>) <xref ref-type="bibr" rid="pgen.1002208-Zhou2">[52]</xref>. Singular enrichment analysis (SEA) was performed using the default parameters, Fisher test, the Yekutieli multi-test adjustment method, a significance level of 0.05, and a minimum number of five mapped entries using the complete set of gene ontology terms. Parametric analysis of gene set enrichment (PAGE) was used with SEA default parameters, with a difference in requiring a minimum of ten mapped entries and FDR cutoff at 0.1.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All MIAME-compliant GeneChip profiling data are available as accession BB64 at the PLEXdb expression resource for plants and plant pathogens (<ext-link ext-link-type="uri" ns0:href="http://www.plexdb.org">www.plexdb.org</ext-link>), accession GSE20416 at NCBI-GEO, as well as accessions GN235, GN236, GN237, GN238 at GeneNetwork (<ext-link ext-link-type="uri" ns0:href="http://www.genenetwork.org">www.genenetwork.org</ext-link>) <xref ref-type="bibr" rid="pgen.1002208-Wang1">[82]</xref>.</p>