<p>The CARe project has been approved by the Committee on the Use of Humans as Experimental Subjects (COUHES) of the Massachusetts Institute of Technology, and by the Institutional Review Boards of each of the nine parent cohorts.</p><p>Affymetrix 6.0 genotyping and QC filtering of African-American samples from the CARe cardiovascular consortium was performed as described previously <xref ref-type="bibr" rid="pgen.1001371-Lettre1">[25]</xref>. After QC filtering for each of ARIC, CARDIA, CFS, JHS and MESA cohorts and subsequent merging, 8,367 samples and 770,390 SNPs remained. To limit relatedness among samples we restricted all analyses to a subset of 6,209 samples in which all pairs have genome-wide relatedness of 0.10 or less (inferred using the <italic>smartrel</italic> program in EIGENSOFT 3.0; see Web Resources). We merged CARe genotype data with genotype data from the HapMap3 project <xref ref-type="bibr" rid="pgen.1001371-Altshuler2">[18]</xref>. HapMap3 samples had been genotyped on both Affymetrix 6.0 and Illumina 1M chips. We excluded SNPs that did not pass QC in HapMap3, as well as A/T and C/G SNPs to avoid allele complementarity issues, leaving 556,698 SNPs for further analysis. (We note that HAPMIX accuracy is insensitive to the number of SNPs, if at least 250,000 SNPs are used <xref ref-type="bibr" rid="pgen.1001371-Price2">[23]</xref>.)</p><p>When run in default mode, HAPMIX outputs local ancestry estimates as the expected probability of 0, 1 or 2 copies of European ancestry at each SNP (see ref. <xref ref-type="bibr" rid="pgen.1001371-Price2">[23]</xref> and Web Resources). However, HAPMIX can also be run in a mode that outputs the inferred joint distribution of local ancestry and allele value, so as to resolve the &#8220;het-het&#8221; case (both genotype and local ancestry heterozygous). In order to obtain integer estimates of local ancestry, one approach is to simply round the probabilities, which however can lead to biased estimates in regions with limited SNP coverage. We chose an alternative approach that does not produce these types of biases: sampling from the probabilities for 0, 1 or 2 European chromosomes at each position. Results in this mode are highly concordant with the default mode, producing correlations of 100% in genome-wide ancestry and 98.8% in local ancestry.</p><p>We selected a random subset of 100,000 autosomal SNPs. For each SNP, we simulated phenotypes for <italic>R</italic>&#8202;=&#8202;1.0 (null model) and <italic>R</italic>&#8202;=&#8202;1.2,1.5,2.0 (causal models). For the null model, we chose random subsets of 1,000 cases and 1,000 controls. For causal models, we chose a random subset of 1,000 controls, and then chose 1,000 cases from the remaining samples so that samples with 0:1:2 reference alleles have relative probabilities 1:<italic>R</italic>:<italic>R</italic>
<sup>2</sup> of being chosen.</p><p>A &#967;<sup>2</sup>(1dof) statistic via the Armitage trend test with adjustment for genome-wide ancestry, as described previously <xref ref-type="bibr" rid="pgen.1001371-Price3">[50]</xref>. Genome-wide ancestry was inferred as the genome-wide average of local ancestry estimates from HAPMIX <xref ref-type="bibr" rid="pgen.1001371-Price2">[23]</xref>. We note that this is &gt;99% correlated to the top eigenvector of a principal components analysis run using CEU and YRI from HapMap3 to compute principal components <xref ref-type="bibr" rid="pgen.1001371-Altshuler2">[18]</xref>, <xref ref-type="bibr" rid="pgen.1001371-Patterson2">[20]</xref>.</p><p>A &#967;<sup>2</sup>(1dof) likelihood ratio test that compares the null hypothesis of case-control odds ratio R&#8202;=&#8202;1 with the alternate hypothesis of R &#8800;1, where R is assumed to be the same across populations, while the allele frequencies are treated as nuisance parameters.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For every local ancestry <italic>X</italic>
<sub>1</sub>
<italic>X</italic>
<sub>2</sub> (AA, AE, or EE) and phenotype <italic>Y</italic> (1 for cases, 0 for controls), let <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e001.jpg" /></inline-formula>, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e002.jpg" /></inline-formula>,<inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e003.jpg" /></inline-formula> denote the counts of individuals with genotypes 2, 1 or 0. Then the SNP1 likelihood is defined as<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001371.e004.jpg" /></disp-formula>where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e005.jpg" /></inline-formula>, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e006.jpg" /></inline-formula> represent the allele frequencies in cases, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e007.jpg" /></inline-formula>, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e008.jpg" /></inline-formula> represent allele frequencies in controls and R is the allelic odds ratio.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Then, the &#967;<sup>2</sup> statistic with 1 degree of freedom is:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001371.e009.jpg" /></disp-formula>
</p><p>A &#967;<sup>2</sup>(1dof) likelihood ratio test that compares the local ancestry in the disease cases to the average local ancestry across the genome in the same disease cases. This is more powerful than comparing cases to controls, since no statistical noise is introduced from controls <xref ref-type="bibr" rid="pgen.1001371-Patterson1">[7]</xref>. However, it is critical when using this approach to infer local ancestry using a method that has been shown not to produce artifactual deviations in average local ancestry in large data sets of controls from the admixed population <xref ref-type="bibr" rid="pgen.1001371-Price2">[23]</xref>.</p><p>Let <italic>&#952;<sub>i</sub></italic> be the genome-wide ancestry of individual <italic>i</italic>, and let <italic>N<sub>i</sub></italic> be the number of European chromosomes in individual <italic>i</italic> at the candidate SNP. Restricting to disease cases only, we define the likelihood as function of ancestry odds ratio <italic>&#937;</italic>, where <italic>&#937;</italic> is the multiplicative risk for disease given one or two European alleles. It follows that the likelihood for 2, 1, or 0 European alleles at individual i is:</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001371.e010.jpg" /></disp-formula>
<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001371.e011.jpg" /></disp-formula>
<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001371.e012.jpg" /></disp-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Then the likelihood is <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e013.jpg" /></inline-formula>, with the a &#967;<sup>2</sup>(1dof) likelihood ratio test defined as: <disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001371.e014.jpg" /></disp-formula>
</p><p>A &#967;<sup>2</sup>(2dof) that sums the SNP1 and the ADM statistics <xref ref-type="bibr" rid="pgen.1001371-Lettre1">[25]</xref>. We note that, since SNP1 conditions out the local ancestry and the ADM statistic employs only the local ancestry, these two tests contain independent information.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">A &#967;2(1dof) test that combines the SNP1 and ADM likelihood functions by using the implied ancestry odds ratio <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e015.jpg" /></inline-formula> under the assumption of a single causal variant with same odds ratio R across the European and African populations.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The MIX likelihood is defined as the product of the likelihoods for SNP1 and ADM as <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e016.jpg" /></inline-formula>, where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e017.jpg" /></inline-formula>is the relative increase in risk per extra European allele under the assumption of single causal variant with odds ratio R. It follows that <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e018.jpg" /></inline-formula> is a function of the SNP odds ratio <italic>R</italic> and the population allele frequencies in controls: <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e019.jpg" /></inline-formula>. We then compute a &#967;<sup>2</sup> statistic with 1 degree of freedom as:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001371.e020.jpg" /></disp-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">A &#967;2(1dof) test that compares the alternate hypothesis of different odds ratios in different ancestries with the null model that assumes the same odds ratio. The likelihood <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e021.jpg" /></inline-formula> extends the SNP1 likelihood by allowing ancestry specific odds ratios (R<sub>A</sub> and R<sub>E</sub>) which leads to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e022.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e023.jpg" /></inline-formula>. We then compute a &#967;<sup>2</sup>(1dof) statistic as: <disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001371.e024.jpg" /></disp-formula>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We incorporate the observed variance of the average local ancestry across the genome assuming that the average local ancestry <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e025.jpg" /></inline-formula>at each SNP is normally distributed with mean <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e026.jpg" /></inline-formula> and standard deviation <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e027.jpg" /></inline-formula>, where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e028.jpg" /></inline-formula> is the ancestry odds ratio. We estimate <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e029.jpg" /></inline-formula> empirically and set <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e030.jpg" /></inline-formula>, where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e031.jpg" /></inline-formula> is the empirical mean across the genome of the per SNP average local ancestry estimates. Then, the admixture likelihood becomes <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e032.jpg" /></inline-formula>. We can then compute a &#967;<sup>2</sup>(1dof) statistic, ADMGC, that incorporates the empirical variance and in the ADM score as:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001371.e033.jpg" /></disp-formula>In a similar manner we can replace <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e034.jpg" /></inline-formula>with <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e035.jpg" /></inline-formula>in the admixture component of the MIX likelihood to compute a new &#967;<sup>2</sup>(1dof) statistic MIXGC, that incorporates the empirical variance of the average local ancestry:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001371.e036.jpg" /></disp-formula>
</p><p>Many of the likelihoods defined above require a multidimensional optimization. The number of parameters optimized in the likelihoods is 3 for the SNP1 score, 1 for the ADM score, 3 for the MIX score and 4 for the HET score. (The HET score can be reduced to two independent 2-parameter optimizations by considering cases and controls separately.) For the ADM score, Newton&#8217;s method was used. For the SNP1, MIX and HET scores, Brent&#8217;s algorithm was used (GSL software library implementation; see Web Resources). The maximization is performed in one dimension over each parameter in turn, repeating for each parameter until the algorithm converges. In rare instances, extreme variation in the slope of the log likelihood as a function of odds ratio can cause the algorithm to not converge; in this situations a simple binary search is used.</p><p>We employed the widely used MACH <xref ref-type="bibr" rid="pgen.1001371-Li1">[51]</xref> imputation method to infer genotypes at untyped SNPs in the CARe African American samples. As reference haplotypes we used either the cosmopolitan approach of providing all the CEU and YRI haplotypes from HapMap Phase 3 data <xref ref-type="bibr" rid="pgen.1001371-Altshuler2">[18]</xref>, or a local ancestry aware approach in which, for every locus in every sample, we provided either YRI, CEU+YRI, CEU reference haplotypes to MACH according to the number of copies of YRI (2/1/0) inferred by HAPMIX. We note that the local ancestry aware approach has been previously shown to boost imputation accuracy in admixed populations [24. 35]. For both strategies we ran MACH in two steps, first by training the model parameters on a random sample of 200 individuals with the rounds parameter set to 50 followed by imputation of all the samples using the trained model from step 1. Importantly, we note that the local ancestry aware approach can be applied as an add-on to any imputation method.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Even when the true odds ratio is the same across populations, different imputation quality across the segments with different ancestries can lead to different estimates for the allelic odds ratios in European versus African segments. We account for this by adjusting the observed allelic odds ratios in the SNP1 and the MIX scores as follows. Following a derivation similar to <xref ref-type="bibr" rid="pgen.1001371-Pritchard1">[52]</xref> (see <xref ref-type="supplementary-material" rid="pgen.1001371.s009">Text S1</xref>) we show that the expected observed odds ratio at an imputed causal SNP with true odds ratio R, is a function of R, the imputation accuracy (as measured by the correlation between true and imputed SNP), and the allele frequency: <disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001371.e037.jpg" /></disp-formula>
<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1001371.e038.jpg" /></disp-formula>Unfortunately we do not know the true genotypes, and thus cannot compute the correlation between the true and imputed genotypes. However, reliable estimates for this correlation have been proposed; here we chose to use MACH <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e039.jpg" /></inline-formula> estimates shown to produce robust estimates of imputation quality <xref ref-type="bibr" rid="pgen.1001371-Zaitlen2">[53]</xref>. To estimate ancestry-specific imputation error rates, we restrict the computation to segments containing both alleles from that ancestry. Given that imputation accuracies are estimated directly from the data, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e040.jpg" /></inline-formula> depend on the term R and the allele frequencies. Then, the likelihood term from the MIX admixture association score becomes <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e041.jpg" /></inline-formula>. As in the previous version of the score, the optimization is done over the three free terms <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e042.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e043.jpg" /></inline-formula>. SNP1 score is updated in a similar fashion.</p><p>We randomly selected 100,000 autosomal SNPs and simulated phenotypes as described above using R&#8202;=&#8202;1.5. For all the compared scores, we computed the maximum statistic over all SNP across a region centered on the SNP of interest (taking the 20 SNPs upstream and 20 SNPs downstream). We computed the maximum of the statistics either over 41 SNPs by including the simulated causal SNP or over 40 SNPs by ignoring the statistics at the simulated causal SNP.</p><p>Case-control phenotypes for coronary heart disease (CHD) and type 2 diabetes (T2D) were ascertained as described previously <xref ref-type="bibr" rid="pgen.1001371-Lettre1">[25]</xref>. In each case, phenotypes were available for only a subset of the five CARe cohorts. Restricting to 6,209 unrelated individuals as defined above, we analyzed 929 cases and 4,150 controls for T2D, and 179 cases and 3,328 controls for CHD. For every analyzed SNP we performed imputation within a region of 10Mb centered on the SNP of interest using the MACH imputation method under the local ancestry aware framework. We assessed the scoring statistics at all SNPs within 100Kb of the SNPs of interest.</p><p>The FGFR2 locus has been associated with breast cancer in women of European and Asian descent <xref ref-type="bibr" rid="pgen.1001371-Hunter1">[36]</xref>, and further fine mapping in African-American women has identified SNP rs2981578 as showing the highest signal of association <xref ref-type="bibr" rid="pgen.1001371-Hunter1">[36]</xref>, <xref ref-type="bibr" rid="pgen.1001371-Udler1">[37]</xref>. We analyzed data from a GWAS including 5,761 unrelated African-American women from 11 epidemiological studies: The Multiethnic Cohort Study (MEC) <xref ref-type="bibr" rid="pgen.1001371-Kolonel1">[54]</xref>, The Los Angeles component of The Women&#8217;s Contraceptive and Reproductive Experiences (CARE) cohort <xref ref-type="bibr" rid="pgen.1001371-Marchbanks1">[55]</xref>, The Women&#8217;s Circle of Health Study (WCHS) <xref ref-type="bibr" rid="pgen.1001371-Ambrosone1">[56]</xref>, The San Francisco Bay Area Breast Cancer Study (SFBC) <xref ref-type="bibr" rid="pgen.1001371-John1">[57]</xref>, The Northern California Breast Cancer Family Registry (NC-BCFR) <xref ref-type="bibr" rid="pgen.1001371-John2">[58]</xref>, <xref ref-type="bibr" rid="pgen.1001371-John3">[59]</xref>, The Carolina Breast Cancer Study (CBCS) <xref ref-type="bibr" rid="pgen.1001371-Newman1">[60]</xref>, The Prostate, Lung, Colorectal, and Ovarian Cancer Screening Trial (PLCO) <xref ref-type="bibr" rid="pgen.1001371-Prorok1">[61]</xref>, The Nashville Breast Health Study (NBHS)<xref ref-type="bibr" rid="pgen.1001371-Zheng1">[62]</xref>, The Wake Forest University Breast Cancer Study (WFBC) <xref ref-type="bibr" rid="pgen.1001371-Smith3">[63]</xref>. Informed consent was obtained from all subjects. Detailed information about the design and organization of each study will be provided elsewhere (C. Haiman and colleagues, unpublished data). Genotyping was conducted using the Illumina Human1M-Duo BeadChip. A total of 1,043,036 SNPs were kept after QC filtering. Imputation was performed using the MACH software, providing as reference all the haplotypes of CEU and YRI HapMap Phase 2 panels). We focused our analysis on all the typed or imputed SNPs, 251 in total, located 100Kb upstream and downstream of SNP rs2981578.</p><p>For each of 100,000 autosomal SNPs, we simulated phenotypes for <italic>&#949;</italic>&#8202;=&#8202;0 (null model) and <italic>&#949;</italic>&#8202;=&#8202;0.05,0.10,0.20 (causal model), using a random subset of 2,000 samples. For the null model, phenotypes were sampled from a normal distribution with mean 0 and variance 1. For the causal model, the mean was shifted to 0:<italic>&#949;</italic>:2<italic>&#949;</italic> for 0:1:2 reference alleles. In each case, we subtracted out the overall phenotypic mean.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Let <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e044.jpg" /></inline-formula> denote genome-wide ancestry, genotype (0, 1 or 2) and phenotype for sample <italic>i</italic>. The model is <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e045.jpg" /></inline-formula>. We compute a &#967;<sup>2</sup>(1dof) statistic as <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e046.jpg" /></inline-formula>, where<inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e047.jpg" /></inline-formula> is adjusted to mean 0 and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e048.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e049.jpg" /></inline-formula> are each adjusted for<inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e050.jpg" /></inline-formula>. We also compute the effect size <italic>&#949;</italic>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Let <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e051.jpg" /></inline-formula> denote local ancestry (0, 1 or 2 European copies) for sample <italic>i</italic>. The model is <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e052.jpg" /></inline-formula>. We compute a &#967;<sup>2</sup>(1dof) statistic as <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e053.jpg" /></inline-formula>, where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e054.jpg" /></inline-formula> is adjusted to mean 0 and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e055.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e056.jpg" /></inline-formula> are each adjusted for <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e057.jpg" /></inline-formula>. We also compute the effect size <italic>&#949;</italic>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The model is <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e058.jpg" /></inline-formula>. We compute a &#967;<sup>2</sup>(1dof) statistic as <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e059.jpg" /></inline-formula>, where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e060.jpg" /></inline-formula> is adjusted to mean 0 and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e061.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e062.jpg" /></inline-formula> are each adjusted for <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e063.jpg" /></inline-formula>. We also compute the effect size <italic>&#949;</italic>.</p><p>We sum the two &#967;<sup>2</sup>(1dof) statistics to produce a &#967;<sup>2</sup>(2dof) statistic.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Let <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e064.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e065.jpg" /></inline-formula>denote the number of reference alleles of African or European local ancestry. If joint local ancestry and phasing information is not available and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e066.jpg" /></inline-formula>, we use expected values <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e067.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e068.jpg" /></inline-formula>, where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e069.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e070.jpg" /></inline-formula> are estimated as above by maximizing <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e071.jpg" /></inline-formula>. The model is <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e072.jpg" /></inline-formula>. We compute a &#967;<sup>2</sup>(2dof) statistic as <italic>N</italic> times the proportion of variance of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e073.jpg" /></inline-formula> jointly predicted by <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e074.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e075.jpg" /></inline-formula>, where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e076.jpg" /></inline-formula> are adjusted to mean 0 and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e077.jpg" /></inline-formula> are adjusted for <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e078.jpg" /></inline-formula>. We also compute the effect sizes <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e079.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e080.jpg" /></inline-formula>. We define the <italic>QHET</italic> score as the &#967;<sup>2</sup>(2dof) statistic minus the <italic>QSNP1</italic> score, thus testing the statistical significance of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1001371.e081.jpg" /></inline-formula>.</p><p>LDL and HDL cholesterol phenotypes in CARe samples were ascertained as described previously. We analyzed 5,801 samples for LDL and 5,946 samples for HDL for which phenotypic data were available, restricting to 6,209 unrelated individuals as defined above. For every analyzed SNP we performed imputation within a region of 10Mb centered on the SNP of interest using the MACH imputation method under the local ancestry aware framework. We assessed the scoring statistics at all SNPs within 100Kb of the SNPs of interest.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://www.hsph.harvard.edu/faculty/alkes-price/software/">http://www.hsph.harvard.edu/faculty/alkes-price/software/</ext-link>(MIXSCORE software)</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://www.hsph.harvard.edu/faculty/alkes-price/software/">http://www.hsph.harvard.edu/faculty/alkes-price/software/</ext-link>(EIGENSOFT software)</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<ext-link ext-link-type="uri" ns0:href="http://www.stats.ox.ac.uk/~myers/software.html">http://www.stats.ox.ac.uk/~myers/software.html</ext-link> (HAPMIX software)</p>