<p>The studies described in this manuscript have been approved by the ethics
committee of the respective institutions: Karolinska Institutet (Sweden),
Scientific Ethics Committee system (Denmark), University of California, Berkeley
(US), National Cancer Institute, National Institutes of Health (US), Mayo Clinic
(US), University of British Columbia (Canada), Yale University (US), University
of Sydney (Australia).</p><p>The SCALE study is a population-based study of the etiology of NHL carried out in
all of Denmark and Sweden during 1999 to 2002 <xref ref-type="bibr" rid="pgen.1001378-Smedby1">[26]</xref>. NHL subtype diagnoses were
reviewed and reclassified according to the World Health Organization (WHO)
classification <xref ref-type="bibr" rid="pgen.1001378-Jaffe1">[1]</xref> as previously described <xref ref-type="bibr" rid="pgen.1001378-Smedby1">[26]</xref>. For this GWAS (SCALE1) we
used DNA from 400 cases with follicular lymphoma (FL; 150 from Denmark and 250
from Sweden) and from 150 Danish controls, individually matched to the Danish FL
cases by sex and age at study inclusion. We also used material collected from
673 control subjects in a separate Swedish population-based case-control study
of rheumatoid arthritis (the Eira study) <xref ref-type="bibr" rid="pgen.1001378-Plenge1">[21]</xref>, <xref ref-type="bibr" rid="pgen.1001378-Plenge2">[27]</xref>. The latter was conducted
during 1996 to 2005 among residents 18 to 70 years of age in the southern and
central parts of Sweden (including 90% of Swedish residents). Hence, the
population controls recruited in this study were considered to represent the
same study population as the Swedish component of the SCALE study with regard to
genetic variation. Genotyping completion rates were similar between cases and
controls; out of 400 cases and 823 controls genotyped, 379 cases (95%)
and 791 controls (96%) were included in the final analysis. Study
subjects used in Stages 2, 3 and validation in other NHL subtypes (<xref ref-type="table" rid="pgen-1001378-t001">Table 1</xref>, <xref ref-type="supplementary-material" rid="pgen.1001378.s007">Table S1</xref>,
<xref ref-type="supplementary-material" rid="pgen.1001378.s008">S2</xref>)
have been previously described <xref ref-type="bibr" rid="pgen.1001378-Conde1">[6]</xref>, <xref ref-type="bibr" rid="pgen.1001378-Cerhan1">[8]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1001378-Spinelli1">[12]</xref>, and details are
available as supporting text (<xref ref-type="supplementary-material" rid="pgen.1001378.s018">Text S1</xref>). For the SCALE2 NHL subtype
validation study, we used the rest of the lymphoma cases with blood samples
originally recruited in SCALE (n&#8202;=&#8202;1869), Danish control
subjects not included in the GWAS (n&#8202;=&#8202;556), a second set
of control subjects from the Eira study (n&#8202;=&#8202;742) and a
third group of controls recruited in a national population-based case-control
study of breast cancer, the Cancer and Hormones Replacement in Sweden (CAHRES)
study <xref ref-type="bibr" rid="pgen.1001378-Magnusson1">[28]</xref>
(n&#8202;=&#8202;720). The control subjects from this study were
randomly selected from the Swedish general population to match the expected age
distribution of the participating breast cancer cases (50 to 74 years).</p><p>Stage I genotyping of 317,503 single nucleotide polymorphisms (SNPs) was done on
the HumanHap300 (version 1.0) array. Validation genotyping was done using
Sequenom iPlex; SNPs in the human leukocyte antigen (HLA) region that failed
primer design for Sequenom assays were genotyped using Taqman (Applied
Biosystems).</p><p>The scan included 317,503 SNPs from the HumanHap300 (version 1.0) array. The
datasets were filtered on the basis of SNP genotyping call rates
(&#8805;&gt;95% completeness), sample completion rate (&#8805;90%),
minor allele frequency (MAF; all subjects as well as cases and controls
separately &#8805;0.03) and non-deviation from Hardy-Weinberg equilibrium (HWE;
p&lt;10<sup>&#8722;6</sup>). We also excluded SNPs with cluster plot
problems, and those on the X and Y chromosomes. Study subjects with gender
discrepancies and/or labelling errors were removed. We also removed individual
samples with evidence of cryptic family relationships (identified using
the&#8211;genome command in PLINK). To detect outliers in terms of population
stratification, we performed principal component (PC) analysis using the
EIGENSTRAT software (<xref ref-type="supplementary-material" rid="pgen.1001378.s004">Figure S4</xref>). A subset of linkage
disequilibrium (LD) thinned SNPs was selected such that all pair-wise
associations had r<sup>2</sup>&lt;0.2, and long-range regions of high LD,
reported to potentially confound genome scans, were removed <xref ref-type="bibr" rid="pgen.1001378-Price1">[29]</xref>. Twenty-five
samples were removed as population outliers on the basis of their values on the
first three PCs. To adjust for possible stratification in our association
analyses we adjusted the regression analyses using the first three PCs; the
number of PCs used for adjustment was determined by plotting the eigenvalues and
locating the position of the &#8220;elbow&#8221; on the scree plot (<xref ref-type="supplementary-material" rid="pgen.1001378.s005">Figure S5</xref>).
Wald tests, treating minor allele counts as continuous covariates were used to
test for association. The genomic inflation factor (&#955;) was calculated to be
1.0283 after adjusting for the first three PCs, suggesting the presence of
minimal stratification. Quantile-quantile plots for the associations before and
after adjustment are shown in <xref ref-type="supplementary-material" rid="pgen.1001378.s006">Figure S6</xref>. Finally, we assessed associations
of age and sex with main genotypes among the control subjects to address the
possibility of confounding by these factors (<xref ref-type="supplementary-material" rid="pgen.1001378.s017">Table S11</xref>).
As there was no evidence of associations of age or sex with genotypes among the
controls, we did not adjust for them in the final main effects analyses of
genotypes.</p><p>In Stage 2, similar quality control measures were applied as in Stage 1,
including genotyping call rate &#8805;95%, sample completion rate
&#8805;90%, and MAF &#8805;0.05. We tested each validation study for
association using trend tests. For meta-analyses across studies and NHL
subtypes, we used the Cochran-Mantel-Haenszel method to calculate the combined
odds ratio and P-value, and &#967;<sup>2</sup> tests for heterogeneity.
Multivariate logistic regression was used to test for independence of SNP
effects. For validation among other NHL subtypes, the control subjects were the
same as those in Stages 2 and 3 for validation in FL for all studies except
SCALE2. Only European-ancestry subjects were included, and the possibility of
population stratification affecting the results has been thoroughly explored and
found to be low in earlier investigations in the same populations <xref ref-type="bibr" rid="pgen.1001378-Conde1">[6]</xref>, <xref ref-type="bibr" rid="pgen.1001378-Cerhan1">[8]</xref>.</p><p>We used IMPUTEv1 for the imputation of SNPs from the 1000 Genomes pilot1 CEU data
(August 2009 release); and the HapMap Phase II release 22 CEU data. We set a
strict threshold for imputation, using only SNPs with confidence scores of
&#8805;0.9, call rates &#8805;90%, non-deviation from Hardy-Weinberg
equilibrium P &gt;0.001 and MAF &gt;0.01. The imputation was done on the Stage 1
samples separately for each of the two reference datasets and SNPs showing a
discordance of &gt;5% between the genotypes imputed with the two datasets
were excluded from further analysis. The data were then merged using HapMap II
as the master dataset to which additional imputed SNPs from the 1000 Genomes
dataset were added. HLA alleles were imputed by identifying tag SNPs <xref ref-type="bibr" rid="pgen.1001378-deBakker2">[15]</xref> from the
genotyped and imputed SNP dataset. We used PLINK for haplotype imputation with
the tag SNPs and downstream association analyses. Only haplotypes with call
rates &gt;90%, MAF&gt;1% and probability thresholds &gt;0.8 were
analyzed.</p><p>For coalescence analysis all 12 SNPs (genotyped in this study and within a region
of &#8764;177 Kb) adjacent to the two SNPs associated with the FL risk were used
to construct haplotypes. These were phased using the PHASE program <xref ref-type="bibr" rid="pgen.1001378-Stephens1">[30]</xref> and
tested for association using PLINK. The ancestral haplotype was constructed from
the chimpanzee (PanTro2) allele whenever possible, and otherwise from the
macaque alleles. An ancestral recombination graph was constructed using the
program Beagle <xref ref-type="bibr" rid="pgen.1001378-Song1">[13]</xref>, <xref ref-type="bibr" rid="pgen.1001378-Lyngs1">[31]</xref> which allows recombination assuming an infinite site
mutation model. After identifying the first recombination event the haplotype
segment before the recombination spot was used to construct a median
&#8211;joining network using the Network program <xref ref-type="bibr" rid="pgen.1001378-Bandelt1">[14]</xref>. The alleles of the
imputed SNP rs9378212 were then phased on each haplotype segment using the PHASE
program.</p><p>The URLs for the data and analytic approaches presented herein are as
follows:</p><p xmlns:ns0="http://www.w3.org/1999/xlink">1000 Genomes <ext-link ext-link-type="uri" ns0:href="http://1000genomes.org">http://1000genomes.org</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">HapMapII <ext-link ext-link-type="uri" ns0:href="http://www.hapmap.org">http://www.hapmap.org</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">IMPUTEv1 <ext-link ext-link-type="uri" ns0:href="https://mathgen.stats.ox.ac.uk/impute/impute_v1.html">https://mathgen.stats.ox.ac.uk/impute/impute_v1.html</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">mRNA by SNP browser <ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/liang/asthma/">http://www.sph.umich.edu/csg/liang/asthma/</ext-link>
</p><p xmlns:ns0="http://www.w3.org/1999/xlink">R script for recombination plot <ext-link ext-link-type="uri" ns0:href="http://www.broadinstitute.org/science/projects/diabetes-genetics-initiative/plotting-genome-wide-association-results">http://www.broadinstitute.org/science/projects/diabetes-genetics-initiative/plotting-genome-wide-association-results</ext-link>
</p>