<p>Following our conjecture that SNPs associated with the genes in a pathway involved in disease will exhibit more within-group similarity than across-group similarity, we propose Pathways of Distinction Analysis (PoDA), a method designed to address the following questions:</p><p>Given some set of SNPs, do we find that, on average, cases are &#8220;closer&#8221; to other cases than to controls (or that controls are &#8220;closer&#8221; to other controls than to cases)?</p><p>If we look for these distinctions systematically over all SNP-sets of potential interest, can we use it to single out SNP-sets which may be associated with disease?</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In PoDA, a set of SNPs are selected, and for each sample we compute whether it is closer to the pool of remaining cases or controls across that SNP set, using the relative distance statistic described below. Once this is done for every sample, the distribution of the relative distance statistic is compared in the cases and controls using a nonparametric statistic, addressing the first question above. This may be carried out amongst all SNP sets of interest, adjusting the <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e012.jpg" /></inline-formula>-value for the multiple hypotheses, to find SNP sets for which cases more strongly resemble the population of remaining cases while controls more strongly resemble the population of remaining controls.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We begin with a discussion of how we measure the relative distance of an individual to the other cases vs. other controls. A simple but computationally intensive approach is to represent each sample by a vector in an <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e013.jpg" /></inline-formula>-dimensional space, where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e014.jpg" /></inline-formula> is the number of SNPs in the group of interest. One can then compute, between each sample pair, their distance in this <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e015.jpg" /></inline-formula>-dimensional space using a Euclidean, Manhattan, or Hamming metric. For each sample, we would define its relative distance statistic as the mean of its distance to other controls minus the mean of its distance to other cases; a sample that is more similar to cases will exhibit a positive statistic, whereas one that is more similar to other controls will exhibit a negative statistic. For the given SNP set, we would then have for each sample a value quantifying its relative distance that was computed without knowledge of that sample's class (i.e., using a leave-one-out scheme) and could then be used in further tests. By doing this for all pathways of interest, one derives a relative distance value for each sample in each pathway.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">This brute-force approach, while conceptually clear, has two significant drawbacks. The first is that the distance computation is <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e016.jpg" /></inline-formula> where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e017.jpg" /></inline-formula> is the total number of samples in the study&#8211;a considerable undertaking, particularly if many SNP sets are to be analyzed, and one that becomes exceedingly troublesome in the context of permutation tests. The second drawback is that because we are taking the mean of the distances, a sample that is situated squarely within a cluster of cases may have a large case-distance value due to the dispersion of cases around it. Both of these issues are circumvented by instead considering the relative distance to the <italic>centroids</italic> of the cases and controls in the <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e018.jpg" /></inline-formula>-dimensional space, a computation that can be performed in <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e019.jpg" /></inline-formula> for all <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e020.jpg" /></inline-formula> samples. It is this approach that PoDA employs, as follows:</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In <xref ref-type="bibr" rid="pgen.1002101-Homer1">[29]</xref>, <xref ref-type="bibr" rid="pgen.1002101-Braun1">[30]</xref>, the authors consider a measure of individual <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e021.jpg" /></inline-formula>'s distance to two population samples <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e022.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e023.jpg" /></inline-formula> at locus <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e024.jpg" /></inline-formula>,<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002101.e025.jpg" /><label>(1)</label></disp-formula>where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e026.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e027.jpg" /></inline-formula> are the minor allele frequencies (MAFs) of SNP <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e028.jpg" /></inline-formula> in samples <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e029.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e030.jpg" /></inline-formula>, and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e031.jpg" /></inline-formula> is <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e032.jpg" /></inline-formula>'s genotype at <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e033.jpg" /></inline-formula> corresponding to homozygous major, heterozygous, and homozygous minor alleles, respectively (i.e., the frequency of minor allele in that individual. The first term quantifies how different <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e034.jpg" /></inline-formula>'s MAF is from <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e035.jpg" /></inline-formula>'s for a given locus <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e036.jpg" /></inline-formula>; the second term quantifies how different <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e037.jpg" /></inline-formula>'s MAF is from <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e038.jpg" /></inline-formula>'s at locus <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e039.jpg" /></inline-formula>; and so <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e040.jpg" /></inline-formula> gives the distance of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e041.jpg" /></inline-formula> relative to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e042.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e043.jpg" /></inline-formula> at locus <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e044.jpg" /></inline-formula>. Since the minor allele frequencies <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e045.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e046.jpg" /></inline-formula> are computed by averaging the genotypes (again, written as <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e047.jpg" /></inline-formula>) in samples <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e048.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e049.jpg" /></inline-formula> respectively, it is clear that <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e050.jpg" /></inline-formula> is the distance from <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e051.jpg" /></inline-formula> to the centroid of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e052.jpg" /></inline-formula> along the coordinate <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e053.jpg" /></inline-formula> (and likewise for the <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e054.jpg" /></inline-formula> term). It can be seen from Eq. 1 that positive <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e055.jpg" /></inline-formula> implies that <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e056.jpg" /></inline-formula> is closer to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e057.jpg" /></inline-formula> than to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e058.jpg" /></inline-formula>, and that negative <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e059.jpg" /></inline-formula> implies that <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e060.jpg" /></inline-formula> is closer to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e061.jpg" /></inline-formula> than to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e062.jpg" /></inline-formula>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">By computing <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e063.jpg" /></inline-formula> at each locus <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e064.jpg" /></inline-formula> and taking the standardized mean across the <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e065.jpg" /></inline-formula> loci, <xref ref-type="bibr" rid="pgen.1002101-Homer1">[29]</xref> obtain a distance score <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e066.jpg" /></inline-formula> which quantifies how close <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e067.jpg" /></inline-formula> is relative to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e068.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e069.jpg" /></inline-formula> across all <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e070.jpg" /></inline-formula> loci under consideration,<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002101.e071.jpg" /><label>(2)</label></disp-formula>where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e072.jpg" /></inline-formula> denotes the mean of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e073.jpg" /></inline-formula> across all loci <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e074.jpg" /></inline-formula>. That is, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e075.jpg" /></inline-formula> provides a means to quantify whether <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e076.jpg" /></inline-formula>'s MAFs are closer to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e077.jpg" /></inline-formula>'s MAFs or <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e078.jpg" /></inline-formula>'s MAFs on average for the loci under consideration. It is instructive to consider the geometrical interpretation of Eq. 2. Is clear upon inspection that the numerator in Eq. 2 is equal, up to a factor of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e079.jpg" /></inline-formula>, to the difference in Manhattan distances between <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e080.jpg" /></inline-formula> and the (nonstandardized) <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e081.jpg" /></inline-formula> centroid and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e082.jpg" /></inline-formula> and the (nonstandardized) <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e083.jpg" /></inline-formula> centroid; in this sense, Eq. 2 resembles a nearest-centroid classifier. However, the denominator scales the relative distances by their variance across the <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e084.jpg" /></inline-formula> SNPs; that is, a sample <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e085.jpg" /></inline-formula> who is consistently closer to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e086.jpg" /></inline-formula> than to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e087.jpg" /></inline-formula> for each of the <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e088.jpg" /></inline-formula> SNPs will obtain a higher <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e089.jpg" /></inline-formula> than an individual who is variously closer to either across the <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e090.jpg" /></inline-formula> SNPs under consideration.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">By assigning the (non-<inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e091.jpg" /></inline-formula>) controls as <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e092.jpg" /></inline-formula> and the (non-<inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e093.jpg" /></inline-formula>) cases as <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e094.jpg" /></inline-formula>, we can compute a statistic <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e095.jpg" /></inline-formula> quantifying <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e096.jpg" /></inline-formula>'s distance to other cases relative to <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e097.jpg" /></inline-formula>'s distance to other controls. If we then apply this systematically to all individuals in the study population (removing that individual, computing the MAF's <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e098.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e099.jpg" /></inline-formula> for the remaining individuals who comprise <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e100.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e101.jpg" /></inline-formula>, and then computing <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e102.jpg" /></inline-formula> in a leave-one-out manner), we can obtain distributions of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e103.jpg" /></inline-formula> statistics in cases and controls that may be compared. Here, the null hypothesis is that case and control <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e104.jpg" /></inline-formula> distributions do not differ, with the alternative hypothesis that the cases have higher <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e105.jpg" /></inline-formula> values than do controls, i.e., that they are closer (via Eqs. 1&#8211;2) to other cases than are controls.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We can use <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e106.jpg" /></inline-formula> in the following manner to answer the questions posed above by applying it in a leave-one-out manner in each pathway:</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For a given pathway <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e107.jpg" /></inline-formula>, select the <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e108.jpg" /></inline-formula> SNPs associated with that pathway;</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For every sample <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e109.jpg" /></inline-formula>, remove <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e110.jpg" /></inline-formula> from the case or control group as needed, and compute <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e111.jpg" /></inline-formula> with respect to the remaining cases and controls using the SNPs chosen in step 1.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Quantify the differences in distribution of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e112.jpg" /></inline-formula>'s for the case samples versus that of the controls and test for significance.</p><p>By systematically carrying out the above steps on all pathways of interest, we can identify pathways for which there appears to be differential homogeneity in cases and controls, indicating that the pathway may play a disease-related role. The details of the algorithm are explained below, and summarized in <xref ref-type="table" rid="pgen-1002101-t001">Table 1</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In <xref ref-type="bibr" rid="pgen.1002101-Braun1">[30]</xref>, we examined Eqs. 1&#8211;2 and found that the magnitude of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e131.jpg" /></inline-formula> is influenced both by the MAF differences <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e132.jpg" /></inline-formula> (that is, how distant the centroids of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e133.jpg" /></inline-formula> and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e134.jpg" /></inline-formula> are) and by correlations between the SNPs under consideration (due to the penalization for variance in <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e135.jpg" /></inline-formula> provided by the denominator of Eq. 2. These properties are extremely well-suited to the application we propose: pathways with few highly-significant SNPs will yield large <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e136.jpg" /></inline-formula> differences (due to the influence of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e137.jpg" /></inline-formula>), as will pathways with SNPs that are highly correlated yet have subtle individual MAF differences, corresponding to the concerted action of multiple SNPs.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">At the same time, however, we wish to ensure that the pathways we select as having differential <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e138.jpg" /></inline-formula> are not being influenced large LD blocks covered by the SNPs in the genes on the pathway. That is, we wish to ensure that the SNP correlations which drive <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e139.jpg" /></inline-formula> are reflective of epistatic effects between different genes rather than recombination events within a gene. To this end, we select a single SNP to represent each gene, based on the desire to detect multi-<italic>gene</italic> rather than multi-SNP effects.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In practice, SNPs are selected as follows: for each pathway represented in the Pathway Interaction Database <xref ref-type="bibr" rid="pgen.1002101-Schaefer1">[32]</xref> (PID, <ext-link ext-link-type="uri" ns0:href="http://www.pid.nci.gov">http://www.pid.nci.gov</ext-link>, containing annotations from BioCarta, Reactome, and the NCI/Nature database <xref ref-type="bibr" rid="pgen.1002101-Schaefer1">[32]</xref>) and KEGG <xref ref-type="bibr" rid="pgen.1002101-Kanehisa1">[33]</xref>, we select the associated genes. Using dbSNP <xref ref-type="bibr" rid="pgen.1002101-Sherry1">[34]</xref>, we retrieve the SNPs associated with the pathway genes that are present in the data, excluding those with <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e140.jpg" /></inline-formula> missing data or with minor allele frequency <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e141.jpg" /></inline-formula> in either case of control group. We necessarily exclude pathways for which only one gene is probed by the remaining SNPs. Because we are interested in <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e142.jpg" /></inline-formula> values that are driven by correlations <italic>across</italic> genes (and not in individual genes covered by many SNPs with high LD), we select for each gene its most significant SNP in a univariate test of association (Fisher's exact test). It should be noted here that while the SNP chosen for each gene is the most significant of that gene's SNPs, it is <italic>not</italic> necessarily significantly associated with disease. Our goal here is not to filter based on SNP significance, but rather to select, for each gene, a single marker that is as informative as possible.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Having selected the SNPs of interest, we compute <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e143.jpg" /></inline-formula> at each locus for every sample by selectively removing it and comparing it to the remaining cases and controls, as described above. For each pathway <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e144.jpg" /></inline-formula>, we compute <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e145.jpg" /></inline-formula> for <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e146.jpg" /></inline-formula> the SNPs <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e147.jpg" /></inline-formula> that comprise it, yielding a distribution of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e148.jpg" /></inline-formula> for cases and another distribution for controls. The difference in the location of the case and control <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e149.jpg" /></inline-formula> distributions is then quantified nonparametrically by computing the Wilcoxon rank sum statistic, defined as<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002101.e150.jpg" /><label>(3)</label></disp-formula>where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e151.jpg" /></inline-formula> is the rank of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e152.jpg" /></inline-formula> amongst all samples <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e153.jpg" /></inline-formula> for a given pathway <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e154.jpg" /></inline-formula>. Eq. 3 thus quantifies, non-parametrically, the degree to which cases are &#8220;closer&#8221; to other cases and controls &#8220;closer&#8221; to other controls across a set of SNPs for all individuals in the GWAS.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To illustrate the above, we consider a simulated GWAS of 250 cases and 250 controls and 50 SNPs, shown in <xref ref-type="fig" rid="pgen-1002101-g001">Figure 1</xref>, and ask whether we are able to detect a 12-SNP pathway in which a subset of SNPs appear to have an epistatic interaction. Alleles were simulated as binomial samples from a source population with MAFs ranging from 0.1 to 0.4 across the 50 SNPs, and case labels were assigned such that a combintion of homozygous minor alleles at SNPs 1 and 2 or 3 (i.e., <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e155.jpg" /></inline-formula>) conferred a three-fold relative risk, mimicking an epistatic interaction between SNPs 1 and 2 and SNPs 1 and 3 (<xref ref-type="fig" rid="pgen-1002101-g001">Figure 1a</xref>). Alone, none of the 50 SNPs showed any association with case status, nor was any SNP significantly out of HWE in either cases or controls. However, the &#8220;shared pattern&#8221; in SNPs 1&#8211;3 is such that a 12 SNP pathway comprising SNPs 1&#8211;12 yields greater <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e156.jpg" /></inline-formula> in cases than in controls as can been seen in <xref ref-type="fig" rid="pgen-1002101-g001">Figure 1b</xref>, while a random 12 SNP pathway selected from the 50 SNPs (that includes SNP 3, but neither SNP 1 or 2) shows no difference in <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e157.jpg" /></inline-formula> values as seen in <xref ref-type="fig" rid="pgen-1002101-g001">Figure 1c</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Alleles at 50 loci for 250 cases and 250 controls were simulated such that each SNP was in HWE and not associated with case status, but homozygous minor (red) at both loci 1 and 2 or 1 and 3 yielded a three-fold relative risk (a). A 12-SNP pathway comprising SNPs 1&#8211;12 shows differential <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e158.jpg" /></inline-formula> distributions (b); a random 12-SNP pathway does not (c). Boxplots are overlayed on the scatterplots of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e159.jpg" /></inline-formula> for clarity.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">While the Wilcoxon statistic <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e160.jpg" /></inline-formula> is normal in the large-sample limit and can be directly compared to a Gaussian, to truly evaluate the significance of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e161.jpg" /></inline-formula> for a given pathway <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e162.jpg" /></inline-formula>, we must address two sources of bias: the number of SNPs per gene, and the size of the pathway. To address these issues, we introduce a normalized Pathway Distinction Score <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e163.jpg" /></inline-formula> that we test for significance using a resampling procedure.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">First, we expect that because we have selected for each gene the single most informative SNP, we are pre-disposed to seeing higher <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e164.jpg" /></inline-formula> for pathways that contain large genes. Because large genes will be more likely to contain highly-significant SNPs by chance, the concern has been raised that <xref ref-type="bibr" rid="pgen.1002101-Wang2">[18]</xref>, <xref ref-type="bibr" rid="pgen.1002101-Kraft1">[35]</xref> selecting the single most significant SNP as a proxy for the gene (as is done here) will lead to a bias toward pathways that contain an abundance of large genes. To account for this, we follow the approach in <xref ref-type="bibr" rid="pgen.1002101-Wang2">[18]</xref> and normalize the score via a permutation-based procedure. First, we permute the phenotype labels and in each permutation recalculate <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e165.jpg" /></inline-formula> as described above, but using the permuted case and control labels. The permuted labels are used both to select the most informative SNP per gene and to compute <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e166.jpg" /></inline-formula>, <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e167.jpg" /></inline-formula>, and <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e168.jpg" /></inline-formula> in Eqns. 1&#8211;3). This yields a distribution of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e169.jpg" /></inline-formula> under the null hypothesis that the magnitude of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e170.jpg" /></inline-formula> is independent of the true case/control classifications. We then normalize the true <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e171.jpg" /></inline-formula> by the distribution from the permutation procedure, yielding a Distinction Score <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e172.jpg" /></inline-formula> for pathway <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e173.jpg" /></inline-formula> that effectively adjusts for different sizes of genes and preserves correlations of SNPs in the same gene:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002101.e174.jpg" /><label>(4)</label></disp-formula>where <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e175.jpg" /></inline-formula> are the set of <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e176.jpg" /></inline-formula> obtained for pathway <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e177.jpg" /></inline-formula> across the permutations. (In practice, 100 permutations are used.) Because the permuted labels are used in the SNP selection, this normalization adjusts for the bias introduced by the fact that large genes have more opportunity to contain significant SNPs by chance. The Pathway Distinction Score <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e178.jpg" /></inline-formula> thus provides a model-free, gene-size adjusted metric for quantifying the degree to which cases are &#8220;closer&#8221; to other cases (higher <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e179.jpg" /></inline-formula>) than controls.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">To test whether <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e180.jpg" /></inline-formula> is significant, we note that larger pathways may yield high <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e181.jpg" /></inline-formula> values simply due to the fact that they sample the case anc control differences more thoroughly. Indeed, the question of significance that we wish to address is not simply whether a pathway permits the distinction of cases and controls, but <italic>whether it does so better than a random collection of as many SNPs</italic>, wherein the SNPs are still selected to be the most informative by gene. To account for the fact that the pathways are of differing sizes, significance of the Distinction Score for a given pathway is assessed through resampling by choosing, at random, the same number of SNPs that are present in that pathway (<inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e182.jpg" /></inline-formula>) from the total set of most-informative-SNP-per-gene and recomputing <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e183.jpg" /></inline-formula> for the random pathway. The <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e184.jpg" /></inline-formula> value is obtained by counting the fraction of random <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e185.jpg" /></inline-formula>-SNP sets which give a larger <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e186.jpg" /></inline-formula> than the true pathway SNPs in <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e187.jpg" /></inline-formula> resamplings. In this way, we are able to detect pathways that yield large differences of case and control <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e188.jpg" /></inline-formula> distributions due to their particular SNPs, rather than simply being the result of choosing many SNPs. The <inline-formula><inline-graphic mimetype="image" ns0:href="pgen.1002101.e189.jpg" /></inline-formula> value obtained addresses the question of whether the pathway under consideration permits greater separation of cases and controls than would a random collection of most-informative-SNP-per-gene, i.e., whether there exists a more extreme aggregated effect in that pathway than expected by chance.</p>