<p>The subjects participating in the GWAS consisted of a total of 16,388 self-identified African-American individuals from 7 population-based cohorts (ARIC, CARDIA, JHS, WHI, HANDLS, Health ABC, and GeneSTAR) that belong to the Continental Origins and Genetic Epidemiology Network (COGENT). Detailed descriptions of each participating COGENT cohort, their quality control practices and study-level analyses are provided in the <xref ref-type="supplementary-material" rid="pgen.1002108.s017">Text S1</xref>. Clinical information of the subjects was collected by self-report and clinical examination. All participants provided written informed consent as approved by local Human Subjects Committees. We excluded study participants on the basis of pregnancy, cancer, or AIDS diagnosis at the time of blood count measurement.</p><p>Certified staff obtained fasting blood samples at the baseline clinic visit. Samples for complete blood count (CBC) analysis were obtained by venipuncture and collected into tubes containing ethylenediaminetetraacetic acid (EDTA). Total circulating WBC count and cell subtype counts were performed at local clinical laboratories using automated hematology cell counters and standardized quality assurance procedures <xref ref-type="bibr" rid="pgen.1002108-Nalls1">[4]</xref>, <xref ref-type="bibr" rid="pgen.1002108-Lo1">[6]</xref>, <xref ref-type="bibr" rid="pgen.1002108-Shimakawa1">[49]</xref>&#8211;<xref ref-type="bibr" rid="pgen.1002108-Margolis1">[51]</xref>. Total WBC count was reported in millions of cells per ml, and was recorded in all 16,388 study participants. Information on WBC subtype was available only in a subset of 7,477 (45.6%) participants from ARIC, CARDIA, JHS, HANDLS, GeneSTAR, and Health ABC. WBC differentials were performed by clinically certified hematology laboratories. The absolute numbers of each type of WBC were calculated by multiplying the proportion of the WBC count comprised by each cell type by the total WBC measure. To evaluate normality of the phenotypes for subsequent regression analyses, we performed Box-Cox likelihood ratio tests on raw WBC phenotypes. On this basis, all WBC traits were natural log transformed to normalize the distributions of the phenotypic data.</p><p>Genome-wide genotyping was performed within each COGENT cohort using methods described under <xref ref-type="supplementary-material" rid="pgen.1002108.s017">Text S1</xref>. DNA samples with a genome-wide genotyping success rate &lt;90%, duplicate discordance or sex mismatch, genetic ancestry outliers (as determined by cluster analysis performed using principal component analysis or multi-dimensional scaling), SNPs with genotyping success rate &lt;95%, monomorphic SNPs, SNPs with minor allele frequency (MAF) &lt;1%, and SNPs that map to several genomic locations were removed from the analyses. Significantly associated SNPs were examined for strong deviations from Hardy&#8211;Weinberg equilibrium and/or raw genotype data examined for abnormal clustering. Participants and SNPs passing basic quality control were imputed to &gt;2.2 million SNPs based on HapMap2 haplotype data using a 1&#8758;1 mixture of Europeans (CEU) and Africans (YRI) as the reference panel. Details of the genotype imputation procedure are described further under <xref ref-type="supplementary-material" rid="pgen.1002108.s017">Text S1</xref>. Prior to discovery meta-analyses, SNPs were excluded if imputation quality metrics (equivalent to the squared correlation between proximal imputed and genotyped SNPs) were less than 0.50.</p><p>For all cohorts, genome-wide association (GWA) analysis for quantitative WBC traits was performed using linear regression adjusted for covariates, implemented in either PLINK v1.07 <xref ref-type="bibr" rid="pgen.1002108-Purcell1">[52]</xref> or MACH2QTL v1.08. Allelic dosage at each SNP was used as the independent variable, adjusted for primary covariates of age, age-squared, sex, and clinic site (if applicable). To adjust for population stratification and global admixture, the principal components were also incorporated as covariates in the regression models (see <xref ref-type="supplementary-material" rid="pgen.1002108.s017">Text S1</xref>). For GeneSTAR, family structure was accounted for in the association tests using linear mixed effects (LME) models implemented in R <xref ref-type="bibr" rid="pgen.1002108-Chen1">[53]</xref>. Although the JHS has a small number of related individuals, extensive analyses showed that results were concordant using linear regression or LME, after genomic control. Therefore, results are presented for JHS using linear regression. For imputed genotypes, we used dosage information (<italic>i.e.</italic> a value between 0.0&#8211;2.0 calculated using the probability of each of the three possible genotypes) in the regression model implemented in PLINK and MACH2QTL (for cohorts with unrelated individuals) or the Maximum Likelihood Estimation (MLE) routines (for GeneSTAR).</p><p>For each WBC phenotype, meta-analyses were conducted using inverse-variance weighted fixed-effects models to combine beta coefficients and standard errors from study level regression results for each SNP to derive a combined p-value and effect estimates. Study level results were corrected for genomic inflation factors (&#955;) by incorporating study specific &#955; estimates into the scaling of the standard errors (SE) of the regression coefficients by multiplying the SE by the squate-root of the genomic inflation factor. The inflation factors for all completed analyses are presented in <xref ref-type="supplementary-material" rid="pgen.1002108.s006">Table S1</xref>. Meta-analyses were implemented in the software METAL <xref ref-type="bibr" rid="pgen.1002108-Willer1">[54]</xref> and were performed independently by another analyst to confirm results. Between-study heterogeneity of results was assessed by using Cochran's <italic>Q</italic> statistic and the <italic>I</italic>
<sup>2</sup> inconsistency metric. For each genome-wide significant or replicated locus, cohort specific-results and overall WBC effect estimates and confidence intervals are summarized using forest plots (<xref ref-type="supplementary-material" rid="pgen.1002108.s003">Figure S3</xref> and <xref ref-type="supplementary-material" rid="pgen.1002108.s004">S4</xref>). The mean and standard deviation WBC count for each genotype class is provided in <xref ref-type="supplementary-material" rid="pgen.1002108.s013">Table S8</xref>.</p><p>To maintain an overall type 1 error rate of 5%, a threshold of &#945;&#8202;=&#8202;2.5&#215;10<sup>&#8722;8</sup> was used to declare genome-wide statistical significance. This threshold has been suggested for African ancestry populations based on estimates of &#8764;2 million independent common variant tests in African genomes <xref ref-type="bibr" rid="pgen.1002108-Peer1">[55]</xref>.</p><p>Given the nonlinear nature of the original phenotype, we performed a sensitivity analysis of whether our results are robust to the assumption of an additive genetic model. We repeated the GWA analysis for the WHI, ARIC, CARDIA, JHS cohorts, the four largest African American cohorts (n&#8202;=&#8202;13,694) using a 2 degree of freedom genotypic model as well as a dominance deviation test, and meta-analyzed the results using METAL.</p><p>To assess in the COGENT African-Americans WBC trait-associated loci previously reported in Europeans or Japanese, we evaluated the African-American meta-analysis results for each index SNP in the regions reported, including consistency of direction of effect, and assessed statistical significance by a simple Bonferroni adjustment based on the total number of SNPs assessed using a 2-sided hypothesis test. In addition, we performed a more exploratory assessment of all SNPs within a 500 kb window that were correlated in African Americans with the European or Japanese index SNP in HapMap CEU or CHB+JPT (<italic>r<sup>2</sup></italic>&#8805;0.5). We adjusted these exploratory regional analyses for multiple testing based on the effective number of SNPs, taking into account pairwise linkage disequilibrium patterns.</p><p>To further assess the potential existence of multiple, independent variants influencing a trait at the same locus (allelic heterogeneity), regression analyses were repeated, conditional on the most strongly associated (index) SNP in that region. Each study repeated the primary GWA analysis, additionally adjusting for the lead SNP in each region under the appropriate regression models. The cohort-specific results were then meta-analyzed in the same way as for the primary GWA study using METAL.</p><p>Replication of novel association findings was performed using GWA data in 3 other ethnic populations: 3,551 Hispanic American women from WHI, 14,767 Japanese from RIKEN, and 19,509 European Americans from CHARGE. Further details of each study population are provided under <xref ref-type="supplementary-material" rid="pgen.1002108.s017">Text S1</xref>. Both genotyped and imputed SNP data were available in the European and Japanese samples, while only genotyped SNP data were available in the Hispanic Americans. To further localize the causal variant responsible for the <italic>CXCL2</italic>-WBC association, we extended the association analysis to include all genotyped and imputed SNPs within a 500 kb region centered at rs9131, the SNP most strongly associated with WBC count in African Americans. We then performed a trans-population meta-analysis of each SNP in this region by combining test statistics from the African American (COGENT), European (CHARGE), and Japanese (RIKEN) association analyses using Fisher's method <xref ref-type="bibr" rid="pgen.1002108-Fisher1">[56]</xref>, which may have some advantages over the standard meta-analytic approach in this setting <xref ref-type="bibr" rid="pgen.1002108-Teo1">[37]</xref>. Nonetheless, we also performed a standard inverse variance-weighted meta-analysis using either fixed or random effects <xref ref-type="bibr" rid="pgen.1002108-DerSimonian1">[57]</xref>, and obtained results similar to Fisher's method.</p><p>For between-study GWA platform consistency, we estimated locus-specific ancestry using Affymetrix 6.0 genotyped SNP data from the 4 largest African-American cohorts (WHI, ARIC, JHS, CARDIA), which constitute &#8764;85% of our total COGENT African American sample. For each African American, locus-specific ancestry (probabilities of whether an individual has 0, 1, or 2 alleles of African ancestry at each locus) was estimated using a Hidden Markov Model and local haplotype structure to detect transitions in ancestry along the genome <xref ref-type="bibr" rid="pgen.1002108-Price1">[58]</xref>, <xref ref-type="bibr" rid="pgen.1002108-Tang1">[59]</xref>. Phased haplotype data from the HapMap CEU and YRI individuals were used as reference panels. To assess the impact of local ancestry on any genome-wide SNP associations, each of the 4 cohorts repeated each SNP genotype-WBC phenotype linear regression model, adjusting for local ancestry proportion as a covariate. In addition, we stratified the SNP genotype-WBC phenotype association test on the number of estimated local European chromosomes (&#8805;1 versus &lt;1) to compare whether variants in genome-wide significant regions have the same versus different effect on African and European ancestral population backgrounds. The cohort-specific results of these analyses were combined using METAL.</p><p>In the GeneSTAR family study, variance components models in the ASSOC subroutine of S.A.G.E. <xref ref-type="bibr" rid="pgen.1002108-SAGE1">[60]</xref> were used to derive maximum likelihood estimates of polygenic (narrow-sense) heritability (<italic>&#963;</italic>
<sup>2</sup>
<sub>g</sub>) using natural-log transformed unadjusted or covariate-adjusted phenotype data. The statistical significance of the heritability estimate was obtained using a likelihood ratio test. In each of the 7 COGENT African American cohorts, the fraction of variance explained was estimated using the formula: 2<italic>pq</italic>&#215;<italic>&#946;</italic>
<sup>2</sup>, where <italic>p</italic> is the frequency of the effect allele of the SNP, <italic>q</italic>&#8202;=&#8202;1&#8722;<italic>p</italic>, and <italic>&#946;</italic> is the additive effect in each population estimated by standardizing WBC to have standard deviation 1.</p>