<p>Serum creatinine was measured as described in <xref ref-type="supplementary-material" rid="pgen.1002264.s014">Text S1</xref>. Serum creatinine was calibrated to NHANES in all studies (including replication cohorts) to account for between-laboratory variation as previously described <xref ref-type="bibr" rid="pgen.1002264-Kottgen1">[3]</xref>, <xref ref-type="bibr" rid="pgen.1002264-Coresh2">[36]</xref>, <xref ref-type="bibr" rid="pgen.1002264-Fox3">[37]</xref>. Glomerular filtration rate was calculated based on serum creatinine (eGFR) with the Modification of Diet in Renal Disease (MDRD) equation <xref ref-type="bibr" rid="pgen.1002264-Klahr1">[38]</xref>. We defined chronic kidney disease (CKD) as eGFR &lt;60 ml/min/1.73 m<sup>2</sup> in accordance with the National Kidney Foundation guidelines; CKD was based on a single serum creatinine measurement as described in <xref ref-type="supplementary-material" rid="pgen.1002264.s014">Text S1</xref>. Urinary albumin to creatinine ratio (UACR, mg/g) was computed as described in <xref ref-type="supplementary-material" rid="pgen.1002264.s014">Text S1</xref>; microalbuminuria was defined as UACR &gt;17 mg/g [men] and &gt;25 mg/g [women].</p><p>We defined diabetes as fasting glucose &#8805;126 mg/dl, self-report, or pharmacologic treatment. Similarly, hypertension was defined as systolic blood pressure &#8805;140 mm Hg, diastolic blood pressure &#8805;90 mm Hg, or pharmacologic treatment.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">For the present study, the CARe consortium genotyped the IBC SNP chip <xref ref-type="bibr" rid="pgen.1002264-Keating1">[15]</xref> in 23767 European Americans and 8110 African Americans, as well as the Affymetrix 6.0 chip in 7382 African Americans. The IBC array contains nearly 50,000 SNPs across 2,000 loci. SNPs were selected using a tagging approach among populations represented in HapMap and the SeattleSNPs project. The array was designed to focus on candidate loci related to cardiovascular disease and its risk factors. More details can be found in the design paper <xref ref-type="bibr" rid="pgen.1002264-Keating1">[15]</xref>. <xref ref-type="supplementary-material" rid="pgen.1002264.s008">Table S1</xref> details the genotyping that was conducted. For the CARe study cohorts, quality control and imputation were conducted centrally using MACH 1.0.16 (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/MaCH/">http://www.sph.umich.edu/csg/abecasis/MaCH/</ext-link>). Imputation results were filtered using thresholds RSQ_HAT value of 0.3 and minor allele frequency 0.01. Fractional counts between 0 and 2 were coded for the imputed genotypes in order to estimate the number of copies of a pre-specified allele. For European samples, the CEU population from HapMap 2 (2.54 million SNPs) was used as the reference panel.</p><p>For African American samples, a 1&#8758;1 combined HapMap 2 CEU+YRI reference panel was used. This panel includes SNPs that were present in both populations, as well as SNPs segregating in one panel and monomorphic and nonmissing in the other (2.74 million altogether). Since the African American samples were genotyped for both the Affymetrix 6.0 and IBC arrays, we were able to analyze imputation performance at non-genotyped SNPs. The use of the CEU+YRI panel resulted in an allelic concordance rate of &#8764;95.6%, calculated as 1 &#8211; 1/2*|imputed_dosage &#8211; chip_dosage| for imputation on the Affymetrix chip. This is similar to rates obtained from African ancestry participants imputed using HapMap 2 YRI individuals <xref ref-type="bibr" rid="pgen.1002264-Huang1">[39]</xref>.</p><p>Trait creation details are described above (Renal Function Indices). Performed centrally but within each individual study, genome-wide association analyses and IBC chip analyses of natural log-transformed eGFR, UACR, CKD, and MA were conducted using linear and logistic regression with an additive genetic model. We adjusted for age, sex and study site (when applicable) and the first 10 principal components; relatedness was accounted for when necessary using linear mixed effect (LME) models for eGFR and UACR and logistic regression via generalized estimating equations (GEE) for CKD and MA. Additional details regarding the discovery cohorts are in <xref ref-type="supplementary-material" rid="pgen.1002264.s014">Text S1</xref>.</p><p>Principal components were generated using EIGENSTRAT <xref ref-type="bibr" rid="pgen.1002264-Price1">[40]</xref> within each study using the CARe African ancestry Affy6.0 genotype data. Two reference populations were included in the principal component analysis of African Americans: 1,178 European Americans from a multiple sclerosis GWA study (from Dr. Phil de Jager and colleagues), and 756 Nigerians from the Yoruba region from a hypertension GWA (provided by Dr. Richard Cooper and colleagues). Importantly, these two underwent extensively quality control procedures to remove population outliers using PCA. Ten principal components were generated for each study and used to adjust for population substructure.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We performed fixed-effect meta-analyses of the IBC chip and genome-wide association data using the inverse-variance weighted approach in METAL (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/Metal/index.html">http://www.sph.umich.edu/csg/abecasis/Metal/index.html</ext-link>). Genomic control correction was applied after calculating the inflation factor lambda (&#955;) within each individual study and after the genome-wide association meta-analysis was performed.</p><p>The standard threshold of p&lt;5&#215;10<sup>&#8722;8</sup> for genome-wide significance in the genome-wide association and p&lt;2.0&#215;10<sup>&#8722;6</sup> in the IBC chip analyses was used. The rationale for the p-value threshold used for the IBC chip is based on an empiric test of the number of independent loci (&#8764;25,000) that appear on the IBC array <xref ref-type="bibr" rid="pgen.1002264-Lo1">[17]</xref>. We selected independent SNPs (pairwise r<sup>2</sup>&lt;0.2) at each locus for replication.</p><p>The R software (v2.9.0) was used for data management, statistical analyses and graphing.</p><p>We developed a set of criteria to validate the lead SNPs and interrogate regions around each of the loci that were previously reported among European ancestry (EA) participants <xref ref-type="bibr" rid="pgen.1002264-Kottgen1">[3]</xref>, <xref ref-type="bibr" rid="pgen.1002264-Kottgen2">[4]</xref> in our African American (AA) CARe samples. For each lead SNP in EA, we looked-up the respective association result with eGFR in AA. To accommodate the difference of LD structure and possible allelic heterogeneity across different ethnicities, we then interrogated the 250 kb flanking region around each lead SNP to determine whether there exist other SNPs with stronger associations with the outcome. We used the following criteria to identify the top AA SNP: 1) the SNP with the smallest association p-value within the region; 2) MAF &gt;0.03; 3) location of the AA lead SNP within the same recombination block of the lead EA SNP, where the recombination block was defined as a 20% recombination rate. The statistical significance of each identified SNP was evaluated using a region-specific Bonferroni correction. We determined the number of independent SNPs based on the variance inflation factor (VIF), which was calculated recursively within a sliding window with size 50 SNPs and pairwise r<sup>2</sup> value of 0.2 using PLINK.</p><p>Finally, each identified top SNP in AA was sent for replication in additional independent AA samples.</p><p>Similar to the interrogation in AA for the EA lead loci, we also interrogated the newly identified loci from the CARe GWAS and the IBC chip in EA participants of the CKDGen consortium.</p><p>Replication analyses were performed using imputed <italic>in silico</italic> genome-wide association data; replication studies conducted the same association analyses as the Stage 1 phase. Details regarding the replication cohorts can be found in <xref ref-type="supplementary-material" rid="pgen.1002264.s014">Text S1</xref>.</p><p>Replication was performed as follows: meta-analysis was conducted in the Stage 2 studies only, and then in the Stage 1 + Stage 2 studies combined. Replication was defined as a direction-consistent Stage 2 beta coefficient; replication p-values are thus represented as one-sided tests. SNPs were declared to replicate when the p-value in the Stage 1 + Stage 2 studies combined was smaller than the p-value in the Stage 1 alone.</p><p>The correlation between ln(UACR) and ln(eGFR) can range from non-significant to as high as 0.237 (p&lt;0.001) in our study; therefore, we examined the cross-trait associations across albuminuria and eGFR phenotypes.</p><p>Zebrafish were maintained in accordance with established procedures and IACUC approval. At the one-cell stage, zebrafish embryos were injected with varying doses of morpholino antisense oligonucleotides (MO, GeneTools, Philomath OR). MO sequences are shown in <xref ref-type="supplementary-material" rid="pgen.1002264.s013">Table S6</xref>. For <italic>in vivo</italic> observations, edema development was documented at 5-days post-fertilization.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">
<italic>In situ</italic> hybridization was performed according to established procedures (<ext-link ext-link-type="uri" ns0:href="http://zfin.org/ZFIN/Methods/ThisseProtocol.html">http://zfin.org/ZFIN/Methods/ThisseProtocol.html</ext-link>).To visualize different regions of the kidney, we used <italic>pax2a</italic> (global kidney marker) <xref ref-type="bibr" rid="pgen.1002264-Drummond2">[41]</xref>, <italic>wt1a</italic> (podocyte marker) <xref ref-type="bibr" rid="pgen.1002264-Drummond2">[41]</xref>, <italic>nephrin</italic> (podocyte marker) <xref ref-type="bibr" rid="pgen.1002264-KramerZucker1">[31]</xref>, <italic>slc20a1a</italic> (proximal tubule) <xref ref-type="bibr" rid="pgen.1002264-Wingert1">[42]</xref>, and <italic>slc12a3</italic> (distal tubule marker) <xref ref-type="bibr" rid="pgen.1002264-Wingert1">[42]</xref>. The morphology of the expression pattern was independently scored by two investigators. Electron microscopy was performed as previously described <xref ref-type="bibr" rid="pgen.1002264-Drummond2">[41]</xref>.</p><p>Dextran clearance was assessed as described previously <xref ref-type="bibr" rid="pgen.1002264-Hentschel1">[20]</xref>, 48 hours after morpholino injection, embryos were manually dechorionated, anesthetized in a 1&#8758;20 dilution of 4 mg/ml Tricaine in egg water and positioned on their back in a 1% agarose injection mold. An equal volume of tetramethylrhodamine dextran (10,000 MW; Invitrogen) was injected into the cardiac sinus venosus of each embryo, after which embryos were returned to fresh egg water. Embryos were imaged by fluorescence microscopy 6 hours post-injection (54 hpf) to demonstrate equal loading, then subsequently imaged at 72 and 96 hpf to evaluate dextran clearance. Confocal images were obtained from agarose embedded embryos using a Zeiss LSM500 microscope.</p>