<p>All yeast strains and plasmids used in this study are listed in <xref ref-type="table" rid="pgen-1002303-t001">Table 1</xref> and <xref ref-type="table" rid="pgen-1002303-t002">Table 2</xref>, respectively. Yeast growth media, and protocols are as described previously <xref ref-type="bibr" rid="pgen.1002303-Au1">[35]</xref>. Plasmid pMB1193 (<italic>2 &#181;m LEU2 GAL1/10-SCM3</italic>) was derived by inserting a PCR-amplified <italic>SCM3</italic> fragment between the <italic>BamHI</italic> and <italic>XhoI</italic> sites of pRS425 <italic>GAL1</italic>. The NES domain of <italic>SCM3</italic> was deleted from pMB1306 using QuickChange Site Directed Mutagenesis Kit (Stratagene) resulting in pMB1393 (<italic>2 &#181;m URA3 GAL1/10-scm3nes&#916;-HA</italic>). pMB1455 (<italic>2 &#181;m LEU2 GAL1/10-scm3hr&#916;-HA</italic>) was constructed by cloning the PCR-amplified <italic>SCM3hr&#916;-HA</italic> fragment from pRB923 between <italic>SpeI</italic> and <italic>XhoI</italic> sites of pRS425 <italic>GAL1</italic>, whereas pMB1490 (<italic>2 &#181;m URA3 GAL1/10-HJURP</italic>) was derived from subcloning the <italic>EcoRI -XhoI</italic> fragment of HJURP (clone id 2820741, Open Biosystem), into pRS426 <italic>GAL1</italic>. Plasmids pMB1520 (<italic>2 &#181;m LEU2 GAL1/10-scm3-&#916;103HA</italic>) and pMB1521 (<italic>2 &#181;m LEU2 GAL1/10-scm3-&#916;140HA</italic>) were constructed by cloning the PCR-amplified <italic>SCM3-&#916;103HA</italic>, and <italic>SCM3-&#916;140HA</italic> fragments from pMB1306 between <italic>SpeI</italic> and <italic>XhoI</italic> sites of pRS425 <italic>GAL1</italic>. Strains were grown in YPD (1% yeast extract, 2% Bacto-peptone, 2% glucose) or in synthetic medium containing either 2% glucose or 2% galactose and supplements to allow selection of plasmids being tested.</p><p>We used an assay developed previously <xref ref-type="bibr" rid="pgen.1002303-Spencer1">[55]</xref> to measure the loss of a non-essential reporter chromosome fragment (CF). Reporter strains were grown to logarithmic phase in synthetic media to maintain the CF and plasmids to be assayed. Cultures were then diluted and plated on synthetic medium to maintain plasmid selection with limiting adenine. The loss of non-essential CF leads to red sectors in an otherwise a white colony. Chromosome loss was measured by counting the percentage of colonies that were at least half red, which represents the loss of reporter chromosome during the first cell division. A minimum of 3000 colonies were assayed from three individual transformants for each strain.</p><p>Viability assays for Ctf3p and COMA complex strains containing <italic>GALSCM3HA</italic> (pMB1306) or vector (pRS426 <italic>GAL1</italic>) were carried out by plating equal numbers of cells from three independent transformants for each strain on synthetic media with either glucose (2%) or galactose (2%) and grown at 30&#176;C for 5&#8211;7 days. At least 2500 colonies were counted for each strain. Percent viability is expressed as the ratio of the number of colonies obtained on galactose media over the number of colonies obtained on glucose plates and normalized to the value of 100.</p><p>All ChIP experiments were carried out (three biological replicates) as described <xref ref-type="bibr" rid="pgen.1002303-Au1">[35]</xref> with minor modifications. Cultures were cross-linked with formaldehyde (1% final concentration) at room temperature for 15 min and excess formaldehyde was quenched with 125 mM glycine for 5 min. Cell pellets were collected by centrifugation and spheroplasts prepared using Zymolyase 100T, followed by sequential washes with PBS (150&#160;mM NaCl, 10&#160;mM sodium phosphate, pH 7.4), and FA buffer (50&#160;mM Na-Hepes pH 7.6, 1 mM EDTA, 1% Triton X-100, 150 mM NaCl, 0.1% Na-deoxycholate). Spheroplasts were resuspended in FA buffer with protease inhibitors (1&#160;mM PMSF, 1&#160;mM benzamidine, 10 &#181;g/ml leupeptin, 10 &#181;g/ml aprotinin) and sonicated on ice at setting 3, 100% duty cycle, for four 12&#160;sec bursts applied 2&#160;min apart to obtain an average fragment size of 400&#160;bp. The resulting soluble fraction was denoted as input and used for chromatin immunoprecipitation. Protein A magnetic beads (Invitrogen Inc.) coated with antibodies were used in immunoprecipitation performed at 4&#176;C for 12 hours. Antibodies used were anti-FLAG (A2220, Sigma), anti-HA (A2095, Sigma), anti-Myc (sc-789 Santa Cruz Inc), anti-histone H4 (clone 62-141-13, Millipore), anti-<italic>HJURP</italic>
<xref ref-type="bibr" rid="pgen.1002303-Foltz1">[26]</xref> or anti-GST (negative control). The immunoprecipitated nucleic acid-protein complexes were washed sequentially at room temperature for 5 min each wash in FA buffer (thrice), FA-HS buffer (50 mM Na-Hepes pH 7.6, 1 mM EDTA, 1% Triton X-100, 500 mM NaCl, 0.1% Na-deoxycholate) (twice), RIPA buffer (10 mM Tris-HCl pH 8.0, 1 mM EDTA, 250 mM LiCl, 0.5% Nonidet P40, 0.5% Na-deoxycholate) (twice), 1 x TE pH 8.0 (twice), and finally resuspended in elution buffer (25 mM Tris-HCl pH 7.6, 10 mM EDTA, 0.5% SDS). Immunoprecipitated DNA was recovered after cross-link reversal at 65&#176;C for 16 hours and RNase A/proteinase K treatment, purified by phenol/chloroform/isoamyl alcohol (25&#8758;24&#8758;1) extractions, and ethanol precipitated (&#8722;20&#176;C for 12 hours). Enriched DNA was recovered by centrifugation and dissolved in TE (10&#160;mM Tris, 1&#160;mM EDTA, pH 8.0).</p><p>Quantitative real time ChIP-PCR was performed using Fast SYBR Green Master Mix in 7500 Fast real Time PCR System (Applied Biosystem Inc.) following manufacturer's instructions. PCR conditions were: denaturation at 95&#176;C for 20 sec, followed by 40 cycles of 95&#176;C for 3 sec, 60&#176;C for 30 sec. The amplification readings were recorded and threshold cycle (Ct) was determined by 7500 Fast System Software v1.4.0 (Applied Biosystem Inc.). The enrichment values were calculated using the &#916;&#916;Ct method <xref ref-type="bibr" rid="pgen.1002303-Livak1">[56]</xref> and are presented as % input.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">ChIP-PCR was done using Hotstar-Taq polymerase Master Mix (Qiagen Inc.), with initial denaturation at 94&#176;C for 15 min, followed by 25 cycles of 94&#176;C for 1 min annealing at 54&#176;C for 1 min and extension at 68&#176;C for 10 sec. The linear amplification conditions were determined by various cycles of amplification using serial dilutions of the input DNA. PCR products were separated on 2% agarose gel, stained with ethidium bromide and photographed. The intensity of PCR amplicons was determined using the software ImageJ 1.43u (<ext-link ext-link-type="uri" ns0:href="http://rsb.info.nih.gov/ij">http://rsb.info.nih.gov/ij</ext-link>). Sequences of primers used in this study are listed in <xref ref-type="table" rid="pgen-1002303-t003">Table 3</xref>.</p><p>Immunoprecipitation experiments were performed as described previously <xref ref-type="bibr" rid="pgen.1002303-Camahort1">[18]</xref>. Protein samples for western blot analysis were prepared using the TCA procedure as described previously <xref ref-type="bibr" rid="pgen.1002303-Kastenmayer1">[57]</xref>, and protein concentration was measured using the Bio-Rad DC protein assay (Bio-Rad, CA). Equal amounts of protein for each sample were size separated on 4&#8211;12% gradient polyacrylamide gels and transferred to nitrocellulose membrane. Primary antibodies used were anti-HA (clone 12CA5, Roche), anti-Myc (Z-5, sc-789, Santa Cruz Inc), anti-histone H3 (ab1791-100, Abcam), anti-histone H4 (clone 62-141-13, Millipore), anti-Flag (A-6457, Molecular Probes) or anti-HJURP <xref ref-type="bibr" rid="pgen.1002303-Foltz1">[26]</xref>. Secondary antibodies used were HRP-conjugated sheep anti-mouse IgG (NA931V&#8211;Amersham) and HRP-conjugated sheep anti-rabbit IgG (NA934V&#8211;Amersham). Rabbit polyclonal antibodies against Tub2p were custom made by Covance, Inc.</p><p>For determining the <italic>CEN</italic> enrichment of Scm3p through the cell cycle, wild-type strain (RC154) with FLAG-tagged Scm3p expressed from its endogenous promoter was used. A wild type strain (JG595) with Myc-tagged Cse4p expressed from its endogenous promoter was used to determine the <italic>CEN</italic> enrichment of Cse4p through the cell cycle. Cells were grown in YPD to logarithmic (LOG) phase at 30&#176;C, treated with 3 &#181;M &#945;-factor (T-6901, Sigma, St. Louis) for 90 minutes to arrest cells in G1, washed, and released into pheromone-free YPD medium. Samples were taken at 15 min time points after release from G1 arrest and used for FACS, protein and ChIP analysis. Additional synchronizations were carried out to confirm the results: cells were arrested in G1 with &#945;-factor as described above and released into YPD medium containing 15 &#181;g/ml nocodazole (M1404, Sigma); cells were arrested in G2/M with nocodazole for 2 hours and released into YPD medium containing 3 &#181;M &#945;-factor (T-6901, Sigma).</p><p>To examine the <italic>CEN</italic> enrichment of overexpressed Scm3p (<italic>GALSCM3</italic>) through the cell cycle, strain RC100 with HA tagged Scm3p integrated at its endogenous locus (only copy in the genome) and expressed from <italic>GAL1</italic> promoter was used. Cells were grown in SC media with 2% galactose to logarithmic (LOG) phase at 30&#176;C, treated with 3 &#181;M &#945;-factor (T-6901, Sigma, St. Louis) for 90 minutes for G1 arrest, washed, and released into pheromone-free SC media with 2% galactose medium. Samples were taken at different time points after release from G1 arrest and were used in FACS and ChIP analysis.</p><p>FACS assays was performed to confirm the cell cycle arrest and release. Cells were fixed in 70% ethanol, washed in 0.2 M Tris buffer, treated with RNase A, Proteinase K, stained with propidium iodide, and analyzed using a Becton-Dickinson FACSort flow cytometer and Cell Quest software (BD Biosciences, Boston, MA). Cell cycle stages were determined by examining cell morphology and nuclear position in propidium iodide-stained cells under the Zeiss Axioskop 2 microscope (Carl Zeiss Inc., Thornwood, USA) as described previously <xref ref-type="bibr" rid="pgen.1002303-Calvert1">[42]</xref>. Cell cycle stages were defined as follows: G1, single cells with undivided nuclei; S-phase, cells showing a small bud with undivided nuclei; metaphase, large budded cells with nucleus at the neck; anaphase, large budded cells with elongated/separated nuclei; and telophase, large budded cells with nuclei separated between mother and daughter cells (see <xref ref-type="fig" rid="pgen-1002303-g002">Figure 2</xref>).</p><p>HeLa cells were plated 24 hours prior to transfection to poly-lysine coated 12mm square coverslips in a 6-well plate. Cells were incubated with DNA and Effectene transfection reagent (Qiagen) according the manufacturer instructions in Optimem (Invitrogen) for 8 hours and at which time the media was replaced with normal growth media (DMEM, 10% FBS, supplemented with Penicillin and Streptomycin). Beginning 24 hours after transfection, cells were incubated with 2mM thymidine for 20 hours, then released from thymidine by washing cells in PBS and incubating cells in normal growth media supplemented with 20mM deoxycytidine for 12 hours prior to fixation. Cells were fixed after in 4% formaldehyde and blocked in PBS with 2% FBS, 2% BSA and 0.1% TritonX-100. Antibody incubates were conducted for 1 hour at room temperature in blocking buffer. Centromeres were detected using anti-centromere autosera (Antibodies incorporated) and Cy5 conjugated goat-anti-human secondary antibodies (Jackson Labs). DNA was stained using 2 &#181;g/ml 4&#8242;,6-diamidino-2-phenylindole (DAPI) and cells were mounted in Prolong Antifade (Invitrogen). Z-stacked images were collected on a Deltavision microscope (Applied Precision Instruments) using a 60X objective. Images were deconvolved and are presented as maximum projections.</p>