<p>All participating studies conducted their research in accordance with their respective institutional scientific and ethical review boards. All human participants provided informed consent and all clinical investigation was conducted in accordance with the Declaration of Helsinki.</p><p>WBC counts were measured in 19,509 subjects in 7 discovery cohorts (The Rotterdam Study (RS), Framingham Heart Study(FHS), the NHLBI's Atherosclerosis Risk in Communities (ARIC) Study, the Age, Gene/Environment Susceptibility &#8211; Reykjavik Study (AGES) Study, Health Aging and Body Composition Study (Health ABC), the Baltimore Longitudinal Study of Aging (BLSA), and the Invecchiare in Chianti Study (InChianti)) and 11,823 subjects in 10 replication cohorts (the Sorbs, the Twins UK cohort (TwinsUK), Kooperative Gesundheitsforschung in der Region Augsburg (KORAF3 &amp; KORAF4) and UK Blood Services Donor Panel 1 (UKBS1) studies, three of the Italian Network on Genetic Isolates (INGI) studies (Carlantino, Val Borbera and Friuli Venezia Giulia), the Rotterdam Study II (RSII) and the Heart and Vascular Health Study (HVH)). In order to study genetic factors affecting variation of these traits within normal ranges, each study excluded all participants with any WBC measure (total WBC or one of the 5 cell subtypes) outside of +/&#8722;2 standard deviations from the mean value for that trait.</p><p>WBC phenotypes were derived from data provided by fluorescence activated cell sorting technologies commonly employed in clinical and epidemiological studies to interrogate common hematological elements found in peripheral blood. Total WBC count was reported in thousands of cells per ml, and sub-type specific cell counts were calculated by multiplying the proportion of the WBC count comprised by each cell type by the total WBC measure. Any subject with a trait value greater than 2 SD from the corresponding mean of that trait in each cohort or missing data for any assayed phenotype were excluded from all analyses. Shapiro-Wilk tests of normality were implemented in the smallest discovery cohorts as the data was available at the time of study design (the InChianti study and the Baltimore Longitudinal Study of Aging, BLSA) to evaluate normality of the phenotypes for analyses. Raw values, natural log transformed and square-root transformed values for each phenotype of interest in these two studies were compared with regard to deviations from normality based assessment of the Shapiro-Wilk test statistic in the two studies. Based upon these reviews, a uniform analysis plan was established for conducting each study-level analyses, analysis, using either log transformation (total WBC count, neutrophil count, and monocyte count) or square-root transformation (basophil count, eosinophil count and lymphocyte count) in order to normalize the distributions of the phenotypic data.</p><p>At the study level, GWAS analyses were conducted on unrelated participants (except for FHS) of confirmed European ancestry based on either multi-dimensional scaling or principal components analyses, concordance between genotypic and self-reported gender, and successfully genotyped at &gt;95% of attempted SNPs. SNPs were filtered based on criteria of minor allele frequency &gt;0.01 (MAF), missingness per SNP&lt;5% and Hardy-Weinberg equilibrium p-value&gt;1.00E-7 (HWE, used to exclude poorly clustered genotypes). Participants and SNPs passing basic quality control were imputed to &gt;2.4 million SNPs based on HapMapII haplotype data. All studies utilized multivariate linear regression to generate study level summary statistics for each phenotype, with allelic dosages at each SNP used as the independent variable and primary covariates of age at hematology assay, current smoking status and sex. Detailed descriptions of participating studies, their quality control practices and study-level analyses which may differ slightly from those described above are provided in <xref ref-type="supplementary-material" rid="pgen.1002113.s019">Text S1</xref>.</p><p>To conduct meta-analyses, all studies submitted summary statistics from the study-level linear regression analyses for each phenotype. Meta-analyses were conducted using inverse-variance weighted fixed-effects models to combine beta coefficients and standard errors from study level regression results for each SNP to derive a combined p-value. Prior to discovery meta-analyses, SNPs were excluded if imputation quality metrics (equivalent to the squared correlation between proximal imputed and genotyped SNPs) were less than 0.30. Study level results were also corrected for genomic inflation factors (&#955;<sub>GC</sub>) by incorporating study specific &#955;<sub>GC</sub> estimates into the scaling of the standard errors (SE) of the regression coefficients by multiplying the SE by the square-root of the genomic inflation factor (see <xref ref-type="supplementary-material" rid="pgen.1002113.s011">Table S1</xref> for study and phenotype specific genomic inflation factors) <xref ref-type="bibr" rid="pgen.1002113-deBakker1">[81]</xref>. Study specific genomic inflation factor estimates for all discovery cohorts were all &lt;1.05 except for 1.12 in the Health ABC analysis of basophil count and 1.09 in the analysis of total white blood cell count in AGES (<xref ref-type="supplementary-material" rid="pgen.1002113.s011">Table S1</xref>). No definitive cause of this inflation could be identified, and of particular interest, the genomic inflation factors for related traits in these two studies were within the normally accepted range. Meta-analyses were implemented using METAL and independently re-analyzed using R to confirm results <xref ref-type="bibr" rid="pgen.1002113-Willer1">[82]</xref>.</p><p>We chose <italic>a priori</italic> to carry over all results from discovery meta-analyses at p-values&lt;5.00E-08 to replication meta-analyses, excluding any SNPs with Cochran's Q test of heterogeneity p-values&lt;0.01 or missing in more than 2 studies. These conservative exclusion criteria caused the exclusion of 6 of 167 genome-wide significant SNPs from replication analyses, and these SNPs did not constitute any new loci of interest. The final number of SNPs for replication analyses was then reduced to 161 candidate SNPs across all phenotypes. For replication meta-analyses of individual SNPs, each phenotype was analyzed separately using similar inverse-variance weighted meta-analyses as in the discovery stage analyses, although no genomic control was used. P-values for significant associations in the replication stage were corrected for the number of SNPs tested per phenotype using the standard Bonferroni correction for multiple testing (total WBC count corrected for 63 SNPs, with a significance threshold of p-value&#8804;7.94E-4; neutrophil count corrected for 46 SNPs, with a significance threshold of p-value&#8804;1.09E-3; basophil count corrected for 1 SNP, with a significance threshold of p-value&#8804;0.05; lymphocyte count corrected for 14 SNPs, with a significance threshold of p-value&#8804;3.57E-3; and monocyte count corrected for 37 SNPs, with a significance threshold of p-value&#8804;1.35E-3). Of the 161 candidate SNPs included in the replication phase, 152 SNPs passed the trait-specific replication p-value thresholds. Ony one genome-wide significant locus failed to replicate, and this locus on chromosome 1q22 contained only one genome-wide significant SNP associated with monocyte count in the discovery phase.</p><p>Of the 152 successfully replicated associations, 109 SNPs were unique, since some SNPs were significant across multiple phenotypes. These replicated SNPs were analyzed in GRAIL to infer a possible biological connection between significant meta-analysis loci. GRAIL was used to mine textual data based on PubMed keywords to examine functional relatedness across phenotypes based on inferred biological interconectivity between genes proximal to meta-analysis results. SNP (rs) identifiers for these associated SNPs were input into the GRAIL webserver as a means of specifying genomic regions of interest in constructing query and seed regions to be analyzed. Genes for text mining of the functional datasource were identified using the LD structure of HapMap2 Release 22 CEU samples, gathering gene identifiers to search indexed abstracts from PubMed last curated on May 2010. Genes in regions of interest were clustered based on keyword similarity. These genes and clusters were then scored based on ranked similarity, adjusting for gene size, to generate p-values evaluating the strength of the functional interconnectivity of genes in the regions of interest. P-values for these functional clusters were then false-discovery rate adjusted (FDR) to correct for multiple testing, with the FDR adjusted p-value of 0.05 considered the threshold for significance.</p><p>For the eQTL analysis, 501 participants with complete genotyping data from the InChianti study were also successfully assayed on Illumina HT12v.3 genome-wide expression arrays using RNA isolated from whole blood. Quality control of the genome-wide expression data included the exclusion of probes with detection p-values&gt;0.01 with missing data for greater than 5% of participants. Samples must have been assayed with at least 95% of the filtered probe sets passing quality control in order to be included in analyses.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">5094 probes passed quality control and were subsequently cubic-spline normalized prior to analysis. In the investigation of possible cis-eQTL associations at regions of interest identified in the meta-analysis, all probes within 500 kb of successfully replicated SNPs from the meta-analysis were identified based on annotations from ReMOAT (<ext-link ext-link-type="uri" ns0:href="http://www.compbio.group.cam.ac.uk/Resources/Annotation/">http://www.compbio.group.cam.ac.uk/Resources/Annotation/</ext-link>) <xref ref-type="bibr" rid="pgen.1002113-BarbosaMorais1">[83]</xref>. Thus, we tested 741 possible cis-eQTLs. Multivariate linear regression was implemented using PLINKv.1.07 <xref ref-type="bibr" rid="pgen.1002113-Purcell1">[84]</xref>, testing the dosage of minor alleles as a predictor of gene expression level for each probe. These linear regression models were adjusted for hybridization batch, amplification batches, sex, smoking, study site and age at baseline of study. The p-values generated by each analysis was corrected for the number of tests, with a minimum threshold of significance at p-value&#8804;6.75E-05.</p>