<p>The study was approved by the Human Subject Institutional Review Boards or the ethnic committees of each institution. All subjects were enrolled after informed consent had been obtained.</p><p>To test the association of <italic>CFH</italic> and <italic>CFHRs</italic> with SLE, we used a large collection of samples from case-control subjects from multiple ethnic groups. These samples were from the collaborative Large Lupus Association Study 2 (LLAS2) and were contributed by participating institutions in the United States, Asia and Europe. According to genetic ancestry, subjects were grouped into four ethnic groups including European American (3,936 cases vs. 3,491 controls), African American (1,679 cases vs. 1,934 controls), Asian (1,265 cases vs. 1,260 controls) and Hispanic enriched for the Amerindian-European admixture (1,492 cases vs. 807 controls). Asians were comprised of Koreans (884 cases vs. 994 controls), Chinese (200 cases vs. 205 controls) and subjects from other East Asian countries such as Japan and Singapore (181 cases vs. 61 controls). African Americans included 275 Gullahs (152 cases vs. 123 controls), who are subjects with African Ancestry.</p><p>To test LD between <italic>CFHR3</italic>-1&#916; and SLE-associated SNPs, we used 275 unrelated European Americans (187 cases vs. 88 controls), 106 African Americans (88 unrelated subjects [58 cases vs. 30 controls] and 18 subjects from 6 SLE trios families), 282 unrelated Chinese (218 cases vs. 64 controls) and 196 Hispanics (157 unrelated subjects [91 cases vs. 66 controls] and 39 subjects from 13 SLE trios families). All of these subjects were enrolled from UCLA.</p><p>To enlarge the sample size of Asians for association test, we used 1,852 Chinese case-control subjects (787 vs. 1065) recruited from Shanghai Renji Hospital, Shanghai Jiao Tong University School of Medicine.</p><p>All SLE patients met the American College of Rheumatology (ACR) criteria for the classification of SLE <xref ref-type="bibr" rid="pgen.1002079-Hochberg1">[38]</xref>.</p><p>LLAS2 samples were processed at the Lupus Genetics Studies Unit of the Oklahoma Medical Research Foundation (OMRF). SNP genotyping was carried out on the Illumina iSelect platform. Subjects with individual genotyping call rate &lt;0.90 were removed because of low data quality. Subjects that were duplicated or first degree related were also removed. Both principal component analysis and global ancestry estimation based on 347 ancestry informative markers were used to detect population stratification and admixture, as described in another LLAS2 report <xref ref-type="bibr" rid="pgen.1002079-Lessard1">[39]</xref>. After removing genetic outliers, a final dataset of 15,864 unrelated subjects (8,372 cases vs. 7,492 controls) was obtained.</p><p>Taqman SNP genotyping assay (Applied Biosystems, California, USA) was used to genotype rs6677604 for subjects who were not recruited into LLAS2.</p><p>MLPA kit &#8220;SALSA MLPA KIT P236-A1 ARMD mix-1&#8221; was used to genotype the <italic>CFH-CFHRs</italic> region according to the manufacture's instruction (MRC-Holland, Amsterdam, The Netherlands). ABI 3730 Genetic Analyzer (Applied Biosystems) was used to run gel electrophoresis. Software Peak Scanner v1.0 (Applied Biosystems) was used to extract peaks generated in electrophoresis. Coffalyser v9.4 (MRC-Holland) was used to readout copy number of target region.</p><p>The HWE test threshold was set at <italic>P</italic>&gt;0.01 for controls and <italic>P</italic>&gt;0.0001 for cases. SNPs failing the HWE test were excluded from association test. SNPs showing genotyping missing rate&gt;5% or showing significantly different genotyping missing rate between cases and controls (missing rate&gt;2% and <italic>P</italic>
<sub>missing</sub>&lt;0.05) were also excluded from association test. In allelic association test (Pearson's &#967;2&#8211;test), the significance level was set at <italic>P</italic>&lt;0.05. Haploview 4.2 was used to estimate pairwise LD values between SNPs, define haplotypes blocks and calculate haplotypic association with SLE. Haplotype-based conditional association analysis was carried out by Plink v1.07. Mantel-Haenszel analysis was performed to generate the meta-analysis <italic>P</italic> value. CaTS was used to calculate statistical power.</p>