<p>The Institutional Review Board at the University of Southern California approved the study protocol.</p><p>Nine studies were genotyped as part of the GWAS of prostate cancer in African American men. Below is a brief description of each study.</p><p>The MEC includes 215,251 men and women aged 45&#8211;75 years at recruitment from Hawaii and California <xref ref-type="bibr" rid="pgen.1001387-Kolonel1">[32]</xref>. The cohort was assembled in 1993&#8211;1996 by mailing a self-administered, 26-page questionnaire to persons identified primarily through the driver's license files. Identification of incident cancer cases is by regular linkage with the Hawaii Tumor Registry and the Los Angeles County Cancer Surveillance Program; both NCI-funded Surveillance, Epidemiology, and End Results registries. From the cancer registries, information is obtained about stage and grade. Collection of biospecimens from incident prostate cases began in California in 1995 and in Hawaii in 1997 and a biorepository was established between 2001 and 2006 from 67,000 MEC participants. The participation rates for providing a blood sample have been greater than 60%. Through January 1, 2008 the African American case-control study in the MEC included 1,094 cases and 1,096 controls.</p><p>The SCCS is a prospective cohort of African and non-African Americans which during 2002&#8211;2009 enrolled approximately 86,000 residents aged 40&#8211;79 years across 12 southern states <xref ref-type="bibr" rid="pgen.1001387-Signorello1">[33]</xref>. Recruitment occurred mainly at community health centers, institutions providing basic health services primarily to the medically uninsured, so that the cohort includes many adults of lower income and educational status. Each study participant completed a detailed baseline questionnaire, and nearly 90% provided a biologic specimen (approximately 45% a blood sample and 45% buccal cells). Follow-up of the cohort is conducted by linkage to national mortality registers and to state cancer registries. Included in this study are 212 incident African American prostate cancer cases and a matched stratified random sample of 419 African American male cohort members without prostate cancer at the index date selected by incidence density sampling.</p><p>The Prostate, Lung, Colorectal, and Ovarian Cancer Screening Trial <xref ref-type="bibr" rid="pgen.1001387-Gohagan1">[34]</xref>, is a randomized, two-arm trial among men and women aged 55&#8211;74 years to determine if screening reduced the mortality from these cancers. Male participants randomized to the intervention arm underwent prostate specific antigen (PSA) screening at baseline and annually for 5 years and digital rectal examination at baseline and annually for 3 years. Sequential blood samples were collected from participants assigned to the screening arm; participation was 93% at the baseline blood draw (1993&#8211;2001). Buccal cell samples were collected from participants in the control arm of the trial; participation was about 85% for this component. Included in this study are 286 African American prostate cancer cases and 269 controls without a history of prostate cancer, matched on age at randomization and study year of the trial.</p><p>The CPS-II Nutrition Cohort includes over 86,000 men and 97,000 women from 21 US states who completed a mailed questionnaire in 1992 (aged 40&#8211;92 years at baseline) <xref ref-type="bibr" rid="pgen.1001387-Calle1">[35]</xref>. Starting in 1997, follow-up questionnaires were sent to surviving cohort members every other year to update exposure information and to ascertain occurrence of new cases of cancer; a &gt;90% response rate has been achieved for each follow-up questionnaire. From 1998&#8211;2001, blood samples were collected in a subgroup of 39,376 cohort members. To further supplement the DNA resources, during 2000&#8211;2001, buccal cell samples were collected by mail from an additional 70,000 cohort members. Incident cancers are verified through medical records, or through state cancer registries or death certificates when the medical record can not be obtained. Genomic DNA from 76 African American prostate cancer cases and 152 age-matched controls were included in stage 1 of the scan.</p><p>Participants in this study were identified from epidemiological prostate cancer studies conducted at the University of Texas M.D. Anderson Cancer Center in the Houston Metropolitan area since 1996. Cases were accrued from six institutions in the Houston Medical Center and were not restricted with respect to Gleason score, stage or PSA. Controls were identified via random-digit-dialing or among hospital visitors and they were frequency matched to cases on age and race. Lifestyle, demographic, and family history data were collected using a standardized questionnaire. These studies contributed 543 African American cases and 474 controls to this study <xref ref-type="bibr" rid="pgen.1001387-Strom1">[36]</xref>.</p><p>Cases in this study were patients 1) undergoing treatment for prostate cancer in the Department of Urology at Johns Hopkins Hospital from 1999 to 2007; 2) undergoing treatment at the Sidney Kimmel Comprehensive Cancer Center from 2003 to 2007; and 3) outside referrals as part of the Hereditary Prostate Cancer Study from 1990 to present. Blood was obtained from groups 2) and 3) while DNA from normal tissue was obtained from group 1). Data are available on age at diagnosis, race, pretreatment prostate-specific antigen (PSA) values, clinical pathology values, and family history. The control subjects were men undergoing disease screening and were not thought to have prostate cancer on the basis of a physical exam and a serum PSA value below 4 ng/ml. Screenings were performed at the Johns Hopkins Applied Physics Lab, at Bethlehem Steel in Baltimore, and at local African American churches in East Baltimore <xref ref-type="bibr" rid="pgen.1001387-Gudmundsson2">[7]</xref>. A total of 368 African American cases and 172 controls contributed to stage 1.</p><p>The LAAPC is a population-based case-control study of aggressive prostate among African Americans in Los Angeles County <xref ref-type="bibr" rid="pgen.1001387-Ingles1">[37]</xref>. Cases were identified through the Los Angeles County Cancer Surveillance Program rapid case ascertainment system and eligible cases included African American men diagnosed with a first primary prostate cancer between January 1, 1999 and December 31, 2003. Eligible cases also had either tumor extension outside the prostate, metastatic prostate cancer in sites other than prostate, or needle biopsy of the prostate with Gleason grade 8 or higher, or Gleason grade 7 and tumor in more than 2/3 of the biopsy cores. Controls were identified by a neighborhood walk algorithm and were men never diagnosed with prostate cancer, and were frequency matched to cases on age (&#177;5 years). For this study, genomic DNA was included for 296 cases and 140 controls. We also included an additional 163 African American controls from the MEC that were frequency matched to cases on age.</p><p>The African American component of this study population comprised 160 men: 75 cases diagnosed with more aggressive prostate cancer and 85 age-matched controls <xref ref-type="bibr" rid="pgen.1001387-Liu1">[38]</xref>. All subjects were recruited and frequency-matched on the major medical institutions in Cleveland, Ohio (i.e., the Cleveland Clinic, University Hospitals of Cleveland, and their affiliates) between 2001 and 2004. The cases were newly diagnosed with histologically confirmed disease: Gleason score 7; tumor stage T2c; or a prostate-specific antigen level &gt;10 ng/ml at diagnosis. Controls were men without a prostate cancer diagnosis who underwent standard annual medical examinations at the collaborating medical institutions.</p><p>Unrelated men self-described as African American were recruited for several case-control studies on genetic risk factors for prostate cancer between the years 2001 and 2005 from the Division of Urology at Howard University Hospital (HUH) in Washington, DC. Control subjects unrelated to the cases and matched for age (&#177;5 years) were also ascertained from the prostate cancer screening population of the Division of Urology at HUH <xref ref-type="bibr" rid="pgen.1001387-Robbins1">[24]</xref>. These studies included 292 cases and 359 controls.</p><p>The study population consists of participants from one of two population-based case-control studies among residents of King County, Washington <xref ref-type="bibr" rid="pgen.1001387-Agalliu1">[39]</xref>, <xref ref-type="bibr" rid="pgen.1001387-Stanford1">[40]</xref>. Incident Caucasian and African American cases with histologically confirmed prostate cancer were ascertained from the Seattle-Puget Sound SEER cancer registry during two time periods, 1993&#8211;1996 and 2002&#8211;2005. Age-matched (5-year age groups) controls were men without a self-reported history of being diagnosed with prostate cancer and were identified using one-step random digit telephone dialing. Controls were ascertained during the same time periods as the cases. A total of 145 incident African American cases and 81 African American controls were included from these studies.</p><p>The Henry Ford Health System (HFHS) recruited cases diagnosed with adenocarcinoma of the prostate of Caucasian or African American race, less than 75 years of age, and living in the metropolitan Detroit tri-county area <xref ref-type="bibr" rid="pgen.1001387-Rybicki1">[41]</xref>. Controls were randomly selected from the same HFHS population base from which cases were drawn. The control sample was frequency matched at a ratio of 3 enrolled cases to 1 control based on race and five-year age stratum. In total, 637 cases and 244 controls were enrolled between January 2002 and December 2004. Of study enrollees, DNA for 234 African Americans cases and 92 controls were included in stage 1 of the scan.</p><p>Genotyping of 7,123 samples from these studies (3,621 cases and 3,502 controls) was conducted using the Illumina Infinium 1 M-Duo bead array at the University of Southern California and the NCI Genotyping Core Facility (PLCO study). Following genotyping samples were removed based on the following exclusion criteria: 1) unknown replicates across studies (n&#8202;=&#8202;24, none within studies); 2) call rates &lt;95% (n&#8202;=&#8202;126); 3) samples with &gt;10% mean heterozygosity on the X chromosome and/or &lt;10% mean intensity on the Y chromosome - we inferred 3 samples to be XX and 6 to be XXY; 4) ancestry outliers (n&#8202;=&#8202;108, discussed below), and; 5) samples that were related (n&#8202;=&#8202;141, discussed below). To assess genotyping reproducibility we included 158 replicate samples; the average concordance rate was 99.99% (&#8805;99.3% for all pairs). Starting with 1,153,397 SNPs, we removed SNPs with &lt;95% call rate, MAFs&lt;1%, or &gt;1 QC mismatch based on sample replicates (n&#8202;=&#8202;105,411). The analysis included 1,047,986 SNPs among 3,425 cases and 3,290 controls.</p><p>We used PLINK to calculate the probabilities of sharing 0, 1, and 2 alleles (Z&#8202;=&#8202;Z0, Z1, Z2) across all possible pairs of samples to determine individuals who were likely to be related to others within and across studies. We identified 167 pairs of related subjects (MZ twin, parent-offspring pairs, full and half-sibling pairs), based on the values of their observed probability vector Z being within 1 SD of the expected values of Z for their respective relationship. The criterion for removal was such that individuals that were connected with a higher number of pairs were chosen for removal. In all other cases, one of the two members was randomly selected for removal. A total of 141 subjects were removed.</p><p>The EIGENSTRAT software was used to calculate eigenvectors that explained genetic differences in ancestry among samples in the study <xref ref-type="bibr" rid="pgen.1001387-Price1">[42]</xref>. The program included data from both HapMap Phase 3 populations and our study, so that comparisons to reference populations of known ethnicity could be made. An individual was subject to filtering from the analysis if his value along eigenvector 1 or 2 was outside of 4 SDs of the mean of each respective eigenvector. We identified 108 individuals who met this criterion. Eigenvector 1 was highly correlated (&#961;&#8202;=&#8202;0.997, p&lt;1&#215;10<sup>&#8722;16</sup>) with percentage of European ancestry, estimated in HAPMIX <xref ref-type="bibr" rid="pgen.1001387-Price2">[43]</xref>. Together the top 10 eigenvectors (used in the analysis) explain 21% of the global genetic variability among subjects.</p><p>At each locus and for each participant, local ancestry was defined as the estimated number of European chromosomes (continuous between 0&#8211;2) carried by the participant, estimated via the HAPMIX program <xref ref-type="bibr" rid="pgen.1001387-Price2">[43]</xref>. To summarize local ancestry at each region, for each individual we averaged across all local ancestry estimates that were within the start and end points of the region (<xref ref-type="supplementary-material" rid="pgen.1001387.s007">Table S5</xref>). We used this average value as an additional covariate in the risk analyses.</p><p>In order to generate a dataset suitable for fine-mapping, we carried out genome-wide imputation using the software MACH <xref ref-type="bibr" rid="pgen.1001387-Li1">[44]</xref>. Phased haplotype data from the founders of the CEU (CEPH) and YRI (Yoruba) HapMap Phase 2 samples were used to infer LD patterns in order to impute ungenotyped markers. The Rsq metric, defined as the observed variance divided by the expected variance, provides a measure of the quality of the imputation at any SNP, and was used as a threshold in determining which SNPs to filter from analysis (Rsq&lt;0.3). Of the 1,539,328 common SNPs (MAF&#8805;0.05) in the YRI population in HapMap Phase 2, we could impute 1,392,294 (90%) with Rsq&#8805;0.8. For all imputed SNPs presented in the Results and Tables reported herein, the average Rsq was 0.92 (estimated in MACH).</p><p>For each typed and imputed SNP, odds ratios (OR) and 95% confidence intervals (95% CI) were estimated using unconditional logistic regression adjusting for age at diagnosis (or age at the reference date for controls), study, the first 10 eigenvalues and local ancestry. For each SNP, we tested for allele dosage effects through a 1 d.f. Wald chi-square trend test.</p><p>We fine-mapped each risk locus in search of 1) a better marker of the index signal in African Americans, and; 2) a novel signal that is independent of the index signal. These analyses included SNPs (genotyped and imputed) spanning 250 kb upstream and 250 kb downstream of each index signal. If the index signal was contained within an LD block (based on the D<sup>&#8242;</sup> statistic) of &gt;250 kb, then the region was extended to include the entire region of LD. Stepwise regression was performed by region to select the most informative risk variants as discussed below, in models adjusted for age, study, global ancestry (the 1<sup>st</sup> eigenvector) and local ancestry. In the stepwise regression we preserved the original sample size by using the mean genotype of typed subjects in place of &#8220;no-calls&#8221; for SNPs with &lt;100% genotyping completion rate.</p><p>Within each known risk locus, it is expected that markers that are associated with risk in African Americans will be correlated with the index signal reported in Europeans. Thus, we identified and tested SNPs that are correlated (r<sup>2</sup>&gt;0.2) with the index signals in Europeans in HapMap (CEU population). Because these variants are not independent and there is a high prior probability that signals exist among such variants, we applied a lenient criteria for keeping them in the stepwise regression. The average number of tags to capture (r<sup>2</sup>&gt;0.8) these SNPs in each region was used as a correction factor, as they define the number of independent tests (p&lt;0.004). For all of the remaining markers that were not correlated with the index signal (in Europeans), we applied a more stringent &#945; level for defining statistical significance. In each risk region, we determined the number of tag SNPs needed to capture all common alleles (MAF&gt;0.05, with r<sup>2</sup>&gt;0.8) in the YRI population in Phase 2 HapMap using single and multi-marker tests. An &#945; of 0.05/the total number of tags was applied to assess statistical significance for any putative novel, independent signal in each region (p&lt;5.6&#215;10<sup>&#8722;6</sup>). For the correlated SNPs we had 80% power to detect an OR of 1.17 per copy for a 20% risk allele, whereas for the novel SNPs the detectable OR for such an allele increased to 1.26 per copy. A similar stepwise analysis was also performed at 8q24 (127.8&#8211;129.0 Mb) for SNPs with nominal p-values&lt;0.05, keeping SNPs if p&lt;0.001 in the multivariate model. This choice of p-value reflects a balance between the need to correct for multiple comparisons and the prior knowledge that this region harbors multiple independent risk alleles for prostate cancer. For SNPs in the 8q24 region we had 80% power to detect an OR of 1.19 per copy for a 20% risk allele. We tested heterogeneity of effect by study for all 76 SNPs presented in <xref ref-type="table" rid="pgen-1001387-t001">Table 1</xref> and <xref ref-type="table" rid="pgen-1001387-t002">Table 2</xref> and we observed 5 significant associations (p&lt;0.05, 3.6 expected) and only 1 at p&lt;0.01 (rs7000448 at 8q24, p&#8202;=&#8202;0.004).</p><p>We modeled the cumulative genetic risk of prostate cancer using the risk variants reported in previous GWAS (total&#8202;=&#8202;40). For regions outside of 8q24 with multiple correlated variants, we selected the SNP with the largest OR in African Americans. At 8q24 we used the seven variants reported in Al Olama et al. <xref ref-type="bibr" rid="pgen.1001387-AlOlama1">[1]</xref>. We compared the results to a model of the SNPs found to be significantly associated with risk in African Americans, which included the index signals if nominally associated with risk in African Americans (p&#8804;0.05) as well as SNPs identified from the stepwise procedures at all loci including 8q24 (total&#8202;=&#8202;27). More specifically, in each case we summed the number of risk alleles for each individual and estimated the odds ratio per allele for this aggregate unweighted allele count variable as an approximate risk score appropriate for unlinked variants with independent effects of approximately the same magnitude for each allele. For individuals missing genotypes for a given SNP, we assigned the average number of risk alleles (2&#215; risk allele frequency) to replace the missing value for that SNP. To address the independence assumption, we compared the betas for each SNP with the betas obtained when all SNPs were included in the same model. We found remarkable consistency in the betas, which supports their associations as being independent (<xref ref-type="supplementary-material" rid="pgen.1001387.s011">Table S9</xref>). We also stratified the risk score analysis by first-degree family history of prostate cancer. We tested for differences in the effect of the risk score by disease severity (advanced disease defined as Gleason 8&#8211;10 and/or non-localized stage vs non-advanced disease defined as Gleason&#8804;7 and localized stage).</p><p>We estimated crudely how much of the familial risk of prostate cancer is explained by the known risk alleles as summarized in the improved risk score. In this study, a first-degree family history of prostate cancer is associated with a relative risk of 1.55 (95% CI, 1.32&#8211;1.81). Making the simplifying assumption that all risk alleles are inherited independently then the correlation between the risk allele count for two first-degree relatives will be equal to 0.5 (i.e. will equal 1/2 the probability of sharing one allele IBD+the probability of sharing two alleles IBD). Making the further assumption that the number of risk alleles is distributed as approximately normal with mean&#8202;=&#8202;30.66 and standard deviation 3.07 alleles in the population (estimated among African American controls) and that in cases the mean is 32.13 alleles with roughly the same standard deviation (3.08), we can approximate the mean number of alleles in individuals of unknown prostate cancer status, but each of whom has a single first-degree relative (brother or father) with the disease as 30.66(1&#8211;0.5<sup>2</sup>)+32.13(0.5<sup>2</sup>)&#8202;=&#8202;31.03. Since this is just 0.37 more alleles than is expected in the control population overall we see that the relative odds of prostate cancer for a man with a brother or father with prostate cancer is only exp(log(1.17)*0.37)&#8202;=&#8202;1.06 higher than an unselected subject (i.e. one not selected on the basis of disease in a first-degree relative). Compared to the approximately 1.55-fold increase in relative risk, this risk score may only explain &#8764;11% [(1.06&#8722;1)/(1.55&#8722;1)&#215;100%] of risk in first-degree relatives of cases, which indicates that many more alleles are required to explain familial aggregation in the African American population.</p>