<p>Human Subject Committees of the participating institutions approved the study. The Discovery dataset was nested in the NeuroGenetics Research Consortium (NGRC) GWAS which successfully identified known PD genes as well as a novel association with <italic>HLA</italic>
<xref ref-type="bibr" rid="pgen.1002237-Hamza1">[5]</xref> which has been widely replicated <xref ref-type="bibr" rid="pgen.1002237-Nalls1">[10]</xref>, <xref ref-type="bibr" rid="pgen.1002237-SimonSanchez2">[19]</xref>. For the present GWAIS, Replication samples were provided by PEG <xref ref-type="bibr" rid="pgen.1002237-Ritz1">[20]</xref> (Parkinson, Environment, and Gene), PAGE <xref ref-type="bibr" rid="pgen.1002237-Chen2">[21]</xref> (Parkinson's, Genes, and Environment from the prospective NIH-AARP Diet and Health Study cohort), and HIHG <xref ref-type="bibr" rid="pgen.1002237-Edwards1">[9]</xref> (Hussman Institute for Human Genomics). Persons with PD had been diagnosed by neurologists using standard criteria <xref ref-type="bibr" rid="pgen.1002237-Hughes1">[22]</xref>, control subjects self-reported as not having PD. Cases and controls were all unrelated, non-Hispanic Caucasian, from United States. The NGRC cohort was clinic-based sequentially ascertained patients, PEG and PAGE were community-based incident cases, HIHG was clinic-based and self-referral cases. The numbers of cases/controls with genotype, coffee/caffeine and key clinical and demographic data were NGRC&#8202;=&#8202;1458/931, PEG&#8202;=&#8202;280/310, PAGE&#8202;=&#8202;525/1474, HIHG&#8202;=&#8202;209/133 (<xref ref-type="supplementary-material" rid="pgen.1002237.s005">Table S1</xref>).</p><p>NGRC, PEG and HIHG had collected lifetime caffeinated-coffee consumption data, measured as cups per day multiplied by the number of years of consumption (ccy) <xref ref-type="bibr" rid="pgen.1002237-Powers1">[12]</xref>, <xref ref-type="bibr" rid="pgen.1002237-Hancock1">[23]</xref>. PAGE had daily mg caffeine intake from all caffeine-containing drinks and foods for 12 months prior to enrollment (1995&#8211;1996) and only incident PD cases diagnosed after 1997 were included in the analysis <xref ref-type="bibr" rid="pgen.1002237-Thompson1">[24]</xref>. Despite the variation in data collection, results were consistent across studies, corroborating robustness of the interaction between coffee/caffeine and <italic>GRIN2A</italic>. We could not, and did not, attempt to distinguish the bioactive ingredient in caffeinated-coffee. Although caffeine has been shown to be neuroprotective, there may be other ingredients in caffeinated-coffee that may affect disease pathogenesis. To classify coffee/caffeine intake, each dataset was treated separately according to the measurements available. The median ccy or mg was determined for controls within each dataset (excluding those with zero intake) and used as the cut-off for heavy drinkers (&gt;median) vs. light drinkers (0 to &#8804;median). The median was 67.5 ccy for NGRC, 74.0 ccy for PEG, 70.0 ccy for HIHG, and 237.8 mg/day for PAGE. For coffee dose, quartiles were defined for each dataset using the full range from zero to maximum intake in controls. Results shown for NGRC, PEG and HIHG are based on lifetime caffeinated-coffee consumption. Truncating coffee use at age-at-onset or age-at-diagnosis in patients did not affect the results. To assess the effects of caffeinated tea and soda, we performed sensitivity analysis in NGRC dataset. Caffeinated soda and tea were commonly and equally consumed by heavy and light coffee drinkers (soda: 80% in both heavy and light drinkers; caffeinated tea: 66% in heavy coffee drinkers and 61% in light coffee drinkers). We repeated GWAIS and stratified GWAS with caffeinated soda and tea as covariates. We also explored association of caffeinated tea and soda with PD expecting an inverse association if caffeine were the bioactive ingredient in coffee.</p><p>The source of DNA was whole blood for NGRC and HIHG, saliva for PAGE, and whole blood (all PD and half of controls) or saliva (half of controls) for PEG. NGRC was genome-wide genotyped using Illumina HumanOmni1-Quad_v1-0_B array and achieved 99.92% call rate and 99.99% reproducibility. GWAS genotyping and statistical quality control (QC) have been published <xref ref-type="bibr" rid="pgen.1002237-Hamza1">[5]</xref>. 811,597 SNPs (excluding Y chromosome SNPs because they are not amenable to sex adjustment) passed GWAS QC and were included in GWAIS. Replication groups genotyped <italic>GRIN2A</italic>_rs4998386. Only one SNP was genotyped for replication; we have no other undisclosed replication results. PEG and HIHG used ABI TaqMan assay-by-design (C__28018721_20), PAGE used Sequenom and all achieved call rates of 96%&#8211;99%.</p><p>The first step was to test the hypothesis that the effect of coffee on PD risk is affected by a gene; ie, test statistical interaction between SNPs and coffee genome-wide. Theoretically, a test of SNP*coffee interaction would have been suitable; however, a pure test of interaction has low power; reportedly, it requires more than four times the sample size that GWAS would require to detect a main effect of similar size (reviewed in <xref ref-type="bibr" rid="pgen.1002237-Thomas1">[2]</xref>). We chose the joint test of SNP main effect and its interaction with coffee as proposed by Kraft et al <xref ref-type="bibr" rid="pgen.1002237-Kraft1">[15]</xref>. We call the test GWAIS for genome-wide <italic>association</italic> and <italic>interaction</italic> study. The main advantage of the joint test is that it does test for interaction and it has more power than pure interaction test when there is a modest SNP marginal effect. Next we performed stratified GWAS in heavy and light drinkers to gain insight to where the interaction signal was coming from and to formulate a hypothesis for replication. We then replicated the top signal and performed pooled analysis. Methods for meta-analysis of the joint test are available <xref ref-type="bibr" rid="pgen.1002237-Manning1">[25]</xref>, <xref ref-type="bibr" rid="pgen.1002237-Aschard1">[26]</xref>; however, since we had individual level data we pooled the datasets.</p><p>The genome-wide genotypes for NGRC had been cleaned previously for GWAS using standard rigorous measures <xref ref-type="bibr" rid="pgen.1002237-Hamza1">[5]</xref>. We had identified two significant principal components (PC1, PC2) marking Jewish/non-Jewish ancestry and European countries of origin <xref ref-type="bibr" rid="pgen.1002237-Hamza1">[5]</xref>. Sex was a significant variable, because PD affects more men than women and our data has a significant gender disparity (<xref ref-type="supplementary-material" rid="pgen.1002237.s005">Table S1</xref>). Controls were older than patients at age at onset, which was by design to minimize the chances that controls were too young to have developed the disease. Nevertheless, we controlled for age at enrolment both for patients and controls to avoid confounding by age-related factors. We examined coffee consumption and the most significant SNP for potential variation by disease related variables, recruitment sites, and ethnic and geographic origins of subjects (<xref ref-type="supplementary-material" rid="pgen.1002237.s006">Table S2</xref>). Smoking was a potential confounder because it is correlated with coffee use and is an independent inverse risk factor of PD. Thus we repeated all analyses with smoking included as a covariate in the model (<xref ref-type="supplementary-material" rid="pgen.1002237.s007">Table S3</xref> without smoking as covariate, <xref ref-type="supplementary-material" rid="pgen.1002237.s008">Table S4</xref> with smoking as covariate). We also repeated analyses with caffeinated tea and soda in the model (<xref ref-type="supplementary-material" rid="pgen.1002237.s009">Table S5</xref>). For details on how the data on tea, soda and smoking were collected in NGRC, see <xref ref-type="bibr" rid="pgen.1002237-Powers1">[12]</xref>.</p><p>811,597 SNP genotypes <xref ref-type="bibr" rid="pgen.1002237-Hamza1">[5]</xref> and lifetime caffeinated-coffee consumption data <xref ref-type="bibr" rid="pgen.1002237-Powers1">[12]</xref> from 1458 persons with PD and 931 controls from NGRC were analyzed. We tested the following models: [SNP+coffee+SNP*coffee+covariate vs. coffee+covariate] henceforth referred to as [SNP+SNP*coffee] joint test <xref ref-type="bibr" rid="pgen.1002237-Kraft1">[15]</xref>. Critically, the main effect of coffee on PD risk was present in both models being compared thus we controlled for coffee in the test. This model conducts a 2 degrees of freedom (df) joint test of SNP marginal effect and its interaction with coffee on PD risk <xref ref-type="bibr" rid="pgen.1002237-Kraft1">[15]</xref>. Sex, age, PC1 and PC2 were included as covariates. We used likelihood ratio test statistics as implemented in PLINK <xref ref-type="bibr" rid="pgen.1002237-Purcell1">[27]</xref>, and tested the Dominant, Additive and Recessive modes of inheritance. GWAIS was repeated once with the addition of smoking as a covariate, and again by addition of caffeinated tea and soda as covariates.</p><p>There were 512 cases and 387 controls in the heavy coffee drinking group and 946 cases and 544 controls in the light coffee drinking group. We tested association of 811,597 SNPs with PD in each group using standard GWAS with 1 df. We used PLINK <xref ref-type="bibr" rid="pgen.1002237-Purcell1">[27]</xref> and adjusted for age, sex, PC1 and PC2. Stratified GWAS were repeated with smoking, caffeinated soda and caffeinated tea added as covariates (<xref ref-type="supplementary-material" rid="pgen.1002237.s008">Tables S4</xref> and <xref ref-type="supplementary-material" rid="pgen.1002237.s009">S5</xref>).</p><p>Based on the main finding in Discovery, we specified the replication hypothesis a-priori as follows: &#8220;Among heavy coffee drinkers, carriers of rs4998386_T allele have a lower risk of PD than carriers of rs4998386_CC genotype&#8221;. Note that we were using the GWAIS as a means to identify the genes that might enhance the inverse association of coffee with PD with the goal of carrying the discovery forward to pharmacogenetic studies. Hence, the replication hypothesis was framed as specified. We used three datasets for replication PEG <xref ref-type="bibr" rid="pgen.1002237-Ritz1">[20]</xref>, PAGE <xref ref-type="bibr" rid="pgen.1002237-Chen2">[21]</xref>, and HIHG <xref ref-type="bibr" rid="pgen.1002237-Edwards1">[9]</xref>. We tested between-study heterogeneity using Breslow-Day test statistics. There was no heterogeneity in coffee use, in rs4998386_CC or in rs4998386_TC genotype frequencies, but rs4998386_TT frequency, which is quite rare, varied significantly across studies. There were a total of 26 cases and 26 controls with rs4998386_TT genotype in Discovery and Replication combined. We found no trend in rs4998386_TT subject characteristics that could help pinpoint the source of heterogeneity (<xref ref-type="supplementary-material" rid="pgen.1002237.s010">Table S6</xref>). Given the unanticipated heterogeneity in rs4998386_TT, we performed genotype-specific analysis (comparing TC to CC, excluding TT) as well as Dominant and Additive models which included TT. Categorical data were analyzed using logistic regression in SAS (version 9.2) and were adjusted for age and sex, and for source of data when data were pooled. Age at onset was analyzed as a continuous variable using linear regression in SAS.</p><p>P values were two-sided for Discovery, one-sided for Replication given the clear directional prior hypothesis <xref ref-type="bibr" rid="pgen.1002237-Neter1">[28]</xref>, and two sided when Discovery and Replication were pooled. There is no agreed-upon significance threshold for GWAIS. The Bonferroni corrected threshold for all 811,597 SNPs on the array is P&lt;6.4&#215;10<sup>&#8722;8</sup>. However, not all 811,597 SNPs are independent due to linkage disequilibrium (LD). SimpleM <xref ref-type="bibr" rid="pgen.1002237-Gao1">[29]</xref> provides a sound Bonferroni-based multiple testing correction method for GWAS based on the estimated number of independent tests, allowing for marker-to-marker LD. It was shown to be the best approximation for permutation, which is computationally prohibitive for GWAS. Using simpleM we calculated the number of independent SNPs genome-wide for NGRC as M<sub>eff</sub>&#8202;=&#8202;430,151; thus the Bonferroni corrected threshold for independent tests was P&lt;1.16&#215;10<sup>&#8722;7</sup>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used IMPUTE v2 <xref ref-type="bibr" rid="pgen.1002237-Howie1">[30]</xref> with HapMap and 1000 Genomes genotypes combined as reference data to infer genotypes for SNPs that were not originally included on the Illumina OMNI-1 array and thus not genotyped in the NGRC dataset. 2,710,971 SNPs were imputed with high reliability (information score &#8805;0.95) and had MAF&gt;0.01, increasing the genomic coverage to 3,522,568 SNPs total (genotyped and imputed). We performed GWAIS and stratified GWAS for the <italic>GRIN2A</italic> region (Chromosome 16, 97 Mb&#8211;102 Mb) using genotype probability data (dose 2-0) in R software <ext-link ext-link-type="uri" ns0:href="http://www.r-project.org/">http://www.r-project.org/</ext-link>.</p><p>Linkage disequilibrium and haplotype blocks were estimated using the Haploview software <xref ref-type="bibr" rid="pgen.1002237-Barrett1">[31]</xref>. Haplotype analysis was performed using hapstat adjusting for sex and age <xref ref-type="bibr" rid="pgen.1002237-Lin1">[32]</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used Golden Helix SNP Variation Suite version 7.2.3 (<ext-link ext-link-type="uri" ns0:href="http://www.goldenhelix.com/">http://www.goldenhelix.com/</ext-link>) and PennCNV <xref ref-type="bibr" rid="pgen.1002237-Wang1">[33]</xref> to explore for deletions or duplications in the <italic>GRIN2A</italic> region. Golden Helix found no CNVs; PennCNV identified two controls with CNVs, which even if confirmed to be real, would not affect the results of the study.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">NGRC genome-wide genotypes, phenotype and environmental data are available at <ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/gap">www.ncbi.nlm.nih.gov/gap</ext-link> study accession number phs000196.v1.p1.</p>