<p>Seven study samples contributed to this meta-analysis of GWA studies on serum DHEAS concentrations, comprising a total of 14,846 men and women of Caucasian origin. The consortium was made up of populations from TwinsUK (n&#8202;=&#8202;4,906), Framingham Heart Study (FHS) (n&#8202;=&#8202;3,183), SHIP (n&#8202;=&#8202;1,832), Rotterdam Study (RS1) (n&#8202;=&#8202;1,597), InCHIANTI (n&#8202;=&#8202;1,182), Health ABC (n&#8202;=&#8202;1,222), and GOOD (n&#8202;=&#8202;924). Full details can be found in <xref ref-type="supplementary-material" rid="pgen.1002025.s005">Text S1</xref>.</p><p>Blood samples were collected from each of the study participants either after overnight fasting or non-fasting and the serum levels of DHEAS were measured by either immunoassay or liquid chromatography tandem mass spectrometry (LC-MS/MS) methods (<xref ref-type="supplementary-material" rid="pgen.1002025.s005">Text S1</xref>). Because the distribution of the serum DHEAS levels was skewed, we log transformed the concentrations and the transformed data used in the subsequent analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Seven study populations were genotyped using a variety of genotyping platforms including Illumina (HumanHap 317k, 550k, 610k, 1M-Duo BeadChip) and Affymetrix (array 500K, 6.0). Each cohort followed a strict quality control on the genotyping data. More details on the quality control and filtering criteria can be found in <xref ref-type="supplementary-material" rid="pgen.1002025.s005">Text S1</xref>. In order to increase genomic coverage and allow the evaluation of the same SNPs across as many study populations as possible, each study imputed genotype data based on the HapMap CEU Build 36. Algorithms were used to infer unobserved genotypes in a probabilistic manner in either MACH (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/MACH">http://www.sph.umich.edu/csg/abecasis/MACH</ext-link>), or IMPUTE <xref ref-type="bibr" rid="pgen.1002025-Marchini1">[61]</xref>. We exclude non-genotyped SNPs with an imputation quality score &lt;0.2 and SNPs with allele frequency &lt;0.01 from meta-analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Each study performed genome-wide association testing for serum concentrations of DHEAS across approximately 2.5 million SNPs under an additive genetic model separately in men and women (<xref ref-type="supplementary-material" rid="pgen.1002025.s005">Text S1</xref>). The analyses were adjusted for age. In addition, the association testing was performed in the combined men and women data with adjustment for age and sex. Studies used PLINK, GenABEL, SNPTEST, QUICKTEST, or MERLIN for the association testing. The summary results from each cohort were meta-analyzed by Z-score pooling method implemented in Metal (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/metal/">http://www.sph.umich.edu/csg/abecasis/metal/</ext-link>). We chose this method to minimize the impact of the different assays used for serum DHEAS measurements. Specifically, for each study, we converted the two-sided P value after adjustment for population stratification by the genomic control method to a Z statistic that was signed to reflect the direction of the association given the reference allele. Each Z score was then weighted; the squared weights were chosen to sum to 1, and each sample-specific weight was proportional to the square root of the effective number of individuals in the sample. We summed the weighted Z statistics across studies and converted the summary Z score to a two-sided P value. We also used I<sup>2</sup> index to assess between-study heterogeneity and the inverse variance weighted method to estimate the effect size. Genome-wide significance was defined as p&lt;5&#215;10<sup>&#8722;8</sup>. The association between the DHEAS-associated SNPs and the related gene expression levels in MuTHER data were examined by mixed linear regression modelling which takes both family structure and batch effects into account. The significance was defined as p&lt;0.006 after accounting for multiple testing (Bonferroni method, correcting 9 independent tests).</p><p>Meta-Analysis Gene-set Enrichment of variaNT Associations (MAGENTA) was used to explore pathway-based associations in the full GWAS dataset. MAGENTA implements a gene set enrichment analysis (GSEA) based approach, the methodology of which is described in Segr&#232; et al <xref ref-type="bibr" rid="pgen.1002025-Segre1">[30]</xref>. Briefly, each gene in the genome is mapped to a single index SNP with the lowest P-value within a 110 kb upstream, 40 kb downstream window. This P-value, representing a gene score, is then corrected for confounding factors such as gene size, SNP density and LD-related properties in a regression model. Genes within the HLA-region were excluded from analysis due to difficulties in accounting for gene density and LD patterns. Each mapped gene in the genome is then ranked by its adjusted gene score. At a given significance threshold (95th and 75th percentiles of all gene scores), the observed number of gene scores in a given pathway, with a ranked score above the specified threshold percentile, is calculated. This observed statistic is then compared to 1,000,000 randomly permuted pathways of identical size. This generates an empirical GSEA P-value for each pathway. Significance was determined when an individual pathway reached a false discovery rate (FDR)&lt;0.05 in either analysis. In total, 2529 pathways from Gene Ontology, PANTHER, KEGG and Ingenuity were tested for enrichment of multiple modest associations with serum DHEAS levels.</p><p>All studies were approved by local ethics committees and all participants provided written informed consent as stated in <xref ref-type="supplementary-material" rid="pgen.1002025.s005">Text S1</xref>.</p>