<p>In all studies, all participants gave informed consent. All studies were approved by their appropriate Research Ethics Committees.</p><p>In population based cohorts, serum creatinine measurements were calibrated to the National Health and Nutrition Examination Study (NHANES) standards in all studies to account for between-laboratory variation across studies, as described previously <xref ref-type="bibr" rid="pgen.1002292-Fox1">[10]</xref>, <xref ref-type="bibr" rid="pgen.1002292-Kttgen1">[16]</xref>, <xref ref-type="bibr" rid="pgen.1002292-Kttgen2">[17]</xref>. Using calibrated serum creatinine, we calculated the estimated glomerular filtration rate (eGFR) with the 4-variable MDRD equation <xref ref-type="bibr" rid="pgen.1002292-Levey2">[49]</xref>.</p><p>For incident CKD, we analyzed studies of incident CKD in eight population-based cohorts in the CKDGen consortium with follow-up available: ARIC, CHS, CoLaus, FHS, KORA S3/F3, KORA S4/F4, the Rotterdam Study and SHIP. Each study's design is shown in <xref ref-type="supplementary-material" rid="pgen.1002292.s004">Text S1</xref>. Incident CKD cases were defined as those free of CKD at baseline (defined as eGFR&#8805;60 ml/min/1.73m<sup>2</sup>) but with a follow-up eGFR&lt;60 ml/min/1.73m<sup>2</sup>. Controls were those free of CKD at baseline and at follow-up.</p><p>For the ESRD analysis, we performed four case control studies of ESRD. Cases were ESRD patients from six cohorts of ESRD patients: CHOICE, ArMORR, GENDIAN, 4D, MMKD and FHKS. Controls were those free of CKD (defined as eGFR&#8805;60 ml/min/1.73m<sup>2</sup>) in three population-based cohorts (KORA F3, KORA F4, SAPHIR) and one type 2 diabetes cohort (GENDIAN). Each study's design is shown in <xref ref-type="supplementary-material" rid="pgen.1002292.s004">Text S1</xref>.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">In each study, we performed age- and sex adjusted logistic regression of incident CKD, with and without additional adjusting for baseline eGFR, or ESRD status with each SNP. In multicenter studies further adjustment for study-center was performed to account for possible differences between recruiting centers. For family-based studies, we applied logistic regression via generalized estimating equations (GEE) to account for the familial relatedness. Study-specific results were then combined by meta-analysis using a fixed effects model, using METAL (<ext-link ext-link-type="uri" ns0:href="http://www.sph.umich.edu/csg/abecasis/Metal/index.html">http://www.sph.umich.edu/csg/abecasis/Metal/index.html</ext-link>) <xref ref-type="bibr" rid="pgen.1002292-Willer1">[50]</xref>. When significant heterogeneity between studies was observed (p for heterogeneity between studies &lt;0.05) we used the random effects model <xref ref-type="bibr" rid="pgen.1002292-DerSimonian1">[51]</xref>. Statistical significance was defined as a one-sided p-value &lt;0.05 for each SNP without adjustment for multiple testing since all SNPs examined had strong prior probabilities of being associated with the outcomes and the same alleles were hypothesized to be associated with lower eGFR, incident CKD, and ESRD.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">We used the QUANTO software for power estimation, assuming an additive genetic model (<ext-link ext-link-type="uri" ns0:href="http://hydra.usc.edu/GxE">http://hydra.usc.edu/GxE</ext-link>) <xref ref-type="bibr" rid="pgen.1002292-Gauderman1">[52]</xref>. For the ESRD analysis and for SNPs with minor allele frequency ranging from 0.2 to 0.4 we had 80&#8211;100% power to detect an OR &#8805; 1.10, whereas power was borderline for an OR of 1.05 to 1.09. For example, for the SNP rs12917707 at <italic>UMOD</italic>, we had 100% power to detect an association with ESRD in the 3775 ESRD cases and 4577 controls assuming that the effect in ESRD would be the same or larger than the effect observed for prevalent CKD previously <xref ref-type="bibr" rid="pgen.1002292-Kttgen1">[16]</xref>, <xref ref-type="bibr" rid="pgen.1002292-Kttgen2">[17]</xref>.</p><p>For the incident CKD analysis, we used the allele dosage information of each of the 16 SNPs from each study's genome wide data set imputed to HAPMAP CEU samples described previously <xref ref-type="bibr" rid="pgen.1002292-Kttgen2">[17]</xref>, <xref ref-type="bibr" rid="pgen.1002292-Chambers1">[18]</xref>. Imputation provides a common SNP panel across all studies to facilitate a meta-analysis across all contributing SNPs. Information on each study's genotyping and imputation platform and quality control procedures are shown in <xref ref-type="supplementary-material" rid="pgen.1002292.s001">Table S1</xref>. <xref ref-type="supplementary-material" rid="pgen.1002292.s002">Table S2</xref> summarizes each SNPs imputation quality.</p><p>De novo genotyping of the 16 SNPs was performed in each of the ESRD case-control studies as described previously <xref ref-type="bibr" rid="pgen.1002292-Kttgen2">[17]</xref>. Briefly, genotyping was performed either on a MassARRAY system using Assay Design v.3.1.2 and the iPLEX&#8482; chemistry (Sequenom, San Diego, USA) at the Helmholtz Zentrum in Munich, Germany (ArMORR, GENDIAN, 4D, MMKD, FHKS, KORA S3/F3-subset without GWAS data, KORA S4/F4-subset without GWAS data, SAPHIR); by using 5&#8242; nuclease allelic discrimination assays on 7900HT Fast Real-Time Taqman PCR genotyping systems (Applied Biosystems, Foster City, CA, USA) at the Innsbruck Medical University (ArMORR, GENDIAN, 4D, MMKD, FHKS, KORA F3-subset without GWAS data, KORA F4-subset without GWAS data, SAPHIR); or as part of a larger panel of 768 SNPs genotyped on the Illumina Bead Station (CHOICE). The SNPs rs347685, rs11959928, rs4744712 and rs12460876 were not available for de novo genotyping on the Sequenom platform, thus the proxy SNPs rs6773343, rs11951093, rs1556751 and rs8101881, with pairwise r<sup>2</sup> of 1.0, 0.87, 0.87 and 1.0 respectively <xref ref-type="bibr" rid="pgen.1002292-Johnson1">[53]</xref>, were included in the MassARRAY multiplex PCR.</p><p>For the obtained duplicate genotypes (9&#8211;22% of the subjects in GENDIAN, 4D, MMKD, FHKS, KORA F3-subset without GWAS data, KORA F4-subset without GWAS data, and SAPHIR; no duplicate genotyping possible due to limited DNA-availability in CHOICE and ArMORR) concordance was 96&#8211;100% (median: 100%). SNPs with a per-study call rate &lt;90% or with a per-study HWE p value &lt;0.0001 were excluded from further analysis (rs6773343 and rs653178 in GENDIAN cases; rs13538, rs267734, rs10109414, rs1394125 in ArMORR, rs6773343, rs10109414, rs1556751, rs653178, rs8101881 in CHOICE). In addition, individual samples with &lt;80% successfully genotyped SNPs were excluded from further analysis. After these exclusions, call rates ranged from 91&#8211;100% (mean: 98%) across all studies and all SNPs.</p>