<p>Anonymised tumour and normal adjacent to tumour prostate tissue samples were either received from the National Cancer Institute (NCI) Cooperative Prostate Cancer Tissue Resource (CPCTR) or from the University of Maryland under protocols reviewed and approved by Institutional Review Boards.</p><p>Breast cancer cell lines MCF-7, MDA-MB-134 (also referred to as MDA134), PMC42, T47D, normal-like breast cell line HB2, prostate cancer cell lines PC3 and 1542-CP, normal-like prostate cell line RWPE-1 and the HCT116 colon cancer cell line were from the Cambridge Research Institute culture collection. T47D, MDA134, PMC42, PC3 and HCT116 were maintained in RPMI, 10% foetal calf serum (FCS) and antibiotics; MCF-7 and HB2 in DMEM, 10% FCS and antibiotics, but HB2 cell medium was supplemented with insulin (5 &#181;g/ml) and hydrocortisone (1 &#181;g/ml) (Sigma, UK). RWPE-1 cells were cultured in Keratinocyte Serum Free Medium (Gibco) and supplemented with 5 ng/ml human recombinant EGF and 0.05 mg/ml bovine pituitary extract (both from Sigma). 1542-CP cells were cultured in keratinocyte serum-free medium (Keratinocyte-SFM, Life Technologies, Grand Island, NY) containing 25 &#181;g/ml bovine pituitary extract, 5 ng/ml epidermal growth factor, 2 mM L-glutamine, 10 mM HEPES buffer, antibiotics, and 5% heat-inactivated FCS.</p><p>Nuclei of all cell lines were harvested and digested with DNase I as previously described <xref ref-type="bibr" rid="pgen.1002165-Udler1">[22]</xref>. For the DNase-seq experiment DNA from DNase-digested nuclei was isolated by standard procedures, separated on a 1.4% agarose gel and gel-purified DNA fragments between 125 and 500 bp were used directly in Illumina pre-amplification. The amplified library was again resolved on an agarose gel to select 200&#8211;300 bp fragments that were gel-purified and sequenced using an Illumina Genome Analyser II, according to manufacturer's instructions. MCF-7 DNase-seq was carried out as part of our analysis of ER responses. Multiple MCF-7 aliquots were transfected with and without a siRNA to GATA3, but no differences in the DHS profile were detected. BWA software <xref ref-type="bibr" rid="pgen.1002165-Li1">[37]</xref> was used for sequence alignment by the CRI Bioinformatics core facility. A representative experiment with low background is shown. Two independent DNase-seq experiments for PC3 and RWPE-1 gave very similar results and again a representative example is shown. Sequence reads were displayed on the UCSC genome browser. The presence of the S-DHS DNase-seq peak was verified in T47D cells (<xref ref-type="supplementary-material" rid="pgen.1002165.s004">Figure S4</xref>). For DNase-chip the generation of libraries of DNase hypersensitive fragments has previously been described <xref ref-type="bibr" rid="pgen.1002165-Udler1">[22]</xref>. Libraries were hybridised to Agilent custom tiling arrays covering 2Mb at 8q24.</p><p>The 3C method <xref ref-type="bibr" rid="pgen.1002165-Dekker1">[38]</xref> was applied to detect physical interactions between genomic regions (such as between promoters and distant enhancers <xref ref-type="bibr" rid="pgen.1002165-West1">[39]</xref>) using a sequence of interest (bait) from 8q24 region. The experimental procedure for this technique is outlined in <xref ref-type="supplementary-material" rid="pgen.1002165.s012">Figure S12</xref>. The 3C experiments were carried out in the normal-like cell lines HB2 and RWPE-1 as many cancer cell lines such as PC3 and MCF-7 carry amplifications across the 8q24 region <xref ref-type="bibr" rid="pgen.1002165-Liu1">[9]</xref>, <xref ref-type="bibr" rid="pgen.1002165-Guan1">[10]</xref>. Briefly, exponentially growing cells were cross-linked with 1% formaldehyde for 10 min at room temperature, washed 3x with cold PBS and scraped into microcentrifuge tubes. Cells were spun at 3000 rpm for 5 mins and resuspended in 1 ml lysis buffer (50 mM Tris-HCl pH 8.0, 1% SDS, 10 mM EDTA). After 10 min cells were centrifuged and resuspended in lysis buffer supplemented with 0.4% NP-40 and 1.8% Triton-X. After centrifugation for 20 seconds at full speed 1.5&#215;10<sup>6</sup> nuclei were resuspended in 300 &#181;l <italic>Eco</italic>RI digestion buffer and 1.8% Triton-X. Nuclei were incubated for 45 minutes at 37&#176;C in a cell shaker and subsequently digested with 1000 U <italic>Eco</italic>RI (NEB) overnight. The enzyme was heat-inactivated and the sample diluted in 1 ml. Ligation was carried out overnight using 4000 U T4 ligase at 16&#176;C. A second digestion step was carried out using 1000 U <italic>Bam</italic>HI (NEB). Genomic DNA was purified after proteinase K treatment using standard protocols. PCRs were carried out using Power SYBR Green Mastermix (Applied Biosystems), 100 ng template DNA, 5 pmol of each primer in a volume of 20 &#181;l (initial 95&#176;C denaturation step, then 1 min at 60&#176;C and 20 secs at 92&#176;C for 40 cycles). Products were separated on a 3% NuSieve agarose gel. The identity of all PCR products was verified by direct sequencing of PCR products.</p><p>EMSAs were carried out as previously described <xref ref-type="bibr" rid="pgen.1002165-Meyer1">[40]</xref>. Oligonucleotide sequences used in the assays are provided in <xref ref-type="supplementary-material" rid="pgen.1002165.s013">Table S1</xref>. Antibodies for supershifts were obtained from Santa Cruz Biotech: &#945;YY1 (sc1703x), &#945;C/EBP&#945; (sc9314x), &#945;SP1 (sc14027x) and &#945;Oct-1 (sc232x) and 2 &#181;l were used per reaction.</p><p>ChIP reactions were carried out as described by Schmidt <italic>et al</italic>. <xref ref-type="bibr" rid="pgen.1002165-Schmidt2">[41]</xref> using 10 &#181;g &#945;YY1 (sc7341x, Santa Cruz Biotech). DNA was purified from the precipitates and used in RT-PCR reactions with primers listed in <xref ref-type="supplementary-material" rid="pgen.1002165.s013">Table S1</xref>. Primer design was carried out using the UCSC genome browser mapability plot (<xref ref-type="supplementary-material" rid="pgen.1002165.s003">Figure S3B</xref>) in conjunction with Primer 3 software. The primer pairs used gave rise to a single reproducible band whose identity was confirmed by sequencing. The RT-PCR protocol includes a dissociation step to confirm that the reaction gave rise to only a single product. Results were normalised against input and normal mouse IgG (sc2025, Santa Cruz Biotech). As positive control the promoter region of the glucocorticoid receptor <xref ref-type="bibr" rid="pgen.1002165-Kim1">[42]</xref> was chosen, as this contains three YY1 sites. Results are given relative to S-DHS-ve, a sequence 5 kb upstream of the S-DHS which we have shown to be in inaccessible chromatin (<xref ref-type="fig" rid="pgen-1002165-g001">Figure 1</xref>). 1542-CP cells and MCF-7 cells are heterozygous for the risk SNP rs378854. Consistent results were obtained in three biological repeats. Error bars denote the technical error in a representative experiment.</p><p>S-DHS_cloning-forward and reverse primers (<xref ref-type="supplementary-material" rid="pgen.1002165.s013">Table S1</xref>) were used to amplify a DNA fragment from genomic DNA of PMC42 breast cancer cell line (heterozygous for rs378854) and cloned using the TopoTA cloning system (Invitrogen). Plasmids were sequenced and clones carrying either the minor or the common allele of rs378854 were selected. The inserts were excised using <italic>Sal</italic>I and <italic>Bam</italic>HI (resulting fragment: chr8:128,392,958-128,393,353) and cloned into pGL3-basic or pGL3-enhancer vector (Promega), cut with <italic>Xho</italic>I and <italic>Bgl</italic>II. The same fragments were inserted into pGL3-promoter vector (Promega) cut with <italic>Bam</italic>HI and <italic>Sal</italic>I. Plasmids were prepared with an endotoxin-free kit (Qiagen) and transiently transfected into PC3 and MCF-7 cells grown in 24 well plates. 0.5 &#181;g reporter plasmid and 0.1 &#181;g &#946;-galactoside transfection control plasmid in 30 &#181;l OptiMEM (Gibco) were incubated for 20 min with 4.5 &#181;l Fugene (Roche) before being added to 50% confluent wells. 10% FCS was added after 2 hours. Cells were harvested in 100 &#181;l lysis buffer after 24 hours and luciferase and &#946;-galatosidase activity was assayed using Promega kits. The data is averaged from two independent transfection experiments, each carried out in triplicate.</p><p>Anonymised tumour and normal adjacent to tumour prostate tissue samples were either received from the National Cancer Institute (NCI) Cooperative Prostate Cancer Tissue Resource (CPCTR) or from the University of Maryland under protocols reviewed and approved by Institutional Review Boards. We also obtained clinical information including age at diagnosis, race, Gleason score and PSA level at diagnosis.</p><p>The frozen samples were homogenized with Tissue Lyser (Qiagen) and divided into two fractions from which total RNA was prepared with MirVana kit (Applied Biosystems) and DNA was prepared with Gentra (Qiagen). The integrity of RNA was confirmed by Bioanalyzer (Agilent).</p><p>cDNA was prepared from 800 ng of total RNA with Superscript III kit and random hexamers (Invitrogen). All expression assays were first evaluated in pooled prostate cDNA samples containing 25 ng, 5 ng, or 1 ng of total RNA per reaction. Genomic DNA and water were used as negative controls for each assay. Based on this test, endogenous controls Beta-2 microglobulin (<italic>B2M</italic>, assay HS_00187842_m1), Cyclophilin (<italic>PPIA, assay</italic> 4326316E) and <italic>MYC</italic> expression (assay Hs00905030) were tested using 5 ng of total RNA per reaction, while <italic>PVT1</italic> expression (assay Hs00413039) was evaluated with 20 ng of total RNA per reaction. Expression of alternative splicing forms of <italic>PVT1</italic> measured by assay Hs01069044 was beyond confident detection level even with 20 ng of total RNA and was not used on individual tissue samples.</p><p>The expression was tested in 4 technical replicates for each sample and assay, technical replicates were averaged and standard deviations were determined.</p><p>TaqMan miRNA expression assays for target miRNAs from <italic>PVT1</italic> gene (002872 for hsa-miR-1204, 002778 for hsa-miR-1205, 002878 for hsa-miR-1206, 002826 for hsa-miR-1207-3p, 241060-mat for hsa-miR-1207-5p, 002880 for hsa-mir-1208) and 3 endogenous controls (001973 for U6 snRNA, 001094 for RNU44 and 001006 for RNU48) as well as other reagents for miRNA expression were purchased from Applied Biosystems. cDNA was prepared from experimentally determined amount of total RNA with TaqMan MicroRNA reverse transcription kit. All assays were first tested on pooled cDNA samples using 10, 20 and 100 ng of total RNA per reaction as input. Based on this test, U6, RNU44 and RNU48 assays were run on all individual prostate samples starting from 20 ng of total RNA per reaction. For hsa-miR-1204, 1205, 1206 and 1207-3p reactions were run starting from 100 ng of total RNA per reaction. Expression of hsa-miR-1207-5p was below detection level and was not tested further. All expression assays were run in technical duplicates and target assays were normalized by a geometric mean of U6, RNU44 and RNU48.</p><p>A custom-designed assay was used to quantify copy number variation (CNV) within intron 1 of <italic>PVT1</italic> gene (PVT1-CNV-F and PVT1-CNV-R, 173 bp) (<xref ref-type="supplementary-material" rid="pgen.1002165.s013">Table S1</xref>). The assay was run on DNA prepared from the prostate tissue samples with 5 ng of DNA, 2x Power SYBR Green master mix (Applied Biosystems) and primers. The results were normalized to copy number of control RNaseP assay quantified in the same DNA samples. All assays were run in four technical replicates on 7900 Sequence Detection System (Applied Biosystems).</p><p>Statistical significance of differences in the reporter assays was determined using a Student's two-sided t-test with a Bonferroni correction for multiple comparisons. The expression values of mRNA and miRNA target assays were normalized by corresponding endogenous controls according to dCt method of relative quantification and tested for normality of distribution. Univariate linear regression was used to analyze expression values in relation to 0, 1 or 2 copies of risk alleles of rs378854. Age, race and <italic>PVT1</italic>-CNV values were tested as possible covariates but were found to have no significant effect. The analysis was performed with SPSS 16.0. Normalized expression values were centered to the mean of the samples heterozygous for rs378854 and plotted with GraphPad Prism5 software.</p>