<p>The immortalized MEFs used in this study were prepared from mice with the following genotypes: <italic>Pcna<sup>+/+</sup></italic> and <italic>Pcna<sup>K164R/K164R</sup></italic>
<xref ref-type="bibr" rid="pgen.1002262-Langerak1">[29]</xref>; <italic>Rad18<sup>+/+</sup></italic> and <italic>Rad18<sup>&#8722;/&#8722;</sup></italic>
<xref ref-type="bibr" rid="pgen.1002262-Tateishi1">[37]</xref>; <italic>Hltf<sup>+/+</sup>Shprh<sup>+/+</sup></italic> and <italic>Hltf<sup>&#8722;/&#8722;</sup>Shprh<sup>&#8722;/&#8722;</sup></italic>
<xref ref-type="bibr" rid="pgen.1002262-Krijger1">[38]</xref>; <italic>Usp1<sup>+/+</sup></italic>, <italic>Usp1<sup>&#8722;/&#8722;</sup></italic>, <italic>Usp1<sup>&#8722;/&#8722;</sup></italic> complemented with wild-type <italic>Usp1</italic>, and <italic>Usp1<sup>&#8722;/&#8722;</sup></italic> complemented with the inactive <italic>Usp1</italic>C90S mutant <xref ref-type="bibr" rid="pgen.1002262-Kim1">[40]</xref>. The immortalized MEFs were cultured in DMEM supplemented with 2 mM L-glutamine, 100 units/ml of penicillin, 100 &#181;g/ml of streptomycin, and 10% FBS. The primary MEFs were cultured in DMEM supplemented with 2 mM L-glutamine, 100 units/ml of penicillin, 100 &#181;g/ml of streptomycin, non-essential amino acids (Biological industries), 2-Mercaptoethanol 50 &#181;M, and 15% FBS. The immortalized MEFs were incubated at 37&#176;C in a 5% CO<sub>2</sub> atmosphere. The primary MEFs were incubated at 37&#176;C in a 5% CO<sub>2</sub> and 4% O<sub>2</sub> atmosphere.</p><p>Separation of cells at G1/S phase of the cell cycle was performed by the elutriation method with the following modifications. The elutriation system consisted of a J6 Beckman elutriation centrifuge with a JE-5.0 rotor equipped with a single standard 5 ml elutriation chamber (Beckman Coulter, Inc., Fullerton, CA, USA), and a masterflex microprocessor pump drive, model 7524-05 (Cole Parmer). The elutriation medium was DMEM supplemented with 1% FBS, maintained at room temperature. The speed and temperature of the rotor were set constant at 3000 rpm and 25&#176;C. Approximately 3&#215;10<sup>8</sup>
<italic>Pcna<sup>+/+</sup></italic> or <italic>Pcna<sup>K164R/K164R</sup></italic> MEFs were harvested from cultures at &#8764;80% confluence, centrifuged, and suspended in 10 ml of DMEM (room temprature) supplemented with 1% FBS. Cell suspensions were introduced to the elutriation chamber at a flow rate of 50 ml/min. After 15 minutes the flow rate was increased by 10 ml/min and three 50 ml fractions were collected at this flow rate. The flow rate was then gradually increased to 160 ml/min in 10 ml/min increments. Three 50 ml fractions were collected after each subsequent increase of the flow rate. The G1/S fraction (analyzed by FACS) was taken for the UV-induced Rpa foci assay.</p><p>For Rpa immunostaining <xref ref-type="bibr" rid="pgen.1002262-Diamant1">[33]</xref>, <italic>Pcna<sup>+/+</sup></italic> and <italic>Pcna<sup>K164R/K164R</sup></italic> MEFs were fractionated by centrifugal elutriation, and cells in the G1/S boundary were seeded on 13 mm glass cover slips coated with 0.01% poly-L-lysine. After 2 h when the cells attached to the slides, the medium was removed and the cells were UV-C irradiated at 254 nm using a low-pressure mercury lamp (TUV 15w G15T8, Philips) at doses of 8 J/m<sup>2</sup>. The dose rate was measured using an UVX Radiometer (UVP) equipped with a 254-nm detector. At various time points after irradiation the cells were washed three times with PBS, pre-extracted with 25 mM HEPES pH 7.4, 50 mM NaCl, 3 mM MgCl<sub>2</sub>, 300 mM sucrose, 1% Triton X-100 for 5 minutes on ice with gentle shaking, and washed for three more times with PBS. The slides were then fixed in 4% paraformaldehyde for 15 minutes at room temperature and washed three times in PBS. Blocking was done in PBS supplemented with 5% normal goat serum for 30 minutes on ice. The cells were incubated for 4 hours on ice with anti-Rpa32 antibodies (AbCam, cat. No. ab2175) that were diluted 1&#8758;200 in blocking solution. After incubation the slides were washed three times in PBS and incubated with a secondary antibody &#8211;goat anti mouse Alexa Fluor 488 (green) diluted 1&#8758;1000, and with DAPI diluted 1&#8758;1000 (both in blocking solution) for 45 minutes on ice. The slides were then washed three times in PBS and mounted on microscope slides using Aqua poly/Mount. Images were captured with a DeltaVision system (Applied Precision) equipped with an Olympus IX71 microscope. Optical images were acquired using CCD camera (Photometrics, Coolsnap HQ) and a 60&#215;/1.42 objective (Olympus). For each cell line at each time point at least 100 cells were counted and the percentage of cells exhibiting Rpa foci was determined.</p><p>(<xref ref-type="supplementary-material" rid="pgen.1002262.s001">Figure S1</xref>) The assay was performed as previously described <xref ref-type="bibr" rid="pgen.1002262-Avkin2">[34]</xref>, and the gapped plasmids with site-specific lesions used in this assay were prepared as previously described as follows: TT CPD and TT 6-4 PP <xref ref-type="bibr" rid="pgen.1002262-Hendel1">[35]</xref>; cisPt-GG <xref ref-type="bibr" rid="pgen.1002262-Shachar1">[6]</xref>. Briefly, cells were co-transfected with a DNA mixture containing 100 ng of a gapped-lesion plasmid (GP-TT-CPD, or GP-TT-6-4 PP, or GP-cisPt-GG; kan<sup>R</sup>), 100 ng of a control gapped plasmid without a lesion (GP20; cm<sup>R</sup>), and 5 &#181;g of the carrier plasmid pUC18, using jetPEI/DNA complexes for the immortalized MEFs or the Lipofectamine 2000 for the primary MEFs. After allowing time for gap filling and lesion bypass, the plasmids were extracted from the cells using alkaline lysis conditions, and used to transform an <italic>E. coli RecA</italic> reporter strain. The percentage of plasmid repair, of which most occurs by TLS, was calculated by dividing the number of transformants obtained from the gap-lesion plasmid (kan<sup>R</sup> colonies) by the number of transformants obtained from the control gapped-plasmid (cm<sup>R</sup> colonies). A small fraction of gap-lesion plasmids can be repaired by non-TLS events, which involve formation of a DSB followed by DSB repair. These are observed as plasmid isolates with large deletions or insertions. To obtain precise TLS extents, the plasmid repair extents were multiplied by the fraction of TLS events out of all plasmid repair events, based on the DNA sequence analysis of the plasmids. To determine the DNA sequence changes that have occurred during plasmid repair, sequence analysis was carried using the TempliPhi DNA Sequencing Template Amplification Kit and the BigDye Terminator v1.1 Cycle Sequencing Kit. Reactions were analyzed by capillary electrophoresis on an ABI 3130XL Genetic Analyzer from Applied Biosystems.</p><p>Two methods were used: depletion of ATP as a measure for viability, and colony forming ability after UV irradiation. For the viability ATP assay <italic>Pcna<sup>+/+</sup></italic> and <italic>Pcna<sup>K164R/K164R</sup></italic> MEFs were seeded in 96-well plates. At 24 h after the seeding, cells were washed twice with Hanks' buffer, and irradiated in Hanks' buffer with UV-C at 254 nm using a low-pressure mercury lamp (TUV 15w G15T8, Philips). UV dose rate was measured using an UVX Radiometer (UVP) equipped with a 254-nm detector. After irradiation, Hanks' buffer was removed and the cells were incubated in a fresh growing medium for additional 48 h. Viability was determined using the CellTiter-Glo Luminescent Cell Viability Assay (Promega). Luminescence was measured using an Infinite&#174; M200 Luminometer (Tecan). Throughout the entire experiment, none of the samples reached cell confluency.</p><p>For the colony forming ability assay <italic>Pcna<sup>+/+</sup></italic>and <italic>Pcna<sup>K164R/K164R</sup></italic> immortalized MEFs were transfected with siRNA against TLS polymerases as described below, and incubated for 48 h. Cells were then trypsinized, counted, and plated in 10-cm Petri dishes. After incubation of 12 h, cells were UV irradiated as described above, and incubated in fresh medium for 10&#8211;12 days. Colonies were fixed and stained with 1% methylene blue (Sigma). Colony forming ability was calculated by dividing the number of colonies in UV-irradiated plates by the number of colonies in unirradiated plates with pre-transfected with the same siRNA.</p><p>The expression of specific DNA polymerase genes was knocked-down in <italic>Pcna<sup>+/+</sup></italic> and <italic>Pcna<sup>K164R/K164R</sup></italic> MEFs by transfection with 50 nM of siRNA pools specific for <italic>PolH</italic>, <italic>Rev3L</italic> or <italic>Rev1</italic>. The siRNAs used were from Dharmacon as follows: <italic>mRev3L</italic> SMARTpool (M-04219), <italic>mPolH</italic> ON-TARGETplus SMARTpool (LU-063800), <italic>mRev1</italic> SMARTpool (M-041898), siGENOME non-targeting siRNA #5 (D-001210), ON-TARGETplus nontargeting Pool (D-001810). Transfection was carried out using HiPerFect (Qiagen), according to the manufacturer recommendations. The effectiveness of knocking down the expression of TLS polymerases was measured by RT-PCR using <bold>t</bold>otal RNA that was extracted from the cells 48 h after transfection with siRNA, using the Perfect-Pure RNA cultured cells kit (5 PRIME). A hundred ng of total RNA was used for cDNA synthesis and RT-PCR by Maxime RT-PCR PreMix kit (iNtRON BIOTECHNOLOGY) according to the manufacturer recommendations. The following primers were used for the RT-PCRs: <named-content content-type="gene">5&#8242;-GTGGTACGAGTCTTCGG-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-TCTTGTGACTCGGGCTG-3&#8242;</named-content> for <italic>mREV3L</italic>, <named-content content-type="gene">5&#8242;-GAAGCCCGAGCATTTGGTG-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-GCCTCTCCTCAAGTTCCAG-3&#8242;</named-content> for <italic>mPOLH</italic>, <named-content content-type="gene">5&#8242;-AGAACGGAGAATGATGGC-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-GGCCCAGGATCCTCAGGTTTGCACACAGG-3&#8242;</named-content> for <italic>mRev1</italic>, <named-content content-type="gene">5&#8242;-ACCACAGTCCATGCCATCAC-3&#8242;</named-content> and <named-content content-type="gene">5&#8242;-TCCACCACCCTGTTGCTGTA-3&#8242;</named-content> for <italic>mGAPDH</italic>. The results of knocking-down the expression of <italic>PolH</italic>, <italic>Rev3L</italic> and <italic>Rev1</italic> are shown in <xref ref-type="supplementary-material" rid="pgen.1002262.s002">Figure S2</xref>.</p>