<p>Work in California was conducted under United States Fish and Wildlife Service (USFWS) permit MA 491 672724-9 issued to United States Geological Survey Biological Resource Discipline (USGS492 BRD). Harbor seal carcasses were gathered and samples processed as part of Northwest Marine Mammal Stranding Network activities authorized under Marine Mammal Protection Act (MMPA) Stranding Agreements (SA), and Section 109(h) (16 U.S.C. 1379(h)). Additional specimens were acquired under MMPA Section 120, and the National Marine Fisheries Service (NMFS) MMPA Research Permit 782&#8211;1702.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Parasite DNA was obtained either from infected host tissues or parasite isolates maintained in tissue culture as described previously <xref ref-type="bibr" rid="pgen.1001261-Wendte1">[25]</xref>. Samples were analyzed using a typing scheme that included the surface antigen markers: SnSAG1, SnSAG3, SnSAG4, SnSAG5, SnSAG6 <xref ref-type="bibr" rid="pgen.1001261-Wendte1">[25]</xref> and 9 microsatellite markers Sn2&#8211;Sn5 and Sn7&#8211;Sn11 originally described by Asmundsson and Rosenthal <xref ref-type="bibr" rid="pgen.1001261-Asmundsson2">[85]</xref> but applied as modified in Wendte et al. <xref ref-type="bibr" rid="pgen.1001261-Wendte1">[25]</xref> and Rejmanek et al. <xref ref-type="bibr" rid="pgen.1001261-Rejmanek1">[23]</xref>. Three additional microsatellite markers were designed by the following method: Publically available <italic>Sarcocystis neurona</italic> expressed sequence tags (ESTs) were downloaded from the NCBI dbEST database (<ext-link ext-link-type="uri" ns0:href="http://www.ncbi.nlm.nih.gov/dbEST">http://www.ncbi.nlm.nih.gov/dbEST</ext-link>) and the <italic>S. neurona</italic> Gene Index (maintained by the Computational Biology and Functional Genomics Laboratory at the Dana Farber Cancer Institute, <ext-link ext-link-type="uri" ns0:href="http://compbio.dfci.harvard.edu/tgi/">http://compbio.dfci.harvard.edu/tgi/</ext-link>) databases. The downloaded ESTs were assembled into contigs using the SeqMan (Lasergene) application. Contig sequences were then processed with the MISA microsatellite identification program (<ext-link ext-link-type="uri" ns0:href="http://pgrc.ipk-gatersleben.de/misa/">http://pgrc.ipk-gatersleben.de/misa/</ext-link>) with the following repeat parameters: definition (unit size-minimum repeats): 2-12, 3-7, 4-5, 5-4, 6-3, 7-3, 8-2, 9-2, 10-2, 11-2, 12-2, 13-2, 14-2, 15-2; interruptions (maximum difference between 2 simple sequence repeats): 25.</p><p>Approximately 50 microsatellites of sufficient length and/or complexity were identified. Three (Sn1520, Sn1863 and Sn515) of these markers were not previously published and possessed sufficient non-redundant flanking sequence to allow for nested primer design and produced robust size-polymorphic PCR amplification products. Primers were validated as described <xref ref-type="bibr" rid="pgen.1001261-Wendte1">[25]</xref> and found to be specific and sensitive for <italic>S. neurona</italic> DNA in tissues (data not shown). The primers designed are as follows: Sn1520 Fext- <named-content content-type="gene">GGGGCAGAACCATCGTAGTA</named-content>, Rext- <named-content content-type="gene">GTGAAGCATTTCCCCTACGA</named-content>, Fint- <named-content content-type="gene">GGCGGTAGTCACTTGCTGA</named-content>, Rint- <named-content content-type="gene">GTGGGAGAAGACGGTCGTTA</named-content>; Sn1863 Fext- <named-content content-type="gene">CATGGCGTGCGTTAACTAAA</named-content>, Rext- <named-content content-type="gene">CGTACAAACACACGCTCCAC</named-content>, Fint- <named-content content-type="gene">CCATTCATCGACAGCGACTA</named-content>, Rint- <named-content content-type="gene">TGAGACAGCCGTCAAACACT</named-content>; Sn515 Fext- <named-content content-type="gene">CTTCTAGCGGCTGTTTCTCC</named-content>, Rext- <named-content content-type="gene">TCTGTGTGGGTGTGGAAGTC</named-content>, Fint- <named-content content-type="gene">GACCCCCTCTCTGCTTCTCT</named-content>, Rint- <named-content content-type="gene">ACGCAAATGCGAACATATCA</named-content>. Representative sequences for each allele at each locus were placed in GenBank under the following accession numbers: Sn1520: HM851251, HM851252, HM851253, HM851254, HM851255; Sn1863: HM851256, HM851257, HM851258, HM851259; Sn515: HM851249, HM851250. PCR, DNA sequencing and analysis were conducted as described previously, except, to control for bias in scoring results, random sample IDs were assigned to samples before sequencing so that sequence analysis for some loci was blinded <xref ref-type="bibr" rid="pgen.1001261-Wendte1">[25]</xref>.</p><p>For this study, <italic>S. neurona</italic> DNA from 15 sea otters and 4 harbor seals was analyzed. Additionally, samples from 21 sea otters, 2 harbor seals, 3 horses, and 2 raccoons previously described by Wendte et al. <xref ref-type="bibr" rid="pgen.1001261-Wendte1">[25]</xref> at the SnSAG antigen loci and MS Sn9, were further typed in this study at the remaining 10 MS loci. Finally, <italic>S. neurona</italic> DNA from 21 sea otters, 1 porpoise, 4 horses, 13 opossums, and 1 cat that was previously typed by Rejmanek et al. <xref ref-type="bibr" rid="pgen.1001261-Rejmanek1">[23]</xref> at SnSAG3, SnSAG4, and MS markers Sn2&#8211;Sn5 and Sn7&#8211;Sn11 were combined with the data in this study for a total sample set that included 87 samples from 57 sea otters, 6 harbor seals, 2 raccoons, 13 opossums, 7 horses, 1 porpoise, and 1 cat. In all, 75 of the 87 samples were from California. Other states represented include Georgia (n&#8202;=&#8202;2 samples), Illinois (n&#8202;=&#8202;1), Missouri (n&#8202;=&#8202;3), Washington (n&#8202;=&#8202;4), and Wisconsin (n&#8202;=&#8202;2). Some overlap existed between the samples typed in this study and those reported by Rejmanek et al.: samples SO4387, SO4413, H1, H2, and H3 in this study are reported as SO1, SO2, Horse 1, Horse 2, and Horse 3 in Rejmanek et al. <xref ref-type="bibr" rid="pgen.1001261-Rejmanek1">[23]</xref>, respectively. Complete information about the sample origins is found in <xref ref-type="supplementary-material" rid="pgen.1001261.s001">Table S1</xref>.</p><p>
<italic>Toxoplasma gondii</italic> isolates from a water cistern (n&#8202;=&#8202;2), chickens (n&#8202;=&#8202;11), and one cat associated with a human waterborne toxoplasmosis outbreak <xref ref-type="bibr" rid="pgen.1001261-Vaudaux1">[44]</xref>, as well as laboratory strain CEP were typed at microsatellite loci B17, B18, TgMA, TUB2, W35 <xref ref-type="bibr" rid="pgen.1001261-Ajzenberg1">[39]</xref> and M95 <xref ref-type="bibr" rid="pgen.1001261-Blackston1">[40]</xref>. Markers were PCR amplified and sequenced to assign alleles as for <italic>S. neurona</italic> markers <xref ref-type="bibr" rid="pgen.1001261-Wendte1">[25]</xref>. Representatives of each microsatellite allele at each locus were placed in Genbank under accession numbers: B17: HM851260&#8211;67; TgMA: HM851268&#8211;73; W35: HM851274&#8211;77; M95: HM851278&#8211;81.</p><p>Because different parts of the genome are likely under different selective pressures, all <italic>S. neurona</italic> samples were categorized by an antigen (Ag) type designated by roman numerals and a microsatellite (MS) type indicated by a lowercase letter designation. Ag types were defined by the presence/absence of mutually exclusive antigen genes (SnSAG1, SnSAG5, or SnSAG6) and the inheritance pattern of alleles at SnSAG3 and SnSAG4 <xref ref-type="bibr" rid="pgen.1001261-Rejmanek1">[23]</xref>, <xref ref-type="bibr" rid="pgen.1001261-Wendte1">[25]</xref>. MS types were assigned on the basis of allele combinations defined by the number of di- or tri- nucleotide repeats at each locus (Sn2&#8211;Sn5 and Sn7&#8211;Sn11, Sn1520, Sn1863). Sn515 was a complex repeat in which each isolate possessed one of two alleles. Samples from the study by Rejmanek et al. <xref ref-type="bibr" rid="pgen.1001261-Rejmanek1">[23]</xref> were not typed at the SnSAG1-5-6 loci, but were placed into Ag groups based on the allelic profile at SnSAG3 and SnSAG4 and by the Ag group their MS type was associated with in samples typed at all markers. For example, based on the alleles at SnSAG3 and SnSAG4, sample SO4 (<xref ref-type="supplementary-material" rid="pgen.1001261.s001">Table S1</xref>) could be placed either in Ag type II or V, but its MS type was only found associated with Ag type II in samples where all markers were typed, making this the most likely, though not definitive, Ag type designation. The <italic>S. neurona</italic> strains assessed by Rejmanek et al. <xref ref-type="bibr" rid="pgen.1001261-Rejmanek1">[23]</xref> were also not typed at MS markers Sn1520, Sn1863, and Sn515. Presumptively classifying these samples into MS types based on alleles at Sn2&#8211;Sn5 and Sn7&#8211;Sn11 is likely accurate, though, since these three markers did not provide additional resolution to MS types for the 46 additional <italic>S. neurona</italic> strains described in this study.</p><p>The alleles present at MS markers Sn2&#8211;Sn5 and Sn7&#8211;Sn11 were used for creation of a multi-locus sequence typing scheme by which all isolates could be compared. The numerical designation of alleles allowed the detection of which MS types formed clonal complexes using the eBURST program <xref ref-type="bibr" rid="pgen.1001261-Feil2">[48]</xref>. Default settings were used which grouped MS types on the basis of sharing alleles at 8 of the 9 markers analyzed.</p><p>To assess <italic>T. gondii</italic> isolates for clonality, MS alleles were combined with previously published DNA sequence analysis at three genetic loci, PCR-RFLP or DNA sequencing at 10 loci, and serologic analysis as described by Vaudaux et al. <xref ref-type="bibr" rid="pgen.1001261-Vaudaux1">[44]</xref>.</p><p>Statistical analyses were performed using GraphPad Prism 5 and &#967;<sup>2</sup> values were considered significant at P&#8202;=&#8202;0.05.</p>