<p>
<italic>Mycosphaerella graminicola</italic> isolates IPO323 and IPO94269 are Dutch field strains that were isolated in 1984 and 1994 from the wheat cultivar Arminda and an unknown cultivar, respectively. Isolate IPO95052 was isolated from a durum (tetraploid) wheat sample from Algeria. All isolates are maintained at the CBS-KNAW Fungal Biodiversity Centre of the Royal Netherlands Academy of Arts and Sciences (Utrecht, the Netherlands) under accession numbers CBS 115943 (IPO323), CBS 115941 (IPO94269) and CBS 115942 (IPO95052). Mycelia of each isolate were used to inoculate 200 mL of YG broth (10 g of yeast extract and 30 g of glucose per L) and were cultured until cloudy by shaking at 120 rpm at 18&#176;C, after which the spores were lyophilized, 50 mg of lyophilised spores were placed in a 2-mL tube and ground with a Hybaid Ribolyser (model n&#176; FP120HY-230) for 10 s at 2500 rpm with a tungsten carbide bead. DNA was extracted using the Promega Wizard Magnetic DNA Purification system for food according to instructions provided by the manufacturer.</p><p>Whole-genome shotgun (WGS) sequencing of the genome of <italic>M. graminicola</italic> used three libraries with insert sizes of 2&#8211;3, 6&#8211;8, and 35&#8211;40 kb. The sequenced reads were screened for vector using cross_match, trimmed for vector and quality, and filtered to remove reads shorter than 100 bases. WGS assembly was done using Jazz, a tool developed at the JGI <xref ref-type="bibr" rid="pgen.1002070-Aparicio1">[44]</xref>. After excluding redundant and short scaffolds, the assembly v1.0 contained 41.2 Mb of sequence in 129 scaffolds, of which 4.0 Mb (7.5%) was in gaps (<xref ref-type="supplementary-material" rid="pgen.1002070.s022">Table S8</xref>). The sequence depth derived from the assembly was 8.88&#177;0.04.</p><p>To perform finishing, the <italic>M. graminicola</italic> WGS assembly was broken down into scaffold-size pieces and each piece was reassembled with phrap. These scaffold pieces were then finished using a Phred/Phrap/Consed pipeline. Initially, all low-quality regions and gaps were targeted with computationally selected sequencing reactions completed with 4&#8758;1 BigDye terminator: dGTP chemistry (Applied Biosystems). These automated rounds included resequencing plasmid subclones and walking on plasmid subclones or fosmids using custom primers.</p><p>Following completion of the automated rounds, a trained finisher manually inspected each assembly. Further reactions were then manually selected to complete the genome. These included additional resequencing reactions and custom primer walks on plasmid subclones or fosmids as described above guided by a genetic map of more than 2,031 sequenced markers plus paired-end reads from a library of Bacterial Artificial Chromosome clones. Smaller repeats in the sequence were resolved by transposon-hopping 8-kb plasmid clones. Fosmid and BAC clones were shotgun sequenced and finished to fill large gaps, resolve larger repeats and to extend into the telomere regions.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Each assembly was then validated by an independent quality assessment. This included a visual examination of subclone paired ends using Orchid (<ext-link ext-link-type="uri" ns0:href="http://www-hagsc.org">http://www-hagsc.org</ext-link>), and visual inspection of high-quality discrepancies and all remaining low-quality areas. All available EST resources were also placed on the assembly to ensure completeness. The finished genome consists of 39,686,251 base pairs of finished sequence with an estimated error rate of less than 1 in 100,000 base pairs. Genome contiguity is very high with a total of 21 chromosomes represented, 19 of which are complete and 20 of which are sequenced from telomere to telomere.</p><p>Both draft (v1.0) and finished (v2.0) assemblies of <italic>M. graminicola</italic> were processed using the JGI annotation pipeline, which combines several gene predictors:1) putative full-length genes from EST cluster consensus sequences; 2) homology-based gene models were predicted using FGENESH+ <xref ref-type="bibr" rid="pgen.1002070-Salamov1">[45]</xref> and Genewise <xref ref-type="bibr" rid="pgen.1002070-Birney1">[46]</xref> seeded by Blastx alignments against sequences from the NCBI non-redundant protein set; 3) <italic>ab initio</italic> gene predictor FGENESH <xref ref-type="bibr" rid="pgen.1002070-Salamov1">[45]</xref> was trained on the set of putative full-length genes and reliable homology-based models. Genewise models were completed using scaffold data to find start and stop codons. ESTs were used to extend, verify, and complete the predicted gene models. Because multiple gene models per locus were often generated, a single representative gene model for each locus was chosen based on homology and EST support and used for further analysis. Those comprised a filtered set of gene models supported by different lines of evidence. These were further curated manually during community annotation and used for analysis.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All predicted gene models were annotated using InterProScan <xref ref-type="bibr" rid="pgen.1002070-Zdobnov1">[47]</xref> and hardware-accelerated double-affine Smith-Waterman alignments (<ext-link ext-link-type="uri" ns0:href="http://www.timelogic.com">www.timelogic.com</ext-link>) against the SwissProt (<ext-link ext-link-type="uri" ns0:href="http://www.expasy.org/sprot">www.expasy.org/sprot</ext-link>) and other specialized databases such as KEGG <xref ref-type="bibr" rid="pgen.1002070-Kanehisa1">[48]</xref>. Finally, KEGG hits were used to map EC numbers (<ext-link ext-link-type="uri" ns0:href="http://www.expasy.org/enzyme/">http://www.expasy.org/enzyme/</ext-link>), and Interpro hits were used to map GO terms <xref ref-type="bibr" rid="pgen.1002070-Ashburner1">[49]</xref>. Predicted proteins also were annotated according to KOG <xref ref-type="bibr" rid="pgen.1002070-Tatusov1">[50]</xref>, <xref ref-type="bibr" rid="pgen.1002070-Koonin1">[51]</xref> classification.</p><p>Following the machine annotation, manual validation and correction of selected gene sets was performed by more than 30 annotators through a jamboree held at the JGI facilities in Walnut Creek, California, USA. Annotators were trained by JGI staff and continue to make modifications as necessary.</p><p>Potential microRNA-like small RNA (milRNAs) loci were annotated using the INFERNAL software tool and based on 454 microRNA families (covarion models) from the RFAM database version 9.1 <xref ref-type="bibr" rid="pgen.1002070-GriffithsJones1">[52]</xref>. milRNAs were predicted if their scores were higher than thresholds, defined individually for each family, in the same way as PFAM domains are predicted.</p><p>Experimental validation of the predicted milRNAs was done by sequencing of an RNA library Total RNA was isolated from spores germinated on water agar of <italic>M. graminicola</italic> isolate IPO323. A small RNA library was prepared according to the protocol for Illumina sequencing; small RNAs from 16&#8211;&#8764;50 nt were isolated from gels, sequenced with an Illumina/Solexa single read DNA 50 cycles Genome Analyzer II, and compared by BLAST search against the list of 535 predicted pre-milRNAs from the genome sequence.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Assembly and annotations of the <italic>M. graminicola</italic> finished genome are available from the JGI Genome Portal at <ext-link ext-link-type="uri" ns0:href="http://www.jgi.doe.gov/Mgraminicola">http://www.jgi.doe.gov/Mgraminicola</ext-link> and were deposited at DDBJ/EMBL/GenBank under the project accession ACPE00000000.</p><p>Whole-genome tiling microarrays were designed by choosing one 50-mer primer every 100 bases spanning the entire finished genome. The arrays were manufactured and hybridized by the Nimblegen Corporation with total DNA extracted from each field isolate.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">The CodonW package (<ext-link ext-link-type="uri" ns0:href="http://codonw.sourceforge.net/">http://codonw.sourceforge.net/</ext-link>) was used for correspondence analysis of codon usage, which mathematically is identical to principal component analysis. CodonW requires as an input a set of coding sequences, usually of individual genes. For chromosome-level analyses coding sequences from the frozen gene catalog models for each chromosomes were concatenated, forming 21 &#8216;superORFs&#8217;, one for each chromosome. Because partial models may introduce some potential frameshifts with internal stop codons they were removed from the analysis; this did not affect the results as their total number is low. CodonW has no graphical outputs, so they were used as inputs for scatter plots in R (<ext-link ext-link-type="uri" ns0:href="http://www.r-project.org/">http://www.r-project.org/</ext-link>).</p><p>For <italic>M. graminicola</italic> only a similar analysis was done for repeats. RepeatScout was run on the genome to produce a set of <italic>ab initio</italic>-identified repeat sequences. From that set 81 distinct repeat sequences, each with an occurrence exceeding 20 times in the genome, were extracted. For each chromosome a vector of length 81 was calculated with the relative frequency of each repeat. A PC analysis was run on the resulting vectors using the standard principal component function pcomp in R. Separation at the repeat level means that these chromosomes have distinct evolutionary profiles not only on the protein-coding level, but also on other parts of the chromosomes, suggesting that entire chromosomes may be transferred horizontally.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Dot plots were generated via MUMMER 3.0 <xref ref-type="bibr" rid="pgen.1002070-Kurtz1">[53]</xref> with data derived from default PROmer comparisons between the <italic>M. graminicola</italic> genome assembly versions 1 and 2 (<ext-link ext-link-type="uri" ns0:href="http://genome.jgi-psf.org/Mycgr3/Mycgr3.home.html">http://genome.jgi-psf.org/Mycgr3/Mycgr3.home.html</ext-link>) and <italic>S. nodorum</italic> SN15 assembly version 2 <xref ref-type="bibr" rid="pgen.1002070-Hane1">[54]</xref>, available under GenBank accessions CH445325&#8211;CH445384, CH445386&#8211;CH445394 and CH959328&#8211;CH959365, or AAGI00000000. Additional comparisons and statistical analyses were made with custom-designed perl scripts.</p><p>Data from the <italic>M. graminicola</italic> version 2 comparison with <italic>S. nodorum</italic> were used to test the efficacy of mesosyntenic comparisons to assist the completion of fungal genomes. The mesosynteny-based prediction of scaffold joining involved 3 stages: determining the percent coverage of scaffolds/chromosomes for each scaffold/chromosome pair (i.e., a function of the number of &#8216;dots&#8217; per &#8216;block&#8217;); determining which scaffold/chromosome pairs were significantly related and forming groups of joined scaffolds; and filtering out background levels of similarity due to sequence redundancy and incomplete genome assemblies.</p><p>Coordinates of homologous regions were obtained from PROmer coordinate outputs (MUMMER 3.0) and used to determine the percent of sequence covered by matches to a sequence from the alternate genome for each sequence pair. Where match coordinates overlapped on the sequence of interest, those matches were merged into a single feature to avoid redundancy. A perl script for conversion of PROmer coordinate outputs to a table of percent coverage is available on request.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Coverage values for each <italic>M. graminicola</italic>-<italic>S. nodorum</italic> sequence pair were subject to a binomial test for significance. The threshold for significance (<italic>Psig</italic>)&#8805;0.95 was:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002070.e001.jpg" /></disp-formula>where <italic>x</italic> is the percent coverage, <italic>n</italic> equals 100, and <italic>p</italic> is the probability of chromosome homology.</p><p>The probability of chromosome homology (<italic>p</italic>) was equal to 1/(21&#215;19), which was derived from the number of <italic>M. graminicola</italic> chromosomes (21) and the approximate PFGE estimate of <italic>S. nodorum</italic> chromosomes (19) <xref ref-type="bibr" rid="pgen.1002070-Cooley1">[55]</xref>. This is the likelihood that a given sequence pair represents related chromosomes. This model assumes that no whole-genome/chromosome duplication events have occurred previously between either fungal genome since divergence from their last common ancestor.</p><p>The significance of percent coverage (<italic>Psig</italic>) was tested bidirectionally for each sequence pair (i.e., for sequence pair A&#8211;B, both coverage of sequence A by B and coverage of sequence B by A were tested). Sequence pairs were significantly related if a test in either direction was successful. A minimum length threshold of 1 kb was also imposed for both sequences. Where multiple scaffolds of <italic>M. graminicola</italic> were significantly related to the same <italic>S. nodorum</italic> scaffold, those <italic>M. graminicola</italic> scaffolds formed a &#8216;joined group&#8217; of candidates for representation of the same chromosome.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">All possible paired combinations of <italic>M. graminicola</italic> scaffolds present within predicted joined groups were subject to filtering for high levels of background similarity as follows:<disp-formula><graphic mimetype="image" position="float" ns0:href="pgen.1002070.e002.jpg" /></disp-formula>The retention score is a measure of the reliability of scaffold join relationships. Joins between <italic>M. graminicola</italic> scaffold pairs with retention scores &lt;0.25 were discarded.</p><p>Annotation of carbohydrate-related enzymes was performed using the Carbohydrate-Active Enzyme database (CAZy) annotation pipeline <xref ref-type="bibr" rid="pgen.1002070-Dean1">[26]</xref>. BLAST was used to compare the predicted proteins of <italic>M. graminicola</italic> to a collection of catalytic and carbohydrate-binding modules derived from CAZy. Significant hits were compared individually by BLAST to assign them to one or more CAZy families. Ambiguous family attributions were processed manually along with all identified models that presented defects (deletions, insertions, splicing issues, etc.).</p><p>Growth profiling of <italic>S. nodorum</italic> and <italic>M. graminicola</italic> was on <italic>Aspergillus niger</italic> minimal medium <xref ref-type="bibr" rid="pgen.1002070-deVries1">[56]</xref>. Cultures were grown at 25 degrees for seven days after which pictures were taken for growth comparison. Carbon sources used were: glucose (Sigma); soluble starch (Difco); alpha-cellulose (Sigma); Guar Gum (Sigma, galactomannan); Oat spelt xylan (Sigma); and Apple Pectin (Sigma).</p><p>Comparisons of sequence content between core and dispensable chromosomes was with Circos <xref ref-type="bibr" rid="pgen.1002070-Krzywinski1">[57]</xref>. This tool draws ribbons connecting sequences that align in different data sets.</p>