<p>Clinical and laboratory studies were approved by the NHGRI IRB. Patients and family members who were enrolled in the clinical protocol gave written informed consent, specifically including WES.</p><p>Sanger sequencing of the following genes was within normal limits: <italic>SPG7</italic>, <italic>CLN8</italic>, <italic>COX10</italic>, <italic>DLD</italic>, <italic>EPM1</italic>, <italic>EPM2A</italic>, <italic>EPM2B</italic>, <italic>PANK2</italic>, <italic>PLA2G6</italic>, <italic>PRICKLE1</italic>, <italic>SCO1</italic>, <italic>SCO2</italic>, <italic>SURF1</italic>, as was an analysis of mtDNA for point mutations, deletions and duplications. A lysosomal enzyme panel and buffy coat electron microscopy were also normal.</p><p>Genomic DNA was extracted from peripheral whole blood, using the Gentra Puregene Blood kit (Qiagen) per manufacturer's standards. An additional chloroform-phenol extraction step was carried out to neutralize infectious agents.</p><p>Illumina HumanOmni1-Quad genotyping arrays were run for one affected child and parents. The deceased brother's DNA was not analyzed on the SNP chip due to sample limitations. Analyses including homozygosity mapping were carried out using Illumina GenomeStudio Software. SNP chip data were also used to verify exome sample IDs for quality control purposes, and to calculate sensitivity and specificity for genotype calling in exome sequence data.</p><p>Solution hybridization exome capture was carried out using the Sureselect Human All exon System (Agilent Technologies, Santa Clara, CA). This technique uses biotinylated RNA baits to hybridize to sequences that correspond to exons <xref ref-type="bibr" rid="pgen.1002325-Gnirke1">[26]</xref>. Manufacturer's protocol version 1.0 compatible with Illumina paired-end sequencing was used, with the exception that DNA fragment size and quality was measured using a 2% agarose gel stained with Sybr Gold instead of using an Agilent Bioanalyser. The manufacturer's specifications state that the capture regions total approximately 38 Mb. This kit covers the 1.22% of the human genome corresponding to the Consensus Conserved Domain Sequences database (CCDS) and greater than 1000 non-coding RNAs. Flowcell preparation and 76bp paired end read sequencing were carried out as per protocol for the GAIIx sequencer (Illumina Inc, San Diego CA)<xref ref-type="bibr" rid="pgen.1002325-Bentley1">[27]</xref>. Approximately two lanes on a GAIIx flowcell were used per exome sample to generate sufficient reads to generate the aligned sequence.</p><p>Image analysis and base calling on all lanes of data were performed using Illumina Genome Analyzer Pipeline software (GAPipeline versions 1.4.0 or greater) with default parameters. The complexity of sequencing libraries was between 97.5%&#8211;99.3%. Complexity is the percentage of unique molecules determined from the first 25 bp in single-fragment reads and the combination of the first 25 bp in both paired-end reads out of total number of reads where the match chromosome field is not &#8220;QC&#8221; and the first 25 bp have no &#8216;N&#8217;s.</p><p xmlns:ns0="http://www.w3.org/1999/xlink">Reads were aligned to a human reference sequence (UCSC assembly hg18, NCBI build 36) using the package called &#8220;efficient large-scale alignment of nucleotide databases&#8221; (ELAND). Reads that aligned uniquely were grouped into genomic sequence intervals of about 100 kb, and reads that failed to align were binned with their paired-end mates. Reads in each bin were subjected to a Smith-Waterman-based local alignment algorithm, <italic>cross_match</italic> using the parameters&#8211;minscore 21 and &#8211;mask level 0 to their respective 100kb genomic sequence (<ext-link ext-link-type="uri" ns0:href="http://www.phrap.org">http://www.phrap.org</ext-link>). A total of 3.04&#8211;3.53 Gb of high-confidence mappable sequence data were generated in autosomal targeted regions per individual (<xref ref-type="supplementary-material" rid="pgen.1002325.s002">Table S1</xref>). Genotypes were called at all positions where there were high-quality sequence bases (Phred-like Q20 or greater) using a Bayesian algorithm (Most Probable Genotype &#8211; MPG; Nancy F Hansen, unpublished observations). The genotype calls were compared against Illumina Human 1M-Quad genotype chips. Genotypes with a MPG score of 10 or greater show &gt;99.89% concordance with SNP Chip data. The targeted regions included the exons of 17,134 genes and total 36,025,890 bases in the human genome. We successfully sequenced 92%&#8211;94% of targeted regions to an average depth per individual of 84&#8211;98 fold redundancy on the autosomes, 90%&#8211;94% of targeted regions and 57-93 fold redundancy on the X-chromosomes and 66%&#8211;68% of targeted regions and 45-51 fold redundancy on the Y chromosome in males. Of the total targeted sequence, 24&#8211;25 Mb of the genome corresponded to protein coding exons as per UCSC known gene annotation (<xref ref-type="supplementary-material" rid="pgen.1002325.s003">Table S2</xref>). This resulted in capturing and sequencing of 79%&#8211;81% of the exome as defined by UCSC known gene annotations. Our custom protein variation annotation pipeline annotated an average of 8,585 missense and 87 termination changes per individual. Our annotation of cSNVs (coding single nucleotide variants) was based on UCSC all known genes. A number of filtering and prioritization steps were applied to reduce the number and to identify potentially pathogenic mutations, similar to the methods used in previous studies <xref ref-type="bibr" rid="pgen.1002325-WoodAllum1">[28]</xref>, <xref ref-type="bibr" rid="pgen.1002325-Ng2">[29]</xref>. Missense variants were sorted by the degree of severity of functional disruption prediction using CDPred. Variants detected in dbSNP (version 130) and in eight HapMap exome data were excluded from being potential disease causing cSNVs <xref ref-type="bibr" rid="pgen.1002325-Ng2">[29]</xref>. Imposing the requirement to satisfy all of the above three filters for severity of amino acid substitution, and not being a common variant resulted in 483-636 potential protein affecting variants (<xref ref-type="supplementary-material" rid="pgen.1002325.s003">Table S2</xref>). Based on the inheritance pattern of the disease we required the variants to follow a Mendelian autosomal recessive pattern of inheritance, the homozygous subset of which yielded two potential genes (<italic>AFG3L2, DMGDH</italic>) (<xref ref-type="supplementary-material" rid="pgen.1002325.s004">Table S3</xref>).</p><p>Wild-type or mutant variants of mature human AFG3L2 and human SPG7 were fused to the mitochondrial targeting sequence of the yeast <italic>m</italic>-AAA protease subunit Yta10 <xref ref-type="bibr" rid="pgen.1002325-Bonn1">[17]</xref>. The human <italic>AFG3L2</italic> gene was mutagenized in yeast expression constructs using the QuikChange XL Site-Directed Mutagenesis Kit (Agilent). Mutations were verified by DNA sequencing. Mutant variants were co-expressed from the multicopy vector YEplac111 under the control of the <italic>ADH1</italic> promoter in &#916;<italic>yta10</italic>&#916;<italic>yta12</italic> yeast cells (YKO200)<xref ref-type="bibr" rid="pgen.1002325-Koppen2">[30]</xref>. Yeast cells were grown under standard growth conditions at 30&#176;C either in YP medium [1% (w/v) yeast extract, 2% (w/v) peptone] or minimal medium [0.67% yeast nitrogen base, 0.15% amino acid mix] supplemented with 40 &#181;g/ml of adenine and tryptophan, 20 &#181;g/ml of histidine and uracile, 60 &#181;g/ml of leucine, and 30 &#181;g/ml lysine in different combinations, both containing 2% (w/v) glucose or for isolation of mitochondria 2% (w/v) galactose and 0.5% (w/v) lactate. To test for respiratory chain activity, yeast cells were grown on YP medium containing 3% (w/v) glycerol as the sole carbon source. Expression of the human <italic>m</italic>-AAA protease subunits and processing of MrpL32 was assessed by immunoblotting using <italic>m</italic>-AAA protease and MrpL32-specific antibodies.</p><p>The following procedures were performed as described previously: Yeast mitochondria isolation <xref ref-type="bibr" rid="pgen.1002325-Tatsuta2">[31]</xref>; yeast BN-PAGE analysis <xref ref-type="bibr" rid="pgen.1002325-Wittig1">[32]</xref>. BN-PAGE on human fibroblasts was performed as previously described using digitonin as detergent (detergent to protein ratio was 2&#8758;1 [w/w]) <xref ref-type="bibr" rid="pgen.1002325-Klement1">[33]</xref>.</p><p>(<xref ref-type="supplementary-material" rid="pgen.1002325.s001">Figure S1</xref>) The coding region of murine <italic>Afg3l2</italic> (GenBank/EMBL/DDBJ accession no. NM_027130) was cloned into the SfiI-EcoRV sites of the vector pcDNA3.1 in frame with HA. The Y615C mutation (corresponding to the human Y616C) was introduced by site-directed mutagenesis.</p><p>To assess targeting of Afg3l2-HA to mitochondria, HeLa cells were co-transfected with a mitochondrially-targeted variant of GFP (Su9-GFP) and Afg3l2-WT or Afg3l2-Y615C constructs using Lipofectamine 2000 (Invitrogen). After 48 hours, cells were collected, fixed with PFA4%/PBS and immunofluorescence was performed as previously described (Errico et al. 2000). AFG3L2 was detected using specific antibodies (1&#8758;200). Immunofluorescences were examined and images acquired using an Axioplan microscope equipped with an Apotome system (Zeiss).</p>